{
  "cells": [
    {
      "cell_type": "markdown",
      "source": [
        "# **SaprotHub: Democratizing Protein Language Model Training, Sharing and Collaboration for the Biology Community**\n",
        "# <font color=\"red\">Note: Google Colab python library has been updated. We are currently updating ColabSaprot and expect to complete this process in 2-3 days. (2025/03/18)<font color=\"red\">\n",
        "\n",
        "<a href=\"https://www.biorxiv.org/content/10.1101/2024.05.24.595648v3\"><img src=\"https://img.shields.io/badge/Paper-bioRxiv-green\" style=\"max-width: 100%;\"></a>\n",
        "<a href=\"https://huggingface.co/SaProtHub\"><img src=\"https://img.shields.io/badge/%F0%9F%A4%97%20Hugging%20Face-red?label=SaprotHub\" style=\"max-width: 100%;\"></a>\n",
        "<a href=\"https://github.com/westlake-repl/SaprotHub\"><img src=\"https://img.shields.io/badge/Github-black?logo=github\" style=\"max-width: 100%;\"></a>\n",
        "<a href=\"https://theopmc.github.io/\"><img src=\"https://img.shields.io/badge/Website-OPMC-yellow\" style=\"max-width: 100%;\"></a>\n",
        "<a href=\"https://cbirt.net/no-coding-required-saprothub-brings-protein-modeling-to-every-biologist/\" alt=\"blog\"><img src=\"https://img.shields.io/badge/Blog-Medium-purple\" /></a>\n",
        "<a href=\"https://x.com/sokrypton/status/1795525127653986415\"><img src=\"https://img.shields.io/badge/Twitter-blue?logo=twitter\" style=\"max-width: 100%;\"></a>\n",
        "\n",
        "\n",
        "- This is **ColabSaprot**, the Colab version of [SaProt](https://github.com/westlake-repl/SaProt) - a pre-trained protein language model designed for [various protein prediction tasks](https://github.com/westlake-repl/SaprotHub/blob/main/task_list.md).\n",
        "\n",
        "- We have built **SaprotHub** ([website](https://huggingface.co/SaProtHub), [paper](https://www.biorxiv.org/content/10.1101/2024.05.24.595648v4)), a repository for community-shared models. Explore them through our specialized [search engine](https://huggingface.co/spaces/SaProtHub/SaprotHub-search).\n",
        "\n",
        "- Visit and contribute to [OPMC](https://theopmc.github.io/).\n",
        "\n",
        "<font color=\"red\">‚ö†Ô∏è **New Notice**: Google Colab recommends Chrome browser. ColabSaprot may not work properly in Safari. Contact {sujin, yuanfajie}@westlake.edu.cn for any inquiries about ColabSaprot. We will try to respond within 24 hours.</font>"
      ],
      "metadata": {
        "id": "i7atSR7woQ-b"
      },
      "id": "i7atSR7woQ-b"
    },
    {
      "cell_type": "markdown",
      "source": [
        "# How to start (üé• [tutorial](https://www.youtube.com/watch?v=nmLtjlCI_7M))\n",
        "\n",
        "## Follow below steps to switch to GPU. Once connected, click the run-button <img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/run_button.png?raw=true' height='25px' width='25px' align='center'> to run ColabSaprot\n",
        "\n",
        "<img src=\"https://github.com/westlake-repl/SaProtHub/blob/main/Figure/Switch_Runtime_2.png?raw=true\" align=\"center\">\n",
        "\n",
        "### GPU selection guide\n",
        "\n",
        "- **T4 GPU:** <font color=\"red\">**Free to use**</font>. Suitable for low-compute tasks such as\n",
        "training Saprot 35M model, protein property prediction, mutational effect prediction and protein sequence design. <font color=\"red\">**Note this GPU is unstable so you may lose connection or have memory issues during long-time training. To avoid this problem you could switch to L4 or A100 GPU.**\n",
        "</font>\n",
        "\n",
        "- **L4 GPU (need Colab Pro):** It has larger memory. Could be used for training Saprot 650M\n",
        "model but is slower than A100 GPU.\n",
        "\n",
        "- **A100 GPU (need Colab Pro):** Much more powerful GPU. Suitable for training large\n",
        "model such as Saprot 650M model."
      ],
      "metadata": {
        "id": "paX3gluumu7J"
      },
      "id": "paX3gluumu7J"
    },
    {
      "cell_type": "code",
      "source": [
        "#@title **Click the run-button**\n",
        "\n",
        "#@markdown ### Hint:\n",
        "#@markdown - It takes 3-8 minutes for the initial installation.\n",
        "#@markdown - The run-button's \"inactive\" state (<img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/run_button.png?raw=true' height='25px' width='25px' align='center'>) indicates the program is on standby and awaits your click to start. When clicked, the run-button transforms to show a dynamic loading animation (<img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/run_button_working.png?raw=true' height='25px' width='25px' align='center'>), confirming the program is launching. Once the interface loads completely, you can begin using the program.\n",
        "\n",
        "#@markdown ### <font color=red> Important: </font>\n",
        "#@markdown - <font color=red>If changing runtime type or encountering connection issues, first click the run-button to stop the program. Once the run-button transforms from (<img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/run_button_working.png?raw=true' height='25px' width='25px' align='center'>) to (<img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/run_button.png?raw=true' height='25px' width='25px' align='center'>), you can reconnect and restart, see [here](https://github.com/westlake-repl/SaprotHub/wiki/SaprotHub-v2-(latest)#2-how-can-i-reconnect-when-encountering-connection-issues).</font>\n",
        "#@markdown - <font color=red>For any unknown issues, clear your session via Runtime > \"Manage Sessions\", see [here](https://github.com/westlake-repl/SaprotHub/wiki/SaprotHub-v2-(latest)#a-simple-way-to-handle-unexpected-issues).</font>\n",
        "################################################################################\n",
        "########################### install saprot #####################################\n",
        "################################################################################\n",
        "# %load_ext autoreload\n",
        "# %autoreload 2\n",
        "\n",
        "import os\n",
        "# Check whether the server is local or from google cloudA model id example\n",
        "root_dir = os.getcwd()\n",
        "\n",
        "from google.colab import output\n",
        "# output.enable_custom_widget_manager()\n",
        "\n",
        "\n",
        "try:\n",
        "  import sys\n",
        "  sys.path.append(f\"{root_dir}/SaprotHub\")\n",
        "  import saprot\n",
        "\n",
        "  os.system(f\"chmod +x {root_dir}/SaprotHub/bin/*\")\n",
        "\n",
        "  ################################################################################\n",
        "  ################################################################################\n",
        "  ################################## global ######################################\n",
        "  ################################################################################\n",
        "  ################################################################################\n",
        "\n",
        "  import ipywidgets\n",
        "  import pandas as pd\n",
        "  import torch\n",
        "  import numpy as np\n",
        "  import lmdb\n",
        "  import base64\n",
        "  import copy\n",
        "  import os\n",
        "  import json\n",
        "  import zipfile\n",
        "  import yaml\n",
        "  import argparse\n",
        "  import pprint\n",
        "  import subprocess\n",
        "  import py3Dmol\n",
        "  import matplotlib.pyplot as plt\n",
        "  import shutil\n",
        "  import torch.nn.functional as F\n",
        "  import warnings\n",
        "  warnings.filterwarnings(\"ignore\", category=FutureWarning)\n",
        "\n",
        "  from loguru import logger\n",
        "  from easydict import EasyDict\n",
        "  from colorama import init, Fore, Back, Style\n",
        "  from IPython.display import clear_output\n",
        "  from huggingface_hub import snapshot_download\n",
        "  from ipywidgets import HTML\n",
        "  from IPython.display import display\n",
        "  from google.colab import widgets\n",
        "  from google.colab import files\n",
        "  from pathlib import Path\n",
        "  from tqdm import tqdm\n",
        "  from datetime import datetime\n",
        "  from transformers import AutoTokenizer, EsmForProteinFolding, EsmTokenizer\n",
        "  from transformers.models.esm.openfold_utils.protein import to_pdb, Protein as OFProtein\n",
        "  from transformers.models.esm.openfold_utils.feats import atom14_to_atom37\n",
        "  from string import ascii_uppercase,ascii_lowercase\n",
        "  from saprot.utils.mpr import MultipleProcessRunnerSimplifier\n",
        "  from saprot.data.parse import get_chain_ids\n",
        "  from saprot.scripts.training import my_load_model\n",
        "  from safetensors import safe_open\n",
        "\n",
        "  print(\"SaProt is installed successfully!\")\n",
        "\n",
        "except Exception:\n",
        "  print(\"Installing SaProt...\")\n",
        "  os.system(f\"rm -rf {root_dir}/SaprotHub\")\n",
        "  # !rm -rf /content/SaprotHub/\n",
        "\n",
        "  !git clone https://github.com/westlake-repl/SaprotHub.git\n",
        "\n",
        "  # !pip install /content/SaprotHub/saprot-0.4.7-py3-none-any.whl\n",
        "  os.system(f\"pip install -r {root_dir}/SaprotHub/requirements.txt\")\n",
        "  # !pip install -r /content/SaprotHub/requirements.txt\n",
        "\n",
        "  os.system(f\"pip install {root_dir}/SaprotHub\")\n",
        "\n",
        "\n",
        "  os.system(f\"mkdir -p {root_dir}/SaprotHub/LMDB\")\n",
        "  os.system(f\"mkdir -p {root_dir}/SaprotHub/bin\")\n",
        "  os.system(f\"mkdir -p {root_dir}/SaprotHub/output\")\n",
        "  os.system(f\"mkdir -p {root_dir}/SaprotHub/datasets\")\n",
        "  os.system(f\"mkdir -p {root_dir}/SaprotHub/adapters/classification/Local\")\n",
        "  os.system(f\"mkdir -p {root_dir}/SaprotHub/adapters/regression/Local\")\n",
        "  os.system(f\"mkdir -p {root_dir}/SaprotHub/adapters/token_classification/Local\")\n",
        "  os.system(f\"mkdir -p {root_dir}/SaprotHub/adapters/pair_classification/Local\")\n",
        "  os.system(f\"mkdir -p {root_dir}/SaprotHub/adapters/pair_regression/Local\")\n",
        "  os.system(f\"mkdir -p {root_dir}/SaprotHub/structures\")\n",
        "\n",
        "  os.system(\"pip install jupyter_ui_poll\")\n",
        "\n",
        "  # !mkdir -p /content/SaprotHub/LMDB\n",
        "  # !mkdir -p /content/SaprotHub/bin\n",
        "  # !mkdir -p /content/SaprotHub/output\n",
        "  # !mkdir -p /content/SaprotHub/datasets\n",
        "  # !mkdir -p /content/SaprotHub/adapters/classification/Local\n",
        "  # !mkdir -p /content/SaprotHub/adapters/regression/Local\n",
        "  # !mkdir -p /content/SaprotHub/adapters/token_classification/Local\n",
        "  # !mkdir -p /content/SaprotHub/adapters/pair_classification/Local\n",
        "  # !mkdir -p /content/SaprotHub/adapters/pair_regression/Local\n",
        "  # !mkdir -p /content/SaprotHub/structures\n",
        "\n",
        "  # !pip install gdown==v4.6.3 --force-reinstall --quiet\n",
        "  # os.system(\n",
        "  #   f\"wget 'https://drive.usercontent.google.com/download?id=1B_9t3n_nlj8Y3Kpc_mMjtMdY0OPYa7Re&export=download&authuser=0' -O {root_dir}/SaprotHub/bin/foldseek\"\n",
        "  # )\n",
        "\n",
        "  os.system(f\"chmod +x {root_dir}/SaprotHub/bin/*\")\n",
        "  # !chmod +x /content/SaprotHub/bin/foldseek\n",
        "  import sys\n",
        "  sys.path.append(f\"{root_dir}/SaprotHub\")\n",
        "\n",
        "  # !mv /content/SaprotHub/ColabSaprotSetup/foldseek /content/SaprotHub/bin/\n",
        "\n",
        "################################################################################\n",
        "################################################################################\n",
        "################################## global ######################################\n",
        "################################################################################\n",
        "################################################################################\n",
        "# Overwrite modules to apply the updates\n",
        "\n",
        "\n",
        "from transformers import AutoTokenizer, EsmForProteinFolding, EsmTokenizer\n",
        "from transformers.models.esm.openfold_utils.protein import to_pdb, Protein as OFProtein\n",
        "from transformers.models.esm.openfold_utils.feats import atom14_to_atom37\n",
        "import ipywidgets\n",
        "import numpy as np\n",
        "import importlib\n",
        "importlib.reload(np)\n",
        "import pandas as pd\n",
        "import torch\n",
        "import lmdb\n",
        "import base64\n",
        "import copy\n",
        "import os\n",
        "import json\n",
        "import zipfile\n",
        "import yaml\n",
        "import argparse\n",
        "import pprint\n",
        "import subprocess\n",
        "import py3Dmol\n",
        "import matplotlib.pyplot as plt\n",
        "import shutil\n",
        "import torch.nn.functional as F\n",
        "import warnings\n",
        "warnings.filterwarnings(\"ignore\", category=FutureWarning)\n",
        "\n",
        "from loguru import logger\n",
        "from easydict import EasyDict\n",
        "from colorama import init, Fore, Back, Style\n",
        "from IPython.display import clear_output\n",
        "from huggingface_hub import snapshot_download\n",
        "from ipywidgets import HTML\n",
        "from IPython.display import display\n",
        "from google.colab import widgets\n",
        "from google.colab import files\n",
        "from pathlib import Path\n",
        "from tqdm import tqdm\n",
        "from datetime import datetime\n",
        "\n",
        "from string import ascii_uppercase,ascii_lowercase\n",
        "from saprot.utils.mpr import MultipleProcessRunnerSimplifier\n",
        "from saprot.data.parse import get_chain_ids\n",
        "from saprot.scripts.training import my_load_model\n",
        "from safetensors import safe_open\n",
        "\n",
        "DATASET_HOME = Path(f'{root_dir}/SaprotHub/datasets')\n",
        "ADAPTER_HOME = Path(f'{root_dir}/SaprotHub/adapters')\n",
        "STRUCTURE_HOME = Path(f\"{root_dir}/SaprotHub/structures\")\n",
        "LMDB_HOME = Path(f'{root_dir}/SaprotHub/LMDB')\n",
        "OUTPUT_HOME = Path(f'{root_dir}/SaprotHub/output')\n",
        "UPLOAD_FILE_HOME = Path(f'{root_dir}/SaprotHub/upload_files')\n",
        "FOLDSEEK_PATH = Path(f\"{root_dir}/SaprotHub/bin/foldseek\")\n",
        "aa_set = {\"A\", \"C\", \"D\", \"E\", \"F\", \"G\", \"H\", \"I\", \"K\", \"L\", \"M\", \"N\", \"P\", \"Q\", \"R\", \"S\", \"T\", \"V\", \"W\", \"Y\"}\n",
        "foldseek_struc_vocab = \"pynwrqhgdlvtmfsaeikc#\"\n",
        "\n",
        "data_type_list = [\"Single AA Sequence\",\n",
        "                  \"Single SA Sequence\",\n",
        "                  \"Single UniProt ID\",\n",
        "                  \"Single PDB/CIF Structure\",\n",
        "                  \"Multiple AA Sequences\",\n",
        "                  \"Multiple SA Sequences\",\n",
        "                  \"Multiple UniProt IDs\",\n",
        "                  \"Multiple PDB/CIF Structures\",\n",
        "                  \"SaprotHub Dataset\",\n",
        "                  \"A pair of AA Sequences\",\n",
        "                  \"A pair of SA Sequences\",\n",
        "                  \"A pair of UniProt IDs\",\n",
        "                  \"A pair of PDB/CIF Structures\",\n",
        "                  \"Multiple pairs of AA Sequences\",\n",
        "                  \"Multiple pairs of SA Sequences\",\n",
        "                  \"Multiple pairs of UniProt IDs\",\n",
        "                  \"Multiple pairs of PDB/CIF Structures\",]\n",
        "\n",
        "data_type_list_single = [\n",
        "    \"Single AA Sequence\",\n",
        "    \"Single SA Sequence\",\n",
        "    \"Single UniProt ID\",\n",
        "    \"Single PDB/CIF Structure\",\n",
        "    \"A pair of AA Sequences\",\n",
        "    \"A pair of SA Sequences\",\n",
        "    \"A pair of UniProt IDs\",\n",
        "    \"A pair of PDB/CIF Structures\",]\n",
        "\n",
        "data_type_list_multiple = [\n",
        "    \"Multiple AA Sequences\",\n",
        "    \"Multiple SA Sequences\",\n",
        "    \"Multiple UniProt IDs\",\n",
        "    \"Multiple PDB/CIF Structures\",\n",
        "    \"Multiple pairs of AA Sequences\",\n",
        "    \"Multiple pairs of SA Sequences\",\n",
        "    \"Multiple pairs of UniProt IDs\",\n",
        "    \"Multiple pairs of PDB/CIF Structures\",]\n",
        "\n",
        "task_type_dict = {\n",
        "  \"Protein-level Classification\": \"classification\",\n",
        "  \"Residue-level Classification\" : \"token_classification\",\n",
        "  \"Protein-level Regression\" : \"regression\",\n",
        "  \"Protein-protein Classification\": \"pair_classification\",\n",
        "  \"Protein-protein Regression\": \"pair_regression\",\n",
        "}\n",
        "model_type_dict = {\n",
        "  \"classification\" : \"saprot/saprot_classification_model\",\n",
        "  \"token_classification\" : \"saprot/saprot_token_classification_model\",\n",
        "  \"regression\" : \"saprot/saprot_regression_model\",\n",
        "  \"pair_classification\" : \"saprot/saprot_pair_classification_model\",\n",
        "  \"pair_regression\" : \"saprot/saprot_pair_regression_model\",\n",
        "}\n",
        "dataset_type_dict = {\n",
        "  \"classification\": \"saprot/saprot_classification_dataset\",\n",
        "  \"token_classification\" : \"saprot/saprot_token_classification_dataset\",\n",
        "  \"regression\": \"saprot/saprot_regression_dataset\",\n",
        "  \"pair_classification\" : \"saprot/saprot_pair_classification_dataset\",\n",
        "  \"pair_regression\" : \"saprot/saprot_pair_regression_dataset\",\n",
        "}\n",
        "training_data_type_dict = {\n",
        "  \"Single AA Sequence\": \"AA\",\n",
        "  \"Single SA Sequence\": \"SA\",\n",
        "  \"Single UniProt ID\": \"SA\",\n",
        "  \"Single PDB/CIF Structure\": \"SA\",\n",
        "  \"Multiple AA Sequences\": \"AA\",\n",
        "  \"Multiple SA Sequences\": \"SA\",\n",
        "  \"Multiple UniProt IDs\": \"SA\",\n",
        "  \"Multiple PDB/CIF Structures\": \"SA\",\n",
        "  \"SaprotHub Dataset\": \"SA\",\n",
        "  \"A pair of AA Sequences\": \"AA\",\n",
        "  \"A pair of SA Sequences\": \"SA\",\n",
        "  \"A pair of UniProt IDs\": \"SA\",\n",
        "  \"A pair of PDB/CIF Structures\": \"SA\",\n",
        "  \"Multiple pairs of AA Sequences\": \"AA\",\n",
        "  \"Multiple pairs of SA Sequences\": \"SA\",\n",
        "  \"Multiple pairs of UniProt IDs\": \"SA\",\n",
        "  \"Multiple pairs of PDB/CIF Structures\": \"SA\",\n",
        "}\n",
        "\n",
        "\n",
        "class font:\n",
        "    RED = '\\033[91m'\n",
        "    GREEN = '\\033[92m'\n",
        "    YELLOW = '\\033[93m'\n",
        "    BLUE = '\\033[94m'\n",
        "\n",
        "    BOLD = '\\033[1m'\n",
        "    UNDERLINE = '\\033[4m'\n",
        "\n",
        "    RESET = '\\033[0m'\n",
        "\n",
        "\n",
        "################################################################################\n",
        "############################### adapters #######################################\n",
        "################################################################################\n",
        "def get_adapters_list(task_type=None):\n",
        "\n",
        "    adapters_list = []\n",
        "\n",
        "    if task_type:\n",
        "      raise\n",
        "      # for file_path in (ADAPTER_HOME / task_type).glob('**/adapter_config.json'):\n",
        "        # adapters_list.append(file_path.relative_to(ADAPTER_HOME / task_type).parent)\n",
        "    else:\n",
        "      for file_path in ADAPTER_HOME.glob('Local/**/adapter_config.json'):\n",
        "        adapters_list.append(file_path.relative_to(ADAPTER_HOME).parent)\n",
        "\n",
        "    return adapters_list\n",
        "\n",
        "def adapters_text(adapters_list):\n",
        "  input = ipywidgets.Text(\n",
        "    value=None,\n",
        "    placeholder='Enter SaprotHub Model ID',\n",
        "    # description='Selected:',\n",
        "    disabled=False)\n",
        "  input.layout.width = '500px'\n",
        "  display(input)\n",
        "\n",
        "  return input\n",
        "\n",
        "def adapters_dropdown(adapters_list):\n",
        "  dropdown = ipywidgets.Dropdown(\n",
        "    # options=[f\"{adapter_path.parent.stem}/{adapter_path.stem}\" for index, adapter_path in enumerate(adapters_list)],\n",
        "    options=adapters_list,\n",
        "    value=None,\n",
        "    placeholder='Select a Local Model here',\n",
        "    # description='Selected:',\n",
        "    disabled=False)\n",
        "  dropdown.layout.width = '500px'\n",
        "\n",
        "  return dropdown\n",
        "\n",
        "def adapters_combobox(adapters_list):\n",
        "  combobox = ipywidgets.Combobox(\n",
        "    options=[f\"{adapter_path.parent.stem}/{adapter_path.stem}\" for index, adapter_path in enumerate(adapters_list)],\n",
        "    value=None,\n",
        "    placeholder='Enter SaprotHub Model repository id or select a Local Model here',\n",
        "    # description='Selected:',\n",
        "    disabled=False)\n",
        "  combobox.layout.width = '500px'\n",
        "  display(combobox)\n",
        "\n",
        "  return combobox\n",
        "\n",
        "def adapters_selectmultiple(adapters_list):\n",
        "  selectmulitiple = ipywidgets.SelectMultiple(\n",
        "  # options=[f\"{adapter_path.parent.stem}/{adapter_path.stem}\" for index, adapter_path in enumerate(adapters_list)],\n",
        "  options=adapters_list,\n",
        "  value=[],\n",
        "  #rows=10,\n",
        "  placeholder='Select multiple models',\n",
        "  # description='Fruits',\n",
        "  disabled=False,\n",
        "  layout={'width': '500px'})\n",
        "  display(selectmulitiple)\n",
        "\n",
        "  return selectmulitiple\n",
        "\n",
        "def adapters_textmultiple(adapters_list):\n",
        "  textmultiple = ipywidgets.Text(\n",
        "  value=None,\n",
        "  placeholder='Enter multiple SaprotHub Model IDs, separated by commas.',\n",
        "  # description='Fruits',\n",
        "  disabled=False,\n",
        "  layout={'width': '500px'})\n",
        "  display(textmultiple)\n",
        "\n",
        "  return textmultiple\n",
        "\n",
        "\n",
        "def select_adapter_from(task_type, use_model_from):\n",
        "  adapters_list = get_adapters_list(task_type)\n",
        "\n",
        "  if use_model_from == 'Trained by yourself on ColabSaprot':\n",
        "    return adapters_dropdown(adapters_list)\n",
        "\n",
        "  elif use_model_from == 'Shared by peers on SaprotHub':\n",
        "    print(Fore.BLUE+\"SaprotHub Model:\"+Style.RESET_ALL)\n",
        "    return adapters_text(adapters_list)\n",
        "\n",
        "  elif use_model_from == \"Saved in your local computer\":\n",
        "    print(Fore.BLUE+\"Click the button to upload the \\\"Model-<task_name>-<model_size>.zip\\\" file of your Model:\"+Style.RESET_ALL)\n",
        "    # 1. upload model.zip\n",
        "    if task_type:\n",
        "      adapter_upload_path = ADAPTER_HOME / task_type / \"Local\"\n",
        "    else:\n",
        "      adapter_upload_path = ADAPTER_HOME / \"Local\"\n",
        "\n",
        "    adapter_zip_path = upload_file(adapter_upload_path)\n",
        "    adapter_path = adapter_upload_path / adapter_zip_path.stem\n",
        "    # 2. unzip model.zip\n",
        "    with zipfile.ZipFile(adapter_zip_path, 'r') as zip_ref:\n",
        "        zip_ref.extractall(adapter_path)\n",
        "    os.remove(adapter_zip_path)\n",
        "    # 3. check adapter_config.json\n",
        "    adapter_config_path = adapter_path / \"adapter_config.json\"\n",
        "    assert adapter_config_path.exists(), f\"Can't find {adapter_config_path}\"\n",
        "\n",
        "    # # 4. move to correct folder\n",
        "    # num_labels, task_type = get_num_labels_and_task_type_by_adapter(adapter_path)\n",
        "    # shutil.move(adapter_path, ADAPTER_HOME / task_type)\n",
        "\n",
        "    return EasyDict({\"value\":  f\"Local/{adapter_zip_path.stem}\"})\n",
        "\n",
        "  elif use_model_from == \"Multi-models on ColabSaprot\":\n",
        "    # 1. select the list of adapters\n",
        "    print(Fore.BLUE+f\"Local Model ({task_type}):\"+Style.RESET_ALL)\n",
        "    print(Fore.BLUE+f\"Multiple values can be selected with \\\"shift\\\" and/or \\\"ctrl\\\" (or \\\"command\\\") pressed and mouse clicks or arrow keys.\"+Style.RESET_ALL)\n",
        "    return adapters_selectmultiple(adapters_list)\n",
        "\n",
        "  elif use_model_from == \"Multi-models on SaprotHub\":\n",
        "    # 1. enter the list of adapters\n",
        "    print(Fore.BLUE+f\"SaprotHub Model IDs, separated by commas ({task_type}):\"+Style.RESET_ALL)\n",
        "    return adapters_textmultiple(adapters_list)\n",
        "\n",
        "\n",
        "\n",
        "################################################################################\n",
        "########################### download dataset ###################################\n",
        "################################################################################\n",
        "def download_dataset(task_name):\n",
        "  import gdown\n",
        "  import tarfile\n",
        "\n",
        "  filepath = LMDB_HOME / f\"{task_name}.tar.gz\"\n",
        "  download_links = {\n",
        "    \"ClinVar\" : \"https://drive.google.com/uc?id=1Le6-v8ddXa1eLJZFo7HPij7NhaBmNUbo\",\n",
        "    \"DeepLoc_cls2\" : \"https://drive.google.com/uc?id=1dGlojkCt1DwUXWiUk4kXRGRNu5sz2uxf\",\n",
        "    \"DeepLoc_cls10\" : \"https://drive.google.com/uc?id=1dGlojkCt1DwUXWiUk4kXRGRNu5sz2uxf\",\n",
        "    \"EC\" : \"https://drive.google.com/uc?id=1VFLFA-jK1tkTZBVbMw8YSsjZqAqlVQVQ\",\n",
        "    \"GO_BP\" : \"https://drive.google.com/uc?id=1DGiGErWbRnEK8jmE2Jpb996By8KVDBfF\",\n",
        "    \"GO_CC\" : \"https://drive.google.com/uc?id=1DGiGErWbRnEK8jmE2Jpb996By8KVDBfF\",\n",
        "    \"GO_MF\" : \"https://drive.google.com/uc?id=1DGiGErWbRnEK8jmE2Jpb996By8KVDBfF\",\n",
        "    \"HumanPPI\" : \"https://drive.google.com/uc?id=1ahgj-IQTtv3Ib5iaiXO_ASh2hskEsvoX\",\n",
        "    \"MetalIonBinding\" : \"https://drive.google.com/uc?id=1rwknPWIHrXKQoiYvgQy4Jd-efspY16x3\",\n",
        "    \"ProteinGym\" : \"https://drive.google.com/uc?id=1L-ODrhfeSjDom-kQ2JNDa2nDEpS8EGfD\",\n",
        "    \"Thermostability\" : \"https://drive.google.com/uc?id=1I9GR1stFDHc8W3FCsiykyrkNprDyUzSz\",\n",
        "  }\n",
        "\n",
        "  try:\n",
        "    gdown.download(download_links[task_name], str(filepath), quiet=False)\n",
        "    with tarfile.open(filepath, 'r:gz') as tar:\n",
        "      tar.extractall(path=str(LMDB_HOME))\n",
        "      print(f\"Extracted: {filepath}\")\n",
        "  except Exception as e:\n",
        "    raise RuntimeError(\"The dataset has not prepared.\")\n",
        "\n",
        "################################################################################\n",
        "############################# upload file ######################################\n",
        "################################################################################\n",
        "def upload_file(upload_path):\n",
        "  upload_path = Path(upload_path)\n",
        "  upload_path.mkdir(parents=True, exist_ok=True)\n",
        "  basepath = Path().resolve()\n",
        "  try:\n",
        "    uploaded = files.upload()\n",
        "    filenames = []\n",
        "    for filename in uploaded.keys():\n",
        "      filenames.append(filename)\n",
        "      shutil.move(basepath / filename, upload_path / filename)\n",
        "    if len(filenames) == 0:\n",
        "      logger.info(\"The uploading process has been interrupted by the user.\")\n",
        "      raise RuntimeError(\"The uploading process has been interrupted by the user.\")\n",
        "  except Exception as e:\n",
        "    logger.error(\"Upload file fail! Please click the button to run again.\")\n",
        "    raise(e)\n",
        "\n",
        "  return upload_path / filenames[0]\n",
        "\n",
        "################################################################################\n",
        "############################ upload dataset ####################################\n",
        "################################################################################\n",
        "\n",
        "def read_csv_dataset(uploaded_csv_path):\n",
        "  df = pd.read_csv(uploaded_csv_path)\n",
        "  df.columns = df.columns.str.lower()\n",
        "  return df\n",
        "\n",
        "def check_column_label_and_stage(csv_dataset_path):\n",
        "  df = read_csv_dataset(csv_dataset_path)\n",
        "  assert {'label', 'stage'}.issubset(df.columns), f\"Make sure your CSV dataset includes both `label` and `stage` columns!\\nCurrent columns: {df.columns}\"\n",
        "  column_values = set(df['stage'].unique())\n",
        "  assert all(value in column_values for value in ['train', 'valid', 'test']), f\"Ensure your dataset includes samples for all three stages: `train`, `valid` and `test`.\\nCurrent columns: {df.columns}\"\n",
        "\n",
        "def get_data_type(csv_dataset_path):\n",
        "  # AA, SA, Pair AA, Pair SA\n",
        "  df = read_csv_dataset(csv_dataset_path)\n",
        "\n",
        "  # AA, SA\n",
        "  if 'protein' in df.columns:\n",
        "    second_token = df.loc[0, 'protein'][1]\n",
        "    if second_token in aa_set:\n",
        "      return \"Multiple AA Sequences\"\n",
        "    elif second_token in foldseek_struc_vocab:\n",
        "      return \"Multiple SA Sequences\"\n",
        "    else:\n",
        "      raise RuntimeError(f\"The sequence in the dataset({csv_dataset_path}) are neither SA Sequences nor AA Sequences. Please check carefully.\")\n",
        "\n",
        "  # Pair AA, Pair SA\n",
        "  elif 'protein_1' in df.columns and 'protein_2' in df.columns:\n",
        "    second_token = df.loc[0, 'protein_1'][1]\n",
        "    if second_token in aa_set:\n",
        "      return \"Multiple pairs of AA Sequences\"\n",
        "    elif second_token in foldseek_struc_vocab:\n",
        "      return \"Multiple pairs of SA Sequences\"\n",
        "    else:\n",
        "      raise RuntimeError(f\"The sequence in the dataset({csv_dataset_path}) are neither SA Sequences nor AA Sequences. Please check carefully.\")\n",
        "\n",
        "  else:\n",
        "    print(df)\n",
        "    raise Exception(\"Please check your dataset format. You are expected to strictly follow the examples we provide.\")\n",
        "\n",
        "def check_task_type_and_data_type(original_task_type, data_type):\n",
        "  if \"Protein-protein\" in original_task_type:\n",
        "    assert data_type == \"SaprotHub Dataset\" or \"pair\" in data_type, f\"The current `data_type`({data_type}) is incompatible with the current `task_type`({original_task_type}). Please use Pair Sequence Datset for {original_task_type} task!\"\n",
        "  else:\n",
        "    assert \"pair\" not in data_type, f\"The current `data_type`({data_type}) is incompatible with the current `task_type`({original_task_type}). Please avoid using the Pair Sequence Dataset({data_type}) for the {original_task_type} task!\"\n",
        "\n",
        "def input_raw_data_by_data_type(data_type):\n",
        "\n",
        "  # 0-2. 0. Single AA Sequence, 1. Single SA Sequence, 2. Single UniProt ID\n",
        "  if data_type in data_type_list[:3]:\n",
        "    input_seq = ipywidgets.Text(\n",
        "      value=None,\n",
        "      placeholder=f'Enter {data_type} here',\n",
        "      disabled=False)\n",
        "    input_seq.layout.width = '500px'\n",
        "    print(Fore.BLUE+f\"{data_type}\"+Style.RESET_ALL)\n",
        "    display(input_seq)\n",
        "    return input_seq\n",
        "\n",
        "  # 3. Single PDB/CIF Structure\n",
        "  elif data_type == 'Single PDB/CIF Structure':\n",
        "    input_chain = ipywidgets.Text(\n",
        "      value=\"A\",\n",
        "      placeholder=f'Enter the name of chain here',\n",
        "      disabled=False)\n",
        "    input_chain.layout.width = '500px'\n",
        "    print(Fore.BLUE+\"Chain (to be extracted from the structure):\"+Style.RESET_ALL)\n",
        "    display(input_chain)\n",
        "\n",
        "    print(Fore.BLUE+\"Click to upload a .pdb/.cif file\"+Style.RESET_ALL)\n",
        "    pdb_file_path = upload_file(STRUCTURE_HOME)\n",
        "    print(input_chain)\n",
        "    return pdb_file_path.stem, input_chain\n",
        "\n",
        "  # 4-7 & 13-16. Multiple Sequences\n",
        "  elif data_type in data_type_list_multiple:\n",
        "    print(Fore.BLUE+f\"Please upload the .csv file which contains {data_type}\"+Style.RESET_ALL)\n",
        "    uploaded_csv_path = upload_file(UPLOAD_FILE_HOME)\n",
        "    print(Fore.BLUE+\"Successfully upload your .csv file!\"+Style.RESET_ALL)\n",
        "    print(\"=\"*100)\n",
        "\n",
        "    if data_type in ['Multiple PDB/CIF Structures', 'Multiple pairs of PDB/CIF Structures']:\n",
        "      # upload and unzip PDB files\n",
        "      print(Fore.BLUE+f\"Please upload your .zip file which contains {data_type} files\"+Style.RESET_ALL)\n",
        "      pdb_zip_path = upload_file(UPLOAD_FILE_HOME)\n",
        "      if pdb_zip_path.suffix != \".zip\":\n",
        "        logger.error(\"The data type does not match. Please click the run button again to upload a .zip file!\")\n",
        "        raise RuntimeError(\"The data type does not match.\")\n",
        "      print(Fore.BLUE+\"Successfully upload your .zip file!\"+Style.RESET_ALL)\n",
        "      print(\"=\"*100)\n",
        "\n",
        "      import zipfile\n",
        "      with zipfile.ZipFile(pdb_zip_path, 'r') as zip_ref:\n",
        "        zip_ref.extractall(STRUCTURE_HOME)\n",
        "\n",
        "    return uploaded_csv_path\n",
        "\n",
        "  # 8. SaprotHub Dataset\n",
        "  elif data_type == \"SaprotHub Dataset\":\n",
        "    input_repo_id = ipywidgets.Text(\n",
        "      value=None,\n",
        "      placeholder=f'Copy and paste the SaprotHub Dataset ID here',\n",
        "      disabled=False)\n",
        "    input_repo_id.layout.width = '500px'\n",
        "    print(Fore.BLUE+f\"{data_type}\"+Style.RESET_ALL)\n",
        "    display(input_repo_id)\n",
        "    return input_repo_id\n",
        "\n",
        "  # 9-11. A pair of seq\n",
        "  elif data_type in [\"A pair of AA Sequences\", \"A pair of SA Sequences\", \"A pair of UniProt IDs\"]:\n",
        "    print()\n",
        "\n",
        "    seq_type = data_type[len(\"A pair of \"):-1]\n",
        "\n",
        "    input_seq1 = ipywidgets.Text(\n",
        "      value=None,\n",
        "      placeholder=f'Enter the {seq_type} of Sequence 1 here',\n",
        "      disabled=False)\n",
        "    input_seq1.layout.width = '500px'\n",
        "    print(Fore.BLUE+f\"Sequence 1:\"+Style.RESET_ALL)\n",
        "    display(input_seq1)\n",
        "\n",
        "    input_seq2 = ipywidgets.Text(\n",
        "      value=None,\n",
        "      placeholder=f'Enter the {seq_type} of Sequence 2 here',\n",
        "      disabled=False)\n",
        "    input_seq2.layout.width = '500px'\n",
        "    print(Fore.BLUE+f\"Sequence 2:\"+Style.RESET_ALL)\n",
        "    display(input_seq2)\n",
        "\n",
        "    return (input_seq1, input_seq2)\n",
        "\n",
        "  # 12. Pair Single PDB/CIF Structure\n",
        "  elif data_type == 'A pair of PDB/CIF Structures':\n",
        "    print(\"Please provide the structure type, chain and your structure file.\")\n",
        "\n",
        "    dropdown_type1 = ipywidgets.Dropdown(\n",
        "      value=\"PDB\",\n",
        "      options=[\"PDB\", \"AF2\"],\n",
        "      disabled=False)\n",
        "    dropdown_type1.layout.width = '500px'\n",
        "    print(Fore.BLUE+\"The first structure type:\"+Style.RESET_ALL)\n",
        "    display(dropdown_type1)\n",
        "\n",
        "    input_chain1 = ipywidgets.Text(\n",
        "      value=\"A\",\n",
        "      placeholder=f'Enter the name of chain of the first structure here',\n",
        "      disabled=False)\n",
        "    input_chain1.layout.width = '500px'\n",
        "    print(Fore.BLUE+\"Chain of the first structure:\"+Style.RESET_ALL)\n",
        "    display(input_chain1)\n",
        "\n",
        "    print(Fore.BLUE+\"Please upload a .pdb/.cif file\"+Style.RESET_ALL)\n",
        "    pdb_file_path1 = upload_file(STRUCTURE_HOME)\n",
        "\n",
        "\n",
        "    dropdown_type2 = ipywidgets.Dropdown(\n",
        "      value=\"PDB\",\n",
        "      options=[\"PDB\", \"AF2\"],\n",
        "      disabled=False)\n",
        "    dropdown_type2.layout.width = '500px'\n",
        "    print(Fore.BLUE+\"The second structure type:\"+Style.RESET_ALL)\n",
        "    display(dropdown_type2)\n",
        "\n",
        "    input_chain2 = ipywidgets.Text(\n",
        "      value=\"A\",\n",
        "      placeholder=f'Enter the name of chain of the second structure here',\n",
        "      disabled=False)\n",
        "    input_chain2.layout.width = '500px'\n",
        "    print(Fore.BLUE+\"Chain of the second structure:\"+Style.RESET_ALL)\n",
        "    display(input_chain2)\n",
        "\n",
        "    print(Fore.BLUE+\"Please upload a .pdb/.cif file\"+Style.RESET_ALL)\n",
        "    pdb_file_path2 = upload_file(STRUCTURE_HOME)\n",
        "    return (pdb_file_path1.stem, dropdown_type1, input_chain1, pdb_file_path2.stem, dropdown_type2, input_chain2)\n",
        "\n",
        "def get_SA_sequence_by_data_type(data_type, raw_data):\n",
        "\n",
        "  # Multiple sequences\n",
        "  # raw_data = upload_files/xxx.csv\n",
        "\n",
        "  # 8. SaprotHub Dataset\n",
        "  if data_type == \"SaprotHub Dataset\":\n",
        "    input_repo_id = raw_data\n",
        "    REPO_ID = input_repo_id.value\n",
        "\n",
        "    if REPO_ID.startswith('/'):\n",
        "      return Path(REPO_ID)\n",
        "\n",
        "    snapshot_download(repo_id=REPO_ID, repo_type=\"dataset\", local_dir=DATASET_HOME / REPO_ID)\n",
        "    csv_dataset_path = DATASET_HOME / REPO_ID / 'dataset.csv'\n",
        "    assert csv_dataset_path.exists(), f\"Can't find {csv_dataset_path}\"\n",
        "    protein_df = read_csv_dataset(csv_dataset_path)\n",
        "\n",
        "    data_type = get_data_type(csv_dataset_path)\n",
        "\n",
        "    return get_SA_sequence_by_data_type(data_type, csv_dataset_path)\n",
        "\n",
        "    # # AA, SA\n",
        "    # if data_type == \"Multiple AA Sequences\":\n",
        "    #   for index, value in protein_df['sequence'].items():\n",
        "    #     sa_seq = ''\n",
        "    #     for aa in value:\n",
        "    #       sa_seq += aa + '#'\n",
        "    #     protein_df.at[index, 'sequence'] = sa_seq\n",
        "\n",
        "    # # Pair AA, Pair SA\n",
        "    # elif data_type in [\"Multiple pairs of AA Sequences\", \"Multiple pairs of SA Sequences\"]:\n",
        "    #   for i in ['1', '2']:\n",
        "    #     if data_type == \"Multiple pairs of AA Sequences\":\n",
        "    #       for index, value in protein_df[f'sequence_{i}'].items():\n",
        "    #         sa_seq = ''\n",
        "    #         for aa in value:\n",
        "    #           sa_seq += aa + '#'\n",
        "    #         protein_df.at[index, f'sequence_{i}'] = sa_seq\n",
        "\n",
        "    #     protein_df[f'name_{i}'] = f'name_{i}'\n",
        "    #     protein_df[f'chain_{i}'] = 'A'\n",
        "\n",
        "    # protein_df.to_csv(csv_dataset_path, index=None)\n",
        "\n",
        "    # return csv_dataset_path\n",
        "\n",
        "  elif data_type in data_type_list_multiple:\n",
        "    uploaded_csv_path = raw_data\n",
        "    csv_dataset_path = DATASET_HOME / uploaded_csv_path\n",
        "    protein_df = read_csv_dataset(uploaded_csv_path)\n",
        "\n",
        "    if 'pair' in data_type:\n",
        "      assert {'protein_1', 'protein_2'}.issubset(protein_df.columns), f\"The CSV dataset ({uploaded_csv_path}) must contain `protein_1` and `protein_2` columns. \\n Current columns:{protein_df.columns}\"\n",
        "    else:\n",
        "      assert 'protein' in protein_df.columns, f\"The CSV Dataset({uploaded_csv_path}) must contain a `protein` column. \\n Current columns:{protein_df.columns}\"\n",
        "\n",
        "    # 4. Multiple AA Sequences\n",
        "    if data_type == 'Multiple AA Sequences':\n",
        "      for index, value in protein_df['protein'].items():\n",
        "        sa_seq = ''\n",
        "        for aa in value:\n",
        "          sa_seq += aa + '#'\n",
        "        protein_df.at[index, 'protein'] = sa_seq\n",
        "\n",
        "      protein_df.to_csv(csv_dataset_path, index=None)\n",
        "      return csv_dataset_path\n",
        "\n",
        "    # 5. Multiple SA Sequences\n",
        "    elif data_type == 'Multiple SA Sequences':\n",
        "      protein_df.to_csv(csv_dataset_path, index=None)\n",
        "      return csv_dataset_path\n",
        "\n",
        "    # 6. Multiple UniProt IDs\n",
        "    elif data_type == 'Multiple UniProt IDs':\n",
        "      protein_list = protein_df.loc[:, 'protein'].tolist()\n",
        "      uniprot2pdb(protein_list)\n",
        "      protein_list = [(uniprot_id, \"AF2\", \"A\") for uniprot_id in protein_list]\n",
        "      mprs = MultipleProcessRunnerSimplifier(protein_list, pdb2sequence, n_process=2, return_results=True)\n",
        "      outputs = mprs.run()\n",
        "\n",
        "      protein_df['protein'] = [output.split(\"\\t\")[1] for output in outputs]\n",
        "      protein_df.to_csv(csv_dataset_path, index=None)\n",
        "      return csv_dataset_path\n",
        "\n",
        "    # 7. Multiple PDB/CIF Structures\n",
        "    elif data_type == 'Multiple PDB/CIF Structures':\n",
        "      # protein_list = [(uniprot_id, type, chain), ...]\n",
        "      # protein_list = [item.split('.')[0] for item in protein_df.iloc[:, 0].tolist()]\n",
        "      # uniprot2pdb(protein_list)\n",
        "      protein_list = []\n",
        "      for row_tuple in protein_df.itertuples(index=False):\n",
        "        assert row_tuple.type in ['PDB', 'AF2'],  \"The type of structure must be either \\\"PDB\\\" or \\\"AF2\\\"!\"\n",
        "        protein_list.append(row_tuple)\n",
        "      mprs = MultipleProcessRunnerSimplifier(protein_list, pdb2sequence, n_process=2, return_results=True)\n",
        "      outputs = mprs.run()\n",
        "\n",
        "      protein_df['protein'] = [output.split(\"\\t\")[1] for output in outputs]\n",
        "      protein_df.to_csv(csv_dataset_path, index=None)\n",
        "      return csv_dataset_path\n",
        "\n",
        "    # 13. Pair Multiple AA Sequences\n",
        "    elif data_type == \"Multiple pairs of AA Sequences\":\n",
        "      for i in ['1', '2']:\n",
        "        for index, value in protein_df[f'protein_{i}'].items():\n",
        "          sa_seq = ''\n",
        "          for aa in value:\n",
        "            sa_seq += aa + '#'\n",
        "          protein_df.at[index, f'protein_{i}'] = sa_seq\n",
        "\n",
        "        protein_df[f'name_{i}'] = f'name_{i}'\n",
        "        protein_df[f'chain_{i}'] = 'A'\n",
        "\n",
        "      protein_df.to_csv(csv_dataset_path, index=None)\n",
        "      return csv_dataset_path\n",
        "\n",
        "    # 14. Pair Multiple SA Sequences\n",
        "    elif data_type == \"Multiple pairs of SA Sequences\":\n",
        "      for i in ['1', '2']:\n",
        "        protein_df[f'name_{i}'] = f'name_{i}'\n",
        "        protein_df[f'chain_{i}'] = 'A'\n",
        "\n",
        "      protein_df.to_csv(csv_dataset_path, index=None)\n",
        "      return csv_dataset_path\n",
        "\n",
        "    # 15. Pair Multiple UniProt IDs\n",
        "    elif data_type == \"Multiple pairs of UniProt IDs\":\n",
        "      for i in ['1', '2']:\n",
        "        protein_list = protein_df.loc[:, f'protein_{i}'].tolist()\n",
        "        uniprot2pdb(protein_list)\n",
        "        protein_df[f'name_{i}'] = protein_list\n",
        "        protein_list = [(uniprot_id, \"AF2\", \"A\") for uniprot_id in protein_list]\n",
        "        mprs = MultipleProcessRunnerSimplifier(protein_list, pdb2sequence, n_process=2, return_results=True)\n",
        "        outputs = mprs.run()\n",
        "\n",
        "        protein_df[f'protein_{i}'] = [output.split(\"\\t\")[1] for output in outputs]\n",
        "        protein_df[f'chain_{i}'] = 'A'\n",
        "\n",
        "      protein_df.to_csv(csv_dataset_path, index=None)\n",
        "      return csv_dataset_path\n",
        "\n",
        "    elif data_type ==  \"Multiple pairs of PDB/CIF Structures\":\n",
        "      # columns: sequence_1, sequence_2, type_1, type_2, chain_1, chain_2, label, stage\n",
        "\n",
        "      # protein_list = [(uniprot_id, type, chain), ...]\n",
        "      # protein_list = [item.split('.')[0] for item in protein_df.iloc[:, 0].tolist()]\n",
        "      # uniprot2pdb(protein_list)\n",
        "\n",
        "      for i in ['1', '2']:\n",
        "        protein_list = []\n",
        "        for index, row in protein_df.iterrows():\n",
        "          assert row[f\"type_{i}\"] in ['PDB', 'AF2'],  \"The type of structure must be either \\\"PDB\\\" or \\\"AF2\\\"!\"\n",
        "          row_tuple = (row[f\"protein_{i}\"], row[f\"type_{i}\"], row[f\"chain_{i}\"])\n",
        "          protein_list.append(row_tuple)\n",
        "        mprs = MultipleProcessRunnerSimplifier(protein_list, pdb2sequence, n_process=2, return_results=True)\n",
        "        outputs = mprs.run()\n",
        "\n",
        "        # add name column, del type column\n",
        "        protein_df[f'name_{i}'] = protein_df[f'protein_{i}'].apply(lambda x: x.split('.')[0])\n",
        "        protein_df.drop(f\"type_{i}\", axis=1, inplace=True)\n",
        "        protein_df[f'protein_{i}'] = [output.split(\"\\t\")[1] for output in outputs]\n",
        "\n",
        "      # columns: name_1, name_2, chain_1, chain_2, sequence_1, sequence_2, label, stage\n",
        "      protein_df.to_csv(csv_dataset_path, index=None)\n",
        "      return csv_dataset_path\n",
        "\n",
        "  else:\n",
        "    # 0. Single AA Sequence\n",
        "    if data_type == 'Single AA Sequence':\n",
        "      input_seq = raw_data\n",
        "      aa_seq = input_seq.value\n",
        "\n",
        "      sa_seq = ''\n",
        "      for aa in aa_seq:\n",
        "          sa_seq += aa + '#'\n",
        "      return sa_seq\n",
        "\n",
        "    # 1. Single SA Sequence\n",
        "    elif data_type == 'Single SA Sequence':\n",
        "      input_seq = raw_data\n",
        "      sa_seq = input_seq.value\n",
        "\n",
        "      return sa_seq\n",
        "\n",
        "    # 2. Single UniProt ID\n",
        "    elif data_type == 'Single UniProt ID':\n",
        "      input_seq = raw_data\n",
        "      uniprot_id = input_seq.value\n",
        "\n",
        "\n",
        "      protein_list = [(uniprot_id, \"AF2\", \"A\")]\n",
        "      uniprot2pdb([protein_list[0][0]])\n",
        "      mprs = MultipleProcessRunnerSimplifier(protein_list, pdb2sequence, n_process=2, return_results=True)\n",
        "      seqs = mprs.run()\n",
        "      sa_seq = seqs[0].split('\\t')[1]\n",
        "      return sa_seq\n",
        "\n",
        "    # 3. Single PDB/CIF Structure\n",
        "    elif data_type == 'Single PDB/CIF Structure':\n",
        "      uniprot_id = raw_data[0]\n",
        "      chain = raw_data[1].value\n",
        "\n",
        "      protein_list = [(uniprot_id, chain)]\n",
        "      mprs = MultipleProcessRunnerSimplifier(protein_list, pdb2sequence, n_process=2, return_results=True)\n",
        "      seqs = mprs.run()\n",
        "      assert len(seqs)>0, \"Unable to convert to SA sequence. Please check the `type`, `chain`, and `.pdb/.cif file`.\"\n",
        "      sa_seq = seqs[0].split('\\t')[1]\n",
        "      return sa_seq\n",
        "\n",
        "    # 9. Pair Single AA Sequences\n",
        "    elif data_type == \"A pair of AA Sequences\":\n",
        "      input_seq_1, input_seq_2 = raw_data\n",
        "      sa_seq1 = get_SA_sequence_by_data_type('Single AA Sequence', input_seq_1)\n",
        "      sa_seq2 = get_SA_sequence_by_data_type('Single AA Sequence', input_seq_2)\n",
        "\n",
        "      return (sa_seq1, sa_seq2)\n",
        "\n",
        "    # 10. Pair Single SA Sequences\n",
        "    elif data_type ==  \"A pair of SA Sequences\":\n",
        "      input_seq_1, input_seq_2 = raw_data\n",
        "      sa_seq1 = get_SA_sequence_by_data_type('Single SA Sequence', input_seq_1)\n",
        "      sa_seq2 = get_SA_sequence_by_data_type('Single SA Sequence', input_seq_2)\n",
        "\n",
        "      return (sa_seq1, sa_seq2)\n",
        "\n",
        "    # 11. Pair Single UniProt IDs\n",
        "    elif data_type ==  \"A pair of UniProt IDs\":\n",
        "      input_seq_1, input_seq_2 = raw_data\n",
        "      sa_seq1 = get_SA_sequence_by_data_type('Single UniProt ID', input_seq_1)\n",
        "      sa_seq2 = get_SA_sequence_by_data_type('Single UniProt ID', input_seq_2)\n",
        "\n",
        "      return (sa_seq1, sa_seq2)\n",
        "\n",
        "    # 12. Pair Single PDB/CIF Structure\n",
        "    elif data_type == \"A pair of PDB/CIF Structures\":\n",
        "      uniprot_id1 = raw_data[0]\n",
        "      struc_type1 = raw_data[1].value\n",
        "      chain1 = raw_data[2].value\n",
        "\n",
        "      protein_list1 = [(uniprot_id1, struc_type1, chain1)]\n",
        "      mprs1 = MultipleProcessRunnerSimplifier(protein_list1, pdb2sequence, n_process=2, return_results=True)\n",
        "      seqs1 = mprs1.run()\n",
        "      sa_seq1 = seqs1[0].split('\\t')[1]\n",
        "\n",
        "      uniprot_id2 = raw_data[3]\n",
        "      struc_type2 = raw_data[4].value\n",
        "      chain2 = raw_data[5].value\n",
        "\n",
        "      protein_list2 = [(uniprot_id2, struc_type2, chain2)]\n",
        "      mprs2 = MultipleProcessRunnerSimplifier(protein_list2, pdb2sequence, n_process=2, return_results=True)\n",
        "      seqs2 = mprs2.run()\n",
        "      sa_seq2 = seqs2[0].split('\\t')[1]\n",
        "      return sa_seq1, sa_seq2\n",
        "\n",
        "\n",
        "\n",
        "\n",
        "################################################################################\n",
        "########################## Download predicted structures #######################\n",
        "################################################################################\n",
        "def uniprot2pdb(uniprot_ids, nprocess=20):\n",
        "  from saprot.utils.downloader import AlphaDBDownloader\n",
        "\n",
        "  os.makedirs(STRUCTURE_HOME, exist_ok=True)\n",
        "  af2_downloader = AlphaDBDownloader(uniprot_ids, \"pdb\", save_dir=STRUCTURE_HOME, n_process=20)\n",
        "  af2_downloader.run()\n",
        "\n",
        "\n",
        "\n",
        "################################################################################\n",
        "############### Form foldseek sequences by multiple processes ##################\n",
        "################################################################################\n",
        "# def pdb2sequence(process_id, idx, uniprot_id, writer):\n",
        "#   from saprot.utils.foldseek_util import get_struc_seq\n",
        "\n",
        "#   try:\n",
        "#     pdb_path = f\"{STRUCTURE_HOME}/{uniprot_id}.pdb\"\n",
        "#     cif_path = f\"{STRUCTURE_HOME}/{uniprot_id}.cif\"\n",
        "#     if Path(pdb_path).exists():\n",
        "#       seq = get_struc_seq(FOLDSEEK_PATH, pdb_path, [\"A\"], process_id=process_id)[\"A\"][-1]\n",
        "#     if Path(cif_path).exists():\n",
        "#       seq = get_struc_seq(FOLDSEEK_PATH, cif_path, [\"A\"], process_id=process_id)[\"A\"][-1]\n",
        "\n",
        "#     writer.write(f\"{uniprot_id}\\t{seq}\\n\")\n",
        "#   except Exception as e:\n",
        "#     print(f\"Error: {uniprot_id}, {e}\")\n",
        "\n",
        "# clear_output(wait=True)\n",
        "# print(\"Installation finished!\")\n",
        "\n",
        "def pdb2sequence(process_id, idx, row_tuple, writer):\n",
        "  file_path = row_tuple[0]\n",
        "  chain = row_tuple[1]\n",
        "  plddt_mask= True\n",
        "\n",
        "  from saprot.utils.foldseek_util import get_struc_seq\n",
        "  try:\n",
        "    seq = get_struc_seq(FOLDSEEK_PATH, file_path, [chain], process_id=process_id, plddt_mask=plddt_mask)[chain][-1]\n",
        "    writer.write(f\"{file_path}\\t{seq}\\n\")\n",
        "\n",
        "  except Exception as e:\n",
        "    raise e\n",
        "\n",
        "\n",
        "pymol_color_list = [\"#33ff33\",\"#00ffff\",\"#ff33cc\",\"#ffff00\",\"#ff9999\",\"#e5e5e5\",\"#7f7fff\",\"#ff7f00\",\n",
        "          \"#7fff7f\",\"#199999\",\"#ff007f\",\"#ffdd5e\",\"#8c3f99\",\"#b2b2b2\",\"#007fff\",\"#c4b200\",\n",
        "          \"#8cb266\",\"#00bfbf\",\"#b27f7f\",\"#fcd1a5\",\"#ff7f7f\",\"#ffbfdd\",\"#7fffff\",\"#ffff7f\",\n",
        "          \"#00ff7f\",\"#337fcc\",\"#d8337f\",\"#bfff3f\",\"#ff7fff\",\"#d8d8ff\",\"#3fffbf\",\"#b78c4c\",\n",
        "          \"#339933\",\"#66b2b2\",\"#ba8c84\",\"#84bf00\",\"#b24c66\",\"#7f7f7f\",\"#3f3fa5\",\"#a5512b\"]\n",
        "\n",
        "alphabet_list = list(ascii_uppercase+ascii_lowercase)\n",
        "\n",
        "\n",
        "def convert_outputs_to_pdb(outputs):\n",
        "\tfinal_atom_positions = atom14_to_atom37(outputs[\"positions\"][-1], outputs)\n",
        "\toutputs = {k: v.to(\"cpu\").numpy() for k, v in outputs.items()}\n",
        "\tfinal_atom_positions = final_atom_positions.cpu().numpy()\n",
        "\tfinal_atom_mask = outputs[\"atom37_atom_exists\"]\n",
        "\tpdbs = []\n",
        "\toutputs[\"plddt\"] *= 100\n",
        "\n",
        "\tfor i in range(outputs[\"aatype\"].shape[0]):\n",
        "\t\taa = outputs[\"aatype\"][i]\n",
        "\t\tpred_pos = final_atom_positions[i]\n",
        "\t\tmask = final_atom_mask[i]\n",
        "\t\tresid = outputs[\"residue_index\"][i] + 1\n",
        "\t\tpred = OFProtein(\n",
        "\t\t    aatype=aa,\n",
        "\t\t    atom_positions=pred_pos,\n",
        "\t\t    atom_mask=mask,\n",
        "\t\t    residue_index=resid,\n",
        "\t\t    b_factors=outputs[\"plddt\"][i],\n",
        "\t\t    chain_index=outputs[\"chain_index\"][i] if \"chain_index\" in outputs else None,\n",
        "\t\t)\n",
        "\t\tpdbs.append(to_pdb(pred))\n",
        "\treturn pdbs\n",
        "\n",
        "\n",
        "# This function is copied from ColabFold!\n",
        "def show_pdb(path, show_sidechains=False, show_mainchains=False, color=\"lddt\"):\n",
        "  file_type = str(path).split(\".\")[-1]\n",
        "  if file_type == \"cif\":\n",
        "    file_type == \"mmcif\"\n",
        "\n",
        "  view = py3Dmol.view(js='https://3dmol.org/build/3Dmol.js',)\n",
        "  view.addModel(open(path,'r').read(),file_type)\n",
        "\n",
        "  if color == \"lDDT\":\n",
        "    view.setStyle({'cartoon': {'colorscheme': {'prop':'b','gradient': 'roygb','min':50,'max':90}}})\n",
        "  elif color == \"rainbow\":\n",
        "    view.setStyle({'cartoon': {'color':'spectrum'}})\n",
        "  elif color == \"chain\":\n",
        "    chains = len(get_chain_ids(path))\n",
        "    for n,chain,color in zip(range(chains),alphabet_list,pymol_color_list):\n",
        "       view.setStyle({'chain':chain},{'cartoon': {'color':color}})\n",
        "\n",
        "  if show_sidechains:\n",
        "    BB = ['C','O','N']\n",
        "    view.addStyle({'and':[{'resn':[\"GLY\",\"PRO\"],'invert':True},{'atom':BB,'invert':True}]},\n",
        "                        {'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "    view.addStyle({'and':[{'resn':\"GLY\"},{'atom':'CA'}]},\n",
        "                        {'sphere':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "    view.addStyle({'and':[{'resn':\"PRO\"},{'atom':['C','O'],'invert':True}]},\n",
        "                        {'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "  if show_mainchains:\n",
        "    BB = ['C','O','N','CA']\n",
        "    view.addStyle({'atom':BB},{'stick':{'colorscheme':f\"WhiteCarbon\",'radius':0.3}})\n",
        "\n",
        "  view.zoomTo()\n",
        "  return view\n",
        "\n",
        "\n",
        "def plot_plddt_legend(dpi=100):\n",
        "  thresh = ['plDDT:','Very low (<50)','Low (60)','OK (70)','Confident (80)','Very high (>90)']\n",
        "  plt.figure(figsize=(1,0.1),dpi=dpi)\n",
        "  ########################################\n",
        "  for c in [\"#FFFFFF\",\"#FF0000\",\"#FFFF00\",\"#00FF00\",\"#00FFFF\",\"#0000FF\"]:\n",
        "    plt.bar(0, 0, color=c)\n",
        "  plt.legend(thresh, frameon=False,\n",
        "             loc='center', ncol=6,\n",
        "             handletextpad=1,\n",
        "             columnspacing=1,\n",
        "             markerscale=0.5,)\n",
        "  plt.axis(False)\n",
        "  return plt\n",
        "\n",
        "\n",
        "################################################################################\n",
        "###############   Download file to local computer   ##################\n",
        "################################################################################\n",
        "def file_download(path: str):\n",
        "  # Automatically download file if the server is from google cloud.\n",
        "  if root_dir == \"/content\":\n",
        "    btn = ipywidgets.Button(\n",
        "      description=\"Download File\",\n",
        "      button_style=\"success\",\n",
        "      )\n",
        "\n",
        "    btn.on_click(lambda b: files.download(path))\n",
        "    display(btn)\n",
        "\n",
        "    files.download(path)\n",
        "\n",
        "  else:\n",
        "    with open(path, \"rb\") as r:\n",
        "      res = r.read()\n",
        "\n",
        "    #FILE\n",
        "    filename = os.path.basename(path)\n",
        "    b64 = base64.b64encode(res)\n",
        "    payload = b64.decode()\n",
        "\n",
        "    #BUTTONS\n",
        "    html_buttons = '''<html>\n",
        "    <head>\n",
        "    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">\n",
        "    </head>\n",
        "    <body>\n",
        "    <a download=\"{filename}\" href=\"data:text/csv;base64,{payload}\" download>\n",
        "    <button class=\"p-Widget jupyter-widgets jupyter-button widget-button mod-warning\">Download File</button>\n",
        "    </a>\n",
        "    </body>\n",
        "    </html>\n",
        "    '''\n",
        "\n",
        "    html_button = html_buttons.format(payload=payload,filename=filename)\n",
        "    display(HTML(html_button))\n",
        "\n",
        "\n",
        "################################################################################\n",
        "############################ MODEL INFO #######################################\n",
        "################################################################################\n",
        "def get_base_model(adapter_path):\n",
        "  adapter_config = Path(adapter_path) / \"adapter_config.json\"\n",
        "  with open(adapter_config, 'r') as f:\n",
        "    adapter_config_dict = json.load(f)\n",
        "    base_model = adapter_config_dict['base_model_name_or_path']\n",
        "    if 'SaProt_650M_AF2' in base_model:\n",
        "      base_model = \"westlake-repl/SaProt_650M_AF2\"\n",
        "    elif 'SaProt_35M_AF2' in base_model:\n",
        "      base_model = \"westlake-repl/SaProt_35M_AF2\"\n",
        "    else:\n",
        "      raise RuntimeError(\"Please ensure the base model is \\\"SaProt_650M_AF2\\\" or \\\"SaProt_35M_AF2\\\"\")\n",
        "  return base_model\n",
        "\n",
        "def check_training_data_type(adapter_path, data_type):\n",
        "  metadata_path = Path(adapter_path) / \"metadata.json\"\n",
        "  if metadata_path.exists():\n",
        "    with open(metadata_path, 'r') as f:\n",
        "      metadata = json.load(f)\n",
        "      required_training_data_type = metadata['training_data_type']\n",
        "  else:\n",
        "    required_training_data_type = \"SA\"\n",
        "\n",
        "  if (required_training_data_type == \"AA\") and (\"AA\" not in data_type):\n",
        "    print(Fore.RED+f\"This model ({adapter_path}) is trained on {required_training_data_type} sequences, and predictions work better with AA sequences.\"+Style.RESET_ALL)\n",
        "    print(Fore.RED+f\"The current data type ({data_type}) includes structural information, which will not be used for predictions.\"+Style.RESET_ALL)\n",
        "    print()\n",
        "    print('='*100)\n",
        "  elif (required_training_data_type == \"SA\") and (\"AA\" in data_type):\n",
        "    print(Fore.RED+f\"This model ({adapter_path}) is trained on {required_training_data_type} sequences, and predictions work better with SA sequences.\"+Style.RESET_ALL)\n",
        "    print(Fore.RED+f\"The current data type ({data_type}) does not include structural information, which may lead to weak prediction performance.\"+Style.RESET_ALL)\n",
        "    print(Fore.RED+f\"If you only have the amino acid sequence, we strongly recommend using AF2 to predict the structure and generate a PDB file before prediction.\"+Style.RESET_ALL)\n",
        "    print()\n",
        "    print('='*100)\n",
        "\n",
        "  return required_training_data_type\n",
        "\n",
        "def mask_struc_token(sequence):\n",
        "    return ''.join('#' if i % 2 == 1 and char.islower() else char for i, char in enumerate(sequence))\n",
        "\n",
        "def get_num_labels_by_adapter(adapter_path):\n",
        "    adapter_path = Path(adapter_path)\n",
        "\n",
        "    if (adapter_path / 'adapter_model.safetensors').exists():\n",
        "        file_path = adapter_path / 'adapter_model.safetensors'\n",
        "        with safe_open(file_path, framework=\"pt\") as f:\n",
        "          if 'base_model.model.classifier.out_proj.bias' in f.keys():\n",
        "              tensor = f.get_tensor('base_model.model.classifier.out_proj.bias')\n",
        "          elif 'base_model.model.classifier.bias' in f.keys():\n",
        "              tensor = f.get_tensor('base_model.model.classifier.bias')\n",
        "          else:\n",
        "              raise KeyError(f\"Neither 'base_model.model.classifier.out_proj.bias' nor 'base_model.model.classifier.bias' found in the file({file_path}).\")\n",
        "\n",
        "    elif (adapter_path / 'adapter_model.bin').exists():\n",
        "      file_path = adapter_path / 'adapter_model.bin'\n",
        "      state_dict = torch.load(file_path)\n",
        "      if 'base_model.model.classifier.out_proj.bias' in state_dict.keys():\n",
        "        tensor = state_dict['base_model.model.classifier.out_proj.bias']\n",
        "      elif 'base_model.model.classifier.bias' in f.keys():\n",
        "        tensor = state_dict['base_model.model.classifier.bias']\n",
        "      else:\n",
        "        raise KeyError(f\"Neither 'base_model.model.classifier.out_proj.bias' nor 'base_model.model.classifier.bias' found in the file({file_path}).\")\n",
        "\n",
        "    else:\n",
        "        raise FileNotFoundError(f\"Neither 'adapter_model.safetensors' nor 'adapter_model.bin' found in the provided path({adapter_path}).\")\n",
        "\n",
        "    num_labels = list(tensor.shape)[0]\n",
        "    return num_labels\n",
        "\n",
        "def get_num_labels_and_task_type_by_adapter(adapter_path):\n",
        "    adapter_path = Path(adapter_path)\n",
        "\n",
        "    task_type = None\n",
        "    if (adapter_path / 'adapter_model.safetensors').exists():\n",
        "      file_path = adapter_path / 'adapter_model.safetensors'\n",
        "      with safe_open(file_path, framework=\"pt\") as f:\n",
        "        if 'base_model.model.classifier.out_proj.bias' in f.keys():\n",
        "          tensor = f.get_tensor('base_model.model.classifier.out_proj.bias')\n",
        "        elif 'base_model.model.classifier.bias' in f.keys():\n",
        "          task_type = 'token_classification'\n",
        "          tensor = f.get_tensor('base_model.model.classifier.bias')\n",
        "        elif 'base_model.model.classifier.2.bias' in f.keys():\n",
        "          task_type = 'pair'\n",
        "          tensor = f.get_tensor('base_model.model.classifier.2.bias')\n",
        "        else:\n",
        "          raise KeyError(f\"Neither 'base_model.model.classifier.out_proj.bias' nor 'base_model.model.classifier.bias' found in the file({file_path}).\")\n",
        "\n",
        "    elif (adapter_path / 'adapter_model.bin').exists():\n",
        "      file_path = adapter_path / 'adapter_model.bin'\n",
        "      state_dict = torch.load(file_path, map_location=\"cpu\")\n",
        "      if 'base_model.model.classifier.out_proj.bias' in state_dict.keys():\n",
        "        tensor = state_dict['base_model.model.classifier.out_proj.bias']\n",
        "      elif 'base_model.model.classifier.bias' in state_dict.keys():\n",
        "        task_type = 'token_classification'\n",
        "        tensor = state_dict['base_model.model.classifier.bias']\n",
        "      elif 'base_model.model.classifier.2.bias' in state_dict.keys():\n",
        "        task_type = 'pair'\n",
        "        tensor = state_dict['base_model.model.classifier.2.bias']\n",
        "      else:\n",
        "        raise KeyError(f\"Neither 'base_model.model.classifier.out_proj.bias' nor 'base_model.model.classifier.bias' found in the file({file_path}).\")\n",
        "\n",
        "    else:\n",
        "        raise FileNotFoundError(f\"Neither 'adapter_model.safetensors' nor 'adapter_model.bin' found in the provided path({adapter_path}).\")\n",
        "\n",
        "    num_labels = list(tensor.shape)[0]\n",
        "    if task_type != 'token_classification':\n",
        "      if task_type != \"pair\":\n",
        "        if num_labels > 1:\n",
        "          task_type = 'classification'\n",
        "        elif num_labels == 1:\n",
        "          task_type = 'regression'\n",
        "\n",
        "      else:\n",
        "        if num_labels > 1:\n",
        "          task_type = 'pair_classification'\n",
        "        elif num_labels == 1:\n",
        "          task_type = 'pair_regression'\n",
        "\n",
        "    return num_labels, task_type\n",
        "\n",
        "################################################################################\n",
        "############################ INFO ##############################################\n",
        "################################################################################\n",
        "clear_output(wait=True)\n",
        "\n",
        "import markdown\n",
        "import ipywidgets\n",
        "import random\n",
        "import threading\n",
        "import time\n",
        "\n",
        "from jupyter_ui_poll import ui_events\n",
        "from ipywidgets import Button, Label, HTML, Layout\n",
        "from IPython.display import display, clear_output\n",
        "from functools import partial\n",
        "from saprot.utils.foldseek_util import get_struc_seq\n",
        "from saprot.utils.generate_lmdb import get_length\n",
        "\n",
        "\n",
        "######################################################################\n",
        "#            Backend functions             #\n",
        "######################################################################\n",
        "def red_print(text, **kwargs):\n",
        "  print(f\"\\033[31m{text}\\033[0m\", **kwargs)\n",
        "\n",
        "\n",
        "def get_upload_file_path(upload_items):\n",
        "  upload_size_box, _, google_drive_id, upload_button = upload_items[:4]\n",
        "  if upload_size_box.value == \"No\":\n",
        "    # User mannually uploads file\n",
        "    save_path = f\"{UPLOAD_FILE_HOME}/{upload_button.description}\"\n",
        "    assert os.path.exists(save_path), \"Please upload your file!\"\n",
        "\n",
        "  else:\n",
        "    # Disable all widgets\n",
        "    for widget in ipywidgets.Widget.widgets.values():\n",
        "      widget.disabled = True\n",
        "\n",
        "    google_drive_dir = f\"{UPLOAD_FILE_HOME}/google_drive/\"\n",
        "    os.makedirs(google_drive_dir, exist_ok=True)\n",
        "\n",
        "    # Download file from Google Drive\n",
        "    print(\"Downloading file from Google Drive...\")\n",
        "    file_link = google_drive_id.value\n",
        "    cmd = f\"cd {google_drive_dir} && gdown --fuzzy {file_link}\"\n",
        "    os.system(cmd)\n",
        "\n",
        "    # Enable all widgets\n",
        "    for widget in ipywidgets.Widget.widgets.values():\n",
        "      widget.disabled = False\n",
        "\n",
        "    # Check file validity\n",
        "    assert len(os.listdir(google_drive_dir)) > 0, \"Download failed. Please ensure your link is correct and publicly available!\"\n",
        "\n",
        "    file = os.listdir(google_drive_dir)[0]\n",
        "    save_path = f\"{UPLOAD_FILE_HOME}/{file}\"\n",
        "    cmd = f\"mv '{google_drive_dir}/{file}' '{save_path}'\"\n",
        "    os.system(cmd)\n",
        "\n",
        "  suffix = os.path.splitext(save_path)[-1]\n",
        "  new_save_path = f\"{os.path.dirname(save_path)}/{time.time()}{suffix}\"\n",
        "  cmd = f\"ln -s '{save_path}' {new_save_path}\"\n",
        "  os.system(cmd)\n",
        "  return new_save_path\n",
        "\n",
        "\n",
        "def set_upload_visibility(upload_items, mode=\"default\"):\n",
        "  assert mode in [\"default\", \"none\"]\n",
        "\n",
        "  if mode == \"default\":\n",
        "    upload_size_box, google_driver_tutorial, google_drive_id, upload_button, google_upload_html = upload_items\n",
        "    upload_size_box.value = \"No\"\n",
        "    upload_size_box.layout.display = None\n",
        "    google_driver_tutorial.layout.display = \"none\"\n",
        "    google_drive_id.layout.display = \"none\"\n",
        "    upload_button.layout.display = None\n",
        "    google_upload_html.layout.display = \"none\"\n",
        "\n",
        "  else:\n",
        "    for item in upload_items:\n",
        "      item.layout.display = \"none\"\n",
        "\n",
        "\n",
        "def get_upload_box(btn_desc=None):\n",
        "  import base64 as _base64\n",
        "  import collections as _collections\n",
        "  from http import server as _http_server\n",
        "  import json as _json\n",
        "  import os as _os\n",
        "  import pkgutil as _pkgutil\n",
        "  import socket as _socket\n",
        "  import socketserver as _socketserver\n",
        "  import urllib as _urllib\n",
        "  import uuid as _uuid\n",
        "\n",
        "  from google.colab import output as _output\n",
        "  import IPython as _IPython\n",
        "\n",
        "\n",
        "  WIDTH = \"500px\"\n",
        "  HEIGHT = \"30px\"\n",
        "\n",
        "  upload_size_box = ipywidgets.RadioButtons(\n",
        "    options=['Yes', 'No'],\n",
        "    value=\"No\",\n",
        "    disabled=False,\n",
        "    layout={'width': 'max-content'}, # If the items' names are long\n",
        "    description=\"Is the file size > 5MB?\",\n",
        "    style={'description_width': 'initial'},\n",
        "    )\n",
        "\n",
        "  google_drive_id = ipywidgets.Text(\n",
        "      placeholder=\"Upload your file to Google Drive and copy its path into this box\",\n",
        "      description=\"File path:\",\n",
        "      layout=Layout(width=WIDTH, height=HEIGHT, display=\"none\")\n",
        "      )\n",
        "  google_driver_tutorial = HTML(markdown.markdown(\n",
        "      \"<a href='https://github.com/westlake-repl/SaprotHub/wiki/SaprotHub-v2-%28latest%29#get-your-file-from-google-drive' target='blank'>Quick tutorial</a>\"\n",
        "      ),layout=Layout(display=\"none\"))\n",
        "\n",
        "  upload_button = ipywidgets.Button(\n",
        "      description=\"Upload your file\" if btn_desc is None else btn_desc,\n",
        "      layout=Layout(width=WIDTH, height=HEIGHT),\n",
        "      button_style=\"info\",\n",
        "      )\n",
        "\n",
        "  multiple = False\n",
        "  upload_id = str(_uuid.uuid4())\n",
        "  input_id = 'files-' + upload_id\n",
        "  output_id = 'result-' + upload_id\n",
        "  files_js = _pkgutil.get_data(\"google.colab.files\", 'resources/files.js').decode('utf8')\n",
        "\n",
        "  google_upload_html = HTML(\n",
        "          \"\"\"\n",
        "      <input type=\"file\" id=\"{input_id}\" name=\"files[]\" {multiple_text} disabled\n",
        "        style=\"border:none\" /><br><font color=red>Note: If you upload the same file, the upload will be ignored.</font></br>\n",
        "      <output id=\"{output_id}\">\n",
        "      Upload widget is only available when the cell has been executed in the\n",
        "      current browser session. Please rerun this cell to enable.\n",
        "      </output>\n",
        "      <script>{files_js}</script> \"\"\".format(\n",
        "              input_id=input_id,\n",
        "              output_id=output_id,\n",
        "              multiple_text='multiple' if multiple else '',\n",
        "              files_js=files_js,\n",
        "          ),layout=Layout(display=\"none\")\n",
        "\n",
        "      )\n",
        "\n",
        "  def switch_file_size(change):\n",
        "    now_type = change[\"new\"]\n",
        "    if now_type == \"No\":\n",
        "      google_drive_id.layout.display = \"none\"\n",
        "      google_driver_tutorial.layout.display = \"none\"\n",
        "      upload_button.layout.display = None\n",
        "\n",
        "    else:\n",
        "      google_drive_id.layout.display = None\n",
        "      google_driver_tutorial.layout.display = None\n",
        "      upload_button.layout.display = \"none\"\n",
        "\n",
        "  def google_file_upload(button):\n",
        "    def _get_unique_filename(filename):\n",
        "      if not _os.path.lexists(filename):\n",
        "        return filename\n",
        "      counter = 1\n",
        "      while True:\n",
        "        path, ext = _os.path.splitext(filename)\n",
        "        new_filename = '{} ({}){}'.format(path, counter, ext)\n",
        "        if not _os.path.lexists(new_filename):\n",
        "          return new_filename\n",
        "        counter += 1\n",
        "\n",
        "    try:\n",
        "      # Disable all widgets\n",
        "      for widget in ipywidgets.Widget.widgets.values():\n",
        "        widget.disabled = True\n",
        "      google_upload_html.disabled = False\n",
        "\n",
        "      ori_desc = button.description\n",
        "      button.description = \"All other buttons are disabled. Please first upload file or cancel upload!\"\n",
        "      button.button_style = \"danger\"\n",
        "      google_upload_html.layout.display = None\n",
        "\n",
        "      # First result is always an indication that the file picker has completed.\n",
        "      filename = None\n",
        "      result = _output.eval_js(\n",
        "          'google.colab._files._uploadFiles(\"{input_id}\", \"{output_id}\")'.format(\n",
        "              input_id=input_id, output_id=output_id\n",
        "          )\n",
        "      )\n",
        "      files = _collections.defaultdict(bytes)\n",
        "\n",
        "      while result['action'] != 'complete':\n",
        "        result = _output.eval_js(\n",
        "            'google.colab._files._uploadFilesContinue(\"{output_id}\")'.format(\n",
        "                output_id=output_id\n",
        "            )\n",
        "        )\n",
        "        if result['action'] != 'append':\n",
        "          # JS side uses a generator of promises to process all of the files- some\n",
        "          # steps may not produce data for the Python side, so just proceed onto the\n",
        "          # next message.\n",
        "          continue\n",
        "        files[result['file']] += _base64.b64decode(result['data'])\n",
        "\n",
        "      uploaded_files = dict(files)\n",
        "      # Mapping from original filename to filename as saved locally.\n",
        "\n",
        "      upload_path = Path(UPLOAD_FILE_HOME)\n",
        "      upload_path.mkdir(parents=True, exist_ok=True)\n",
        "      for filename, data in uploaded_files.items():\n",
        "        save_path = upload_path / filename\n",
        "        with open(save_path, 'wb') as f:\n",
        "          f.write(data)\n",
        "\n",
        "    except Exception as e:\n",
        "      raise e\n",
        "\n",
        "    finally:\n",
        "      # Enable items\n",
        "      for widget in ipywidgets.Widget.widgets.values():\n",
        "        widget.disabled = False\n",
        "\n",
        "      button.description = filename if filename is not None else ori_desc\n",
        "      button.button_style = \"info\"\n",
        "      google_upload_html.layout.display = \"none\"\n",
        "\n",
        "  # Set click events\n",
        "  upload_size_box.observe(switch_file_size, names=\"value\")\n",
        "  upload_button.on_click(google_file_upload)\n",
        "\n",
        "  upload_hbox = ipywidgets.HBox([upload_size_box, google_drive_id, upload_button, google_upload_html])\n",
        "  upload_items = [\n",
        "      upload_size_box,\n",
        "      google_driver_tutorial,\n",
        "      google_drive_id,\n",
        "      upload_button,\n",
        "      google_upload_html\n",
        "  ]\n",
        "\n",
        "  return upload_items\n",
        "\n",
        "\n",
        "# Disable a button during function execution and enable it after execution done\n",
        "def disable_wrapper(func):\n",
        "  def wrapper(button):\n",
        "    button.disabled = True\n",
        "    try:\n",
        "      func(button)\n",
        "\n",
        "    except Exception as e:\n",
        "      raise e\n",
        "\n",
        "    finally:\n",
        "      button.disabled = False\n",
        "\n",
        "  return wrapper\n",
        "\n",
        "\n",
        "def show_upload_file(change, button):\n",
        "  name = list(button.value.keys())[0]\n",
        "  button.description = name\n",
        "  button._counter = 1\n",
        "\n",
        "\n",
        "# Convert protein structures into structure-aware sequences\n",
        "def pdb2sa(pdb_list):\n",
        "  from saprot.utils.foldseek_util import get_struc_seq\n",
        "  from utils.mpr import MultipleProcessRunnerSimplifier\n",
        "\n",
        "  def do(process_id, idx, item, writer):\n",
        "    path, chain = item\n",
        "    name = os.path.basename(path)\n",
        "    sa_seq = get_struc_seq(FOLDSEEK_PATH, path, process_id=process_id, plddt_mask=True, chains=[chain])[chain][-1]\n",
        "    writer.write(f\"{sa_seq}\\n\")\n",
        "\n",
        "  mprs = MultipleProcessRunnerSimplifier(pdb_list, do, n_process=1, return_results=True)\n",
        "  outputs = mprs.run()\n",
        "  return outputs\n",
        "\n",
        "\n",
        "def save_uploaded_file(button):\n",
        "  upload_path = Path(UPLOAD_FILE_HOME)\n",
        "  upload_path.mkdir(parents=True, exist_ok=True)\n",
        "  basepath = Path().resolve()\n",
        "\n",
        "  # Write to specific path\n",
        "  assert len(list(button.value.keys())) > 0, \"\\033[31m\\nYou must upload the data file firstly!\\033[0m\"\n",
        "\n",
        "  name = list(button.value.keys())[0]\n",
        "  content = button.value[name][\"content\"]\n",
        "  save_path = upload_path / name\n",
        "\n",
        "  with open(save_path, \"wb\") as wb:\n",
        "    wb.write(content)\n",
        "\n",
        "  return name, str(save_path)\n",
        "\n",
        "\n",
        "# Protein property prediction\n",
        "def make_predictions(df, rows, num_labels, model_type, model_arg):\n",
        "  task_type = load_task_type_from_model(model_type, str(model_arg).split(\"\\n\")[0].strip())\n",
        "  original_task_type = task_type\n",
        "  task_type = task_type_dict[task_type]\n",
        "\n",
        "  if model_type == \"Multi-models on SaprotHub\":\n",
        "    #1. get adapter_list\n",
        "    repo_id_list = [repo_id.strip() for repo_id in model_arg.strip().split(\"\\n\")]\n",
        "    #2. download adapters\n",
        "    for repo_id in repo_id_list:\n",
        "      snapshot_download(repo_id=repo_id, repo_type=\"model\", local_dir=ADAPTER_HOME / repo_id)\n",
        "    config_list = [EasyDict({'lora_config_path': ADAPTER_HOME / repo_id}) for repo_id in repo_id_list]\n",
        "\n",
        "    assert len(config_list) > 0, \"Please choose at least one model!\"\n",
        "    base_model = get_base_model(ADAPTER_HOME / config_list[0].lora_config_path)\n",
        "\n",
        "    lora_kwargs = EasyDict({\n",
        "      \"is_trainable\": False,\n",
        "      \"num_lora\": len(config_list),\n",
        "      \"config_list\": config_list\n",
        "    })\n",
        "\n",
        "  else:\n",
        "    if model_type == \"Shared by peers on SaprotHub\":\n",
        "      snapshot_download(repo_id=model_arg, repo_type=\"model\", local_dir=ADAPTER_HOME / model_arg)\n",
        "\n",
        "    adapter_path = ADAPTER_HOME / model_arg\n",
        "    base_model = get_base_model(adapter_path)\n",
        "    lora_kwargs = {\n",
        "      \"is_trainable\": False,\n",
        "      \"num_lora\": 1,\n",
        "      \"config_list\": [{\"lora_config_path\": adapter_path}]\n",
        "    }\n",
        "\n",
        "  from saprot.config.config_dict import Default_config\n",
        "  config = copy.deepcopy(Default_config)\n",
        "\n",
        "  # task\n",
        "  if task_type in [ \"classification\", \"token_classification\", \"pair_classification\"]:\n",
        "    config.model.kwargs.num_labels = num_labels\n",
        "\n",
        "  # base model\n",
        "  config.model.model_py_path = model_type_dict[task_type]\n",
        "  config.model.kwargs.config_path = base_model\n",
        "\n",
        "  # lora\n",
        "  config.model.kwargs.lora_kwargs = lora_kwargs\n",
        "\n",
        "  # Load model\n",
        "  model = my_load_model(config.model)\n",
        "  tokenizer = EsmTokenizer.from_pretrained(config.model.kwargs.config_path)\n",
        "  device = \"cuda\" if torch.cuda.is_available() else \"cpu\"\n",
        "  model.to(device)\n",
        "\n",
        "  # Start prediction\n",
        "  logits = []\n",
        "  pred_labels = []\n",
        "  if task_type in [\"pair_classification\", \"pair_regression\"]:\n",
        "    for sa_seq_1, sa_seq_2 in tqdm(rows):\n",
        "      input_1 = tokenizer(sa_seq_1, return_tensors=\"pt\")\n",
        "      input_1 = {k: v.to(device) for k, v in input_1.items()}\n",
        "      input_2 = tokenizer(sa_seq_2, return_tensors=\"pt\")\n",
        "      input_2 = {k: v.to(device) for k, v in input_2.items()}\n",
        "\n",
        "      with torch.no_grad():\n",
        "        pred = model(input_1, input_2)\n",
        "\n",
        "      if \"regression\" in task_type:\n",
        "        pred_labels.append(pred.item())\n",
        "      else:\n",
        "        logits.append(pred[0].softmax(dim=-1).cpu().numpy().tolist())\n",
        "        pred_labels.append(pred.argmax(dim=-1)[0].cpu().numpy().tolist())\n",
        "\n",
        "  else:\n",
        "    for sa_seq in tqdm(rows):\n",
        "      inputs = tokenizer(sa_seq, return_tensors=\"pt\")\n",
        "      inputs = {k: v.to(device) for k, v in inputs.items()}\n",
        "      with torch.no_grad():\n",
        "        pred = model(inputs)\n",
        "\n",
        "      if \"regression\" in task_type:\n",
        "        pred_labels.append(pred.item())\n",
        "      else:\n",
        "        logits.append(pred[0].softmax(dim=-1).cpu().numpy().tolist())\n",
        "        pred_labels.append(pred.argmax(dim=-1)[0].cpu().numpy().tolist())\n",
        "\n",
        "  if \"classification\" in task_type:\n",
        "    df[\"predicted_probabilities\"] = logits\n",
        "  df[\"predicted_label\"] = pred_labels\n",
        "\n",
        "  return_label = pred_labels[0] if len(pred_labels) == 1 else None\n",
        "\n",
        "  # Save predictions\n",
        "  timestamp = str(datetime.now().strftime(\"%Y%m%d%H%M%S\"))\n",
        "  output_file = OUTPUT_HOME / f'output_{timestamp}.csv'\n",
        "  df.to_csv(output_file, index=False)\n",
        "\n",
        "  return return_label, output_file\n",
        "\n",
        "\n",
        "# Get data type that is compatible with the model\n",
        "def load_data_type_from_model(model_type, model_arg):\n",
        "  if model_type == \"Official SaProt (35M)\":\n",
        "    return \"SA\"\n",
        "\n",
        "  elif model_type == \"Official SaProt (650M)\":\n",
        "    return \"SA\"\n",
        "\n",
        "  else:\n",
        "    adapter_path = ADAPTER_HOME / model_arg\n",
        "\n",
        "    if model_type == \"Shared by peers on SaprotHub\":\n",
        "      snapshot_download(repo_id=model_arg, repo_type=\"model\", local_dir=adapter_path)\n",
        "\n",
        "    metadata_path = Path(adapter_path) / \"metadata.json\"\n",
        "    with open(metadata_path, 'r') as f:\n",
        "      metadata = json.load(f)\n",
        "      return metadata['training_data_type']\n",
        "\n",
        "\n",
        "# Get task type that is compatible with the model\n",
        "def load_task_type_from_model(model_type, model_arg):\n",
        "  try:\n",
        "    adapter_path = ADAPTER_HOME / model_arg\n",
        "\n",
        "    if model_type == \"Shared by peers on SaprotHub\" or model_type == \"Multi-models on SaprotHub\":\n",
        "      snapshot_download(repo_id=model_arg, repo_type=\"model\", local_dir=adapter_path)\n",
        "\n",
        "    metadata_path = Path(adapter_path) / \"metadata.json\"\n",
        "    with open(metadata_path, 'r') as f:\n",
        "      metadata = json.load(f)\n",
        "      return metadata['training_task_type']\n",
        "\n",
        "  except Exception as e:\n",
        "    raise Exception(\"\\033[31mPlease check your model input!\\033[0m\")\n",
        "\n",
        "\n",
        "def generate_download_btn(path: str):\n",
        "  if root_dir == \"/content\":\n",
        "    btn = ipywidgets.Button(\n",
        "      description=\"Download File\",\n",
        "      button_style=\"success\",\n",
        "      )\n",
        "\n",
        "    btn.on_click(lambda b: files.download(path))\n",
        "    return btn\n",
        "\n",
        "  else:\n",
        "    with open(path, \"rb\") as r:\n",
        "      res = r.read()\n",
        "\n",
        "    #FILE\n",
        "    filename = os.path.basename(path)\n",
        "    b64 = base64.b64encode(res)\n",
        "    payload = b64.decode()\n",
        "\n",
        "    #BUTTONS\n",
        "    html_buttons = '''<html>\n",
        "    <head>\n",
        "    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1\">\n",
        "    </head>\n",
        "    <body>\n",
        "    <a download=\"{filename}\" href=\"data:text/csv;base64,{payload}\" download>\n",
        "    <button class=\"p-Widget jupyter-widgets jupyter-button widget-button mod-warning\">Download File</button>\n",
        "    </a>\n",
        "    </body>\n",
        "    </html>\n",
        "    '''\n",
        "\n",
        "    html_button = html_buttons.format(payload=payload,filename=filename)\n",
        "    return HTML(html_button)\n",
        "\n",
        "\n",
        "def load_embedding_generation_model(model_type, model_arg):\n",
        "  if model_type == \"Official SaProt (35M)\":\n",
        "    base_model = \"westlake-repl/SaProt_35M_AF2\"\n",
        "    lora_kwargs = None\n",
        "\n",
        "  elif model_type == \"Official SaProt (650M)\":\n",
        "    base_model = \"westlake-repl/SaProt_650M_AF2\"\n",
        "    lora_kwargs = None\n",
        "\n",
        "  else:\n",
        "    adapter_path = ADAPTER_HOME / model_arg\n",
        "\n",
        "    if model_type == \"Shared by peers on SaprotHub\":\n",
        "      snapshot_download(repo_id=model_arg, repo_type=\"model\", local_dir=adapter_path)\n",
        "\n",
        "    base_model = get_base_model(adapter_path)\n",
        "    lora_kwargs = {\n",
        "      \"is_trainable\": False,\n",
        "      \"num_lora\": 1,\n",
        "      \"config_list\": [{\"lora_config_path\": adapter_path}]\n",
        "    }\n",
        "\n",
        "  from saprot.config.config_dict import Default_config\n",
        "  config = copy.deepcopy(Default_config)\n",
        "\n",
        "  if model_type in [\"Official SaProt (35M)\", \"Official SaProt (650M)\"]:\n",
        "    num_labels, task_type = 1, 'classification'\n",
        "  else:\n",
        "    num_labels, task_type = get_num_labels_and_task_type_by_adapter(lora_kwargs[\"config_list\"][0][\"lora_config_path\"])\n",
        "\n",
        "  config.model.kwargs.num_labels = num_labels\n",
        "  # base model\n",
        "  config.model.model_py_path = model_type_dict[task_type]\n",
        "  config.model.kwargs.config_path = base_model\n",
        "  # lora\n",
        "  config.model.kwargs.lora_kwargs = lora_kwargs\n",
        "\n",
        "  model = my_load_model(config.model)\n",
        "  tokenizer = EsmTokenizer.from_pretrained(config.model.kwargs.config_path)\n",
        "  device = \"cuda\" if torch.cuda.is_available() else \"cpu\"\n",
        "  model.to(device)\n",
        "\n",
        "  return model\n",
        "\n",
        "\n",
        "def generate_embeddings(protein_list, model_type, model_arg):\n",
        "  # If the uploaded file is protein structure\n",
        "  if isinstance(protein_list, list):\n",
        "    from saprot.utils.foldseek_util import get_struc_seq\n",
        "    from utils.mpr import MultipleProcessRunnerSimplifier\n",
        "\n",
        "    def do(process_id, idx, path, writer):\n",
        "      name = os.path.basename(path)\n",
        "      seq_dict = get_struc_seq(FOLDSEEK_PATH, path, plddt_mask=True, process_id=process_id)\n",
        "      for chain, seq_tuple in seq_dict.items():\n",
        "        writer.write(f\"{name}.chain-{chain}\\t{seq_tuple[-1]}\\n\")\n",
        "\n",
        "\n",
        "    mprs = MultipleProcessRunnerSimplifier(protein_list, do, n_process=2, return_results=True)\n",
        "    outputs = mprs.run()\n",
        "\n",
        "  # Else process fasta file\n",
        "  else:\n",
        "    from Bio import SeqIO\n",
        "    outputs = []\n",
        "    for record in SeqIO.parse(protein_list, \"fasta\"):\n",
        "      aa_seq = str(record.seq)\n",
        "      sa_seq = \"\".join(aa + \"#\" for aa in aa_seq)\n",
        "      outputs.append(f\"{record.id}\\t{sa_seq}\\n\")\n",
        "\n",
        "  model = load_embedding_generation_model(model_type, model_arg)\n",
        "\n",
        "  save_name_path = OUTPUT_HOME / \"embedding_seqs.fasta\"\n",
        "  save_embedding_path = OUTPUT_HOME / \"embeddings.pt\"\n",
        "\n",
        "  embedding_list = []\n",
        "  name_list = []\n",
        "  with torch.no_grad(), open(save_name_path, \"w\") as w:\n",
        "    for line in tqdm(outputs):\n",
        "      name, sa_seq = line.strip().split(\"\\t\")\n",
        "      w.write(f\">{name}\\n{sa_seq}\\n\")\n",
        "      embedding = model.get_hidden_states_from_seqs([sa_seq], reduction='mean')\n",
        "      embedding_list.append(embedding[0])\n",
        "      name_list.append(name)\n",
        "\n",
        "  embeddings = torch.stack(embedding_list)\n",
        "  torch.save(embeddings, save_embedding_path)\n",
        "\n",
        "  # Compress the fasta and embedding file into a .zip file\n",
        "  zip_path = OUTPUT_HOME / \"generated_embedding.zip\"\n",
        "  !cd $OUTPUT_HOME && zip -r $zip_path \"embedding_seqs.fasta\" \"embeddings.pt\"\n",
        "\n",
        "  return zip_path\n",
        "\n",
        "\n",
        "def load_zeroshot_model():\n",
        "  try:\n",
        "    zero_shot_model\n",
        "  except Exception:\n",
        "    from saprot.model.saprot.saprot_foldseek_mutation_model import SaprotFoldseekMutationModel\n",
        "    base_model = \"westlake-repl/SaProt_650M_AF2\"\n",
        "    config = {\n",
        "      \"foldseek_path\": None,\n",
        "      \"config_path\": base_model,\n",
        "      \"load_pretrained\": True,\n",
        "    }\n",
        "\n",
        "    zero_shot_model = SaprotFoldseekMutationModel(**config)\n",
        "    device = \"cuda\" if torch.cuda.is_available() else \"cpu\"\n",
        "    zero_shot_model.to(device)\n",
        "\n",
        "  return zero_shot_model\n",
        "\n",
        "\n",
        "# Zero-shot prediction\n",
        "def predict_mut(sa_seq, mut_info):\n",
        "  zero_shot_model = load_zeroshot_model()\n",
        "  score = zero_shot_model.predict_mut(sa_seq, mut_info)\n",
        "  return score\n",
        "\n",
        "\n",
        "# Zero-shot prediction for single-site saturation mutagenesis\n",
        "def predict_all_mut(sa_seq):\n",
        "  zero_shot_model = load_zeroshot_model()\n",
        "\n",
        "  timestamp = datetime.now().strftime(\"%y%m%d%H%M%S\")\n",
        "  output_path = OUTPUT_HOME / f'{timestamp}_prediction_output.csv'\n",
        "\n",
        "  mut_dicts = []\n",
        "  aa_seq = sa_seq[0::2]\n",
        "  for i in tqdm(range(len(aa_seq)), leave=False, desc=f\"Predicting\"):\n",
        "    mut_dict = zero_shot_model.predict_pos_mut(sa_seq, i+1)\n",
        "    mut_dicts.append(mut_dict)\n",
        "\n",
        "  mut_list = [{'mutation': key, 'score': value} for d in mut_dicts for key, value in d.items()]\n",
        "  df = pd.DataFrame(mut_list)\n",
        "  df.to_csv(output_path, index=None)\n",
        "  return output_path\n",
        "\n",
        "\n",
        "# Perform protein sequence design\n",
        "def inverse_folding(aa_seq, struc_seq, method, num_samples):\n",
        "  try:\n",
        "    saprot_if_model\n",
        "\n",
        "  except Exception:\n",
        "    from saprot.model.saprot.saprot_if_model import SaProtIFModel\n",
        "\n",
        "    base_model = \"westlake-repl/SaProt_650M_AF2_inverse_folding\"\n",
        "    config = {\n",
        "        \"config_path\": base_model,\n",
        "        \"load_pretrained\": True,\n",
        "    }\n",
        "\n",
        "    saprot_if_model = SaProtIFModel(**config)\n",
        "    tokenizer = saprot_if_model.tokenizer\n",
        "    device = \"cuda\" if torch.cuda.is_available() else \"cpu\"\n",
        "    saprot_if_model.to(device)\n",
        "\n",
        "  pred_aa_seqs = saprot_if_model.predict(aa_seq, struc_seq, method=method, num_samples=num_samples)\n",
        "  return pred_aa_seqs\n",
        "\n",
        "\n",
        "# Perform protein structure using ESMFold\n",
        "def predict_structure(seq):\n",
        "  try:\n",
        "    esmfold\n",
        "  except Exception:\n",
        "    tokenizer = AutoTokenizer.from_pretrained(\"facebook/esmfold_v1\")\n",
        "    esmfold = EsmForProteinFolding.from_pretrained(\"facebook/esmfold_v1\")\n",
        "    esmfold.esm = esmfold.esm.half()\n",
        "    esmfold.trunk.set_chunk_size(64)\n",
        "\n",
        "    device = \"cuda\" if torch.cuda.is_available() else \"cpu\"\n",
        "    esmfold.to(device)\n",
        "\n",
        "  tokenized_input = tokenizer(\n",
        "    [seq],\n",
        "    return_tensors=\"pt\",\n",
        "    add_special_tokens=False,\n",
        "    )['input_ids']\n",
        "\n",
        "  tokenized_input = tokenized_input.to(esmfold.device)\n",
        "  with torch.no_grad():\n",
        "    output = esmfold(tokenized_input)\n",
        "\n",
        "  save_path = f\"{root_dir}/SaprotHub/output/predicted_structure.pdb\"\n",
        "  pdb = convert_outputs_to_pdb(output)\n",
        "  with open(save_path, \"w\") as f:\n",
        "    f.write(\"\".join(pdb))\n",
        "\n",
        "  return save_path\n",
        "\n",
        "\n",
        "######################################################################\n",
        "#          Training or Prediction            #\n",
        "######################################################################\n",
        "def train_or_pred():\n",
        "  global refresh_module\n",
        "  refresh_module = train_or_pred\n",
        "\n",
        "  question = HTML(markdown.markdown(\"## Please choose what you want to do with ColabSaprot\"))\n",
        "  option_intro = HTML(markdown.markdown(\n",
        "    \"<a href='https://github.com/westlake-repl/SaprotHub/wiki/SaprotHub-v2-(latest)#introduction-of-options' target='blank'>Introduction of these options</a>\"\n",
        "    ))\n",
        "\n",
        "  train_btn = Button(description='I want to train my own model', layout=Layout(width='400px', height='30px'))\n",
        "  pred_btn = Button(description='I want to use existing models to make prediction', layout=Layout(width='400px', height='30px'))\n",
        "  share_btn = Button(description='I want to share my model publicly', layout=Layout(width='400px', height='30px'))\n",
        "\n",
        "  items = [\n",
        "      question,\n",
        "      option_intro,\n",
        "      train_btn,\n",
        "      pred_btn,\n",
        "      share_btn\n",
        "      ]\n",
        "\n",
        "  # Set click events\n",
        "  train_btn.on_click(partial(jump, next=choose_training_task))\n",
        "  pred_btn.on_click(partial(jump, next=choose_pred_task))\n",
        "  share_btn.on_click(partial(jump, next=share_model))\n",
        "\n",
        "  display(*items)\n",
        "\n",
        "\n",
        "\n",
        "######################################################################\n",
        "#          Choose training task             #\n",
        "######################################################################\n",
        "def choose_training_task():\n",
        "  global refresh_module\n",
        "  refresh_module = choose_training_task\n",
        "\n",
        "  title = HTML(markdown.markdown(\"## Please finish the setting of your training task\"))\n",
        "  WIDTH = \"500px\"\n",
        "  HEIGHT= \"30px\"\n",
        "\n",
        "  task_hint = HTML(markdown.markdown(\"### Task setting:\"))\n",
        "  task_name = ipywidgets.Text(\n",
        "              value=\"Demo\",\n",
        "              placeholder=f'Enter the task name',\n",
        "              disabled=False,\n",
        "              description=\"Name your task:\",\n",
        "              layout=Layout(width=WIDTH, height=HEIGHT),\n",
        "              style={'description_width': 'initial'},\n",
        "              )\n",
        "  task_type = ipywidgets.Dropdown(\n",
        "            options=['Protein-level Classification', 'Protein-level Regression', 'Residue-level Classification', \"Protein-protein Classification\", \"Protein-protein Regression\"],\n",
        "            value='Protein-level Classification',\n",
        "            description='Task type:',\n",
        "            disabled=False,\n",
        "            layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "          )\n",
        "\n",
        "  num_label = ipywidgets.BoundedIntText(\n",
        "        value=2,\n",
        "        min=2,\n",
        "        max=100000000,\n",
        "        step=1,\n",
        "        description='Number of categories:',\n",
        "        disabled=False,\n",
        "        style={'description_width': 'initial'},\n",
        "        layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "      )\n",
        "\n",
        "  task_intro_dict = {\n",
        "      'Protein-level Classification': \"<font color=red>What is **Protein-level Classification:** Given a protein, you have some categories and you want to predict which category the protein belongs to.</font>\",\n",
        "      \"Protein-level Regression\": \"<font color=red>What is **Protein-level Regression:** Given a protein, you want to predict a score about its property such as stability or enzyme activity.</font>\",\n",
        "      \"Residue-level Classification\": \"<font color=red>What is **Residue-level Classification:** Given a protein, you have some categories and for every amino acid you want to predict which category it belongs to.</font>\",\n",
        "      \"Protein-protein Classification\": \"<font color=red>What is **Protein-protein Classification:** Given a pair of proteins, you have some categories and you want to predict which category the pair belongs to.</font>\",\n",
        "      \"Protein-protein Regression\": \"<font color=red>What is **Protein-protein Regression:** Given a pair of proteins, you want to predict a score about its property such as binding affinity.</font>\"\n",
        "  }\n",
        "\n",
        "  task_intro = HTML(markdown.markdown(\n",
        "    \"<font color=red>What is **Protein-level Classification:** Given a protein, you have some categories and you want to predict which category the protein belongs to.</font>\"\n",
        "    ), layout=Layout(width=WIDTH))\n",
        "\n",
        "  model_hint = HTML(markdown.markdown(\"### Model setting:\"))\n",
        "  model_type = ipywidgets.Dropdown(\n",
        "            options=['Official SaProt (35M)', \"Official SaProt (650M)\", \"Trained by yourself on ColabSaprot\", \"Shared by peers on SaprotHub\", \"Saved in your local computer\"],\n",
        "            value='Official SaProt (35M)',\n",
        "            description='Base model:',\n",
        "            disabled=False,\n",
        "            layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "          )\n",
        "  model_arg_box = HTML(markdown.markdown(\"dummy box\"))\n",
        "  model_arg_box.layout.display = \"none\"\n",
        "\n",
        "  upload_model_items = get_upload_box(btn_desc=\"Upload Model-<task_name>-<model_size>.zip file\")\n",
        "  set_upload_visibility(upload_model_items, mode=\"none\")\n",
        "\n",
        "  saprot_650m_hint = HTML(markdown.markdown(\n",
        "      \"<font color=red>Warning: You are training Saprot 650M version, please switch to A100 GPU (need Colab Pro) to avoid the **Out of Memory** problem!</font>\"\n",
        "      ), layout=Layout(display=\"none\"))\n",
        "\n",
        "  saprothub_link = HTML(markdown.markdown(\n",
        "      \"<font color=red>You could search models using our <a href='https://huggingface.co/spaces/SaProtHub/SaprotHub-search' target='blank'>search engine</a> or from <a href='https://huggingface.co/SaProtHub' target='blank'>SaprotHub</a>\\n\\n\"\n",
        "      \"A model id example: <a href='https://huggingface.co/SaProtHub/Model-Binary_Localization-650M' target='blank'>SaProtHub/Model-Binary_Localization-650M</a></font>\"\n",
        "      ), layout=Layout(display=\"none\"))\n",
        "\n",
        "  dataset_hint = HTML(markdown.markdown(\"### Dataset setting:\"))\n",
        "  data_type_hint = HTML(markdown.markdown(\"**Your training data type:**\"))\n",
        "  data_type = ipywidgets.RadioButtons(\n",
        "      options=['protein sequence', 'protein structure'],\n",
        "      disabled=False,\n",
        "      style={'description_width': 'initial'},\n",
        "      )\n",
        "\n",
        "  data_src_hint = HTML(markdown.markdown(\"**Where is your data from:**\"))\n",
        "  data_src_type = ipywidgets.RadioButtons(\n",
        "      options=['SaprotHub in HuggingFace', 'Upload mannually'],\n",
        "      value=\"Upload mannually\",\n",
        "      layout={'width': 'max-content'}, # If the items' names are long\n",
        "      disabled=False,\n",
        "      style={'description_width': 'initial'},\n",
        "      )\n",
        "\n",
        "  # data_split_hint = HTML(markdown.markdown(\"**Have you already split your data into training, validation and test set?**\"))\n",
        "  # data_split_type = ipywidgets.RadioButtons(\n",
        "  #             options=['Yes', 'No (We will randomly split your data with 8:1:1 for train:valid:test)'],\n",
        "  #           layout={'width': 'max-content'}, # If the items' names are long\n",
        "  #             disabled=False,\n",
        "  #             style={'description_width': 'initial'},\n",
        "  #         )\n",
        "\n",
        "  saprothub_data_id = ipywidgets.Text(\n",
        "              value=None,\n",
        "              placeholder=f'Enter SaprotHub dataset id in HuggingFace',\n",
        "              disabled=False,\n",
        "              description=\"Dataset id:\",\n",
        "              layout=Layout(width=WIDTH, height=HEIGHT, display=\"none\"),\n",
        "              )\n",
        "  saprothub_data_id_hint = HTML(markdown.markdown(\n",
        "      \"<font color=red>You could search datasets using our <a href='https://huggingface.co/spaces/SaProtHub/SaprotHub-search' target='blank'>search engine</a> or from <a href='https://huggingface.co/SaProtHub' target='blank'>SaprotHub</a>\\n\\n\"\n",
        "      \"A dataset id example: <a href='https://huggingface.co/datasets/SaProtHub/Dataset-Binary_Localization-DeepLoc' target='blank'>SaProtHub/Dataset-Binary_Localization-DeepLoc</a></font>\"\n",
        "      ), layout=Layout(display=\"none\"))\n",
        "\n",
        "  upload_hint = HTML(markdown.markdown(\n",
        "            \"**Upload your training data:**\\n\\n\"\n",
        "            \"<img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/train/dataset/aa_classification.jpg?raw=true' height='200px' width='700px' align='center'>\\n\\n\"\n",
        "            \"Please upload a ``.csv`` file that contains all protein sequences to train. Please strictly follow the format above **(column names are also needed in the csv file)**.\"\n",
        "            ))\n",
        "  data_example_hint = HTML(markdown.markdown(\n",
        "            f\"<a href='https://github.com/westlake-repl/SaprotHub/tree/main/upload_files/ColabSaprot_v2_example_csv_dataset/train/' target='blank'>Here</a> are some toy examples.\"\n",
        "            ))\n",
        "\n",
        "  upliad_items_hint_1 = HTML(markdown.markdown(\n",
        "            \"**Upload .csv file:**\"\n",
        "            ))\n",
        "  upload_items_1 = get_upload_box()\n",
        "\n",
        "  upliad_items_hint_2 = HTML(markdown.markdown(\n",
        "          \"**Upload .zip file:**\"\n",
        "          ), layout=Layout(display=\"none\"))\n",
        "  upload_items_2 = get_upload_box()\n",
        "  set_upload_visibility(upload_items_2, mode=\"none\")\n",
        "\n",
        "  hyperparameter_hint = HTML(markdown.markdown(\n",
        "      \"### Training hyper-parameters:\\n\\n\"\n",
        "      \"<a href='https://github.com/westlake-repl/SaprotHub/wiki/SaprotHub-v2-%28latest%29#hyper-parameters' target='blank'>What are these hyper-parameters?</a>\"\n",
        "      ))\n",
        "  batch_size = ipywidgets.Dropdown(\n",
        "            options=[\"Adaptive\", \"1\", \"2\", \"4\", \"8\", \"16\", \"32\", \"64\", \"128\", \"256\"],\n",
        "            value='Adaptive',\n",
        "            description='Batch size:',\n",
        "            disabled=False,\n",
        "            layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "            )\n",
        "\n",
        "  epoch = ipywidgets.BoundedIntText(\n",
        "        value=5,\n",
        "        min=1,\n",
        "        max=100,\n",
        "        step=1,\n",
        "        description='Epoch:',\n",
        "        disabled=False,\n",
        "        style={'description_width': 'initial'},\n",
        "        layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "      )\n",
        "\n",
        "  lr = ipywidgets.BoundedFloatText(\n",
        "      value=5e-4,\n",
        "      min=0,\n",
        "      max=1e-2,\n",
        "      step=1e-4,\n",
        "      description='Learning rate:',\n",
        "      disabled=False,\n",
        "      style={'description_width': 'initial'},\n",
        "      layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "  )\n",
        "\n",
        "  advanced_setting = Button(description='Show advanced setting‚¨áÔ∏è', layout=Layout(width=WIDTH, height=HEIGHT))\n",
        "  lora_hint = HTML(markdown.markdown(f\"### Lora config\"))\n",
        "  lora_r = ipywidgets.BoundedIntText(\n",
        "        value=8,\n",
        "        min=1,\n",
        "        max=128,\n",
        "        step=1,\n",
        "        description='r:',\n",
        "        disabled=False,\n",
        "        style={'description_width': 'initial'},\n",
        "        layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "      )\n",
        "  lora_dropout_box = ipywidgets.FloatText(\n",
        "      value=0.0,\n",
        "      description='Lora dropout:',\n",
        "      disabled=False,\n",
        "      style={'description_width': 'initial'},\n",
        "      layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "  )\n",
        "  lora_alpha_box = ipywidgets.BoundedIntText(\n",
        "        value=16,\n",
        "        min=1,\n",
        "        max=128,\n",
        "        step=1,\n",
        "        description='Lora alpha:',\n",
        "        disabled=False,\n",
        "        style={'description_width': 'initial'},\n",
        "        layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "      )\n",
        "\n",
        "  trainer_hint = HTML(markdown.markdown(f\"### PyTorch Lightning trainer config\"))\n",
        "  val_check_interval_box = ipywidgets.FloatText(\n",
        "      value=0.5,\n",
        "      description='val_check_interval:',\n",
        "      disabled=False,\n",
        "      style={'description_width': 'initial'},\n",
        "      layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "  )\n",
        "  limit_train_batches_box = ipywidgets.FloatText(\n",
        "      value=1.0,\n",
        "      description='limit_train_batches:',\n",
        "      disabled=False,\n",
        "      style={'description_width': 'initial'},\n",
        "      layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "  )\n",
        "  limit_val_batches_box = ipywidgets.FloatText(\n",
        "      value=1.0,\n",
        "      description='limit_val_batches:',\n",
        "      disabled=False,\n",
        "      style={'description_width': 'initial'},\n",
        "      layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "  )\n",
        "  limit_test_batches_box = ipywidgets.FloatText(\n",
        "      value=1.0,\n",
        "      description='limit_test_batches:',\n",
        "      disabled=False,\n",
        "      style={'description_width': 'initial'},\n",
        "      layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "  )\n",
        "\n",
        "  start_hint = HTML(markdown.markdown(f\"### {'-'*75}\"))\n",
        "  stable_control_hint = HTML(markdown.markdown(\n",
        "      \"<font color=red>Warning: Your session may disconnect during long-time (e.g. >2 hours) training. \"\n",
        "      \"See our quick fix <a href='https://github.com/westlake-repl/SaprotHub/wiki/SaprotHub-v2-(latest)#how-to-handle-long-time-training' target='blank'>here</a>.</font>\"\n",
        "      ))\n",
        "  start_btn = Button(description='Start training', layout=Layout(width=WIDTH, height=HEIGHT), button_style=\"info\")\n",
        "\n",
        "\n",
        "  items = [\n",
        "      title,\n",
        "      task_hint,\n",
        "      task_name,\n",
        "      task_type,\n",
        "      num_label,\n",
        "      task_intro,\n",
        "      model_hint,\n",
        "      model_type,\n",
        "      model_arg_box,\n",
        "      saprot_650m_hint,\n",
        "      saprothub_link,\n",
        "      dataset_hint,\n",
        "      data_src_hint,\n",
        "      data_src_type,\n",
        "      data_type_hint,\n",
        "      data_type,\n",
        "      saprothub_data_id,\n",
        "      saprothub_data_id_hint,\n",
        "      upload_hint,\n",
        "      data_example_hint,\n",
        "      upliad_items_hint_1,\n",
        "      *upload_items_1,\n",
        "      upliad_items_hint_2,\n",
        "      *upload_items_2,\n",
        "      hyperparameter_hint,\n",
        "      batch_size,\n",
        "      epoch,\n",
        "      lr,\n",
        "      advanced_setting,\n",
        "      lora_hint,\n",
        "      lora_r,\n",
        "      lora_dropout_box,\n",
        "      lora_alpha_box,\n",
        "      trainer_hint,\n",
        "      limit_train_batches_box,\n",
        "      limit_val_batches_box,\n",
        "      limit_test_batches_box,\n",
        "      start_hint,\n",
        "      stable_control_hint,\n",
        "      start_btn\n",
        "      ]\n",
        "\n",
        "  # Used to replace input box based on model type\n",
        "  model_arg_box_idx = items.index(model_arg_box)\n",
        "\n",
        "  # Set click events\n",
        "  def change_task_type(change):\n",
        "    now_type = change[\"new\"]\n",
        "    task_intro.value = markdown.markdown(task_intro_dict[now_type])\n",
        "    set_upload_hint()\n",
        "    if \"Classification\" in now_type:\n",
        "      num_label.layout.display = None\n",
        "    else:\n",
        "      num_label.layout.display = \"none\"\n",
        "\n",
        "  def change_model_type(change):\n",
        "    model_type_value = change[\"new\"]\n",
        "    task_type_value = task_type_dict[task_type.value]\n",
        "\n",
        "    saprot_650m_hint.layout.display = None if \"650M\" in model_type_value else \"none\"\n",
        "\n",
        "    if model_type_value == \"Trained by yourself on ColabSaprot\":\n",
        "      model_arg_box = select_adapter_from(None, use_model_from=model_type_value)\n",
        "      model_arg_box.layout.width = WIDTH\n",
        "      model_arg_box.description = \"Select your local model:\"\n",
        "      model_arg_box.style = {'description_width': 'initial'}\n",
        "      items[model_arg_box_idx] = model_arg_box\n",
        "      saprothub_link.layout.display = \"none\"\n",
        "      custom_display(*items)\n",
        "\n",
        "    elif model_type_value == \"Shared by peers on SaprotHub\":\n",
        "      model_arg_box = select_adapter_from(None, use_model_from=model_type_value)\n",
        "      model_arg_box.layout.width = WIDTH\n",
        "      model_arg_box.placeholder = \"Enter SaprotHub model id in HuggingFace\"\n",
        "      model_arg_box.description = \"Model id:\"\n",
        "      items[model_arg_box_idx] = model_arg_box\n",
        "      saprothub_link.layout.display = None\n",
        "      custom_display(*items)\n",
        "\n",
        "    elif model_type_value == \"Saved in your local computer\":\n",
        "      set_upload_visibility(upload_model_items, \"default\")\n",
        "      new_model_arg_box = upload_model_items\n",
        "      new_items = items[:model_arg_box_idx] + new_model_arg_box + items[model_arg_box_idx+1:]\n",
        "      custom_display(*new_items)\n",
        "      saprothub_link.layout.display = \"none\"\n",
        "\n",
        "    else:\n",
        "      items[model_arg_box_idx].layout.display = \"none\"\n",
        "      saprothub_link.layout.display = \"none\"\n",
        "      custom_display(*items)\n",
        "\n",
        "  def change_data_type(change):\n",
        "    now_type = change[\"new\"]\n",
        "    if now_type == \"protein sequence\":\n",
        "      upliad_items_hint_2.layout.display = \"none\"\n",
        "      set_upload_visibility(upload_items_2, \"none\")\n",
        "    else:\n",
        "      upliad_items_hint_2.layout.display = None\n",
        "      set_upload_visibility(upload_items_2, \"default\")\n",
        "\n",
        "    set_upload_hint()\n",
        "\n",
        "  def change_data_src_type(change):\n",
        "    data_src_type_value = change[\"new\"]\n",
        "    if data_src_type_value == \"SaprotHub in HuggingFace\":\n",
        "      saprothub_data_id.layout.display = None\n",
        "      saprothub_data_id_hint.layout.display = None\n",
        "      for item in [data_type_hint, data_type, upload_hint, data_example_hint]:\n",
        "        item.layout.display = \"none\"\n",
        "\n",
        "      upliad_items_hint_1.layout.display = \"none\"\n",
        "      set_upload_visibility(upload_items_1, \"none\")\n",
        "      upliad_items_hint_2.layout.display = \"none\"\n",
        "      set_upload_visibility(upload_items_2, \"none\")\n",
        "\n",
        "    else:\n",
        "      saprothub_data_id.layout.display = \"none\"\n",
        "      saprothub_data_id_hint.layout.display = \"none\"\n",
        "      for item in [data_type_hint, data_type, upload_hint, data_example_hint]:\n",
        "        item.layout.display = None\n",
        "\n",
        "      upliad_items_hint_1.layout.display = None\n",
        "      set_upload_visibility(upload_items_1, \"default\")\n",
        "\n",
        "      if data_type.value == \"protein structure\":\n",
        "        upliad_items_hint_2.layout.display = None\n",
        "        set_upload_visibility(upload_items_2, \"default\")\n",
        "\n",
        "  def set_upload_hint():\n",
        "    if task_type.value == \"Protein-level Classification\" and data_type.value == \"protein sequence\":\n",
        "      upload_hint.value = markdown.markdown(\n",
        "              \"**Upload your training data:**\\n\\n\"\n",
        "              \"<img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/train/dataset/aa_classification.jpg?raw=true' height='200px' width='700px' align='center'>\\n\\n\"\n",
        "              \"Please upload a ``.csv`` file that contains all protein sequences to train. Please strictly follow the format above **(column names are also needed in the csv file)**.\"\n",
        "              )\n",
        "\n",
        "    elif task_type.value == \"Protein-level Regression\" and data_type.value == \"protein sequence\":\n",
        "      upload_hint.value = markdown.markdown(\n",
        "              \"**Upload your training data:**\\n\\n\"\n",
        "              \"<img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/train/dataset/aa_regression.jpg?raw=true' height='200px' width='700px' align='center'>\\n\\n\"\n",
        "              \"Please upload a ``.csv`` file that contains all protein sequences to train. Please strictly follow the format above **(column names are also needed in the csv file)**.\"\n",
        "              )\n",
        "\n",
        "    elif task_type.value == \"Residue-level Classification\" and data_type.value == \"protein sequence\":\n",
        "      upload_hint.value = markdown.markdown(\n",
        "              \"**Upload your training data:**\\n\\n\"\n",
        "              \"<img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/train/dataset/aa_token_classification.jpg?raw=true' height='200px' width='800px' align='center'>\\n\\n\"\n",
        "              \"Please upload a ``.csv`` file that contains all protein sequences to train. Please strictly follow the format above **(column names are also needed in the csv file)**.\"\n",
        "              )\n",
        "\n",
        "    elif task_type.value == \"Protein-protein Classification\" and data_type.value == \"protein sequence\":\n",
        "      upload_hint.value = markdown.markdown(\n",
        "              \"**Upload your training data:**\\n\\n\"\n",
        "              \"<img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/train/dataset/aa_pair_classification.jpg?raw=true' height='200px' width='800px' align='center'>\\n\\n\"\n",
        "              \"Please upload a ``.csv`` file that contains all protein sequences to train. Please strictly follow the format above **(column names are also needed in the csv file)**.\"\n",
        "              )\n",
        "\n",
        "    elif task_type.value == \"Protein-protein Regression\" and data_type.value == \"protein sequence\":\n",
        "      upload_hint.value = markdown.markdown(\n",
        "              \"**Upload your training data:**\\n\\n\"\n",
        "              \"<img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/train/dataset/aa_pair_regression.jpg?raw=true' height='200px' width='800px' align='center'>\\n\\n\"\n",
        "              \"Please upload a ``.csv`` file that contains all protein sequences to train. Please strictly follow the format above **(column names are also needed in the csv file)**.\"\n",
        "              )\n",
        "\n",
        "    elif task_type.value == \"Protein-level Classification\" and data_type.value == \"protein structure\":\n",
        "      upload_hint.value = markdown.markdown(\n",
        "              \"**Upload your training data:**\\n\\n\"\n",
        "              \"<img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/train/dataset/struc_classification.jpg?raw=true' height='200px' width='550px' align='center'>\\n\\n\"\n",
        "              \"Please upload a ``.csv`` file and a ``.zip`` file that stores all protein structures to train. Please strictly follow the format above **(column names are also needed in the csv file)**.\"\n",
        "              )\n",
        "\n",
        "    elif task_type.value == \"Protein-level Regression\" and data_type.value == \"protein structure\":\n",
        "      upload_hint.value = markdown.markdown(\n",
        "              \"**Upload your training data:**\\n\\n\"\n",
        "              \"<img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/train/dataset/struc_regression.jpg?raw=true' height='200px' width='550px' align='center'>\\n\\n\"\n",
        "              \"Please upload a ``.csv`` file and a ``.zip`` file that stores all protein structures to train. Please strictly follow the format above **(column names are also needed in the csv file)**.\"\n",
        "              )\n",
        "\n",
        "    elif task_type.value == \"Residue-level Classification\" and data_type.value == \"protein structure\":\n",
        "      upload_hint.value = markdown.markdown(\n",
        "              \"**Upload your training data:**\\n\\n\"\n",
        "              \"<img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/train/dataset/struc_token_classification.jpg?raw=true' height='200px' width='700px' align='center'>\\n\\n\"\n",
        "              \"Please upload a ``.csv`` file and a ``.zip`` file that stores all protein structures to train. Please strictly follow the format above **(column names are also needed in the csv file)**.\"\n",
        "              )\n",
        "\n",
        "    elif task_type.value == \"Protein-protein Classification\" and data_type.value == \"protein structure\":\n",
        "      upload_hint.value = markdown.markdown(\n",
        "              \"**Upload your training data:**\\n\\n\"\n",
        "              \"<img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/train/dataset/struc_pair_classification.jpg?raw=true' height='200px' width='800px' align='center'>\\n\\n\"\n",
        "              \"Please upload a ``.csv`` file and a ``.zip`` file that stores all protein structures to train. Please strictly follow the format above **(column names are also needed in the csv file)**.\"\n",
        "              )\n",
        "\n",
        "    elif task_type.value == \"Protein-protein Regression\" and data_type.value == \"protein structure\":\n",
        "      upload_hint.value = markdown.markdown(\n",
        "              \"**Upload your training data:**\\n\\n\"\n",
        "              \"<img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/train/dataset/struc_pair_regression.jpg?raw=true' height='200px' width='800px' align='center'>\\n\\n\"\n",
        "              \"Please upload a ``.csv`` file and a ``.zip`` file that stores all protein structures to train. Please strictly follow the format above **(column names are also needed in the csv file)**.\"\n",
        "              )\n",
        "\n",
        "    else:\n",
        "      raise\n",
        "\n",
        "  def click_advanced_setting(button):\n",
        "    if button.description == \"Show advanced setting‚¨áÔ∏è\":\n",
        "      button.description = \"Hide advanced setting‚¨áÔ∏è\"\n",
        "      for item in [lora_hint, lora_r, lora_dropout_box, lora_alpha_box, trainer_hint, limit_train_batches_box, limit_val_batches_box, limit_test_batches_box,]:\n",
        "        item.layout.display = None\n",
        "    else:\n",
        "      button.description = \"Show advanced setting‚¨áÔ∏è\"\n",
        "      for item in [lora_hint, lora_r, lora_dropout_box, lora_alpha_box, trainer_hint, limit_train_batches_box, limit_val_batches_box, limit_test_batches_box,]:\n",
        "        item.layout.display = \"none\"\n",
        "\n",
        "\n",
        "  def start_training(button):\n",
        "    print(\"Start training...\")\n",
        "\n",
        "    task_type_value = task_type.value\n",
        "    model_type_value = model_type.value\n",
        "    model_arg = items[model_arg_box_idx].value\n",
        "\n",
        "    # Check compatibility between chosen task and model\n",
        "    if model_type_value == \"Shared by peers on SaprotHub\" or model_type_value == \"Trained by yourself on ColabSaprot\" or model_type_value == \"Saved in your local computer\":\n",
        "      if model_type_value == \"Saved in your local computer\":\n",
        "        zip_path = get_upload_file_path(upload_model_items)\n",
        "        name = os.path.basename(zip_path)\n",
        "        save_dir = ADAPTER_HOME / \"Upload\" / name.rsplit('.', 1)[0]\n",
        "        os.makedirs(save_dir, exist_ok=True)\n",
        "        # unzip model.zip\n",
        "        with zipfile.ZipFile(zip_path, 'r') as zip_ref:\n",
        "            zip_ref.extractall(save_dir)\n",
        "        os.remove(zip_path)\n",
        "        model_arg = save_dir\n",
        "\n",
        "      model_task_type = load_task_type_from_model(model_type_value, model_arg)\n",
        "      assert task_type_value == model_task_type, f\"The model you choose was trained on {model_task_type} task, which is not suitable for the {task_type_value} task you choose!\"\n",
        "\n",
        "    ######################################################################\n",
        "    #            Start training              #\n",
        "    ######################################################################\n",
        "    from saprot.config.config_dict import Default_config\n",
        "    config = copy.deepcopy(Default_config)\n",
        "\n",
        "    # training config\n",
        "    GPU_batch_size = 0\n",
        "    accumulate_grad_batches = 0\n",
        "    num_workers = 2\n",
        "    seed = 20000812\n",
        "\n",
        "    # lora config\n",
        "    r = lora_r.value\n",
        "    lora_dropout = lora_dropout_box.value\n",
        "    lora_alpha = lora_alpha_box.value\n",
        "\n",
        "    # dataset config\n",
        "    val_check_interval = val_check_interval_box.value\n",
        "    limit_train_batches = limit_train_batches_box.value\n",
        "    limit_val_batches = limit_val_batches_box.value\n",
        "    limit_test_batches = limit_test_batches_box.value\n",
        "\n",
        "    mask_struc_ratio=None\n",
        "    if torch.cuda.is_available() is False:\n",
        "      raise BaseException(\"Please switch your Runtime to a GPU!\")\n",
        "\n",
        "    # Task config\n",
        "    task_name_value = task_name.value.replace(\" \", \"_\")\n",
        "    task_type_value = task_type.value\n",
        "    original_task_type = task_type_value\n",
        "    task_type_value = task_type_dict[task_type_value]\n",
        "\n",
        "    #####################################################################\n",
        "    #            Model config               #\n",
        "    #####################################################################\n",
        "    base_model = model_type.value\n",
        "    # continue learning\n",
        "    if base_model in [\"Trained by yourself on ColabSaprot\", \"Shared by peers on SaprotHub\", \"Saved in your local computer\"]:\n",
        "      continue_learning = True\n",
        "    else:\n",
        "      continue_learning = False\n",
        "\n",
        "    # base_model\n",
        "    if continue_learning:\n",
        "      adapter_path = ADAPTER_HOME / model_arg\n",
        "      print(f\"Training on an existing model: {adapter_path}\")\n",
        "\n",
        "      if base_model == \"Shared by peers on SaprotHub\":\n",
        "        if not adapter_path.exists():\n",
        "          snapshot_download(repo_id=model_arg, repo_type=\"model\", local_dir=adapter_path)\n",
        "\n",
        "      adapter_config_path = Path(adapter_path) / \"adapter_config.json\"\n",
        "      assert adapter_config_path.exists(), f\"Can't find {adapter_config_path}\"\n",
        "      with open(adapter_config_path, 'r') as f:\n",
        "        adapter_config = json.load(f)\n",
        "        base_model = adapter_config['base_model_name_or_path']\n",
        "\n",
        "    elif base_model == \"Official SaProt (35M)\":\n",
        "      base_model = \"westlake-repl/SaProt_35M_AF2\"\n",
        "\n",
        "    elif base_model == \"Official SaProt (650M)\":\n",
        "      base_model = \"westlake-repl/SaProt_650M_AF2\"\n",
        "\n",
        "    # model size and model name\n",
        "    if base_model == \"westlake-repl/SaProt_650M_AF2\":\n",
        "      model_size = \"650M\"\n",
        "      model_name = f\"Model-{task_name_value}-{model_size}\"\n",
        "    elif base_model == \"westlake-repl/SaProt_35M_AF2\":\n",
        "      model_size = \"35M\"\n",
        "      model_name = f\"Model-{task_name_value}-{model_size}\"\n",
        "\n",
        "    config.setting.run_mode = \"train\"\n",
        "    config.setting.seed = seed\n",
        "\n",
        "    if task_type_value in [\"classification\", \"token_classification\", \"pair_classification\"]:\n",
        "      config.model.kwargs.num_labels = num_label.value\n",
        "\n",
        "    config.model.model_py_path = model_type_dict[task_type_value]\n",
        "    config.model.kwargs.config_path = base_model\n",
        "    config.dataset.kwargs.tokenizer = base_model\n",
        "\n",
        "    config.model.save_path = str(ADAPTER_HOME / \"Local\" / f\"{task_type_value}\" / model_name)\n",
        "\n",
        "    if task_type_value in [\"regression\", \"pair_regression\"]:\n",
        "      config.model.kwargs.extra_config = {}\n",
        "      config.model.kwargs.extra_config.attention_probs_dropout_prob=0\n",
        "      config.model.kwargs.extra_config.hidden_dropout_prob=0\n",
        "\n",
        "    config.model.kwargs.lora_kwargs = EasyDict({\n",
        "      \"is_trainable\": True,\n",
        "      \"num_lora\": 1,\n",
        "      \"r\": r,\n",
        "      \"lora_dropout\": lora_dropout,\n",
        "      \"lora_alpha\": lora_alpha,\n",
        "      \"config_list\": []})\n",
        "    if continue_learning:\n",
        "      config.model.kwargs.lora_kwargs.config_list.append({\"lora_config_path\": adapter_path})\n",
        "\n",
        "    #####################################################################\n",
        "    #            Dataset config              #\n",
        "    #####################################################################\n",
        "    if data_src_type.value == \"SaprotHub in HuggingFace\":\n",
        "      data_src_type_value = \"SaprotHub Dataset\"\n",
        "      raw_data = saprothub_data_id\n",
        "      csv_dataset_path = get_SA_sequence_by_data_type(data_src_type_value, raw_data)\n",
        "\n",
        "      # Get the meta-data of the dataset\n",
        "      metadata_path = DATASET_HOME / saprothub_data_id.value / 'metadata.json'\n",
        "      metadata = json.load(open(metadata_path, \"r\"))\n",
        "\n",
        "      # Check compatibility between chosen task and data\n",
        "      assert metadata[\"training_task_type\"] == task_type.value, f\"This dataset is for '{metadata['training_task_type']}', which is not suitable for your task '{task_type.value}'\"\n",
        "\n",
        "      # Check compatibility between chosen data and model\n",
        "      if model_type_value == \"Shared by peers on SaprotHub\" or model_type_value == \"Trained by yourself on ColabSaprot\":\n",
        "        model_data_type = load_data_type_from_model(model_type_value, model_arg)\n",
        "\n",
        "        if model_data_type != metadata[\"training_data_type\"]:\n",
        "          if model_data_type == \"SA\":\n",
        "            raise Exception(\"Error: The model you choose should be trained on protein structures. So you have to choose protein structure dataset!\")\n",
        "          else:\n",
        "            raise Exception(\"Error: The model you choose should be trained on protein sequences. So you have to choose protein sequence dataset!\")\n",
        "\n",
        "    else:\n",
        "      data_type_value = data_type.value\n",
        "      metadata = {\n",
        "          \"training_task_type\": task_type.value,\n",
        "          \"training_data_type\": \"AA\" if data_type_value == \"protein sequence\" else \"SA\"\n",
        "      }\n",
        "\n",
        "      csv_path = get_upload_file_path(upload_items_1)\n",
        "      assert csv_path.endswith(\".csv\"), \"Please upload file with correct format (.csv)!\"\n",
        "\n",
        "      if data_type_value == \"protein sequence\":\n",
        "        if \"Protein-protein\" not in task_type.value:\n",
        "          processed_data_type = \"Multiple AA Sequences\"\n",
        "        else:\n",
        "          processed_data_type = \"Multiple pairs of AA Sequences\"\n",
        "\n",
        "        tmp_path = f\"{csv_path}.tmp\"\n",
        "        os.system(f\"cp {csv_path} {tmp_path}\")\n",
        "        csv_dataset_path = get_SA_sequence_by_data_type(processed_data_type, tmp_path)\n",
        "\n",
        "        # df = pd.read_csv(csv_dataset_path)\n",
        "        # print(df)\n",
        "        # raise\n",
        "\n",
        "      else:\n",
        "        zip_path = get_upload_file_path(upload_items_2)\n",
        "        zip_name = os.path.basename(zip_path)\n",
        "        assert zip_path.endswith(\".zip\"), \"Please upload file with correct format (.zip)!\"\n",
        "\n",
        "        # Unzip structures\n",
        "        upload_path = Path(UPLOAD_FILE_HOME)\n",
        "        upload_path.mkdir(parents=True, exist_ok=True)\n",
        "        prefix = zip_name.rsplit(\".\", 1)[0]\n",
        "        struc_dir = upload_path / prefix\n",
        "        with zipfile.ZipFile(zip_path, 'r') as zip_ref:\n",
        "          zip_ref.extractall(struc_dir)\n",
        "\n",
        "        # If the unzipped file is a directory\n",
        "        unzipped_files = os.listdir(struc_dir)\n",
        "        if len(unzipped_files) == 1:\n",
        "          file_path = f\"{struc_dir}/{unzipped_files[0]}\"\n",
        "          if os.path.isdir(file_path):\n",
        "            struc_dir = file_path\n",
        "\n",
        "        df = pd.read_csv(csv_path)\n",
        "        if \"Protein-protein\" not in task_type.value:\n",
        "          pdbs = []\n",
        "          for pdb_name, chain in df[[\"protein\", \"chain\"]].values:\n",
        "            pdb_path = f\"{struc_dir}/{pdb_name}\"\n",
        "            pdbs.append([pdb_path, chain])\n",
        "\n",
        "          sa_seqs = pdb2sa(pdbs)\n",
        "          df[\"protein\"] = sa_seqs\n",
        "\n",
        "        else:\n",
        "          pdbs = []\n",
        "          for pdb_name, chain in df[[\"protein_1\", \"chain_1\"]].values:\n",
        "            pdb_path = f\"{struc_dir}/{pdb_name}\"\n",
        "            pdbs.append([pdb_path, chain])\n",
        "\n",
        "          sa_seqs = pdb2sa(pdbs)\n",
        "          df[\"name_1\"] = df[\"protein_1\"]\n",
        "          df[\"protein_1\"] = sa_seqs\n",
        "\n",
        "          pdbs = []\n",
        "          for pdb_name, chain in df[[\"protein_2\", \"chain_2\"]].values:\n",
        "            pdb_path = f\"{struc_dir}/{pdb_name}\"\n",
        "            pdbs.append([pdb_path, chain])\n",
        "\n",
        "          sa_seqs = pdb2sa(pdbs)\n",
        "          df[\"name_2\"] = df[\"protein_2\"]\n",
        "          df[\"protein_2\"] = sa_seqs\n",
        "\n",
        "        tmp_path = f\"{csv_path}.tmp\"\n",
        "        os.system(f\"cp {csv_path} {tmp_path}\")\n",
        "        df.to_csv(tmp_path, index=False)\n",
        "        csv_dataset_path = tmp_path\n",
        "\n",
        "    # Rename columns to match backend functions\n",
        "    df = pd.read_csv(csv_dataset_path)\n",
        "    if \"Protein-protein\" in task_type.value:\n",
        "      df = df.rename(columns={\"protein_1\": \"sequence_1\", \"protein_2\": \"sequence_2\"})\n",
        "    else:\n",
        "      df = df.rename(columns={\"protein\": \"sequence\"})\n",
        "\n",
        "    if \"stage\" not in df.columns:\n",
        "      print(\"The 'stage' column was not provided. We will randomly split the data into training, validation and test set.\")\n",
        "      train_len = int(0.8 * len(df))\n",
        "      valid_len = int(0.1 * len(df))\n",
        "      test_len = len(df) - train_len - valid_len\n",
        "      stage = [\"train\"] * train_len + [\"valid\"] * valid_len + [\"test\"] * test_len\n",
        "\n",
        "      # assert train_len > 0 and valid_len > 0 and test_len > 0, \"Please provide more data for splitting\"\n",
        "      if train_len == 0 or valid_len == 0 or test_len == 0:\n",
        "        red_print(\"Error: The size of your data is too small. Please provide more data for training (>= 10).\")\n",
        "        return\n",
        "\n",
        "\n",
        "      # Shuffle the stage labels\n",
        "      random.shuffle(stage)\n",
        "      df[\"stage\"] = stage\n",
        "\n",
        "    csv_for_lmdb_path = f\"{csv_dataset_path}.for_lmdb\"\n",
        "    df.to_csv(csv_for_lmdb_path, index=False)\n",
        "    check_column_label_and_stage(csv_for_lmdb_path)\n",
        "\n",
        "    # Check the number of labels for classification task\n",
        "    if task_type_value in [\"classification\", \"token_classification\", \"pair_classification\"]:\n",
        "      max_label = 0\n",
        "\n",
        "      try:\n",
        "        for label in df[\"label\"].values:\n",
        "          if task_type_value == \"token_classification\":\n",
        "            label_ids = [int(i.strip()) for i in label.strip().split(\",\")]\n",
        "            max_label = max([max_label] + label_ids)\n",
        "\n",
        "          else:\n",
        "            max_label = max(max_label, int(label))\n",
        "\n",
        "      except Exception:\n",
        "        raise Exception(\"Please make sure you upload the correct data according to the task type you choose!\")\n",
        "\n",
        "      assert num_label.value > max_label, f\"The number of category you set ({num_label.value}) should match the dataset you upload ({max_label+1})!\"\n",
        "\n",
        "    from saprot.utils.construct_lmdb import construct_lmdb\n",
        "    try:\n",
        "      construct_lmdb(csv_for_lmdb_path, LMDB_HOME, task_name_value, task_type_value)\n",
        "\n",
        "    except Exception:\n",
        "      raise Exception(\"Please make sure you upload the correct data according to the task type you choose!\")\n",
        "\n",
        "    lmdb_dataset_path = LMDB_HOME / task_name_value\n",
        "\n",
        "    config.dataset.dataset_py_path = dataset_type_dict[task_type_value]\n",
        "\n",
        "    config.dataset.train_lmdb = str(lmdb_dataset_path / \"train\")\n",
        "    config.dataset.valid_lmdb = str(lmdb_dataset_path / \"valid\")\n",
        "    config.dataset.test_lmdb = str(lmdb_dataset_path / \"test\")\n",
        "\n",
        "    # num_workers\n",
        "    config.dataset.dataloader_kwargs.num_workers = num_workers\n",
        "\n",
        "    def get_accumulate_grad_samples(num_samples):\n",
        "        if num_samples > 3200:\n",
        "            return 64\n",
        "        elif 1600 < num_samples <= 3200:\n",
        "            return 32\n",
        "        elif 800 < num_samples <= 1600:\n",
        "            return 16\n",
        "        elif 400 < num_samples <= 800:\n",
        "            return 8\n",
        "        elif 200 < num_samples <= 400:\n",
        "            return 4\n",
        "        elif 100 < num_samples <= 200:\n",
        "            return 2\n",
        "        else:\n",
        "            return 1\n",
        "\n",
        "    # advanced config\n",
        "    if (GPU_batch_size > 0) and (accumulate_grad_batches > 0):\n",
        "      config.dataset.dataloader_kwargs.batch_size = GPU_batch_size\n",
        "      config.Trainer.accumulate_grad_batches= accumulate_grad_batches\n",
        "\n",
        "    elif (GPU_batch_size == 0) and (accumulate_grad_batches == 0):\n",
        "\n",
        "      # batch_size\n",
        "      # if base_model == \"westlake-repl/SaProt_650M_AF2\" and root_dir == \"/content\":\n",
        "      #   GPU_batch_size = 1\n",
        "      # else:\n",
        "      #   GPU_batch_size_dict = {\n",
        "      #     \"Tesla T4\": 2,\n",
        "      #     \"NVIDIA L4\": 2,\n",
        "      #     \"NVIDIA A100-SXM4-40GB\": 4,\n",
        "      #     }\n",
        "      #   GPU_name = torch.cuda.get_device_name(0)\n",
        "      #   GPU_batch_size = GPU_batch_size_dict[GPU_name] if GPU_name in GPU_batch_size_dict else 2\n",
        "\n",
        "      #   if task_type_value in [\"pair_classification\", \"pair_regression\"]:\n",
        "      #     GPU_batch_size = int(max(GPU_batch_size / 2, 1))\n",
        "      GPU_batch_size = 1\n",
        "      config.dataset.dataloader_kwargs.batch_size = GPU_batch_size\n",
        "\n",
        "      # accumulate_grad_batches\n",
        "      if batch_size.value == \"Adaptive\":\n",
        "        num_samples = get_length(config.dataset.train_lmdb)\n",
        "        accumulate_grad_samples = get_accumulate_grad_samples(num_samples)\n",
        "\n",
        "      else:\n",
        "        accumulate_grad_samples = int(batch_size.value)\n",
        "\n",
        "      accumulate_grad_batches = max(int(accumulate_grad_samples / GPU_batch_size), 1)\n",
        "\n",
        "      config.Trainer.accumulate_grad_batches= accumulate_grad_batches\n",
        "\n",
        "      # For test\n",
        "      config.Trainer.accumulate_grad_batches= 64\n",
        "      config.dataset.dataloader_kwargs.batch_size = 1\n",
        "\n",
        "    else:\n",
        "      raise BaseException(f\"Please make sure `GPU_batch_size`({GPU_batch_size}) and `accumulate_grad_batches`({accumulate_grad_batches}) are both greater than zero!\")\n",
        "\n",
        "    #####################################################################\n",
        "    #          Hyper-parameter config            #\n",
        "    #####################################################################\n",
        "    epoch_value = epoch.value\n",
        "    lr_value = lr.value\n",
        "\n",
        "    #####################################################################\n",
        "    #               Trainer               #\n",
        "    #####################################################################\n",
        "    config.Trainer.accelerator = \"gpu\" if torch.cuda.is_available() else \"cpu\"\n",
        "\n",
        "    # epoch\n",
        "    config.Trainer.max_epochs = epoch_value\n",
        "    # test only: load the existing model\n",
        "    if config.Trainer.max_epochs == 0 and continue_learning:\n",
        "      config.model.save_path = config.model.kwargs.lora_kwargs.config_list[0]['lora_config_path']\n",
        "\n",
        "    # learning rate\n",
        "    config.model.lr_scheduler_kwargs.init_lr = lr_value\n",
        "\n",
        "    # trainer\n",
        "    config.Trainer.limit_train_batches=limit_train_batches\n",
        "    config.Trainer.limit_val_batches=limit_val_batches\n",
        "    config.Trainer.limit_test_batches=limit_test_batches\n",
        "    config.Trainer.val_check_interval=val_check_interval\n",
        "\n",
        "    # strategy\n",
        "    strategy = {\n",
        "        # - deepspeed\n",
        "        # 'class': 'DeepSpeedStrategy',\n",
        "        # 'stage': 2\n",
        "\n",
        "        # - None\n",
        "        # 'class': None,\n",
        "\n",
        "        # - DP\n",
        "        # 'class': 'DataParallelStrategy',\n",
        "\n",
        "        # - DDP\n",
        "        # 'class': 'DDPStrategy',\n",
        "        # 'find_unused_parameter': True\n",
        "    }\n",
        "    config.Trainer.strategy = strategy\n",
        "\n",
        "    ####################################################################\n",
        "    #          Run the training task           #\n",
        "    ####################################################################\n",
        "    print('='*100)\n",
        "    print(Fore.BLUE+f\"Training task type: {task_type_value}\"+Style.RESET_ALL)\n",
        "    print(Fore.BLUE+f\"Dataset: {lmdb_dataset_path}\"+Style.RESET_ALL)\n",
        "    print(Fore.BLUE+f\"Base Model: {config.model.kwargs.config_path}\"+Style.RESET_ALL)\n",
        "    if continue_learning:\n",
        "      print(Fore.BLUE+f\"Existing model: {config.model.kwargs.lora_kwargs.config_list[0]['lora_config_path']}\"+Style.RESET_ALL)\n",
        "    print('='*100)\n",
        "    pprint.pprint(config)\n",
        "    print('='*100)\n",
        "\n",
        "    # Save the metadata file\n",
        "    def add_training_data_type_to_config(metadata_path, metadata):\n",
        "      with open(metadata_path, 'w') as file:\n",
        "          json.dump(metadata, file, indent=4)\n",
        "    metadata_path = Path(config.model.save_path) / \"metadata.json\"\n",
        "    os.makedirs(config.model.save_path, exist_ok=True)\n",
        "    add_training_data_type_to_config(metadata_path, metadata)\n",
        "\n",
        "    from saprot.scripts.training import finetune\n",
        "    finetune(config)\n",
        "\n",
        "\n",
        "    ####################################################################\n",
        "    #            Modify README              #\n",
        "    ####################################################################\n",
        "    name = model_name\n",
        "    description = '<slot name=\\'description\\'>'\n",
        "\n",
        "    with open(f'{config.model.save_path}/adapter_config.json', 'r') as f:\n",
        "      lora_config = json.load(f)\n",
        "\n",
        "    readme = f'''\n",
        "---\n",
        "\n",
        "base_model: {base_model} \\n\n",
        "library_name: peft\n",
        "\n",
        "---\n",
        "\\n\n",
        "\n",
        "# Model Card for {name}\n",
        "{description}\n",
        "\n",
        "## Task type\n",
        "{original_task_type}\n",
        "\n",
        "## Model input type\n",
        "{metadata[\"training_data_type\"]} Sequence\n",
        "\n",
        "## LoRA config\n",
        "\n",
        "- **r:** {lora_config['r']}\n",
        "- **lora_dropout:** {lora_config['lora_dropout']}\n",
        "- **lora_alpha:** {lora_config['lora_alpha']}\n",
        "- **target_modules:** {lora_config['target_modules']}\n",
        "- **modules_to_save:** {lora_config['modules_to_save']}\n",
        "\n",
        "## Training config\n",
        "\n",
        "- **optimizer:**\n",
        "  - **class:** AdamW\n",
        "  - **betas:** (0.9, 0.98)\n",
        "  - **weight_decay:** 0.01\n",
        "- **learning rate:** {config.model.lr_scheduler_kwargs.init_lr}\n",
        "- **epoch:** {config.Trainer.max_epochs}\n",
        "- **batch size:** {config.dataset.dataloader_kwargs.batch_size * config.Trainer.accumulate_grad_batches}\n",
        "- **precision:** 16-mixed \\n\n",
        "'''\n",
        "    # Write the markdown output to a file\n",
        "    with open(f\"{config.model.save_path}/README.md\", \"w\") as file:\n",
        "      file.write(readme)\n",
        "\n",
        "    ####################################################################\n",
        "    #            Save the adapter            #\n",
        "    ####################################################################\n",
        "    print(Fore.BLUE)\n",
        "    print(f\"Model is saved to \\\"{config.model.save_path}\\\" on Colab Server\")\n",
        "    print(Style.RESET_ALL)\n",
        "\n",
        "    adapter_zip = Path(config.model.save_path) / f\"{model_name}.zip\"\n",
        "    cmd = f\"cd {config.model.save_path} && zip -r {adapter_zip} 'adapter_config.json' 'adapter_model.safetensors' 'README.md' 'metadata.json'\"\n",
        "    os.system(cmd)\n",
        "    # !cd $config.model.save_path && zip -r $adapter_zip \"adapter_config.json\" \"adapter_model.safetensors\" \"README.md\" \"metadata.json\"\n",
        "    # !cd $config.model.save_path && zip -r $adapter_zip \"adapter_config.json\" \"adapter_model.safetensors\" \"adapter_model.bin\" \"README.md\" \"metadata.json\"\n",
        "    print(\"Click to download the model to your local computer\")\n",
        "    if adapter_zip.exists():\n",
        "      file_download(adapter_zip)\n",
        "\n",
        "    finish_hint = HTML(markdown.markdown(\n",
        "        f\"## The training is completed, you can then:\\n\\n\"\n",
        "        \"- **Train other models or continually train this model with more epochs:**\\n\\n\"\n",
        "        \"\\t1. Click the ``Refresh`` button.\\n\\n\"\n",
        "        \"\\t2. Reset your training configuration and start a new training task.\\n\\n\"\n",
        "        \"- **Use this model for prediction:**\\n\\n\"\n",
        "        \"\\t1. Click the ``Go back`` button.\\n\\n\"\n",
        "        \"\\t2. Choose ``I want to use existing models to make prediction`` module.\\n\\n\"\n",
        "        \"\\t3. Choose ``Protein property prediction`` module.\\n\\n\"\n",
        "        \"\\t4. Set the base model option to ``Trained by yourself on ColabSaprot`` and then choose the model you trained for prediction.\\n\\n\"\n",
        "        \"- **Upload this trained model to SaprotHub:**\\n\\n\"\n",
        "        \"\\t1. Click the ``Go back`` button.\\n\\n\"\n",
        "        \"\\t2. Choose ``I want to share my model publicly`` module.\\n\\n\"\n",
        "        \"\\t3. Complete a basic description of your model and upload it.\"\n",
        "        ))\n",
        "    display(finish_hint)\n",
        "\n",
        "  # Set click events\n",
        "  task_type.observe(change_task_type, names='value')\n",
        "  model_type.observe(change_model_type, names='value')\n",
        "  data_type.observe(change_data_type, names=\"value\")\n",
        "  data_src_type.observe(change_data_src_type, names='value')\n",
        "  advanced_setting.on_click(click_advanced_setting)\n",
        "\n",
        "  start_btn.on_click(\n",
        "      # disable_wrapper(\n",
        "      #     lambda btn: start_thread(start_training, (btn,))\n",
        "      #     )\n",
        "      lambda btn: start_thread(start_training, (btn,))\n",
        "      )\n",
        "\n",
        "  # Set default state\n",
        "  for item in [lora_hint, lora_r, lora_dropout_box, lora_alpha_box, trainer_hint, limit_train_batches_box, limit_val_batches_box, limit_test_batches_box,]:\n",
        "    item.layout.display = \"none\"\n",
        "\n",
        "  display(*items)\n",
        "\n",
        "\n",
        "######################################################################\n",
        "#          Choose prediction task              #\n",
        "######################################################################\n",
        "def choose_pred_task():\n",
        "  global refresh_module\n",
        "  refresh_module = choose_pred_task\n",
        "\n",
        "  WIDTH = \"500px\"\n",
        "\n",
        "  question = HTML(markdown.markdown(\"## ColabSaprot supports multiple prediction tasks, which one would you like to choose?\"))\n",
        "  normal_pred = Button(description='Protein property prediction', layout=Layout(width='500px', height='30px'), button_style=\"info\")\n",
        "  normal_intro = HTML(markdown.markdown(\n",
        "    f\"This section enables property prediction using well-trained models for various protein characteristics, \"\n",
        "    \"including stability, subcellular localization, and solubility, etc. Users can leverage either models \"\n",
        "    \"available in <a href='https://huggingface.co/SaProtHub' target='blank'>SaprotHub</a> or their previously trained models for predictions. The model input for \"\n",
        "    \"predicting should be consistent with its training format.\"), layout=Layout(width=WIDTH))\n",
        "\n",
        "  zeroshot_pred = Button(description='Mutational effect prediction', layout=Layout(width='500px', height='30px'), button_style=\"info\")\n",
        "  zeroshot_intro = HTML(markdown.markdown(\n",
        "  f\"The Mutational Effect Prediction section utilizes the Saprot 650M model. By analyzing the predicted mutation \"\n",
        "  \"scores, users can quickly identify mutations that are likely to enhance specific protein functions, such as \"\n",
        "  \"enzyme activity.\"\n",
        "  ), layout=Layout(width=WIDTH))\n",
        "\n",
        "  design_pred = Button(description='Protein sequence design', layout=Layout(width='500px', height='30px'), button_style=\"info\")\n",
        "  design_intro = HTML(markdown.markdown(\n",
        "    f\"The Sequence Design section enables generation of diverse protein sequences compatible with a given \"\n",
        "    \"structural backbone. Users can input their desired backbone coordinates and obtain novel sequences \"\n",
        "    \"optimized for that scaffold.\"\n",
        "    ), layout=Layout(width=WIDTH))\n",
        "\n",
        "  repr_pred = Button(description='Obtain protein-level embeddings', layout=Layout(width='500px', height='30px'), button_style=\"info\")\n",
        "  repr_intro = HTML(markdown.markdown(\n",
        "    f\"The Protein Embedding section enables extraction of sequence or structure embeddings using either \"\n",
        "    \"standard Saprot models or custom fine-tuned models for specialized analysis tasks.\"\n",
        "    ), layout=Layout(width=WIDTH))\n",
        "\n",
        "  back_btn = Button(description='Go back', layout=Layout(width='500px', height='30px'))\n",
        "\n",
        "  sep_hint = HTML(markdown.markdown(f\"### {'-'*75}\"))\n",
        "\n",
        "  items = [\n",
        "      question,\n",
        "      sep_hint,\n",
        "      normal_pred,\n",
        "      normal_intro,\n",
        "      sep_hint,\n",
        "      zeroshot_pred,\n",
        "      zeroshot_intro,\n",
        "      sep_hint,\n",
        "      design_pred,\n",
        "      design_intro,\n",
        "      sep_hint,\n",
        "      repr_pred,\n",
        "      repr_intro,\n",
        "      sep_hint,\n",
        "      # back_btn\n",
        "      ]\n",
        "\n",
        "  # Set click events\n",
        "  normal_pred.on_click(partial(jump, next=protein_property_prediction))\n",
        "  zeroshot_pred.on_click(partial(jump, next=start_mut_pred))\n",
        "  design_pred.on_click(partial(jump, next=protein_sequence_design))\n",
        "  repr_pred.on_click(partial(jump, next=obtain_protein_embedding))\n",
        "  back_btn.on_click(partial(jump, next=train_or_pred))\n",
        "\n",
        "  display(*items)\n",
        "\n",
        "\n",
        "######################################################################\n",
        "#          Protein property prediction          #\n",
        "######################################################################\n",
        "def protein_property_prediction():\n",
        "  global refresh_module\n",
        "  refresh_module = protein_property_prediction\n",
        "\n",
        "  WIDTH = \"500px\"\n",
        "  HEIGHT = \"30px\"\n",
        "  hint = HTML(markdown.markdown(\"## Protein property prediction\"))\n",
        "\n",
        "  task_hint = HTML(markdown.markdown(\"### Choose the prediction task:\"))\n",
        "  task_type_box = ipywidgets.Dropdown(\n",
        "            options=['Protein-level Classification', 'Protein-level Regression', 'Residue-level Classification', \"Protein-protein Classification\", \"Protein-protein Regression\"],\n",
        "            value='Protein-level Classification',\n",
        "            description='Task type:',\n",
        "            disabled=False,\n",
        "            layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "          )\n",
        "\n",
        "  task_intro_dict = {\n",
        "      'Protein-level Classification': \"<font color=red>What is **Protein-level Classification:** Given a protein, you have some categories and you want to predict which category the protein belongs to.</font>\",\n",
        "      \"Protein-level Regression\": \"<font color=red>What is **Protein-level Regression:** Given a protein, you want to predict a score about its property such as stability or enzyme activity.</font>\",\n",
        "      \"Residue-level Classification\": \"<font color=red>What is **Residue-level Classification:** Given a protein, you have some categories and for every amino acid you want to predict which category it belongs to.</font>\",\n",
        "      \"Protein-protein Classification\": \"<font color=red>What is **Protein-protein Classification:** Given a pair of proteins, you have some categories and you want to predict which category the pair belongs to.</font>\",\n",
        "      \"Protein-protein Regression\": \"<font color=red>What is **Protein-protein Regression:** Given a pair of proteins, you want to predict a score about its property such as binding affinity.</font>\"\n",
        "  }\n",
        "\n",
        "  task_intro = HTML(markdown.markdown(\n",
        "    \"<font color=red>What is **Protein-level Classification:** Given a protein, you have some categories and you want to predict which category the protein belongs to.</font>\"\n",
        "    ), layout=Layout(width=WIDTH))\n",
        "\n",
        "  num_label_box = ipywidgets.BoundedIntText(\n",
        "        value=2,\n",
        "        min=2,\n",
        "        max=100000000,\n",
        "        step=1,\n",
        "        description='Number of categories:',\n",
        "        disabled=False,\n",
        "        style={'description_width': 'initial'},\n",
        "        layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "      )\n",
        "\n",
        "  model_hint = HTML(markdown.markdown(\"### Choose the model for prediction:\"))\n",
        "  model_type_box = ipywidgets.Dropdown(\n",
        "            options=[\"Trained by yourself on ColabSaprot\", \"Shared by peers on SaprotHub\", \"Saved in your local computer\", \"Multi-models on SaprotHub\"],\n",
        "            value='Trained by yourself on ColabSaprot',\n",
        "            description='Base model:',\n",
        "            disabled=False,\n",
        "            layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "          )\n",
        "  model_arg_box = select_adapter_from(None, use_model_from=model_type_box.value)\n",
        "  model_arg_box.layout.width = WIDTH\n",
        "  model_arg_box.description = \"Select your local model:\"\n",
        "  model_arg_box.style = {'description_width': 'initial'}\n",
        "\n",
        "  upload_model_items = get_upload_box(btn_desc=\"Upload Model-<task_name>-<model_size>.zip file\")\n",
        "  set_upload_visibility(upload_model_items, mode=\"none\")\n",
        "\n",
        "  saprothub_link = HTML(markdown.markdown(\n",
        "      \"<font color=red>You could search models using our <a href='https://huggingface.co/spaces/SaProtHub/SaprotHub-search' target='blank'>search engine</a> or from <a href='https://huggingface.co/SaProtHub' target='blank'>SaprotHub</a>\\n\\n\"\n",
        "      \"A model id example: <a href='https://huggingface.co/SaProtHub/Model-Binary_Localization-650M' target='blank'>SaProtHub/Model-Binary_Localization-650M</a>\\n\\n\"\n",
        "      ), layout=Layout(display=\"none\"))\n",
        "\n",
        "  data_type_hint = HTML(markdown.markdown(\"### Uploaded data type:\"))\n",
        "  data_type_box = ipywidgets.RadioButtons(\n",
        "      options=['protein sequence', 'protein structure'],\n",
        "      value=\"protein structure\",\n",
        "      disabled=False,\n",
        "      style={'description_width': 'initial'},\n",
        "      )\n",
        "  saprothub_data_type_hint = HTML(markdown.markdown(\n",
        "    \"<font color=red>Note: For models with input type '(Structure-aware) sequence' in SaprotHub, protein structure input is required.</font>\",\n",
        "    ), layout=Layout(display=\"none\"))\n",
        "\n",
        "  upload_type_hint = HTML(markdown.markdown(\"### Choose the number of protein:\"))\n",
        "  upload_type_box = ipywidgets.RadioButtons(\n",
        "        options=['Single file', 'Multiple files'],\n",
        "        layout={'width': 'max-content'}, # If the items' names are long\n",
        "        disabled=False,\n",
        "        style={'description_width': 'initial'},\n",
        "        )\n",
        "\n",
        "  input_seq_hint = HTML(markdown.markdown(\"### Input proteins:\"))\n",
        "  input_seq_box_1 = ipywidgets.Text(\n",
        "                value=\"\",\n",
        "                placeholder=f'Input protein sequence',\n",
        "                disabled=False,\n",
        "                description=\"Protein sequence:\",\n",
        "                layout=Layout(width=WIDTH, height=HEIGHT, display=\"none\"),\n",
        "                style={'description_width': 'initial'},\n",
        "                )\n",
        "  input_seq_box_2 = ipywidgets.Text(\n",
        "                value=\"\",\n",
        "                placeholder=f'Input protein sequence',\n",
        "                disabled=False,\n",
        "                description=\"Protein sequence:\",\n",
        "                layout=Layout(width=WIDTH, height=HEIGHT, display=\"none\"),\n",
        "                style={'description_width': 'initial'},\n",
        "                )\n",
        "\n",
        "  chain_hint_1 = HTML(markdown.markdown(\"Chain (to be extracted from the structure):\"))\n",
        "  input_chain_1 = ipywidgets.Text(value=\"A\",placeholder=f'Enter the name of chain here', layout=Layout(width=\"250px\", height=HEIGHT))\n",
        "  upload_hint_1 = HTML(markdown.markdown(\"**Upload protein structure (.pdb / .cif file):**\"))\n",
        "  upload_items_1 = get_upload_box()\n",
        "\n",
        "  chain_hint_2 = HTML(markdown.markdown(\"Chain (to be extracted from the structure):\"), layout=Layout(display=\"none\"))\n",
        "  input_chain_2 = ipywidgets.Text(value=\"A\",placeholder=f'Enter the name of chain here', layout=Layout(width=\"250px\", height=HEIGHT, display=\"none\"))\n",
        "  upload_hint_2 = HTML(markdown.markdown(\"**Upload the second protein structure (.pdb / .cif file):**\"), layout=Layout(display=\"none\"))\n",
        "  upload_items_2 = get_upload_box()\n",
        "  set_upload_visibility(upload_items_2, mode=\"none\")\n",
        "\n",
        "  save_path_hint = HTML(layout={\"display\": \"none\"})\n",
        "  start_hint = HTML(markdown.markdown(f\"### {'-'*75}\"))\n",
        "  start_btn = Button(description='Start prediction', layout=Layout(width=WIDTH, height=HEIGHT), button_style=\"info\")\n",
        "\n",
        "  items = [\n",
        "      hint,\n",
        "      task_hint,\n",
        "      task_type_box,\n",
        "      num_label_box,\n",
        "      task_intro,\n",
        "      model_hint,\n",
        "      model_type_box,\n",
        "      model_arg_box,\n",
        "      saprothub_link,\n",
        "      data_type_hint,\n",
        "      data_type_box,\n",
        "      saprothub_data_type_hint,\n",
        "      upload_type_hint,\n",
        "      upload_type_box,\n",
        "      input_seq_hint,\n",
        "      input_seq_box_1,\n",
        "      input_seq_box_2,\n",
        "      upload_hint_1,\n",
        "      ipywidgets.HBox([chain_hint_1, input_chain_1]),\n",
        "      *upload_items_1,\n",
        "      upload_hint_2,\n",
        "      ipywidgets.HBox([chain_hint_2, input_chain_2]),\n",
        "      *upload_items_2,\n",
        "      save_path_hint,\n",
        "      start_hint,\n",
        "      start_btn\n",
        "      ]\n",
        "\n",
        "  # Used to replace input box based on model type\n",
        "  model_arg_box_idx = items.index(model_arg_box)\n",
        "\n",
        "  # Set click events\n",
        "  def change_task_type(change):\n",
        "    set_input_format()\n",
        "    now_type = change[\"new\"]\n",
        "    task_intro.value = markdown.markdown(task_intro_dict[now_type])\n",
        "\n",
        "    if \"Classification\" in now_type:\n",
        "      num_label_box.layout.display = None\n",
        "    else:\n",
        "      num_label_box.layout.display = \"none\"\n",
        "\n",
        "  def change_model_type(change):\n",
        "      model_type_value = change[\"new\"]\n",
        "      if model_type_value == \"Shared by peers on SaprotHub\" or model_type_value == \"Multi-models on SaprotHub\":\n",
        "        saprothub_data_type_hint.layout.display = None\n",
        "      else:\n",
        "        saprothub_data_type_hint.layout.display = \"none\"\n",
        "\n",
        "      if model_type_value == \"Trained by yourself on ColabSaprot\":\n",
        "        new_model_arg_box = select_adapter_from(None, use_model_from=model_type_value)\n",
        "        new_model_arg_box.layout.width = WIDTH\n",
        "        new_model_arg_box.description = \"Select your local model:\"\n",
        "        new_model_arg_box.style = {'description_width': 'initial'}\n",
        "        saprothub_link.layout.display = \"none\"\n",
        "\n",
        "      elif model_type_value == \"Shared by peers on SaprotHub\":\n",
        "        new_model_arg_box = select_adapter_from(None, use_model_from=model_type_value)\n",
        "        new_model_arg_box.layout.width = WIDTH\n",
        "        new_model_arg_box.placeholder = \"Enter SaprotHub model id in HuggingFace\"\n",
        "        new_model_arg_box.description = \"Model id:\"\n",
        "        saprothub_link.layout.display = None\n",
        "\n",
        "      elif model_type_value == \"Saved in your local computer\":\n",
        "        set_upload_visibility(upload_model_items, \"default\")\n",
        "        new_model_arg_box = upload_model_items\n",
        "        saprothub_link.layout.display = \"none\"\n",
        "\n",
        "      elif model_type_value == \"Multi-models on SaprotHub\":\n",
        "        new_model_arg_box = ipywidgets.Textarea(\n",
        "                        value=None,\n",
        "                        placeholder='Enter SaprotHub model ids, one line for one model.\\ne.g.:\\nSaprotHub/model_1\\nSaprotHub/model_2\\n\\nNote: These models should have the same model size, i.e. both are 650M or 35M.',\n",
        "                        description='SaprotHub model ids:',\n",
        "                        disabled=False,\n",
        "                        style={'description_width': 'initial'},\n",
        "                        layout=Layout(width=WIDTH, height=\"140px\"),\n",
        "                        )\n",
        "        saprothub_link.layout.display = \"none\"\n",
        "\n",
        "      else:\n",
        "        new_model_arg_box = HTML(markdown.markdown(\"dummy box\"))\n",
        "        new_model_arg_box.layout.display = \"none\"\n",
        "        saprothub_link.layout.display = \"none\"\n",
        "\n",
        "      if isinstance(new_model_arg_box, list):\n",
        "        new_items = items[:model_arg_box_idx] + new_model_arg_box + items[model_arg_box_idx+1:]\n",
        "        custom_display(*new_items)\n",
        "\n",
        "      else:\n",
        "        items[model_arg_box_idx] = new_model_arg_box\n",
        "        custom_display(*items)\n",
        "\n",
        "  def set_input_format():\n",
        "    data_type = data_type_box.value\n",
        "    upload_type = upload_type_box.value\n",
        "    task_type = task_type_box.value\n",
        "    if \"Protein-protein\" not in task_type:\n",
        "      chain_hint_2.layout.display = \"none\"\n",
        "      input_chain_2.layout.display = \"none\"\n",
        "      upload_hint_2.layout.display = \"none\"\n",
        "      if data_type == \"protein sequence\":\n",
        "        if upload_type == \"Single file\":\n",
        "          chain_hint_1.layout.display = \"none\"\n",
        "          input_chain_1.layout.display = \"none\"\n",
        "          upload_hint_1.layout.display = \"none\"\n",
        "          set_upload_visibility(upload_items_1, mode=\"none\")\n",
        "          chain_hint_2.layout.display = \"none\"\n",
        "          input_chain_2.layout.display = \"none\"\n",
        "          upload_hint_2.layout.display = \"none\"\n",
        "          set_upload_visibility(upload_items_2, mode=\"none\")\n",
        "          input_seq_box_1.layout.display = None\n",
        "          input_seq_box_1.description = \"Protein sequence:\"\n",
        "          input_seq_box_2.layout.display = \"none\"\n",
        "\n",
        "        else:\n",
        "          chain_hint_1.layout.display = \"none\"\n",
        "          input_chain_1.layout.display = \"none\"\n",
        "          chain_hint_2.layout.display = \"none\"\n",
        "          input_chain_2.layout.display = \"none\"\n",
        "          upload_hint_2.layout.display = \"none\"\n",
        "          set_upload_visibility(upload_items_2, mode=\"none\")\n",
        "          upload_hint_1.layout.display = None\n",
        "          set_upload_visibility(upload_items_1, mode=\"default\")\n",
        "          input_seq_box_1.layout.display = \"none\"\n",
        "          input_seq_box_2.layout.display = \"none\"\n",
        "          upload_hint_1.value = markdown.markdown(\n",
        "              \"<img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/prediction/dataset/new_aa.jpg?raw=true' height='200px' width='300px' align='center'>\\n\\n\"\n",
        "              \"Please upload a ``.csv`` file that contains all protein sequences to predict. Please strictly follow the format above **(column names are also needed in the csv file)**. \"\n",
        "              \"Then click the start button.\"\n",
        "              )\n",
        "\n",
        "      else:\n",
        "        upload_hint_1.layout.display = None\n",
        "        set_upload_visibility(upload_items_1, mode=\"default\")\n",
        "        input_seq_box_1.layout.display = \"none\"\n",
        "        input_seq_box_2.layout.display = \"none\"\n",
        "        if upload_type == \"Single file\":\n",
        "          set_upload_visibility(upload_items_2, mode=\"none\")\n",
        "          chain_hint_1.layout.display = None\n",
        "          input_chain_1.layout.display = None\n",
        "          upload_hint_1.value = markdown.markdown(\"**Upload protein structure:**\")\n",
        "\n",
        "        else:\n",
        "          chain_hint_1.layout.display = \"none\"\n",
        "          input_chain_1.layout.display = \"none\"\n",
        "          set_upload_visibility(upload_items_2, mode=\"default\")\n",
        "          upload_hint_1.value = markdown.markdown(\n",
        "              \"<img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/prediction/dataset/new_pdb.jpg?raw=true' height='200px' width='500px' align='center'>\\n\\n\"\n",
        "              \"Please upload a ``.csv`` file and a ``.zip`` file that stores all protein structures to predict. Please strictly follow the format above **(column names are also needed in the csv file)**. \"\n",
        "              \"Then click the start button.\\n\\n**Upload .csv file:**\"\n",
        "              )\n",
        "          upload_hint_2.layout.display = None\n",
        "          upload_hint_2.value = markdown.markdown(\"**Upload .zip file:**\")\n",
        "    else:\n",
        "      if data_type == \"protein sequence\":\n",
        "        if upload_type == \"Single file\":\n",
        "          upload_hint_1.layout.display = \"none\"\n",
        "          set_upload_visibility(upload_items_1, mode=\"none\")\n",
        "          upload_hint_2.layout.display = \"none\"\n",
        "          set_upload_visibility(upload_items_2, mode=\"none\")\n",
        "          chain_hint_1.layout.display = \"none\"\n",
        "          input_chain_1.layout.display = \"none\"\n",
        "          chain_hint_2.layout.display = \"none\"\n",
        "          input_chain_2.layout.display = \"none\"\n",
        "          input_seq_box_1.layout.display = None\n",
        "          input_seq_box_2.layout.display = None\n",
        "          input_seq_box_1.description = \"Protein sequence 1:\"\n",
        "          input_seq_box_2.description = \"Protein sequence 2:\"\n",
        "\n",
        "        else:\n",
        "          upload_hint_1.layout.display = None\n",
        "          set_upload_visibility(upload_items_1, mode=\"default\")\n",
        "          upload_hint_2.layout.display = \"none\"\n",
        "          set_upload_visibility(upload_items_2, mode=\"none\")\n",
        "          chain_hint_1.layout.display = \"none\"\n",
        "          input_chain_1.layout.display = \"none\"\n",
        "          chain_hint_2.layout.display = \"none\"\n",
        "          input_chain_2.layout.display = \"none\"\n",
        "          input_seq_box_1.layout.display = \"none\"\n",
        "          input_seq_box_2.layout.display = \"none\"\n",
        "          upload_hint_1.value = markdown.markdown(\n",
        "              \"<img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/prediction/dataset/pair_aa.jpg?raw=true' height='240px' width='300px' align='center'>\\n\\n\"\n",
        "              \"Please upload a ``.csv`` file that contains all protein sequences to predict. Please strictly follow the format above **(column names are also needed in the csv file)**. \"\n",
        "              \"Then click the start button.\"\n",
        "              )\n",
        "\n",
        "      else:\n",
        "        upload_hint_1.layout.display = None\n",
        "        set_upload_visibility(upload_items_1, mode=\"default\")\n",
        "        input_seq_box_1.layout.display = \"none\"\n",
        "        input_seq_box_2.layout.display = \"none\"\n",
        "        if upload_type == \"Single file\":\n",
        "          upload_hint_2.layout.display = None\n",
        "          set_upload_visibility(upload_items_2, mode=\"default\")\n",
        "          chain_hint_1.layout.display = None\n",
        "          input_chain_1.layout.display = None\n",
        "          chain_hint_2.layout.display = None\n",
        "          input_chain_2.layout.display = None\n",
        "          upload_hint_1.value = markdown.markdown(\"**Upload protein structure 1:**\")\n",
        "          upload_hint_2.value = markdown.markdown(\"**Upload protein structure 2:**\")\n",
        "\n",
        "        else:\n",
        "          chain_hint_1.layout.display = \"none\"\n",
        "          input_chain_1.layout.display = \"none\"\n",
        "          chain_hint_2.layout.display = \"none\"\n",
        "          input_chain_2.layout.display = \"none\"\n",
        "          set_upload_visibility(upload_items_2, mode=\"default\")\n",
        "          upload_hint_1.value = markdown.markdown(\n",
        "              \"<img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/prediction/dataset/pair_pdb.jpg?raw=true' height='200px' width='900px' align='center'>\\n\\n\"\n",
        "              \"Please upload a ``.csv`` file and a ``.zip`` file that stores all protein structures to predict. Please strictly follow the format above **(column names are also needed in the csv file)**. \"\n",
        "              \"Then click the start button.\\n\\n**Upload .csv file:**\"\n",
        "              )\n",
        "          upload_hint_2.layout.display = None\n",
        "          upload_hint_2.value = markdown.markdown(\"**Upload .zip file:**\")\n",
        "\n",
        "  def change_data_type(change):\n",
        "    set_input_format()\n",
        "\n",
        "  def change_upload_type(change):\n",
        "    set_input_format()\n",
        "\n",
        "  def start_prediction(button):\n",
        "    print(\"Start prediction....\")\n",
        "\n",
        "    task_type = task_type_box.value\n",
        "    model_type = model_type_box.value\n",
        "    model_arg = items[model_arg_box_idx].value\n",
        "\n",
        "    if model_type == \"Saved in your local computer\":\n",
        "      zip_path = get_upload_file_path(upload_model_items)\n",
        "      name = os.path.basename(zip_path)\n",
        "      save_dir = ADAPTER_HOME / \"Upload\" / name.rsplit('.', 1)[0]\n",
        "      os.makedirs(save_dir, exist_ok=True)\n",
        "      # unzip model.zip\n",
        "      with zipfile.ZipFile(zip_path, 'r') as zip_ref:\n",
        "          zip_ref.extractall(save_dir)\n",
        "      os.remove(zip_path)\n",
        "      model_arg = save_dir\n",
        "\n",
        "    if model_type == \"Multi-models on SaprotHub\":\n",
        "      model_arg_list = model_arg.strip().split(\"\\n\")\n",
        "      for arg in model_arg_list:\n",
        "        model_task_type = load_task_type_from_model(model_type, arg.strip())\n",
        "        # Check compatibility between chosen task and model\n",
        "        assert task_type == model_task_type, f\"The model {arg.strip()} was trained on {model_task_type} task, which is not suitable for the {task_type} task you choose!\"\n",
        "\n",
        "    else:\n",
        "      model_task_type = load_task_type_from_model(model_type, model_arg)\n",
        "      # Check compatibility between chosen task and model\n",
        "      assert task_type == model_task_type, f\"The model you choose was trained on {model_task_type} task, which is not suitable for the {task_type} task you choose!\"\n",
        "\n",
        "    # Check data type compatibility\n",
        "    input_data_type = \"AA\" if data_type_box.value == \"protein sequence\" else \"SA\"\n",
        "\n",
        "    if model_type == \"Multi-models on SaprotHub\":\n",
        "      model_arg_list = model_arg.strip().split(\"\\n\")\n",
        "      for arg in model_arg_list:\n",
        "        model_data_type = load_data_type_from_model(model_type, arg.strip())\n",
        "        if model_data_type != input_data_type:\n",
        "          if model_data_type == \"SA\":\n",
        "            raise Exception(f\"Error: The model {arg.strip()} was trained on protein structures. So you have to upload protein structures!\")\n",
        "          else:\n",
        "            raise Exception(f\"Error: The model {arg.strip()} was trained on protein sequences. So you have to upload protein sequences!\")\n",
        "\n",
        "    else:\n",
        "      model_data_type = load_data_type_from_model(model_type, model_arg)\n",
        "      if model_data_type != input_data_type:\n",
        "        if model_data_type == \"SA\":\n",
        "          raise Exception(\"Error: The model you choose was trained on protein structures. So you have to upload protein structures!\")\n",
        "        else:\n",
        "          raise Exception(\"Error: The model you choose was trained on protein sequences. So you have to upload protein sequences!\")\n",
        "\n",
        "    # Process input\n",
        "    data_type = data_type_box.value\n",
        "    upload_type = upload_type_box.value\n",
        "    rows = []\n",
        "    if \"Protein-protein\" not in task_type and data_type == \"protein sequence\" and upload_type == \"Single file\":\n",
        "      df = pd.DataFrame(columns=[\"protein\"])\n",
        "      aa_seq = input_seq_box_1.value\n",
        "      df = df._append({\"protein\": aa_seq}, ignore_index=True)\n",
        "\n",
        "      sa_seq = \"\".join(aa + \"#\" for aa in aa_seq)\n",
        "      rows.append(sa_seq)\n",
        "\n",
        "    elif \"Protein-protein\" not in task_type and data_type == \"protein sequence\" and upload_type == \"Multiple files\":\n",
        "      csv_path = get_upload_file_path(upload_items_1)\n",
        "      assert csv_path.endswith(\".csv\"), \"Please upload file with correct format (.csv)!\"\n",
        "\n",
        "      df = pd.read_csv(csv_path)\n",
        "      rows = []\n",
        "      for aa_seq in df[\"protein\"].values:\n",
        "        sa_seq = \"\".join(aa + \"#\" for aa in aa_seq)\n",
        "        rows.append(sa_seq)\n",
        "\n",
        "    elif \"Protein-protein\" not in task_type and data_type == \"protein structure\" and upload_type == \"Single file\":\n",
        "      pdb_path = get_upload_file_path(upload_items_1)\n",
        "      name = os.path.basename(pdb_path)\n",
        "      assert name.endswith(\".pdb\") or name.endswith(\".cif\"), \"Please upload file with correct format (.pdb / .cif)!\"\n",
        "\n",
        "      chain = input_chain_1.value\n",
        "      sa_seq = get_struc_seq(FOLDSEEK_PATH, pdb_path, chains=[chain], plddt_mask=True)[chain][-1]\n",
        "      rows.append(sa_seq)\n",
        "\n",
        "      df = pd.DataFrame(columns=[\"protein\"])\n",
        "      df = df._append({\"protein\": name}, ignore_index=True)\n",
        "\n",
        "    elif \"Protein-protein\" not in task_type and data_type == \"protein structure\" and upload_type == \"Multiple files\":\n",
        "      csv_path = get_upload_file_path(upload_items_1)\n",
        "      csv_name = os.path.basename(csv_path)\n",
        "      zip_path = get_upload_file_path(upload_items_2)\n",
        "      zip_name = os.path.basename(zip_path)\n",
        "      assert csv_path.endswith(\".csv\"), \"Please upload file with correct format (.csv)!\"\n",
        "      assert zip_path.endswith(\".zip\"), \"Please upload file with correct format (.zip)!\"\n",
        "\n",
        "      # Unzip structures\n",
        "      upload_path = Path(UPLOAD_FILE_HOME)\n",
        "      upload_path.mkdir(parents=True, exist_ok=True)\n",
        "      prefix = zip_name.rsplit(\".\", 1)[0]\n",
        "      struc_dir = upload_path / prefix\n",
        "      with zipfile.ZipFile(zip_path, 'r') as zip_ref:\n",
        "          zip_ref.extractall(struc_dir)\n",
        "\n",
        "      # If the unzipped file is a directory\n",
        "      unzipped_files = os.listdir(struc_dir)\n",
        "      if len(unzipped_files) == 1:\n",
        "        file_path = f\"{struc_dir}/{unzipped_files[0]}\"\n",
        "        if os.path.isdir(file_path):\n",
        "          struc_dir = file_path\n",
        "\n",
        "      df = pd.read_csv(csv_path)\n",
        "      pdbs = []\n",
        "      for pdb_name, chain in df[[\"protein\", \"chain\"]].values:\n",
        "        pdb_path = f\"{struc_dir}/{pdb_name}\"\n",
        "        pdbs.append([pdb_path, chain])\n",
        "      rows = pdb2sa(pdbs)\n",
        "\n",
        "    elif \"Protein-protein\" in task_type and data_type == \"protein sequence\" and upload_type == \"Single file\":\n",
        "      df = pd.DataFrame(columns=[\"protein_1\", \"protein_2\"])\n",
        "      aa_seq_1 = input_seq_box_1.value\n",
        "      aa_seq_2 = input_seq_box_2.value\n",
        "      df = df._append({\"protein_1\": aa_seq_1, \"protein_2\": aa_seq_2}, ignore_index=True)\n",
        "\n",
        "      sa_seq_1 = \"\".join(aa + \"#\" for aa in aa_seq_1)\n",
        "      sa_seq_2 = \"\".join(aa + \"#\" for aa in aa_seq_2)\n",
        "      rows.append([sa_seq_1, sa_seq_2])\n",
        "\n",
        "    elif \"Protein-protein\" in task_type and data_type == \"protein sequence\" and upload_type == \"Multiple files\":\n",
        "      csv_path = get_upload_file_path(upload_items_1)\n",
        "      assert csv_path.endswith(\".csv\"), \"Please upload file with correct format (.csv)!\"\n",
        "\n",
        "      df = pd.read_csv(csv_path)\n",
        "      rows = []\n",
        "      for aa_seq_1, aa_seq_2 in df[[\"protein_1\", \"protein_2\"]].values:\n",
        "        sa_seq_1 = \"\".join(aa + \"#\" for aa in aa_seq_1)\n",
        "        sa_seq_2 = \"\".join(aa + \"#\" for aa in aa_seq_2)\n",
        "        rows.append([sa_seq_1, sa_seq_2])\n",
        "\n",
        "    elif \"Protein-protein\" in task_type and data_type == \"protein structure\" and upload_type == \"Single file\":\n",
        "      pdb_path_1 = get_upload_file_path(upload_items_1)\n",
        "      name_1 = os.path.basename(pdb_path_1)\n",
        "      pdb_path_2 = get_upload_file_path(upload_items_2)\n",
        "      name_2 = os.path.basename(pdb_path_2)\n",
        "      assert name_1.endswith(\".pdb\") or name_1.endswith(\".cif\"), \"Please upload file with correct format (.pdb / .cif)!\"\n",
        "      assert name_2.endswith(\".pdb\") or name_2.endswith(\".cif\"), \"Please upload file with correct format (.pdb / .cif)!\"\n",
        "\n",
        "      chain_1 = input_chain_1.value\n",
        "      sa_seq_1 = get_struc_seq(FOLDSEEK_PATH, pdb_path_1, chains=[chain_1], plddt_mask=True)[chain_1][-1]\n",
        "\n",
        "      chain_2 = input_chain_2.value\n",
        "      sa_seq_2 = get_struc_seq(FOLDSEEK_PATH, pdb_path_2, chains=[chain_2], plddt_mask=True)[chain_2][-1]\n",
        "\n",
        "      rows.append([sa_seq_1, sa_seq_2])\n",
        "\n",
        "      df = pd.DataFrame(columns=[\"protein_1\", \"protein_2\"])\n",
        "      df = df._append({\"protein_1\": name_1, \"protein_2\": name_2}, ignore_index=True)\n",
        "\n",
        "    elif \"Protein-protein\" in task_type and data_type == \"protein structure\" and upload_type == \"Multiple files\":\n",
        "      csv_path = get_upload_file_path(upload_items_1)\n",
        "      csv_name = os.path.basename(csv_path)\n",
        "      zip_path = get_upload_file_path(upload_items_2)\n",
        "      zip_name = os.path.basename(zip_path)\n",
        "      assert csv_path.endswith(\".csv\"), \"Please upload file with correct format (.csv)!\"\n",
        "      assert zip_path.endswith(\".zip\"), \"Please upload file with correct format (.zip)!\"\n",
        "\n",
        "      # Unzip structures\n",
        "      upload_path = Path(UPLOAD_FILE_HOME)\n",
        "      upload_path.mkdir(parents=True, exist_ok=True)\n",
        "      prefix = zip_name.rsplit(\".\", 1)[0]\n",
        "      struc_dir = upload_path / prefix\n",
        "      with zipfile.ZipFile(zip_path, 'r') as zip_ref:\n",
        "          zip_ref.extractall(struc_dir)\n",
        "\n",
        "      # If the unzipped file is a directory\n",
        "      unzipped_files = os.listdir(struc_dir)\n",
        "      if len(unzipped_files) == 1:\n",
        "        file_path = f\"{struc_dir}/{unzipped_files[0]}\"\n",
        "        if os.path.isdir(file_path):\n",
        "          struc_dir = file_path\n",
        "\n",
        "      df = pd.read_csv(csv_path)\n",
        "      pdbs_1 = []\n",
        "      for pdb_name, chain in df[[\"protein_1\", \"chain_1\"]].values:\n",
        "        pdb_path = f\"{struc_dir}/{pdb_name}\"\n",
        "        pdbs_1.append([pdb_path, chain])\n",
        "\n",
        "      sa_seqs_1 = pdb2sa(pdbs_1)\n",
        "\n",
        "      pdbs_2 = []\n",
        "      for pdb_name, chain in df[[\"protein_2\", \"chain_2\"]].values:\n",
        "        pdb_path = f\"{struc_dir}/{pdb_name}\"\n",
        "        pdbs_2.append([pdb_path, chain])\n",
        "\n",
        "      sa_seqs_2 = pdb2sa(pdbs_2)\n",
        "\n",
        "      rows = list(zip(sa_seqs_1, sa_seqs_2))\n",
        "\n",
        "    else:\n",
        "      raise\n",
        "\n",
        "    return_label, output_file = make_predictions(df, rows, num_label_box.value, model_type, model_arg)\n",
        "\n",
        "    report_str = \"\"\n",
        "    if return_label is not None:\n",
        "      report_str = f\"The predicted label is **{return_label}**\"\n",
        "\n",
        "    dividing_line = HTML(markdown.markdown(f\"### {'-'*75}\"))\n",
        "    pred_hint = HTML(markdown.markdown(\n",
        "      f\"{report_str}\\n\\n\"\n",
        "      f\"All prediction results have been saved to ``{output_file}``. You can click to download the results.\\n\\n\"\n",
        "      )\n",
        "    )\n",
        "    download_btn = generate_download_btn(output_file)\n",
        "\n",
        "    new_items = items + [dividing_line, pred_hint, download_btn]\n",
        "    custom_display(*new_items)\n",
        "\n",
        "\n",
        "  task_type_box.observe(change_task_type, names=\"value\")\n",
        "  model_type_box.observe(change_model_type, names=\"value\")\n",
        "  data_type_box.observe(change_data_type, names=\"value\")\n",
        "  upload_type_box.observe(change_upload_type, names='value')\n",
        "  start_btn.on_click(\n",
        "      lambda btn: start_thread(start_prediction, (btn,))\n",
        "      )\n",
        "\n",
        "  display(*items)\n",
        "\n",
        "\n",
        "######################################################################\n",
        "#       Start mutational effect predction         #\n",
        "######################################################################\n",
        "def start_mut_pred():\n",
        "  global refresh_module\n",
        "  refresh_module = start_mut_pred\n",
        "\n",
        "  WIDTH = \"500px\"\n",
        "  question = HTML(markdown.markdown(\"## Please choose the mutation task you want to perform\"))\n",
        "  sep_hint = HTML(markdown.markdown(f\"### {'-'*75}\"))\n",
        "\n",
        "  single_btn = Button(description='Single-site or Multi-site mutagenesis', layout=Layout(width=WIDTH, height='30px'), button_style=\"info\")\n",
        "  single_intro = HTML(markdown.markdown(\n",
        "      f\"You have to specify a mutation mannually and the model will output a score to predict whether this mutation \"\n",
        "      \"is good or not.\"\n",
        "      ), layout=Layout(width=WIDTH))\n",
        "\n",
        "  all_btn = Button(description='Single-site saturation mutagenesis', layout=Layout(width=WIDTH, height='30px'), button_style=\"info\")\n",
        "  all_intro = HTML(markdown.markdown(\n",
        "      f\"You don't have to specify mutation mannually. The model will predict scores for all single-site mutations given a protein, \"\n",
        "      \"e.g. given a protein with the length of 200 the model will output 200 * 20 scores, each for a single-site mutation.\"\n",
        "      ), layout=Layout(width=WIDTH))\n",
        "\n",
        "  items = [\n",
        "      question,\n",
        "      sep_hint,\n",
        "      single_btn,\n",
        "      single_intro,\n",
        "      sep_hint,\n",
        "      all_btn,\n",
        "      all_intro,\n",
        "      sep_hint,\n",
        "  ]\n",
        "\n",
        "  # Set click events\n",
        "  single_btn.on_click(partial(jump, next=single_mut_pred))\n",
        "  all_btn.on_click(partial(jump, next=saturation_mut_pred))\n",
        "\n",
        "  display(*items)\n",
        "\n",
        "\n",
        "######################################################################\n",
        "#      Single-site or Multi-site mutagenesis        #\n",
        "######################################################################\n",
        "def single_mut_pred():\n",
        "  global refresh_module\n",
        "  refresh_module = single_mut_pred\n",
        "\n",
        "  hint = HTML(markdown.markdown(\"# Single-site or Multi-site mutagenesis\\n\\n## Please upload the protein structure\\n If you only have protein sequence, you could use <a href='https://alphafoldserver.com' target='blank'>AlphaFold server</a> to predict its structure and upload it here.\"))\n",
        "\n",
        "  chain_hint = HTML(markdown.markdown(\"Chain (to be extracted from the structure):\"))\n",
        "  input_chain = ipywidgets.Text(value=\"A\",placeholder=f'Enter the name of chain here', layout=Layout(width='500px', height='30px'))\n",
        "  upload_hint = HTML(markdown.markdown(\"Upload the protein structure:\"))\n",
        "  # upload_btn = ipywidgets.FileUpload(accept='',multiple=False, description=\"Upload protein structure (.pdb / .cif file)\", layout=Layout(width='500px', height='30px'))\n",
        "  # upload_btn = ipywidgets.Button(description=\"Upload protein structure (.pdb / .cif file)\", layout=Layout(width='500px', height='30px'))\n",
        "  upload_items = get_upload_box()\n",
        "  upload_ok_btn = ipywidgets.Button(\n",
        "      description=\"Submit\",\n",
        "      layout=Layout(width='500px', height='30px'),\n",
        "      button_style=\"info\",\n",
        "      )\n",
        "\n",
        "  items = [\n",
        "      hint,\n",
        "      chain_hint,\n",
        "      input_chain,\n",
        "      upload_hint,\n",
        "      *upload_items,\n",
        "      upload_ok_btn,\n",
        "  ]\n",
        "\n",
        "  # Set click events\n",
        "  def on_upload_file(button):\n",
        "    save_path = get_upload_file_path(upload_items)\n",
        "    name = os.path.basename(save_path)\n",
        "    assert name.endswith(\".pdb\") or name.endswith(\".cif\"), \"Please upload file with correct format (.pdb / .cif)!\"\n",
        "\n",
        "    chain = input_chain.value\n",
        "    protein_list = [(save_path, chain)]\n",
        "    mprs = MultipleProcessRunnerSimplifier(protein_list, pdb2sequence, n_process=2, return_results=True, verbose=False)\n",
        "    seqs = mprs.run()\n",
        "\n",
        "    assert len(seqs) != 0, f\"The specified chain '{chain}' does not exist in the structure!\"\n",
        "\n",
        "    sa_seq = seqs[0].split(\"\\t\")[-1]\n",
        "    aa_seq = sa_seq[0::2]\n",
        "    struc_seq = sa_seq[1::2].replace(\"#\", \"\\#\")\n",
        "\n",
        "    seq_info = HTML(markdown.markdown(f\"**{name}**\\n\\n**Foldseek sequence (\\\"#\\\" means low pLDDT positions that are masked):**\\n\\n{struc_seq}\\n\\n**Amino acid sequence:**\\n\\n{aa_seq}\"))\n",
        "\n",
        "    # Mutation information box\n",
        "    input_hint = HTML(markdown.markdown(\n",
        "        \"**Please input the mutation information:**\"\n",
        "        )\n",
        "    )\n",
        "    pred_num_box = ipywidgets.RadioButtons(\n",
        "      options=['Single variant', 'multiple variants'],\n",
        "      disabled=False,\n",
        "      layout={'width': 'max-content'}, # If the items' names are long\n",
        "      description=\"How many variants do you want to predict?\",\n",
        "      style={'description_width': 'initial'},\n",
        "      )\n",
        "\n",
        "    mut_hint =  HTML(markdown.markdown(\n",
        "        \"For single-site mutation, e.g. M1E means mutating the amino acid M to E at first position. \"\n",
        "        \"For multi-site mutation, you are expected to separate each position by ':', e.g. M1E:P2V\"\n",
        "        )\n",
        "    )\n",
        "\n",
        "    upload_csv_hint = HTML(markdown.markdown(\n",
        "              \"<img src='https://github.com/westlake-repl/SaProtHub/blob/dev/Figure/prediction/dataset/mut.jpg?raw=true' height='200px' width='400px' align='center'>\\n\\n\"\n",
        "              \"Please upload a ``.csv`` file that contains all mutations to predict. Please strictly follow the format above **(column names are also needed in the csv file)**. \"\n",
        "              \"Then click the start button.\"\n",
        "              ), layout=Layout(display=\"none\"))\n",
        "\n",
        "    upload_csv_items = get_upload_box()\n",
        "    for item in upload_csv_items[:4]:\n",
        "      item.layout.display = \"none\"\n",
        "\n",
        "    input_mut = ipywidgets.Text(\n",
        "        placeholder=\"Enter mutation information here, e.g. M1E\", layout=Layout(width='1000px', height='30px'))\n",
        "    submit_btn = Button(description='Calculate mutation score', layout=Layout(width='500px', height='30px'), button_style=\"info\")\n",
        "\n",
        "\n",
        "    new_items = items + [\n",
        "        seq_info,\n",
        "        input_hint,\n",
        "        pred_num_box,\n",
        "        mut_hint,\n",
        "        input_mut,\n",
        "        upload_csv_hint,\n",
        "        *upload_csv_items,\n",
        "        submit_btn\n",
        "        ]\n",
        "\n",
        "    def change_pred_num(change):\n",
        "      now_type = change[\"new\"]\n",
        "      if now_type == \"Single variant\":\n",
        "        upload_csv_hint.layout.display = \"none\"\n",
        "        set_upload_visibility(upload_csv_items, \"none\")\n",
        "        input_mut.layout.display = None\n",
        "\n",
        "      else:\n",
        "        upload_csv_hint.layout.display = None\n",
        "        set_upload_visibility(upload_csv_items, \"default\")\n",
        "        input_mut.layout.display = \"none\"\n",
        "\n",
        "    # Click to calculate mutation score\n",
        "    def calc_mut_score(button):\n",
        "      if pred_num_box.value == \"Single variant\":\n",
        "        mut_info = input_mut.value\n",
        "        # Skip empty input\n",
        "        if mut_info.strip() == \"\":\n",
        "          return\n",
        "\n",
        "        for single in mut_info.split(\":\"):\n",
        "          # Assert position valid\n",
        "          pos = int(single[1:-1])\n",
        "          assert 0 < pos <= len(aa_seq), f\"The mutation position should be greater than 0 and not greater than the length of sequence ({len(aa_seq)})\"\n",
        "\n",
        "          # Assert amino acid valid\n",
        "          ori_aa = single[0]\n",
        "          assert ori_aa == aa_seq[pos-1], f\"The amino acid at position {pos} is not {ori_aa}! Please refer to the amino acid sequence above to find correct position (the counting starts from 1).\"\n",
        "\n",
        "        print(f\"Predict the mutation score for {mut_info}...\")\n",
        "        score = predict_mut(sa_seq, mut_info)\n",
        "        score_hint = HTML(markdown.markdown(\n",
        "            f\"The score for {mut_info} is <font color=red>{score.item()}</font>. **Please check <a href='https://github.com/westlake-repl/SaprotHub/wiki/SaprotHub-v2-(latest)#introduction-of-saprot-mutation-score' target='blank'>here</a> for the description of the mutation score.**\"))\n",
        "        new_items = items + [seq_info, input_hint, pred_num_box, input_mut, upload_csv_hint, mut_hint, *upload_csv_items, submit_btn, score_hint]\n",
        "        custom_display(*new_items)\n",
        "\n",
        "      else:\n",
        "        save_path = get_upload_file_path(upload_csv_items)\n",
        "        name = os.path.basename(save_path)\n",
        "        assert name.endswith(\".csv\"), \"Please upload file with correct format (.csv)!\"\n",
        "\n",
        "        df = pd.read_csv(save_path)\n",
        "        print(f\"Predicting...\")\n",
        "        zeroshot_model = load_zeroshot_model()\n",
        "        scores = []\n",
        "        for mut_info in tqdm(df[\"mutation\"].values):\n",
        "          for single in mut_info.split(\":\"):\n",
        "            # Assert position valid\n",
        "            pos = int(single[1:-1])\n",
        "            assert 0 < pos <= len(aa_seq), f\"For {mut_info}, The mutation position should be greater than 0 and not greater than the length of sequence ({len(aa_seq)})\"\n",
        "\n",
        "            # Assert amino acid valid\n",
        "            ori_aa = single[0]\n",
        "            assert ori_aa == aa_seq[pos-1], f\"For {mut_info}, The amino acid at position {pos} is not {ori_aa}! Please refer to the amino acid sequence above to find correct position (the counting starts from 1).\"\n",
        "\n",
        "          score = zeroshot_model.predict_mut(sa_seq, mut_info)\n",
        "          scores.append(score.item())\n",
        "\n",
        "        df[\"score\"] = scores\n",
        "        save_path = OUTPUT_HOME / \"mutation_results.csv\"\n",
        "        df.to_csv(save_path, index=False)\n",
        "        download_btn = generate_download_btn(save_path)\n",
        "\n",
        "        pred_hint = HTML(markdown.markdown(\n",
        "          f\"Prediction results have been saved to ``{save_path}``. You can click to download the results.\"\n",
        "          )\n",
        "        )\n",
        "\n",
        "        new_items = items + [seq_info, input_hint, pred_num_box, input_mut, upload_csv_hint, mut_hint, *upload_csv_items, submit_btn, pred_hint, download_btn]\n",
        "        custom_display(*new_items)\n",
        "\n",
        "    pred_num_box.observe(change_pred_num, names=\"value\")\n",
        "    submit_btn.on_click(\n",
        "        lambda btn: start_thread(calc_mut_score, (btn,))\n",
        "        )\n",
        "    custom_display(*new_items)\n",
        "\n",
        "  upload_ok_btn.on_click(on_upload_file)\n",
        "  display(*items)\n",
        "\n",
        "\n",
        "######################################################################\n",
        "#       Single-site saturation mutagenesis         #\n",
        "######################################################################\n",
        "def saturation_mut_pred():\n",
        "  global refresh_module\n",
        "  refresh_module = saturation_mut_pred\n",
        "\n",
        "  hint = HTML(markdown.markdown(\"# Single-site saturation mutagenesis\\n\\n## Please upload the protein structure\\n If you only have protein sequence, you could use <a href='https://alphafoldserver.com' target='blank'>AlphaFold server</a> to predict its structure and upload it here.\"))\n",
        "\n",
        "  chain_hint = HTML(markdown.markdown(\"Chain (to be extracted from the structure):\"))\n",
        "  input_chain = ipywidgets.Text(value=\"A\",placeholder=f'Enter the name of chain here', layout=Layout(width='500px', height='30px'))\n",
        "  upload_hint = HTML(markdown.markdown(\"Upload the protein structure:\"))\n",
        "  upload_items = get_upload_box()\n",
        "  upload_ok_btn = ipywidgets.Button(\n",
        "      description=\"Submit\",\n",
        "      layout=Layout(width='500px', height='30px'),\n",
        "      button_style=\"info\",\n",
        "      )\n",
        "\n",
        "  items = [\n",
        "      hint,\n",
        "      chain_hint,\n",
        "      input_chain,\n",
        "      upload_hint,\n",
        "      *upload_items,\n",
        "      upload_ok_btn,\n",
        "  ]\n",
        "\n",
        "  # Set click events\n",
        "  def on_upload_file(change):\n",
        "    save_path = get_upload_file_path(upload_items)\n",
        "    name = os.path.basename(save_path)\n",
        "    assert name.endswith(\".pdb\") or name.endswith(\".cif\"), \"Please upload file with correct format (.pdb / .cif)!\"\n",
        "\n",
        "    chain = input_chain.value\n",
        "    protein_list = [(save_path, chain)]\n",
        "    mprs = MultipleProcessRunnerSimplifier(protein_list, pdb2sequence, n_process=2, return_results=True, verbose=False)\n",
        "    seqs = mprs.run()\n",
        "\n",
        "    assert len(seqs) != 0, f\"The specified chain '{chain}' does not exist in the structure!\"\n",
        "\n",
        "    sa_seq = seqs[0].split(\"\\t\")[-1]\n",
        "    aa_seq = sa_seq[0::2]\n",
        "    struc_seq = sa_seq[1::2].replace(\"#\", \"\\#\")\n",
        "\n",
        "    seq_info = HTML(markdown.markdown(f\"**{name}**\\n\\n**Foldseek sequence (\\\"#\\\" means low pLDDT positions that are masked):**\\n\\n{struc_seq}\\n\\n**Amino acid sequence:**\\n\\n{aa_seq}\"))\n",
        "    submit_btn = Button(description='Calculate mutation score for all single-site mutations', layout=Layout(width='500px', height='30px'), button_style=\"info\")\n",
        "\n",
        "\n",
        "    new_items = items + [seq_info, submit_btn]\n",
        "\n",
        "    # Click to calculate mutation score\n",
        "    def calc_mut_score(button):\n",
        "      print(f\"Predicting mutation scores for all single-site mutations...\")\n",
        "      save_path = predict_all_mut(sa_seq)\n",
        "      score_hint = HTML(markdown.markdown(f\"The result has been saved to {save_path}. You can click to download the file.\\n\\n\"\n",
        "      \"**Please check <a href='https://github.com/westlake-repl/SaprotHub/wiki/SaprotHub-v2-(latest)#introduction-of-saprot-mutation-score' target='blank'>here</a> for the description of the mutation score.**\"))\n",
        "      download_btn = generate_download_btn(save_path)\n",
        "\n",
        "      new_items = items + [seq_info, submit_btn, score_hint, download_btn]\n",
        "      custom_display(*new_items)\n",
        "\n",
        "    submit_btn.on_click(\n",
        "        # disable_wrapper(\n",
        "        #     lambda btn: start_thread(calc_mut_score, (btn,))\n",
        "        #     )\n",
        "        lambda btn: start_thread(calc_mut_score, (btn,))\n",
        "          )\n",
        "    custom_display(*new_items)\n",
        "\n",
        "  upload_ok_btn.on_click(on_upload_file)\n",
        "\n",
        "  display(*items)\n",
        "\n",
        "\n",
        "#####################################################################\n",
        "#          Protein sequence design           #\n",
        "#####################################################################\n",
        "def protein_sequence_design():\n",
        "  global refresh_module\n",
        "  refresh_module = protein_sequence_design\n",
        "\n",
        "  title = HTML(markdown.markdown(\"## Protein sequence design\"))\n",
        "  WIDTH = \"500px\"\n",
        "  HEIGHT= \"30px\"\n",
        "\n",
        "  task_hint = HTML(markdown.markdown(\"### Please upload the structure backbone:\"))\n",
        "  chain_hint = HTML(markdown.markdown(\"Chain (to be extracted from the structure):\"))\n",
        "  input_chain = ipywidgets.Text(value=\"A\",placeholder=f'Enter the name of chain here', layout=Layout(width=WIDTH, height=HEIGHT))\n",
        "  upload_hint = HTML(markdown.markdown(\"Upload the protein structure:\"))\n",
        "  upload_items = get_upload_box()\n",
        "  upload_ok_btn = ipywidgets.Button(\n",
        "      description=\"Submit\",\n",
        "      layout=Layout(width='500px', height='30px'),\n",
        "      button_style=\"info\",\n",
        "      )\n",
        "\n",
        "  items = [\n",
        "      title,\n",
        "      task_hint,\n",
        "      chain_hint,\n",
        "      input_chain,\n",
        "      upload_hint,\n",
        "      *upload_items,\n",
        "      upload_ok_btn,\n",
        "  ]\n",
        "\n",
        "  # Set click events\n",
        "  def parse_structure(change):\n",
        "    save_path = get_upload_file_path(upload_items)\n",
        "    name = os.path.basename(save_path)\n",
        "    assert name.endswith(\".pdb\") or name.endswith(\".cif\"), \"Please upload file with correct format (.pdb / .cif)!\"\n",
        "\n",
        "    chain = input_chain.value\n",
        "    protein_list = [(save_path, chain)]\n",
        "    mprs = MultipleProcessRunnerSimplifier(protein_list, pdb2sequence, n_process=2, return_results=True, verbose=False)\n",
        "    seqs = mprs.run()\n",
        "\n",
        "    assert len(seqs) != 0, f\"The specified chain '{chain}' does not exist in the structure!\"\n",
        "\n",
        "    sa_seq = seqs[0].split(\"\\t\")[-1]\n",
        "    aa_seq = sa_seq[0::2]\n",
        "    struc_seq = sa_seq[1::2].replace(\"#\", \"\\#\")\n",
        "\n",
        "    seq_info = HTML(markdown.markdown(\n",
        "        f\"**Foldseek sequence of chain {chain} (\\\"#\\\" means low pLDDT positions that are masked):**\\n\\n{struc_seq}\\n\\n**Backbone visualization of {name} (chains are displayed with different colors):**\"\n",
        "        ), layout=Layout(width=WIDTH))\n",
        "\n",
        "    # Sampling config\n",
        "    sampling_method_box = ipywidgets.Dropdown(\n",
        "              options=['argmax', 'multinomial'],\n",
        "              value='argmax',\n",
        "              description='Sampling methodology:',\n",
        "              disabled=False,\n",
        "              layout=Layout(width=WIDTH, height=HEIGHT),\n",
        "              style={'description_width': 'initial'},\n",
        "            )\n",
        "    sampling_num = ipywidgets.BoundedIntText(\n",
        "          value=1,\n",
        "          min=1,\n",
        "          max=100,\n",
        "          step=1,\n",
        "          description='Sampling num:',\n",
        "          disabled=False,\n",
        "          style={'description_width': 'initial'},\n",
        "          layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "        )\n",
        "    sampling_intro = HTML(markdown.markdown(\n",
        "        f\"*For each position, Saprot will predict the probability of every amino acid occuring at this position. ***argmax*** means the model \"\n",
        "        \"will choose the amino acid with highest probability as its prediction. ***multinomial*** means the model will sample amino acids based on \"\n",
        "        \"the probability distribution.*\"\n",
        "        ), layout=Layout(width=WIDTH))\n",
        "\n",
        "    save_name = ipywidgets.Text(\n",
        "          value=\"predicted_seq\",\n",
        "          description='Save name:',\n",
        "          disabled=False,\n",
        "          style={'description_width': 'initial'},\n",
        "          layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "        )\n",
        "\n",
        "\n",
        "    pred_hint = HTML(markdown.markdown(f\"### {'-'*75}\"))\n",
        "    pred_btn = Button(description='Predict protein sequence', layout=Layout(width=WIDTH, height=HEIGHT), button_style=\"info\")\n",
        "\n",
        "    esmfold_hint = HTML(markdown.markdown(\n",
        "        f\"### You could use ESMFold to evaluate the quality of the predicted sequence\\n\\n\"\n",
        "        \"#### <font color=red>Warning: If you are using Google T4 GPU, it will run out of memory!</font>\"\n",
        "        ))\n",
        "    esmfold_input_box = ipywidgets.Text(\n",
        "          value=\"\",\n",
        "          placeholder = \"Copy the predicted sequence and paste it here\",\n",
        "          disabled=False,\n",
        "          style={'description_width': 'initial'},\n",
        "          layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "        )\n",
        "    esmfold_pred_btn = Button(description='Predict structure', layout=Layout(width=WIDTH, height=HEIGHT), button_style=\"info\")\n",
        "\n",
        "    def change_sampling_method(change):\n",
        "      sampling_method = change[\"new\"]\n",
        "      if sampling_method == \"multinomial\":\n",
        "        sampling_num.layout.display = None\n",
        "      else:\n",
        "        sampling_num.layout.display = \"none\"\n",
        "\n",
        "    # Predict protein sequence\n",
        "    def predict(button):\n",
        "      print(\"Predicting sequence...\")\n",
        "\n",
        "      struc_seq = sa_seq[1::2]\n",
        "      masked_aa_seq = \"#\" * len(struc_seq)\n",
        "      methods = sampling_method_box.value\n",
        "      num_samples = sampling_num.value\n",
        "\n",
        "      pred_seqs = inverse_folding(masked_aa_seq, struc_seq, methods, num_samples)\n",
        "      print(pred_seqs)\n",
        "\n",
        "      save_seq_path = f\"{root_dir}/SaprotHub/output/{save_name.value}.fasta\"\n",
        "      pred_seq_hint = \"**Predicted sequences are listed below:**\\n\\n\"\n",
        "      with open(save_seq_path, \"w\") as w:\n",
        "        for i, aa_seq in enumerate(pred_seqs):\n",
        "          pred_seq_hint += f\"{i+1}. {aa_seq}\\n\\n\"\n",
        "          w.write(f\">predicted_seq_{i}\\n{aa_seq}\\n\")\n",
        "\n",
        "      pred_seq_hint += \"You can click the button to dowanload the predictions.\"\n",
        "      pred_seq_hint = HTML(markdown.markdown(pred_seq_hint))\n",
        "      download_btn = generate_download_btn(save_seq_path)\n",
        "\n",
        "      # Update the display\n",
        "      clear_output()\n",
        "      display(*items)\n",
        "      display(seq_info)\n",
        "      show_pdb(save_path, color=\"chain\").show()\n",
        "      display(sampling_method_box, sampling_num, sampling_intro, pred_hint, pred_btn, pred_seq_hint, download_btn, esmfold_hint, esmfold_input_box, esmfold_pred_btn)\n",
        "      display(*system_widgets)\n",
        "\n",
        "      def esmfold_pred(button):\n",
        "        print(\"Predicting structure...\")\n",
        "        aa_seq = esmfold_input_box.value\n",
        "        pred_pdb_path = predict_structure(aa_seq)\n",
        "\n",
        "        # Update the display after using ESMFold\n",
        "        clear_output()\n",
        "        display(*items)\n",
        "        display(seq_info)\n",
        "        show_pdb(save_path, color=\"chain\").show()\n",
        "        display(sampling_method_box, sampling_num, sampling_intro, pred_hint, pred_btn, pred_seq_hint, download_btn, esmfold_hint, esmfold_input_box, esmfold_pred_btn)\n",
        "        print(\"Predicted structure by ESMFold:\")\n",
        "\n",
        "        # Display predicted structure\n",
        "        color = \"lDDT\"\n",
        "        show_sidechains = False\n",
        "        show_mainchains = False\n",
        "        show_pdb(pred_pdb_path, show_sidechains, show_mainchains, color).show()\n",
        "        if color == \"lDDT\":\n",
        "          plot_plddt_legend().show()\n",
        "        display(*system_widgets)\n",
        "\n",
        "\n",
        "      esmfold_pred_btn.on_click(\n",
        "          # disable_wrapper(\n",
        "          #     lambda btn: start_thread(esmfold_pred, (btn,))\n",
        "          #     )\n",
        "          lambda btn: start_thread(esmfold_pred, (btn,))\n",
        "          )\n",
        "\n",
        "\n",
        "    # Set click events\n",
        "    sampling_method_box.observe(change_sampling_method, names=\"value\")\n",
        "    pred_btn.on_click(\n",
        "        # disable_wrapper(\n",
        "            # lambda btn: start_thread(predict, (btn,))\n",
        "            # )\n",
        "            lambda btn: start_thread(predict, (btn,))\n",
        "        )\n",
        "\n",
        "    # Set default state\n",
        "    sampling_num.layout.display = \"none\"\n",
        "\n",
        "    # Update the display\n",
        "    clear_output()\n",
        "    display(*items)\n",
        "    display(seq_info)\n",
        "    show_pdb(save_path, color=\"chain\").show()\n",
        "    display(sampling_method_box, sampling_num, sampling_intro, pred_hint, pred_btn)\n",
        "    display(*system_widgets)\n",
        "\n",
        "  upload_ok_btn.on_click(parse_structure)\n",
        "  display(*items)\n",
        "\n",
        "\n",
        "def obtain_protein_embedding():\n",
        "  global refresh_module\n",
        "  refresh_module = obtain_protein_embedding\n",
        "\n",
        "  WIDTH = \"500px\"\n",
        "  HEIGHT = \"30px\"\n",
        "  hint = HTML(markdown.markdown(\"## Obtain protein-level embeddings\"))\n",
        "\n",
        "  data_type_hint = HTML(markdown.markdown(\"### Uploaded data type:\"))\n",
        "  data_type_box = ipywidgets.RadioButtons(\n",
        "      options=['protein sequence', 'protein structure'],\n",
        "      value=\"protein structure\",\n",
        "      disabled=False,\n",
        "      style={'description_width': 'initial'},\n",
        "      )\n",
        "  saprothub_data_type_hint = HTML(markdown.markdown(\n",
        "    \"<font color=red>Note: For models with input type '(Structure-aware) sequence' in SaprotHub, protein structure input is required.</font>\",\n",
        "    ), layout=Layout(display=\"none\"))\n",
        "\n",
        "  model_hint = HTML(markdown.markdown(\"### Choose the model for embedding generation:\"))\n",
        "  model_type_box = ipywidgets.Dropdown(\n",
        "            options=['Official SaProt (35M)', \"Official SaProt (650M)\", \"Trained by yourself on ColabSaprot\", \"Shared by peers on SaprotHub\", \"Saved in your local computer\"],\n",
        "            value='Official SaProt (35M)',\n",
        "            description='Base model:',\n",
        "            disabled=False,\n",
        "            layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "          )\n",
        "  model_arg_box = HTML(markdown.markdown(\"dummy box\"))\n",
        "  model_arg_box.layout.display = \"none\"\n",
        "\n",
        "  upload_model_items = get_upload_box(btn_desc=\"Upload Model-<task_name>-<model_size>.zip file\")\n",
        "  set_upload_visibility(upload_model_items, mode=\"none\")\n",
        "\n",
        "  saprothub_link = HTML(markdown.markdown(\n",
        "    \"<font color=red>You could search models using our <a href='https://huggingface.co/spaces/SaProtHub/SaprotHub-search' target='blank'>search engine</a> or from <a href='https://huggingface.co/SaProtHub' target='blank'>SaprotHub</a>\\n\\n\"\n",
        "    \"A model id example: <a href='https://huggingface.co/SaProtHub/Model-Binary_Localization-650M' target='blank'>SaProtHub/Model-Binary_Localization-650M</a></font>\"\n",
        "    ), layout=Layout(display=\"none\"))\n",
        "\n",
        "  upload_type_hint = HTML(markdown.markdown(\"### Choose the number of protein structure:\"))\n",
        "  upload_type_box = ipywidgets.RadioButtons(\n",
        "        options=['Single file', 'Multiple files'],\n",
        "        layout={'width': 'max-content'}, # If the items' names are long\n",
        "        disabled=False,\n",
        "        style={'description_width': 'initial'},\n",
        "        )\n",
        "\n",
        "  upload_hint = HTML(markdown.markdown(\"**Upload the protein structure (.pdb / .cif file):**\"))\n",
        "  upload_items = get_upload_box()\n",
        "  upload_ok_btn = ipywidgets.Button(\n",
        "      description=\"Submit\",\n",
        "      layout=Layout(width='500px', height='30px'),\n",
        "      button_style=\"info\",\n",
        "      )\n",
        "\n",
        "  save_path_hint = HTML(layout={\"display\": \"none\"})\n",
        "  start_hint = HTML(markdown.markdown(f\"### {'-'*75}\"))\n",
        "  start_btn = Button(description='Start embedding generation', layout=Layout(width=WIDTH, height=HEIGHT), button_style=\"info\")\n",
        "\n",
        "  # Set items to display\n",
        "  items = [\n",
        "      hint,\n",
        "      model_hint,\n",
        "      model_type_box,\n",
        "      model_arg_box,\n",
        "      saprothub_link,\n",
        "      data_type_hint,\n",
        "      data_type_box,\n",
        "      saprothub_data_type_hint,\n",
        "      upload_type_hint,\n",
        "      upload_type_box,\n",
        "      upload_hint,\n",
        "      *upload_items,\n",
        "      # upload_ok_btn,\n",
        "      save_path_hint,\n",
        "      start_hint,\n",
        "      start_btn\n",
        "  ]\n",
        "\n",
        "  # Used to replace input box based on model type\n",
        "  model_arg_box_idx = items.index(model_arg_box)\n",
        "\n",
        "  # Set click events\n",
        "  def change_model_type(change):\n",
        "    model_type_value = change[\"new\"]\n",
        "    if model_type_value == \"Shared by peers on SaprotHub\":\n",
        "      saprothub_data_type_hint.layout.display = None\n",
        "    else:\n",
        "      saprothub_data_type_hint.layout.display = \"none\"\n",
        "\n",
        "    if model_type_value == \"Trained by yourself on ColabSaprot\":\n",
        "      new_model_arg_box = select_adapter_from(None, use_model_from=model_type_value)\n",
        "      new_model_arg_box.layout.width = WIDTH\n",
        "      new_model_arg_box.description = \"Select your local model:\"\n",
        "      new_model_arg_box.style = {'description_width': 'initial'}\n",
        "      saprothub_link.layout.display = \"none\"\n",
        "\n",
        "    elif model_type_value == \"Shared by peers on SaprotHub\":\n",
        "      new_model_arg_box = select_adapter_from(None, use_model_from=model_type_value)\n",
        "      new_model_arg_box.layout.width = WIDTH\n",
        "      new_model_arg_box.placeholder = \"Enter SaprotHub model id in HuggingFace\"\n",
        "      new_model_arg_box.description = \"Model id:\"\n",
        "      saprothub_link.layout.display = None\n",
        "\n",
        "    elif model_type_value == \"Saved in your local computer\":\n",
        "      set_upload_visibility(upload_model_items, \"default\")\n",
        "      new_model_arg_box = upload_model_items\n",
        "      saprothub_link.layout.display = \"none\"\n",
        "\n",
        "    else:\n",
        "      new_model_arg_box = HTML(markdown.markdown(\"dummy box\"))\n",
        "      new_model_arg_box.layout.display = \"none\"\n",
        "      saprothub_link.layout.display = \"none\"\n",
        "\n",
        "    if isinstance(new_model_arg_box, list):\n",
        "      new_items = items[:model_arg_box_idx] + new_model_arg_box + items[model_arg_box_idx+1:]\n",
        "      custom_display(*new_items)\n",
        "\n",
        "    else:\n",
        "      items[model_arg_box_idx] = new_model_arg_box\n",
        "      custom_display(*items)\n",
        "\n",
        "  def change_data_type(change):\n",
        "    data_type_value = change[\"new\"]\n",
        "    if data_type_value == \"protein sequence\":\n",
        "      upload_type_hint.layout.display = \"none\"\n",
        "      upload_type_box.layout.display = \"none\"\n",
        "      upload_hint.value = markdown.markdown(\"**Upload the protein sequences (.fasta file):**\")\n",
        "\n",
        "    else:\n",
        "      upload_type_hint.layout.display = None\n",
        "      upload_type_box.layout.display = None\n",
        "      if upload_type_box.value == \"Single file\":\n",
        "        upload_hint.value = markdown.markdown(\"**Upload the protein structure (.pdb / .cif file):**\")\n",
        "      else:\n",
        "        upload_hint.value = markdown.markdown(\"**Upload the protein structure (a .zip file that contains all structure files):**\")\n",
        "\n",
        "  def change_upload_type(change):\n",
        "    now_type = change[\"new\"]\n",
        "    if now_type == \"Single file\":\n",
        "      upload_hint.value = markdown.markdown(\"**Upload the protein structure (.pdb / .cif file):**\")\n",
        "    else:\n",
        "      upload_hint.value = markdown.markdown(\"**Upload the protein structure (a .zip file that contains all structure files):**\")\n",
        "\n",
        "  # def on_upload_file(button):\n",
        "  #   save_path = get_upload_file_path(upload_items)\n",
        "  #   name = os.path.basename(save_path)\n",
        "\n",
        "  #   # Check the file format\n",
        "  #   if data_type_box.value == \"protein sequence\":\n",
        "  #     assert name.endswith(\".fasta\"), \"Please upload file with correct format (.fasta)!\"\n",
        "\n",
        "  #   elif upload_type_box.value == \"Single file\":\n",
        "  #     assert name.endswith(\".pdb\") or name.endswith(\".cif\"), \"Please upload file with correct format (.pdb / .cif)!\"\n",
        "\n",
        "  #   else:\n",
        "  #     assert name.endswith(\".zip\"), \"Please upload file with correct format (.zip)!\"\n",
        "\n",
        "  #   save_path_hint.value = markdown.markdown(f\"The uploaded file is saved to '{save_path}'\")\n",
        "  #   for item in [save_path_hint, start_hint, start_btn]:\n",
        "  #     item.layout.display = None\n",
        "\n",
        "  #   if name.endswith(\".fasta\"):\n",
        "  #     pdb_list = save_path\n",
        "\n",
        "  #   elif name.endswith(\".zip\"):\n",
        "  #     save_dir = save_path.rsplit(\".\", 1)[0]\n",
        "  #     with zipfile.ZipFile(save_path, 'r') as zip_ref:\n",
        "  #         zip_ref.extractall(save_dir)\n",
        "\n",
        "  #     # If the unzipped file is a directory\n",
        "  #     unzipped_files = os.listdir(save_dir)\n",
        "  #     if len(unzipped_files) == 1:\n",
        "  #       file_path = f\"{save_dir}/{unzipped_files[0]}\"\n",
        "  #       if os.path.isdir(file_path):\n",
        "  #         save_dir = file_path\n",
        "\n",
        "  #     pdb_list = [f\"{save_dir}/{file}\" for file in os.listdir(save_dir)]\n",
        "\n",
        "  #   else:\n",
        "  #     pdb_list = [str(save_path)]\n",
        "\n",
        "  #   # Enable click for embedding generation\n",
        "  #   start_btn.on_click(\n",
        "  #       lambda btn: start_thread(partial(start_generation, pdb_list=pdb_list), (btn,))\n",
        "  #       )\n",
        "\n",
        "  def start_generation(button):\n",
        "    save_path = get_upload_file_path(upload_items)\n",
        "    name = os.path.basename(save_path)\n",
        "\n",
        "    # Check the file format\n",
        "    if data_type_box.value == \"protein sequence\":\n",
        "      assert name.endswith(\".fasta\"), \"Please upload file with correct format (.fasta)!\"\n",
        "\n",
        "    elif upload_type_box.value == \"Single file\":\n",
        "      assert name.endswith(\".pdb\") or name.endswith(\".cif\"), \"Please upload file with correct format (.pdb / .cif)!\"\n",
        "\n",
        "    else:\n",
        "      assert name.endswith(\".zip\"), \"Please upload file with correct format (.zip)!\"\n",
        "\n",
        "    if name.endswith(\".fasta\"):\n",
        "      pdb_list = save_path\n",
        "\n",
        "    elif name.endswith(\".zip\"):\n",
        "      save_dir = save_path.rsplit(\".\", 1)[0]\n",
        "      with zipfile.ZipFile(save_path, 'r') as zip_ref:\n",
        "          zip_ref.extractall(save_dir)\n",
        "\n",
        "      # If the unzipped file is a directory\n",
        "      unzipped_files = os.listdir(save_dir)\n",
        "      if len(unzipped_files) == 1:\n",
        "        file_path = f\"{save_dir}/{unzipped_files[0]}\"\n",
        "        if os.path.isdir(file_path):\n",
        "          save_dir = file_path\n",
        "\n",
        "      pdb_list = [f\"{save_dir}/{file}\" for file in os.listdir(save_dir)]\n",
        "\n",
        "    else:\n",
        "      pdb_list = [str(save_path)]\n",
        "\n",
        "    # Check data type compatibility\n",
        "    upload_data_type = \"AA\" if data_type_box.value == \"protein sequence\" else \"SA\"\n",
        "\n",
        "    if model_type_box.value == \"Saved in your local computer\":\n",
        "      zip_path = get_upload_file_path(upload_model_items)\n",
        "      name = os.path.basename(zip_path)\n",
        "      save_dir = ADAPTER_HOME / \"Upload\" / name.rsplit('.', 1)[0]\n",
        "      os.makedirs(save_dir, exist_ok=True)\n",
        "      # unzip model.zip\n",
        "      with zipfile.ZipFile(zip_path, 'r') as zip_ref:\n",
        "          zip_ref.extractall(save_dir)\n",
        "      os.remove(zip_path)\n",
        "      model_arg = save_dir\n",
        "\n",
        "    else:\n",
        "      model_arg = items[model_arg_box_idx].value\n",
        "\n",
        "    model_data_type = load_data_type_from_model(model_type_box.value, model_arg)\n",
        "    if model_data_type != upload_data_type:\n",
        "      if model_data_type == \"SA\":\n",
        "        raise Exception(\"Error: The model you choose was trained on protein structures. So you have to upload protein structures!\")\n",
        "      else:\n",
        "        raise Exception(\"Error: The model you choose was trained on protein sequences. So you have to upload protein sequences!\")\n",
        "\n",
        "    print(\"Start generation...\")\n",
        "    zip_path = generate_embeddings(pdb_list, model_type_box.value, model_arg)\n",
        "\n",
        "    dividing_line = HTML(markdown.markdown(f\"### {'-'*75}\"))\n",
        "    zip_hint = HTML(markdown.markdown(\n",
        "      f\"The embedding files have been saved to ``{zip_path}``. You can click to download the file.\\n\\n\"\n",
        "      \"There are two files in the zip file, ``embedding_seqs.fasta`` and ``embeddings.pt``.\\n\\n\"\n",
        "      \"``embedding_seqs.fasta`` contains proteins used for generating embeddings.\\n\\n\"\n",
        "      \"``embeddings.pt`` is the generated embeddings with the shape ``[N, D]``, where ``N`` is the number of sequences and ``D`` is the hidden dimension.\\n\\n\"\n",
        "      \"The order of sequences in ``fasta`` file is consistent with the order of embeddings in ``pt`` file.\"\n",
        "      ))\n",
        "    download_btn = generate_download_btn(zip_path)\n",
        "\n",
        "    new_items = items + [dividing_line, zip_hint, download_btn]\n",
        "    custom_display(*new_items)\n",
        "\n",
        "\n",
        "  model_type_box.observe(change_model_type, names=\"value\")\n",
        "  data_type_box.observe(change_data_type, names=\"value\")\n",
        "  upload_type_box.observe(change_upload_type, names='value')\n",
        "  start_btn.on_click(lambda btn: start_thread(start_generation, (btn,)))\n",
        "  display(*items)\n",
        "\n",
        "\n",
        "def share_model():\n",
        "  global refresh_module\n",
        "  refresh_module = share_model\n",
        "\n",
        "  from huggingface_hub import notebook_login\n",
        "  notebook_login()\n",
        "\n",
        "  WIDTH = \"500px\"\n",
        "  HEIGHT = \"30px\"\n",
        "\n",
        "  sep_hint = HTML(markdown.markdown(f\"### {'-'*75}\\n\\n## Please first login to HuggingFace (see above)\"))\n",
        "\n",
        "  upload_hint = HTML(markdown.markdown(f\"### Choose the model yout want to share\"))\n",
        "\n",
        "  share_model_type_box = ipywidgets.Dropdown(\n",
        "              options=[\"Trained by yourself on ColabSaprot\", \"Saved in your local computer\"],\n",
        "              value='Trained by yourself on ColabSaprot',\n",
        "              description='Base model:',\n",
        "              disabled=False,\n",
        "              layout=Layout(width=WIDTH, height=HEIGHT)\n",
        "            )\n",
        "  model_arg_box = select_adapter_from(None, use_model_from=share_model_type_box.value)\n",
        "  model_arg_box.layout.width = WIDTH\n",
        "  model_arg_box.description = \"Select your local model:\"\n",
        "  model_arg_box.style = {'description_width': 'initial'}\n",
        "\n",
        "  upload_model_items = get_upload_box(btn_desc=\"Upload Model-<task_name>-<model_size>.zip file\")\n",
        "  set_upload_visibility(upload_model_items, mode=\"none\")\n",
        "\n",
        "\n",
        "  desc_hint = HTML(markdown.markdown(\n",
        "      f\"### Describe your model\\n\\n\"\n",
        "      \"<font color=red>A more accurate model description enables easier retrieval by other researchers on SaprotHub.</font>\"\n",
        "      ))\n",
        "\n",
        "  task_id_box = ipywidgets.Text(\n",
        "                value=\"\",\n",
        "                placeholder=f'Input the name of your task, e.g. Subcellular_Localization.',\n",
        "                disabled=False,\n",
        "                description=\"Task name:\",\n",
        "                style={'description_width': 'initial'},\n",
        "                layout=Layout(width=WIDTH, height=HEIGHT),\n",
        "                )\n",
        "\n",
        "  task_desc_box = ipywidgets.Textarea(\n",
        "                placeholder=f'Briefly introduce your task.\\n\\nExample:\\n\\nThe task is to predict the catalytic activity of laccase enzymes in lignin degradation based on their protein sequences and structures.',\n",
        "                disabled=False,\n",
        "                description=\"Task description:\",\n",
        "                layout=Layout(width=WIDTH, height=\"200px\"),\n",
        "                style={'description_width': 'initial'},\n",
        "                )\n",
        "  label_meaning_box = ipywidgets.Textarea(\n",
        "                placeholder=f'Briefly introduce the label range/categories predicted by your model.\\n\\nExamples:\\n\\n- Label \"1\" indicates soluble proteins, while \"0\" indicates membrane-bound proteins.\\n\\n- The labels represent fluorescence intensity values ranging from 0 to 1.',\n",
        "                disabled=False,\n",
        "                description=\"Label meaning:\",\n",
        "                layout=Layout(width=WIDTH, height=\"200px\"),\n",
        "                style={'description_width': 'initial'},\n",
        "                )\n",
        "\n",
        "  data_desc_box = ipywidgets.Textarea(\n",
        "                placeholder=f\"Briefly introduce your training data. How was it created or collected? How many training examples? Other details?\",\n",
        "                disabled=False,\n",
        "                description=\"Data description (optional):\",\n",
        "                layout=Layout(width=WIDTH, height=\"200px\"),\n",
        "                style={'description_width': 'initial'},\n",
        "                )\n",
        "\n",
        "  data_src_box = ipywidgets.Text(\n",
        "                value=\"\",\n",
        "                placeholder=f'Training data URL/link or referenced literature.',\n",
        "                disabled=False,\n",
        "                description=\"Data source (optional):\",\n",
        "                style={'description_width': 'initial'},\n",
        "                layout=Layout(width=WIDTH, height=HEIGHT),\n",
        "                )\n",
        "\n",
        "  model_performance_box = ipywidgets.Text(\n",
        "                value=\"\",\n",
        "                placeholder=f'Prediction accuracy on your test set.',\n",
        "                disabled=False,\n",
        "                description=\"Model performance (optional):\",\n",
        "                style={'description_width': 'initial'},\n",
        "                layout=Layout(width=WIDTH, height=HEIGHT),\n",
        "                )\n",
        "\n",
        "  citation_box = ipywidgets.Text(\n",
        "                value=\"\",\n",
        "                placeholder=f'Add your preferred citation here for users to cite.',\n",
        "                disabled=False,\n",
        "                description=\"Citation (optional):\",\n",
        "                style={'description_width': 'initial'},\n",
        "                layout=Layout(width=WIDTH, height=\"HEIGHT\"),\n",
        "                )\n",
        "\n",
        "  upload_hint = HTML(markdown.markdown(\n",
        "    \"<font color=red>Note: Model input type and task type will be detected automatically.</font>\"\n",
        "    ))\n",
        "\n",
        "  upload_to_hf_btn = Button(description='Upload to HuggingFace', layout=Layout(width=WIDTH, height=HEIGHT), button_style=\"info\")\n",
        "\n",
        "\n",
        "  items = [\n",
        "      sep_hint,\n",
        "      upload_hint,\n",
        "      share_model_type_box,\n",
        "      model_arg_box,\n",
        "      desc_hint,\n",
        "      task_id_box,\n",
        "      task_desc_box,\n",
        "      label_meaning_box,\n",
        "      data_desc_box,\n",
        "      data_src_box,\n",
        "      model_performance_box,\n",
        "      citation_box,\n",
        "      upload_hint,\n",
        "      upload_to_hf_btn,\n",
        "  ]\n",
        "\n",
        "  # Used to replace input box based on model type\n",
        "  model_arg_box_idx = items.index(model_arg_box)\n",
        "\n",
        "  # Set click events\n",
        "  def change_model_type(change):\n",
        "    model_type_value = change[\"new\"]\n",
        "\n",
        "    if model_type_value == \"Trained by yourself on ColabSaprot\":\n",
        "      new_model_arg_box = select_adapter_from(None, use_model_from=model_type_value)\n",
        "      new_model_arg_box.layout.width = WIDTH\n",
        "      new_model_arg_box.description = \"Select your local model:\"\n",
        "      new_model_arg_box.style = {'description_width': 'initial'}\n",
        "\n",
        "    elif model_type_value == \"Saved in your local computer\":\n",
        "      set_upload_visibility(upload_model_items, \"default\")\n",
        "      new_model_arg_box = upload_model_items\n",
        "\n",
        "    else:\n",
        "      raise KeyError(model_type_value)\n",
        "\n",
        "    clear_output()\n",
        "    notebook_login()\n",
        "    if isinstance(new_model_arg_box, list):\n",
        "      new_items = items[:model_arg_box_idx] + new_model_arg_box + items[model_arg_box_idx+1:]\n",
        "      display(*new_items)\n",
        "\n",
        "    else:\n",
        "      items[model_arg_box_idx] = new_model_arg_box\n",
        "      display(*items)\n",
        "\n",
        "    display(*system_widgets)\n",
        "\n",
        "  def upload_to_huggingface(button):\n",
        "    from huggingface_hub import HfApi, Repository, ModelFilter\n",
        "    api = HfApi()\n",
        "    user = api.whoami()\n",
        "\n",
        "    name = task_id_box.value.replace(\" \", \"_\")\n",
        "    assert name != \"\", \"Please input your task name!\"\n",
        "\n",
        "    task_desc = task_desc_box.value\n",
        "    assert task_desc != \"\", \"Please introduce your task!\"\n",
        "\n",
        "    label_meaning = label_meaning_box.value\n",
        "    assert label_meaning != \"\", \"Please introduce the meaning of the label predicted by your model!\"\n",
        "\n",
        "    model_performance = model_performance_box.value\n",
        "    data_desc = data_desc_box.value\n",
        "    data_src = data_src_box.value\n",
        "    citation = citation_box.value\n",
        "\n",
        "    repo_name = user['name'] + '/' + name\n",
        "    local_dir = Path(f\"{root_dir}/SaprotHub/model_to_push\") / repo_name\n",
        "    local_dir.mkdir(parents=True, exist_ok=True)\n",
        "\n",
        "    repo_list = [repo.id for repo in api.list_models(filter=ModelFilter(author=user['name']))]\n",
        "    if repo_name not in repo_list:\n",
        "      api.create_repo(repo_name, private=False)\n",
        "\n",
        "    repo = Repository(local_dir=local_dir, clone_from=repo_name)\n",
        "\n",
        "    model_arg = items[model_arg_box_idx].value\n",
        "    if share_model_type_box.value == \"Saved in your local computer\":\n",
        "      zip_path = get_upload_file_path(upload_model_items)\n",
        "      zip_name = os.path.basename(zip_path)\n",
        "      save_dir = ADAPTER_HOME / \"Upload\" / zip_name.rsplit('.', 1)[0]\n",
        "      os.makedirs(save_dir, exist_ok=True)\n",
        "      # unzip model.zip\n",
        "      with zipfile.ZipFile(zip_path, 'r') as zip_ref:\n",
        "          zip_ref.extractall(save_dir)\n",
        "      model_arg = f\"Upload/{zip_name.rsplit('.', 1)[0]}\"\n",
        "\n",
        "    command = f\"cp '{ADAPTER_HOME}/{model_arg}'/* '{local_dir}/'\"\n",
        "    subprocess.run(command, shell=True)\n",
        "\n",
        "    ######################################################################\n",
        "    #             Modify README              #\n",
        "    ######################################################################\n",
        "    import json\n",
        "\n",
        "    md_path = local_dir / \"README.md\"\n",
        "\n",
        "    # Obtain base model\n",
        "    adapter_config_path = local_dir / \"adapter_config.json\"\n",
        "    with open(adapter_config_path, \"r\") as r:\n",
        "      config_dict = json.load(r)\n",
        "      base_model = config_dict[\"base_model_name_or_path\"]\n",
        "\n",
        "    model_desc = \"## Base model\\n\\n\" \\\n",
        "            f\"[{base_model}](https://huggingface.co/{base_model})\\n\\n\" \\\n",
        "            \"## Task description\\n\\n\" \\\n",
        "            f\"{task_desc}\\n\\n\" \\\n",
        "            \"## Label meaning\\n\\n\" \\\n",
        "            f\"{label_meaning}\\n\\n\"\n",
        "\n",
        "    if data_desc != \"\":\n",
        "      model_desc += \"## Data description\\n\\n\" \\\n",
        "              f\"{data_desc}\\n\\n\" \\\n",
        "\n",
        "    if data_src != \"\":\n",
        "      model_desc += \"## Data source\\n\\n\" \\\n",
        "              f\"{data_src}\\n\\n\" \\\n",
        "\n",
        "    if model_performance != \"\":\n",
        "      model_desc += \"## Model performance (on test set)\\n\\n\" \\\n",
        "              f\"{model_performance}\\n\\n\" \\\n",
        "\n",
        "    if citation != \"\":\n",
        "      model_desc += \"## Citation\\n\\n\" \\\n",
        "              \"Please consider citing the following work if you find the model useful:\\n\\n\" \\\n",
        "              f\"{citation}\\n\\n\" \\\n",
        "\n",
        "    replace_data = {\n",
        "        '<slot name=\\'description\\'>': model_desc,\n",
        "    }\n",
        "\n",
        "    with open(md_path, \"r\") as file:\n",
        "        content = file.read()\n",
        "\n",
        "    for key, value in replace_data.items():\n",
        "        if value != \"\":\n",
        "            content = content.replace(key, value)\n",
        "\n",
        "    with open(md_path, \"w\") as file:\n",
        "        file.write(content)\n",
        "\n",
        "    ######################################################################\n",
        "    #          Push to HuggingFace             #\n",
        "    ######################################################################\n",
        "    repo.push_to_hub(commit_message=\"Upload adapter model\")\n",
        "    url = f\"https://huggingface.co/{user['name']}/{name}\"\n",
        "    finish_hint = HTML(markdown.markdown(\n",
        "        f\"**You model has been successfully uploaded to <a href='{url}' target='blank'>{url}</a>!**\\n\\n\"\n",
        "        \"**You can further move your model from personal repository to official SaprotHub, see <a href='https://github.com/westlake-repl/SaprotHub/wiki/SaprotHub-v2-(latest)#share-your-model-to-official-saprothub' target='blank'>here</a>!**\\n\\n\"\n",
        "        ))\n",
        "    display(finish_hint)\n",
        "\n",
        "  share_model_type_box.observe(change_model_type, names=\"value\")\n",
        "  upload_to_hf_btn.on_click(upload_to_huggingface)\n",
        "  display(*items)\n",
        "\n",
        "\n",
        "# Stop the running task\n",
        "def stop_thread(button):\n",
        "  global now_thread\n",
        "  if now_thread:\n",
        "    print(\"\\033[31mInterupt the running task...\\033[0m\")\n",
        "    while now_thread.is_alive():\n",
        "      import inspect\n",
        "      import ctypes\n",
        "\n",
        "      # Stop the thread\n",
        "      exctype = SystemExit\n",
        "      tid = ctypes.c_long(now_thread.ident)\n",
        "      if not inspect.isclass(exctype):\n",
        "          exctype = type(exctype)\n",
        "      res = ctypes.pythonapi.PyThreadState_SetAsyncExc(tid, ctypes.py_object(exctype))\n",
        "      if res == 0:\n",
        "          # raise ValueError(\"invalid thread id\")\n",
        "          continue\n",
        "      elif res != 1:\n",
        "          # \"\"\"if it returns a number greater than one, you're in trouble,\n",
        "          # and you should call it again with exc=NULL to revert the effect\"\"\"\n",
        "          ctypes.pythonapi.PyThreadState_SetAsyncExc(tid, None)\n",
        "          # raise SystemError(\"PyThreadState_SetAsyncExc failed\")\n",
        "          continue\n",
        "\n",
        "    print(\"\\033[31m\\nTask interrupted by user!\\033[0m\")\n",
        "    now_thread = None\n",
        "\n",
        "  else:\n",
        "    print(\"\\033[31m\\nNo running task to be stopped!\\033[0m\")\n",
        "\n",
        "\n",
        "# Start a new thread when executing a task\n",
        "def start_thread(func, args=tuple()):\n",
        "  # def test(button):\n",
        "  #   save_path = \"/sujin/temp/test.tsv\"\n",
        "  #   with open(save_path, \"w\") as w:\n",
        "  #     for i in range(1000):\n",
        "  #       print(i)\n",
        "  #       w.write(f\"{i}\\n\")\n",
        "  #       w.flush()\n",
        "  #       time.sleep(0.1)\n",
        "\n",
        "  # disable_wrapper(func)(*args)\n",
        "  thread = threading.Thread(target=disable_wrapper(func), args=args)\n",
        "  thread.start()\n",
        "\n",
        "  global now_thread\n",
        "  now_thread = thread\n",
        "\n",
        "\n",
        "# Jump to next page\n",
        "def jump(button, next=None):\n",
        "  # Stop the running task\n",
        "  stop_thread(None)\n",
        "\n",
        "  clear_output()\n",
        "\n",
        "  if next is None:\n",
        "    # Refresh current module\n",
        "    refresh_module()\n",
        "\n",
        "  else:\n",
        "    # Jump to next module\n",
        "    next()\n",
        "\n",
        "  display(*system_widgets)\n",
        "\n",
        "\n",
        "def custom_display(*args, **kwargs):\n",
        "  clear_output()\n",
        "  display(*args, **kwargs)\n",
        "  display(*system_widgets)\n",
        "\n",
        "\n",
        "# Current running thread\n",
        "now_thread = None\n",
        "\n",
        "HINT = HTML(markdown.markdown(\"**<font color=red>Note: At any time you can click below buttons to stop and restart</font>**\"))\n",
        "stop_btn = Button(description=\"Stop\", button_style=\"danger\")\n",
        "back_btn = Button(description=\"Go back\", button_style=\"success\")\n",
        "refresh_btn = Button(description=\"Refresh\", button_style=\"success\")\n",
        "system_hint = HTML(markdown.markdown(\n",
        "    \"**Go back:** Stop the running task and go back to the first interface\\n\\n\"\n",
        "    \"**Refresh**: Stop the running task and refresh the current interface\\n\\n\"\n",
        "    \"**Stop**: Stop the running task\"\n",
        "    ))\n",
        "\n",
        "refresh_module = None\n",
        "back_btn.on_click(partial(jump, next=train_or_pred))\n",
        "refresh_btn.on_click(jump)\n",
        "stop_btn.on_click(stop_thread)\n",
        "\n",
        "train_or_pred()\n",
        "system_widgets = [HINT, ipywidgets.HBox([back_btn, refresh_btn, stop_btn]), system_hint]\n",
        "display(*system_widgets)\n",
        "\n",
        "with ui_events() as poll:\n",
        "  try:\n",
        "    # start_thread(None, (None,))\n",
        "    while True:\n",
        "      poll(10)\n",
        "      # print(now_thread)\n",
        "      time.sleep(0.1)\n",
        "\n",
        "  except Exception:\n",
        "    pass\n",
        "\n",
        "  finally:\n",
        "    stop_thread(None)\n",
        "    clear_output()\n",
        "    end_hint = HTML(markdown.markdown(\"## The program is interrupted. You could click the run-button to restart.\"))\n",
        "    display(end_hint)"
      ],
      "metadata": {
        "id": "LHKi8xI-aU4E",
        "cellView": "form",
        "outputId": "c212c303-fc75-41d5-cb9e-8f049a76c1e1",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000,
          "referenced_widgets": [
            "cb892f8a799b49a79e68f86a2214cc32",
            "31116bf082be4db4a0445dacfd75adf8",
            "b2c0ad6e0eec45b0b7d715432b5ede67",
            "6f56172384284742809974592f2c56f9",
            "e97ed5724e2a49ddbf3fd14ef1470b55",
            "905790c74150464f8727a378ab71a248",
            "afd0e84440484bdf97b1a8252bda0249",
            "7c90febfd5394f648248398805e99620",
            "c329a7db5be0435595c2bbdf7b072de8",
            "4f0f730b7266466b81143e5cc98f1f38",
            "5944035516eb48efa3bab5e5d4c047e0",
            "84c915f4c24b47019c3a57fd64584b0c",
            "1e3e43d7104a46bc82b43ca8d0cbc786",
            "153fcbc5e217499eb8d0d404d2ab9915",
            "2ab7a019c0af47e78954085d5df3bac7",
            "99cac01948ec49e28342c51c0f8a7cb7",
            "0883c3d8f0f94f56b46bcbb4d1997c48",
            "c2399df31a0540ed802cd557481b3dbd",
            "3eccfa1db44a4344b43dc726d20a512b",
            "75d32a15b59449c78511319420a6335e",
            "f4c926ba549c4e8cb73d505ed00eebdc",
            "234301fa55ac438d99d13083ca14747e",
            "53f47a56a11b45ed962800ebce4e0326",
            "59ac90a30a644eeaabc24b8cafd11278",
            "a584fa36c41f457c949110f63f22ee2b",
            "e944b600328242e4a2b37ee73f370037",
            "a34d2e740cf849408137abc6a474f818",
            "bebe165d768d44689d04b95e5529d1a4",
            "d8b0829952da4a70ac337f26256af992",
            "fa4c378424fe4ffc903b08584dd6ae47",
            "b8e0bee67db8463c9924cd7c09f4ca6e",
            "1dec01a5339c4cf69f8f80437f220a30",
            "fdcc3a61ae9c4844980c0f3fe6d4597a",
            "6ef3b636cc274069946cbffb38cc190a",
            "6d813806bc49426396e6b0fd77669627",
            "e33c5526e0a741fa8b625653280c6fae",
            "131a7eb8af1c4e359243b1c77270cd4f",
            "126cc102032b483288f5368954881958",
            "444a0ec1fea2499ea163722c54831b38",
            "3757ba3aeafe4941b9f31a16cc972b5d",
            "429d2ee9e7ab4e809306043612194632",
            "776ad3846df34250a96370dfe2ed8832",
            "3290bce4312345edbb9efdcc218ddda3",
            "bc4c5346a9aa4df990c197a8c44f70f8",
            "49168b26cc2c4f13bf6c368841ba975e",
            "71557d0108404be8aced4d707743dc59",
            "db95609d0f1143759c5938be8a88ddd9",
            "91557ea908cf4b2680a193fe32d96001",
            "55937092eb6b4bfeb492b1b0b66fb996",
            "d7e95855731e4332a46f9e9f9711b63e",
            "6ff6e7cd96044210a83d26e6f3030f9e",
            "d15a3542c1564708a1cb5d96e5a084a5",
            "c15dc1d27091430c83e8c99524ac7a38",
            "feddedbe2ee54a9cac2fbea88e27172e",
            "7f45b74269744c05ba7a99af9423c01b",
            "3fd98403ec1a402a8fec39c898bc3134",
            "579c36f72f5e48b7a1319f34b150dc6d",
            "1ad70eee4d07494bb9d14d378b75aafd",
            "7fa1247efd5a4cd1af121a241dcf56cf",
            "4381953e5b964201b48c3e0f07c6a7a8",
            "74583fff751a4c0daa90522e3e23319d",
            "f6ee8c85fccd4f998fada4bde46598b8",
            "033205f8a7714146b4b750323bf9a3e2",
            "107447b6cb0e4e8685008bdc5445e6c0",
            "93ed434d4e4248e9888c12916376514e",
            "54822490622c4729b0d93f9bf9d8f536",
            "78124bcbe7ac4464aca0e97785d389ea",
            "f7ef258e9cf540939f611f10ae48d5d2",
            "43a6c8d9b31345fb815e8e2d04d142a7",
            "e6c963ef9e7b446aaa8889d7a252143a",
            "ed9665034f78414fb70918210e013070",
            "e5a2715f8f514d3c9f7eb18bc4ca198f",
            "f3f62515c2e243b59f7f82a07c8b307d",
            "3086f99a43fd4867aa06fc1985adf9cd",
            "745169f17570408dbea231bfd45888a1",
            "0c4f36c100ef49c8b99bb92f51417803",
            "9fab2e2685d4443799ab51bed7057ca0",
            "e481bd8d7a844e74acc75e6a7e2d0ecb",
            "16417102ff5d41c1a83e61c56f425de9",
            "fabede3b814e4e47a2b9199c1ae0458c",
            "842be5fc77934a7d930230ec77c072a7",
            "5dde927371f549b2a56d0944ec07ff2d",
            "4ca8b09e41644f05a7f1ff178b138418",
            "e26629d88f764328bd324abc05bac619",
            "0318445a8dd14013abb3e8c2270eabbe",
            "aff7949f8ce14342981f650d3e6c5327",
            "c4af9f968ff14e70a3c436b4acf8cf84",
            "39234706a8b145c08bd4f3070c2511e6",
            "3596f01850f349ce88e8be87abdc18cb",
            "75b99b20537944aeb1deea199c756e41",
            "3ccc6beee0cd40c98001062e97634d4c",
            "3212c28719a24c50a77be0a886681d4c",
            "0999c826dff54b2e8767ee4f85c7b58e",
            "cd5978e89963496284182842414ba373",
            "865069f1a237426ea1beb1b37867db4b",
            "c7a6d65400324c5ea6ce76d80ddaf921",
            "9c606953d00c44d0b1067f37eafcdae6",
            "0cee1d8fb7ee44dd8305c34d24772beb",
            "700fe055b07d466ca49bb28f45d1dd8c",
            "1ecfcf1451f542f4a2426cf600c5405d",
            "83ef374662834a9facb743377207acac",
            "3ecc0304993a4badac618fc1f072f045",
            "a72202a60c9648f3889546e5c5fb85e5",
            "425e2e46cf4948749afee64d51772e35",
            "b8bd64125e7c4082bfa7992d56c1f745",
            "42142b648d25435592c2cd412abae2c8",
            "fafe0a31a1b24e6b81f4dbf808ef882f",
            "0bde97d05de44b5190444541ad0ec38a",
            "685e62f8fe1943ff82ff1f0c642f57b6",
            "0527f077b3914d84b3225016150abc48",
            "eb6e64c5b0b34d8081c88e032aaf6fdc",
            "a54569f2806a4f3daf0ec81078787519",
            "8d4889f9613b48d4a41616a9bc99add5",
            "c2c9fec6578346478b6244b3a6db2176",
            "3d82d2dd59c142b5b351df4f715cb565",
            "6bbe3cf2f3d7437b99b3882309d5c344",
            "e556d9a6341b4b959bc1267167860e5f",
            "3c37c9fc693a4cab88d5f346ad686463",
            "a54de0c45e6e47418945f198f3fc0500",
            "53393a656e7f4450ae08541b972b4d75",
            "d4a09839fa964cc4ab0a63dfb798db70",
            "0d718fcee69d49f9a4d6953603c8e912",
            "572888144abf4345ae5b0f41f0457acc",
            "7032ff2e022746e39d9389e28c67b5e5",
            "915f0e154d304c7d8cb6f2ae183c355e",
            "ac45630d0bb74b3694e9fe9231cc96eb",
            "8f7c63d564804352a86173123e9c567d",
            "7f3eb6775b0b4049852cf45bc3c325fa",
            "b157ba138a314ff48d8247dc1895b970",
            "d22367a764864b198755c00beb4e8a9b",
            "2cacff63343949c1ac6412a465eb6fb7",
            "95dcb311094f44fdba62a5253d83b0bf",
            "9ae785c8c8304b12a3926c9a7e506adb",
            "7e20bb2da89e4753b57e2a9678d8a9a3",
            "0a0feface3a4407686eef7e3a90be88e",
            "25cc738bc6ca4e9cb817e4d4633a3fa5",
            "1a4c9ca971b24e6084038b1d718c6de0",
            "0a753a8a60614f65bb440c80727d76f9",
            "230d4790e9cb4b058926eaf064405ea1",
            "31dae71c0ae04187ad1802e8df826f87",
            "cd907850ca024b20971715368f018664",
            "e68866bbfa3e4d2eaeec08ad4c9c8afa",
            "0bbcc91ab0ab4a9fb91c5b996e0c1829",
            "30fe7b0e433a4bf19dc8bda8c5660cde",
            "835135854bde4df58361f55bdfee936d",
            "b879a1097e8747dc90e2fd4873fe4a61",
            "18f9708b2aee49a18045725bed317cff",
            "e13621fc126142378f91e6865624b4d2",
            "f055a639b2c7476c9a9c160f48d3cd24",
            "ec9dee9766004a01ba65a23bc867342d",
            "bfe0d6e243c745c7b4a91d024fd4d1f1",
            "14290117e08945b2995351621d915666",
            "41c2e69509d743cc8b6b33c5a5b916a6",
            "021de2c3ab284151aa226fcee9596873",
            "cfcd072b06434b0281391d86555d834f",
            "04e1fc6fc23f475fa6ecdc3210225928",
            "3516f2d47c8e4aa6aa1173e56c776589",
            "448afc138e0a4888905e6d4b780a55dd",
            "f89bc13783314b379a6ce2b3a4fe2f43",
            "663b48cc54484b6b8eee3ebc7e0c30d3",
            "14315ef2e22b468d8eb538c77662cb47",
            "0b0cba32e3694402bc9036abab46993e",
            "f9134f53502d4adfb65692b0422bc101",
            "5f30f4b7a5a348468ec91651bec24e88",
            "5ba9fc68af354bbf896746fa8cb1ebf0",
            "0c5af4b1b42040eebdaab9ad9d14ce19",
            "66f274d78e444de193e952c37087a51c",
            "ac397aa11a1149f79d48735027d704eb",
            "9a8173f1fe3f4125a509823737170499",
            "4dde9740edb5472083597c01d496d88b",
            "b884f523af2f47f18f69da35baed22e8",
            "1617bd4931114311b1e0f5eb93887582",
            "20d4faece52248b58dda81eaa21b6fc0",
            "690daf1037fc44e7bb2081bb4b925e83",
            "d822bdc17f904f5aac71464412167942",
            "b793a44d6d4749fcac9240417c3cb87f",
            "b53d5879bde54f3cbb6b59dea6ac46cf",
            "a53fe7a3ab984f36836ca6b1947f6c52",
            "1fd5878a9c454e11a6b5517b66a36403",
            "03962db06c784275bc2e16a81c253b41",
            "c942691f13934b7f86a82d7daad16097",
            "d9cb38272f76435e989dd432a26e68a2",
            "5e060167622c4f3ea4c29a0626d6c0e4",
            "6d7d6979ffe74f468b7eb6b3c3aafafb",
            "e987f8de9db644b2b2226a5163a2fa0d",
            "1341d23d8a5e49aa85595ebec1b72220",
            "f6dc9ae6b23e4ada9c3414915e647b17",
            "e9aa8bbc4b8d4589ba653cc69237d70b",
            "3b4323756ca64e0f89afc0fc8c61418e",
            "0ddd93e7bab947ba94864af87b6362c3",
            "05201e9c8a6544deaeed712140073e81",
            "e91d225d33cb4d3eb8bb8cf34d8749aa",
            "571d2b84906d4a87adfe77f1449ffbcb",
            "b155c5def25b46728fe8f03050f72f2b",
            "ee34ea4748ae4d3d89320b31d6662f79",
            "7ce642e79f574c7ebc79cb52050eb31c",
            "1ceb03de8458467d8da1231757f982b3",
            "55138d9d4e584e90b60c687f2848c430",
            "8f54f69eb0264944879f003de7da4b43",
            "5ad7202b8dff481d80be9d1d0686588e",
            "c54c486c9f8d4151aae04cf93d09c6f5",
            "ab859c29b67641249ed3a391d6a3f607",
            "8403502d130f4e9abf7eba0091f48091",
            "08c185d7c7204904b9d9ef3a285be90f",
            "4902a6111f87440db2145b576fc75a8e",
            "fc066b6a54fc4acdbff9e4ed9bdb43a4",
            "b02e285f17c6494da4582accc2e945bc",
            "c99ee45b34174aabab065748325949fc",
            "fd244b32346142cca415edf38d2056c6",
            "cdeb7cbf4af74aa793149643a26e3403",
            "94ecdefcd9594129b025c2a87e05d83b",
            "47359149efa2418e81cc4d23307b1414",
            "3ed70a3857a24b39b71ba5a32dcdcb90",
            "d626ed4d4b604fc4a7019e321ce91a22",
            "5ca5b1edd3284635b61bf9023433b924",
            "d0d00befdc3946b4ac72d08259c5eec2"
          ]
        }
      },
      "id": "LHKi8xI-aU4E",
      "execution_count": null,
      "outputs": [
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "HTML(value='<h2>Protein sequence design</h2>')"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "c7a6d65400324c5ea6ce76d80ddaf921"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "HTML(value='<h3>Please upload the structure backbone:</h3>')"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "700fe055b07d466ca49bb28f45d1dd8c"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "HTML(value='<p>Chain (to be extracted from the structure):</p>')"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "3ecc0304993a4badac618fc1f072f045"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "Text(value='A', layout=Layout(height='30px', width='500px'), placeholder='Enter the name of chain here')"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "b8bd64125e7c4082bfa7992d56c1f745"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "HTML(value='<p>Upload the protein structure:</p>')"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "0bde97d05de44b5190444541ad0ec38a"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "RadioButtons(description='Is the file size > 5MB?', index=1, layout=Layout(width='max-content'), options=('Yes‚Ä¶"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "eb6e64c5b0b34d8081c88e032aaf6fdc"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "HTML(value=\"<p><a href='https://github.com/westlake-repl/SaprotHub/wiki/SaprotHub-v2-%28latest%29#get-your-fil‚Ä¶"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "c2c9fec6578346478b6244b3a6db2176"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "Text(value='', description='File path:', layout=Layout(display='none', height='30px', width='500px'), placehol‚Ä¶"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "e556d9a6341b4b959bc1267167860e5f"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "Button(button_style='info', description='001-chai1.pdb', layout=Layout(height='30px', width='500px'), style=Bu‚Ä¶"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "53393a656e7f4450ae08541b972b4d75"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "HTML(value='\\n      <input type=\"file\" id=\"files-7c75b2e4-9bd8-4539-b4fd-ac9789867c03\" name=\"files[]\"  disable‚Ä¶"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "572888144abf4345ae5b0f41f0457acc"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "Button(button_style='info', description='Submit', layout=Layout(height='30px', width='500px'), style=ButtonSty‚Ä¶"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "ac45630d0bb74b3694e9fe9231cc96eb"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "HTML(value='<p><strong>Foldseek sequence of chain A (\"#\" means low pLDDT positions that are masked):</strong><‚Ä¶"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "b157ba138a314ff48d8247dc1895b970"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "application/3dmoljs_load.v0": "<div id=\"3dmolviewer_1742372256039599\"  style=\"position: relative; width: 640px; height: 480px;\">\n        <p id=\"3dmolwarning_1742372256039599\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n        </div>\n<script>\n\nvar loadScriptAsync = function(uri){\n  return new Promise((resolve, reject) => {\n    //this is to ignore the existence of requirejs amd\n    var savedexports, savedmodule;\n    if (typeof exports !== 'undefined') savedexports = exports;\n    else exports = {}\n    if (typeof module !== 'undefined') savedmodule = module;\n    else module = {}\n\n    var tag = document.createElement('script');\n    tag.src = uri;\n    tag.async = true;\n    tag.onload = () => {\n        exports = savedexports;\n        module = savedmodule;\n        resolve();\n    };\n  var firstScriptTag = document.getElementsByTagName('script')[0];\n  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n});\n};\n\nif(typeof $3Dmolpromise === 'undefined') {\n$3Dmolpromise = null;\n  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n}\n\nvar viewer_1742372256039599 = null;\nvar warn = document.getElementById(\"3dmolwarning_1742372256039599\");\nif(warn) {\n    warn.parentNode.removeChild(warn);\n}\n$3Dmolpromise.then(function() {\nviewer_1742372256039599 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1742372256039599\"),{backgroundColor:\"white\"});\nviewer_1742372256039599.zoomTo();\n\tviewer_1742372256039599.addModel(\"ATOM      1  N   MET A   1     -27.698 -28.468 -31.491  1.00 37.85      A    N  \\nATOM      2  CA  MET A   1     -26.538 -28.314 -30.616  1.00 43.90      A    C  \\nATOM      3  C   MET A   1     -25.381 -27.672 -31.350  1.00 43.61      A    C  \\nATOM      4  O   MET A   1     -24.600 -26.941 -30.736  1.00 41.03      A    O  \\nATOM      5  CB  MET A   1     -26.113 -29.677 -30.027  1.00 37.01      A    C  \\nATOM      6  CG  MET A   1     -27.117 -30.312 -29.076  1.00 37.68      A    C  \\nATOM      7  SD  MET A   1     -26.655 -31.905 -28.452  1.00 37.85      A    S  \\nATOM      8  CE  MET A   1     -25.334 -31.405 -27.350  1.00 31.57      A    C  \\nATOM      9  N   SER A   2     -25.273 -28.007 -32.614  1.00 54.59      A    N  \\nATOM     10  CA  SER A   2     -24.216 -27.414 -33.427  1.00 59.61      A    C  \\nATOM     11  C   SER A   2     -24.461 -25.938 -33.664  1.00 60.18      A    C  \\nATOM     12  O   SER A   2     -23.539 -25.156 -33.832  1.00 57.16      A    O  \\nATOM     13  CB  SER A   2     -24.072 -28.141 -34.756  1.00 54.61      A    C  \\nATOM     14  OG  SER A   2     -25.265 -28.009 -35.519  1.00 51.31      A    O  \\nATOM     15  N   ASN A   3     -25.742 -25.559 -33.696  1.00 55.15      A    N  \\nATOM     16  CA  ASN A   3     -26.063 -24.138 -33.866  1.00 59.47      A    C  \\nATOM     17  C   ASN A   3     -25.781 -23.356 -32.598  1.00 59.93      A    C  \\nATOM     18  O   ASN A   3     -25.398 -22.188 -32.664  1.00 58.24      A    O  \\nATOM     19  CB  ASN A   3     -27.533 -23.968 -34.289  1.00 55.02      A    C  \\nATOM     20  CG  ASN A   3     -27.791 -24.444 -35.689  1.00 52.56      A    C  \\nATOM     21  ND2 ASN A   3     -28.932 -25.089 -35.886  1.00 47.22      A    N  \\nATOM     22  OD1 ASN A   3     -27.008 -24.202 -36.580  1.00 44.78      A    O  \\nATOM     23  N   LYS A   4     -25.979 -23.988 -31.443  1.00 53.58      A    N  \\nATOM     24  CA  LYS A   4     -25.708 -23.296 -30.178  1.00 58.31      A    C  \\nATOM     25  C   LYS A   4     -24.236 -23.097 -29.961  1.00 59.36      A    C  \\nATOM     26  O   LYS A   4     -23.807 -22.081 -29.396  1.00 58.08      A    O  \\nATOM     27  CB  LYS A   4     -26.313 -24.058 -29.010  1.00 54.65      A    C  \\nATOM     28  CG  LYS A   4     -27.840 -24.071 -29.000  1.00 52.08      A    C  \\nATOM     29  CD  LYS A   4     -28.393 -24.854 -27.861  1.00 47.90      A    C  \\nATOM     30  CE  LYS A   4     -29.918 -24.866 -27.833  1.00 39.43      A    C  \\nATOM     31  NZ  LYS A   4     -30.452 -25.610 -26.668  1.00 34.86      A    N1+\\nATOM     32  N   VAL A   5     -23.440 -24.074 -30.358  1.00 57.03      A    N  \\nATOM     33  CA  VAL A   5     -21.988 -23.940 -30.272  1.00 61.05      A    C  \\nATOM     34  C   VAL A   5     -21.470 -22.890 -31.213  1.00 61.96      A    C  \\nATOM     35  O   VAL A   5     -20.566 -22.118 -30.887  1.00 60.33      A    O  \\nATOM     36  CB  VAL A   5     -21.309 -25.303 -30.482  1.00 57.55      A    C  \\nATOM     37  CG1 VAL A   5     -21.748 -26.273 -29.402  1.00 57.63      A    C  \\nATOM     38  CG2 VAL A   5     -19.807 -25.168 -30.493  1.00 56.03      A    C  \\nATOM     39  N   SER A   6     -22.064 -22.832 -32.386  1.00 61.71      A    N  \\nATOM     40  CA  SER A   6     -21.648 -21.796 -33.346  1.00 65.53      A    C  \\nATOM     41  C   SER A   6     -22.087 -20.407 -32.900  1.00 66.06      A    C  \\nATOM     42  O   SER A   6     -21.425 -19.425 -33.203  1.00 64.27      A    O  \\nATOM     43  CB  SER A   6     -22.243 -22.109 -34.725  1.00 62.12      A    C  \\nATOM     44  OG  SER A   6     -23.675 -22.003 -34.697  1.00 54.71      A    O  \\nATOM     45  N   ARG A   7     -23.182 -20.340 -32.162  1.00 60.92      A    N  \\nATOM     46  CA  ARG A   7     -23.619 -19.039 -31.639  1.00 65.26      A    C  \\nATOM     47  C   ARG A   7     -22.756 -18.571 -30.502  1.00 66.29      A    C  \\nATOM     48  O   ARG A   7     -22.514 -17.349 -30.337  1.00 65.30      A    O  \\nATOM     49  CB  ARG A   7     -25.085 -19.092 -31.196  1.00 61.65      A    C  \\nATOM     50  CG  ARG A   7     -26.072 -19.183 -32.366  1.00 57.08      A    C  \\nATOM     51  CD  ARG A   7     -27.502 -19.318 -31.907  1.00 52.65      A    C  \\nATOM     52  NE  ARG A   7     -28.444 -19.422 -32.998  1.00 47.03      A    N  \\nATOM     53  CZ  ARG A   7     -29.753 -19.554 -32.840  1.00 45.97      A    C  \\nATOM     54  NH1 ARG A   7     -30.288 -19.572 -31.649  1.00 38.79      A    N1+\\nATOM     55  NH2 ARG A   7     -30.534 -19.678 -33.905  1.00 36.64      A    N  \\nATOM     56  N   THR A   8     -22.302 -19.501 -29.657  1.00 58.96      A    N  \\nATOM     57  CA  THR A   8     -21.379 -19.110 -28.585  1.00 61.14      A    C  \\nATOM     58  C   THR A   8     -20.036 -18.711 -29.121  1.00 62.16      A    C  \\nATOM     59  O   THR A   8     -19.371 -17.842 -28.560  1.00 61.06      A    O  \\nATOM     60  CB  THR A   8     -21.242 -20.260 -27.593  1.00 57.01      A    C  \\nATOM     61  CG2 THR A   8     -22.583 -20.559 -26.920  1.00 53.46      A    C  \\nATOM     62  OG1 THR A   8     -20.769 -21.436 -28.258  1.00 56.02      A    O  \\nATOM     63  N   ASP A   9     -19.602 -19.346 -30.199  1.00 65.02      A    N  \\nATOM     64  CA  ASP A   9     -18.359 -18.945 -30.841  1.00 69.20      A    C  \\nATOM     65  C   ASP A   9     -18.478 -17.597 -31.519  1.00 69.94      A    C  \\nATOM     66  O   ASP A   9     -17.516 -16.832 -31.565  1.00 69.21      A    O  \\nATOM     67  CB  ASP A   9     -17.886 -20.016 -31.839  1.00 66.86      A    C  \\nATOM     68  CG  ASP A   9     -17.472 -21.299 -31.149  1.00 62.83      A    C  \\nATOM     69  OD1 ASP A   9     -17.004 -21.220 -30.003  1.00 55.10      A    O  \\nATOM     70  OD2 ASP A   9     -17.604 -22.371 -31.777  1.00 56.63      A    O1-\\nATOM     71  N   SER A  10     -19.643 -17.311 -32.035  1.00 66.15      A    N  \\nATOM     72  CA  SER A  10     -19.871 -15.998 -32.620  1.00 68.97      A    C  \\nATOM     73  C   SER A  10     -19.962 -14.912 -31.544  1.00 69.07      A    C  \\nATOM     74  O   SER A  10     -19.538 -13.770 -31.771  1.00 68.69      A    O  \\nATOM     75  CB  SER A  10     -21.133 -15.999 -33.475  1.00 66.85      A    C  \\nATOM     76  OG  SER A  10     -22.283 -16.229 -32.663  1.00 57.90      A    O  \\nATOM     77  N   LEU A  11     -20.495 -15.271 -30.385  1.00 65.25      A    N  \\nATOM     78  CA  LEU A  11     -20.533 -14.318 -29.283  1.00 67.80      A    C  \\nATOM     79  C   LEU A  11     -19.159 -14.079 -28.687  1.00 68.57      A    C  \\nATOM     80  O   LEU A  11     -18.862 -12.999 -28.177  1.00 67.49      A    O  \\nATOM     81  CB  LEU A  11     -21.474 -14.819 -28.198  1.00 63.85      A    C  \\nATOM     82  CG  LEU A  11     -22.970 -14.756 -28.519  1.00 59.63      A    C  \\nATOM     83  CD1 LEU A  11     -23.436 -13.331 -28.658  1.00 60.06      A    C  \\nATOM     84  CD2 LEU A  11     -23.766 -15.467 -27.432  1.00 58.55      A    C  \\nATOM     85  N   LYS A  12     -18.288 -15.100 -28.727  1.00 65.13      A    N  \\nATOM     86  CA  LYS A  12     -16.927 -14.923 -28.277  1.00 66.72      A    C  \\nATOM     87  C   LYS A  12     -16.123 -14.024 -29.213  1.00 66.74      A    C  \\nATOM     88  O   LYS A  12     -15.278 -13.245 -28.773  1.00 66.72      A    O  \\nATOM     89  CB  LYS A  12     -16.221 -16.256 -28.129  1.00 65.04      A    C  \\nATOM     90  CG  LYS A  12     -16.689 -17.077 -26.932  1.00 60.90      A    C  \\nATOM     91  CD  LYS A  12     -16.019 -18.405 -26.851  1.00 58.20      A    C  \\nATOM     92  CE  LYS A  12     -16.515 -19.217 -25.668  1.00 52.14      A    C  \\nATOM     93  NZ  LYS A  12     -15.844 -20.524 -25.563  1.00 46.35      A    N1+\\nATOM     94  N   LYS A  13     -16.440 -14.093 -30.469  1.00 62.63      A    N  \\nATOM     95  CA  LYS A  13     -15.786 -13.216 -31.444  1.00 64.54      A    C  \\nATOM     96  C   LYS A  13     -16.295 -11.789 -31.361  1.00 63.85      A    C  \\nATOM     97  O   LYS A  13     -15.537 -10.837 -31.617  1.00 63.33      A    O  \\nATOM     98  CB  LYS A  13     -15.943 -13.757 -32.860  1.00 62.52      A    C  \\nATOM     99  CG  LYS A  13     -15.166 -15.039 -33.143  1.00 57.87      A    C  \\nATOM    100  CD  LYS A  13     -15.407 -15.520 -34.553  1.00 54.50      A    C  \\nATOM    101  CE  LYS A  13     -14.695 -16.788 -34.842  1.00 48.48      A    C  \\nATOM    102  NZ  LYS A  13     -14.879 -17.286 -36.194  1.00 43.57      A    N1+\\nATOM    103  N   THR A  14     -17.551 -11.640 -31.025  1.00 65.47      A    N  \\nATOM    104  CA  THR A  14     -18.099 -10.285 -30.860  1.00 65.69      A    C  \\nATOM    105  C   THR A  14     -17.641  -9.661 -29.532  1.00 64.98      A    C  \\nATOM    106  O   THR A  14     -17.450  -8.450 -29.482  1.00 64.14      A    O  \\nATOM    107  CB  THR A  14     -19.618 -10.292 -30.919  1.00 63.17      A    C  \\nATOM    108  CG2 THR A  14     -20.069 -10.794 -32.279  1.00 58.28      A    C  \\nATOM    109  OG1 THR A  14     -20.158 -11.138 -29.893  1.00 61.42      A    O  \\nATOM    110  N   LEU A  15     -17.459 -10.496 -28.518  1.00 59.61      A    N  \\nATOM    111  CA  LEU A  15     -16.943  -9.966 -27.266  1.00 61.72      A    C  \\nATOM    112  C   LEU A  15     -15.471  -9.600 -27.352  1.00 62.02      A    C  \\nATOM    113  O   LEU A  15     -15.035  -8.658 -26.717  1.00 61.67      A    O  \\nATOM    114  CB  LEU A  15     -17.169 -10.968 -26.133  1.00 59.47      A    C  \\nATOM    115  CG  LEU A  15     -18.601 -11.147 -25.625  1.00 55.97      A    C  \\nATOM    116  CD1 LEU A  15     -19.118  -9.875 -24.979  1.00 56.08      A    C  \\nATOM    117  CD2 LEU A  15     -18.691 -12.310 -24.661  1.00 55.18      A    C  \\nATOM    118  N   SER A  16     -14.742 -10.348 -28.128  1.00 65.39      A    N  \\nATOM    119  CA  SER A  16     -13.328 -10.050 -28.349  1.00 67.36      A    C  \\nATOM    120  C   SER A  16     -13.174  -8.761 -29.164  1.00 66.22      A    C  \\nATOM    121  O   SER A  16     -12.247  -7.960 -28.917  1.00 65.25      A    O  \\nATOM    122  CB  SER A  16     -12.591 -11.191 -29.028  1.00 65.27      A    C  \\nATOM    123  OG  SER A  16     -13.082 -11.393 -30.332  1.00 58.30      A    O  \\nATOM    124  N   ARG A  17     -14.038  -8.562 -30.098  1.00 60.22      A    N  \\nATOM    125  CA  ARG A  17     -14.012  -7.321 -30.900  1.00 63.48      A    C  \\nATOM    126  C   ARG A  17     -14.501  -6.104 -30.094  1.00 62.80      A    C  \\nATOM    127  O   ARG A  17     -13.988  -4.997 -30.292  1.00 61.27      A    O  \\nATOM    128  CB  ARG A  17     -14.868  -7.459 -32.160  1.00 60.92      A    C  \\nATOM    129  CG  ARG A  17     -14.315  -8.442 -33.188  1.00 57.06      A    C  \\nATOM    130  CD  ARG A  17     -15.243  -8.591 -34.374  1.00 53.33      A    C  \\nATOM    131  NE  ARG A  17     -14.731  -9.524 -35.353  1.00 47.78      A    N  \\nATOM    132  CZ  ARG A  17     -15.378  -9.901 -36.456  1.00 46.89      A    C  \\nATOM    133  NH1 ARG A  17     -16.575  -9.388 -36.716  1.00 40.05      A    N1+\\nATOM    134  NH2 ARG A  17     -14.821 -10.765 -37.285  1.00 38.60      A    N  \\nATOM    135  N   SER A  18     -15.435  -6.348 -29.192  1.00 54.68      A    N  \\nATOM    136  CA  SER A  18     -15.922  -5.246 -28.376  1.00 58.21      A    C  \\nATOM    137  C   SER A  18     -14.903  -4.869 -27.295  1.00 58.35      A    C  \\nATOM    138  O   SER A  18     -14.759  -3.697 -26.936  1.00 58.57      A    O  \\nATOM    139  CB  SER A  18     -17.265  -5.598 -27.729  1.00 55.43      A    C  \\nATOM    140  OG  SER A  18     -17.105  -6.662 -26.795  1.00 53.12      A    O  \\nATOM    141  N   LYS A  19     -14.171  -5.859 -26.771  1.00 54.50      A    N  \\nATOM    142  CA  LYS A  19     -13.099  -5.569 -25.816  1.00 56.57      A    C  \\nATOM    143  C   LYS A  19     -11.925  -4.848 -26.495  1.00 57.40      A    C  \\nATOM    144  O   LYS A  19     -11.285  -3.989 -25.882  1.00 57.19      A    O  \\nATOM    145  CB  LYS A  19     -12.588  -6.842 -25.175  1.00 54.13      A    C  \\nATOM    146  CG  LYS A  19     -13.531  -7.439 -24.152  1.00 51.30      A    C  \\nATOM    147  CD  LYS A  19     -12.980  -8.705 -23.594  1.00 47.99      A    C  \\nATOM    148  CE  LYS A  19     -13.938  -9.335 -22.588  1.00 43.17      A    C  \\nATOM    149  NZ  LYS A  19     -13.370 -10.562 -21.992  1.00 39.47      A    N1+\\nATOM    150  N   SER A  20     -11.643  -5.186 -27.726  1.00 56.66      A    N  \\nATOM    151  CA  SER A  20     -10.599  -4.467 -28.475  1.00 60.56      A    C  \\nATOM    152  C   SER A  20     -11.043  -3.061 -28.827  1.00 60.93      A    C  \\nATOM    153  O   SER A  20     -10.242  -2.150 -28.905  1.00 60.99      A    O  \\nATOM    154  CB  SER A  20     -10.242  -5.231 -29.729  1.00 58.78      A    C  \\nATOM    155  OG  SER A  20     -11.334  -5.289 -30.627  1.00 53.00      A    O  \\nATOM    156  N   SER A  21     -12.350  -2.876 -29.013  1.00 56.39      A    N  \\nATOM    157  CA  SER A  21     -12.846  -1.530 -29.301  1.00 60.18      A    C  \\nATOM    158  C   SER A  21     -12.861  -0.651 -28.042  1.00 60.44      A    C  \\nATOM    159  O   SER A  21     -12.642   0.548 -28.106  1.00 60.14      A    O  \\nATOM    160  CB  SER A  21     -14.250  -1.610 -29.910  1.00 58.28      A    C  \\nATOM    161  OG  SER A  21     -15.168  -2.108 -28.966  1.00 51.91      A    O  \\nATOM    162  N   LEU A  22     -13.120  -1.282 -26.899  1.00 57.13      A    N  \\nATOM    163  CA  LEU A  22     -13.090  -0.536 -25.635  1.00 60.02      A    C  \\nATOM    164  C   LEU A  22     -11.673  -0.156 -25.240  1.00 61.19      A    C  \\nATOM    165  O   LEU A  22     -11.443   0.905 -24.667  1.00 61.22      A    O  \\nATOM    166  CB  LEU A  22     -13.759  -1.334 -24.519  1.00 56.77      A    C  \\nATOM    167  CG  LEU A  22     -15.261  -1.492 -24.581  1.00 53.85      A    C  \\nATOM    168  CD1 LEU A  22     -15.936  -0.127 -24.470  1.00 53.60      A    C  \\nATOM    169  CD2 LEU A  22     -15.740  -2.418 -23.485  1.00 53.01      A    C  \\nATOM    170  N   THR A  23     -10.743  -1.045 -25.523  1.00 57.36      A    N  \\nATOM    171  CA  THR A  23      -9.328  -0.711 -25.272  1.00 60.66      A    C  \\nATOM    172  C   THR A  23      -8.827   0.384 -26.208  1.00 61.17      A    C  \\nATOM    173  O   THR A  23      -8.012   1.193 -25.823  1.00 61.07      A    O  \\nATOM    174  CB  THR A  23      -8.433  -1.940 -25.382  1.00 58.52      A    C  \\nATOM    175  CG2 THR A  23      -8.818  -2.960 -24.320  1.00 55.10      A    C  \\nATOM    176  OG1 THR A  23      -8.564  -2.536 -26.680  1.00 57.50      A    O  \\nATOM    177  N   ASN A  24      -9.345   0.389 -27.415  1.00 62.80      A    N  \\nATOM    178  CA  ASN A  24      -8.970   1.445 -28.350  1.00 63.27      A    C  \\nATOM    179  C   ASN A  24      -9.598   2.782 -27.973  1.00 63.00      A    C  \\nATOM    180  O   ASN A  24      -9.004   3.826 -28.164  1.00 63.35      A    O  \\nATOM    181  CB  ASN A  24      -9.389   1.060 -29.772  1.00 61.61      A    C  \\nATOM    182  CG  ASN A  24      -8.549  -0.063 -30.338  1.00 59.29      A    C  \\nATOM    183  ND2 ASN A  24      -9.197  -0.965 -31.048  1.00 51.62      A    N  \\nATOM    184  OD1 ASN A  24      -7.347  -0.111 -30.114  1.00 51.44      A    O  \\nATOM    185  N   LEU A  25     -10.813   2.721 -27.417  1.00 59.28      A    N  \\nATOM    186  CA  LEU A  25     -11.445   3.955 -26.946  1.00 61.39      A    C  \\nATOM    187  C   LEU A  25     -10.787   4.471 -25.677  1.00 62.03      A    C  \\nATOM    188  O   LEU A  25     -10.748   5.674 -25.424  1.00 61.73      A    O  \\nATOM    189  CB  LEU A  25     -12.915   3.723 -26.709  1.00 58.92      A    C  \\nATOM    190  CG  LEU A  25     -13.792   3.511 -27.939  1.00 55.14      A    C  \\nATOM    191  CD1 LEU A  25     -13.771   4.747 -28.830  1.00 55.42      A    C  \\nATOM    192  CD2 LEU A  25     -15.204   3.167 -27.536  1.00 54.56      A    C  \\nATOM    193  N   PHE A  26     -10.279   3.543 -24.861  1.00 53.08      A    N  \\nATOM    194  CA  PHE A  26      -9.603   3.937 -23.661  1.00 56.50      A    C  \\nATOM    195  C   PHE A  26      -8.226   4.543 -23.979  1.00 57.05      A    C  \\nATOM    196  O   PHE A  26      -7.760   5.452 -23.271  1.00 57.51      A    O  \\nATOM    197  CB  PHE A  26      -9.487   2.758 -22.711  1.00 53.93      A    C  \\nATOM    198  CG  PHE A  26      -8.852   3.071 -21.388  1.00 51.33      A    C  \\nATOM    199  CD1 PHE A  26      -9.525   3.775 -20.430  1.00 48.13      A    C  \\nATOM    200  CD2 PHE A  26      -7.565   2.645 -21.124  1.00 47.08      A    C  \\nATOM    201  CE1 PHE A  26      -8.955   4.074 -19.221  1.00 43.65      A    C  \\nATOM    202  CE2 PHE A  26      -6.988   2.927 -19.888  1.00 41.71      A    C  \\nATOM    203  CZ  PHE A  26      -7.675   3.651 -18.954  1.00 42.22      A    C  \\nATOM    204  N   LYS A  27      -7.588   4.046 -25.022  1.00 60.81      A    N  \\nATOM    205  CA  LYS A  27      -6.315   4.635 -25.461  1.00 63.46      A    C  \\nATOM    206  C   LYS A  27      -6.531   5.999 -26.102  1.00 63.22      A    C  \\nATOM    207  O   LYS A  27      -5.687   6.886 -25.963  1.00 62.78      A    O  \\nATOM    208  CB  LYS A  27      -5.610   3.699 -26.421  1.00 61.70      A    C  \\nATOM    209  CG  LYS A  27      -5.064   2.437 -25.791  1.00 59.74      A    C  \\nATOM    210  CD  LYS A  27      -4.427   1.544 -26.826  1.00 56.01      A    C  \\nATOM    211  CE  LYS A  27      -3.883   0.294 -26.199  1.00 52.11      A    C  \\nATOM    212  NZ  LYS A  27      -3.238  -0.609 -27.194  1.00 45.89      A    N1+\\nATOM    213  N   LYS A  28      -7.639   6.170 -26.781  1.00 56.47      A    N  \\nATOM    214  CA  LYS A  28      -7.963   7.481 -27.336  1.00 59.14      A    C  \\nATOM    215  C   LYS A  28      -8.265   8.502 -26.276  1.00 58.37      A    C  \\nATOM    216  O   LYS A  28      -7.980   9.693 -26.439  1.00 57.58      A    O  \\nATOM    217  CB  LYS A  28      -9.142   7.377 -28.310  1.00 56.81      A    C  \\nATOM    218  CG  LYS A  28      -8.836   6.666 -29.607  1.00 53.49      A    C  \\nATOM    219  CD  LYS A  28     -10.053   6.560 -30.481  1.00 50.79      A    C  \\nATOM    220  CE  LYS A  28      -9.733   5.821 -31.766  1.00 44.60      A    C  \\nATOM    221  NZ  LYS A  28     -10.920   5.705 -32.623  1.00 40.56      A    N1+\\nATOM    222  N   ASN A  29      -8.833   8.026 -25.162  1.00 53.60      A    N  \\nATOM    223  CA  ASN A  29      -9.089   8.951 -24.056  1.00 55.73      A    C  \\nATOM    224  C   ASN A  29      -7.805   9.347 -23.344  1.00 55.81      A    C  \\nATOM    225  O   ASN A  29      -7.670  10.459 -22.854  1.00 55.66      A    O  \\nATOM    226  CB  ASN A  29     -10.061   8.321 -23.058  1.00 52.64      A    C  \\nATOM    227  CG  ASN A  29     -11.469   8.202 -23.614  1.00 51.01      A    C  \\nATOM    228  ND2 ASN A  29     -12.097   7.085 -23.311  1.00 45.64      A    N  \\nATOM    229  OD1 ASN A  29     -11.962   9.072 -24.303  1.00 43.88      A    O  \\nATOM    230  N   LYS A  30      -6.855   8.430 -23.311  1.00 55.01      A    N  \\nATOM    231  CA  LYS A  30      -5.555   8.763 -22.743  1.00 57.35      A    C  \\nATOM    232  C   LYS A  30      -4.743   9.689 -23.633  1.00 56.95      A    C  \\nATOM    233  O   LYS A  30      -3.960  10.518 -23.143  1.00 56.26      A    O  \\nATOM    234  CB  LYS A  30      -4.759   7.493 -22.453  1.00 54.81      A    C  \\nATOM    235  CG  LYS A  30      -5.304   6.676 -21.308  1.00 51.51      A    C  \\nATOM    236  CD  LYS A  30      -4.441   5.464 -21.062  1.00 48.54      A    C  \\nATOM    237  CE  LYS A  30      -4.934   4.611 -19.897  1.00 42.06      A    C  \\nATOM    238  NZ  LYS A  30      -4.057   3.460 -19.617  1.00 39.20      A    N1+\\nATOM    239  N   ARG A  31      -4.933   9.592 -24.921  1.00 57.98      A    N  \\nATOM    240  CA  ARG A  31      -4.260  10.482 -25.843  1.00 60.57      A    C  \\nATOM    241  C   ARG A  31      -4.841  11.900 -25.791  1.00 60.12      A    C  \\nATOM    242  O   ARG A  31      -4.122  12.872 -25.975  1.00 58.98      A    O  \\nATOM    243  CB  ARG A  31      -4.356   9.941 -27.279  1.00 57.44      A    C  \\nATOM    244  CG  ARG A  31      -3.575   8.689 -27.546  1.00 54.47      A    C  \\nATOM    245  CD  ARG A  31      -3.782   8.207 -28.975  1.00 50.81      A    C  \\nATOM    246  NE  ARG A  31      -3.063   6.961 -29.213  1.00 44.43      A    N  \\nATOM    247  CZ  ARG A  31      -3.151   6.265 -30.359  1.00 43.97      A    C  \\nATOM    248  NH1 ARG A  31      -3.890   6.718 -31.343  1.00 36.29      A    N1+\\nATOM    249  NH2 ARG A  31      -2.489   5.136 -30.499  1.00 34.48      A    N  \\nATOM    250  N   LYS A  32      -6.151  11.970 -25.515  1.00 54.18      A    N  \\nATOM    251  CA  LYS A  32      -6.777  13.298 -25.395  1.00 56.12      A    C  \\nATOM    252  C   LYS A  32      -6.354  14.027 -24.122  1.00 55.12      A    C  \\nATOM    253  O   LYS A  32      -6.195  15.239 -24.142  1.00 55.10      A    O  \\nATOM    254  CB  LYS A  32      -8.295  13.163 -25.449  1.00 54.02      A    C  \\nATOM    255  CG  LYS A  32      -8.833  12.751 -26.813  1.00 52.32      A    C  \\nATOM    256  CD  LYS A  32     -10.341  12.582 -26.794  1.00 49.32      A    C  \\nATOM    257  CE  LYS A  32     -10.845  12.203 -28.154  1.00 44.14      A    C  \\nATOM    258  NZ  LYS A  32     -12.311  12.015 -28.185  1.00 40.04      A    N1+\\nATOM    259  N   HIS A  33      -6.176  13.262 -23.052  1.00 51.71      A    N  \\nATOM    260  CA  HIS A  33      -5.706  13.862 -21.825  1.00 53.98      A    C  \\nATOM    261  C   HIS A  33      -4.243  14.247 -21.906  1.00 52.97      A    C  \\nATOM    262  O   HIS A  33      -3.784  15.158 -21.238  1.00 52.35      A    O  \\nATOM    263  CB  HIS A  33      -5.951  12.964 -20.624  1.00 51.18      A    C  \\nATOM    264  CG  HIS A  33      -7.385  12.814 -20.247  1.00 50.14      A    C  \\nATOM    265  CD2 HIS A  33      -8.244  11.751 -20.330  1.00 46.85      A    C  \\nATOM    266  ND1 HIS A  33      -8.120  13.862 -19.720  1.00 43.45      A    N  \\nATOM    267  CE1 HIS A  33      -9.344  13.440 -19.471  1.00 43.10      A    C  \\nATOM    268  NE2 HIS A  33      -9.453  12.180 -19.818  1.00 40.39      A    N  \\nATOM    269  N   ALA A  34      -3.490  13.524 -22.715  1.00 51.12      A    N  \\nATOM    270  CA  ALA A  34      -2.072  13.824 -22.893  1.00 54.84      A    C  \\nATOM    271  C   ALA A  34      -1.863  15.063 -23.750  1.00 54.40      A    C  \\nATOM    272  O   ALA A  34      -0.959  15.846 -23.505  1.00 53.71      A    O  \\nATOM    273  CB  ALA A  34      -1.349  12.619 -23.503  1.00 52.74      A    C  \\nATOM    274  N   ARG A  35      -2.710  15.236 -24.751  1.00 49.90      A    N  \\nATOM    275  CA  ARG A  35      -2.607  16.406 -25.612  1.00 54.04      A    C  \\nATOM    276  C   ARG A  35      -3.064  17.694 -24.915  1.00 52.99      A    C  \\nATOM    277  O   ARG A  35      -2.541  18.756 -25.226  1.00 50.66      A    O  \\nATOM    278  CB  ARG A  35      -3.409  16.198 -26.887  1.00 50.29      A    C  \\nATOM    279  CG  ARG A  35      -2.874  15.159 -27.813  1.00 48.90      A    C  \\nATOM    280  CD  ARG A  35      -3.781  14.904 -29.007  1.00 43.94      A    C  \\nATOM    281  NE  ARG A  35      -3.279  13.885 -29.891  1.00 36.74      A    N  \\nATOM    282  CZ  ARG A  35      -3.906  13.478 -30.992  1.00 36.62      A    C  \\nATOM    283  NH1 ARG A  35      -5.060  14.022 -31.353  1.00 31.46      A    N1+\\nATOM    284  NH2 ARG A  35      -3.362  12.528 -31.758  1.00 29.02      A    N  \\nATOM    285  N   ASP A  36      -3.980  17.557 -23.998  1.00 46.65      A    N  \\nATOM    286  CA  ASP A  36      -4.424  18.736 -23.255  1.00 49.47      A    C  \\nATOM    287  C   ASP A  36      -3.387  19.172 -22.219  1.00 49.26      A    C  \\nATOM    288  O   ASP A  36      -3.234  20.343 -21.942  1.00 47.02      A    O  \\nATOM    289  CB  ASP A  36      -5.744  18.459 -22.570  1.00 46.18      A    C  \\nATOM    290  CG  ASP A  36      -6.907  18.324 -23.555  1.00 44.97      A    C  \\nATOM    291  OD1 ASP A  36      -6.819  18.898 -24.661  1.00 38.68      A    O  \\nATOM    292  OD2 ASP A  36      -7.890  17.635 -23.210  1.00 39.62      A    O1-\\nATOM    293  N   ASP A  37      -2.671  18.201 -21.656  1.00 46.49      A    N  \\nATOM    294  CA  ASP A  37      -1.644  18.530 -20.678  1.00 49.31      A    C  \\nATOM    295  C   ASP A  37      -0.361  19.062 -21.343  1.00 49.22      A    C  \\nATOM    296  O   ASP A  37       0.418  19.741 -20.696  1.00 47.13      A    O  \\nATOM    297  CB  ASP A  37      -1.313  17.337 -19.800  1.00 45.92      A    C  \\nATOM    298  CG  ASP A  37      -2.463  16.952 -18.877  1.00 44.38      A    C  \\nATOM    299  OD1 ASP A  37      -3.218  17.839 -18.447  1.00 37.22      A    O  \\nATOM    300  OD2 ASP A  37      -2.604  15.728 -18.580  1.00 36.99      A    O1-\\nATOM    301  N   ASP A  38      -0.169  18.741 -22.583  1.00 45.57      A    N  \\nATOM    302  CA  ASP A  38       1.015  19.202 -23.307  1.00 47.82      A    C  \\nATOM    303  C   ASP A  38       0.842  20.610 -23.861  1.00 47.90      A    C  \\nATOM    304  O   ASP A  38       1.813  21.218 -24.283  1.00 45.63      A    O  \\nATOM    305  CB  ASP A  38       1.373  18.233 -24.436  1.00 44.16      A    C  \\nATOM    306  CG  ASP A  38       1.850  16.885 -23.925  1.00 43.87      A    C  \\nATOM    307  OD1 ASP A  38       2.450  16.845 -22.852  1.00 36.34      A    O  \\nATOM    308  OD2 ASP A  38       1.596  15.868 -24.628  1.00 36.28      A    O1-\\nATOM    309  N   GLU A  39      -0.386  21.074 -23.892  1.00 47.71      A    N  \\nATOM    310  CA  GLU A  39      -0.644  22.415 -24.386  1.00 50.73      A    C  \\nATOM    311  C   GLU A  39      -0.261  23.448 -23.341  1.00 50.51      A    C  \\nATOM    312  O   GLU A  39      -0.711  23.421 -22.201  1.00 48.28      A    O  \\nATOM    313  CB  GLU A  39      -2.101  22.601 -24.770  1.00 47.56      A    C  \\nATOM    314  CG  GLU A  39      -2.526  21.768 -25.976  1.00 46.71      A    C  \\nATOM    315  CD  GLU A  39      -3.998  21.949 -26.324  1.00 43.31      A    C  \\nATOM    316  OE1 GLU A  39      -4.694  22.641 -25.556  1.00 39.43      A    O  \\nATOM    317  OE2 GLU A  39      -4.441  21.383 -27.348  1.00 36.98      A    O1-\\nATOM    318  N   ASP A  40       0.611  24.370 -23.741  1.00 46.59      A    N  \\nATOM    319  CA  ASP A  40       1.081  25.414 -22.861  1.00 47.74      A    C  \\nATOM    320  C   ASP A  40       0.245  26.660 -23.078  1.00 47.37      A    C  \\nATOM    321  O   ASP A  40       0.047  27.119 -24.197  1.00 46.52      A    O  \\nATOM    322  CB  ASP A  40       2.571  25.712 -23.095  1.00 44.64      A    C  \\nATOM    323  CG  ASP A  40       3.463  24.554 -22.696  1.00 43.33      A    C  \\nATOM    324  OD1 ASP A  40       3.076  23.775 -21.776  1.00 37.62      A    O  \\nATOM    325  OD2 ASP A  40       4.570  24.400 -23.287  1.00 37.54      A    O1-\\nATOM    326  N   ASP A  41      -0.213  27.207 -21.979  1.00 45.41      A    N  \\nATOM    327  CA  ASP A  41      -1.025  28.424 -22.011  1.00 47.31      A    C  \\nATOM    328  C   ASP A  41      -0.096  29.616 -21.764  1.00 47.10      A    C  \\nATOM    329  O   ASP A  41       0.506  29.744 -20.704  1.00 45.85      A    O  \\nATOM    330  CB  ASP A  41      -2.141  28.390 -20.999  1.00 44.01      A    C  \\nATOM    331  CG  ASP A  41      -3.203  27.352 -21.348  1.00 42.98      A    C  \\nATOM    332  OD1 ASP A  41      -3.375  27.068 -22.553  1.00 38.02      A    O  \\nATOM    333  OD2 ASP A  41      -3.853  26.838 -20.425  1.00 37.73      A    O1-\\nATOM    334  N   GLU A  42      -0.008  30.472 -22.743  1.00 40.91      A    N  \\nATOM    335  CA  GLU A  42       0.855  31.624 -22.670  1.00 43.44      A    C  \\nATOM    336  C   GLU A  42       0.088  32.771 -22.037  1.00 42.98      A    C  \\nATOM    337  O   GLU A  42      -0.971  33.190 -22.516  1.00 41.44      A    O  \\nATOM    338  CB  GLU A  42       1.397  32.039 -24.025  1.00 40.30      A    C  \\nATOM    339  CG  GLU A  42       2.334  31.020 -24.644  1.00 39.40      A    C  \\nATOM    340  CD  GLU A  42       2.880  31.459 -25.999  1.00 37.72      A    C  \\nATOM    341  OE1 GLU A  42       2.474  32.542 -26.486  1.00 34.74      A    O  \\nATOM    342  OE2 GLU A  42       3.723  30.755 -26.577  1.00 32.40      A    O1-\\nATOM    343  N   ASP A  43       0.629  33.262 -20.941  1.00 47.07      A    N  \\nATOM    344  CA  ASP A  43       0.028  34.389 -20.233  1.00 48.99      A    C  \\nATOM    345  C   ASP A  43       0.579  35.712 -20.775  1.00 48.10      A    C  \\nATOM    346  O   ASP A  43       1.315  35.754 -21.771  1.00 46.87      A    O  \\nATOM    347  CB  ASP A  43       0.279  34.258 -18.727  1.00 46.07      A    C  \\nATOM    348  CG  ASP A  43      -0.480  33.107 -18.103  1.00 44.83      A    C  \\nATOM    349  OD1 ASP A  43      -1.574  32.769 -18.607  1.00 39.17      A    O  \\nATOM    350  OD2 ASP A  43       0.005  32.548 -17.125  1.00 39.74      A    O1-\\nATOM    351  N   GLU A  44       0.185  36.772 -20.125  1.00 45.47      A    N  \\nATOM    352  CA  GLU A  44       0.656  38.090 -20.548  1.00 48.32      A    C  \\nATOM    353  C   GLU A  44       2.142  38.295 -20.256  1.00 47.79      A    C  \\nATOM    354  O   GLU A  44       2.839  39.015 -20.974  1.00 45.69      A    O  \\nATOM    355  CB  GLU A  44      -0.158  39.223 -19.892  1.00 44.96      A    C  \\nATOM    356  CG  GLU A  44      -1.612  39.249 -20.322  1.00 44.05      A    C  \\nATOM    357  CD  GLU A  44      -2.393  40.373 -19.620  1.00 42.26      A    C  \\nATOM    358  OE1 GLU A  44      -1.805  41.029 -18.732  1.00 38.80      A    O  \\nATOM    359  OE2 GLU A  44      -3.582  40.547 -19.969  1.00 36.49      A    O1-\\nATOM    360  N   ASN A  45       2.607  37.670 -19.199  1.00 45.81      A    N  \\nATOM    361  CA  ASN A  45       3.983  37.796 -18.795  1.00 47.51      A    C  \\nATOM    362  C   ASN A  45       4.874  36.719 -19.407  1.00 46.77      A    C  \\nATOM    363  O   ASN A  45       6.055  36.698 -19.140  1.00 45.46      A    O  \\nATOM    364  CB  ASN A  45       4.113  37.783 -17.279  1.00 44.53      A    C  \\nATOM    365  CG  ASN A  45       3.475  38.989 -16.636  1.00 43.03      A    C  \\nATOM    366  ND2 ASN A  45       2.810  38.782 -15.519  1.00 40.10      A    N  \\nATOM    367  OD1 ASN A  45       3.567  40.090 -17.158  1.00 37.33      A    O  \\nATOM    368  N   GLY A  46       4.278  35.816 -20.186  1.00 42.44      A    N  \\nATOM    369  CA  GLY A  46       5.045  34.746 -20.787  1.00 43.17      A    C  \\nATOM    370  C   GLY A  46       5.302  33.576 -19.843  1.00 44.40      A    C  \\nATOM    371  O   GLY A  46       6.242  32.791 -20.068  1.00 43.72      A    O  \\nATOM    372  N   ILE A  47       4.517  33.449 -18.793  1.00 44.58      A    N  \\nATOM    373  CA  ILE A  47       4.676  32.389 -17.813  1.00 47.13      A    C  \\nATOM    374  C   ILE A  47       3.914  31.186 -18.319  1.00 47.36      A    C  \\nATOM    375  O   ILE A  47       2.673  31.199 -18.411  1.00 46.29      A    O  \\nATOM    376  CB  ILE A  47       4.198  32.817 -16.425  1.00 44.74      A    C  \\nATOM    377  CG1 ILE A  47       5.022  33.988 -15.909  1.00 46.53      A    C  \\nATOM    378  CG2 ILE A  47       4.290  31.647 -15.455  1.00 43.63      A    C  \\nATOM    379  CD1 ILE A  47       4.530  34.606 -14.647  1.00 42.22      A    C  \\nATOM    380  N   SER A  48       4.633  30.128 -18.629  1.00 44.45      A    N  \\nATOM    381  CA  SER A  48       3.997  28.892 -19.094  1.00 48.79      A    C  \\nATOM    382  C   SER A  48       3.685  28.048 -17.881  1.00 48.22      A    C  \\nATOM    383  O   SER A  48       4.491  27.907 -16.955  1.00 47.46      A    O  \\nATOM    384  CB  SER A  48       4.891  28.146 -20.055  1.00 46.14      A    C  \\nATOM    385  OG  SER A  48       6.086  27.722 -19.420  1.00 43.11      A    O  \\nATOM    386  N   SER A  49       2.524  27.441 -17.878  1.00 44.55      A    N  \\nATOM    387  CA  SER A  49       2.071  26.582 -16.807  1.00 47.59      A    C  \\nATOM    388  C   SER A  49       2.127  25.143 -17.246  1.00 47.00      A    C  \\nATOM    389  O   SER A  49       1.558  24.757 -18.267  1.00 45.65      A    O  \\nATOM    390  CB  SER A  49       0.658  26.973 -16.357  1.00 44.76      A    C  \\nATOM    391  OG  SER A  49      -0.279  26.691 -17.396  1.00 40.63      A    O  \\nATOM    392  N   ASN A  50       2.820  24.345 -16.467  1.00 44.07      A    N  \\nATOM    393  CA  ASN A  50       2.923  22.931 -16.745  1.00 46.40      A    C  \\nATOM    394  C   ASN A  50       3.070  22.172 -15.433  1.00 46.30      A    C  \\nATOM    395  O   ASN A  50       4.018  22.392 -14.679  1.00 45.12      A    O  \\nATOM    396  CB  ASN A  50       4.108  22.613 -17.675  1.00 43.40      A    C  \\nATOM    397  CG  ASN A  50       4.048  21.180 -18.212  1.00 41.99      A    C  \\nATOM    398  ND2 ASN A  50       5.034  20.800 -19.021  1.00 37.75      A    N  \\nATOM    399  OD1 ASN A  50       3.125  20.421 -17.903  1.00 36.23      A    O  \\nATOM    400  N   ASP A  51       2.124  21.308 -15.164  1.00 45.55      A    N  \\nATOM    401  CA  ASP A  51       2.151  20.546 -13.913  1.00 47.56      A    C  \\nATOM    402  C   ASP A  51       3.143  19.409 -13.952  1.00 47.88      A    C  \\nATOM    403  O   ASP A  51       3.724  19.053 -12.932  1.00 46.90      A    O  \\nATOM    404  CB  ASP A  51       0.740  20.000 -13.618  1.00 43.81      A    C  \\nATOM    405  CG  ASP A  51      -0.249  21.101 -13.301  1.00 42.70      A    C  \\nATOM    406  OD1 ASP A  51       0.158  22.151 -12.757  1.00 37.73      A    O  \\nATOM    407  OD2 ASP A  51      -1.445  20.907 -13.633  1.00 37.69      A    O1-\\nATOM    408  N   LYS A  52       3.336  18.824 -15.130  1.00 43.86      A    N  \\nATOM    409  CA  LYS A  52       4.259  17.730 -15.244  1.00 46.14      A    C  \\nATOM    410  C   LYS A  52       5.714  18.180 -15.143  1.00 45.99      A    C  \\nATOM    411  O   LYS A  52       6.566  17.440 -14.647  1.00 45.25      A    O  \\nATOM    412  CB  LYS A  52       4.048  16.976 -16.568  1.00 43.40      A    C  \\nATOM    413  CG  LYS A  52       2.700  16.287 -16.665  1.00 42.81      A    C  \\nATOM    414  CD  LYS A  52       2.579  15.545 -17.972  1.00 40.30      A    C  \\nATOM    415  CE  LYS A  52       1.257  14.852 -18.099  1.00 35.00      A    C  \\nATOM    416  NZ  LYS A  52       1.141  14.129 -19.390  1.00 31.55      A    N1+\\nATOM    417  N   SER A  53       5.977  19.387 -15.604  1.00 45.28      A    N  \\nATOM    418  CA  SER A  53       7.329  19.928 -15.495  1.00 48.37      A    C  \\nATOM    419  C   SER A  53       7.651  20.359 -14.068  1.00 48.04      A    C  \\nATOM    420  O   SER A  53       8.814  20.307 -13.651  1.00 47.20      A    O  \\nATOM    421  CB  SER A  53       7.515  21.098 -16.440  1.00 45.82      A    C  \\nATOM    422  OG  SER A  53       6.682  22.163 -16.066  1.00 42.03      A    O  \\nATOM    423  N   LEU A  54       6.630  20.758 -13.315  1.00 44.05      A    N  \\nATOM    424  CA  LEU A  54       6.844  21.127 -11.927  1.00 47.06      A    C  \\nATOM    425  C   LEU A  54       7.105  19.921 -11.035  1.00 47.21      A    C  \\nATOM    426  O   LEU A  54       7.880  20.011 -10.085  1.00 46.14      A    O  \\nATOM    427  CB  LEU A  54       5.641  21.901 -11.388  1.00 44.08      A    C  \\nATOM    428  CG  LEU A  54       5.398  23.279 -11.994  1.00 42.46      A    C  \\nATOM    429  CD1 LEU A  54       6.550  24.206 -11.668  1.00 42.05      A    C  \\nATOM    430  CD2 LEU A  54       4.107  23.845 -11.491  1.00 41.49      A    C  \\nATOM    431  N   THR A  55       6.481  18.806 -11.373  1.00 45.15      A    N  \\nATOM    432  CA  THR A  55       6.722  17.598 -10.589  1.00 45.85      A    C  \\nATOM    433  C   THR A  55       8.118  17.011 -10.856  1.00 45.54      A    C  \\nATOM    434  O   THR A  55       8.719  16.435  -9.951  1.00 44.61      A    O  \\nATOM    435  CB  THR A  55       5.658  16.555 -10.885  1.00 42.76      A    C  \\nATOM    436  CG2 THR A  55       4.263  17.073 -10.522  1.00 39.92      A    C  \\nATOM    437  OG1 THR A  55       5.677  16.192 -12.247  1.00 41.52      A    O  \\nATOM    438  N   LYS A  56       8.588  17.145 -12.073  1.00 44.73      A    N  \\nATOM    439  CA  LYS A  56       9.931  16.685 -12.396  1.00 47.43      A    C  \\nATOM    440  C   LYS A  56      11.018  17.513 -11.741  1.00 46.72      A    C  \\nATOM    441  O   LYS A  56      12.068  16.996 -11.362  1.00 46.04      A    O  \\nATOM    442  CB  LYS A  56      10.142  16.657 -13.911  1.00 45.65      A    C  \\nATOM    443  CG  LYS A  56       9.318  15.609 -14.626  1.00 44.07      A    C  \\nATOM    444  CD  LYS A  56       9.561  15.657 -16.104  1.00 41.72      A    C  \\nATOM    445  CE  LYS A  56       8.742  14.608 -16.846  1.00 36.03      A    C  \\nATOM    446  NZ  LYS A  56       9.003  14.613 -18.306  1.00 34.01      A    N1+\\nATOM    447  N   ILE A  57      10.743  18.808 -11.602  1.00 39.53      A    N  \\nATOM    448  CA  ILE A  57      11.705  19.689 -10.921  1.00 41.97      A    C  \\nATOM    449  C   ILE A  57      11.720  19.398  -9.427  1.00 42.29      A    C  \\nATOM    450  O   ILE A  57      12.787  19.383  -8.810  1.00 41.09      A    O  \\nATOM    451  CB  ILE A  57      11.386  21.163 -11.227  1.00 39.09      A    C  \\nATOM    452  CG1 ILE A  57      11.601  21.452 -12.699  1.00 42.32      A    C  \\nATOM    453  CG2 ILE A  57      12.255  22.089 -10.389  1.00 39.71      A    C  \\nATOM    454  CD1 ILE A  57      11.113  22.801 -13.159  1.00 38.47      A    C  \\nATOM    455  N   ASP A  58      10.575  19.129  -8.867  1.00 40.54      A    N  \\nATOM    456  CA  ASP A  58      10.526  18.843  -7.435  1.00 43.26      A    C  \\nATOM    457  C   ASP A  58      11.162  17.508  -7.092  1.00 43.47      A    C  \\nATOM    458  O   ASP A  58      11.726  17.346  -6.017  1.00 41.78      A    O  \\nATOM    459  CB  ASP A  58       9.073  18.867  -6.945  1.00 40.03      A    C  \\nATOM    460  CG  ASP A  58       8.459  20.250  -6.965  1.00 38.96      A    C  \\nATOM    461  OD1 ASP A  58       9.201  21.228  -6.748  1.00 33.23      A    O  \\nATOM    462  OD2 ASP A  58       7.237  20.367  -7.178  1.00 32.13      A    O1-\\nATOM    463  N   ASN A  59      11.052  16.555  -8.001  1.00 44.95      A    N  \\nATOM    464  CA  ASN A  59      11.675  15.264  -7.776  1.00 47.05      A    C  \\nATOM    465  C   ASN A  59      13.192  15.355  -7.918  1.00 47.09      A    C  \\nATOM    466  O   ASN A  59      13.929  14.612  -7.264  1.00 45.06      A    O  \\nATOM    467  CB  ASN A  59      11.122  14.199  -8.716  1.00 43.24      A    C  \\nATOM    468  CG  ASN A  59       9.688  13.814  -8.405  1.00 42.67      A    C  \\nATOM    469  ND2 ASN A  59       8.885  13.579  -9.423  1.00 38.09      A    N  \\nATOM    470  OD1 ASN A  59       9.306  13.749  -7.234  1.00 36.98      A    O  \\nATOM    471  N   GLU A  60      13.666  16.242  -8.761  1.00 45.70      A    N  \\nATOM    472  CA  GLU A  60      15.086  16.447  -8.903  1.00 49.77      A    C  \\nATOM    473  C   GLU A  60      15.684  17.227  -7.736  1.00 49.36      A    C  \\nATOM    474  O   GLU A  60      16.852  17.076  -7.440  1.00 47.57      A    O  \\nATOM    475  CB  GLU A  60      15.396  17.144 -10.219  1.00 46.61      A    C  \\nATOM    476  CG  GLU A  60      15.113  16.296 -11.464  1.00 45.32      A    C  \\nATOM    477  CD  GLU A  60      15.382  17.044 -12.768  1.00 41.65      A    C  \\nATOM    478  OE1 GLU A  60      15.684  18.250 -12.718  1.00 38.82      A    O  \\nATOM    479  OE2 GLU A  60      15.277  16.402 -13.841  1.00 36.13      A    O1-\\nATOM    480  N   ASP A  61      14.861  18.054  -7.111  1.00 43.97      A    N  \\nATOM    481  CA  ASP A  61      15.355  18.806  -5.982  1.00 46.24      A    C  \\nATOM    482  C   ASP A  61      15.427  17.960  -4.705  1.00 46.34      A    C  \\nATOM    483  O   ASP A  61      16.274  18.221  -3.845  1.00 44.36      A    O  \\nATOM    484  CB  ASP A  61      14.475  20.025  -5.727  1.00 42.52      A    C  \\nATOM    485  CG  ASP A  61      14.619  21.077  -6.803  1.00 41.58      A    C  \\nATOM    486  OD1 ASP A  61      15.722  21.207  -7.367  1.00 35.92      A    O  \\nATOM    487  OD2 ASP A  61      13.624  21.786  -7.070  1.00 35.32      A    O1-\\nATOM    488  N   GLU A  62      14.556  16.980  -4.598  1.00 43.55      A    N  \\nATOM    489  CA  GLU A  62      14.563  16.114  -3.435  1.00 45.83      A    C  \\nATOM    490  C   GLU A  62      15.698  15.113  -3.480  1.00 45.66      A    C  \\nATOM    491  O   GLU A  62      16.126  14.603  -2.438  1.00 43.73      A    O  \\nATOM    492  CB  GLU A  62      13.234  15.392  -3.290  1.00 42.93      A    C  \\nATOM    493  CG  GLU A  62      12.062  16.281  -2.956  1.00 42.85      A    C  \\nATOM    494  CD  GLU A  62      10.751  15.551  -2.816  1.00 39.74      A    C  \\nATOM    495  OE1 GLU A  62      10.732  14.344  -3.137  1.00 35.68      A    O  \\nATOM    496  OE2 GLU A  62       9.750  16.180  -2.379  1.00 33.59      A    O1-\\nATOM    497  N   ASP A  63      16.197  14.807  -4.656  1.00 43.35      A    N  \\nATOM    498  CA  ASP A  63      17.291  13.858  -4.822  1.00 46.04      A    C  \\nATOM    499  C   ASP A  63      18.660  14.545  -4.733  1.00 45.97      A    C  \\nATOM    500  O   ASP A  63      19.665  13.844  -4.720  1.00 44.16      A    O  \\nATOM    501  CB  ASP A  63      17.153  13.124  -6.154  1.00 42.24      A    C  \\nATOM    502  CG  ASP A  63      15.978  12.173  -6.189  1.00 41.35      A    C  \\nATOM    503  OD1 ASP A  63      15.602  11.657  -5.105  1.00 37.05      A    O  \\nATOM    504  OD2 ASP A  63      15.425  11.929  -7.275  1.00 36.52      A    O1-\\nATOM    505  N   ASP A  64      18.661  15.844  -4.678  1.00 42.88      A    N  \\nATOM    506  CA  ASP A  64      19.931  16.549  -4.611  1.00 44.35      A    C  \\nATOM    507  C   ASP A  64      20.543  16.442  -3.227  1.00 44.62      A    C  \\nATOM    508  O   ASP A  64      19.844  16.472  -2.209  1.00 42.89      A    O  \\nATOM    509  CB  ASP A  64      19.751  18.021  -4.984  1.00 40.83      A    C  \\nATOM    510  CG  ASP A  64      19.442  18.224  -6.461  1.00 40.12      A    C  \\nATOM    511  OD1 ASP A  64      19.894  17.376  -7.290  1.00 35.63      A    O  \\nATOM    512  OD2 ASP A  64      18.744  19.203  -6.796  1.00 34.57      A    O1-\\nATOM    513  N   THR A  65      21.836  16.297  -3.214  1.00 46.92      A    N  \\nATOM    514  CA  THR A  65      22.554  16.185  -1.958  1.00 49.56      A    C  \\nATOM    515  C   THR A  65      22.859  17.546  -1.351  1.00 50.34      A    C  \\nATOM    516  O   THR A  65      22.979  17.658  -0.131  1.00 48.69      A    O  \\nATOM    517  CB  THR A  65      23.853  15.404  -2.189  1.00 45.75      A    C  \\nATOM    518  CG2 THR A  65      23.539  14.001  -2.707  1.00 44.35      A    C  \\nATOM    519  OG1 THR A  65      24.682  16.087  -3.120  1.00 46.27      A    O  \\nATOM    520  N   ASP A  66      22.996  18.544  -2.160  1.00 47.79      A    N  \\nATOM    521  CA  ASP A  66      23.314  19.892  -1.669  1.00 51.42      A    C  \\nATOM    522  C   ASP A  66      22.062  20.522  -1.100  1.00 52.41      A    C  \\nATOM    523  O   ASP A  66      21.036  20.667  -1.780  1.00 51.19      A    O  \\nATOM    524  CB  ASP A  66      23.884  20.764  -2.800  1.00 48.37      A    C  \\nATOM    525  CG  ASP A  66      25.253  20.278  -3.241  1.00 47.16      A    C  \\nATOM    526  OD1 ASP A  66      25.977  19.654  -2.426  1.00 40.28      A    O  \\nATOM    527  OD2 ASP A  66      25.597  20.497  -4.414  1.00 40.99      A    O1-\\nATOM    528  N   THR A  67      22.152  20.893   0.158  1.00 52.99      A    N  \\nATOM    529  CA  THR A  67      20.992  21.475   0.829  1.00 54.44      A    C  \\nATOM    530  C   THR A  67      20.832  22.966   0.510  1.00 54.84      A    C  \\nATOM    531  O   THR A  67      19.706  23.460   0.487  1.00 54.06      A    O  \\nATOM    532  CB  THR A  67      21.111  21.278   2.344  1.00 51.60      A    C  \\nATOM    533  CG2 THR A  67      21.162  19.789   2.653  1.00 48.14      A    C  \\nATOM    534  OG1 THR A  67      22.308  21.894   2.826  1.00 49.97      A    O  \\nATOM    535  N   SER A  68      21.927  23.654   0.276  1.00 54.08      A    N  \\nATOM    536  CA  SER A  68      21.835  25.089   0.018  1.00 57.41      A    C  \\nATOM    537  C   SER A  68      21.269  25.384  -1.354  1.00 57.50      A    C  \\nATOM    538  O   SER A  68      20.423  26.284  -1.517  1.00 55.74      A    O  \\nATOM    539  CB  SER A  68      23.214  25.745   0.177  1.00 53.95      A    C  \\nATOM    540  OG  SER A  68      24.111  25.284  -0.805  1.00 49.99      A    O  \\nATOM    541  N   ARG A  69      21.691  24.609  -2.330  1.00 57.23      A    N  \\nATOM    542  CA  ARG A  69      21.165  24.820  -3.702  1.00 59.30      A    C  \\nATOM    543  C   ARG A  69      19.726  24.373  -3.821  1.00 59.66      A    C  \\nATOM    544  O   ARG A  69      18.941  25.007  -4.543  1.00 58.97      A    O  \\nATOM    545  CB  ARG A  69      22.039  24.073  -4.718  1.00 56.46      A    C  \\nATOM    546  CG  ARG A  69      23.456  24.607  -4.814  1.00 53.52      A    C  \\nATOM    547  CD  ARG A  69      24.248  23.850  -5.828  1.00 49.90      A    C  \\nATOM    548  NE  ARG A  69      25.592  24.386  -5.945  1.00 45.58      A    N  \\nATOM    549  CZ  ARG A  69      26.550  23.883  -6.729  1.00 44.72      A    C  \\nATOM    550  NH1 ARG A  69      26.299  22.792  -7.463  1.00 38.83      A    N1+\\nATOM    551  NH2 ARG A  69      27.749  24.456  -6.779  1.00 36.22      A    N  \\nATOM    552  N   SER A  70      19.374  23.313  -3.105  1.00 54.93      A    N  \\nATOM    553  CA  SER A  70      18.003  22.855  -3.144  1.00 56.99      A    C  \\nATOM    554  C   SER A  70      17.070  23.810  -2.395  1.00 57.40      A    C  \\nATOM    555  O   SER A  70      15.887  23.907  -2.717  1.00 57.49      A    O  \\nATOM    556  CB  SER A  70      17.898  21.451  -2.581  1.00 54.91      A    C  \\nATOM    557  OG  SER A  70      18.235  21.439  -1.208  1.00 49.75      A    O  \\nATOM    558  N   HIS A  71      17.595  24.478  -1.376  1.00 55.95      A    N  \\nATOM    559  CA  HIS A  71      16.780  25.471  -0.672  1.00 57.03      A    C  \\nATOM    560  C   HIS A  71      16.515  26.706  -1.535  1.00 57.63      A    C  \\nATOM    561  O   HIS A  71      15.437  27.273  -1.460  1.00 58.13      A    O  \\nATOM    562  CB  HIS A  71      17.488  25.855   0.625  1.00 55.46      A    C  \\nATOM    563  CG  HIS A  71      16.740  26.866   1.431  1.00 52.61      A    C  \\nATOM    564  CD2 HIS A  71      16.995  28.180   1.663  1.00 47.89      A    C  \\nATOM    565  ND1 HIS A  71      15.571  26.583   2.108  1.00 46.38      A    N  \\nATOM    566  CE1 HIS A  71      15.144  27.668   2.718  1.00 44.27      A    C  \\nATOM    567  NE2 HIS A  71      15.992  28.652   2.478  1.00 41.79      A    N  \\nATOM    568  N   LYS A  72      17.491  27.065  -2.338  1.00 53.63      A    N  \\nATOM    569  CA  LYS A  72      17.309  28.224  -3.212  1.00 56.36      A    C  \\nATOM    570  C   LYS A  72      16.370  27.935  -4.369  1.00 56.27      A    C  \\nATOM    571  O   LYS A  72      15.602  28.805  -4.792  1.00 56.13      A    O  \\nATOM    572  CB  LYS A  72      18.650  28.724  -3.757  1.00 54.72      A    C  \\nATOM    573  CG  LYS A  72      19.576  29.304  -2.701  1.00 51.89      A    C  \\nATOM    574  CD  LYS A  72      20.882  29.758  -3.307  1.00 47.88      A    C  \\nATOM    575  CE  LYS A  72      21.816  30.335  -2.256  1.00 43.23      A    C  \\nATOM    576  NZ  LYS A  72      23.111  30.770  -2.827  1.00 39.42      A    N1+\\nATOM    577  N   ARG A  73      16.456  26.695  -4.884  1.00 55.15      A    N  \\nATOM    578  CA  ARG A  73      15.539  26.327  -5.979  1.00 56.50      A    C  \\nATOM    579  C   ARG A  73      14.106  26.206  -5.486  1.00 57.25      A    C  \\nATOM    580  O   ARG A  73      13.178  26.582  -6.197  1.00 57.23      A    O  \\nATOM    581  CB  ARG A  73      16.002  25.000  -6.588  1.00 53.75      A    C  \\nATOM    582  CG  ARG A  73      17.276  25.121  -7.376  1.00 50.32      A    C  \\nATOM    583  CD  ARG A  73      17.731  23.790  -7.934  1.00 46.61      A    C  \\nATOM    584  NE  ARG A  73      18.964  23.880  -8.667  1.00 41.70      A    N  \\nATOM    585  CZ  ARG A  73      19.597  22.839  -9.191  1.00 40.79      A    C  \\nATOM    586  NH1 ARG A  73      19.099  21.624  -9.078  1.00 34.94      A    N1+\\nATOM    587  NH2 ARG A  73      20.754  23.012  -9.839  1.00 32.52      A    N  \\nATOM    588  N   ARG A  74      13.937  25.683  -4.269  1.00 53.12      A    N  \\nATOM    589  CA  ARG A  74      12.578  25.589  -3.713  1.00 56.23      A    C  \\nATOM    590  C   ARG A  74      11.996  26.954  -3.366  1.00 56.92      A    C  \\nATOM    591  O   ARG A  74      10.792  27.135  -3.407  1.00 56.26      A    O  \\nATOM    592  CB  ARG A  74      12.581  24.687  -2.481  1.00 52.71      A    C  \\nATOM    593  CG  ARG A  74      12.813  23.216  -2.774  1.00 49.74      A    C  \\nATOM    594  CD  ARG A  74      12.902  22.402  -1.502  1.00 45.30      A    C  \\nATOM    595  NE  ARG A  74      13.179  20.988  -1.770  1.00 40.13      A    N  \\nATOM    596  CZ  ARG A  74      13.341  20.051  -0.824  1.00 38.74      A    C  \\nATOM    597  NH1 ARG A  74      13.239  20.381   0.455  1.00 33.39      A    N1+\\nATOM    598  NH2 ARG A  74      13.607  18.811  -1.166  1.00 31.35      A    N  \\nATOM    599  N   HIS A  75      12.869  27.893  -3.037  1.00 55.13      A    N  \\nATOM    600  CA  HIS A  75      12.407  29.232  -2.721  1.00 57.17      A    C  \\nATOM    601  C   HIS A  75      11.912  29.949  -3.986  1.00 57.87      A    C  \\nATOM    602  O   HIS A  75      10.935  30.689  -3.953  1.00 58.48      A    O  \\nATOM    603  CB  HIS A  75      13.502  30.047  -2.054  1.00 54.39      A    C  \\nATOM    604  CG  HIS A  75      13.079  31.432  -1.669  1.00 51.13      A    C  \\nATOM    605  CD2 HIS A  75      13.405  32.650  -2.187  1.00 47.76      A    C  \\nATOM    606  ND1 HIS A  75      12.214  31.696  -0.618  1.00 44.90      A    N  \\nATOM    607  CE1 HIS A  75      12.042  33.014  -0.542  1.00 43.81      A    C  \\nATOM    608  NE2 HIS A  75      12.751  33.602  -1.488  1.00 41.71      A    N  \\nATOM    609  N   VAL A  76      12.601  29.717  -5.103  1.00 54.41      A    N  \\nATOM    610  CA  VAL A  76      12.191  30.323  -6.368  1.00 57.12      A    C  \\nATOM    611  C   VAL A  76      10.937  29.634  -6.908  1.00 58.46      A    C  \\nATOM    612  O   VAL A  76      10.044  30.291  -7.446  1.00 58.91      A    O  \\nATOM    613  CB  VAL A  76      13.363  30.251  -7.396  1.00 55.05      A    C  \\nATOM    614  CG1 VAL A  76      14.523  31.083  -6.908  1.00 54.89      A    C  \\nATOM    615  CG2 VAL A  76      12.917  30.755  -8.769  1.00 53.41      A    C  \\nATOM    616  N   GLU A  77      10.887  28.308  -6.743  1.00 54.48      A    N  \\nATOM    617  CA  GLU A  77       9.699  27.571  -7.211  1.00 57.39      A    C  \\nATOM    618  C   GLU A  77       8.460  27.889  -6.391  1.00 58.20      A    C  \\nATOM    619  O   GLU A  77       7.374  27.994  -6.935  1.00 57.98      A    O  \\nATOM    620  CB  GLU A  77       9.971  26.076  -7.180  1.00 54.16      A    C  \\nATOM    621  CG  GLU A  77      10.969  25.624  -8.231  1.00 50.62      A    C  \\nATOM    622  CD  GLU A  77      11.268  24.120  -8.146  1.00 45.39      A    C  \\nATOM    623  OE1 GLU A  77      10.761  23.457  -7.219  1.00 43.08      A    O  \\nATOM    624  OE2 GLU A  77      12.023  23.621  -9.002  1.00 40.70      A    O1-\\nATOM    625  N   ASP A  78       8.651  28.078  -5.101  1.00 54.81      A    N  \\nATOM    626  CA  ASP A  78       7.505  28.452  -4.270  1.00 56.67      A    C  \\nATOM    627  C   ASP A  78       7.057  29.878  -4.522  1.00 57.27      A    C  \\nATOM    628  O   ASP A  78       5.873  30.190  -4.425  1.00 57.58      A    O  \\nATOM    629  CB  ASP A  78       7.835  28.257  -2.781  1.00 54.08      A    C  \\nATOM    630  CG  ASP A  78       7.910  26.797  -2.406  1.00 51.34      A    C  \\nATOM    631  OD1 ASP A  78       7.214  25.979  -3.016  1.00 46.99      A    O  \\nATOM    632  OD2 ASP A  78       8.680  26.448  -1.464  1.00 48.45      A    O1-\\nATOM    633  N   ALA A  79       7.999  30.748  -4.832  1.00 57.69      A    N  \\nATOM    634  CA  ALA A  79       7.640  32.128  -5.172  1.00 59.90      A    C  \\nATOM    635  C   ALA A  79       6.984  32.210  -6.536  1.00 60.64      A    C  \\nATOM    636  O   ALA A  79       6.110  33.041  -6.773  1.00 61.34      A    O  \\nATOM    637  CB  ALA A  79       8.856  33.011  -5.111  1.00 58.63      A    C  \\nATOM    638  N   ALA A  80       7.432  31.355  -7.457  1.00 56.51      A    N  \\nATOM    639  CA  ALA A  80       6.829  31.333  -8.782  1.00 58.82      A    C  \\nATOM    640  C   ALA A  80       5.447  30.671  -8.769  1.00 59.63      A    C  \\nATOM    641  O   ALA A  80       4.563  31.046  -9.537  1.00 60.34      A    O  \\nATOM    642  CB  ALA A  80       7.735  30.605  -9.771  1.00 57.24      A    C  \\nATOM    643  N   LYS A  81       5.278  29.678  -7.886  1.00 56.25      A    N  \\nATOM    644  CA  LYS A  81       3.966  29.050  -7.752  1.00 57.57      A    C  \\nATOM    645  C   LYS A  81       2.947  29.975  -7.098  1.00 57.29      A    C  \\nATOM    646  O   LYS A  81       1.776  29.950  -7.451  1.00 58.24      A    O  \\nATOM    647  CB  LYS A  81       4.068  27.745  -6.972  1.00 55.40      A    C  \\nATOM    648  CG  LYS A  81       4.782  26.624  -7.695  1.00 52.73      A    C  \\nATOM    649  CD  LYS A  81       4.971  25.426  -6.822  1.00 50.04      A    C  \\nATOM    650  CE  LYS A  81       5.745  24.323  -7.535  1.00 47.79      A    C  \\nATOM    651  NZ  LYS A  81       6.038  23.172  -6.668  1.00 43.12      A    N1+\\nATOM    652  N   GLU A  82       3.424  30.819  -6.205  1.00 51.77      A    N  \\nATOM    653  CA  GLU A  82       2.547  31.804  -5.608  1.00 54.45      A    C  \\nATOM    654  C   GLU A  82       2.150  32.891  -6.618  1.00 55.20      A    C  \\nATOM    655  O   GLU A  82       1.055  33.453  -6.542  1.00 56.06      A    O  \\nATOM    656  CB  GLU A  82       3.192  32.463  -4.397  1.00 52.42      A    C  \\nATOM    657  CG  GLU A  82       3.286  31.541  -3.188  1.00 49.58      A    C  \\nATOM    658  CD  GLU A  82       3.944  32.199  -2.005  1.00 45.47      A    C  \\nATOM    659  OE1 GLU A  82       4.434  33.369  -2.130  1.00 42.76      A    O  \\nATOM    660  OE2 GLU A  82       4.004  31.563  -0.924  1.00 40.65      A    O1-\\nATOM    661  N   LYS A  83       3.016  33.181  -7.561  1.00 53.67      A    N  \\nATOM    662  CA  LYS A  83       2.681  34.139  -8.599  1.00 54.98      A    C  \\nATOM    663  C   LYS A  83       1.696  33.558  -9.572  1.00 55.19      A    C  \\nATOM    664  O   LYS A  83       0.814  34.284 -10.078  1.00 55.81      A    O  \\nATOM    665  CB  LYS A  83       3.947  34.582  -9.337  1.00 52.55      A    C  \\nATOM    666  CG  LYS A  83       4.900  35.450  -8.499  1.00 48.99      A    C  \\nATOM    667  CD  LYS A  83       6.136  35.825  -9.277  1.00 45.52      A    C  \\nATOM    668  CE  LYS A  83       7.079  36.674  -8.467  1.00 40.03      A    C  \\nATOM    669  NZ  LYS A  83       8.265  37.080  -9.203  1.00 37.00      A    N1+\\nATOM    670  N   LEU A  84       1.808  32.260  -9.871  1.00 54.28      A    N  \\nATOM    671  CA  LEU A  84       0.847  31.622 -10.752  1.00 55.55      A    C  \\nATOM    672  C   LEU A  84      -0.524  31.498 -10.090  1.00 56.14      A    C  \\nATOM    673  O   LEU A  84      -1.562  31.591 -10.764  1.00 56.06      A    O  \\nATOM    674  CB  LEU A  84       1.342  30.238 -11.151  1.00 52.50      A    C  \\nATOM    675  CG  LEU A  84       2.570  30.177 -12.069  1.00 49.28      A    C  \\nATOM    676  CD1 LEU A  84       2.216  30.766 -13.421  1.00 49.96      A    C  \\nATOM    677  CD2 LEU A  84       3.053  28.757 -12.221  1.00 49.22      A    C  \\nATOM    678  N   ARG A  85      -0.525  31.294  -8.790  1.00 48.18      A    N  \\nATOM    679  CA  ARG A  85      -1.797  31.228  -8.070  1.00 49.39      A    C  \\nATOM    680  C   ARG A  85      -2.467  32.605  -8.001  1.00 50.15      A    C  \\nATOM    681  O   ARG A  85      -3.686  32.704  -7.987  1.00 49.28      A    O  \\nATOM    682  CB  ARG A  85      -1.577  30.636  -6.690  1.00 46.51      A    C  \\nATOM    683  CG  ARG A  85      -1.271  29.167  -6.665  1.00 43.88      A    C  \\nATOM    684  CD  ARG A  85      -1.073  28.662  -5.269  1.00 39.89      A    C  \\nATOM    685  NE  ARG A  85      -0.724  27.256  -5.217  1.00 36.76      A    N  \\nATOM    686  CZ  ARG A  85      -0.490  26.579  -4.104  1.00 36.15      A    C  \\nATOM    687  NH1 ARG A  85      -0.557  27.179  -2.941  1.00 32.28      A    N1+\\nATOM    688  NH2 ARG A  85      -0.173  25.295  -4.164  1.00 29.72      A    N  \\nATOM    689  N   LEU A  86      -1.680  33.634  -7.934  1.00 53.51      A    N  \\nATOM    690  CA  LEU A  86      -2.245  34.997  -7.909  1.00 55.19      A    C  \\nATOM    691  C   LEU A  86      -2.775  35.403  -9.293  1.00 55.79      A    C  \\nATOM    692  O   LEU A  86      -3.770  36.114  -9.392  1.00 56.35      A    O  \\nATOM    693  CB  LEU A  86      -1.210  35.995  -7.433  1.00 52.76      A    C  \\nATOM    694  CG  LEU A  86      -0.838  35.927  -5.950  1.00 48.93      A    C  \\nATOM    695  CD1 LEU A  86      -2.045  36.244  -5.082  1.00 47.95      A    C  \\nATOM    696  CD2 LEU A  86       0.290  36.882  -5.640  1.00 47.33      A    C  \\nATOM    697  N   ILE A  87      -2.083  34.956 -10.336  1.00 58.33      A    N  \\nATOM    698  CA  ILE A  87      -2.538  35.250 -11.700  1.00 58.41      A    C  \\nATOM    699  C   ILE A  87      -3.827  34.469 -11.999  1.00 58.31      A    C  \\nATOM    700  O   ILE A  87      -4.777  35.016 -12.578  1.00 58.11      A    O  \\nATOM    701  CB  ILE A  87      -1.439  34.937 -12.716  1.00 56.40      A    C  \\nATOM    702  CG1 ILE A  87      -0.273  35.922 -12.520  1.00 58.36      A    C  \\nATOM    703  CG2 ILE A  87      -1.938  35.024 -14.144  1.00 54.86      A    C  \\nATOM    704  CD1 ILE A  87       0.964  35.559 -13.285  1.00 53.19      A    C  \\nATOM    705  N   ALA A  88      -3.852  33.205 -11.620  1.00 50.24      A    N  \\nATOM    706  CA  ALA A  88      -5.034  32.383 -11.834  1.00 51.75      A    C  \\nATOM    707  C   ALA A  88      -6.201  32.835 -10.970  1.00 52.12      A    C  \\nATOM    708  O   ALA A  88      -7.357  32.676 -11.359  1.00 51.88      A    O  \\nATOM    709  CB  ALA A  88      -4.726  30.908 -11.580  1.00 49.09      A    C  \\nATOM    710  N   PHE A  89      -5.894  33.356  -9.806  1.00 48.02      A    N  \\nATOM    711  CA  PHE A  89      -6.946  33.904  -8.970  1.00 50.63      A    C  \\nATOM    712  C   PHE A  89      -7.492  35.213  -9.518  1.00 51.12      A    C  \\nATOM    713  O   PHE A  89      -8.699  35.465  -9.455  1.00 50.74      A    O  \\nATOM    714  CB  PHE A  89      -6.423  34.112  -7.532  1.00 48.33      A    C  \\nATOM    715  CG  PHE A  89      -7.469  34.614  -6.562  1.00 46.35      A    C  \\nATOM    716  CD1 PHE A  89      -8.452  33.748  -6.065  1.00 44.95      A    C  \\nATOM    717  CD2 PHE A  89      -7.477  35.924  -6.170  1.00 45.02      A    C  \\nATOM    718  CE1 PHE A  89      -9.402  34.198  -5.190  1.00 42.66      A    C  \\nATOM    719  CE2 PHE A  89      -8.411  36.382  -5.288  1.00 43.02      A    C  \\nATOM    720  CZ  PHE A  89      -9.380  35.530  -4.798  1.00 40.79      A    C  \\nATOM    721  N   GLN A  90      -6.631  36.046 -10.073  1.00 52.44      A    N  \\nATOM    722  CA  GLN A  90      -7.094  37.294 -10.660  1.00 53.99      A    C  \\nATOM    723  C   GLN A  90      -7.834  37.069 -11.974  1.00 54.22      A    C  \\nATOM    724  O   GLN A  90      -8.697  37.869 -12.330  1.00 54.33      A    O  \\nATOM    725  CB  GLN A  90      -5.914  38.232 -10.877  1.00 51.38      A    C  \\nATOM    726  CG  GLN A  90      -5.303  38.768  -9.597  1.00 49.74      A    C  \\nATOM    727  CD  GLN A  90      -4.080  39.633  -9.827  1.00 46.09      A    C  \\nATOM    728  NE2 GLN A  90      -3.492  40.148  -8.759  1.00 42.97      A    N  \\nATOM    729  OE1 GLN A  90      -3.665  39.842 -10.955  1.00 44.17      A    O  \\nATOM    730  N   LYS A  91      -7.489  35.994 -12.673  1.00 51.31      A    N  \\nATOM    731  CA  LYS A  91      -8.217  35.663 -13.900  1.00 52.74      A    C  \\nATOM    732  C   LYS A  91      -9.614  35.125 -13.613  1.00 53.12      A    C  \\nATOM    733  O   LYS A  91     -10.532  35.293 -14.408  1.00 51.97      A    O  \\nATOM    734  CB  LYS A  91      -7.445  34.694 -14.768  1.00 49.53      A    C  \\nATOM    735  CG  LYS A  91      -6.221  35.277 -15.423  1.00 47.99      A    C  \\nATOM    736  CD  LYS A  91      -5.501  34.252 -16.276  1.00 45.45      A    C  \\nATOM    737  CE  LYS A  91      -4.302  34.835 -16.948  1.00 42.17      A    C  \\nATOM    738  NZ  LYS A  91      -3.571  33.830 -17.781  1.00 38.07      A    N1+\\nATOM    739  N   ASN A  92      -9.773  34.462 -12.462  1.00 50.92      A    N  \\nATOM    740  CA  ASN A  92     -11.075  33.959 -12.091  1.00 51.60      A    C  \\nATOM    741  C   ASN A  92     -12.018  35.070 -11.657  1.00 51.57      A    C  \\nATOM    742  O   ASN A  92     -13.223  34.943 -11.771  1.00 50.26      A    O  \\nATOM    743  CB  ASN A  92     -10.942  32.923 -10.969  1.00 48.51      A    C  \\nATOM    744  CG  ASN A  92     -10.318  31.622 -11.420  1.00 46.83      A    C  \\nATOM    745  ND2 ASN A  92      -9.489  31.045 -10.588  1.00 46.27      A    N  \\nATOM    746  OD1 ASN A  92     -10.600  31.150 -12.520  1.00 44.39      A    O  \\nATOM    747  N   GLU A  93     -11.463  36.159 -11.160  1.00 49.52      A    N  \\nATOM    748  CA  GLU A  93     -12.276  37.315 -10.777  1.00 49.53      A    C  \\nATOM    749  C   GLU A  93     -12.720  38.115 -12.002  1.00 49.38      A    C  \\nATOM    750  O   GLU A  93     -13.786  38.730 -11.978  1.00 47.61      A    O  \\nATOM    751  CB  GLU A  93     -11.514  38.207  -9.809  1.00 45.46      A    C  \\nATOM    752  CG  GLU A  93     -11.245  37.579  -8.459  1.00 43.74      A    C  \\nATOM    753  CD  GLU A  93     -10.516  38.496  -7.517  1.00 40.38      A    C  \\nATOM    754  OE1 GLU A  93     -10.090  39.594  -7.961  1.00 38.45      A    O  \\nATOM    755  OE2 GLU A  93     -10.373  38.148  -6.317  1.00 37.28      A    O1-\\nATOM    756  N   LYS A  94     -11.896  38.132 -13.026  1.00 47.87      A    N  \\nATOM    757  CA  LYS A  94     -12.221  38.868 -14.250  1.00 49.27      A    C  \\nATOM    758  C   LYS A  94     -13.189  38.133 -15.156  1.00 48.98      A    C  \\nATOM    759  O   LYS A  94     -13.832  38.733 -16.019  1.00 47.69      A    O  \\nATOM    760  CB  LYS A  94     -10.957  39.209 -15.024  1.00 46.31      A    C  \\nATOM    761  CG  LYS A  94     -10.040  40.182 -14.317  1.00 45.00      A    C  \\nATOM    762  CD  LYS A  94      -8.813  40.491 -15.182  1.00 42.44      A    C  \\nATOM    763  CE  LYS A  94      -7.876  41.427 -14.458  1.00 38.00      A    C  \\nATOM    764  NZ  LYS A  94      -6.660  41.747 -15.261  1.00 34.15      A    N1+\\nATOM    765  N   LEU A  95     -13.282  36.798 -14.987  1.00 46.80      A    N  \\nATOM    766  CA  LEU A  95     -14.184  36.004 -15.796  1.00 47.87      A    C  \\nATOM    767  C   LEU A  95     -15.605  36.165 -15.312  1.00 48.08      A    C  \\nATOM    768  O   LEU A  95     -15.872  36.512 -14.153  1.00 47.19      A    O  \\nATOM    769  CB  LEU A  95     -13.788  34.530 -15.779  1.00 45.01      A    C  \\nATOM    770  CG  LEU A  95     -12.494  34.134 -16.478  1.00 44.00      A    C  \\nATOM    771  CD1 LEU A  95     -12.631  34.369 -17.971  1.00 42.56      A    C  \\nATOM    772  CD2 LEU A  95     -12.153  32.702 -16.179  1.00 42.03      A    C  \\nATOM    773  N   LEU A  96     -16.538  35.878 -16.200  1.00 49.24      A    N  \\nATOM    774  CA  LEU A  96     -17.959  35.950 -15.887  1.00 51.65      A    C  \\nATOM    775  C   LEU A  96     -18.470  34.598 -15.431  1.00 53.05      A    C  \\nATOM    776  O   LEU A  96     -17.732  33.601 -15.427  1.00 50.61      A    O  \\nATOM    777  CB  LEU A  96     -18.747  36.449 -17.098  1.00 46.01      A    C  \\nATOM    778  CG  LEU A  96     -18.437  37.874 -17.536  1.00 43.79      A    C  \\nATOM    779  CD1 LEU A  96     -18.787  38.866 -16.464  1.00 43.27      A    C  \\nATOM    780  CD2 LEU A  96     -19.196  38.199 -18.819  1.00 42.27      A    C  \\nATOM    781  N   ASP A  97     -19.704  34.549 -15.054  1.00 45.49      A    N  \\nATOM    782  CA  ASP A  97     -20.345  33.314 -14.601  1.00 46.75      A    C  \\nATOM    783  C   ASP A  97     -20.650  32.440 -15.788  1.00 47.94      A    C  \\nATOM    784  O   ASP A  97     -21.300  32.824 -16.725  1.00 46.23      A    O  \\nATOM    785  CB  ASP A  97     -21.603  33.620 -13.813  1.00 42.35      A    C  \\nATOM    786  CG  ASP A  97     -21.334  34.306 -12.486  1.00 41.88      A    C  \\nATOM    787  OD1 ASP A  97     -20.265  34.032 -11.881  1.00 37.36      A    O  \\nATOM    788  OD2 ASP A  97     -22.174  35.109 -12.054  1.00 35.97      A    O1-\\nATOM    789  N   GLY A  98     -20.150  31.228 -15.705  1.00 56.16      A    N  \\nATOM    790  CA  GLY A  98     -20.355  30.240 -16.766  1.00 57.28      A    C  \\nATOM    791  C   GLY A  98     -19.165  30.019 -17.678  1.00 60.56      A    C  \\nATOM    792  O   GLY A  98     -19.146  29.048 -18.429  1.00 59.19      A    O  \\nATOM    793  N   LYS A  99     -18.197  30.888 -17.598  1.00 62.20      A    N  \\nATOM    794  CA  LYS A  99     -17.010  30.779 -18.438  1.00 64.90      A    C  \\nATOM    795  C   LYS A  99     -15.868  30.157 -17.651  1.00 66.22      A    C  \\nATOM    796  O   LYS A  99     -15.800  30.219 -16.422  1.00 64.84      A    O  \\nATOM    797  CB  LYS A  99     -16.592  32.135 -19.006  1.00 61.00      A    C  \\nATOM    798  CG  LYS A  99     -17.582  32.706 -20.002  1.00 56.79      A    C  \\nATOM    799  CD  LYS A  99     -17.134  34.041 -20.533  1.00 52.17      A    C  \\nATOM    800  CE  LYS A  99     -18.111  34.607 -21.546  1.00 47.05      A    C  \\nATOM    801  NZ  LYS A  99     -17.727  35.941 -22.044  1.00 41.00      A    N1+\\nATOM    802  N   LEU A 100     -14.970  29.540 -18.385  1.00 69.28      A    N  \\nATOM    803  CA  LEU A 100     -13.839  28.854 -17.784  1.00 71.16      A    C  \\nATOM    804  C   LEU A 100     -12.578  29.172 -18.547  1.00 71.21      A    C  \\nATOM    805  O   LEU A 100     -12.600  29.255 -19.781  1.00 69.91      A    O  \\nATOM    806  CB  LEU A 100     -14.053  27.342 -17.744  1.00 68.26      A    C  \\nATOM    807  CG  LEU A 100     -15.144  26.805 -16.816  1.00 65.46      A    C  \\nATOM    808  CD1 LEU A 100     -14.821  27.124 -15.376  1.00 66.88      A    C  \\nATOM    809  CD2 LEU A 100     -15.316  25.295 -17.016  1.00 67.20      A    C  \\nATOM    810  N   PRO A 101     -11.446  29.338 -17.867  1.00 69.67      A    N  \\nATOM    811  CA  PRO A 101     -10.184  29.528 -18.573  1.00 71.25      A    C  \\nATOM    812  C   PRO A 101      -9.639  28.226 -19.108  1.00 72.12      A    C  \\nATOM    813  O   PRO A 101     -10.241  27.169 -18.933  1.00 70.80      A    O  \\nATOM    814  CB  PRO A 101      -9.278  30.141 -17.507  1.00 68.25      A    C  \\nATOM    815  CG  PRO A 101      -9.803  29.587 -16.219  1.00 67.27      A    C  \\nATOM    816  CD  PRO A 101     -11.283  29.518 -16.422  1.00 70.06      A    C  \\nATOM    817  N   LYS A 102      -8.492  28.324 -19.740  1.00 67.52      A    N  \\nATOM    818  CA  LYS A 102      -7.868  27.139 -20.325  1.00 68.15      A    C  \\nATOM    819  C   LYS A 102      -7.287  26.240 -19.242  1.00 69.24      A    C  \\nATOM    820  O   LYS A 102      -7.122  25.032 -19.463  1.00 69.17      A    O  \\nATOM    821  CB  LYS A 102      -6.781  27.513 -21.311  1.00 64.61      A    C  \\nATOM    822  CG  LYS A 102      -7.293  28.214 -22.579  1.00 60.18      A    C  \\nATOM    823  CD  LYS A 102      -6.164  28.526 -23.534  1.00 56.49      A    C  \\nATOM    824  CE  LYS A 102      -6.650  29.244 -24.774  1.00 51.08      A    C  \\nATOM    825  NZ  LYS A 102      -5.521  29.567 -25.692  1.00 43.94      A    N1+\\nATOM    826  N   GLU A 103      -6.958  26.813 -18.091  1.00 70.17      A    N  \\nATOM    827  CA  GLU A 103      -6.398  26.034 -17.018  1.00 70.51      A    C  \\nATOM    828  C   GLU A 103      -7.447  25.160 -16.334  1.00 72.04      A    C  \\nATOM    829  O   GLU A 103      -7.160  24.047 -15.919  1.00 70.82      A    O  \\nATOM    830  CB  GLU A 103      -5.714  26.942 -15.983  1.00 67.24      A    C  \\nATOM    831  CG  GLU A 103      -4.506  27.669 -16.547  1.00 62.69      A    C  \\nATOM    832  CD  GLU A 103      -3.850  28.578 -15.527  1.00 56.33      A    C  \\nATOM    833  OE1 GLU A 103      -4.353  28.658 -14.404  1.00 52.58      A    O  \\nATOM    834  OE2 GLU A 103      -2.778  29.194 -15.861  1.00 50.95      A    O1-\\nATOM    835  N   LEU A 104      -8.670  25.668 -16.253  1.00 72.31      A    N  \\nATOM    836  CA  LEU A 104      -9.743  24.896 -15.629  1.00 73.94      A    C  \\nATOM    837  C   LEU A 104     -10.303  23.861 -16.589  1.00 74.77      A    C  \\nATOM    838  O   LEU A 104     -11.017  22.950 -16.149  1.00 73.89      A    O  \\nATOM    839  CB  LEU A 104     -10.857  25.811 -15.144  1.00 71.56      A    C  \\nATOM    840  CG  LEU A 104     -10.506  26.750 -13.979  1.00 70.49      A    C  \\nATOM    841  CD1 LEU A 104     -10.084  25.944 -12.754  1.00 70.16      A    C  \\nATOM    842  CD2 LEU A 104     -11.658  27.678 -13.667  1.00 70.51      A    C  \\nATOM    843  N   ARG A 105     -10.052  23.980 -17.862  1.00 73.57      A    N  \\nATOM    844  CA  ARG A 105     -10.494  22.984 -18.832  1.00 74.21      A    C  \\nATOM    845  C   ARG A 105      -9.647  21.733 -18.770  1.00 74.57      A    C  \\nATOM    846  O   ARG A 105     -10.061  20.698 -19.279  1.00 74.18      A    O  \\nATOM    847  CB  ARG A 105     -10.440  23.566 -20.248  1.00 71.41      A    C  \\nATOM    848  CG  ARG A 105     -11.483  24.655 -20.521  1.00 67.24      A    C  \\nATOM    849  CD  ARG A 105     -11.341  25.189 -21.941  1.00 64.67      A    C  \\nATOM    850  NE  ARG A 105     -12.320  26.233 -22.208  1.00 61.82      A    N  \\nATOM    851  CZ  ARG A 105     -12.402  26.886 -23.359  1.00 59.29      A    C  \\nATOM    852  NH1 ARG A 105     -11.598  26.594 -24.352  1.00 50.75      A    N1+\\nATOM    853  NH2 ARG A 105     -13.319  27.830 -23.530  1.00 51.57      A    N  \\nATOM    854  N   LYS A 106      -8.506  21.821 -18.120  1.00 75.33      A    N  \\nATOM    855  CA  LYS A 106      -7.635  20.653 -17.998  1.00 75.74      A    C  \\nATOM    856  C   LYS A 106      -8.089  19.719 -16.901  1.00 76.73      A    C  \\nATOM    857  O   LYS A 106      -7.721  18.537 -16.928  1.00 75.54      A    O  \\nATOM    858  CB  LYS A 106      -6.183  21.104 -17.741  1.00 72.72      A    C  \\nATOM    859  CG  LYS A 106      -5.548  21.875 -18.863  1.00 67.54      A    C  \\nATOM    860  CD  LYS A 106      -4.126  22.240 -18.533  1.00 64.96      A    C  \\nATOM    861  CE  LYS A 106      -3.479  23.043 -19.623  1.00 59.59      A    C  \\nATOM    862  NZ  LYS A 106      -2.092  23.449 -19.288  1.00 51.56      A    N1+\\nATOM    863  N   TYR A 107      -8.860  20.208 -15.962  1.00 79.73      A    N  \\nATOM    864  CA  TYR A 107      -9.297  19.392 -14.825  1.00 81.94      A    C  \\nATOM    865  C   TYR A 107     -10.732  18.909 -14.952  1.00 83.25      A    C  \\nATOM    866  O   TYR A 107     -11.185  18.140 -14.133  1.00 81.87      A    O  \\nATOM    867  CB  TYR A 107      -9.142  20.162 -13.518  1.00 80.71      A    C  \\nATOM    868  CG  TYR A 107      -7.696  20.515 -13.192  1.00 80.76      A    C  \\nATOM    869  CD1 TYR A 107      -6.864  19.567 -12.624  1.00 79.71      A    C  \\nATOM    870  CD2 TYR A 107      -7.191  21.766 -13.461  1.00 80.20      A    C  \\nATOM    871  CE1 TYR A 107      -5.538  19.855 -12.317  1.00 77.71      A    C  \\nATOM    872  CE2 TYR A 107      -5.868  22.079 -13.170  1.00 79.29      A    C  \\nATOM    873  CZ  TYR A 107      -5.062  21.116 -12.604  1.00 77.97      A    C  \\nATOM    874  OH  TYR A 107      -3.731  21.405 -12.305  1.00 73.03      A    O  \\nATOM    875  N   ARG A 108     -11.437  19.362 -15.995  1.00 79.60      A    N  \\nATOM    876  CA  ARG A 108     -12.828  18.951 -16.172  1.00 80.64      A    C  \\nATOM    877  C   ARG A 108     -12.883  17.582 -16.854  1.00 81.20      A    C  \\nATOM    878  O   ARG A 108     -12.026  17.248 -17.660  1.00 80.99      A    O  \\nATOM    879  CB  ARG A 108     -13.609  19.988 -16.981  1.00 77.66      A    C  \\nATOM    880  CG  ARG A 108     -13.058  20.230 -18.379  1.00 69.88      A    C  \\nATOM    881  CD  ARG A 108     -13.858  21.298 -19.087  1.00 66.81      A    C  \\nATOM    882  NE  ARG A 108     -13.395  21.509 -20.474  1.00 63.05      A    N  \\nATOM    883  CZ  ARG A 108     -13.895  22.423 -21.296  1.00 58.19      A    C  \\nATOM    884  NH1 ARG A 108     -14.917  23.159 -20.909  1.00 50.40      A    N1+\\nATOM    885  NH2 ARG A 108     -13.381  22.562 -22.524  1.00 49.86      A    N  \\nATOM    886  N   PRO A 109     -13.897  16.772 -16.569  1.00 80.39      A    N  \\nATOM    887  CA  PRO A 109     -14.030  15.487 -17.253  1.00 80.32      A    C  \\nATOM    888  C   PRO A 109     -14.591  15.657 -18.661  1.00 80.00      A    C  \\nATOM    889  O   PRO A 109     -14.931  16.751 -19.101  1.00 77.90      A    O  \\nATOM    890  CB  PRO A 109     -15.015  14.740 -16.363  1.00 78.51      A    C  \\nATOM    891  CG  PRO A 109     -15.832  15.781 -15.747  1.00 79.08      A    C  \\nATOM    892  CD  PRO A 109     -14.905  16.944 -15.502  1.00 81.90      A    C  \\nATOM    893  N   GLN A 110     -14.643  14.527 -19.340  1.00 78.49      A    N  \\nATOM    894  CA  GLN A 110     -15.169  14.518 -20.702  1.00 77.99      A    C  \\nATOM    895  C   GLN A 110     -16.693  14.401 -20.699  1.00 78.24      A    C  \\nATOM    896  O   GLN A 110     -17.231  13.448 -20.133  1.00 75.78      A    O  \\nATOM    897  CB  GLN A 110     -14.548  13.388 -21.498  1.00 74.96      A    C  \\nATOM    898  CG  GLN A 110     -13.059  13.517 -21.716  1.00 68.17      A    C  \\nATOM    899  CD  GLN A 110     -12.484  12.365 -22.522  1.00 62.44      A    C  \\nATOM    900  NE2 GLN A 110     -11.207  12.463 -22.880  1.00 54.73      A    N  \\nATOM    901  OE1 GLN A 110     -13.189  11.403 -22.800  1.00 57.81      A    O  \\nATOM    902  N   GLY A 111     -17.330  15.372 -21.276  1.00 78.11      A    N  \\nATOM    903  CA  GLY A 111     -18.762  15.384 -21.361  1.00 77.63      A    C  \\nATOM    904  C   GLY A 111     -19.471  16.099 -20.237  1.00 79.18      A    C  \\nATOM    905  O   GLY A 111     -20.680  16.310 -20.300  1.00 77.04      A    O  \\nATOM    906  N   TYR A 112     -18.737  16.447 -19.195  1.00 80.06      A    N  \\nATOM    907  CA  TYR A 112     -19.292  17.116 -18.042  1.00 80.98      A    C  \\nATOM    908  C   TYR A 112     -18.666  18.477 -17.876  1.00 79.93      A    C  \\nATOM    909  O   TYR A 112     -17.515  18.711 -18.225  1.00 77.96      A    O  \\nATOM    910  CB  TYR A 112     -19.095  16.285 -16.770  1.00 80.20      A    C  \\nATOM    911  CG  TYR A 112     -19.864  14.984 -16.794  1.00 80.96      A    C  \\nATOM    912  CD1 TYR A 112     -21.189  14.933 -16.403  1.00 80.43      A    C  \\nATOM    913  CD2 TYR A 112     -19.259  13.795 -17.202  1.00 80.67      A    C  \\nATOM    914  CE1 TYR A 112     -21.895  13.746 -16.417  1.00 79.31      A    C  \\nATOM    915  CE2 TYR A 112     -19.952  12.607 -17.222  1.00 80.26      A    C  \\nATOM    916  CZ  TYR A 112     -21.271  12.606 -16.829  1.00 79.55      A    C  \\nATOM    917  OH  TYR A 112     -21.980  11.419 -16.873  1.00 76.11      A    O  \\nATOM    918  N   SER A 113     -19.443  19.386 -17.306  1.00 80.47      A    N  \\nATOM    919  CA  SER A 113     -19.005  20.770 -17.096  1.00 80.27      A    C  \\nATOM    920  C   SER A 113     -18.318  20.880 -15.750  1.00 80.90      A    C  \\nATOM    921  O   SER A 113     -18.162  19.909 -14.997  1.00 79.25      A    O  \\nATOM    922  CB  SER A 113     -20.201  21.713 -17.177  1.00 76.90      A    C  \\nATOM    923  OG  SER A 113     -21.130  21.454 -16.146  1.00 70.73      A    O  \\nATOM    924  N   LEU A 114     -17.893  22.094 -15.427  1.00 80.74      A    N  \\nATOM    925  CA  LEU A 114     -17.199  22.381 -14.186  1.00 80.94      A    C  \\nATOM    926  C   LEU A 114     -17.602  23.766 -13.696  1.00 79.85      A    C  \\nATOM    927  O   LEU A 114     -17.569  24.719 -14.479  1.00 76.62      A    O  \\nATOM    928  CB  LEU A 114     -15.691  22.281 -14.358  1.00 79.88      A    C  \\nATOM    929  CG  LEU A 114     -14.809  22.509 -13.113  1.00 79.41      A    C  \\nATOM    930  CD1 LEU A 114     -15.060  21.416 -12.087  1.00 79.69      A    C  \\nATOM    931  CD2 LEU A 114     -13.332  22.566 -13.493  1.00 79.48      A    C  \\nATOM    932  N   ASN A 115     -17.961  23.844 -12.445  1.00 81.50      A    N  \\nATOM    933  CA  ASN A 115     -18.353  25.109 -11.859  1.00 80.60      A    C  \\nATOM    934  C   ASN A 115     -17.137  25.968 -11.556  1.00 79.84      A    C  \\nATOM    935  O   ASN A 115     -16.004  25.469 -11.473  1.00 77.40      A    O  \\nATOM    936  CB  ASN A 115     -19.180  24.915 -10.581  1.00 78.98      A    C  \\nATOM    937  CG  ASN A 115     -20.556  24.336 -10.870  1.00 76.80      A    C  \\nATOM    938  ND2 ASN A 115     -21.006  23.429 -10.017  1.00 71.36      A    N  \\nATOM    939  OD1 ASN A 115     -21.205  24.686 -11.850  1.00 70.51      A    O  \\nATOM    940  N   LEU A 116     -17.383  27.247 -11.362  1.00 75.70      A    N  \\nATOM    941  CA  LEU A 116     -16.297  28.178 -11.079  1.00 75.16      A    C  \\nATOM    942  C   LEU A 116     -15.946  28.133  -9.589  1.00 75.63      A    C  \\nATOM    943  O   LEU A 116     -16.816  27.877  -8.736  1.00 74.27      A    O  \\nATOM    944  CB  LEU A 116     -16.687  29.600 -11.459  1.00 71.97      A    C  \\nATOM    945  CG  LEU A 116     -16.861  29.891 -12.943  1.00 69.28      A    C  \\nATOM    946  CD1 LEU A 116     -15.558  29.691 -13.676  1.00 67.97      A    C  \\nATOM    947  CD2 LEU A 116     -17.383  31.296 -13.171  1.00 67.43      A    C  \\nATOM    948  N   PRO A 117     -14.689  28.377  -9.241  1.00 77.57      A    N  \\nATOM    949  CA  PRO A 117     -14.319  28.408  -7.815  1.00 76.94      A    C  \\nATOM    950  C   PRO A 117     -14.841  29.678  -7.146  1.00 76.67      A    C  \\nATOM    951  O   PRO A 117     -15.059  30.689  -7.805  1.00 74.60      A    O  \\nATOM    952  CB  PRO A 117     -12.808  28.369  -7.856  1.00 75.11      A    C  \\nATOM    953  CG  PRO A 117     -12.460  28.968  -9.176  1.00 73.88      A    C  \\nATOM    954  CD  PRO A 117     -13.538  28.495 -10.096  1.00 76.27      A    C  \\nATOM    955  N   PRO A 118     -15.054  29.625  -5.838  1.00 79.55      A    N  \\nATOM    956  CA  PRO A 118     -15.559  30.810  -5.136  1.00 78.86      A    C  \\nATOM    957  C   PRO A 118     -14.456  31.862  -4.987  1.00 77.65      A    C  \\nATOM    958  O   PRO A 118     -13.274  31.579  -5.149  1.00 75.20      A    O  \\nATOM    959  CB  PRO A 118     -15.995  30.260  -3.775  1.00 78.32      A    C  \\nATOM    960  CG  PRO A 118     -15.129  29.096  -3.590  1.00 78.87      A    C  \\nATOM    961  CD  PRO A 118     -14.958  28.480  -4.936  1.00 81.76      A    C  \\nATOM    962  N   LYS A 119     -14.878  33.081  -4.669  1.00 72.67      A    N  \\nATOM    963  CA  LYS A 119     -13.942  34.205  -4.537  1.00 71.08      A    C  \\nATOM    964  C   LYS A 119     -13.757  34.673  -3.086  1.00 71.07      A    C  \\nATOM    965  O   LYS A 119     -12.736  35.263  -2.766  1.00 67.34      A    O  \\nATOM    966  CB  LYS A 119     -14.393  35.388  -5.393  1.00 66.92      A    C  \\nATOM    967  CG  LYS A 119     -14.386  35.087  -6.881  1.00 61.96      A    C  \\nATOM    968  CD  LYS A 119     -14.881  36.287  -7.690  1.00 56.94      A    C  \\nATOM    969  CE  LYS A 119     -14.894  35.984  -9.153  1.00 51.38      A    C  \\nATOM    970  NZ  LYS A 119     -15.400  37.103  -9.957  1.00 44.79      A    N1+\\nATOM    971  N   ASP A 120     -14.752  34.392  -2.265  1.00 74.15      A    N  \\nATOM    972  CA  ASP A 120     -14.688  34.854  -0.879  1.00 73.76      A    C  \\nATOM    973  C   ASP A 120     -13.915  33.861  -0.010  1.00 74.28      A    C  \\nATOM    974  O   ASP A 120     -12.855  34.171   0.521  1.00 72.08      A    O  \\nATOM    975  CB  ASP A 120     -16.091  35.075  -0.323  1.00 69.80      A    C  \\nATOM    976  CG  ASP A 120     -16.820  36.199  -1.020  1.00 64.96      A    C  \\nATOM    977  OD1 ASP A 120     -16.141  37.164  -1.452  1.00 59.24      A    O  \\nATOM    978  OD2 ASP A 120     -18.062  36.150  -1.095  1.00 63.94      A    O1-\\nATOM    979  N   ARG A 121     -14.469  32.670   0.132  1.00 76.73      A    N  \\nATOM    980  CA  ARG A 121     -13.866  31.624   0.936  1.00 77.93      A    C  \\nATOM    981  C   ARG A 121     -12.824  30.871   0.106  1.00 78.94      A    C  \\nATOM    982  O   ARG A 121     -12.917  30.852  -1.125  1.00 78.50      A    O  \\nATOM    983  CB  ARG A 121     -14.942  30.694   1.462  1.00 76.55      A    C  \\nATOM    984  CG  ARG A 121     -15.729  29.992   0.390  1.00 74.83      A    C  \\nATOM    985  CD  ARG A 121     -16.808  29.123   0.989  1.00 74.67      A    C  \\nATOM    986  NE  ARG A 121     -17.567  28.367  -0.025  1.00 71.43      A    N  \\nATOM    987  CZ  ARG A 121     -18.586  27.575   0.217  1.00 72.42      A    C  \\nATOM    988  NH1 ARG A 121     -19.026  27.425   1.439  1.00 61.81      A    N1+\\nATOM    989  NH2 ARG A 121     -19.183  26.961  -0.791  1.00 65.73      A    N  \\nATOM    990  N   PRO A 122     -11.852  30.247   0.757  1.00 79.88      A    N  \\nATOM    991  CA  PRO A 122     -10.875  29.466  -0.001  1.00 80.37      A    C  \\nATOM    992  C   PRO A 122     -11.493  28.179  -0.554  1.00 82.44      A    C  \\nATOM    993  O   PRO A 122     -12.542  27.727  -0.094  1.00 81.54      A    O  \\nATOM    994  CB  PRO A 122      -9.791  29.144   1.035  1.00 77.55      A    C  \\nATOM    995  CG  PRO A 122     -10.502  29.226   2.318  1.00 74.78      A    C  \\nATOM    996  CD  PRO A 122     -11.504  30.316   2.176  1.00 76.97      A    C  \\nATOM    997  N   ILE A 123     -10.829  27.632  -1.533  1.00 83.65      A    N  \\nATOM    998  CA  ILE A 123     -11.341  26.480  -2.242  1.00 85.51      A    C  \\nATOM    999  C   ILE A 123     -11.089  25.235  -1.401  1.00 86.91      A    C  \\nATOM   1000  O   ILE A 123      -9.952  24.941  -1.025  1.00 85.93      A    O  \\nATOM   1001  CB  ILE A 123     -10.690  26.333  -3.627  1.00 83.90      A    C  \\nATOM   1002  CG1 ILE A 123     -10.972  27.574  -4.445  1.00 81.35      A    C  \\nATOM   1003  CG2 ILE A 123     -11.224  25.101  -4.360  1.00 82.91      A    C  \\nATOM   1004  CD1 ILE A 123     -10.188  27.658  -5.704  1.00 77.22      A    C  \\nATOM   1005  N   ARG A 124     -12.127  24.484  -1.108  1.00 89.19      A    N  \\nATOM   1006  CA  ARG A 124     -12.038  23.288  -0.301  1.00 90.59      A    C  \\nATOM   1007  C   ARG A 124     -12.217  22.075  -1.204  1.00 91.25      A    C  \\nATOM   1008  O   ARG A 124     -13.310  21.832  -1.725  1.00 90.70      A    O  \\nATOM   1009  CB  ARG A 124     -13.114  23.307   0.803  1.00 90.67      A    C  \\nATOM   1010  CG  ARG A 124     -12.928  24.438   1.810  1.00 88.53      A    C  \\nATOM   1011  CD  ARG A 124     -14.048  24.482   2.833  1.00 86.72      A    C  \\nATOM   1012  NE  ARG A 124     -14.003  23.284   3.683  1.00 81.43      A    N  \\nATOM   1013  CZ  ARG A 124     -13.289  23.203   4.797  1.00 80.88      A    C  \\nATOM   1014  NH1 ARG A 124     -12.572  24.240   5.188  1.00 69.83      A    N1+\\nATOM   1015  NH2 ARG A 124     -13.319  22.097   5.508  1.00 71.58      A    N  \\nATOM   1016  N   ILE A 125     -11.154  21.342  -1.393  1.00 92.65      A    N  \\nATOM   1017  CA  ILE A 125     -11.158  20.170  -2.260  1.00 92.72      A    C  \\nATOM   1018  C   ILE A 125     -11.149  18.930  -1.397  1.00 93.42      A    C  \\nATOM   1019  O   ILE A 125     -10.269  18.753  -0.549  1.00 92.65      A    O  \\nATOM   1020  CB  ILE A 125      -9.953  20.177  -3.228  1.00 91.56      A    C  \\nATOM   1021  CG1 ILE A 125     -10.032  21.376  -4.155  1.00 89.83      A    C  \\nATOM   1022  CG2 ILE A 125      -9.916  18.884  -4.029  1.00 90.56      A    C  \\nATOM   1023  CD1 ILE A 125      -8.825  21.533  -5.019  1.00 86.82      A    C  \\nATOM   1024  N   TYR A 126     -12.090  18.066  -1.613  1.00 92.99      A    N  \\nATOM   1025  CA  TYR A 126     -12.221  16.821  -0.861  1.00 93.54      A    C  \\nATOM   1026  C   TYR A 126     -11.576  15.704  -1.676  1.00 93.22      A    C  \\nATOM   1027  O   TYR A 126     -11.962  15.447  -2.806  1.00 91.91      A    O  \\nATOM   1028  CB  TYR A 126     -13.676  16.506  -0.548  1.00 93.41      A    C  \\nATOM   1029  CG  TYR A 126     -13.882  15.163   0.102  1.00 93.89      A    C  \\nATOM   1030  CD1 TYR A 126     -13.597  14.975   1.443  1.00 92.71      A    C  \\nATOM   1031  CD2 TYR A 126     -14.380  14.096  -0.644  1.00 92.25      A    C  \\nATOM   1032  CE1 TYR A 126     -13.791  13.739   2.037  1.00 91.71      A    C  \\nATOM   1033  CE2 TYR A 126     -14.568  12.879  -0.046  1.00 91.48      A    C  \\nATOM   1034  CZ  TYR A 126     -14.283  12.703   1.291  1.00 91.54      A    C  \\nATOM   1035  OH  TYR A 126     -14.489  11.490   1.874  1.00 89.12      A    O  \\nATOM   1036  N   ALA A 127     -10.583  15.061  -1.098  1.00 93.90      A    N  \\nATOM   1037  CA  ALA A 127      -9.893  13.957  -1.758  1.00 93.04      A    C  \\nATOM   1038  C   ALA A 127     -10.427  12.638  -1.205  1.00 91.86      A    C  \\nATOM   1039  O   ALA A 127     -10.232  12.299  -0.032  1.00 89.94      A    O  \\nATOM   1040  CB  ALA A 127      -8.381  14.049  -1.556  1.00 92.36      A    C  \\nATOM   1041  N   ASP A 128     -11.090  11.904  -2.068  1.00 88.62      A    N  \\nATOM   1042  CA  ASP A 128     -11.721  10.640  -1.688  1.00 86.17      A    C  \\nATOM   1043  C   ASP A 128     -10.764   9.506  -1.998  1.00 85.24      A    C  \\nATOM   1044  O   ASP A 128     -10.273   9.373  -3.115  1.00 80.58      A    O  \\nATOM   1045  CB  ASP A 128     -13.030  10.448  -2.415  1.00 82.21      A    C  \\nATOM   1046  CG  ASP A 128     -13.860   9.303  -1.868  1.00 78.96      A    C  \\nATOM   1047  OD1 ASP A 128     -13.408   8.647  -0.889  1.00 70.75      A    O  \\nATOM   1048  OD2 ASP A 128     -14.948   9.041  -2.406  1.00 72.99      A    O1-\\nATOM   1049  N   GLY A 129     -10.524   8.682  -1.013  1.00 89.01      A    N  \\nATOM   1050  CA  GLY A 129      -9.615   7.551  -1.194  1.00 88.31      A    C  \\nATOM   1051  C   GLY A 129      -9.534   6.725   0.079  1.00 89.35      A    C  \\nATOM   1052  O   GLY A 129      -9.901   7.185   1.169  1.00 87.97      A    O  \\nATOM   1053  N   VAL A 130      -9.065   5.515  -0.073  1.00 88.55      A    N  \\nATOM   1054  CA  VAL A 130      -8.858   4.649   1.087  1.00 89.18      A    C  \\nATOM   1055  C   VAL A 130      -7.577   4.999   1.794  1.00 90.53      A    C  \\nATOM   1056  O   VAL A 130      -7.531   5.026   3.014  1.00 88.69      A    O  \\nATOM   1057  CB  VAL A 130      -8.888   3.172   0.662  1.00 87.15      A    C  \\nATOM   1058  CG1 VAL A 130     -10.213   2.862  -0.042  1.00 84.22      A    C  \\nATOM   1059  CG2 VAL A 130      -8.682   2.269   1.844  1.00 84.65      A    C  \\nATOM   1060  N   PHE A 131      -6.545   5.302   1.031  1.00 89.84      A    N  \\nATOM   1061  CA  PHE A 131      -5.232   5.690   1.526  1.00 90.84      A    C  \\nATOM   1062  C   PHE A 131      -4.597   4.602   2.375  1.00 90.76      A    C  \\nATOM   1063  O   PHE A 131      -3.980   4.871   3.385  1.00 86.98      A    O  \\nATOM   1064  CB  PHE A 131      -5.267   7.027   2.272  1.00 91.44      A    C  \\nATOM   1065  CG  PHE A 131      -5.724   8.196   1.418  1.00 92.86      A    C  \\nATOM   1066  CD1 PHE A 131      -4.860   8.780   0.517  1.00 91.32      A    C  \\nATOM   1067  CD2 PHE A 131      -7.022   8.645   1.492  1.00 91.63      A    C  \\nATOM   1068  CE1 PHE A 131      -5.269   9.828  -0.265  1.00 90.23      A    C  \\nATOM   1069  CE2 PHE A 131      -7.433   9.710   0.705  1.00 90.98      A    C  \\nATOM   1070  CZ  PHE A 131      -6.568  10.290  -0.179  1.00 90.23      A    C  \\nATOM   1071  N   ASP A 132      -4.781   3.357   1.936  1.00 89.51      A    N  \\nATOM   1072  CA  ASP A 132      -4.130   2.244   2.600  1.00 88.82      A    C  \\nATOM   1073  C   ASP A 132      -2.722   2.086   2.022  1.00 89.04      A    C  \\nATOM   1074  O   ASP A 132      -2.536   2.085   0.817  1.00 86.98      A    O  \\nATOM   1075  CB  ASP A 132      -4.947   0.969   2.399  1.00 87.50      A    C  \\nATOM   1076  CG  ASP A 132      -4.466  -0.199   3.239  1.00 87.90      A    C  \\nATOM   1077  OD1 ASP A 132      -3.662   0.040   4.172  1.00 82.93      A    O  \\nATOM   1078  OD2 ASP A 132      -4.896  -1.335   2.973  1.00 83.03      A    O1-\\nATOM   1079  N   LEU A 133      -1.751   1.963   2.920  1.00 91.30      A    N  \\nATOM   1080  CA  LEU A 133      -0.326   1.873   2.544  1.00 91.04      A    C  \\nATOM   1081  C   LEU A 133       0.054   3.047   1.661  1.00 90.65      A    C  \\nATOM   1082  O   LEU A 133       0.233   2.916   0.446  1.00 87.17      A    O  \\nATOM   1083  CB  LEU A 133      -0.028   0.543   1.844  1.00 89.71      A    C  \\nATOM   1084  CG  LEU A 133      -0.187  -0.702   2.716  1.00 89.36      A    C  \\nATOM   1085  CD1 LEU A 133       0.861  -0.707   3.815  1.00 88.25      A    C  \\nATOM   1086  CD2 LEU A 133      -0.100  -1.944   1.872  1.00 87.84      A    C  \\nATOM   1087  N   PHE A 134       0.189   4.192   2.313  1.00 92.69      A    N  \\nATOM   1088  CA  PHE A 134       0.399   5.457   1.623  1.00 92.54      A    C  \\nATOM   1089  C   PHE A 134       1.669   5.450   0.782  1.00 92.18      A    C  \\nATOM   1090  O   PHE A 134       2.771   5.368   1.290  1.00 89.20      A    O  \\nATOM   1091  CB  PHE A 134       0.434   6.591   2.679  1.00 91.61      A    C  \\nATOM   1092  CG  PHE A 134       0.391   7.981   2.071  1.00 92.68      A    C  \\nATOM   1093  CD1 PHE A 134      -0.800   8.539   1.686  1.00 90.37      A    C  \\nATOM   1094  CD2 PHE A 134       1.559   8.721   1.938  1.00 90.03      A    C  \\nATOM   1095  CE1 PHE A 134      -0.857   9.800   1.167  1.00 88.01      A    C  \\nATOM   1096  CE2 PHE A 134       1.511   9.997   1.413  1.00 88.23      A    C  \\nATOM   1097  CZ  PHE A 134       0.303  10.544   1.032  1.00 88.42      A    C  \\nATOM   1098  N   HIS A 135       1.474   5.491  -0.527  1.00 91.26      A    N  \\nATOM   1099  CA  HIS A 135       2.562   5.347  -1.468  1.00 89.96      A    C  \\nATOM   1100  C   HIS A 135       2.771   6.671  -2.226  1.00 89.31      A    C  \\nATOM   1101  O   HIS A 135       2.238   7.710  -1.857  1.00 89.33      A    O  \\nATOM   1102  CB  HIS A 135       2.311   4.187  -2.435  1.00 87.96      A    C  \\nATOM   1103  CG  HIS A 135       0.987   4.221  -3.109  1.00 90.50      A    C  \\nATOM   1104  CD2 HIS A 135      -0.192   3.628  -2.791  1.00 83.90      A    C  \\nATOM   1105  ND1 HIS A 135       0.728   4.972  -4.244  1.00 84.74      A    N  \\nATOM   1106  CE1 HIS A 135      -0.527   4.825  -4.591  1.00 83.82      A    C  \\nATOM   1107  NE2 HIS A 135      -1.115   4.010  -3.736  1.00 85.67      A    N  \\nATOM   1108  N   LEU A 136       3.542   6.598  -3.283  1.00 90.64      A    N  \\nATOM   1109  CA  LEU A 136       3.854   7.780  -4.072  1.00 88.68      A    C  \\nATOM   1110  C   LEU A 136       2.644   8.264  -4.895  1.00 88.68      A    C  \\nATOM   1111  O   LEU A 136       2.537   9.457  -5.180  1.00 87.12      A    O  \\nATOM   1112  CB  LEU A 136       5.052   7.499  -4.973  1.00 86.01      A    C  \\nATOM   1113  CG  LEU A 136       5.606   8.632  -5.816  1.00 81.23      A    C  \\nATOM   1114  CD1 LEU A 136       6.133   9.750  -4.919  1.00 81.57      A    C  \\nATOM   1115  CD2 LEU A 136       6.730   8.095  -6.689  1.00 80.45      A    C  \\nATOM   1116  N   GLY A 137       1.798   7.326  -5.270  1.00 89.22      A    N  \\nATOM   1117  CA  GLY A 137       0.624   7.713  -6.049  1.00 88.84      A    C  \\nATOM   1118  C   GLY A 137      -0.329   8.616  -5.302  1.00 89.81      A    C  \\nATOM   1119  O   GLY A 137      -0.940   9.489  -5.889  1.00 88.52      A    O  \\nATOM   1120  N   HIS A 138      -0.442   8.373  -3.995  1.00 90.70      A    N  \\nATOM   1121  CA  HIS A 138      -1.285   9.261  -3.197  1.00 90.50      A    C  \\nATOM   1122  C   HIS A 138      -0.641  10.633  -3.020  1.00 90.43      A    C  \\nATOM   1123  O   HIS A 138      -1.344  11.628  -2.924  1.00 90.18      A    O  \\nATOM   1124  CB  HIS A 138      -1.585   8.609  -1.868  1.00 91.30      A    C  \\nATOM   1125  CG  HIS A 138      -2.423   7.391  -1.939  1.00 92.64      A    C  \\nATOM   1126  CD2 HIS A 138      -3.760   7.214  -1.955  1.00 88.95      A    C  \\nATOM   1127  ND1 HIS A 138      -1.873   6.133  -2.039  1.00 87.65      A    N  \\nATOM   1128  CE1 HIS A 138      -2.846   5.248  -2.089  1.00 88.28      A    C  \\nATOM   1129  NE2 HIS A 138      -4.004   5.886  -2.044  1.00 88.52      A    N  \\nATOM   1130  N   MET A 139       0.662  10.655  -2.915  1.00 90.83      A    N  \\nATOM   1131  CA  MET A 139       1.342  11.939  -2.756  1.00 89.52      A    C  \\nATOM   1132  C   MET A 139       1.278  12.788  -4.010  1.00 88.80      A    C  \\nATOM   1133  O   MET A 139       1.215  14.019  -3.941  1.00 88.27      A    O  \\nATOM   1134  CB  MET A 139       2.812  11.736  -2.342  1.00 88.90      A    C  \\nATOM   1135  CG  MET A 139       2.967  11.098  -0.953  1.00 80.57      A    C  \\nATOM   1136  SD  MET A 139       4.693  10.951  -0.433  1.00 68.07      A    S  \\nATOM   1137  CE  MET A 139       5.209   9.543  -1.391  1.00 60.92      A    C  \\nATOM   1138  N   LYS A 140       1.282  12.116  -5.163  1.00 87.96      A    N  \\nATOM   1139  CA  LYS A 140       1.150  12.864  -6.421  1.00 86.28      A    C  \\nATOM   1140  C   LYS A 140      -0.272  13.315  -6.662  1.00 86.34      A    C  \\nATOM   1141  O   LYS A 140      -0.511  14.357  -7.291  1.00 85.21      A    O  \\nATOM   1142  CB  LYS A 140       1.634  12.008  -7.594  1.00 83.93      A    C  \\nATOM   1143  CG  LYS A 140       3.134  11.777  -7.617  1.00 81.69      A    C  \\nATOM   1144  CD  LYS A 140       3.557  10.978  -8.845  1.00 77.28      A    C  \\nATOM   1145  CE  LYS A 140       5.040  10.747  -8.880  1.00 73.42      A    C  \\nATOM   1146  NZ  LYS A 140       5.449   9.926 -10.055  1.00 65.90      A    N1+\\nATOM   1147  N   GLN A 141      -1.238  12.525  -6.185  1.00 89.23      A    N  \\nATOM   1148  CA  GLN A 141      -2.639  12.919  -6.343  1.00 89.01      A    C  \\nATOM   1149  C   GLN A 141      -2.976  14.102  -5.430  1.00 89.20      A    C  \\nATOM   1150  O   GLN A 141      -3.682  15.019  -5.824  1.00 88.50      A    O  \\nATOM   1151  CB  GLN A 141      -3.553  11.742  -6.036  1.00 88.51      A    C  \\nATOM   1152  CG  GLN A 141      -5.034  12.051  -6.263  1.00 83.54      A    C  \\nATOM   1153  CD  GLN A 141      -5.932  10.869  -5.980  1.00 81.70      A    C  \\nATOM   1154  NE2 GLN A 141      -7.218  11.034  -6.210  1.00 72.78      A    N  \\nATOM   1155  OE1 GLN A 141      -5.445   9.821  -5.558  1.00 75.54      A    O  \\nATOM   1156  N   LEU A 142      -2.458  14.063  -4.208  1.00 91.74      A    N  \\nATOM   1157  CA  LEU A 142      -2.697  15.162  -3.282  1.00 91.84      A    C  \\nATOM   1158  C   LEU A 142      -1.930  16.411  -3.691  1.00 90.95      A    C  \\nATOM   1159  O   LEU A 142      -2.351  17.525  -3.383  1.00 89.68      A    O  \\nATOM   1160  CB  LEU A 142      -2.348  14.756  -1.856  1.00 92.72      A    C  \\nATOM   1161  CG  LEU A 142      -3.220  13.661  -1.242  1.00 93.18      A    C  \\nATOM   1162  CD1 LEU A 142      -4.648  14.124  -1.110  1.00 92.13      A    C  \\nATOM   1163  CD2 LEU A 142      -2.673  13.222   0.105  1.00 92.70      A    C  \\nATOM   1164  N   GLU A 143      -0.817  16.219  -4.354  1.00 88.78      A    N  \\nATOM   1165  CA  GLU A 143      -0.074  17.358  -4.880  1.00 87.20      A    C  \\nATOM   1166  C   GLU A 143      -0.837  18.036  -6.025  1.00 86.57      A    C  \\nATOM   1167  O   GLU A 143      -0.850  19.263  -6.105  1.00 85.16      A    O  \\nATOM   1168  CB  GLU A 143       1.322  16.918  -5.335  1.00 85.27      A    C  \\nATOM   1169  CG  GLU A 143       2.208  18.056  -5.787  1.00 83.37      A    C  \\nATOM   1170  CD  GLU A 143       3.577  17.596  -6.239  1.00 80.96      A    C  \\nATOM   1171  OE1 GLU A 143       3.840  16.382  -6.165  1.00 76.36      A    O  \\nATOM   1172  OE2 GLU A 143       4.409  18.438  -6.633  1.00 73.12      A    O1-\\nATOM   1173  N   GLN A 144      -1.475  17.234  -6.857  1.00 88.12      A    N  \\nATOM   1174  CA  GLN A 144      -2.293  17.808  -7.941  1.00 87.64      A    C  \\nATOM   1175  C   GLN A 144      -3.541  18.469  -7.400  1.00 87.82      A    C  \\nATOM   1176  O   GLN A 144      -4.044  19.411  -8.017  1.00 87.34      A    O  \\nATOM   1177  CB  GLN A 144      -2.661  16.706  -8.928  1.00 86.88      A    C  \\nATOM   1178  CG  GLN A 144      -1.509  16.149  -9.724  1.00 84.51      A    C  \\nATOM   1179  CD  GLN A 144      -1.920  15.018 -10.652  1.00 82.90      A    C  \\nATOM   1180  NE2 GLN A 144      -0.977  14.484 -11.394  1.00 74.14      A    N  \\nATOM   1181  OE1 GLN A 144      -3.089  14.644 -10.717  1.00 76.58      A    O  \\nATOM   1182  N   CYS A 145      -4.040  17.998  -6.271  1.00 89.09      A    N  \\nATOM   1183  CA  CYS A 145      -5.185  18.651  -5.666  1.00 89.21      A    C  \\nATOM   1184  C   CYS A 145      -4.818  19.978  -5.033  1.00 87.91      A    C  \\nATOM   1185  O   CYS A 145      -5.560  20.939  -5.081  1.00 85.57      A    O  \\nATOM   1186  CB  CYS A 145      -5.822  17.736  -4.619  1.00 90.28      A    C  \\nATOM   1187  SG  CYS A 145      -6.519  16.221  -5.303  1.00 89.50      A    S  \\nATOM   1188  N   LYS A 146      -3.623  20.028  -4.428  1.00 87.62      A    N  \\nATOM   1189  CA  LYS A 146      -3.206  21.252  -3.751  1.00 85.58      A    C  \\nATOM   1190  C   LYS A 146      -2.745  22.308  -4.738  1.00 84.46      A    C  \\nATOM   1191  O   LYS A 146      -3.000  23.484  -4.533  1.00 81.98      A    O  \\nATOM   1192  CB  LYS A 146      -2.083  20.927  -2.762  1.00 84.29      A    C  \\nATOM   1193  CG  LYS A 146      -1.679  22.084  -1.892  1.00 82.46      A    C  \\nATOM   1194  CD  LYS A 146      -2.798  22.545  -0.980  1.00 81.66      A    C  \\nATOM   1195  CE  LYS A 146      -2.354  23.662  -0.066  1.00 80.32      A    C  \\nATOM   1196  NZ  LYS A 146      -3.474  24.153   0.774  1.00 76.88      A    N1+\\nATOM   1197  N   LYS A 147      -2.122  21.889  -5.792  1.00 81.52      A    N  \\nATOM   1198  CA  LYS A 147      -1.579  22.799  -6.803  1.00 79.98      A    C  \\nATOM   1199  C   LYS A 147      -2.555  23.098  -7.939  1.00 79.96      A    C  \\nATOM   1200  O   LYS A 147      -2.141  23.608  -8.972  1.00 77.83      A    O  \\nATOM   1201  CB  LYS A 147      -0.266  22.288  -7.354  1.00 77.99      A    C  \\nATOM   1202  CG  LYS A 147       0.862  22.222  -6.351  1.00 74.35      A    C  \\nATOM   1203  CD  LYS A 147       2.125  21.646  -6.968  1.00 70.97      A    C  \\nATOM   1204  CE  LYS A 147       3.265  21.610  -5.977  1.00 66.82      A    C  \\nATOM   1205  NZ  LYS A 147       4.474  21.003  -6.548  1.00 59.13      A    N1+\\nATOM   1206  N   SER A 148      -3.812  22.759  -7.746  1.00 81.66      A    N  \\nATOM   1207  CA  SER A 148      -4.802  23.014  -8.792  1.00 80.72      A    C  \\nATOM   1208  C   SER A 148      -5.273  24.442  -8.775  1.00 78.91      A    C  \\nATOM   1209  O   SER A 148      -5.241  25.138  -9.792  1.00 75.69      A    O  \\nATOM   1210  CB  SER A 148      -5.974  22.056  -8.616  1.00 80.76      A    C  \\nATOM   1211  OG  SER A 148      -6.639  22.253  -7.375  1.00 75.68      A    O  \\nATOM   1212  N   PHE A 149      -5.704  24.900  -7.612  1.00 79.11      A    N  \\nATOM   1213  CA  PHE A 149      -6.211  26.252  -7.435  1.00 78.73      A    C  \\nATOM   1214  C   PHE A 149      -5.426  26.970  -6.358  1.00 77.93      A    C  \\nATOM   1215  O   PHE A 149      -4.885  26.319  -5.464  1.00 76.13      A    O  \\nATOM   1216  CB  PHE A 149      -7.687  26.235  -7.051  1.00 78.15      A    C  \\nATOM   1217  CG  PHE A 149      -8.607  25.641  -8.101  1.00 79.05      A    C  \\nATOM   1218  CD1 PHE A 149      -9.020  26.383  -9.186  1.00 79.09      A    C  \\nATOM   1219  CD2 PHE A 149      -9.032  24.323  -7.992  1.00 79.37      A    C  \\nATOM   1220  CE1 PHE A 149      -9.843  25.842 -10.133  1.00 76.95      A    C  \\nATOM   1221  CE2 PHE A 149      -9.865  23.779  -8.940  1.00 78.80      A    C  \\nATOM   1222  CZ  PHE A 149     -10.273  24.530 -10.021  1.00 76.81      A    C  \\nATOM   1223  N   PRO A 150      -5.345  28.285  -6.401  1.00 74.26      A    N  \\nATOM   1224  CA  PRO A 150      -4.696  29.018  -5.309  1.00 72.57      A    C  \\nATOM   1225  C   PRO A 150      -5.584  29.057  -4.056  1.00 72.53      A    C  \\nATOM   1226  O   PRO A 150      -6.805  29.074  -4.154  1.00 67.92      A    O  \\nATOM   1227  CB  PRO A 150      -4.504  30.426  -5.882  1.00 68.93      A    C  \\nATOM   1228  CG  PRO A 150      -5.583  30.557  -6.911  1.00 68.28      A    C  \\nATOM   1229  CD  PRO A 150      -5.710  29.174  -7.499  1.00 70.81      A    C  \\nATOM   1230  N   ASN A 151      -4.956  29.031  -2.916  1.00 76.24      A    N  \\nATOM   1231  CA  ASN A 151      -5.629  29.027  -1.632  1.00 75.91      A    C  \\nATOM   1232  C   ASN A 151      -6.564  27.840  -1.492  1.00 77.79      A    C  \\nATOM   1233  O   ASN A 151      -7.754  27.970  -1.258  1.00 75.60      A    O  \\nATOM   1234  CB  ASN A 151      -6.372  30.339  -1.360  1.00 71.09      A    C  \\nATOM   1235  CG  ASN A 151      -5.435  31.529  -1.178  1.00 63.30      A    C  \\nATOM   1236  ND2 ASN A 151      -5.798  32.648  -1.762  1.00 57.97      A    N  \\nATOM   1237  OD1 ASN A 151      -4.420  31.402  -0.532  1.00 55.83      A    O  \\nATOM   1238  N   VAL A 152      -6.002  26.663  -1.650  1.00 82.53      A    N  \\nATOM   1239  CA  VAL A 152      -6.769  25.424  -1.615  1.00 84.83      A    C  \\nATOM   1240  C   VAL A 152      -6.561  24.741  -0.275  1.00 85.19      A    C  \\nATOM   1241  O   VAL A 152      -5.416  24.513   0.150  1.00 83.27      A    O  \\nATOM   1242  CB  VAL A 152      -6.377  24.479  -2.757  1.00 84.64      A    C  \\nATOM   1243  CG1 VAL A 152      -6.764  25.108  -4.073  1.00 83.98      A    C  \\nATOM   1244  CG2 VAL A 152      -7.038  23.126  -2.598  1.00 83.44      A    C  \\nATOM   1245  N   THR A 153      -7.625  24.430   0.388  1.00 87.18      A    N  \\nATOM   1246  CA  THR A 153      -7.607  23.621   1.593  1.00 87.92      A    C  \\nATOM   1247  C   THR A 153      -7.969  22.204   1.201  1.00 89.21      A    C  \\nATOM   1248  O   THR A 153      -9.089  21.939   0.745  1.00 87.94      A    O  \\nATOM   1249  CB  THR A 153      -8.578  24.145   2.656  1.00 87.02      A    C  \\nATOM   1250  CG2 THR A 153      -8.255  25.567   3.017  1.00 81.51      A    C  \\nATOM   1251  OG1 THR A 153      -9.913  24.086   2.138  1.00 79.86      A    O  \\nATOM   1252  N   LEU A 154      -7.034  21.307   1.365  1.00 91.56      A    N  \\nATOM   1253  CA  LEU A 154      -7.232  19.917   0.976  1.00 92.21      A    C  \\nATOM   1254  C   LEU A 154      -7.784  19.121   2.150  1.00 92.23      A    C  \\nATOM   1255  O   LEU A 154      -7.202  19.128   3.247  1.00 91.14      A    O  \\nATOM   1256  CB  LEU A 154      -5.908  19.309   0.476  1.00 92.31      A    C  \\nATOM   1257  CG  LEU A 154      -5.989  17.867  -0.036  1.00 92.04      A    C  \\nATOM   1258  CD1 LEU A 154      -6.897  17.786  -1.244  1.00 90.43      A    C  \\nATOM   1259  CD2 LEU A 154      -4.599  17.365  -0.392  1.00 91.11      A    C  \\nATOM   1260  N   ILE A 155      -8.862  18.441   1.915  1.00 91.50      A    N  \\nATOM   1261  CA  ILE A 155      -9.539  17.645   2.928  1.00 91.33      A    C  \\nATOM   1262  C   ILE A 155      -9.552  16.205   2.445  1.00 91.73      A    C  \\nATOM   1263  O   ILE A 155     -10.050  15.906   1.365  1.00 90.75      A    O  \\nATOM   1264  CB  ILE A 155     -10.982  18.140   3.171  1.00 90.88      A    C  \\nATOM   1265  CG1 ILE A 155     -10.953  19.603   3.649  1.00 89.19      A    C  \\nATOM   1266  CG2 ILE A 155     -11.651  17.267   4.240  1.00 90.21      A    C  \\nATOM   1267  CD1 ILE A 155     -12.323  20.223   3.709  1.00 85.22      A    C  \\nATOM   1268  N   CYS A 156      -9.004  15.324   3.252  1.00 92.62      A    N  \\nATOM   1269  CA  CYS A 156      -8.925  13.906   2.912  1.00 92.63      A    C  \\nATOM   1270  C   CYS A 156      -9.872  13.109   3.780  1.00 91.83      A    C  \\nATOM   1271  O   CYS A 156      -9.778  13.135   5.023  1.00 90.21      A    O  \\nATOM   1272  CB  CYS A 156      -7.505  13.366   3.049  1.00 92.30      A    C  \\nATOM   1273  SG  CYS A 156      -6.389  14.048   1.796  1.00 89.41      A    S  \\nATOM   1274  N   GLY A 157     -10.765  12.395   3.167  1.00 92.79      A    N  \\nATOM   1275  CA  GLY A 157     -11.709  11.565   3.885  1.00 91.86      A    C  \\nATOM   1276  C   GLY A 157     -11.247  10.132   3.948  1.00 92.15      A    C  \\nATOM   1277  O   GLY A 157     -11.037   9.474   2.951  1.00 90.01      A    O  \\nATOM   1278  N   VAL A 158     -11.106   9.661   5.178  1.00 89.92      A    N  \\nATOM   1279  CA  VAL A 158     -10.694   8.293   5.457  1.00 89.88      A    C  \\nATOM   1280  C   VAL A 158     -11.920   7.502   5.864  1.00 89.40      A    C  \\nATOM   1281  O   VAL A 158     -12.498   7.762   6.920  1.00 88.13      A    O  \\nATOM   1282  CB  VAL A 158      -9.606   8.238   6.531  1.00 89.45      A    C  \\nATOM   1283  CG1 VAL A 158      -8.394   9.000   6.101  1.00 88.31      A    C  \\nATOM   1284  CG2 VAL A 158      -9.250   6.791   6.856  1.00 87.13      A    C  \\nATOM   1285  N   PRO A 159     -12.367   6.562   5.056  1.00 89.24      A    N  \\nATOM   1286  CA  PRO A 159     -13.495   5.731   5.454  1.00 88.03      A    C  \\nATOM   1287  C   PRO A 159     -13.111   4.730   6.522  1.00 87.59      A    C  \\nATOM   1288  O   PRO A 159     -11.948   4.449   6.740  1.00 86.09      A    O  \\nATOM   1289  CB  PRO A 159     -13.923   5.041   4.146  1.00 87.03      A    C  \\nATOM   1290  CG  PRO A 159     -12.666   4.975   3.365  1.00 86.59      A    C  \\nATOM   1291  CD  PRO A 159     -11.911   6.243   3.687  1.00 88.39      A    C  \\nATOM   1292  N   SER A 160     -14.125   4.210   7.189  1.00 86.04      A    N  \\nATOM   1293  CA  SER A 160     -13.918   3.266   8.260  1.00 84.76      A    C  \\nATOM   1294  C   SER A 160     -13.468   1.913   7.718  1.00 85.11      A    C  \\nATOM   1295  O   SER A 160     -13.533   1.650   6.524  1.00 83.56      A    O  \\nATOM   1296  CB  SER A 160     -15.164   3.114   9.106  1.00 83.09      A    C  \\nATOM   1297  OG  SER A 160     -16.195   2.521   8.350  1.00 77.61      A    O  \\nATOM   1298  N   ASP A 161     -13.028   1.062   8.634  1.00 84.56      A    N  \\nATOM   1299  CA  ASP A 161     -12.586  -0.260   8.230  1.00 82.69      A    C  \\nATOM   1300  C   ASP A 161     -13.757  -1.154   7.835  1.00 83.02      A    C  \\nATOM   1301  O   ASP A 161     -13.596  -2.060   7.015  1.00 82.14      A    O  \\nATOM   1302  CB  ASP A 161     -11.791  -0.923   9.337  1.00 81.93      A    C  \\nATOM   1303  CG  ASP A 161     -10.438  -0.264   9.582  1.00 82.07      A    C  \\nATOM   1304  OD1 ASP A 161      -9.860   0.278   8.604  1.00 78.05      A    O  \\nATOM   1305  OD2 ASP A 161      -9.958  -0.298  10.728  1.00 78.09      A    O1-\\nATOM   1306  N   LYS A 162     -14.928  -0.901   8.400  1.00 81.31      A    N  \\nATOM   1307  CA  LYS A 162     -16.093  -1.684   8.072  1.00 79.71      A    C  \\nATOM   1308  C   LYS A 162     -16.598  -1.407   6.666  1.00 80.10      A    C  \\nATOM   1309  O   LYS A 162     -17.043  -2.325   5.977  1.00 79.39      A    O  \\nATOM   1310  CB  LYS A 162     -17.213  -1.434   9.086  1.00 77.90      A    C  \\nATOM   1311  CG  LYS A 162     -16.896  -1.884  10.494  1.00 70.98      A    C  \\nATOM   1312  CD  LYS A 162     -18.065  -1.604  11.428  1.00 64.80      A    C  \\nATOM   1313  CE  LYS A 162     -17.699  -2.056  12.850  1.00 57.79      A    C  \\nATOM   1314  NZ  LYS A 162     -18.817  -1.839  13.794  1.00 49.52      A    N1+\\nATOM   1315  N   VAL A 163     -16.541  -0.166   6.252  1.00 79.83      A    N  \\nATOM   1316  CA  VAL A 163     -16.991   0.191   4.918  1.00 79.22      A    C  \\nATOM   1317  C   VAL A 163     -15.991  -0.293   3.876  1.00 79.26      A    C  \\nATOM   1318  O   VAL A 163     -16.378  -0.821   2.831  1.00 78.44      A    O  \\nATOM   1319  CB  VAL A 163     -17.214   1.714   4.821  1.00 78.13      A    C  \\nATOM   1320  CG1 VAL A 163     -18.215   2.172   5.850  1.00 75.73      A    C  \\nATOM   1321  CG2 VAL A 163     -17.680   2.103   3.430  1.00 76.30      A    C  \\nATOM   1322  N   THR A 164     -14.713  -0.143   4.147  1.00 82.87      A    N  \\nATOM   1323  CA  THR A 164     -13.692  -0.586   3.215  1.00 82.97      A    C  \\nATOM   1324  C   THR A 164     -13.628  -2.103   3.130  1.00 81.67      A    C  \\nATOM   1325  O   THR A 164     -13.347  -2.637   2.057  1.00 79.16      A    O  \\nATOM   1326  CB  THR A 164     -12.324  -0.036   3.578  1.00 83.99      A    C  \\nATOM   1327  CG2 THR A 164     -12.329   1.487   3.505  1.00 78.35      A    C  \\nATOM   1328  OG1 THR A 164     -11.983  -0.432   4.903  1.00 77.22      A    O  \\nATOM   1329  N   HIS A 165     -13.897  -2.793   4.246  1.00 82.37      A    N  \\nATOM   1330  CA  HIS A 165     -13.858  -4.241   4.210  1.00 80.31      A    C  \\nATOM   1331  C   HIS A 165     -15.080  -4.799   3.494  1.00 80.60      A    C  \\nATOM   1332  O   HIS A 165     -14.995  -5.866   2.880  1.00 78.79      A    O  \\nATOM   1333  CB  HIS A 165     -13.775  -4.811   5.632  1.00 78.58      A    C  \\nATOM   1334  CG  HIS A 165     -12.405  -4.607   6.266  1.00 67.88      A    C  \\nATOM   1335  CD2 HIS A 165     -11.294  -4.036   5.785  1.00 62.20      A    C  \\nATOM   1336  ND1 HIS A 165     -12.130  -5.039   7.539  1.00 61.80      A    N  \\nATOM   1337  CE1 HIS A 165     -10.864  -4.735   7.804  1.00 60.53      A    C  \\nATOM   1338  NE2 HIS A 165     -10.349  -4.133   6.773  1.00 57.65      A    N  \\nATOM   1339  N   LYS A 166     -16.197  -4.122   3.598  1.00 75.46      A    N  \\nATOM   1340  CA  LYS A 166     -17.412  -4.615   2.942  1.00 73.64      A    C  \\nATOM   1341  C   LYS A 166     -17.437  -4.279   1.467  1.00 74.27      A    C  \\nATOM   1342  O   LYS A 166     -17.835  -5.103   0.649  1.00 73.37      A    O  \\nATOM   1343  CB  LYS A 166     -18.641  -4.011   3.653  1.00 71.38      A    C  \\nATOM   1344  CG  LYS A 166     -19.950  -4.530   3.112  1.00 66.03      A    C  \\nATOM   1345  CD  LYS A 166     -21.110  -3.978   3.874  1.00 61.31      A    C  \\nATOM   1346  CE  LYS A 166     -22.437  -4.518   3.312  1.00 54.88      A    C  \\nATOM   1347  NZ  LYS A 166     -23.606  -3.970   4.020  1.00 48.47      A    N1+\\nATOM   1348  N   LEU A 167     -17.051  -3.080   1.118  1.00 76.15      A    N  \\nATOM   1349  CA  LEU A 167     -17.135  -2.656  -0.281  1.00 75.99      A    C  \\nATOM   1350  C   LEU A 167     -15.898  -3.059  -1.067  1.00 75.60      A    C  \\nATOM   1351  O   LEU A 167     -15.988  -3.776  -2.067  1.00 73.56      A    O  \\nATOM   1352  CB  LEU A 167     -17.331  -1.150  -0.367  1.00 74.83      A    C  \\nATOM   1353  CG  LEU A 167     -18.651  -0.607   0.234  1.00 74.33      A    C  \\nATOM   1354  CD1 LEU A 167     -19.851  -1.181  -0.486  1.00 73.24      A    C  \\nATOM   1355  CD2 LEU A 167     -18.647   0.903   0.193  1.00 72.64      A    C  \\nATOM   1356  N   LYS A 168     -14.753  -2.599  -0.616  1.00 76.58      A    N  \\nATOM   1357  CA  LYS A 168     -13.519  -2.846  -1.377  1.00 76.65      A    C  \\nATOM   1358  C   LYS A 168     -12.903  -4.179  -0.972  1.00 75.98      A    C  \\nATOM   1359  O   LYS A 168     -12.835  -5.106  -1.775  1.00 72.79      A    O  \\nATOM   1360  CB  LYS A 168     -12.524  -1.715  -1.181  1.00 74.98      A    C  \\nATOM   1361  CG  LYS A 168     -11.259  -1.842  -1.993  1.00 72.18      A    C  \\nATOM   1362  CD  LYS A 168     -10.374  -0.610  -1.802  1.00 70.76      A    C  \\nATOM   1363  CE  LYS A 168      -9.100  -0.702  -2.608  1.00 68.33      A    C  \\nATOM   1364  NZ  LYS A 168      -8.228   0.483  -2.452  1.00 61.91      A    N1+\\nATOM   1365  N   GLY A 169     -12.420  -4.258   0.239  1.00 79.12      A    N  \\nATOM   1366  CA  GLY A 169     -11.778  -5.465   0.740  1.00 78.46      A    C  \\nATOM   1367  C   GLY A 169     -11.047  -5.215   2.041  1.00 80.44      A    C  \\nATOM   1368  O   GLY A 169     -11.309  -4.237   2.738  1.00 78.57      A    O  \\nATOM   1369  N   LEU A 170     -10.130  -6.092   2.343  1.00 80.33      A    N  \\nATOM   1370  CA  LEU A 170      -9.355  -5.978   3.554  1.00 80.41      A    C  \\nATOM   1371  C   LEU A 170      -8.309  -4.902   3.407  1.00 82.57      A    C  \\nATOM   1372  O   LEU A 170      -7.775  -4.657   2.316  1.00 81.24      A    O  \\nATOM   1373  CB  LEU A 170      -8.690  -7.299   3.914  1.00 77.63      A    C  \\nATOM   1374  CG  LEU A 170      -9.621  -8.448   4.273  1.00 74.44      A    C  \\nATOM   1375  CD1 LEU A 170     -10.404  -8.133   5.545  1.00 73.97      A    C  \\nATOM   1376  CD2 LEU A 170      -8.846  -9.751   4.430  1.00 74.11      A    C  \\nATOM   1377  N   THR A 171      -7.959  -4.279   4.517  1.00 83.12      A    N  \\nATOM   1378  CA  THR A 171      -6.949  -3.233   4.577  1.00 84.53      A    C  \\nATOM   1379  C   THR A 171      -5.846  -3.647   5.506  1.00 84.53      A    C  \\nATOM   1380  O   THR A 171      -6.077  -4.352   6.500  1.00 82.07      A    O  \\nATOM   1381  CB  THR A 171      -7.528  -1.895   5.019  1.00 84.21      A    C  \\nATOM   1382  CG2 THR A 171      -8.598  -1.452   4.037  1.00 81.05      A    C  \\nATOM   1383  OG1 THR A 171      -8.118  -2.018   6.337  1.00 80.44      A    O  \\nATOM   1384  N   VAL A 172      -4.643  -3.218   5.214  1.00 87.78      A    N  \\nATOM   1385  CA  VAL A 172      -3.493  -3.558   6.040  1.00 86.63      A    C  \\nATOM   1386  C   VAL A 172      -3.404  -2.653   7.233  1.00 86.59      A    C  \\nATOM   1387  O   VAL A 172      -3.414  -3.087   8.389  1.00 83.49      A    O  \\nATOM   1388  CB  VAL A 172      -2.195  -3.502   5.212  1.00 85.49      A    C  \\nATOM   1389  CG1 VAL A 172      -2.260  -4.475   4.036  1.00 84.89      A    C  \\nATOM   1390  CG2 VAL A 172      -0.992  -3.825   6.088  1.00 85.04      A    C  \\nATOM   1391  N   LEU A 173      -3.367  -1.347   6.994  1.00 87.57      A    N  \\nATOM   1392  CA  LEU A 173      -3.343  -0.372   8.072  1.00 87.49      A    C  \\nATOM   1393  C   LEU A 173      -4.758  -0.121   8.593  1.00 88.05      A    C  \\nATOM   1394  O   LEU A 173      -5.736  -0.281   7.852  1.00 86.07      A    O  \\nATOM   1395  CB  LEU A 173      -2.702   0.940   7.624  1.00 87.60      A    C  \\nATOM   1396  CG  LEU A 173      -1.234   0.907   7.203  1.00 88.16      A    C  \\nATOM   1397  CD1 LEU A 173      -0.353   0.550   8.390  1.00 84.56      A    C  \\nATOM   1398  CD2 LEU A 173      -0.825   2.261   6.640  1.00 85.93      A    C  \\nATOM   1399  N   THR A 174      -4.843   0.264   9.823  1.00 85.45      A    N  \\nATOM   1400  CA  THR A 174      -6.137   0.595  10.430  1.00 83.75      A    C  \\nATOM   1401  C   THR A 174      -6.536   2.016   9.990  1.00 84.57      A    C  \\nATOM   1402  O   THR A 174      -5.766   2.743   9.391  1.00 84.43      A    O  \\nATOM   1403  CB  THR A 174      -6.083   0.508  11.931  1.00 82.60      A    C  \\nATOM   1404  CG2 THR A 174      -5.663  -0.906  12.348  1.00 77.85      A    C  \\nATOM   1405  OG1 THR A 174      -5.131   1.442  12.436  1.00 78.51      A    O  \\nATOM   1406  N   ASP A 175      -7.777   2.355  10.317  1.00 85.48      A    N  \\nATOM   1407  CA  ASP A 175      -8.257   3.685   9.956  1.00 83.74      A    C  \\nATOM   1408  C   ASP A 175      -7.586   4.771  10.762  1.00 84.54      A    C  \\nATOM   1409  O   ASP A 175      -7.387   5.883  10.272  1.00 84.25      A    O  \\nATOM   1410  CB  ASP A 175      -9.780   3.747  10.114  1.00 81.70      A    C  \\nATOM   1411  CG  ASP A 175     -10.240   3.337  11.493  1.00 79.72      A    C  \\nATOM   1412  OD1 ASP A 175      -9.394   2.967  12.337  1.00 73.50      A    O  \\nATOM   1413  OD2 ASP A 175     -11.470   3.366  11.751  1.00 74.71      A    O1-\\nATOM   1414  N   LYS A 176      -7.234   4.469  11.986  1.00 84.08      A    N  \\nATOM   1415  CA  LYS A 176      -6.519   5.442  12.808  1.00 83.24      A    C  \\nATOM   1416  C   LYS A 176      -5.098   5.660  12.277  1.00 84.61      A    C  \\nATOM   1417  O   LYS A 176      -4.584   6.775  12.371  1.00 84.14      A    O  \\nATOM   1418  CB  LYS A 176      -6.480   4.991  14.255  1.00 81.44      A    C  \\nATOM   1419  CG  LYS A 176      -7.847   4.951  14.918  1.00 73.20      A    C  \\nATOM   1420  CD  LYS A 176      -7.750   4.483  16.341  1.00 67.46      A    C  \\nATOM   1421  CE  LYS A 176      -9.132   4.471  16.979  1.00 58.90      A    C  \\nATOM   1422  NZ  LYS A 176      -9.091   4.027  18.383  1.00 50.89      A    N1+\\nATOM   1423  N   GLN A 177      -4.497   4.630  11.748  1.00 86.50      A    N  \\nATOM   1424  CA  GLN A 177      -3.189   4.739  11.161  1.00 85.85      A    C  \\nATOM   1425  C   GLN A 177      -3.250   5.435   9.804  1.00 86.62      A    C  \\nATOM   1426  O   GLN A 177      -2.313   6.150   9.435  1.00 87.49      A    O  \\nATOM   1427  CB  GLN A 177      -2.552   3.370  11.023  1.00 86.05      A    C  \\nATOM   1428  CG  GLN A 177      -2.277   2.711  12.348  1.00 85.87      A    C  \\nATOM   1429  CD  GLN A 177      -1.674   1.332  12.198  1.00 83.49      A    C  \\nATOM   1430  NE2 GLN A 177      -1.330   0.718  13.304  1.00 78.61      A    N  \\nATOM   1431  OE1 GLN A 177      -1.525   0.823  11.083  1.00 81.80      A    O  \\nATOM   1432  N   ARG A 178      -4.329   5.237   9.086  1.00 90.06      A    N  \\nATOM   1433  CA  ARG A 178      -4.496   5.933   7.810  1.00 90.15      A    C  \\nATOM   1434  C   ARG A 178      -4.748   7.401   8.028  1.00 90.20      A    C  \\nATOM   1435  O   ARG A 178      -4.364   8.218   7.214  1.00 90.42      A    O  \\nATOM   1436  CB  ARG A 178      -5.632   5.292   6.983  1.00 90.15      A    C  \\nATOM   1437  CG  ARG A 178      -5.302   3.903   6.495  1.00 89.48      A    C  \\nATOM   1438  CD  ARG A 178      -6.460   3.347   5.686  1.00 89.06      A    C  \\nATOM   1439  NE  ARG A 178      -7.618   3.056   6.520  1.00 85.53      A    N  \\nATOM   1440  CZ  ARG A 178      -8.811   2.736   6.026  1.00 88.28      A    C  \\nATOM   1441  NH1 ARG A 178      -8.986   2.696   4.718  1.00 76.61      A    N1+\\nATOM   1442  NH2 ARG A 178      -9.811   2.490   6.835  1.00 79.32      A    N  \\nATOM   1443  N   CYS A 179      -5.400   7.754   9.125  1.00 89.54      A    N  \\nATOM   1444  CA  CYS A 179      -5.587   9.153   9.446  1.00 89.16      A    C  \\nATOM   1445  C   CYS A 179      -4.287   9.778   9.954  1.00 88.94      A    C  \\nATOM   1446  O   CYS A 179      -4.054  10.965   9.743  1.00 87.70      A    O  \\nATOM   1447  CB  CYS A 179      -6.683   9.324  10.489  1.00 88.00      A    C  \\nATOM   1448  SG  CYS A 179      -8.321   8.881   9.894  1.00 84.39      A    S  \\nATOM   1449  N   GLU A 180      -3.460   8.977  10.594  1.00 87.08      A    N  \\nATOM   1450  CA  GLU A 180      -2.195   9.514  11.077  1.00 85.28      A    C  \\nATOM   1451  C   GLU A 180      -1.218   9.795   9.955  1.00 85.96      A    C  \\nATOM   1452  O   GLU A 180      -0.477  10.774  10.001  1.00 85.40      A    O  \\nATOM   1453  CB  GLU A 180      -1.575   8.568  12.105  1.00 83.58      A    C  \\nATOM   1454  CG  GLU A 180      -0.326   9.121  12.786  1.00 75.21      A    C  \\nATOM   1455  CD  GLU A 180       0.221   8.179  13.838  1.00 70.27      A    C  \\nATOM   1456  OE1 GLU A 180      -0.365   7.097  14.040  1.00 66.78      A    O  \\nATOM   1457  OE2 GLU A 180       1.239   8.543  14.495  1.00 61.00      A    O1-\\nATOM   1458  N   THR A 181      -1.223   8.965   8.933  1.00 89.58      A    N  \\nATOM   1459  CA  THR A 181      -0.342   9.166   7.795  1.00 89.51      A    C  \\nATOM   1460  C   THR A 181      -0.766  10.380   6.962  1.00 89.79      A    C  \\nATOM   1461  O   THR A 181       0.080  10.966   6.297  1.00 88.37      A    O  \\nATOM   1462  CB  THR A 181      -0.286   7.925   6.912  1.00 89.39      A    C  \\nATOM   1463  CG2 THR A 181       0.329   6.777   7.645  1.00 83.92      A    C  \\nATOM   1464  OG1 THR A 181      -1.608   7.564   6.500  1.00 81.65      A    O  \\nATOM   1465  N   LEU A 182      -2.019  10.736   7.018  1.00 90.72      A    N  \\nATOM   1466  CA  LEU A 182      -2.469  11.917   6.305  1.00 90.50      A    C  \\nATOM   1467  C   LEU A 182      -2.239  13.202   7.073  1.00 89.52      A    C  \\nATOM   1468  O   LEU A 182      -2.143  14.264   6.483  1.00 87.86      A    O  \\nATOM   1469  CB  LEU A 182      -3.942  11.799   5.937  1.00 91.21      A    C  \\nATOM   1470  CG  LEU A 182      -4.282  10.732   4.904  1.00 91.77      A    C  \\nATOM   1471  CD1 LEU A 182      -3.618  11.068   3.582  1.00 91.16      A    C  \\nATOM   1472  CD2 LEU A 182      -5.778  10.619   4.722  1.00 91.41      A    C  \\nATOM   1473  N   THR A 183      -2.143  13.112   8.403  1.00 87.90      A    N  \\nATOM   1474  CA  THR A 183      -1.873  14.325   9.194  1.00 84.82      A    C  \\nATOM   1475  C   THR A 183      -0.438  14.784   9.013  1.00 83.79      A    C  \\nATOM   1476  O   THR A 183      -0.170  15.977   9.046  1.00 81.54      A    O  \\nATOM   1477  CB  THR A 183      -2.158  14.079  10.671  1.00 83.29      A    C  \\nATOM   1478  CG2 THR A 183      -3.635  13.849  10.869  1.00 72.50      A    C  \\nATOM   1479  OG1 THR A 183      -1.428  12.961  11.147  1.00 74.45      A    O  \\nATOM   1480  N   HIS A 184       0.471  13.845   8.843  1.00 86.15      A    N  \\nATOM   1481  CA  HIS A 184       1.869  14.189   8.649  1.00 84.17      A    C  \\nATOM   1482  C   HIS A 184       2.201  14.491   7.211  1.00 85.17      A    C  \\nATOM   1483  O   HIS A 184       3.349  14.793   6.894  1.00 83.07      A    O  \\nATOM   1484  CB  HIS A 184       2.761  13.053   9.162  1.00 82.17      A    C  \\nATOM   1485  CG  HIS A 184       2.710  12.850  10.642  1.00 80.30      A    C  \\nATOM   1486  CD2 HIS A 184       2.128  11.859  11.374  1.00 75.43      A    C  \\nATOM   1487  ND1 HIS A 184       3.268  13.718  11.537  1.00 72.82      A    N  \\nATOM   1488  CE1 HIS A 184       3.050  13.288  12.773  1.00 73.48      A    C  \\nATOM   1489  NE2 HIS A 184       2.362  12.162  12.699  1.00 74.05      A    N  \\nATOM   1490  N   CYS A 185       1.216  14.430   6.336  1.00 87.29      A    N  \\nATOM   1491  CA  CYS A 185       1.445  14.768   4.935  1.00 86.96      A    C  \\nATOM   1492  C   CYS A 185       1.542  16.277   4.778  1.00 86.14      A    C  \\nATOM   1493  O   CYS A 185       1.073  17.050   5.623  1.00 82.61      A    O  \\nATOM   1494  CB  CYS A 185       0.334  14.205   4.049  1.00 86.47      A    C  \\nATOM   1495  SG  CYS A 185       0.588  14.526   2.291  1.00 83.47      A    S  \\nATOM   1496  N   ARG A 186       2.179  16.701   3.731  1.00 84.99      A    N  \\nATOM   1497  CA  ARG A 186       2.437  18.114   3.543  1.00 82.37      A    C  \\nATOM   1498  C   ARG A 186       1.219  18.868   3.005  1.00 81.96      A    C  \\nATOM   1499  O   ARG A 186       0.888  19.951   3.460  1.00 79.74      A    O  \\nATOM   1500  CB  ARG A 186       3.617  18.331   2.576  1.00 79.26      A    C  \\nATOM   1501  CG  ARG A 186       4.020  19.766   2.376  1.00 71.01      A    C  \\nATOM   1502  CD  ARG A 186       5.227  19.890   1.474  1.00 64.21      A    C  \\nATOM   1503  NE  ARG A 186       5.607  21.267   1.222  1.00 54.81      A    N  \\nATOM   1504  CZ  ARG A 186       6.631  21.638   0.472  1.00 56.29      A    C  \\nATOM   1505  NH1 ARG A 186       7.411  20.700  -0.088  1.00 44.71      A    N1+\\nATOM   1506  NH2 ARG A 186       6.913  22.924   0.311  1.00 45.10      A    N  \\nATOM   1507  N   TRP A 187       0.579  18.287   2.003  1.00 86.10      A    N  \\nATOM   1508  CA  TRP A 187      -0.510  18.972   1.320  1.00 85.20      A    C  \\nATOM   1509  C   TRP A 187      -1.835  18.773   1.991  1.00 86.44      A    C  \\nATOM   1510  O   TRP A 187      -2.800  19.484   1.650  1.00 83.49      A    O  \\nATOM   1511  CB  TRP A 187      -0.562  18.486  -0.137  1.00 83.66      A    C  \\nATOM   1512  CG  TRP A 187       0.676  18.829  -0.894  1.00 85.39      A    C  \\nATOM   1513  CD1 TRP A 187       0.985  20.013  -1.470  1.00 81.73      A    C  \\nATOM   1514  CD2 TRP A 187       1.799  17.975  -1.152  1.00 83.72      A    C  \\nATOM   1515  CE2 TRP A 187       2.721  18.689  -1.899  1.00 82.38      A    C  \\nATOM   1516  CE3 TRP A 187       2.073  16.650  -0.833  1.00 83.27      A    C  \\nATOM   1517  NE1 TRP A 187       2.209  19.936  -2.083  1.00 77.31      A    N  \\nATOM   1518  CZ2 TRP A 187       3.922  18.138  -2.323  1.00 76.97      A    C  \\nATOM   1519  CZ3 TRP A 187       3.269  16.122  -1.253  1.00 76.67      A    C  \\nATOM   1520  CH2 TRP A 187       4.177  16.845  -1.983  1.00 74.46      A    C  \\nATOM   1521  N   VAL A 188      -1.935  17.840   2.920  1.00 88.29      A    N  \\nATOM   1522  CA  VAL A 188      -3.202  17.582   3.587  1.00 89.02      A    C  \\nATOM   1523  C   VAL A 188      -3.400  18.573   4.713  1.00 88.00      A    C  \\nATOM   1524  O   VAL A 188      -2.514  18.737   5.566  1.00 85.18      A    O  \\nATOM   1525  CB  VAL A 188      -3.279  16.138   4.099  1.00 89.85      A    C  \\nATOM   1526  CG1 VAL A 188      -3.221  15.186   2.919  1.00 88.20      A    C  \\nATOM   1527  CG2 VAL A 188      -4.569  15.913   4.858  1.00 88.40      A    C  \\nATOM   1528  N   ASP A 189      -4.530  19.237   4.749  1.00 87.20      A    N  \\nATOM   1529  CA  ASP A 189      -4.820  20.213   5.785  1.00 85.15      A    C  \\nATOM   1530  C   ASP A 189      -5.761  19.640   6.824  1.00 85.26      A    C  \\nATOM   1531  O   ASP A 189      -5.481  19.714   8.020  1.00 82.81      A    O  \\nATOM   1532  CB  ASP A 189      -5.406  21.481   5.171  1.00 83.25      A    C  \\nATOM   1533  CG  ASP A 189      -4.412  22.208   4.296  1.00 82.90      A    C  \\nATOM   1534  OD1 ASP A 189      -3.216  22.201   4.622  1.00 79.43      A    O  \\nATOM   1535  OD2 ASP A 189      -4.842  22.788   3.279  1.00 78.91      A    O1-\\nATOM   1536  N   GLU A 190      -6.877  19.089   6.377  1.00 87.68      A    N  \\nATOM   1537  CA  GLU A 190      -7.856  18.539   7.301  1.00 86.74      A    C  \\nATOM   1538  C   GLU A 190      -8.151  17.105   6.916  1.00 87.39      A    C  \\nATOM   1539  O   GLU A 190      -8.316  16.786   5.748  1.00 87.16      A    O  \\nATOM   1540  CB  GLU A 190      -9.149  19.350   7.281  1.00 85.49      A    C  \\nATOM   1541  CG  GLU A 190      -9.001  20.773   7.802  1.00 76.93      A    C  \\nATOM   1542  CD  GLU A 190     -10.295  21.573   7.730  1.00 76.27      A    C  \\nATOM   1543  OE1 GLU A 190     -11.314  21.024   7.264  1.00 73.32      A    O  \\nATOM   1544  OE2 GLU A 190     -10.290  22.754   8.156  1.00 66.56      A    O1-\\nATOM   1545  N   VAL A 191      -8.237  16.251   7.920  1.00 86.82      A    N  \\nATOM   1546  CA  VAL A 191      -8.531  14.829   7.713  1.00 86.96      A    C  \\nATOM   1547  C   VAL A 191      -9.853  14.520   8.395  1.00 86.59      A    C  \\nATOM   1548  O   VAL A 191      -9.995  14.681   9.591  1.00 83.88      A    O  \\nATOM   1549  CB  VAL A 191      -7.402  13.931   8.256  1.00 85.82      A    C  \\nATOM   1550  CG1 VAL A 191      -6.115  14.184   7.526  1.00 85.25      A    C  \\nATOM   1551  CG2 VAL A 191      -7.817  12.479   8.160  1.00 85.69      A    C  \\nATOM   1552  N   ILE A 192     -10.802  14.076   7.600  1.00 87.34      A    N  \\nATOM   1553  CA  ILE A 192     -12.107  13.695   8.115  1.00 85.98      A    C  \\nATOM   1554  C   ILE A 192     -12.090  12.179   8.318  1.00 85.72      A    C  \\nATOM   1555  O   ILE A 192     -12.022  11.420   7.347  1.00 84.60      A    O  \\nATOM   1556  CB  ILE A 192     -13.234  14.109   7.173  1.00 85.16      A    C  \\nATOM   1557  CG1 ILE A 192     -13.266  15.640   7.029  1.00 84.61      A    C  \\nATOM   1558  CG2 ILE A 192     -14.585  13.603   7.677  1.00 84.48      A    C  \\nATOM   1559  CD1 ILE A 192     -14.217  16.157   6.001  1.00 81.32      A    C  \\nATOM   1560  N   ALA A 193     -12.182  11.772   9.556  1.00 85.23      A    N  \\nATOM   1561  CA  ALA A 193     -12.240  10.342   9.873  1.00 84.18      A    C  \\nATOM   1562  C   ALA A 193     -13.676   9.846   9.734  1.00 84.39      A    C  \\nATOM   1563  O   ALA A 193     -14.617  10.613   9.879  1.00 81.41      A    O  \\nATOM   1564  CB  ALA A 193     -11.705  10.063  11.264  1.00 82.42      A    C  \\nATOM   1565  N   ASN A 194     -13.791   8.568   9.454  1.00 83.83      A    N  \\nATOM   1566  CA  ASN A 194     -15.088   7.915   9.246  1.00 83.84      A    C  \\nATOM   1567  C   ASN A 194     -15.857   8.594   8.116  1.00 84.32      A    C  \\nATOM   1568  O   ASN A 194     -17.016   9.002   8.280  1.00 81.98      A    O  \\nATOM   1569  CB  ASN A 194     -15.918   7.857  10.528  1.00 81.26      A    C  \\nATOM   1570  CG  ASN A 194     -15.292   6.972  11.585  1.00 76.81      A    C  \\nATOM   1571  ND2 ASN A 194     -15.196   7.485  12.795  1.00 71.01      A    N  \\nATOM   1572  OD1 ASN A 194     -14.909   5.843  11.307  1.00 72.24      A    O  \\nATOM   1573  N   ALA A 195     -15.227   8.693   6.983  1.00 88.10      A    N  \\nATOM   1574  CA  ALA A 195     -15.852   9.304   5.829  1.00 88.35      A    C  \\nATOM   1575  C   ALA A 195     -16.700   8.282   5.089  1.00 87.69      A    C  \\nATOM   1576  O   ALA A 195     -16.347   7.089   5.057  1.00 85.74      A    O  \\nATOM   1577  CB  ALA A 195     -14.789   9.871   4.886  1.00 88.36      A    C  \\nATOM   1578  N   PRO A 196     -17.805   8.689   4.490  1.00 87.77      A    N  \\nATOM   1579  CA  PRO A 196     -18.629   7.740   3.744  1.00 87.14      A    C  \\nATOM   1580  C   PRO A 196     -17.972   7.353   2.434  1.00 86.65      A    C  \\nATOM   1581  O   PRO A 196     -17.161   8.093   1.888  1.00 85.00      A    O  \\nATOM   1582  CB  PRO A 196     -19.932   8.495   3.524  1.00 85.59      A    C  \\nATOM   1583  CG  PRO A 196     -19.503   9.913   3.512  1.00 85.06      A    C  \\nATOM   1584  CD  PRO A 196     -18.404  10.023   4.530  1.00 87.73      A    C  \\nATOM   1585  N   TRP A 197     -18.352   6.180   1.944  1.00 82.94      A    N  \\nATOM   1586  CA  TRP A 197     -17.788   5.690   0.694  1.00 82.48      A    C  \\nATOM   1587  C   TRP A 197     -18.353   6.473  -0.504  1.00 82.14      A    C  \\nATOM   1588  O   TRP A 197     -17.611   6.837  -1.409  1.00 79.73      A    O  \\nATOM   1589  CB  TRP A 197     -18.054   4.208   0.536  1.00 81.32      A    C  \\nATOM   1590  CG  TRP A 197     -17.468   3.601  -0.692  1.00 80.60      A    C  \\nATOM   1591  CD1 TRP A 197     -18.107   3.352  -1.863  1.00 78.21      A    C  \\nATOM   1592  CD2 TRP A 197     -16.106   3.145  -0.873  1.00 80.26      A    C  \\nATOM   1593  CE2 TRP A 197     -16.001   2.648  -2.168  1.00 79.08      A    C  \\nATOM   1594  CE3 TRP A 197     -14.982   3.144  -0.057  1.00 79.13      A    C  \\nATOM   1595  NE1 TRP A 197     -17.245   2.784  -2.759  1.00 76.66      A    N  \\nATOM   1596  CZ2 TRP A 197     -14.790   2.117  -2.666  1.00 75.18      A    C  \\nATOM   1597  CZ3 TRP A 197     -13.790   2.632  -0.542  1.00 74.96      A    C  \\nATOM   1598  CH2 TRP A 197     -13.708   2.098  -1.864  1.00 74.24      A    C  \\nATOM   1599  N   CYS A 198     -19.650   6.690  -0.485  1.00 83.80      A    N  \\nATOM   1600  CA  CYS A 198     -20.313   7.441  -1.541  1.00 83.95      A    C  \\nATOM   1601  C   CYS A 198     -20.457   8.902  -1.104  1.00 84.54      A    C  \\nATOM   1602  O   CYS A 198     -21.059   9.188  -0.085  1.00 82.46      A    O  \\nATOM   1603  CB  CYS A 198     -21.681   6.857  -1.871  1.00 81.19      A    C  \\nATOM   1604  SG  CYS A 198     -21.565   5.217  -2.603  1.00 76.12      A    S  \\nATOM   1605  N   VAL A 199     -19.885   9.800  -1.901  1.00 86.66      A    N  \\nATOM   1606  CA  VAL A 199     -19.926  11.212  -1.581  1.00 86.82      A    C  \\nATOM   1607  C   VAL A 199     -21.280  11.765  -1.973  1.00 86.27      A    C  \\nATOM   1608  O   VAL A 199     -21.634  11.852  -3.156  1.00 83.44      A    O  \\nATOM   1609  CB  VAL A 199     -18.801  11.959  -2.321  1.00 86.14      A    C  \\nATOM   1610  CG1 VAL A 199     -17.446  11.429  -1.877  1.00 81.45      A    C  \\nATOM   1611  CG2 VAL A 199     -18.873  13.448  -2.004  1.00 79.76      A    C  \\nATOM   1612  N   THR A 200     -22.046  12.142  -0.988  1.00 86.90      A    N  \\nATOM   1613  CA  THR A 200     -23.381  12.696  -1.190  1.00 86.51      A    C  \\nATOM   1614  C   THR A 200     -23.313  14.224  -1.218  1.00 87.40      A    C  \\nATOM   1615  O   THR A 200     -22.373  14.818  -0.696  1.00 87.05      A    O  \\nATOM   1616  CB  THR A 200     -24.334  12.231  -0.097  1.00 84.29      A    C  \\nATOM   1617  CG2 THR A 200     -24.414  10.720  -0.063  1.00 80.71      A    C  \\nATOM   1618  OG1 THR A 200     -23.869  12.702   1.188  1.00 81.85      A    O  \\nATOM   1619  N   PRO A 201     -24.280  14.875  -1.842  1.00 87.39      A    N  \\nATOM   1620  CA  PRO A 201     -24.291  16.328  -1.822  1.00 86.47      A    C  \\nATOM   1621  C   PRO A 201     -24.592  16.935  -0.465  1.00 87.93      A    C  \\nATOM   1622  O   PRO A 201     -24.276  18.106  -0.222  1.00 86.83      A    O  \\nATOM   1623  CB  PRO A 201     -25.371  16.719  -2.856  1.00 84.20      A    C  \\nATOM   1624  CG  PRO A 201     -26.213  15.520  -2.935  1.00 82.80      A    C  \\nATOM   1625  CD  PRO A 201     -25.312  14.354  -2.785  1.00 85.76      A    C  \\nATOM   1626  N   GLU A 202     -25.179  16.152   0.434  1.00 87.07      A    N  \\nATOM   1627  CA  GLU A 202     -25.373  16.641   1.798  1.00 86.60      A    C  \\nATOM   1628  C   GLU A 202     -24.046  16.697   2.547  1.00 87.98      A    C  \\nATOM   1629  O   GLU A 202     -23.836  17.575   3.387  1.00 87.22      A    O  \\nATOM   1630  CB  GLU A 202     -26.378  15.754   2.534  1.00 85.00      A    C  \\nATOM   1631  CG  GLU A 202     -26.743  16.234   3.923  1.00 75.68      A    C  \\nATOM   1632  CD  GLU A 202     -27.782  15.352   4.582  1.00 65.34      A    C  \\nATOM   1633  OE1 GLU A 202     -28.175  14.323   3.963  1.00 60.44      A    O  \\nATOM   1634  OE2 GLU A 202     -28.205  15.670   5.719  1.00 56.28      A    O1-\\nATOM   1635  N   PHE A 203     -23.167  15.777   2.253  1.00 88.92      A    N  \\nATOM   1636  CA  PHE A 203     -21.849  15.779   2.846  1.00 89.02      A    C  \\nATOM   1637  C   PHE A 203     -20.997  16.945   2.313  1.00 89.17      A    C  \\nATOM   1638  O   PHE A 203     -20.155  17.475   3.035  1.00 88.50      A    O  \\nATOM   1639  CB  PHE A 203     -21.165  14.453   2.565  1.00 89.56      A    C  \\nATOM   1640  CG  PHE A 203     -19.780  14.353   3.139  1.00 90.87      A    C  \\nATOM   1641  CD1 PHE A 203     -19.598  14.148   4.508  1.00 89.66      A    C  \\nATOM   1642  CD2 PHE A 203     -18.650  14.451   2.322  1.00 90.28      A    C  \\nATOM   1643  CE1 PHE A 203     -18.345  14.045   5.045  1.00 88.31      A    C  \\nATOM   1644  CE2 PHE A 203     -17.376  14.347   2.878  1.00 89.69      A    C  \\nATOM   1645  CZ  PHE A 203     -17.225  14.132   4.229  1.00 88.98      A    C  \\nATOM   1646  N   LEU A 204     -21.266  17.349   1.101  1.00 92.16      A    N  \\nATOM   1647  CA  LEU A 204     -20.535  18.483   0.535  1.00 91.71      A    C  \\nATOM   1648  C   LEU A 204     -20.967  19.790   1.180  1.00 91.19      A    C  \\nATOM   1649  O   LEU A 204     -20.149  20.694   1.366  1.00 90.00      A    O  \\nATOM   1650  CB  LEU A 204     -20.760  18.552  -0.974  1.00 91.33      A    C  \\nATOM   1651  CG  LEU A 204     -20.242  17.389  -1.825  1.00 90.82      A    C  \\nATOM   1652  CD1 LEU A 204     -18.749  17.267  -1.683  1.00 89.00      A    C  \\nATOM   1653  CD2 LEU A 204     -20.655  17.569  -3.254  1.00 89.24      A    C  \\nATOM   1654  N   LYS A 205     -22.242  19.896   1.511  1.00 89.64      A    N  \\nATOM   1655  CA  LYS A 205     -22.747  21.147   2.099  1.00 88.13      A    C  \\nATOM   1656  C   LYS A 205     -22.404  21.270   3.550  1.00 87.61      A    C  \\nATOM   1657  O   LYS A 205     -22.229  22.386   4.061  1.00 85.64      A    O  \\nATOM   1658  CB  LYS A 205     -24.257  21.243   1.891  1.00 87.91      A    C  \\nATOM   1659  CG  LYS A 205     -24.699  21.397   0.439  1.00 76.74      A    C  \\nATOM   1660  CD  LYS A 205     -26.197  21.482   0.328  1.00 71.72      A    C  \\nATOM   1661  CE  LYS A 205     -26.644  21.629  -1.130  1.00 61.47      A    C  \\nATOM   1662  NZ  LYS A 205     -28.114  21.676  -1.267  1.00 53.66      A    N1+\\nATOM   1663  N   ASP A 206     -22.335  20.147   4.263  1.00 87.92      A    N  \\nATOM   1664  CA  ASP A 206     -22.028  20.206   5.677  1.00 86.03      A    C  \\nATOM   1665  C   ASP A 206     -20.569  20.555   5.922  1.00 86.68      A    C  \\nATOM   1666  O   ASP A 206     -20.251  21.258   6.876  1.00 84.29      A    O  \\nATOM   1667  CB  ASP A 206     -22.372  18.877   6.338  1.00 85.05      A    C  \\nATOM   1668  CG  ASP A 206     -23.857  18.630   6.387  1.00 80.68      A    C  \\nATOM   1669  OD1 ASP A 206     -24.626  19.597   6.479  1.00 75.53      A    O  \\nATOM   1670  OD2 ASP A 206     -24.254  17.456   6.347  1.00 75.78      A    O1-\\nATOM   1671  N   HIS A 207     -19.667  20.080   5.065  1.00 87.51      A    N  \\nATOM   1672  CA  HIS A 207     -18.264  20.344   5.231  1.00 86.38      A    C  \\nATOM   1673  C   HIS A 207     -17.752  21.454   4.310  1.00 86.55      A    C  \\nATOM   1674  O   HIS A 207     -16.577  21.772   4.332  1.00 86.14      A    O  \\nATOM   1675  CB  HIS A 207     -17.481  19.057   4.944  1.00 85.80      A    C  \\nATOM   1676  CG  HIS A 207     -17.699  18.004   5.978  1.00 86.36      A    C  \\nATOM   1677  CD2 HIS A 207     -17.026  17.680   7.086  1.00 79.56      A    C  \\nATOM   1678  ND1 HIS A 207     -18.782  17.148   5.901  1.00 79.11      A    N  \\nATOM   1679  CE1 HIS A 207     -18.745  16.327   6.926  1.00 79.94      A    C  \\nATOM   1680  NE2 HIS A 207     -17.689  16.599   7.650  1.00 80.95      A    N  \\nATOM   1681  N   LYS A 208     -18.656  22.053   3.556  1.00 88.59      A    N  \\nATOM   1682  CA  LYS A 208     -18.354  23.143   2.645  1.00 88.21      A    C  \\nATOM   1683  C   LYS A 208     -17.261  22.751   1.640  1.00 89.43      A    C  \\nATOM   1684  O   LYS A 208     -16.235  23.404   1.527  1.00 88.16      A    O  \\nATOM   1685  CB  LYS A 208     -17.979  24.409   3.393  1.00 85.92      A    C  \\nATOM   1686  CG  LYS A 208     -19.122  24.994   4.210  1.00 81.78      A    C  \\nATOM   1687  CD  LYS A 208     -18.681  26.209   4.977  1.00 75.61      A    C  \\nATOM   1688  CE  LYS A 208     -19.818  26.789   5.818  1.00 69.90      A    C  \\nATOM   1689  NZ  LYS A 208     -19.391  27.935   6.635  1.00 59.63      A    N1+\\nATOM   1690  N   ILE A 209     -17.499  21.680   0.938  1.00 90.35      A    N  \\nATOM   1691  CA  ILE A 209     -16.559  21.140  -0.031  1.00 91.38      A    C  \\nATOM   1692  C   ILE A 209     -16.935  21.643  -1.408  1.00 90.91      A    C  \\nATOM   1693  O   ILE A 209     -18.060  21.458  -1.869  1.00 89.69      A    O  \\nATOM   1694  CB  ILE A 209     -16.510  19.617   0.018  1.00 91.98      A    C  \\nATOM   1695  CG1 ILE A 209     -15.983  19.154   1.383  1.00 91.06      A    C  \\nATOM   1696  CG2 ILE A 209     -15.665  19.067  -1.092  1.00 91.22      A    C  \\nATOM   1697  CD1 ILE A 209     -16.176  17.696   1.619  1.00 87.33      A    C  \\nATOM   1698  N   ASP A 210     -16.000  22.315  -2.067  1.00 90.99      A    N  \\nATOM   1699  CA  ASP A 210     -16.268  22.887  -3.379  1.00 89.77      A    C  \\nATOM   1700  C   ASP A 210     -16.083  21.856  -4.485  1.00 89.33      A    C  \\nATOM   1701  O   ASP A 210     -16.956  21.751  -5.376  1.00 87.57      A    O  \\nATOM   1702  CB  ASP A 210     -15.362  24.083  -3.627  1.00 88.70      A    C  \\nATOM   1703  CG  ASP A 210     -15.681  25.249  -2.700  1.00 88.47      A    C  \\nATOM   1704  OD1 ASP A 210     -16.859  25.394  -2.306  1.00 84.09      A    O  \\nATOM   1705  OD2 ASP A 210     -14.764  25.997  -2.366  1.00 83.73      A    O1-\\nATOM   1706  N   TYR A 211     -15.005  21.121  -4.434  1.00 90.35      A    N  \\nATOM   1707  CA  TYR A 211     -14.690  20.137  -5.473  1.00 90.47      A    C  \\nATOM   1708  C   TYR A 211     -14.320  18.804  -4.851  1.00 91.29      A    C  \\nATOM   1709  O   TYR A 211     -13.835  18.754  -3.715  1.00 91.32      A    O  \\nATOM   1710  CB  TYR A 211     -13.572  20.624  -6.394  1.00 89.70      A    C  \\nATOM   1711  CG  TYR A 211     -13.899  21.862  -7.172  1.00 89.33      A    C  \\nATOM   1712  CD1 TYR A 211     -14.703  21.803  -8.304  1.00 88.02      A    C  \\nATOM   1713  CD2 TYR A 211     -13.392  23.105  -6.806  1.00 88.10      A    C  \\nATOM   1714  CE1 TYR A 211     -14.997  22.925  -9.055  1.00 86.31      A    C  \\nATOM   1715  CE2 TYR A 211     -13.680  24.249  -7.520  1.00 87.33      A    C  \\nATOM   1716  CZ  TYR A 211     -14.469  24.153  -8.641  1.00 86.54      A    C  \\nATOM   1717  OH  TYR A 211     -14.764  25.272  -9.374  1.00 84.47      A    O  \\nATOM   1718  N   VAL A 212     -14.528  17.763  -5.616  1.00 92.19      A    N  \\nATOM   1719  CA  VAL A 212     -14.146  16.410  -5.231  1.00 92.44      A    C  \\nATOM   1720  C   VAL A 212     -13.098  15.936  -6.208  1.00 91.99      A    C  \\nATOM   1721  O   VAL A 212     -13.346  15.848  -7.397  1.00 90.62      A    O  \\nATOM   1722  CB  VAL A 212     -15.345  15.452  -5.201  1.00 92.39      A    C  \\nATOM   1723  CG1 VAL A 212     -16.374  15.910  -4.166  1.00 91.26      A    C  \\nATOM   1724  CG2 VAL A 212     -14.884  14.037  -4.878  1.00 91.46      A    C  \\nATOM   1725  N   ALA A 213     -11.911  15.643  -5.694  1.00 92.50      A    N  \\nATOM   1726  CA  ALA A 213     -10.797  15.239  -6.538  1.00 91.88      A    C  \\nATOM   1727  C   ALA A 213     -10.704  13.708  -6.569  1.00 90.83      A    C  \\nATOM   1728  O   ALA A 213     -10.667  13.056  -5.529  1.00 88.78      A    O  \\nATOM   1729  CB  ALA A 213      -9.493  15.837  -6.040  1.00 91.38      A    C  \\nATOM   1730  N   HIS A 214     -10.656  13.201  -7.778  1.00 89.56      A    N  \\nATOM   1731  CA  HIS A 214     -10.508  11.766  -7.985  1.00 87.93      A    C  \\nATOM   1732  C   HIS A 214     -10.040  11.537  -9.428  1.00 87.31      A    C  \\nATOM   1733  O   HIS A 214     -10.006  12.461 -10.232  1.00 86.12      A    O  \\nATOM   1734  CB  HIS A 214     -11.810  11.003  -7.675  1.00 86.33      A    C  \\nATOM   1735  CG  HIS A 214     -11.614   9.533  -7.491  1.00 85.54      A    C  \\nATOM   1736  CD2 HIS A 214     -11.244   8.801  -6.424  1.00 81.24      A    C  \\nATOM   1737  ND1 HIS A 214     -11.800   8.644  -8.525  1.00 80.61      A    N  \\nATOM   1738  CE1 HIS A 214     -11.555   7.418  -8.088  1.00 80.21      A    C  \\nATOM   1739  NE2 HIS A 214     -11.237   7.473  -6.818  1.00 81.48      A    N  \\nATOM   1740  N   ASP A 215      -9.654  10.321  -9.713  1.00 85.40      A    N  \\nATOM   1741  CA  ASP A 215      -9.163   9.994 -11.062  1.00 83.47      A    C  \\nATOM   1742  C   ASP A 215     -10.308   9.868 -12.053  1.00 82.79      A    C  \\nATOM   1743  O   ASP A 215     -11.486   9.784 -11.699  1.00 79.77      A    O  \\nATOM   1744  CB  ASP A 215      -8.326   8.716 -11.010  1.00 79.42      A    C  \\nATOM   1745  CG  ASP A 215      -9.061   7.564 -10.370  1.00 75.94      A    C  \\nATOM   1746  OD1 ASP A 215     -10.181   7.765  -9.876  1.00 67.42      A    O  \\nATOM   1747  OD2 ASP A 215      -8.520   6.434 -10.364  1.00 72.01      A    O1-\\nATOM   1748  N   ASP A 216      -9.947   9.847 -13.324  1.00 78.25      A    N  \\nATOM   1749  CA  ASP A 216     -10.928   9.790 -14.396  1.00 76.57      A    C  \\nATOM   1750  C   ASP A 216     -11.527   8.397 -14.541  1.00 76.76      A    C  \\nATOM   1751  O   ASP A 216     -12.585   8.232 -15.166  1.00 73.87      A    O  \\nATOM   1752  CB  ASP A 216     -10.305  10.234 -15.724  1.00 72.38      A    C  \\nATOM   1753  CG  ASP A 216     -11.337  10.585 -16.784  1.00 66.38      A    C  \\nATOM   1754  OD1 ASP A 216     -12.536  10.619 -16.451  1.00 60.39      A    O  \\nATOM   1755  OD2 ASP A 216     -10.942  10.840 -17.924  1.00 62.80      A    O1-\\nATOM   1756  N   ILE A 217     -10.859   7.386 -14.001  1.00 75.05      A    N  \\nATOM   1757  CA  ILE A 217     -11.357   6.005 -14.100  1.00 75.04      A    C  \\nATOM   1758  C   ILE A 217     -12.590   5.846 -13.223  1.00 75.71      A    C  \\nATOM   1759  O   ILE A 217     -12.512   6.053 -12.011  1.00 72.85      A    O  \\nATOM   1760  CB  ILE A 217     -10.262   5.020 -13.685  1.00 71.54      A    C  \\nATOM   1761  CG1 ILE A 217      -9.028   5.180 -14.587  1.00 67.71      A    C  \\nATOM   1762  CG2 ILE A 217     -10.760   3.585 -13.781  1.00 66.75      A    C  \\nATOM   1763  CD1 ILE A 217      -7.826   4.405 -14.119  1.00 62.08      A    C  \\nATOM   1764  N   PRO A 218     -13.729   5.462 -13.803  1.00 75.36      A    N  \\nATOM   1765  CA  PRO A 218     -14.926   5.288 -12.990  1.00 74.88      A    C  \\nATOM   1766  C   PRO A 218     -14.867   4.007 -12.177  1.00 74.77      A    C  \\nATOM   1767  O   PRO A 218     -14.344   2.980 -12.608  1.00 70.73      A    O  \\nATOM   1768  CB  PRO A 218     -16.058   5.228 -14.034  1.00 71.28      A    C  \\nATOM   1769  CG  PRO A 218     -15.380   4.686 -15.233  1.00 69.96      A    C  \\nATOM   1770  CD  PRO A 218     -13.988   5.258 -15.218  1.00 72.61      A    C  \\nATOM   1771  N   TYR A 219     -15.390   4.064 -10.983  1.00 74.15      A    N  \\nATOM   1772  CA  TYR A 219     -15.471   2.918 -10.099  1.00 74.74      A    C  \\nATOM   1773  C   TYR A 219     -16.812   2.254 -10.258  1.00 74.19      A    C  \\nATOM   1774  O   TYR A 219     -17.852   2.893 -10.149  1.00 71.43      A    O  \\nATOM   1775  CB  TYR A 219     -15.227   3.307  -8.648  1.00 72.24      A    C  \\nATOM   1776  CG  TYR A 219     -15.176   2.128  -7.689  1.00 68.07      A    C  \\nATOM   1777  CD1 TYR A 219     -16.228   1.830  -6.888  1.00 63.35      A    C  \\nATOM   1778  CD2 TYR A 219     -14.035   1.321  -7.634  1.00 62.81      A    C  \\nATOM   1779  CE1 TYR A 219     -16.192   0.750  -6.005  1.00 59.17      A    C  \\nATOM   1780  CE2 TYR A 219     -13.986   0.253  -6.774  1.00 59.12      A    C  \\nATOM   1781  CZ  TYR A 219     -15.055  -0.016  -5.954  1.00 58.66      A    C  \\nATOM   1782  OH  TYR A 219     -14.987  -1.096  -5.089  1.00 54.54      A    O  \\nATOM   1783  N   VAL A 220     -16.779   0.953 -10.483  1.00 69.53      A    N  \\nATOM   1784  CA  VAL A 220     -17.979   0.174 -10.692  1.00 68.96      A    C  \\nATOM   1785  C   VAL A 220     -18.429  -0.384  -9.353  1.00 69.36      A    C  \\nATOM   1786  O   VAL A 220     -17.690  -1.083  -8.662  1.00 66.74      A    O  \\nATOM   1787  CB  VAL A 220     -17.745  -0.968 -11.685  1.00 64.44      A    C  \\nATOM   1788  CG1 VAL A 220     -17.306  -0.424 -13.024  1.00 63.30      A    C  \\nATOM   1789  CG2 VAL A 220     -19.003  -1.783 -11.830  1.00 62.03      A    C  \\nATOM   1790  N   SER A 221     -19.630  -0.044  -8.971  1.00 71.53      A    N  \\nATOM   1791  CA  SER A 221     -20.217  -0.500  -7.740  1.00 71.49      A    C  \\nATOM   1792  C   SER A 221     -21.513  -1.258  -8.041  1.00 71.45      A    C  \\nATOM   1793  O   SER A 221     -21.948  -1.352  -9.171  1.00 69.07      A    O  \\nATOM   1794  CB  SER A 221     -20.495   0.659  -6.779  1.00 67.93      A    C  \\nATOM   1795  OG  SER A 221     -21.492   1.529  -7.316  1.00 64.04      A    O  \\nATOM   1796  N   ALA A 222     -22.094  -1.809  -6.966  1.00 66.03      A    N  \\nATOM   1797  CA  ALA A 222     -23.350  -2.546  -7.138  1.00 66.35      A    C  \\nATOM   1798  C   ALA A 222     -24.515  -1.610  -7.344  1.00 67.49      A    C  \\nATOM   1799  O   ALA A 222     -25.508  -1.975  -7.967  1.00 64.66      A    O  \\nATOM   1800  CB  ALA A 222     -23.573  -3.429  -5.923  1.00 62.30      A    C  \\nATOM   1801  N   ASP A 223     -24.422  -0.417  -6.806  1.00 67.03      A    N  \\nATOM   1802  CA  ASP A 223     -25.515   0.545  -6.938  1.00 67.51      A    C  \\nATOM   1803  C   ASP A 223     -25.516   1.194  -8.311  1.00 68.64      A    C  \\nATOM   1804  O   ASP A 223     -26.549   1.233  -8.991  1.00 65.49      A    O  \\nATOM   1805  CB  ASP A 223     -25.420   1.642  -5.843  1.00 62.47      A    C  \\nATOM   1806  CG  ASP A 223     -25.662   1.073  -4.453  1.00 58.69      A    C  \\nATOM   1807  OD1 ASP A 223     -26.399   0.078  -4.331  1.00 53.15      A    O  \\nATOM   1808  OD2 ASP A 223     -25.099   1.633  -3.486  1.00 56.33      A    O1-\\nATOM   1809  N   SER A 224     -24.385   1.695  -8.731  1.00 71.90      A    N  \\nATOM   1810  CA  SER A 224     -24.287   2.428  -9.990  1.00 72.49      A    C  \\nATOM   1811  C   SER A 224     -22.995   2.040 -10.711  1.00 73.15      A    C  \\nATOM   1812  O   SER A 224     -22.082   1.458 -10.104  1.00 69.69      A    O  \\nATOM   1813  CB  SER A 224     -24.338   3.937  -9.752  1.00 69.41      A    C  \\nATOM   1814  OG  SER A 224     -23.214   4.374  -9.030  1.00 63.84      A    O  \\nATOM   1815  N   ASP A 225     -22.957   2.371 -11.965  1.00 69.39      A    N  \\nATOM   1816  CA  ASP A 225     -21.748   2.077 -12.753  1.00 70.97      A    C  \\nATOM   1817  C   ASP A 225     -20.603   3.018 -12.388  1.00 73.28      A    C  \\nATOM   1818  O   ASP A 225     -19.456   2.667 -12.609  1.00 71.17      A    O  \\nATOM   1819  CB  ASP A 225     -22.054   2.218 -14.247  1.00 66.08      A    C  \\nATOM   1820  CG  ASP A 225     -22.977   1.128 -14.739  1.00 61.94      A    C  \\nATOM   1821  OD1 ASP A 225     -22.913  -0.007 -14.215  1.00 55.24      A    O  \\nATOM   1822  OD2 ASP A 225     -23.774   1.405 -15.673  1.00 56.17      A    O1-\\nATOM   1823  N   ASP A 226     -20.935   4.163 -11.855  1.00 79.38      A    N  \\nATOM   1824  CA  ASP A 226     -19.914   5.121 -11.421  1.00 80.39      A    C  \\nATOM   1825  C   ASP A 226     -20.431   5.840 -10.176  1.00 81.00      A    C  \\nATOM   1826  O   ASP A 226     -21.555   6.333 -10.166  1.00 80.23      A    O  \\nATOM   1827  CB  ASP A 226     -19.603   6.150 -12.527  1.00 78.63      A    C  \\nATOM   1828  CG  ASP A 226     -18.354   6.979 -12.225  1.00 77.95      A    C  \\nATOM   1829  OD1 ASP A 226     -17.729   6.766 -11.178  1.00 74.65      A    O  \\nATOM   1830  OD2 ASP A 226     -18.039   7.863 -13.041  1.00 74.83      A    O1-\\nATOM   1831  N   ILE A 227     -19.603   5.874  -9.142  1.00 80.32      A    N  \\nATOM   1832  CA  ILE A 227     -20.027   6.517  -7.896  1.00 81.22      A    C  \\nATOM   1833  C   ILE A 227     -19.968   8.041  -8.048  1.00 82.41      A    C  \\nATOM   1834  O   ILE A 227     -20.778   8.742  -7.440  1.00 81.18      A    O  \\nATOM   1835  CB  ILE A 227     -19.127   6.022  -6.743  1.00 78.59      A    C  \\nATOM   1836  CG1 ILE A 227     -19.254   4.502  -6.601  1.00 71.74      A    C  \\nATOM   1837  CG2 ILE A 227     -19.522   6.687  -5.422  1.00 70.64      A    C  \\nATOM   1838  CD1 ILE A 227     -18.340   3.914  -5.592  1.00 67.19      A    C  \\nATOM   1839  N   TYR A 228     -19.074   8.537  -8.837  1.00 85.68      A    N  \\nATOM   1840  CA  TYR A 228     -18.859   9.964  -9.030  1.00 86.52      A    C  \\nATOM   1841  C   TYR A 228     -19.692  10.550 -10.144  1.00 86.31      A    C  \\nATOM   1842  O   TYR A 228     -19.334  11.588 -10.720  1.00 85.70      A    O  \\nATOM   1843  CB  TYR A 228     -17.381  10.257  -9.254  1.00 87.07      A    C  \\nATOM   1844  CG  TYR A 228     -16.517   9.884  -8.077  1.00 87.58      A    C  \\nATOM   1845  CD1 TYR A 228     -16.410  10.707  -6.977  1.00 86.89      A    C  \\nATOM   1846  CD2 TYR A 228     -15.803   8.684  -8.092  1.00 87.27      A    C  \\nATOM   1847  CE1 TYR A 228     -15.620  10.373  -5.909  1.00 85.73      A    C  \\nATOM   1848  CE2 TYR A 228     -15.013   8.337  -7.016  1.00 87.02      A    C  \\nATOM   1849  CZ  TYR A 228     -14.926   9.186  -5.919  1.00 85.75      A    C  \\nATOM   1850  OH  TYR A 228     -14.148   8.852  -4.845  1.00 83.42      A    O  \\nATOM   1851  N   LYS A 229     -20.809   9.902 -10.478  1.00 85.06      A    N  \\nATOM   1852  CA  LYS A 229     -21.699  10.428 -11.512  1.00 84.55      A    C  \\nATOM   1853  C   LYS A 229     -22.425  11.697 -11.106  1.00 84.83      A    C  \\nATOM   1854  O   LYS A 229     -22.459  12.656 -11.888  1.00 84.61      A    O  \\nATOM   1855  CB  LYS A 229     -22.702   9.380 -11.980  1.00 83.19      A    C  \\nATOM   1856  CG  LYS A 229     -22.075   8.261 -12.813  1.00 77.89      A    C  \\nATOM   1857  CD  LYS A 229     -23.134   7.329 -13.342  1.00 71.64      A    C  \\nATOM   1858  CE  LYS A 229     -22.512   6.212 -14.168  1.00 65.74      A    C  \\nATOM   1859  NZ  LYS A 229     -23.528   5.262 -14.672  1.00 56.51      A    N1+\\nATOM   1860  N   PRO A 230     -23.052  11.784  -9.919  1.00 86.16      A    N  \\nATOM   1861  CA  PRO A 230     -23.716  13.036  -9.575  1.00 85.92      A    C  \\nATOM   1862  C   PRO A 230     -22.775  14.196  -9.256  1.00 86.79      A    C  \\nATOM   1863  O   PRO A 230     -23.184  15.344  -9.316  1.00 85.94      A    O  \\nATOM   1864  CB  PRO A 230     -24.568  12.684  -8.352  1.00 84.28      A    C  \\nATOM   1865  CG  PRO A 230     -23.812  11.588  -7.707  1.00 83.15      A    C  \\nATOM   1866  CD  PRO A 230     -23.240  10.806  -8.842  1.00 85.50      A    C  \\nATOM   1867  N   ILE A 231     -21.523  13.872  -8.937  1.00 88.52      A    N  \\nATOM   1868  CA  ILE A 231     -20.525  14.937  -8.721  1.00 88.85      A    C  \\nATOM   1869  C   ILE A 231     -20.155  15.576 -10.050  1.00 88.83      A    C  \\nATOM   1870  O   ILE A 231     -19.975  16.789 -10.137  1.00 88.44      A    O  \\nATOM   1871  CB  ILE A 231     -19.304  14.379  -7.993  1.00 89.19      A    C  \\nATOM   1872  CG1 ILE A 231     -19.722  13.721  -6.676  1.00 87.67      A    C  \\nATOM   1873  CG2 ILE A 231     -18.298  15.472  -7.696  1.00 87.77      A    C  \\nATOM   1874  CD1 ILE A 231     -18.630  12.994  -5.999  1.00 84.19      A    C  \\nATOM   1875  N   LYS A 232     -20.060  14.767 -11.079  1.00 87.77      A    N  \\nATOM   1876  CA  LYS A 232     -19.769  15.289 -12.421  1.00 87.21      A    C  \\nATOM   1877  C   LYS A 232     -20.956  16.065 -12.989  1.00 86.96      A    C  \\nATOM   1878  O   LYS A 232     -20.764  16.972 -13.785  1.00 85.67      A    O  \\nATOM   1879  CB  LYS A 232     -19.385  14.136 -13.349  1.00 86.49      A    C  \\nATOM   1880  CG  LYS A 232     -18.086  13.440 -12.985  1.00 85.88      A    C  \\nATOM   1881  CD  LYS A 232     -17.784  12.288 -13.900  1.00 84.20      A    C  \\nATOM   1882  CE  LYS A 232     -16.510  11.577 -13.505  1.00 81.79      A    C  \\nATOM   1883  NZ  LYS A 232     -16.234  10.412 -14.390  1.00 77.49      A    N1+\\nATOM   1884  N   GLU A 233     -22.163  15.685 -12.603  1.00 85.24      A    N  \\nATOM   1885  CA  GLU A 233     -23.345  16.401 -13.090  1.00 85.01      A    C  \\nATOM   1886  C   GLU A 233     -23.531  17.735 -12.380  1.00 84.82      A    C  \\nATOM   1887  O   GLU A 233     -24.064  18.677 -12.969  1.00 83.17      A    O  \\nATOM   1888  CB  GLU A 233     -24.582  15.551 -12.904  1.00 83.75      A    C  \\nATOM   1889  CG  GLU A 233     -24.621  14.310 -13.782  1.00 80.44      A    C  \\nATOM   1890  CD  GLU A 233     -25.878  13.473 -13.563  1.00 76.61      A    C  \\nATOM   1891  OE1 GLU A 233     -26.678  13.817 -12.679  1.00 71.73      A    O  \\nATOM   1892  OE2 GLU A 233     -26.051  12.459 -14.290  1.00 69.28      A    O1-\\nATOM   1893  N   MET A 234     -23.098  17.811 -11.139  1.00 87.10      A    N  \\nATOM   1894  CA  MET A 234     -23.158  19.076 -10.406  1.00 86.46      A    C  \\nATOM   1895  C   MET A 234     -22.014  20.011 -10.749  1.00 86.24      A    C  \\nATOM   1896  O   MET A 234     -22.045  21.162 -10.337  1.00 85.09      A    O  \\nATOM   1897  CB  MET A 234     -23.167  18.818  -8.888  1.00 85.97      A    C  \\nATOM   1898  CG  MET A 234     -24.437  18.163  -8.379  1.00 83.40      A    C  \\nATOM   1899  SD  MET A 234     -24.358  17.798  -6.623  1.00 82.88      A    S  \\nATOM   1900  CE  MET A 234     -24.575  19.435  -5.936  1.00 71.06      A    C  \\nATOM   1901  N   GLY A 235     -21.033  19.527 -11.459  1.00 86.01      A    N  \\nATOM   1902  CA  GLY A 235     -19.894  20.338 -11.809  1.00 85.55      A    C  \\nATOM   1903  C   GLY A 235     -18.876  20.502 -10.717  1.00 86.32      A    C  \\nATOM   1904  O   GLY A 235     -18.134  21.487 -10.695  1.00 84.67      A    O  \\nATOM   1905  N   LYS A 236     -18.814  19.539  -9.815  1.00 88.26      A    N  \\nATOM   1906  CA  LYS A 236     -17.868  19.575  -8.698  1.00 88.79      A    C  \\nATOM   1907  C   LYS A 236     -16.961  18.370  -8.719  1.00 89.40      A    C  \\nATOM   1908  O   LYS A 236     -16.885  17.617  -7.760  1.00 88.76      A    O  \\nATOM   1909  CB  LYS A 236     -18.589  19.690  -7.379  1.00 88.32      A    C  \\nATOM   1910  CG  LYS A 236     -19.379  20.950  -7.165  1.00 87.85      A    C  \\nATOM   1911  CD  LYS A 236     -20.055  20.987  -5.822  1.00 85.45      A    C  \\nATOM   1912  CE  LYS A 236     -20.836  22.276  -5.627  1.00 82.97      A    C  \\nATOM   1913  NZ  LYS A 236     -21.515  22.316  -4.309  1.00 75.07      A    N1+\\nATOM   1914  N   PHE A 237     -16.273  18.166  -9.801  1.00 89.40      A    N  \\nATOM   1915  CA  PHE A 237     -15.384  17.026  -9.961  1.00 89.97      A    C  \\nATOM   1916  C   PHE A 237     -14.109  17.437 -10.662  1.00 89.40      A    C  \\nATOM   1917  O   PHE A 237     -14.115  17.977 -11.754  1.00 88.38      A    O  \\nATOM   1918  CB  PHE A 237     -16.083  15.905 -10.745  1.00 89.70      A    C  \\nATOM   1919  CG  PHE A 237     -15.246  14.645 -10.875  1.00 90.00      A    C  \\nATOM   1920  CD1 PHE A 237     -15.129  13.778  -9.801  1.00 89.34      A    C  \\nATOM   1921  CD2 PHE A 237     -14.595  14.349 -12.061  1.00 89.03      A    C  \\nATOM   1922  CE1 PHE A 237     -14.365  12.631  -9.907  1.00 88.31      A    C  \\nATOM   1923  CE2 PHE A 237     -13.828  13.204 -12.165  1.00 88.53      A    C  \\nATOM   1924  CZ  PHE A 237     -13.732  12.343 -11.091  1.00 87.99      A    C  \\nATOM   1925  N   LEU A 238     -12.987  17.157 -10.006  1.00 89.04      A    N  \\nATOM   1926  CA  LEU A 238     -11.676  17.479 -10.536  1.00 88.86      A    C  \\nATOM   1927  C   LEU A 238     -10.945  16.205 -10.913  1.00 89.13      A    C  \\nATOM   1928  O   LEU A 238     -10.765  15.325 -10.056  1.00 88.09      A    O  \\nATOM   1929  CB  LEU A 238     -10.859  18.268  -9.524  1.00 87.45      A    C  \\nATOM   1930  CG  LEU A 238     -11.365  19.674  -9.191  1.00 85.41      A    C  \\nATOM   1931  CD1 LEU A 238     -11.366  20.543 -10.428  1.00 82.73      A    C  \\nATOM   1932  CD2 LEU A 238     -10.516  20.298  -8.106  1.00 83.28      A    C  \\nATOM   1933  N   THR A 239     -10.525  16.097 -12.145  1.00 88.08      A    N  \\nATOM   1934  CA  THR A 239      -9.817  14.918 -12.614  1.00 87.88      A    C  \\nATOM   1935  C   THR A 239      -8.347  15.041 -12.237  1.00 87.40      A    C  \\nATOM   1936  O   THR A 239      -7.676  16.016 -12.579  1.00 85.11      A    O  \\nATOM   1937  CB  THR A 239      -9.966  14.735 -14.124  1.00 86.76      A    C  \\nATOM   1938  CG2 THR A 239     -11.430  14.545 -14.474  1.00 79.99      A    C  \\nATOM   1939  OG1 THR A 239      -9.469  15.916 -14.788  1.00 79.25      A    O  \\nATOM   1940  N   THR A 240      -7.867  14.031 -11.513  1.00 88.01      A    N  \\nATOM   1941  CA  THR A 240      -6.468  13.974 -11.111  1.00 87.27      A    C  \\nATOM   1942  C   THR A 240      -5.831  12.747 -11.755  1.00 85.89      A    C  \\nATOM   1943  O   THR A 240      -6.374  11.652 -11.658  1.00 83.43      A    O  \\nATOM   1944  CB  THR A 240      -6.286  13.937  -9.611  1.00 87.25      A    C  \\nATOM   1945  CG2 THR A 240      -6.911  15.179  -8.979  1.00 84.13      A    C  \\nATOM   1946  OG1 THR A 240      -6.927  12.775  -9.061  1.00 83.60      A    O  \\nATOM   1947  N   GLN A 241      -4.696  12.951 -12.369  1.00 83.03      A    N  \\nATOM   1948  CA  GLN A 241      -4.007  11.873 -13.039  1.00 81.17      A    C  \\nATOM   1949  C   GLN A 241      -3.257  11.015 -12.017  1.00 80.97      A    C  \\nATOM   1950  O   GLN A 241      -2.405  11.515 -11.290  1.00 79.25      A    O  \\nATOM   1951  CB  GLN A 241      -3.042  12.413 -14.098  1.00 77.72      A    C  \\nATOM   1952  CG  GLN A 241      -3.712  13.141 -15.228  1.00 69.18      A    C  \\nATOM   1953  CD  GLN A 241      -2.726  13.655 -16.258  1.00 63.22      A    C  \\nATOM   1954  NE2 GLN A 241      -3.231  14.167 -17.391  1.00 54.57      A    N  \\nATOM   1955  OE1 GLN A 241      -1.504  13.599 -16.061  1.00 57.66      A    O  \\nATOM   1956  N   ARG A 242      -3.573   9.740 -11.992  1.00 76.15      A    N  \\nATOM   1957  CA  ARG A 242      -2.932   8.824 -11.091  1.00 74.92      A    C  \\nATOM   1958  C   ARG A 242      -1.673   8.257 -11.758  1.00 75.58      A    C  \\nATOM   1959  O   ARG A 242      -1.485   8.356 -12.973  1.00 73.92      A    O  \\nATOM   1960  CB  ARG A 242      -3.853   7.659 -10.684  1.00 71.50      A    C  \\nATOM   1961  CG  ARG A 242      -5.043   8.130  -9.860  1.00 65.85      A    C  \\nATOM   1962  CD  ARG A 242      -5.933   6.949  -9.471  1.00 63.31      A    C  \\nATOM   1963  NE  ARG A 242      -5.282   6.091  -8.488  1.00 59.55      A    N  \\nATOM   1964  CZ  ARG A 242      -5.831   4.971  -8.035  1.00 57.74      A    C  \\nATOM   1965  NH1 ARG A 242      -7.029   4.590  -8.472  1.00 50.34      A    N1+\\nATOM   1966  NH2 ARG A 242      -5.226   4.256  -7.108  1.00 49.99      A    N  \\nATOM   1967  N   THR A 243      -0.810   7.705 -10.942  1.00 78.58      A    N  \\nATOM   1968  CA  THR A 243       0.404   7.082 -11.441  1.00 78.39      A    C  \\nATOM   1969  C   THR A 243       0.099   5.696 -11.996  1.00 78.41      A    C  \\nATOM   1970  O   THR A 243      -0.946   5.103 -11.723  1.00 75.16      A    O  \\nATOM   1971  CB  THR A 243       1.465   6.978 -10.356  1.00 76.95      A    C  \\nATOM   1972  CG2 THR A 243       1.792   8.356  -9.801  1.00 73.99      A    C  \\nATOM   1973  OG1 THR A 243       0.994   6.150  -9.305  1.00 75.30      A    O  \\nATOM   1974  N   ASN A 244       1.018   5.188 -12.770  1.00 76.38      A    N  \\nATOM   1975  CA  ASN A 244       0.856   3.872 -13.382  1.00 75.61      A    C  \\nATOM   1976  C   ASN A 244       1.718   2.850 -12.684  1.00 76.08      A    C  \\nATOM   1977  O   ASN A 244       2.913   3.057 -12.503  1.00 72.61      A    O  \\nATOM   1978  CB  ASN A 244       1.206   3.920 -14.867  1.00 71.02      A    C  \\nATOM   1979  CG  ASN A 244       0.208   4.740 -15.670  1.00 64.22      A    C  \\nATOM   1980  ND2 ASN A 244       0.720   5.519 -16.623  1.00 58.17      A    N  \\nATOM   1981  OD1 ASN A 244      -0.987   4.667 -15.453  1.00 55.86      A    O  \\nATOM   1982  N   GLY A 245       1.087   1.759 -12.254  1.00 75.88      A    N  \\nATOM   1983  CA  GLY A 245       1.829   0.694 -11.614  1.00 74.74      A    C  \\nATOM   1984  C   GLY A 245       2.090   0.904 -10.124  1.00 76.84      A    C  \\nATOM   1985  O   GLY A 245       2.727   0.073  -9.509  1.00 74.30      A    O  \\nATOM   1986  N   ILE A 246       1.613   2.000  -9.569  1.00 80.18      A    N  \\nATOM   1987  CA  ILE A 246       1.820   2.303  -8.155  1.00 80.38      A    C  \\nATOM   1988  C   ILE A 246       0.483   2.480  -7.508  1.00 81.52      A    C  \\nATOM   1989  O   ILE A 246      -0.211   3.468  -7.738  1.00 79.37      A    O  \\nATOM   1990  CB  ILE A 246       2.690   3.555  -7.954  1.00 78.46      A    C  \\nATOM   1991  CG1 ILE A 246       4.073   3.359  -8.591  1.00 69.47      A    C  \\nATOM   1992  CG2 ILE A 246       2.848   3.849  -6.477  1.00 71.92      A    C  \\nATOM   1993  CD1 ILE A 246       4.918   4.605  -8.568  1.00 65.94      A    C  \\nATOM   1994  N   SER A 247       0.110   1.514  -6.710  1.00 82.00      A    N  \\nATOM   1995  CA  SER A 247      -1.152   1.524  -5.975  1.00 82.70      A    C  \\nATOM   1996  C   SER A 247      -1.006   0.667  -4.735  1.00 84.39      A    C  \\nATOM   1997  O   SER A 247       0.026   0.046  -4.493  1.00 83.53      A    O  \\nATOM   1998  CB  SER A 247      -2.307   1.021  -6.852  1.00 79.36      A    C  \\nATOM   1999  OG  SER A 247      -2.148  -0.355  -7.138  1.00 75.37      A    O  \\nATOM   2000  N   THR A 248      -2.053   0.657  -3.931  1.00 84.93      A    N  \\nATOM   2001  CA  THR A 248      -2.034  -0.193  -2.757  1.00 85.21      A    C  \\nATOM   2002  C   THR A 248      -2.144  -1.647  -3.111  1.00 84.62      A    C  \\nATOM   2003  O   THR A 248      -1.620  -2.512  -2.414  1.00 83.76      A    O  \\nATOM   2004  CB  THR A 248      -3.164   0.198  -1.810  1.00 86.03      A    C  \\nATOM   2005  CG2 THR A 248      -3.000   1.622  -1.364  1.00 82.93      A    C  \\nATOM   2006  OG1 THR A 248      -4.433   0.052  -2.483  1.00 82.04      A    O  \\nATOM   2007  N   SER A 249      -2.822  -1.941  -4.206  1.00 81.59      A    N  \\nATOM   2008  CA  SER A 249      -2.971  -3.326  -4.639  1.00 79.69      A    C  \\nATOM   2009  C   SER A 249      -1.667  -3.856  -5.225  1.00 79.49      A    C  \\nATOM   2010  O   SER A 249      -1.390  -5.057  -5.133  1.00 78.62      A    O  \\nATOM   2011  CB  SER A 249      -4.091  -3.459  -5.662  1.00 77.41      A    C  \\nATOM   2012  OG  SER A 249      -5.345  -3.115  -5.074  1.00 71.30      A    O  \\nATOM   2013  N   ASP A 250      -0.895  -2.984  -5.819  1.00 82.56      A    N  \\nATOM   2014  CA  ASP A 250       0.383  -3.419  -6.370  1.00 80.92      A    C  \\nATOM   2015  C   ASP A 250       1.376  -3.754  -5.279  1.00 81.52      A    C  \\nATOM   2016  O   ASP A 250       2.236  -4.624  -5.461  1.00 80.71      A    O  \\nATOM   2017  CB  ASP A 250       0.970  -2.344  -7.289  1.00 78.81      A    C  \\nATOM   2018  CG  ASP A 250       0.137  -2.140  -8.537  1.00 76.14      A    C  \\nATOM   2019  OD1 ASP A 250      -0.465  -3.123  -9.015  1.00 73.79      A    O  \\nATOM   2020  OD2 ASP A 250       0.063  -0.995  -9.042  1.00 73.33      A    O1-\\nATOM   2021  N   ILE A 251       1.269  -3.058  -4.156  1.00 85.18      A    N  \\nATOM   2022  CA  ILE A 251       2.151  -3.344  -3.015  1.00 85.52      A    C  \\nATOM   2023  C   ILE A 251       1.803  -4.704  -2.420  1.00 84.61      A    C  \\nATOM   2024  O   ILE A 251       2.672  -5.478  -2.045  1.00 83.98      A    O  \\nATOM   2025  CB  ILE A 251       2.081  -2.220  -1.977  1.00 87.12      A    C  \\nATOM   2026  CG1 ILE A 251       2.534  -0.913  -2.602  1.00 87.33      A    C  \\nATOM   2027  CG2 ILE A 251       2.940  -2.545  -0.776  1.00 87.10      A    C  \\nATOM   2028  CD1 ILE A 251       2.247   0.295  -1.740  1.00 86.03      A    C  \\nATOM   2029  N   ILE A 252       0.502  -4.996  -2.338  1.00 82.76      A    N  \\nATOM   2030  CA  ILE A 252       0.058  -6.268  -1.809  1.00 81.65      A    C  \\nATOM   2031  C   ILE A 252       0.469  -7.412  -2.728  1.00 80.21      A    C  \\nATOM   2032  O   ILE A 252       0.884  -8.479  -2.253  1.00 78.67      A    O  \\nATOM   2033  CB  ILE A 252      -1.469  -6.251  -1.561  1.00 82.09      A    C  \\nATOM   2034  CG1 ILE A 252      -1.801  -5.202  -0.495  1.00 83.18      A    C  \\nATOM   2035  CG2 ILE A 252      -1.983  -7.606  -1.130  1.00 82.03      A    C  \\nATOM   2036  CD1 ILE A 252      -3.267  -4.930  -0.372  1.00 81.74      A    C  \\nATOM   2037  N   THR A 253       0.368  -7.206  -4.015  1.00 77.36      A    N  \\nATOM   2038  CA  THR A 253       0.800  -8.230  -4.952  1.00 74.75      A    C  \\nATOM   2039  C   THR A 253       2.301  -8.440  -4.932  1.00 74.22      A    C  \\nATOM   2040  O   THR A 253       2.780  -9.529  -5.178  1.00 72.97      A    O  \\nATOM   2041  CB  THR A 253       0.339  -7.921  -6.369  1.00 71.24      A    C  \\nATOM   2042  CG2 THR A 253      -1.171  -7.883  -6.429  1.00 68.06      A    C  \\nATOM   2043  OG1 THR A 253       0.854  -6.635  -6.765  1.00 71.11      A    O  \\nATOM   2044  N   LYS A 254       3.042  -7.379  -4.615  1.00 78.36      A    N  \\nATOM   2045  CA  LYS A 254       4.488  -7.518  -4.479  1.00 77.40      A    C  \\nATOM   2046  C   LYS A 254       4.854  -8.287  -3.217  1.00 77.14      A    C  \\nATOM   2047  O   LYS A 254       5.897  -8.953  -3.177  1.00 76.75      A    O  \\nATOM   2048  CB  LYS A 254       5.183  -6.150  -4.466  1.00 75.96      A    C  \\nATOM   2049  CG  LYS A 254       5.216  -5.491  -5.828  1.00 70.50      A    C  \\nATOM   2050  CD  LYS A 254       5.860  -4.122  -5.746  1.00 67.41      A    C  \\nATOM   2051  CE  LYS A 254       5.933  -3.456  -7.097  1.00 62.76      A    C  \\nATOM   2052  NZ  LYS A 254       6.564  -2.105  -7.029  1.00 56.34      A    N1+\\nATOM   2053  N   ILE A 255       4.025  -8.168  -2.198  1.00 78.24      A    N  \\nATOM   2054  CA  ILE A 255       4.289  -8.912  -0.966  1.00 78.25      A    C  \\nATOM   2055  C   ILE A 255       3.921 -10.377  -1.153  1.00 77.19      A    C  \\nATOM   2056  O   ILE A 255       4.651 -11.273  -0.708  1.00 76.50      A    O  \\nATOM   2057  CB  ILE A 255       3.509  -8.286   0.223  1.00 79.22      A    C  \\nATOM   2058  CG1 ILE A 255       4.056  -6.889   0.480  1.00 80.53      A    C  \\nATOM   2059  CG2 ILE A 255       3.649  -9.110   1.487  1.00 79.89      A    C  \\nATOM   2060  CD1 ILE A 255       3.192  -6.064   1.397  1.00 79.49      A    C  \\nATOM   2061  N   ILE A 256       2.805 -10.640  -1.834  1.00 72.72      A    N  \\nATOM   2062  CA  ILE A 256       2.333 -12.002  -2.043  1.00 70.40      A    C  \\nATOM   2063  C   ILE A 256       3.266 -12.746  -2.976  1.00 70.30      A    C  \\nATOM   2064  O   ILE A 256       3.592 -13.910  -2.746  1.00 68.44      A    O  \\nATOM   2065  CB  ILE A 256       0.890 -11.991  -2.571  1.00 67.62      A    C  \\nATOM   2066  CG1 ILE A 256      -0.052 -11.510  -1.453  1.00 69.21      A    C  \\nATOM   2067  CG2 ILE A 256       0.467 -13.371  -3.031  1.00 65.91      A    C  \\nATOM   2068  CD1 ILE A 256      -1.435 -11.198  -1.950  1.00 64.14      A    C  \\nATOM   2069  N   ARG A 257       3.706 -12.097  -4.042  1.00 68.31      A    N  \\nATOM   2070  CA  ARG A 257       4.628 -12.767  -4.977  1.00 67.99      A    C  \\nATOM   2071  C   ARG A 257       5.937 -13.165  -4.293  1.00 68.80      A    C  \\nATOM   2072  O   ARG A 257       6.436 -14.270  -4.509  1.00 67.01      A    O  \\nATOM   2073  CB  ARG A 257       4.930 -11.832  -6.171  1.00 64.42      A    C  \\nATOM   2074  CG  ARG A 257       3.790 -11.654  -7.115  1.00 58.91      A    C  \\nATOM   2075  CD  ARG A 257       4.106 -10.638  -8.179  1.00 54.92      A    C  \\nATOM   2076  NE  ARG A 257       2.998 -10.422  -9.115  1.00 48.67      A    N  \\nATOM   2077  CZ  ARG A 257       3.015  -9.560 -10.106  1.00 47.19      A    C  \\nATOM   2078  NH1 ARG A 257       4.088  -8.820 -10.324  1.00 41.47      A    N1+\\nATOM   2079  NH2 ARG A 257       1.953  -9.446 -10.903  1.00 39.56      A    N  \\nATOM   2080  N   ASP A 258       6.471 -12.276  -3.472  1.00 69.78      A    N  \\nATOM   2081  CA  ASP A 258       7.696 -12.570  -2.736  1.00 69.26      A    C  \\nATOM   2082  C   ASP A 258       7.393 -12.497  -1.247  1.00 69.86      A    C  \\nATOM   2083  O   ASP A 258       7.854 -11.593  -0.541  1.00 68.94      A    O  \\nATOM   2084  CB  ASP A 258       8.809 -11.603  -3.129  1.00 67.03      A    C  \\nATOM   2085  CG  ASP A 258       9.293 -11.805  -4.552  1.00 64.99      A    C  \\nATOM   2086  OD1 ASP A 258       9.280 -12.936  -5.024  1.00 62.97      A    O  \\nATOM   2087  OD2 ASP A 258       9.690 -10.792  -5.181  1.00 64.14      A    O1-\\nATOM   2088  N   TYR A 259       6.608 -13.472  -0.794  1.00 68.08      A    N  \\nATOM   2089  CA  TYR A 259       6.250 -13.480   0.624  1.00 67.82      A    C  \\nATOM   2090  C   TYR A 259       7.410 -13.977   1.489  1.00 68.71      A    C  \\nATOM   2091  O   TYR A 259       7.649 -13.480   2.579  1.00 68.28      A    O  \\nATOM   2092  CB  TYR A 259       5.006 -14.343   0.839  1.00 64.70      A    C  \\nATOM   2093  CG  TYR A 259       4.576 -14.431   2.292  1.00 62.03      A    C  \\nATOM   2094  CD1 TYR A 259       3.945 -13.350   2.895  1.00 58.04      A    C  \\nATOM   2095  CD2 TYR A 259       4.790 -15.581   3.035  1.00 58.46      A    C  \\nATOM   2096  CE1 TYR A 259       3.530 -13.399   4.191  1.00 56.28      A    C  \\nATOM   2097  CE2 TYR A 259       4.383 -15.636   4.355  1.00 56.44      A    C  \\nATOM   2098  CZ  TYR A 259       3.757 -14.540   4.927  1.00 54.63      A    C  \\nATOM   2099  OH  TYR A 259       3.348 -14.610   6.245  1.00 48.37      A    O  \\nATOM   2100  N   ASP A 260       8.090 -15.020   1.010  1.00 64.50      A    N  \\nATOM   2101  CA  ASP A 260       9.168 -15.632   1.798  1.00 63.88      A    C  \\nATOM   2102  C   ASP A 260      10.342 -14.700   1.983  1.00 64.90      A    C  \\nATOM   2103  O   ASP A 260      11.026 -14.758   3.017  1.00 63.91      A    O  \\nATOM   2104  CB  ASP A 260       9.582 -16.953   1.150  1.00 60.23      A    C  \\nATOM   2105  CG  ASP A 260       9.960 -16.789  -0.310  1.00 56.11      A    C  \\nATOM   2106  OD1 ASP A 260       9.865 -15.673  -0.857  1.00 52.59      A    O  \\nATOM   2107  OD2 ASP A 260      10.368 -17.799  -0.926  1.00 54.43      A    O1-\\nATOM   2108  N   LYS A 261      10.558 -13.827   1.027  1.00 68.39      A    N  \\nATOM   2109  CA  LYS A 261      11.681 -12.892   1.107  1.00 67.89      A    C  \\nATOM   2110  C   LYS A 261      11.474 -11.848   2.191  1.00 68.55      A    C  \\nATOM   2111  O   LYS A 261      12.398 -11.500   2.918  1.00 68.15      A    O  \\nATOM   2112  CB  LYS A 261      11.915 -12.214  -0.234  1.00 65.69      A    C  \\nATOM   2113  CG  LYS A 261      12.374 -13.158  -1.319  1.00 63.55      A    C  \\nATOM   2114  CD  LYS A 261      12.573 -12.442  -2.625  1.00 62.18      A    C  \\nATOM   2115  CE  LYS A 261      13.016 -13.405  -3.708  1.00 58.97      A    C  \\nATOM   2116  NZ  LYS A 261      13.209 -12.737  -5.036  1.00 52.66      A    N1+\\nATOM   2117  N   TYR A 262      10.246 -11.350   2.313  1.00 71.62      A    N  \\nATOM   2118  CA  TYR A 262       9.935 -10.387   3.367  1.00 71.51      A    C  \\nATOM   2119  C   TYR A 262       9.821 -11.067   4.727  1.00 70.73      A    C  \\nATOM   2120  O   TYR A 262      10.091 -10.445   5.745  1.00 71.49      A    O  \\nATOM   2121  CB  TYR A 262       8.671  -9.616   3.037  1.00 72.36      A    C  \\nATOM   2122  CG  TYR A 262       8.809  -8.723   1.827  1.00 73.62      A    C  \\nATOM   2123  CD1 TYR A 262       9.393  -7.490   1.933  1.00 73.90      A    C  \\nATOM   2124  CD2 TYR A 262       8.361  -9.134   0.582  1.00 73.44      A    C  \\nATOM   2125  CE1 TYR A 262       9.506  -6.656   0.829  1.00 70.83      A    C  \\nATOM   2126  CE2 TYR A 262       8.467  -8.334  -0.516  1.00 71.95      A    C  \\nATOM   2127  CZ  TYR A 262       9.061  -7.095  -0.408  1.00 71.70      A    C  \\nATOM   2128  OH  TYR A 262       9.153  -6.280  -1.486  1.00 66.28      A    O  \\nATOM   2129  N   LEU A 263       9.400 -12.329   4.707  1.00 62.29      A    N  \\nATOM   2130  CA  LEU A 263       9.295 -13.061   5.979  1.00 62.19      A    C  \\nATOM   2131  C   LEU A 263      10.692 -13.337   6.554  1.00 62.91      A    C  \\nATOM   2132  O   LEU A 263      10.896 -13.210   7.773  1.00 62.50      A    O  \\nATOM   2133  CB  LEU A 263       8.527 -14.366   5.762  1.00 59.52      A    C  \\nATOM   2134  CG  LEU A 263       8.304 -15.251   6.986  1.00 55.51      A    C  \\nATOM   2135  CD1 LEU A 263       7.456 -14.515   8.011  1.00 55.42      A    C  \\nATOM   2136  CD2 LEU A 263       7.643 -16.550   6.569  1.00 54.76      A    C  \\nATOM   2137  N   LEU A 264      11.615 -13.690   5.697  1.00 59.56      A    N  \\nATOM   2138  CA  LEU A 264      12.992 -13.896   6.160  1.00 59.52      A    C  \\nATOM   2139  C   LEU A 264      13.649 -12.602   6.574  1.00 60.33      A    C  \\nATOM   2140  O   LEU A 264      14.520 -12.618   7.453  1.00 60.31      A    O  \\nATOM   2141  CB  LEU A 264      13.800 -14.578   5.078  1.00 56.60      A    C  \\nATOM   2142  CG  LEU A 264      13.416 -16.026   4.752  1.00 52.70      A    C  \\nATOM   2143  CD1 LEU A 264      13.712 -16.914   5.954  1.00 52.85      A    C  \\nATOM   2144  CD2 LEU A 264      14.161 -16.518   3.523  1.00 51.87      A    C  \\nATOM   2145  N   ARG A 265      13.241 -11.499   5.989  1.00 64.10      A    N  \\nATOM   2146  CA  ARG A 265      13.768 -10.203   6.393  1.00 65.18      A    C  \\nATOM   2147  C   ARG A 265      13.275  -9.845   7.767  1.00 64.86      A    C  \\nATOM   2148  O   ARG A 265      14.011  -9.230   8.555  1.00 64.91      A    O  \\nATOM   2149  CB  ARG A 265      13.392  -9.141   5.342  1.00 64.09      A    C  \\nATOM   2150  CG  ARG A 265      14.029  -7.790   5.598  1.00 61.28      A    C  \\nATOM   2151  CD  ARG A 265      13.501  -6.783   4.580  1.00 60.42      A    C  \\nATOM   2152  NE  ARG A 265      12.090  -6.446   4.810  1.00 57.50      A    N  \\nATOM   2153  CZ  ARG A 265      11.396  -5.612   4.081  1.00 56.87      A    C  \\nATOM   2154  NH1 ARG A 265      11.963  -4.987   3.055  1.00 51.79      A    N1+\\nATOM   2155  NH2 ARG A 265      10.116  -5.378   4.349  1.00 50.06      A    N  \\nATOM   2156  N   ASN A 266      12.053 -10.206   8.079  1.00 66.96      A    N  \\nATOM   2157  CA  ASN A 266      11.492  -9.866   9.389  1.00 66.38      A    C  \\nATOM   2158  C   ASN A 266      12.146 -10.667  10.508  1.00 65.89      A    C  \\nATOM   2159  O   ASN A 266      12.335 -10.172  11.619  1.00 65.19      A    O  \\nATOM   2160  CB  ASN A 266       9.989 -10.077   9.398  1.00 64.69      A    C  \\nATOM   2161  CG  ASN A 266       9.260  -9.078   8.524  1.00 63.60      A    C  \\nATOM   2162  ND2 ASN A 266       8.231  -9.538   7.837  1.00 60.58      A    N  \\nATOM   2163  OD1 ASN A 266       9.608  -7.904   8.489  1.00 60.83      A    O  \\nATOM   2164  N   PHE A 267      12.497 -11.931  10.226  1.00 55.86      A    N  \\nATOM   2165  CA  PHE A 267      13.178 -12.712  11.223  1.00 55.60      A    C  \\nATOM   2166  C   PHE A 267      14.663 -12.354  11.291  1.00 56.04      A    C  \\nATOM   2167  O   PHE A 267      15.300 -12.574  12.317  1.00 56.08      A    O  \\nATOM   2168  CB  PHE A 267      13.032 -14.205  10.928  1.00 53.02      A    C  \\nATOM   2169  CG  PHE A 267      11.662 -14.749  11.233  1.00 50.33      A    C  \\nATOM   2170  CD1 PHE A 267      11.275 -14.956  12.543  1.00 47.99      A    C  \\nATOM   2171  CD2 PHE A 267      10.803 -15.045  10.213  1.00 47.33      A    C  \\nATOM   2172  CE1 PHE A 267      10.034 -15.474  12.829  1.00 45.28      A    C  \\nATOM   2173  CE2 PHE A 267       9.548 -15.552  10.494  1.00 45.75      A    C  \\nATOM   2174  CZ  PHE A 267       9.169 -15.762  11.809  1.00 42.07      A    C  \\nATOM   2175  N   ALA A 268      15.195 -11.812  10.224  1.00 57.41      A    N  \\nATOM   2176  CA  ALA A 268      16.603 -11.394  10.247  1.00 58.19      A    C  \\nATOM   2177  C   ALA A 268      16.783 -10.116  11.059  1.00 58.35      A    C  \\nATOM   2178  O   ALA A 268      17.778  -9.957  11.777  1.00 57.88      A    O  \\nATOM   2179  CB  ALA A 268      17.119 -11.162   8.837  1.00 55.74      A    C  \\nATOM   2180  N   ARG A 269      15.814  -9.231  10.982  1.00 61.80      A    N  \\nATOM   2181  CA  ARG A 269      15.868  -7.995  11.764  1.00 61.58      A    C  \\nATOM   2182  C   ARG A 269      15.577  -8.276  13.245  1.00 61.09      A    C  \\nATOM   2183  O   ARG A 269      16.133  -7.598  14.121  1.00 59.70      A    O  \\nATOM   2184  CB  ARG A 269      14.839  -6.982  11.236  1.00 59.01      A    C  \\nATOM   2185  CG  ARG A 269      15.216  -6.446   9.852  1.00 55.29      A    C  \\nATOM   2186  CD  ARG A 269      14.174  -5.478   9.312  1.00 51.42      A    C  \\nATOM   2187  NE  ARG A 269      14.132  -4.233  10.046  1.00 46.43      A    N  \\nATOM   2188  CZ  ARG A 269      13.141  -3.881  10.851  1.00 46.39      A    C  \\nATOM   2189  NH1 ARG A 269      12.084  -4.687  11.052  1.00 40.02      A    N1+\\nATOM   2190  NH2 ARG A 269      13.197  -2.708  11.459  1.00 39.13      A    N  \\nATOM   2191  N   GLY A 270      14.752  -9.255  13.492  1.00 53.37      A    N  \\nATOM   2192  CA  GLY A 270      14.481  -9.645  14.874  1.00 53.72      A    C  \\nATOM   2193  C   GLY A 270      15.634 -10.408  15.500  1.00 54.21      A    C  \\nATOM   2194  O   GLY A 270      15.789 -10.405  16.731  1.00 53.34      A    O  \\nATOM   2195  N   ALA A 271      16.411 -11.086  14.696  1.00 51.02      A    N  \\nATOM   2196  CA  ALA A 271      17.565 -11.816  15.199  1.00 51.49      A    C  \\nATOM   2197  C   ALA A 271      18.741 -10.896  15.483  1.00 52.25      A    C  \\nATOM   2198  O   ALA A 271      19.572 -11.210  16.354  1.00 51.27      A    O  \\nATOM   2199  CB  ALA A 271      17.982 -12.902  14.211  1.00 48.40      A    C  \\nATOM   2200  N   THR A 272      18.837  -9.800  14.767  1.00 52.10      A    N  \\nATOM   2201  CA  THR A 272      19.939  -8.878  15.005  1.00 52.72      A    C  \\nATOM   2202  C   THR A 272      19.663  -7.976  16.186  1.00 52.70      A    C  \\nATOM   2203  O   THR A 272      20.607  -7.533  16.855  1.00 51.68      A    O  \\nATOM   2204  CB  THR A 272      20.202  -8.038  13.757  1.00 49.80      A    C  \\nATOM   2205  CG2 THR A 272      20.619  -8.922  12.597  1.00 46.35      A    C  \\nATOM   2206  OG1 THR A 272      19.010  -7.309  13.390  1.00 48.37      A    O  \\nATOM   2207  N   ARG A 273      18.410  -7.679  16.457  1.00 53.75      A    N  \\nATOM   2208  CA  ARG A 273      18.063  -6.789  17.566  1.00 54.89      A    C  \\nATOM   2209  C   ARG A 273      17.998  -7.543  18.889  1.00 54.16      A    C  \\nATOM   2210  O   ARG A 273      18.423  -7.005  19.924  1.00 52.71      A    O  \\nATOM   2211  CB  ARG A 273      16.731  -6.097  17.299  1.00 52.50      A    C  \\nATOM   2212  CG  ARG A 273      16.764  -5.135  16.153  1.00 50.25      A    C  \\nATOM   2213  CD  ARG A 273      15.417  -4.520  15.891  1.00 47.08      A    C  \\nATOM   2214  NE  ARG A 273      15.455  -3.562  14.764  1.00 43.35      A    N  \\nATOM   2215  CZ  ARG A 273      14.394  -2.871  14.354  1.00 43.67      A    C  \\nATOM   2216  NH1 ARG A 273      13.226  -3.022  14.955  1.00 37.95      A    N1+\\nATOM   2217  NH2 ARG A 273      14.502  -2.015  13.350  1.00 36.03      A    N  \\nATOM   2218  N   GLN A 274      17.482  -8.753  18.871  1.00 48.86      A    N  \\nATOM   2219  CA  GLN A 274      17.340  -9.522  20.081  1.00 50.31      A    C  \\nATOM   2220  C   GLN A 274      18.563 -10.385  20.325  1.00 50.13      A    C  \\nATOM   2221  O   GLN A 274      19.382 -10.628  19.447  1.00 48.88      A    O  \\nATOM   2222  CB  GLN A 274      16.092 -10.408  20.021  1.00 47.53      A    C  \\nATOM   2223  CG  GLN A 274      14.793  -9.608  19.947  1.00 46.71      A    C  \\nATOM   2224  CD  GLN A 274      13.561 -10.483  19.835  1.00 44.54      A    C  \\nATOM   2225  NE2 GLN A 274      12.383  -9.856  19.870  1.00 39.59      A    N  \\nATOM   2226  OE1 GLN A 274      13.654 -11.707  19.728  1.00 41.99      A    O  \\nATOM   2227  N   GLU A 275      18.674 -10.891  21.549  1.00 46.67      A    N  \\nATOM   2228  CA  GLU A 275      19.783 -11.728  21.923  1.00 48.58      A    C  \\nATOM   2229  C   GLU A 275      19.464 -13.176  21.562  1.00 48.69      A    C  \\nATOM   2230  O   GLU A 275      18.341 -13.568  21.315  1.00 47.07      A    O  \\nATOM   2231  CB  GLU A 275      20.066 -11.609  23.403  1.00 45.47      A    C  \\nATOM   2232  CG  GLU A 275      20.535 -10.244  23.862  1.00 44.17      A    C  \\nATOM   2233  CD  GLU A 275      20.784 -10.182  25.353  1.00 41.37      A    C  \\nATOM   2234  OE1 GLU A 275      20.481 -11.175  26.045  1.00 39.37      A    O  \\nATOM   2235  OE2 GLU A 275      21.273  -9.125  25.851  1.00 37.50      A    O1-\\nATOM   2236  N   LEU A 276      20.521 -13.956  21.536  1.00 45.52      A    N  \\nATOM   2237  CA  LEU A 276      20.412 -15.380  21.197  1.00 48.78      A    C  \\nATOM   2238  C   LEU A 276      20.347 -16.203  22.487  1.00 49.53      A    C  \\nATOM   2239  O   LEU A 276      21.364 -16.455  23.117  1.00 48.06      A    O  \\nATOM   2240  CB  LEU A 276      21.582 -15.835  20.330  1.00 44.34      A    C  \\nATOM   2241  CG  LEU A 276      21.716 -15.126  18.975  1.00 42.71      A    C  \\nATOM   2242  CD1 LEU A 276      20.507 -15.455  18.109  1.00 42.89      A    C  \\nATOM   2243  CD2 LEU A 276      23.000 -15.566  18.256  1.00 41.95      A    C  \\nATOM   2244  N   ASN A 277      19.168 -16.612  22.840  1.00 47.85      A    N  \\nATOM   2245  CA  ASN A 277      18.928 -17.360  24.053  1.00 50.43      A    C  \\nATOM   2246  C   ASN A 277      18.034 -18.548  23.759  1.00 51.25      A    C  \\nATOM   2247  O   ASN A 277      17.549 -18.729  22.637  1.00 50.30      A    O  \\nATOM   2248  CB  ASN A 277      18.303 -16.478  25.142  1.00 46.59      A    C  \\nATOM   2249  CG  ASN A 277      19.258 -15.446  25.703  1.00 45.14      A    C  \\nATOM   2250  ND2 ASN A 277      18.760 -14.244  25.947  1.00 42.40      A    N  \\nATOM   2251  OD1 ASN A 277      20.440 -15.734  25.909  1.00 40.90      A    O  \\nATOM   2252  N   VAL A 278      17.846 -19.358  24.764  1.00 54.41      A    N  \\nATOM   2253  CA  VAL A 278      16.972 -20.534  24.619  1.00 56.14      A    C  \\nATOM   2254  C   VAL A 278      15.515 -20.099  24.537  1.00 57.77      A    C  \\nATOM   2255  O   VAL A 278      14.773 -20.554  23.654  1.00 57.22      A    O  \\nATOM   2256  CB  VAL A 278      17.181 -21.495  25.793  1.00 53.31      A    C  \\nATOM   2257  CG1 VAL A 278      18.627 -21.997  25.806  1.00 54.36      A    C  \\nATOM   2258  CG2 VAL A 278      16.237 -22.692  25.686  1.00 52.18      A    C  \\nATOM   2259  N   SER A 279      15.116 -19.202  25.412  1.00 57.85      A    N  \\nATOM   2260  CA  SER A 279      13.743 -18.695  25.397  1.00 59.69      A    C  \\nATOM   2261  C   SER A 279      13.483 -17.827  24.189  1.00 60.29      A    C  \\nATOM   2262  O   SER A 279      12.332 -17.724  23.734  1.00 59.88      A    O  \\nATOM   2263  CB  SER A 279      13.454 -17.918  26.686  1.00 57.67      A    C  \\nATOM   2264  OG  SER A 279      14.213 -16.750  26.734  1.00 52.89      A    O  \\nATOM   2265  N   TRP A 280      14.503 -17.196  23.665  1.00 54.83      A    N  \\nATOM   2266  CA  TRP A 280      14.342 -16.408  22.470  1.00 56.36      A    C  \\nATOM   2267  C   TRP A 280      14.158 -17.309  21.241  1.00 57.04      A    C  \\nATOM   2268  O   TRP A 280      13.395 -17.000  20.339  1.00 55.89      A    O  \\nATOM   2269  CB  TRP A 280      15.556 -15.473  22.257  1.00 53.25      A    C  \\nATOM   2270  CG  TRP A 280      15.426 -14.594  21.054  1.00 53.02      A    C  \\nATOM   2271  CD1 TRP A 280      14.660 -13.492  20.974  1.00 49.92      A    C  \\nATOM   2272  CD2 TRP A 280      16.079 -14.774  19.801  1.00 51.60      A    C  \\nATOM   2273  CE2 TRP A 280      15.657 -13.755  18.990  1.00 49.22      A    C  \\nATOM   2274  CE3 TRP A 280      16.962 -15.731  19.321  1.00 45.24      A    C  \\nATOM   2275  NE1 TRP A 280      14.808 -12.948  19.701  1.00 47.01      A    N  \\nATOM   2276  CZ2 TRP A 280      16.099 -13.637  17.671  1.00 44.61      A    C  \\nATOM   2277  CZ3 TRP A 280      17.397 -15.606  18.016  1.00 45.83      A    C  \\nATOM   2278  CH2 TRP A 280      16.971 -14.563  17.206  1.00 44.46      A    C  \\nATOM   2279  N   LEU A 281      14.856 -18.438  21.227  1.00 52.63      A    N  \\nATOM   2280  CA  LEU A 281      14.692 -19.385  20.130  1.00 54.49      A    C  \\nATOM   2281  C   LEU A 281      13.371 -20.135  20.222  1.00 55.57      A    C  \\nATOM   2282  O   LEU A 281      12.831 -20.526  19.179  1.00 55.42      A    O  \\nATOM   2283  CB  LEU A 281      15.862 -20.371  20.100  1.00 50.89      A    C  \\nATOM   2284  CG  LEU A 281      17.229 -19.813  19.694  1.00 48.21      A    C  \\nATOM   2285  CD1 LEU A 281      17.205 -19.321  18.252  1.00 46.77      A    C  \\nATOM   2286  CD2 LEU A 281      18.316 -20.861  19.896  1.00 46.35      A    C  \\nATOM   2287  N   LYS A 282      12.880 -20.332  21.413  1.00 55.98      A    N  \\nATOM   2288  CA  LYS A 282      11.577 -20.989  21.547  1.00 57.77      A    C  \\nATOM   2289  C   LYS A 282      10.450 -20.105  21.047  1.00 59.00      A    C  \\nATOM   2290  O   LYS A 282       9.518 -20.575  20.414  1.00 59.18      A    O  \\nATOM   2291  CB  LYS A 282      11.309 -21.345  23.017  1.00 54.84      A    C  \\nATOM   2292  CG  LYS A 282      12.151 -22.501  23.552  1.00 52.06      A    C  \\nATOM   2293  CD  LYS A 282      11.827 -22.774  25.003  1.00 48.89      A    C  \\nATOM   2294  CE  LYS A 282      12.679 -23.907  25.549  1.00 45.72      A    C  \\nATOM   2295  NZ  LYS A 282      12.405 -24.160  26.972  1.00 40.61      A    N1+\\nATOM   2296  N   LYS A 283      10.573 -18.816  21.290  1.00 56.67      A    N  \\nATOM   2297  CA  LYS A 283       9.575 -17.890  20.776  1.00 58.11      A    C  \\nATOM   2298  C   LYS A 283       9.689 -17.678  19.281  1.00 59.04      A    C  \\nATOM   2299  O   LYS A 283       8.677 -17.466  18.605  1.00 58.50      A    O  \\nATOM   2300  CB  LYS A 283       9.676 -16.535  21.519  1.00 54.96      A    C  \\nATOM   2301  CG  LYS A 283       9.301 -16.584  22.954  1.00 52.26      A    C  \\nATOM   2302  CD  LYS A 283       9.461 -15.233  23.614  1.00 48.79      A    C  \\nATOM   2303  CE  LYS A 283       9.105 -15.291  25.085  1.00 44.26      A    C  \\nATOM   2304  NZ  LYS A 283       9.240 -14.009  25.770  1.00 39.34      A    N1+\\nATOM   2305  N   ASN A 284      10.897 -17.789  18.766  1.00 53.57      A    N  \\nATOM   2306  CA  ASN A 284      11.070 -17.691  17.337  1.00 55.79      A    C  \\nATOM   2307  C   ASN A 284      10.558 -18.925  16.617  1.00 56.56      A    C  \\nATOM   2308  O   ASN A 284      10.040 -18.812  15.501  1.00 55.63      A    O  \\nATOM   2309  CB  ASN A 284      12.558 -17.467  16.989  1.00 53.30      A    C  \\nATOM   2310  CG  ASN A 284      13.017 -16.061  17.336  1.00 50.93      A    C  \\nATOM   2311  ND2 ASN A 284      14.232 -15.970  17.829  1.00 48.84      A    N  \\nATOM   2312  OD1 ASN A 284      12.291 -15.100  17.153  1.00 47.07      A    O  \\nATOM   2313  N   GLU A 285      10.671 -20.092  17.259  1.00 54.31      A    N  \\nATOM   2314  CA  GLU A 285      10.121 -21.291  16.656  1.00 56.08      A    C  \\nATOM   2315  C   GLU A 285       8.596 -21.280  16.720  1.00 57.31      A    C  \\nATOM   2316  O   GLU A 285       7.939 -21.780  15.800  1.00 57.18      A    O  \\nATOM   2317  CB  GLU A 285      10.670 -22.544  17.320  1.00 53.46      A    C  \\nATOM   2318  CG  GLU A 285      12.154 -22.784  17.092  1.00 50.08      A    C  \\nATOM   2319  CD  GLU A 285      12.470 -23.196  15.677  1.00 46.15      A    C  \\nATOM   2320  OE1 GLU A 285      11.633 -23.886  15.052  1.00 44.02      A    O  \\nATOM   2321  OE2 GLU A 285      13.569 -22.850  15.177  1.00 41.84      A    O1-\\nATOM   2322  N   LEU A 286       8.044 -20.748  17.776  1.00 63.69      A    N  \\nATOM   2323  CA  LEU A 286       6.607 -20.649  17.898  1.00 66.01      A    C  \\nATOM   2324  C   LEU A 286       6.039 -19.640  16.898  1.00 67.24      A    C  \\nATOM   2325  O   LEU A 286       4.966 -19.846  16.339  1.00 67.91      A    O  \\nATOM   2326  CB  LEU A 286       6.167 -20.323  19.310  1.00 64.15      A    C  \\nATOM   2327  CG  LEU A 286       6.446 -21.392  20.370  1.00 59.92      A    C  \\nATOM   2328  CD1 LEU A 286       5.653 -22.655  20.079  1.00 60.30      A    C  \\nATOM   2329  CD2 LEU A 286       6.103 -20.857  21.753  1.00 59.50      A    C  \\nATOM   2330  N   GLU A 287       6.737 -18.542  16.723  1.00 62.38      A    N  \\nATOM   2331  CA  GLU A 287       6.311 -17.571  15.743  1.00 63.81      A    C  \\nATOM   2332  C   GLU A 287       6.545 -18.056  14.332  1.00 64.38      A    C  \\nATOM   2333  O   GLU A 287       5.842 -17.688  13.402  1.00 64.68      A    O  \\nATOM   2334  CB  GLU A 287       7.005 -16.235  15.981  1.00 61.76      A    C  \\nATOM   2335  CG  GLU A 287       6.558 -15.519  17.232  1.00 57.60      A    C  \\nATOM   2336  CD  GLU A 287       7.264 -14.197  17.459  1.00 53.13      A    C  \\nATOM   2337  OE1 GLU A 287       8.175 -13.855  16.661  1.00 51.52      A    O  \\nATOM   2338  OE2 GLU A 287       6.937 -13.502  18.447  1.00 48.15      A    O1-\\nATOM   2339  N   PHE A 288       7.553 -18.905  14.174  1.00 53.50      A    N  \\nATOM   2340  CA  PHE A 288       7.819 -19.502  12.868  1.00 55.28      A    C  \\nATOM   2341  C   PHE A 288       6.712 -20.448  12.458  1.00 56.85      A    C  \\nATOM   2342  O   PHE A 288       6.269 -20.452  11.305  1.00 57.15      A    O  \\nATOM   2343  CB  PHE A 288       9.167 -20.244  12.908  1.00 52.18      A    C  \\nATOM   2344  CG  PHE A 288       9.519 -20.952  11.634  1.00 49.57      A    C  \\nATOM   2345  CD1 PHE A 288       9.954 -20.235  10.537  1.00 46.85      A    C  \\nATOM   2346  CD2 PHE A 288       9.438 -22.326  11.545  1.00 45.86      A    C  \\nATOM   2347  CE1 PHE A 288      10.287 -20.878   9.362  1.00 43.75      A    C  \\nATOM   2348  CE2 PHE A 288       9.780 -22.962  10.379  1.00 42.78      A    C  \\nATOM   2349  CZ  PHE A 288      10.221 -22.250   9.282  1.00 40.83      A    C  \\nATOM   2350  N   LYS A 289       6.243 -21.283  13.396  1.00 58.48      A    N  \\nATOM   2351  CA  LYS A 289       5.143 -22.199  13.131  1.00 59.79      A    C  \\nATOM   2352  C   LYS A 289       3.842 -21.461  12.907  1.00 60.99      A    C  \\nATOM   2353  O   LYS A 289       2.993 -21.918  12.159  1.00 61.42      A    O  \\nATOM   2354  CB  LYS A 289       4.994 -23.210  14.268  1.00 57.02      A    C  \\nATOM   2355  CG  LYS A 289       6.164 -24.169  14.358  1.00 53.86      A    C  \\nATOM   2356  CD  LYS A 289       5.986 -25.165  15.480  1.00 49.88      A    C  \\nATOM   2357  CE  LYS A 289       7.157 -26.099  15.617  1.00 47.03      A    C  \\nATOM   2358  NZ  LYS A 289       7.000 -27.050  16.729  1.00 42.86      A    N1+\\nATOM   2359  N   LYS A 290       3.674 -20.333  13.572  1.00 65.50      A    N  \\nATOM   2360  CA  LYS A 290       2.479 -19.538  13.373  1.00 66.36      A    C  \\nATOM   2361  C   LYS A 290       2.460 -18.850  12.027  1.00 67.04      A    C  \\nATOM   2362  O   LYS A 290       1.404 -18.744  11.399  1.00 68.01      A    O  \\nATOM   2363  CB  LYS A 290       2.317 -18.499  14.489  1.00 64.94      A    C  \\nATOM   2364  CG  LYS A 290       1.958 -19.091  15.822  1.00 61.33      A    C  \\nATOM   2365  CD  LYS A 290       1.834 -18.012  16.876  1.00 58.35      A    C  \\nATOM   2366  CE  LYS A 290       1.460 -18.602  18.229  1.00 54.25      A    C  \\nATOM   2367  NZ  LYS A 290       1.337 -17.569  19.273  1.00 47.76      A    N1+\\nATOM   2368  N   HIS A 291       3.612 -18.438  11.554  1.00 57.56      A    N  \\nATOM   2369  CA  HIS A 291       3.679 -17.860  10.221  1.00 58.73      A    C  \\nATOM   2370  C   HIS A 291       3.550 -18.897   9.135  1.00 59.51      A    C  \\nATOM   2371  O   HIS A 291       3.022 -18.611   8.064  1.00 59.35      A    O  \\nATOM   2372  CB  HIS A 291       4.987 -17.081  10.042  1.00 55.70      A    C  \\nATOM   2373  CG  HIS A 291       5.063 -15.814  10.840  1.00 52.91      A    C  \\nATOM   2374  CD2 HIS A 291       5.881 -15.463  11.861  1.00 50.18      A    C  \\nATOM   2375  ND1 HIS A 291       4.231 -14.756  10.623  1.00 51.69      A    N  \\nATOM   2376  CE1 HIS A 291       4.525 -13.791  11.479  1.00 49.34      A    C  \\nATOM   2377  NE2 HIS A 291       5.515 -14.184  12.234  1.00 47.23      A    N  \\nATOM   2378  N   ILE A 292       4.006 -20.131   9.408  1.00 56.14      A    N  \\nATOM   2379  CA  ILE A 292       3.827 -21.207   8.438  1.00 57.03      A    C  \\nATOM   2380  C   ILE A 292       2.369 -21.625   8.391  1.00 58.34      A    C  \\nATOM   2381  O   ILE A 292       1.813 -21.910   7.329  1.00 58.65      A    O  \\nATOM   2382  CB  ILE A 292       4.771 -22.377   8.730  1.00 53.81      A    C  \\nATOM   2383  CG1 ILE A 292       6.218 -21.958   8.473  1.00 54.85      A    C  \\nATOM   2384  CG2 ILE A 292       4.456 -23.582   7.872  1.00 52.15      A    C  \\nATOM   2385  CD1 ILE A 292       7.234 -22.964   8.936  1.00 49.01      A    C  \\nATOM   2386  N   LYS A 293       1.737 -21.693   9.561  1.00 64.60      A    N  \\nATOM   2387  CA  LYS A 293       0.340 -22.064   9.619  1.00 64.85      A    C  \\nATOM   2388  C   LYS A 293      -0.561 -21.016   8.959  1.00 65.80      A    C  \\nATOM   2389  O   LYS A 293      -1.514 -21.377   8.267  1.00 67.05      A    O  \\nATOM   2390  CB  LYS A 293      -0.094 -22.288  11.087  1.00 63.56      A    C  \\nATOM   2391  CG  LYS A 293      -1.526 -22.785  11.252  1.00 59.57      A    C  \\nATOM   2392  CD  LYS A 293      -1.823 -23.043  12.703  1.00 55.64      A    C  \\nATOM   2393  CE  LYS A 293      -3.275 -23.510  12.872  1.00 50.53      A    C  \\nATOM   2394  NZ  LYS A 293      -3.609 -23.797  14.267  1.00 45.06      A    N1+\\nATOM   2395  N   ASP A 294      -0.266 -19.774   9.180  1.00 65.96      A    N  \\nATOM   2396  CA  ASP A 294      -1.033 -18.703   8.524  1.00 67.57      A    C  \\nATOM   2397  C   ASP A 294      -0.719 -18.642   7.041  1.00 67.61      A    C  \\nATOM   2398  O   ASP A 294      -1.563 -18.224   6.247  1.00 68.20      A    O  \\nATOM   2399  CB  ASP A 294      -0.758 -17.359   9.172  1.00 66.61      A    C  \\nATOM   2400  CG  ASP A 294      -1.288 -17.284  10.597  1.00 64.15      A    C  \\nATOM   2401  OD1 ASP A 294      -2.305 -17.951  10.881  1.00 62.41      A    O  \\nATOM   2402  OD2 ASP A 294      -0.690 -16.538  11.407  1.00 63.34      A    O1-\\nATOM   2403  N   PHE A 295       0.463 -19.058   6.664  1.00 57.44      A    N  \\nATOM   2404  CA  PHE A 295       0.799 -19.097   5.244  1.00 58.98      A    C  \\nATOM   2405  C   PHE A 295       0.117 -20.252   4.543  1.00 60.03      A    C  \\nATOM   2406  O   PHE A 295      -0.287 -20.132   3.383  1.00 59.89      A    O  \\nATOM   2407  CB  PHE A 295       2.310 -19.153   5.065  1.00 55.36      A    C  \\nATOM   2408  CG  PHE A 295       2.758 -19.241   3.627  1.00 52.14      A    C  \\nATOM   2409  CD1 PHE A 295       2.706 -18.123   2.828  1.00 49.86      A    C  \\nATOM   2410  CD2 PHE A 295       3.232 -20.427   3.099  1.00 49.64      A    C  \\nATOM   2411  CE1 PHE A 295       3.104 -18.190   1.512  1.00 48.11      A    C  \\nATOM   2412  CE2 PHE A 295       3.629 -20.482   1.782  1.00 48.47      A    C  \\nATOM   2413  CZ  PHE A 295       3.571 -19.372   0.997  1.00 44.84      A    C  \\nATOM   2414  N   ARG A 296      -0.009 -21.388   5.228  1.00 57.09      A    N  \\nATOM   2415  CA  ARG A 296      -0.710 -22.520   4.634  1.00 58.55      A    C  \\nATOM   2416  C   ARG A 296      -2.209 -22.288   4.570  1.00 59.87      A    C  \\nATOM   2417  O   ARG A 296      -2.859 -22.760   3.628  1.00 60.17      A    O  \\nATOM   2418  CB  ARG A 296      -0.427 -23.790   5.444  1.00 55.58      A    C  \\nATOM   2419  CG  ARG A 296       1.005 -24.296   5.330  1.00 52.17      A    C  \\nATOM   2420  CD  ARG A 296       1.215 -25.548   6.156  1.00 49.12      A    C  \\nATOM   2421  NE  ARG A 296       2.570 -26.054   6.015  1.00 45.18      A    N  \\nATOM   2422  CZ  ARG A 296       3.034 -27.121   6.674  1.00 43.98      A    C  \\nATOM   2423  NH1 ARG A 296       2.230 -27.795   7.478  1.00 39.04      A    N1+\\nATOM   2424  NH2 ARG A 296       4.276 -27.525   6.490  1.00 36.90      A    N  \\nATOM   2425  N   GLU A 297      -2.766 -21.594   5.525  1.00 67.67      A    N  \\nATOM   2426  CA  GLU A 297      -4.180 -21.235   5.460  1.00 68.20      A    C  \\nATOM   2427  C   GLU A 297      -4.452 -20.216   4.367  1.00 68.36      A    C  \\nATOM   2428  O   GLU A 297      -5.526 -20.243   3.766  1.00 68.82      A    O  \\nATOM   2429  CB  GLU A 297      -4.649 -20.721   6.799  1.00 67.12      A    C  \\nATOM   2430  CG  GLU A 297      -4.703 -21.777   7.887  1.00 63.67      A    C  \\nATOM   2431  CD  GLU A 297      -5.159 -21.226   9.229  1.00 59.86      A    C  \\nATOM   2432  OE1 GLU A 297      -5.387 -20.002   9.329  1.00 59.07      A    O  \\nATOM   2433  OE2 GLU A 297      -5.266 -22.025  10.184  1.00 55.07      A    O1-\\nATOM   2434  N   TYR A 298      -3.511 -19.355   4.145  1.00 62.23      A    N  \\nATOM   2435  CA  TYR A 298      -3.645 -18.425   3.029  1.00 63.02      A    C  \\nATOM   2436  C   TYR A 298      -3.345 -19.103   1.687  1.00 63.30      A    C  \\nATOM   2437  O   TYR A 298      -3.844 -18.659   0.663  1.00 62.66      A    O  \\nATOM   2438  CB  TYR A 298      -2.712 -17.229   3.235  1.00 61.00      A    C  \\nATOM   2439  CG  TYR A 298      -2.821 -16.158   2.184  1.00 58.42      A    C  \\nATOM   2440  CD1 TYR A 298      -3.948 -15.350   2.125  1.00 57.33      A    C  \\nATOM   2441  CD2 TYR A 298      -1.814 -15.964   1.230  1.00 56.63      A    C  \\nATOM   2442  CE1 TYR A 298      -4.055 -14.360   1.159  1.00 54.73      A    C  \\nATOM   2443  CE2 TYR A 298      -1.915 -14.972   0.282  1.00 55.13      A    C  \\nATOM   2444  CZ  TYR A 298      -3.047 -14.187   0.228  1.00 53.18      A    C  \\nATOM   2445  OH  TYR A 298      -3.159 -13.232  -0.733  1.00 53.48      A    O  \\nATOM   2446  N   PHE A 299      -2.580 -20.149   1.729  1.00 57.25      A    N  \\nATOM   2447  CA  PHE A 299      -2.250 -20.875   0.482  1.00 58.93      A    C  \\nATOM   2448  C   PHE A 299      -3.431 -21.723   0.028  1.00 59.31      A    C  \\nATOM   2449  O   PHE A 299      -3.675 -21.819  -1.197  1.00 58.41      A    O  \\nATOM   2450  CB  PHE A 299      -1.009 -21.738   0.697  1.00 55.81      A    C  \\nATOM   2451  CG  PHE A 299      -0.578 -22.502  -0.555  1.00 53.11      A    C  \\nATOM   2452  CD1 PHE A 299      -0.890 -23.829  -0.706  1.00 50.45      A    C  \\nATOM   2453  CD2 PHE A 299       0.128 -21.853  -1.545  1.00 50.36      A    C  \\nATOM   2454  CE1 PHE A 299      -0.504 -24.517  -1.820  1.00 48.93      A    C  \\nATOM   2455  CE2 PHE A 299       0.512 -22.523  -2.681  1.00 49.14      A    C  \\nATOM   2456  CZ  PHE A 299       0.204 -23.860  -2.823  1.00 47.05      A    C  \\nATOM   2457  N   LYS A 300      -4.125 -22.341   0.959  1.00 62.15      A    N  \\nATOM   2458  CA  LYS A 300      -5.289 -23.160   0.608  1.00 62.76      A    C  \\nATOM   2459  C   LYS A 300      -6.440 -22.312   0.082  1.00 62.76      A    C  \\nATOM   2460  O   LYS A 300      -7.266 -22.834  -0.666  1.00 62.62      A    O  \\nATOM   2461  CB  LYS A 300      -5.739 -23.981   1.806  1.00 60.68      A    C  \\nATOM   2462  CG  LYS A 300      -4.800 -25.072   2.213  1.00 56.81      A    C  \\nATOM   2463  CD  LYS A 300      -5.306 -25.819   3.422  1.00 53.11      A    C  \\nATOM   2464  CE  LYS A 300      -4.335 -26.925   3.845  1.00 48.11      A    C  \\nATOM   2465  NZ  LYS A 300      -4.858 -27.684   5.009  1.00 42.99      A    N1+\\nATOM   2466  N   LYS A 301      -6.507 -21.079   0.486  1.00 67.15      A    N  \\nATOM   2467  CA  LYS A 301      -7.568 -20.195  -0.024  1.00 67.58      A    C  \\nATOM   2468  C   LYS A 301      -7.282 -19.735  -1.436  1.00 67.43      A    C  \\nATOM   2469  O   LYS A 301      -8.164 -19.773  -2.305  1.00 66.17      A    O  \\nATOM   2470  CB  LYS A 301      -7.781 -19.019   0.892  1.00 65.84      A    C  \\nATOM   2471  CG  LYS A 301      -8.333 -19.370   2.259  1.00 61.02      A    C  \\nATOM   2472  CD  LYS A 301      -8.520 -18.138   3.115  1.00 57.63      A    C  \\nATOM   2473  CE  LYS A 301      -9.064 -18.523   4.484  1.00 51.45      A    C  \\nATOM   2474  NZ  LYS A 301      -9.271 -17.330   5.340  1.00 46.40      A    N1+\\nATOM   2475  N   ASN A 302      -6.058 -19.328  -1.668  1.00 63.85      A    N  \\nATOM   2476  CA  ASN A 302      -5.657 -18.834  -2.986  1.00 64.33      A    C  \\nATOM   2477  C   ASN A 302      -4.705 -19.833  -3.630  1.00 63.77      A    C  \\nATOM   2478  O   ASN A 302      -3.495 -19.669  -3.620  1.00 62.94      A    O  \\nATOM   2479  CB  ASN A 302      -5.011 -17.456  -2.895  1.00 61.51      A    C  \\nATOM   2480  CG  ASN A 302      -5.995 -16.400  -2.424  1.00 59.53      A    C  \\nATOM   2481  ND2 ASN A 302      -5.557 -15.548  -1.506  1.00 58.61      A    N  \\nATOM   2482  OD1 ASN A 302      -7.120 -16.349  -2.891  1.00 58.31      A    O  \\nATOM   2483  N   GLN A 303      -5.303 -20.891  -4.184  1.00 58.78      A    N  \\nATOM   2484  CA  GLN A 303      -4.503 -21.917  -4.865  1.00 58.76      A    C  \\nATOM   2485  C   GLN A 303      -4.294 -21.609  -6.344  1.00 58.05      A    C  \\nATOM   2486  O   GLN A 303      -3.294 -22.008  -6.919  1.00 56.75      A    O  \\nATOM   2487  CB  GLN A 303      -5.208 -23.272  -4.735  1.00 55.23      A    C  \\nATOM   2488  CG  GLN A 303      -5.244 -23.800  -3.303  1.00 52.62      A    C  \\nATOM   2489  CD  GLN A 303      -5.931 -25.140  -3.206  1.00 48.76      A    C  \\nATOM   2490  NE2 GLN A 303      -5.905 -25.738  -2.030  1.00 44.97      A    N  \\nATOM   2491  OE1 GLN A 303      -6.477 -25.644  -4.197  1.00 47.28      A    O  \\nATOM   2492  N   GLU A 304      -5.248 -20.921  -6.928  1.00 54.72      A    N  \\nATOM   2493  CA  GLU A 304      -5.149 -20.633  -8.360  1.00 55.49      A    C  \\nATOM   2494  C   GLU A 304      -4.185 -19.488  -8.661  1.00 55.76      A    C  \\nATOM   2495  O   GLU A 304      -3.508 -19.503  -9.687  1.00 54.20      A    O  \\nATOM   2496  CB  GLU A 304      -6.514 -20.319  -8.937  1.00 51.82      A    C  \\nATOM   2497  CG  GLU A 304      -7.479 -21.508  -8.886  1.00 49.50      A    C  \\nATOM   2498  CD  GLU A 304      -8.844 -21.154  -9.434  1.00 46.27      A    C  \\nATOM   2499  OE1 GLU A 304      -9.079 -19.975  -9.756  1.00 44.11      A    O  \\nATOM   2500  OE2 GLU A 304      -9.700 -22.068  -9.533  1.00 41.74      A    O1-\\nATOM   2501  N   SER A 305      -4.155 -18.520  -7.778  1.00 59.03      A    N  \\nATOM   2502  CA  SER A 305      -3.282 -17.374  -8.030  1.00 60.08      A    C  \\nATOM   2503  C   SER A 305      -1.855 -17.637  -7.598  1.00 59.71      A    C  \\nATOM   2504  O   SER A 305      -0.897 -17.322  -8.312  1.00 58.85      A    O  \\nATOM   2505  CB  SER A 305      -3.817 -16.120  -7.310  1.00 57.46      A    C  \\nATOM   2506  OG  SER A 305      -3.802 -16.340  -5.907  1.00 53.75      A    O  \\nATOM   2507  N   LEU A 306      -1.695 -18.228  -6.410  1.00 54.24      A    N  \\nATOM   2508  CA  LEU A 306      -0.363 -18.498  -5.884  1.00 55.78      A    C  \\nATOM   2509  C   LEU A 306       0.173 -19.823  -6.439  1.00 55.46      A    C  \\nATOM   2510  O   LEU A 306      -0.541 -20.790  -6.571  1.00 53.78      A    O  \\nATOM   2511  CB  LEU A 306      -0.368 -18.550  -4.367  1.00 52.24      A    C  \\nATOM   2512  CG  LEU A 306      -0.766 -17.270  -3.624  1.00 50.12      A    C  \\nATOM   2513  CD1 LEU A 306       0.251 -16.173  -3.933  1.00 51.16      A    C  \\nATOM   2514  CD2 LEU A 306      -0.848 -17.517  -2.130  1.00 50.31      A    C  \\nATOM   2515  N   ASN A 307       1.442 -19.826  -6.724  1.00 51.14      A    N  \\nATOM   2516  CA  ASN A 307       2.112 -21.004  -7.241  1.00 52.39      A    C  \\nATOM   2517  C   ASN A 307       3.397 -21.233  -6.445  1.00 53.09      A    C  \\nATOM   2518  O   ASN A 307       4.501 -21.127  -6.962  1.00 50.68      A    O  \\nATOM   2519  CB  ASN A 307       2.398 -20.873  -8.739  1.00 47.69      A    C  \\nATOM   2520  CG  ASN A 307       1.145 -20.903  -9.571  1.00 46.24      A    C  \\nATOM   2521  ND2 ASN A 307       1.081 -20.015 -10.562  1.00 44.65      A    N  \\nATOM   2522  OD1 ASN A 307       0.251 -21.701  -9.335  1.00 42.44      A    O  \\nATOM   2523  N   VAL A 308       3.213 -21.567  -5.163  1.00 49.22      A    N  \\nATOM   2524  CA  VAL A 308       4.349 -21.759  -4.280  1.00 50.57      A    C  \\nATOM   2525  C   VAL A 308       4.200 -23.118  -3.589  1.00 51.48      A    C  \\nATOM   2526  O   VAL A 308       3.128 -23.731  -3.617  1.00 49.48      A    O  \\nATOM   2527  CB  VAL A 308       4.469 -20.644  -3.233  1.00 46.01      A    C  \\nATOM   2528  CG1 VAL A 308       4.857 -19.326  -3.880  1.00 46.83      A    C  \\nATOM   2529  CG2 VAL A 308       3.182 -20.479  -2.478  1.00 45.05      A    C  \\nATOM   2530  N   ASN A 309       5.265 -23.544  -2.973  1.00 54.55      A    N  \\nATOM   2531  CA  ASN A 309       5.288 -24.785  -2.220  1.00 56.41      A    C  \\nATOM   2532  C   ASN A 309       5.613 -24.462  -0.772  1.00 57.38      A    C  \\nATOM   2533  O   ASN A 309       6.644 -23.866  -0.479  1.00 57.07      A    O  \\nATOM   2534  CB  ASN A 309       6.296 -25.762  -2.788  1.00 52.72      A    C  \\nATOM   2535  CG  ASN A 309       5.910 -26.272  -4.163  1.00 50.46      A    C  \\nATOM   2536  ND2 ASN A 309       6.885 -26.354  -5.048  1.00 49.15      A    N  \\nATOM   2537  OD1 ASN A 309       4.759 -26.594  -4.424  1.00 46.35      A    O  \\nATOM   2538  N   SER A 310       4.726 -24.891   0.114  1.00 54.78      A    N  \\nATOM   2539  CA  SER A 310       4.922 -24.591   1.533  1.00 56.78      A    C  \\nATOM   2540  C   SER A 310       6.045 -25.431   2.138  1.00 58.09      A    C  \\nATOM   2541  O   SER A 310       6.706 -25.000   3.095  1.00 57.22      A    O  \\nATOM   2542  CB  SER A 310       3.622 -24.798   2.315  1.00 52.84      A    C  \\nATOM   2543  OG  SER A 310       3.262 -26.181   2.304  1.00 49.29      A    O  \\nATOM   2544  N   LYS A 311       6.245 -26.636   1.621  1.00 56.12      A    N  \\nATOM   2545  CA  LYS A 311       7.306 -27.486   2.144  1.00 58.32      A    C  \\nATOM   2546  C   LYS A 311       8.682 -26.968   1.748  1.00 59.71      A    C  \\nATOM   2547  O   LYS A 311       9.623 -27.053   2.544  1.00 59.52      A    O  \\nATOM   2548  CB  LYS A 311       7.137 -28.918   1.595  1.00 55.13      A    C  \\nATOM   2549  CG  LYS A 311       5.912 -29.640   2.094  1.00 52.04      A    C  \\nATOM   2550  CD  LYS A 311       5.842 -31.037   1.542  1.00 49.03      A    C  \\nATOM   2551  CE  LYS A 311       4.634 -31.778   2.050  1.00 44.83      A    C  \\nATOM   2552  NZ  LYS A 311       4.532 -33.154   1.504  1.00 40.66      A    N1+\\nATOM   2553  N   ASP A 312       8.782 -26.415   0.581  1.00 60.89      A    N  \\nATOM   2554  CA  ASP A 312      10.069 -25.835   0.143  1.00 62.42      A    C  \\nATOM   2555  C   ASP A 312      10.379 -24.546   0.884  1.00 63.58      A    C  \\nATOM   2556  O   ASP A 312      11.544 -24.259   1.176  1.00 63.16      A    O  \\nATOM   2557  CB  ASP A 312      10.038 -25.593  -1.373  1.00 59.78      A    C  \\nATOM   2558  CG  ASP A 312       9.943 -26.875  -2.161  1.00 56.71      A    C  \\nATOM   2559  OD1 ASP A 312      10.498 -27.888  -1.701  1.00 53.67      A    O  \\nATOM   2560  OD2 ASP A 312       9.363 -26.848  -3.253  1.00 55.67      A    O1-\\nATOM   2561  N   LEU A 313       9.356 -23.769   1.159  1.00 57.83      A    N  \\nATOM   2562  CA  LEU A 313       9.571 -22.528   1.899  1.00 59.57      A    C  \\nATOM   2563  C   LEU A 313       9.890 -22.785   3.366  1.00 60.31      A    C  \\nATOM   2564  O   LEU A 313      10.592 -22.000   3.997  1.00 59.44      A    O  \\nATOM   2565  CB  LEU A 313       8.352 -21.618   1.788  1.00 56.77      A    C  \\nATOM   2566  CG  LEU A 313       8.079 -21.022   0.408  1.00 53.36      A    C  \\nATOM   2567  CD1 LEU A 313       9.215 -20.119   0.023  1.00 52.98      A    C  \\nATOM   2568  CD2 LEU A 313       6.757 -20.299   0.393  1.00 53.09      A    C  \\nATOM   2569  N   TYR A 314       9.371 -23.896   3.884  1.00 55.12      A    N  \\nATOM   2570  CA  TYR A 314       9.712 -24.251   5.262  1.00 56.73      A    C  \\nATOM   2571  C   TYR A 314      11.173 -24.650   5.370  1.00 58.22      A    C  \\nATOM   2572  O   TYR A 314      11.841 -24.317   6.366  1.00 57.90      A    O  \\nATOM   2573  CB  TYR A 314       8.813 -25.380   5.746  1.00 53.68      A    C  \\nATOM   2574  CG  TYR A 314       9.079 -25.800   7.171  1.00 50.58      A    C  \\nATOM   2575  CD1 TYR A 314       8.607 -25.060   8.228  1.00 47.88      A    C  \\nATOM   2576  CD2 TYR A 314       9.801 -26.965   7.453  1.00 47.59      A    C  \\nATOM   2577  CE1 TYR A 314       8.828 -25.449   9.539  1.00 45.12      A    C  \\nATOM   2578  CE2 TYR A 314      10.023 -27.359   8.751  1.00 46.02      A    C  \\nATOM   2579  CZ  TYR A 314       9.526 -26.594   9.777  1.00 42.55      A    C  \\nATOM   2580  OH  TYR A 314       9.766 -26.970  11.079  1.00 43.32      A    O  \\nATOM   2581  N   PHE A 315      11.690 -25.359   4.397  1.00 59.20      A    N  \\nATOM   2582  CA  PHE A 315      13.099 -25.772   4.407  1.00 60.83      A    C  \\nATOM   2583  C   PHE A 315      14.005 -24.576   4.254  1.00 61.69      A    C  \\nATOM   2584  O   PHE A 315      15.068 -24.527   4.886  1.00 61.63      A    O  \\nATOM   2585  CB  PHE A 315      13.360 -26.788   3.302  1.00 57.85      A    C  \\nATOM   2586  CG  PHE A 315      12.704 -28.133   3.496  1.00 54.25      A    C  \\nATOM   2587  CD1 PHE A 315      12.418 -28.580   4.771  1.00 52.19      A    C  \\nATOM   2588  CD2 PHE A 315      12.387 -28.938   2.413  1.00 51.97      A    C  \\nATOM   2589  CE1 PHE A 315      11.840 -29.801   4.960  1.00 50.37      A    C  \\nATOM   2590  CE2 PHE A 315      11.800 -30.159   2.614  1.00 51.06      A    C  \\nATOM   2591  CZ  PHE A 315      11.515 -30.595   3.890  1.00 47.36      A    C  \\nATOM   2592  N   GLU A 316      13.621 -23.623   3.436  1.00 61.76      A    N  \\nATOM   2593  CA  GLU A 316      14.482 -22.476   3.192  1.00 62.99      A    C  \\nATOM   2594  C   GLU A 316      14.502 -21.533   4.395  1.00 63.44      A    C  \\nATOM   2595  O   GLU A 316      15.558 -21.004   4.731  1.00 63.44      A    O  \\nATOM   2596  CB  GLU A 316      14.001 -21.718   1.942  1.00 60.29      A    C  \\nATOM   2597  CG  GLU A 316      14.167 -22.519   0.659  1.00 56.21      A    C  \\nATOM   2598  CD  GLU A 316      13.658 -21.770  -0.564  1.00 52.62      A    C  \\nATOM   2599  OE1 GLU A 316      13.081 -20.670  -0.375  1.00 50.12      A    O  \\nATOM   2600  OE2 GLU A 316      13.836 -22.263  -1.707  1.00 47.21      A    O1-\\nATOM   2601  N   VAL A 317      13.376 -21.360   5.027  1.00 58.87      A    N  \\nATOM   2602  CA  VAL A 317      13.333 -20.516   6.232  1.00 59.90      A    C  \\nATOM   2603  C   VAL A 317      14.043 -21.183   7.402  1.00 60.91      A    C  \\nATOM   2604  O   VAL A 317      14.744 -20.523   8.155  1.00 60.00      A    O  \\nATOM   2605  CB  VAL A 317      11.892 -20.162   6.571  1.00 56.17      A    C  \\nATOM   2606  CG1 VAL A 317      11.289 -19.313   5.459  1.00 57.57      A    C  \\nATOM   2607  CG2 VAL A 317      11.774 -19.399   7.852  1.00 55.60      A    C  \\nATOM   2608  N   ARG A 318      13.884 -22.495   7.512  1.00 57.24      A    N  \\nATOM   2609  CA  ARG A 318      14.509 -23.234   8.591  1.00 58.52      A    C  \\nATOM   2610  C   ARG A 318      16.029 -23.196   8.479  1.00 59.95      A    C  \\nATOM   2611  O   ARG A 318      16.750 -23.043   9.483  1.00 60.91      A    O  \\nATOM   2612  CB  ARG A 318      14.002 -24.654   8.662  1.00 55.65      A    C  \\nATOM   2613  CG  ARG A 318      14.593 -25.515   9.777  1.00 52.40      A    C  \\nATOM   2614  CD  ARG A 318      14.158 -25.018  11.126  1.00 49.60      A    C  \\nATOM   2615  NE  ARG A 318      14.665 -25.876  12.191  1.00 46.99      A    N  \\nATOM   2616  CZ  ARG A 318      14.537 -25.612  13.470  1.00 46.35      A    C  \\nATOM   2617  NH1 ARG A 318      13.934 -24.487  13.861  1.00 41.12      A    N1+\\nATOM   2618  NH2 ARG A 318      15.002 -26.438  14.370  1.00 39.54      A    N  \\nATOM   2619  N   GLU A 319      16.544 -23.377   7.274  1.00 63.08      A    N  \\nATOM   2620  CA  GLU A 319      17.969 -23.376   7.051  1.00 63.57      A    C  \\nATOM   2621  C   GLU A 319      18.596 -22.009   7.283  1.00 63.61      A    C  \\nATOM   2622  O   GLU A 319      19.723 -21.913   7.745  1.00 63.62      A    O  \\nATOM   2623  CB  GLU A 319      18.306 -23.894   5.656  1.00 61.21      A    C  \\nATOM   2624  CG  GLU A 319      18.021 -25.366   5.444  1.00 57.51      A    C  \\nATOM   2625  CD  GLU A 319      18.340 -25.827   4.040  1.00 53.42      A    C  \\nATOM   2626  OE1 GLU A 319      18.778 -24.997   3.200  1.00 53.28      A    O  \\nATOM   2627  OE2 GLU A 319      18.161 -27.035   3.754  1.00 48.80      A    O1-\\nATOM   2628  N   ILE A 320      17.849 -20.990   6.962  1.00 64.74      A    N  \\nATOM   2629  CA  ILE A 320      18.339 -19.630   7.182  1.00 64.98      A    C  \\nATOM   2630  C   ILE A 320      18.337 -19.300   8.672  1.00 65.23      A    C  \\nATOM   2631  O   ILE A 320      19.249 -18.648   9.159  1.00 63.60      A    O  \\nATOM   2632  CB  ILE A 320      17.526 -18.621   6.357  1.00 62.08      A    C  \\nATOM   2633  CG1 ILE A 320      17.757 -18.865   4.866  1.00 62.42      A    C  \\nATOM   2634  CG2 ILE A 320      17.882 -17.197   6.694  1.00 59.93      A    C  \\nATOM   2635  CD1 ILE A 320      16.842 -18.081   3.959  1.00 56.87      A    C  \\nATOM   2636  N   LEU A 321      17.334 -19.783   9.382  1.00 61.02      A    N  \\nATOM   2637  CA  LEU A 321      17.294 -19.538  10.817  1.00 61.67      A    C  \\nATOM   2638  C   LEU A 321      18.370 -20.329  11.552  1.00 62.45      A    C  \\nATOM   2639  O   LEU A 321      18.833 -19.907  12.606  1.00 61.64      A    O  \\nATOM   2640  CB  LEU A 321      15.924 -19.892  11.386  1.00 58.14      A    C  \\nATOM   2641  CG  LEU A 321      14.732 -19.001  11.028  1.00 54.34      A    C  \\nATOM   2642  CD1 LEU A 321      14.954 -17.574  11.530  1.00 54.20      A    C  \\nATOM   2643  CD2 LEU A 321      13.451 -19.581  11.580  1.00 52.96      A    C  \\nATOM   2644  N   LEU A 322      18.723 -21.492  11.025  1.00 61.44      A    N  \\nATOM   2645  CA  LEU A 322      19.821 -22.247  11.620  1.00 62.28      A    C  \\nATOM   2646  C   LEU A 322      21.191 -21.658  11.306  1.00 62.76      A    C  \\nATOM   2647  O   LEU A 322      22.086 -21.724  12.133  1.00 62.07      A    O  \\nATOM   2648  CB  LEU A 322      19.757 -23.716  11.160  1.00 59.53      A    C  \\nATOM   2649  CG  LEU A 322      18.604 -24.533  11.716  1.00 56.42      A    C  \\nATOM   2650  CD1 LEU A 322      18.719 -24.661  13.226  1.00 57.13      A    C  \\nATOM   2651  CD2 LEU A 322      18.551 -25.913  11.063  1.00 55.93      A    C  \\nATOM   2652  N   ARG A 323      21.302 -21.056  10.152  1.00 64.20      A    N  \\nATOM   2653  CA  ARG A 323      22.570 -20.451   9.789  1.00 64.68      A    C  \\nATOM   2654  C   ARG A 323      22.798 -19.099  10.479  1.00 64.03      A    C  \\nATOM   2655  O   ARG A 323      23.939 -18.691  10.664  1.00 63.44      A    O  \\nATOM   2656  CB  ARG A 323      22.684 -20.252   8.282  1.00 62.11      A    C  \\nATOM   2657  CG  ARG A 323      22.810 -21.525   7.484  1.00 58.40      A    C  \\nATOM   2658  CD  ARG A 323      22.876 -21.256   6.010  1.00 56.34      A    C  \\nATOM   2659  NE  ARG A 323      22.977 -22.501   5.221  1.00 54.52      A    N  \\nATOM   2660  CZ  ARG A 323      23.062 -22.539   3.909  1.00 52.28      A    C  \\nATOM   2661  NH1 ARG A 323      23.025 -21.426   3.197  1.00 47.11      A    N1+\\nATOM   2662  NH2 ARG A 323      23.174 -23.710   3.279  1.00 46.12      A    N  \\nATOM   2663  N   LYS A 324      21.715 -18.464  10.867  1.00 60.10      A    N  \\nATOM   2664  CA  LYS A 324      21.866 -17.172  11.569  1.00 60.73      A    C  \\nATOM   2665  C   LYS A 324      22.021 -17.339  13.073  1.00 61.06      A    C  \\nATOM   2666  O   LYS A 324      22.711 -16.557  13.698  1.00 60.19      A    O  \\nATOM   2667  CB  LYS A 324      20.664 -16.283  11.266  1.00 57.64      A    C  \\nATOM   2668  CG  LYS A 324      20.642 -15.751   9.857  1.00 53.78      A    C  \\nATOM   2669  CD  LYS A 324      19.439 -14.875   9.595  1.00 50.58      A    C  \\nATOM   2670  CE  LYS A 324      19.445 -14.330   8.175  1.00 46.12      A    C  \\nATOM   2671  NZ  LYS A 324      18.289 -13.449   7.884  1.00 42.49      A    N1+\\nATOM   2672  N   THR A 325      21.410 -18.372  13.620  1.00 58.79      A    N  \\nATOM   2673  CA  THR A 325      21.568 -18.605  15.060  1.00 59.46      A    C  \\nATOM   2674  C   THR A 325      22.861 -19.361  15.353  1.00 60.07      A    C  \\nATOM   2675  O   THR A 325      23.615 -18.971  16.260  1.00 59.18      A    O  \\nATOM   2676  CB  THR A 325      20.365 -19.409  15.588  1.00 56.75      A    C  \\nATOM   2677  CG2 THR A 325      19.100 -18.577  15.447  1.00 52.62      A    C  \\nATOM   2678  OG1 THR A 325      20.226 -20.633  14.864  1.00 55.04      A    O  \\nATOM   2679  N   LEU A 326      23.115 -20.420  14.616  1.00 60.59      A    N  \\nATOM   2680  CA  LEU A 326      24.337 -21.176  14.821  1.00 61.30      A    C  \\nATOM   2681  C   LEU A 326      25.526 -20.487  14.179  1.00 60.95      A    C  \\nATOM   2682  O   LEU A 326      26.627 -20.471  14.757  1.00 59.06      A    O  \\nATOM   2683  CB  LEU A 326      24.194 -22.583  14.254  1.00 58.88      A    C  \\nATOM   2684  CG  LEU A 326      23.178 -23.498  14.947  1.00 56.48      A    C  \\nATOM   2685  CD1 LEU A 326      23.592 -23.760  16.376  1.00 58.55      A    C  \\nATOM   2686  CD2 LEU A 326      23.007 -24.784  14.186  1.00 57.63      A    C  \\nATOM   2687  N   GLY A 327      25.330 -19.922  13.031  1.00 62.24      A    N  \\nATOM   2688  CA  GLY A 327      26.399 -19.306  12.273  1.00 63.98      A    C  \\nATOM   2689  C   GLY A 327      26.753 -20.083  11.037  1.00 65.61      A    C  \\nATOM   2690  O   GLY A 327      26.444 -21.278  10.937  1.00 64.13      A    O  \\nATOM   2691  N   ASN A 328      27.391 -19.424  10.119  1.00 63.43      A    N  \\nATOM   2692  CA  ASN A 328      27.718 -20.053   8.858  1.00 64.47      A    C  \\nATOM   2693  C   ASN A 328      28.825 -21.086   9.020  1.00 65.06      A    C  \\nATOM   2694  O   ASN A 328      28.843 -22.098   8.308  1.00 64.95      A    O  \\nATOM   2695  CB  ASN A 328      28.142 -19.006   7.816  1.00 61.79      A    C  \\nATOM   2696  CG  ASN A 328      26.972 -18.157   7.378  1.00 58.82      A    C  \\nATOM   2697  ND2 ASN A 328      27.201 -16.862   7.252  1.00 54.91      A    N  \\nATOM   2698  OD1 ASN A 328      25.883 -18.654   7.149  1.00 54.40      A    O  \\nATOM   2699  N   LYS A 329      29.724 -20.848   9.965  1.00 62.74      A    N  \\nATOM   2700  CA  LYS A 329      30.815 -21.789  10.191  1.00 63.63      A    C  \\nATOM   2701  C   LYS A 329      30.354 -23.001  10.969  1.00 64.55      A    C  \\nATOM   2702  O   LYS A 329      30.758 -24.140  10.666  1.00 63.45      A    O  \\nATOM   2703  CB  LYS A 329      31.973 -21.103  10.933  1.00 60.72      A    C  \\nATOM   2704  CG  LYS A 329      32.662 -20.038  10.105  1.00 57.63      A    C  \\nATOM   2705  CD  LYS A 329      33.793 -19.396  10.873  1.00 52.41      A    C  \\nATOM   2706  CE  LYS A 329      34.510 -18.351  10.086  1.00 47.66      A    C  \\nATOM   2707  NZ  LYS A 329      35.619 -17.706  10.818  1.00 40.83      A    N1+\\nATOM   2708  N   LEU A 330      29.510 -22.808  11.957  1.00 61.23      A    N  \\nATOM   2709  CA  LEU A 330      29.016 -23.910  12.775  1.00 63.23      A    C  \\nATOM   2710  C   LEU A 330      28.035 -24.791  12.021  1.00 64.11      A    C  \\nATOM   2711  O   LEU A 330      27.965 -26.000  12.238  1.00 63.75      A    O  \\nATOM   2712  CB  LEU A 330      28.381 -23.398  14.058  1.00 60.71      A    C  \\nATOM   2713  CG  LEU A 330      29.312 -22.730  15.067  1.00 56.38      A    C  \\nATOM   2714  CD1 LEU A 330      30.355 -23.716  15.560  1.00 57.26      A    C  \\nATOM   2715  CD2 LEU A 330      28.533 -22.119  16.201  1.00 57.27      A    C  \\nATOM   2716  N   TYR A 331      27.241 -24.174  11.147  1.00 62.80      A    N  \\nATOM   2717  CA  TYR A 331      26.277 -24.926  10.377  1.00 63.73      A    C  \\nATOM   2718  C   TYR A 331      26.962 -25.828   9.348  1.00 63.87      A    C  \\nATOM   2719  O   TYR A 331      26.521 -26.944   9.098  1.00 63.36      A    O  \\nATOM   2720  CB  TYR A 331      25.302 -23.990   9.666  1.00 61.67      A    C  \\nATOM   2721  CG  TYR A 331      24.270 -24.677   8.817  1.00 60.78      A    C  \\nATOM   2722  CD1 TYR A 331      23.212 -25.341   9.404  1.00 60.26      A    C  \\nATOM   2723  CD2 TYR A 331      24.371 -24.653   7.435  1.00 59.88      A    C  \\nATOM   2724  CE1 TYR A 331      22.267 -25.988   8.617  1.00 60.10      A    C  \\nATOM   2725  CE2 TYR A 331      23.435 -25.284   6.660  1.00 59.99      A    C  \\nATOM   2726  CZ  TYR A 331      22.380 -25.959   7.251  1.00 59.38      A    C  \\nATOM   2727  OH  TYR A 331      21.429 -26.583   6.490  1.00 55.56      A    O  \\nATOM   2728  N   LYS A 332      28.046 -25.353   8.782  1.00 65.21      A    N  \\nATOM   2729  CA  LYS A 332      28.798 -26.160   7.810  1.00 65.61      A    C  \\nATOM   2730  C   LYS A 332      29.575 -27.269   8.519  1.00 65.50      A    C  \\nATOM   2731  O   LYS A 332      29.790 -28.333   7.948  1.00 63.90      A    O  \\nATOM   2732  CB  LYS A 332      29.750 -25.306   6.998  1.00 63.34      A    C  \\nATOM   2733  CG  LYS A 332      29.012 -24.373   6.037  1.00 59.55      A    C  \\nATOM   2734  CD  LYS A 332      29.992 -23.533   5.238  1.00 56.04      A    C  \\nATOM   2735  CE  LYS A 332      29.278 -22.587   4.272  1.00 51.67      A    C  \\nATOM   2736  NZ  LYS A 332      30.196 -21.751   3.498  1.00 46.04      A    N1+\\nATOM   2737  N   LYS A 333      29.982 -27.030   9.764  1.00 64.86      A    N  \\nATOM   2738  CA  LYS A 333      30.699 -28.057  10.522  1.00 64.99      A    C  \\nATOM   2739  C   LYS A 333      29.765 -29.151  10.998  1.00 64.92      A    C  \\nATOM   2740  O   LYS A 333      30.181 -30.297  11.112  1.00 64.68      A    O  \\nATOM   2741  CB  LYS A 333      31.466 -27.434  11.682  1.00 63.09      A    C  \\nATOM   2742  CG  LYS A 333      32.638 -26.594  11.266  1.00 59.81      A    C  \\nATOM   2743  CD  LYS A 333      33.327 -26.006  12.464  1.00 56.60      A    C  \\nATOM   2744  CE  LYS A 333      34.498 -25.132  12.052  1.00 52.19      A    C  \\nATOM   2745  NZ  LYS A 333      35.202 -24.557  13.205  1.00 46.95      A    N1+\\nATOM   2746  N   LEU A 334      28.522 -28.798  11.305  1.00 63.19      A    N  \\nATOM   2747  CA  LEU A 334      27.548 -29.792  11.715  1.00 63.56      A    C  \\nATOM   2748  C   LEU A 334      27.112 -30.666  10.533  1.00 62.89      A    C  \\nATOM   2749  O   LEU A 334      26.793 -31.844  10.714  1.00 61.20      A    O  \\nATOM   2750  CB  LEU A 334      26.329 -29.136  12.350  1.00 60.18      A    C  \\nATOM   2751  CG  LEU A 334      26.551 -28.541  13.736  1.00 57.36      A    C  \\nATOM   2752  CD1 LEU A 334      26.910 -29.620  14.734  1.00 57.80      A    C  \\nATOM   2753  CD2 LEU A 334      25.318 -27.781  14.175  1.00 57.16      A    C  \\nATOM   2754  N   LEU A 335      27.069 -30.083   9.371  1.00 63.84      A    N  \\nATOM   2755  CA  LEU A 335      26.794 -30.879   8.164  1.00 65.17      A    C  \\nATOM   2756  C   LEU A 335      27.986 -31.726   7.764  1.00 64.17      A    C  \\nATOM   2757  O   LEU A 335      27.823 -32.817   7.214  1.00 62.41      A    O  \\nATOM   2758  CB  LEU A 335      26.400 -29.969   7.008  1.00 62.17      A    C  \\nATOM   2759  CG  LEU A 335      25.062 -29.263   7.164  1.00 59.03      A    C  \\nATOM   2760  CD1 LEU A 335      23.933 -30.283   7.263  1.00 59.66      A    C  \\nATOM   2761  CD2 LEU A 335      24.837 -28.316   5.985  1.00 59.12      A    C  \\nATOM   2762  N   GLY A 336      29.186 -31.255   8.050  1.00 64.37      A    N  \\nATOM   2763  CA  GLY A 336      30.375 -32.019   7.735  1.00 63.54      A    C  \\nATOM   2764  C   GLY A 336      30.648 -33.149   8.700  1.00 63.88      A    C  \\nATOM   2765  O   GLY A 336      31.228 -34.173   8.334  1.00 60.51      A    O  \\nATOM   2766  N   ASN A 337      30.209 -33.009   9.957  1.00 59.43      A    N  \\nATOM   2767  CA  ASN A 337      30.382 -34.040  10.946  1.00 59.98      A    C  \\nATOM   2768  C   ASN A 337      29.372 -35.178  10.803  1.00 59.19      A    C  \\nATOM   2769  O   ASN A 337      29.526 -36.193  11.480  1.00 56.48      A    O  \\nATOM   2770  CB  ASN A 337      30.276 -33.454  12.355  1.00 56.17      A    C  \\nATOM   2771  CG  ASN A 337      31.485 -32.626  12.707  1.00 53.49      A    C  \\nATOM   2772  ND2 ASN A 337      31.263 -31.520  13.385  1.00 50.58      A    N  \\nATOM   2773  OD1 ASN A 337      32.619 -32.966  12.361  1.00 48.24      A    O  \\nATOM   2774  N   ASN A 338      28.343 -34.989  10.002  1.00 53.39      A    N  \\nATOM   2775  CA  ASN A 338      27.295 -35.970   9.712  1.00 54.38      A    C  \\nATOM   2776  C   ASN A 338      26.556 -36.383  10.970  1.00 54.54      A    C  \\nATOM   2777  O   ASN A 338      26.138 -37.522  11.118  1.00 51.84      A    O  \\nATOM   2778  CB  ASN A 338      27.840 -37.194   8.977  1.00 48.85      A    C  \\nATOM   2779  CG  ASN A 338      28.297 -36.879   7.564  1.00 46.59      A    C  \\nATOM   2780  ND2 ASN A 338      29.421 -37.461   7.169  1.00 43.79      A    N  \\nATOM   2781  OD1 ASN A 338      27.665 -36.119   6.862  1.00 41.98      A    O  \\nATOM   2782  N   LYS A 339      26.378 -35.430  11.894  1.00 49.42      A    N  \\nATOM   2783  CA  LYS A 339      25.630 -35.701  13.106  1.00 52.03      A    C  \\nATOM   2784  C   LYS A 339      24.268 -34.997  13.097  1.00 52.51      A    C  \\nATOM   2785  O   LYS A 339      23.257 -35.598  13.422  1.00 51.45      A    O  \\nATOM   2786  CB  LYS A 339      26.425 -35.275  14.338  1.00 48.27      A    C  \\nATOM   2787  CG  LYS A 339      27.651 -36.098  14.599  1.00 46.53      A    C  \\nATOM   2788  CD  LYS A 339      28.421 -35.640  15.830  1.00 43.83      A    C  \\nATOM   2789  CE  LYS A 339      29.636 -36.504  16.096  1.00 38.18      A    C  \\nATOM   2790  NZ  LYS A 339      30.374 -36.071  17.295  1.00 35.60      A    N1+\\nATOM   2791  N   PHE A 340      24.264 -33.759  12.694  1.00 53.67      A    N  \\nATOM   2792  CA  PHE A 340      23.057 -32.955  12.654  1.00 57.10      A    C  \\nATOM   2793  C   PHE A 340      22.459 -32.999  11.253  1.00 57.68      A    C  \\nATOM   2794  O   PHE A 340      23.072 -32.501  10.304  1.00 56.24      A    O  \\nATOM   2795  CB  PHE A 340      23.380 -31.526  13.089  1.00 54.53      A    C  \\nATOM   2796  CG  PHE A 340      22.167 -30.601  13.051  1.00 52.46      A    C  \\nATOM   2797  CD1 PHE A 340      21.184 -30.681  14.025  1.00 51.22      A    C  \\nATOM   2798  CD2 PHE A 340      22.056 -29.661  12.039  1.00 51.02      A    C  \\nATOM   2799  CE1 PHE A 340      20.096 -29.836  13.989  1.00 48.48      A    C  \\nATOM   2800  CE2 PHE A 340      20.977 -28.790  12.011  1.00 49.58      A    C  \\nATOM   2801  CZ  PHE A 340      20.008 -28.885  12.978  1.00 47.70      A    C  \\nATOM   2802  N   ASN A 341      21.289 -33.583  11.156  1.00 56.67      A    N  \\nATOM   2803  CA  ASN A 341      20.588 -33.656   9.880  1.00 59.14      A    C  \\nATOM   2804  C   ASN A 341      19.407 -32.706   9.913  1.00 59.50      A    C  \\nATOM   2805  O   ASN A 341      18.382 -33.015  10.506  1.00 57.75      A    O  \\nATOM   2806  CB  ASN A 341      20.113 -35.095   9.605  1.00 55.77      A    C  \\nATOM   2807  CG  ASN A 341      21.264 -36.036   9.341  1.00 53.12      A    C  \\nATOM   2808  ND2 ASN A 341      21.177 -37.232   9.931  1.00 49.88      A    N  \\nATOM   2809  OD1 ASN A 341      22.197 -35.716   8.632  1.00 46.92      A    O  \\nATOM   2810  N   PRO A 342      19.535 -31.537   9.336  1.00 51.59      A    N  \\nATOM   2811  CA  PRO A 342      18.448 -30.558   9.396  1.00 53.55      A    C  \\nATOM   2812  C   PRO A 342      17.277 -30.921   8.512  1.00 54.03      A    C  \\nATOM   2813  O   PRO A 342      16.153 -30.444   8.754  1.00 53.19      A    O  \\nATOM   2814  CB  PRO A 342      19.122 -29.275   8.918  1.00 50.80      A    C  \\nATOM   2815  CG  PRO A 342      20.167 -29.740   7.985  1.00 50.96      A    C  \\nATOM   2816  CD  PRO A 342      20.676 -31.015   8.566  1.00 51.47      A    C  \\nATOM   2817  N   LYS A 343      17.473 -31.758   7.531  1.00 48.90      A    N  \\nATOM   2818  CA  LYS A 343      16.366 -32.153   6.645  1.00 51.77      A    C  \\nATOM   2819  C   LYS A 343      15.408 -33.091   7.361  1.00 51.92      A    C  \\nATOM   2820  O   LYS A 343      14.211 -33.049   7.087  1.00 49.98      A    O  \\nATOM   2821  CB  LYS A 343      16.905 -32.808   5.386  1.00 48.66      A    C  \\nATOM   2822  CG  LYS A 343      17.651 -31.829   4.469  1.00 46.58      A    C  \\nATOM   2823  CD  LYS A 343      18.165 -32.533   3.229  1.00 43.86      A    C  \\nATOM   2824  CE  LYS A 343      18.924 -31.589   2.339  1.00 39.67      A    C  \\nATOM   2825  NZ  LYS A 343      19.434 -32.247   1.104  1.00 38.46      A    N1+\\nATOM   2826  N   ASN A 344      15.932 -33.927   8.254  1.00 52.34      A    N  \\nATOM   2827  CA  ASN A 344      15.059 -34.822   8.998  1.00 53.13      A    C  \\nATOM   2828  C   ASN A 344      14.291 -34.077  10.068  1.00 53.75      A    C  \\nATOM   2829  O   ASN A 344      13.193 -34.471  10.434  1.00 52.71      A    O  \\nATOM   2830  CB  ASN A 344      15.876 -35.953   9.613  1.00 50.55      A    C  \\nATOM   2831  CG  ASN A 344      16.424 -36.908   8.574  1.00 48.16      A    C  \\nATOM   2832  ND2 ASN A 344      17.633 -37.361   8.767  1.00 46.30      A    N  \\nATOM   2833  OD1 ASN A 344      15.739 -37.236   7.607  1.00 43.84      A    O  \\nATOM   2834  N   ILE A 345      14.854 -33.007  10.585  1.00 48.88      A    N  \\nATOM   2835  CA  ILE A 345      14.144 -32.191  11.562  1.00 51.23      A    C  \\nATOM   2836  C   ILE A 345      13.044 -31.378  10.901  1.00 51.51      A    C  \\nATOM   2837  O   ILE A 345      11.951 -31.244  11.442  1.00 50.32      A    O  \\nATOM   2838  CB  ILE A 345      15.128 -31.289  12.333  1.00 47.72      A    C  \\nATOM   2839  CG1 ILE A 345      16.125 -32.151  13.138  1.00 51.32      A    C  \\nATOM   2840  CG2 ILE A 345      14.409 -30.358  13.287  1.00 48.05      A    C  \\nATOM   2841  CD1 ILE A 345      17.224 -31.335  13.770  1.00 45.14      A    C  \\nATOM   2842  N   HIS A 346      13.341 -30.854   9.736  1.00 50.11      A    N  \\nATOM   2843  CA  HIS A 346      12.328 -30.086   9.012  1.00 52.18      A    C  \\nATOM   2844  C   HIS A 346      11.177 -30.975   8.546  1.00 53.19      A    C  \\nATOM   2845  O   HIS A 346      10.035 -30.557   8.567  1.00 52.56      A    O  \\nATOM   2846  CB  HIS A 346      12.944 -29.382   7.814  1.00 48.24      A    C  \\nATOM   2847  CG  HIS A 346      13.778 -28.188   8.178  1.00 45.78      A    C  \\nATOM   2848  CD2 HIS A 346      13.459 -26.873   8.150  1.00 43.33      A    C  \\nATOM   2849  ND1 HIS A 346      15.104 -28.301   8.602  1.00 43.17      A    N  \\nATOM   2850  CE1 HIS A 346      15.534 -27.079   8.823  1.00 39.93      A    C  \\nATOM   2851  NE2 HIS A 346      14.586 -26.199   8.559  1.00 36.23      A    N  \\nATOM   2852  N   SER A 347      11.506 -32.217   8.173  1.00 52.90      A    N  \\nATOM   2853  CA  SER A 347      10.450 -33.160   7.768  1.00 54.94      A    C  \\nATOM   2854  C   SER A 347       9.654 -33.667   8.952  1.00 55.23      A    C  \\nATOM   2855  O   SER A 347       8.489 -34.027   8.806  1.00 54.46      A    O  \\nATOM   2856  CB  SER A 347      11.060 -34.315   6.993  1.00 51.85      A    C  \\nATOM   2857  OG  SER A 347      11.868 -35.123   7.832  1.00 50.69      A    O  \\nATOM   2858  N   HIS A 348      10.279 -33.732  10.116  1.00 54.28      A    N  \\nATOM   2859  CA  HIS A 348       9.586 -34.191  11.312  1.00 57.01      A    C  \\nATOM   2860  C   HIS A 348       8.621 -33.130  11.840  1.00 57.52      A    C  \\nATOM   2861  O   HIS A 348       7.590 -33.448  12.413  1.00 58.33      A    O  \\nATOM   2862  CB  HIS A 348      10.573 -34.585  12.392  1.00 55.25      A    C  \\nATOM   2863  CG  HIS A 348       9.930 -35.077  13.651  1.00 52.92      A    C  \\nATOM   2864  CD2 HIS A 348       9.605 -36.340  14.032  1.00 49.47      A    C  \\nATOM   2865  ND1 HIS A 348       9.602 -34.243  14.673  1.00 47.23      A    N  \\nATOM   2866  CE1 HIS A 348       9.099 -34.960  15.661  1.00 47.31      A    C  \\nATOM   2867  NE2 HIS A 348       9.072 -36.225  15.309  1.00 45.74      A    N  \\nATOM   2868  N   GLU A 349       8.976 -31.866  11.653  1.00 50.31      A    N  \\nATOM   2869  CA  GLU A 349       8.105 -30.790  12.140  1.00 52.92      A    C  \\nATOM   2870  C   GLU A 349       6.962 -30.497  11.195  1.00 53.79      A    C  \\nATOM   2871  O   GLU A 349       5.938 -29.982  11.629  1.00 54.28      A    O  \\nATOM   2872  CB  GLU A 349       8.931 -29.529  12.384  1.00 51.20      A    C  \\nATOM   2873  CG  GLU A 349       9.918 -29.642  13.531  1.00 48.15      A    C  \\nATOM   2874  CD  GLU A 349      10.723 -28.374  13.749  1.00 44.93      A    C  \\nATOM   2875  OE1 GLU A 349      10.602 -27.434  12.943  1.00 43.12      A    O  \\nATOM   2876  OE2 GLU A 349      11.483 -28.314  14.737  1.00 40.64      A    O1-\\nATOM   2877  N   ILE A 350       7.147 -30.801   9.934  1.00 47.84      A    N  \\nATOM   2878  CA  ILE A 350       6.073 -30.557   8.954  1.00 49.39      A    C  \\nATOM   2879  C   ILE A 350       4.918 -31.534   9.183  1.00 50.49      A    C  \\nATOM   2880  O   ILE A 350       3.748 -31.153   9.067  1.00 50.53      A    O  \\nATOM   2881  CB  ILE A 350       6.634 -30.647   7.515  1.00 46.77      A    C  \\nATOM   2882  CG1 ILE A 350       7.504 -29.436   7.211  1.00 46.91      A    C  \\nATOM   2883  CG2 ILE A 350       5.501 -30.737   6.494  1.00 44.50      A    C  \\nATOM   2884  CD1 ILE A 350       8.262 -29.518   5.922  1.00 43.25      A    C  \\nATOM   2885  N   LYS A 351       5.242 -32.778   9.528  1.00 52.81      A    N  \\nATOM   2886  CA  LYS A 351       4.187 -33.798   9.745  1.00 54.40      A    C  \\nATOM   2887  C   LYS A 351       3.339 -33.436  10.951  1.00 54.85      A    C  \\nATOM   2888  O   LYS A 351       2.148 -33.671  10.954  1.00 55.00      A    O  \\nATOM   2889  CB  LYS A 351       4.819 -35.159   9.912  1.00 52.14      A    C  \\nATOM   2890  CG  LYS A 351       5.440 -35.704   8.641  1.00 49.21      A    C  \\nATOM   2891  CD  LYS A 351       6.068 -37.060   8.882  1.00 45.98      A    C  \\nATOM   2892  CE  LYS A 351       6.719 -37.595   7.623  1.00 41.29      A    C  \\nATOM   2893  NZ  LYS A 351       7.305 -38.928   7.798  1.00 36.02      A    N1+\\nATOM   2894  N   LYS A 352       3.963 -32.904  11.990  1.00 53.92      A    N  \\nATOM   2895  CA  LYS A 352       3.207 -32.488  13.165  1.00 56.28      A    C  \\nATOM   2896  C   LYS A 352       2.410 -31.205  12.918  1.00 57.03      A    C  \\nATOM   2897  O   LYS A 352       1.358 -31.003  13.556  1.00 57.11      A    O  \\nATOM   2898  CB  LYS A 352       4.130 -32.316  14.386  1.00 54.34      A    C  \\nATOM   2899  CG  LYS A 352       4.725 -33.602  14.879  1.00 52.12      A    C  \\nATOM   2900  CD  LYS A 352       5.572 -33.394  16.116  1.00 48.94      A    C  \\nATOM   2901  CE  LYS A 352       6.167 -34.678  16.608  1.00 45.23      A    C  \\nATOM   2902  NZ  LYS A 352       6.993 -34.508  17.817  1.00 40.17      A    N1+\\nATOM   2903  N   LEU A 353       2.891 -30.375  12.063  1.00 48.40      A    N  \\nATOM   2904  CA  LEU A 353       2.144 -29.178  11.704  1.00 50.68      A    C  \\nATOM   2905  C   LEU A 353       0.968 -29.502  10.786  1.00 51.05      A    C  \\nATOM   2906  O   LEU A 353      -0.085 -28.861  10.877  1.00 51.54      A    O  \\nATOM   2907  CB  LEU A 353       3.040 -28.143  11.051  1.00 48.85      A    C  \\nATOM   2908  CG  LEU A 353       4.056 -27.457  11.953  1.00 46.49      A    C  \\nATOM   2909  CD1 LEU A 353       3.357 -26.598  12.993  1.00 46.94      A    C  \\nATOM   2910  CD2 LEU A 353       4.990 -26.582  11.119  1.00 46.22      A    C  \\nATOM   2911  N   GLN A 354       1.126 -30.474   9.925  1.00 51.17      A    N  \\nATOM   2912  CA  GLN A 354       0.061 -30.849   9.007  1.00 53.23      A    C  \\nATOM   2913  C   GLN A 354      -1.089 -31.562   9.720  1.00 54.03      A    C  \\nATOM   2914  O   GLN A 354      -2.238 -31.462   9.313  1.00 54.54      A    O  \\nATOM   2915  CB  GLN A 354       0.577 -31.728   7.881  1.00 51.09      A    C  \\nATOM   2916  CG  GLN A 354       1.451 -31.020   6.877  1.00 48.30      A    C  \\nATOM   2917  CD  GLN A 354       1.947 -31.914   5.779  1.00 45.03      A    C  \\nATOM   2918  NE2 GLN A 354       2.659 -31.336   4.813  1.00 41.01      A    N  \\nATOM   2919  OE1 GLN A 354       1.703 -33.115   5.810  1.00 43.47      A    O  \\nATOM   2920  N   ASP A 355      -0.750 -32.288  10.787  1.00 55.89      A    N  \\nATOM   2921  CA  ASP A 355      -1.803 -32.983  11.543  1.00 56.34      A    C  \\nATOM   2922  C   ASP A 355      -2.601 -32.031  12.399  1.00 56.74      A    C  \\nATOM   2923  O   ASP A 355      -3.725 -32.343  12.790  1.00 56.79      A    O  \\nATOM   2924  CB  ASP A 355      -1.179 -34.072  12.410  1.00 54.51      A    C  \\nATOM   2925  CG  ASP A 355      -0.605 -35.210  11.588  1.00 52.00      A    C  \\nATOM   2926  OD1 ASP A 355      -1.120 -35.485  10.504  1.00 49.43      A    O  \\nATOM   2927  OD2 ASP A 355       0.357 -35.834  12.065  1.00 50.37      A    O1-\\nATOM   2928  N   ARG A 356      -2.042 -30.866  12.722  1.00 49.35      A    N  \\nATOM   2929  CA  ARG A 356      -2.751 -29.886  13.538  1.00 51.35      A    C  \\nATOM   2930  C   ARG A 356      -3.570 -28.896  12.717  1.00 51.14      A    C  \\nATOM   2931  O   ARG A 356      -4.345 -28.143  13.292  1.00 50.83      A    O  \\nATOM   2932  CB  ARG A 356      -1.792 -29.134  14.437  1.00 49.51      A    C  \\nATOM   2933  CG  ARG A 356      -1.154 -29.973  15.503  1.00 47.28      A    C  \\nATOM   2934  CD  ARG A 356      -0.150 -29.159  16.325  1.00 45.06      A    C  \\nATOM   2935  NE  ARG A 356       0.497 -29.962  17.340  1.00 42.56      A    N  \\nATOM   2936  CZ  ARG A 356       1.394 -29.497  18.204  1.00 40.03      A    C  \\nATOM   2937  NH1 ARG A 356       1.771 -28.209  18.107  1.00 34.31      A    N1+\\nATOM   2938  NH2 ARG A 356       1.960 -30.271  19.075  1.00 32.32      A    N  \\nATOM   2939  N   LEU A 357      -3.445 -28.927  11.419  1.00 46.91      A    N  \\nATOM   2940  CA  LEU A 357      -4.247 -28.053  10.570  1.00 49.01      A    C  \\nATOM   2941  C   LEU A 357      -5.712 -28.528  10.490  1.00 49.32      A    C  \\nATOM   2942  O   LEU A 357      -6.609 -27.719  10.336  1.00 49.36      A    O  \\nATOM   2943  CB  LEU A 357      -3.675 -27.982   9.153  1.00 47.01      A    C  \\nATOM   2944  CG  LEU A 357      -2.324 -27.277   8.998  1.00 44.47      A    C  \\nATOM   2945  CD1 LEU A 357      -2.408 -25.813   9.396  1.00 44.28      A    C  \\nATOM   2946  CD2 LEU A 357      -1.836 -27.417   7.571  1.00 43.56      A    C  \\nATOM   2947  N   SER A 358      -5.907 -29.830  10.590  1.00 50.46      A    N  \\nATOM   2948  CA  SER A 358      -7.256 -30.358  10.555  1.00 51.95      A    C  \\nATOM   2949  C   SER A 358      -8.009 -30.167  11.860  1.00 51.21      A    C  \\nATOM   2950  O   SER A 358      -9.222 -29.958  11.869  1.00 49.91      A    O  \\nATOM   2951  CB  SER A 358      -7.238 -31.850  10.175  1.00 49.28      A    C  \\nATOM   2952  OG  SER A 358      -6.599 -32.616  11.175  1.00 44.19      A    O  \\nATOM   2953  N   ASP A 359      -7.294 -30.221  12.966  1.00 53.13      A    N  \\nATOM   2954  CA  ASP A 359      -7.946 -30.140  14.263  1.00 54.39      A    C  \\nATOM   2955  C   ASP A 359      -7.824 -28.770  14.902  1.00 53.73      A    C  \\nATOM   2956  O   ASP A 359      -8.806 -28.164  15.315  1.00 52.88      A    O  \\nATOM   2957  CB  ASP A 359      -7.365 -31.206  15.210  1.00 52.73      A    C  \\nATOM   2958  CG  ASP A 359      -7.700 -32.619  14.761  1.00 50.09      A    C  \\nATOM   2959  OD1 ASP A 359      -8.779 -32.819  14.178  1.00 45.01      A    O  \\nATOM   2960  OD2 ASP A 359      -6.866 -33.519  14.997  1.00 45.73      A    O1-\\nATOM   2961  N   ILE A 360      -6.598 -28.293  15.057  1.00 47.55      A    N  \\nATOM   2962  CA  ILE A 360      -6.363 -27.032  15.751  1.00 50.07      A    C  \\nATOM   2963  C   ILE A 360      -6.689 -25.869  14.827  1.00 49.73      A    C  \\nATOM   2964  O   ILE A 360      -6.124 -25.767  13.730  1.00 48.31      A    O  \\nATOM   2965  CB  ILE A 360      -4.927 -26.930  16.282  1.00 46.57      A    C  \\nATOM   2966  CG1 ILE A 360      -4.675 -28.048  17.314  1.00 49.68      A    C  \\nATOM   2967  CG2 ILE A 360      -4.687 -25.575  16.921  1.00 45.58      A    C  \\nATOM   2968  CD1 ILE A 360      -3.252 -28.131  17.765  1.00 44.40      A    C  \\nATOM   2969  N   SER A 361      -7.581 -25.032  15.254  1.00 44.13      A    N  \\nATOM   2970  CA  SER A 361      -7.894 -23.788  14.536  1.00 46.47      A    C  \\nATOM   2971  C   SER A 361      -7.260 -22.625  15.287  1.00 46.39      A    C  \\nATOM   2972  O   SER A 361      -7.607 -22.342  16.431  1.00 44.65      A    O  \\nATOM   2973  CB  SER A 361      -9.389 -23.607  14.359  1.00 43.06      A    C  \\nATOM   2974  OG  SER A 361     -10.003 -23.442  15.630  1.00 38.67      A    O  \\nATOM   2975  N   ASP A 362      -6.315 -21.998  14.648  1.00 50.98      A    N  \\nATOM   2976  CA  ASP A 362      -5.644 -20.859  15.248  1.00 53.14      A    C  \\nATOM   2977  C   ASP A 362      -6.248 -19.580  14.738  1.00 52.80      A    C  \\nATOM   2978  O   ASP A 362      -6.742 -19.520  13.607  1.00 51.90      A    O  \\nATOM   2979  CB  ASP A 362      -4.131 -20.903  14.935  1.00 50.56      A    C  \\nATOM   2980  CG  ASP A 362      -3.332 -19.922  15.795  1.00 49.17      A    C  \\nATOM   2981  OD1 ASP A 362      -3.914 -19.209  16.625  1.00 44.21      A    O  \\nATOM   2982  OD2 ASP A 362      -2.091 -19.893  15.577  1.00 45.90      A    O1-\\nATOM   2983  N   SER A 363      -6.210 -18.560  15.570  1.00 46.81      A    N  \\nATOM   2984  CA  SER A 363      -6.700 -17.241  15.182  1.00 49.36      A    C  \\nATOM   2985  C   SER A 363      -5.579 -16.475  14.517  1.00 49.32      A    C  \\nATOM   2986  O   SER A 363      -4.550 -16.204  15.126  1.00 48.19      A    O  \\nATOM   2987  CB  SER A 363      -7.239 -16.473  16.376  1.00 45.66      A    C  \\nATOM   2988  OG  SER A 363      -6.204 -16.200  17.314  1.00 43.34      A    O  \\nATOM   2989  N   SER A 364      -5.776 -16.148  13.271  1.00 46.37      A    N  \\nATOM   2990  CA  SER A 364      -4.799 -15.394  12.516  1.00 49.50      A    C  \\nATOM   2991  C   SER A 364      -5.296 -13.995  12.250  1.00 49.31      A    C  \\nATOM   2992  O   SER A 364      -6.394 -13.605  12.655  1.00 48.10      A    O  \\nATOM   2993  CB  SER A 364      -4.486 -16.095  11.203  1.00 45.80      A    C  \\nATOM   2994  OG  SER A 364      -5.627 -16.115  10.372  1.00 43.35      A    O  \\nATOM   2995  N   SER A 365      -4.482 -13.222  11.593  1.00 47.56      A    N  \\nATOM   2996  CA  SER A 365      -4.878 -11.848  11.253  1.00 50.85      A    C  \\nATOM   2997  C   SER A 365      -5.966 -11.821  10.196  1.00 51.16      A    C  \\nATOM   2998  O   SER A 365      -6.796 -10.920  10.191  1.00 49.83      A    O  \\nATOM   2999  CB  SER A 365      -3.665 -11.056  10.780  1.00 47.22      A    C  \\nATOM   3000  OG  SER A 365      -3.172 -11.564   9.552  1.00 43.68      A    O  \\nATOM   3001  N   GLN A 366      -5.957 -12.818   9.338  1.00 51.38      A    N  \\nATOM   3002  CA  GLN A 366      -7.001 -12.888   8.314  1.00 53.32      A    C  \\nATOM   3003  C   GLN A 366      -8.359 -13.260   8.909  1.00 53.07      A    C  \\nATOM   3004  O   GLN A 366      -9.389 -12.771   8.468  1.00 52.60      A    O  \\nATOM   3005  CB  GLN A 366      -6.623 -13.879   7.225  1.00 50.54      A    C  \\nATOM   3006  CG  GLN A 366      -5.389 -13.498   6.416  1.00 48.45      A    C  \\nATOM   3007  CD  GLN A 366      -5.038 -14.511   5.371  1.00 45.57      A    C  \\nATOM   3008  NE2 GLN A 366      -4.007 -14.228   4.585  1.00 40.62      A    N  \\nATOM   3009  OE1 GLN A 366      -5.695 -15.548   5.268  1.00 42.82      A    O  \\nATOM   3010  N   LYS A 367      -8.326 -14.143   9.910  1.00 45.79      A    N  \\nATOM   3011  CA  LYS A 367      -9.569 -14.544  10.552  1.00 48.80      A    C  \\nATOM   3012  C   LYS A 367     -10.150 -13.431  11.411  1.00 48.27      A    C  \\nATOM   3013  O   LYS A 367     -11.373 -13.303  11.539  1.00 46.59      A    O  \\nATOM   3014  CB  LYS A 367      -9.355 -15.795  11.405  1.00 45.46      A    C  \\nATOM   3015  CG  LYS A 367      -9.010 -17.038  10.593  1.00 44.28      A    C  \\nATOM   3016  CD  LYS A 367      -8.843 -18.242  11.483  1.00 41.28      A    C  \\nATOM   3017  CE  LYS A 367      -8.453 -19.473  10.692  1.00 36.30      A    C  \\nATOM   3018  NZ  LYS A 367      -8.324 -20.666  11.551  1.00 33.39      A    N1+\\nATOM   3019  N   ASP A 368      -9.283 -12.630  12.014  1.00 45.48      A    N  \\nATOM   3020  CA  ASP A 368      -9.772 -11.486  12.780  1.00 47.57      A    C  \\nATOM   3021  C   ASP A 368     -10.353 -10.420  11.874  1.00 47.40      A    C  \\nATOM   3022  O   ASP A 368     -11.291  -9.725  12.271  1.00 45.46      A    O  \\nATOM   3023  CB  ASP A 368      -8.636 -10.900  13.604  1.00 44.69      A    C  \\nATOM   3024  CG  ASP A 368      -8.191 -11.833  14.723  1.00 43.71      A    C  \\nATOM   3025  OD1 ASP A 368      -9.022 -12.577  15.265  1.00 37.86      A    O  \\nATOM   3026  OD2 ASP A 368      -6.999 -11.801  15.057  1.00 37.14      A    O1-\\nATOM   3027  N   ASN A 369      -9.814 -10.306  10.699  1.00 47.15      A    N  \\nATOM   3028  CA  ASN A 369     -10.377  -9.354   9.741  1.00 49.04      A    C  \\nATOM   3029  C   ASN A 369     -11.705  -9.839   9.177  1.00 49.39      A    C  \\nATOM   3030  O   ASN A 369     -12.566  -9.020   8.846  1.00 47.36      A    O  \\nATOM   3031  CB  ASN A 369      -9.370  -9.085   8.613  1.00 45.33      A    C  \\nATOM   3032  CG  ASN A 369      -8.152  -8.325   9.086  1.00 45.02      A    C  \\nATOM   3033  ND2 ASN A 369      -6.975  -8.700   8.604  1.00 40.99      A    N  \\nATOM   3034  OD1 ASN A 369      -8.255  -7.387   9.864  1.00 39.77      A    O  \\nATOM   3035  N   ASP A 370     -11.860 -11.134   9.065  1.00 44.16      A    N  \\nATOM   3036  CA  ASP A 370     -13.107 -11.693   8.567  1.00 46.74      A    C  \\nATOM   3037  C   ASP A 370     -14.221 -11.550   9.614  1.00 46.85      A    C  \\nATOM   3038  O   ASP A 370     -15.389 -11.372   9.253  1.00 45.06      A    O  \\nATOM   3039  CB  ASP A 370     -12.944 -13.145   8.178  1.00 43.89      A    C  \\nATOM   3040  CG  ASP A 370     -12.112 -13.342   6.932  1.00 43.13      A    C  \\nATOM   3041  OD1 ASP A 370     -12.128 -12.453   6.068  1.00 36.68      A    O  \\nATOM   3042  OD2 ASP A 370     -11.451 -14.406   6.821  1.00 36.59      A    O1-\\nATOM   3043  N   ASP A 371     -13.844 -11.596  10.876  1.00 46.25      A    N  \\nATOM   3044  CA  ASP A 371     -14.834 -11.372  11.918  1.00 47.83      A    C  \\nATOM   3045  C   ASP A 371     -15.255  -9.920  11.991  1.00 47.61      A    C  \\nATOM   3046  O   ASP A 371     -16.400  -9.607  12.307  1.00 46.07      A    O  \\nATOM   3047  CB  ASP A 371     -14.318 -11.842  13.272  1.00 44.62      A    C  \\nATOM   3048  CG  ASP A 371     -14.159 -13.351  13.361  1.00 44.05      A    C  \\nATOM   3049  OD1 ASP A 371     -14.961 -14.062  12.711  1.00 37.18      A    O  \\nATOM   3050  OD2 ASP A 371     -13.261 -13.800  14.066  1.00 37.31      A    O1-\\nATOM   3051  N   GLU A 372     -14.344  -9.029  11.683  1.00 40.22      A    N  \\nATOM   3052  CA  GLU A 372     -14.700  -7.633  11.612  1.00 43.10      A    C  \\nATOM   3053  C   GLU A 372     -15.569  -7.328  10.406  1.00 42.86      A    C  \\nATOM   3054  O   GLU A 372     -16.409  -6.434  10.439  1.00 40.66      A    O  \\nATOM   3055  CB  GLU A 372     -13.416  -6.767  11.563  1.00 39.73      A    C  \\nATOM   3056  CG  GLU A 372     -12.602  -6.807  12.824  1.00 39.13      A    C  \\nATOM   3057  CD  GLU A 372     -11.339  -5.981  12.741  1.00 37.12      A    C  \\nATOM   3058  OE1 GLU A 372     -11.074  -5.416  11.661  1.00 33.50      A    O  \\nATOM   3059  OE2 GLU A 372     -10.603  -5.893  13.755  1.00 31.29      A    O1-\\nATOM   3060  N   ASP A 373     -15.340  -8.061   9.334  1.00 43.46      A    N  \\nATOM   3061  CA  ASP A 373     -16.184  -7.895   8.157  1.00 45.20      A    C  \\nATOM   3062  C   ASP A 373     -17.568  -8.490   8.374  1.00 44.74      A    C  \\nATOM   3063  O   ASP A 373     -18.560  -7.941   7.853  1.00 42.97      A    O  \\nATOM   3064  CB  ASP A 373     -15.510  -8.543   6.934  1.00 41.71      A    C  \\nATOM   3065  CG  ASP A 373     -14.278  -7.801   6.478  1.00 40.91      A    C  \\nATOM   3066  OD1 ASP A 373     -14.217  -6.559   6.627  1.00 35.19      A    O  \\nATOM   3067  OD2 ASP A 373     -13.334  -8.452   5.949  1.00 35.21      A    O1-\\nATOM   3068  N   GLU A 374     -17.630  -9.561   9.113  1.00 40.60      A    N  \\nATOM   3069  CA  GLU A 374     -18.922 -10.133   9.421  1.00 42.74      A    C  \\nATOM   3070  C   GLU A 374     -19.696  -9.230  10.405  1.00 42.50      A    C  \\nATOM   3071  O   GLU A 374     -20.915  -9.147  10.332  1.00 40.47      A    O  \\nATOM   3072  CB  GLU A 374     -18.806 -11.544   9.979  1.00 39.94      A    C  \\nATOM   3073  CG  GLU A 374     -18.361 -12.558   8.962  1.00 39.00      A    C  \\nATOM   3074  CD  GLU A 374     -18.187 -13.947   9.553  1.00 37.10      A    C  \\nATOM   3075  OE1 GLU A 374     -18.339 -14.109  10.774  1.00 33.26      A    O  \\nATOM   3076  OE2 GLU A 374     -17.883 -14.899   8.772  1.00 31.41      A    O1-\\nATOM   3077  N   ASP A 375     -18.974  -8.615  11.312  1.00 45.23      A    N  \\nATOM   3078  CA  ASP A 375     -19.608  -7.669  12.221  1.00 47.32      A    C  \\nATOM   3079  C   ASP A 375     -20.044  -6.391  11.478  1.00 46.84      A    C  \\nATOM   3080  O   ASP A 375     -20.984  -5.740  11.895  1.00 45.13      A    O  \\nATOM   3081  CB  ASP A 375     -18.656  -7.299  13.367  1.00 43.74      A    C  \\nATOM   3082  CG  ASP A 375     -18.422  -8.469  14.312  1.00 42.74      A    C  \\nATOM   3083  OD1 ASP A 375     -19.327  -9.318  14.458  1.00 36.90      A    O  \\nATOM   3084  OD2 ASP A 375     -17.310  -8.511  14.891  1.00 37.25      A    O1-\\nATOM   3085  N   ALA A 376     -19.338  -6.069  10.445  1.00 41.56      A    N  \\nATOM   3086  CA  ALA A 376     -19.727  -4.927   9.635  1.00 43.52      A    C  \\nATOM   3087  C   ALA A 376     -20.914  -5.238   8.740  1.00 43.56      A    C  \\nATOM   3088  O   ALA A 376     -21.684  -4.345   8.384  1.00 42.52      A    O  \\nATOM   3089  CB  ALA A 376     -18.565  -4.443   8.772  1.00 41.26      A    C  \\nATOM   3090  N   ASP A 377     -21.067  -6.514   8.395  1.00 45.13      A    N  \\nATOM   3091  CA  ASP A 377     -22.205  -6.916   7.591  1.00 47.57      A    C  \\nATOM   3092  C   ASP A 377     -23.490  -6.959   8.420  1.00 47.24      A    C  \\nATOM   3093  O   ASP A 377     -24.588  -6.747   7.880  1.00 45.71      A    O  \\nATOM   3094  CB  ASP A 377     -21.944  -8.280   6.933  1.00 44.31      A    C  \\nATOM   3095  CG  ASP A 377     -20.888  -8.225   5.861  1.00 42.62      A    C  \\nATOM   3096  OD1 ASP A 377     -20.783  -7.200   5.166  1.00 37.23      A    O  \\nATOM   3097  OD2 ASP A 377     -20.138  -9.231   5.711  1.00 37.56      A    O1-\\nATOM   3098  N   GLU A 378     -23.334  -7.277   9.703  1.00 43.38      A    N  \\nATOM   3099  CA  GLU A 378     -24.511  -7.293  10.542  1.00 46.15      A    C  \\nATOM   3100  C   GLU A 378     -24.983  -5.906  10.890  1.00 45.83      A    C  \\nATOM   3101  O   GLU A 378     -26.194  -5.698  11.101  1.00 44.31      A    O  \\nATOM   3102  CB  GLU A 378     -24.213  -8.086  11.817  1.00 43.81      A    C  \\nATOM   3103  CG  GLU A 378     -23.969  -9.560  11.589  1.00 43.19      A    C  \\nATOM   3104  CD  GLU A 378     -23.641 -10.300  12.869  1.00 40.31      A    C  \\nATOM   3105  OE1 GLU A 378     -23.514  -9.634  13.933  1.00 37.73      A    O  \\nATOM   3106  OE2 GLU A 378     -23.512 -11.555  12.823  1.00 36.03      A    O1-\\nATOM   3107  N   GLU A 379     -24.070  -4.977  11.027  1.00 44.02      A    N  \\nATOM   3108  CA  GLU A 379     -24.417  -3.605  11.281  1.00 47.55      A    C  \\nATOM   3109  C   GLU A 379     -24.750  -2.885   9.996  1.00 47.19      A    C  \\nATOM   3110  O   GLU A 379     -24.538  -3.358   8.910  1.00 44.76      A    O  \\nATOM   3111  CB  GLU A 379     -23.281  -2.899  12.019  1.00 43.42      A    C  \\nATOM   3112  CG  GLU A 379     -23.029  -3.430  13.417  1.00 42.49      A    C  \\nATOM   3113  CD  GLU A 379     -21.862  -2.736  14.097  1.00 39.67      A    C  \\nATOM   3114  OE1 GLU A 379     -21.208  -1.869  13.470  1.00 35.85      A    O  \\nATOM   3115  OE2 GLU A 379     -21.588  -3.043  15.286  1.00 33.38      A    O1-\\nATOM   3116  N   SER A 380     -25.304  -1.684  10.175  1.00 40.67      A    N  \\nATOM   3117  CA  SER A 380     -25.706  -0.889   9.028  1.00 42.89      A    C  \\nATOM   3118  C   SER A 380     -24.493  -0.292   8.345  1.00 42.33      A    C  \\nATOM   3119  O   SER A 380     -23.504   0.066   8.992  1.00 41.03      A    O  \\nATOM   3120  CB  SER A 380     -26.687   0.196   9.427  1.00 39.52      A    C  \\nATOM   3121  OG  SER A 380     -26.055   1.126  10.316  1.00 37.88      A    O  \\nATOM   3122  N   SER A 381     -24.563  -0.180   7.062  1.00 42.87      A    N  \\nATOM   3123  CA  SER A 381     -23.474   0.387   6.266  1.00 47.10      A    C  \\nATOM   3124  C   SER A 381     -23.496   1.917   6.254  1.00 46.87      A    C  \\nATOM   3125  O   SER A 381     -22.568   2.530   5.743  1.00 45.45      A    O  \\nATOM   3126  CB  SER A 381     -23.527  -0.132   4.840  1.00 43.63      A    C  \\nATOM   3127  OG  SER A 381     -24.702   0.315   4.184  1.00 39.97      A    O  \\nATOM   3128  N   GLU A 382     -24.576   2.490   6.761  1.00 45.12      A    N  \\nATOM   3129  CA  GLU A 382     -24.681   3.955   6.770  1.00 47.81      A    C  \\nATOM   3130  C   GLU A 382     -23.793   4.535   7.860  1.00 47.96      A    C  \\nATOM   3131  O   GLU A 382     -23.719   4.016   8.974  1.00 46.59      A    O  \\nATOM   3132  CB  GLU A 382     -26.120   4.385   6.995  1.00 45.32      A    C  \\nATOM   3133  CG  GLU A 382     -27.031   4.012   5.839  1.00 44.10      A    C  \\nATOM   3134  CD  GLU A 382     -28.455   4.453   6.047  1.00 40.77      A    C  \\nATOM   3135  OE1 GLU A 382     -28.783   4.959   7.152  1.00 39.39      A    O  \\nATOM   3136  OE2 GLU A 382     -29.295   4.243   5.129  1.00 36.80      A    O1-\\nATOM   3137  N   ILE A 383     -23.160   5.632   7.542  1.00 42.09      A    N  \\nATOM   3138  CA  ILE A 383     -22.263   6.300   8.452  1.00 46.50      A    C  \\nATOM   3139  C   ILE A 383     -22.973   7.516   9.019  1.00 47.21      A    C  \\nATOM   3140  O   ILE A 383     -23.452   8.369   8.266  1.00 45.68      A    O  \\nATOM   3141  CB  ILE A 383     -20.945   6.705   7.739  1.00 42.59      A    C  \\nATOM   3142  CG1 ILE A 383     -20.192   5.455   7.283  1.00 45.99      A    C  \\nATOM   3143  CG2 ILE A 383     -20.058   7.522   8.689  1.00 43.08      A    C  \\nATOM   3144  CD1 ILE A 383     -18.987   5.774   6.438  1.00 41.45      A    C  \\nATOM   3145  N   ASP A 384     -23.053   7.588  10.321  1.00 48.34      A    N  \\nATOM   3146  CA  ASP A 384     -23.695   8.711  10.974  1.00 50.27      A    C  \\nATOM   3147  C   ASP A 384     -22.785   9.943  10.943  1.00 50.63      A    C  \\nATOM   3148  O   ASP A 384     -21.566   9.825  10.827  1.00 49.72      A    O  \\nATOM   3149  CB  ASP A 384     -24.087   8.382  12.419  1.00 47.63      A    C  \\nATOM   3150  CG  ASP A 384     -25.171   7.320  12.468  1.00 46.05      A    C  \\nATOM   3151  OD1 ASP A 384     -26.041   7.311  11.568  1.00 41.00      A    O  \\nATOM   3152  OD2 ASP A 384     -25.157   6.524  13.420  1.00 41.06      A    O1-\\nATOM   3153  N   ALA A 385     -23.406  11.090  11.073  1.00 47.29      A    N  \\nATOM   3154  CA  ALA A 385     -22.624  12.330  11.094  1.00 50.71      A    C  \\nATOM   3155  C   ALA A 385     -22.040  12.627  12.445  1.00 51.59      A    C  \\nATOM   3156  O   ALA A 385     -21.085  13.393  12.574  1.00 51.50      A    O  \\nATOM   3157  CB  ALA A 385     -23.497  13.506  10.653  1.00 48.58      A    C  \\nATOM   3158  N   GLU A 386     -22.599  12.011  13.471  1.00 45.25      A    N  \\nATOM   3159  CA  GLU A 386     -22.105  12.236  14.813  1.00 48.99      A    C  \\nATOM   3160  C   GLU A 386     -20.829  11.468  15.103  1.00 49.40      A    C  \\nATOM   3161  O   GLU A 386     -20.046  11.857  15.988  1.00 49.43      A    O  \\nATOM   3162  CB  GLU A 386     -23.171  11.863  15.837  1.00 47.44      A    C  \\nATOM   3163  CG  GLU A 386     -24.427  12.707  15.767  1.00 45.49      A    C  \\nATOM   3164  CD  GLU A 386     -25.443  12.341  16.810  1.00 42.48      A    C  \\nATOM   3165  OE1 GLU A 386     -25.219  11.343  17.538  1.00 41.24      A    O  \\nATOM   3166  OE2 GLU A 386     -26.485  13.041  16.917  1.00 37.75      A    O1-\\nATOM   3167  N   GLU A 387     -20.592  10.396  14.371  1.00 42.74      A    N  \\nATOM   3168  CA  GLU A 387     -19.388   9.584  14.559  1.00 45.91      A    C  \\nATOM   3169  C   GLU A 387     -18.196  10.132  13.788  1.00 45.86      A    C  \\nATOM   3170  O   GLU A 387     -17.110   9.603  13.904  1.00 45.81      A    O  \\nATOM   3171  CB  GLU A 387     -19.669   8.138  14.150  1.00 44.31      A    C  \\nATOM   3172  CG  GLU A 387     -20.670   7.428  15.025  1.00 43.12      A    C  \\nATOM   3173  CD  GLU A 387     -20.942   6.015  14.579  1.00 41.40      A    C  \\nATOM   3174  OE1 GLU A 387     -20.440   5.622  13.502  1.00 40.58      A    O  \\nATOM   3175  OE2 GLU A 387     -21.631   5.278  15.320  1.00 36.88      A    O1-\\nATOM   3176  N   ARG A 388     -18.420  11.148  13.002  1.00 44.12      A    N  \\nATOM   3177  CA  ARG A 388     -17.332  11.717  12.216  1.00 47.27      A    C  \\nATOM   3178  C   ARG A 388     -16.453  12.588  13.098  1.00 47.14      A    C  \\nATOM   3179  O   ARG A 388     -16.905  13.353  13.931  1.00 47.32      A    O  \\nATOM   3180  CB  ARG A 388     -17.887  12.541  11.058  1.00 45.36      A    C  \\nATOM   3181  CG  ARG A 388     -18.684  11.729  10.064  1.00 43.51      A    C  \\nATOM   3182  CD  ARG A 388     -19.364  12.599   9.030  1.00 41.10      A    C  \\nATOM   3183  NE  ARG A 388     -20.175  11.791   8.117  1.00 37.13      A    N  \\nATOM   3184  CZ  ARG A 388     -20.970  12.300   7.179  1.00 37.06      A    C  \\nATOM   3185  NH1 ARG A 388     -21.086  13.611   7.036  1.00 33.10      A    N1+\\nATOM   3186  NH2 ARG A 388     -21.647  11.496   6.396  1.00 30.47      A    N  \\nATOM   3187  N   GLU A 389     -15.162  12.456  12.893  1.00 46.00      A    N  \\nATOM   3188  CA  GLU A 389     -14.180  13.174  13.663  1.00 48.91      A    C  \\nATOM   3189  C   GLU A 389     -13.173  13.830  12.741  1.00 48.98      A    C  \\nATOM   3190  O   GLU A 389     -12.679  13.225  11.817  1.00 50.22      A    O  \\nATOM   3191  CB  GLU A 389     -13.478  12.281  14.670  1.00 47.90      A    C  \\nATOM   3192  CG  GLU A 389     -14.370  11.752  15.772  1.00 46.76      A    C  \\nATOM   3193  CD  GLU A 389     -13.640  10.873  16.768  1.00 43.08      A    C  \\nATOM   3194  OE1 GLU A 389     -12.461  10.581  16.523  1.00 42.13      A    O  \\nATOM   3195  OE2 GLU A 389     -14.257  10.471  17.765  1.00 37.41      A    O1-\\nATOM   3196  N   ALA A 390     -12.881  15.057  13.043  1.00 46.90      A    N  \\nATOM   3197  CA  ALA A 390     -11.894  15.806  12.271  1.00 49.76      A    C  \\nATOM   3198  C   ALA A 390     -10.585  15.830  13.053  1.00 50.18      A    C  \\nATOM   3199  O   ALA A 390     -10.515  16.321  14.175  1.00 50.73      A    O  \\nATOM   3200  CB  ALA A 390     -12.364  17.236  12.007  1.00 48.47      A    C  \\nATOM   3201  N   ILE A 391      -9.584  15.271  12.444  1.00 45.11      A    N  \\nATOM   3202  CA  ILE A 391      -8.277  15.167  13.086  1.00 48.63      A    C  \\nATOM   3203  C   ILE A 391      -7.455  16.401  12.744  1.00 48.08      A    C  \\nATOM   3204  O   ILE A 391      -7.169  16.684  11.564  1.00 47.91      A    O  \\nATOM   3205  CB  ILE A 391      -7.552  13.878  12.675  1.00 46.74      A    C  \\nATOM   3206  CG1 ILE A 391      -8.356  12.646  13.138  1.00 51.88      A    C  \\nATOM   3207  CG2 ILE A 391      -6.142  13.841  13.255  1.00 47.37      A    C  \\nATOM   3208  CD1 ILE A 391      -7.868  11.359  12.569  1.00 46.27      A    C  \\nATOM   3209  N   ILE A 392      -7.090  17.122  13.753  1.00 40.46      A    N  \\nATOM   3210  CA  ILE A 392      -6.253  18.298  13.601  1.00 44.20      A    C  \\nATOM   3211  C   ILE A 392      -4.788  17.875  13.654  1.00 44.31      A    C  \\nATOM   3212  O   ILE A 392      -4.399  17.007  14.427  1.00 43.88      A    O  \\nATOM   3213  CB  ILE A 392      -6.586  19.349  14.678  1.00 41.52      A    C  \\nATOM   3214  CG1 ILE A 392      -8.050  19.758  14.568  1.00 46.67      A    C  \\nATOM   3215  CG2 ILE A 392      -5.695  20.584  14.542  1.00 42.53      A    C  \\nATOM   3216  CD1 ILE A 392      -8.534  20.606  15.730  1.00 42.83      A    C  \\nATOM   3217  N   ARG A 393      -3.996  18.497  12.826  1.00 46.33      A    N  \\nATOM   3218  CA  ARG A 393      -2.567  18.181  12.803  1.00 48.29      A    C  \\nATOM   3219  C   ARG A 393      -1.922  18.577  14.128  1.00 47.79      A    C  \\nATOM   3220  O   ARG A 393      -2.102  19.666  14.645  1.00 46.94      A    O  \\nATOM   3221  CB  ARG A 393      -1.878  18.889  11.647  1.00 45.81      A    C  \\nATOM   3222  CG  ARG A 393      -2.263  18.405  10.267  1.00 44.71      A    C  \\nATOM   3223  CD  ARG A 393      -1.509  19.123   9.191  1.00 43.31      A    C  \\nATOM   3224  NE  ARG A 393      -1.845  18.644   7.837  1.00 40.64      A    N  \\nATOM   3225  CZ  ARG A 393      -1.317  19.108   6.732  1.00 39.44      A    C  \\nATOM   3226  NH1 ARG A 393      -0.408  20.063   6.789  1.00 34.32      A    N1+\\nATOM   3227  NH2 ARG A 393      -1.677  18.610   5.553  1.00 32.16      A    N  \\nATOM   3228  N   SER A 394      -1.129  17.649  14.630  1.00 42.26      A    N  \\nATOM   3229  CA  SER A 394      -0.443  17.878  15.899  1.00 44.37      A    C  \\nATOM   3230  C   SER A 394       0.920  17.222  15.833  1.00 44.14      A    C  \\nATOM   3231  O   SER A 394       1.213  16.377  15.009  1.00 43.75      A    O  \\nATOM   3232  CB  SER A 394      -1.266  17.330  17.085  1.00 42.10      A    C  \\nATOM   3233  OG  SER A 394      -1.359  15.917  17.018  1.00 38.85      A    O  \\nATOM   3234  N   LYS A 395       1.754  17.655  16.764  1.00 38.19      A    N  \\nATOM   3235  CA  LYS A 395       3.122  17.111  16.822  1.00 42.74      A    C  \\nATOM   3236  C   LYS A 395       3.130  15.722  17.416  1.00 42.64      A    C  \\nATOM   3237  O   LYS A 395       3.735  14.792  16.876  1.00 41.25      A    O  \\nATOM   3238  CB  LYS A 395       4.005  18.031  17.641  1.00 39.12      A    C  \\nATOM   3239  CG  LYS A 395       4.258  19.366  16.967  1.00 38.36      A    C  \\nATOM   3240  CD  LYS A 395       5.172  20.238  17.795  1.00 36.91      A    C  \\nATOM   3241  CE  LYS A 395       5.446  21.562  17.168  1.00 32.19      A    C  \\nATOM   3242  NZ  LYS A 395       6.345  22.408  17.952  1.00 30.48      A    N1+\\nATOM   3243  N   SER A 396       2.462  15.565  18.534  1.00 40.03      A    N  \\nATOM   3244  CA  SER A 396       2.388  14.271  19.212  1.00 43.33      A    C  \\nATOM   3245  C   SER A 396       1.147  13.520  18.751  1.00 43.16      A    C  \\nATOM   3246  O   SER A 396       0.122  14.147  18.470  1.00 42.47      A    O  \\nATOM   3247  CB  SER A 396       2.370  14.445  20.728  1.00 40.95      A    C  \\nATOM   3248  OG  SER A 396       1.209  15.144  21.134  1.00 38.39      A    O  \\nATOM   3249  N   PRO A 397       1.193  12.218  18.705  1.00 38.17      A    N  \\nATOM   3250  CA  PRO A 397      -0.002  11.433  18.341  1.00 40.22      A    C  \\nATOM   3251  C   PRO A 397      -1.030  11.442  19.475  1.00 40.57      A    C  \\nATOM   3252  O   PRO A 397      -0.682  11.626  20.638  1.00 39.95      A    O  \\nATOM   3253  CB  PRO A 397       0.544  10.039  18.080  1.00 37.64      A    C  \\nATOM   3254  CG  PRO A 397       1.769   9.975  18.920  1.00 38.07      A    C  \\nATOM   3255  CD  PRO A 397       2.374  11.342  18.870  1.00 39.56      A    C  \\nATOM   3256  N   PRO A 398      -2.282  11.230  19.169  1.00 41.20      A    N  \\nATOM   3257  CA  PRO A 398      -3.282  11.164  20.226  1.00 44.56      A    C  \\nATOM   3258  C   PRO A 398      -3.189   9.831  20.976  1.00 45.33      A    C  \\nATOM   3259  O   PRO A 398      -2.446   8.919  20.592  1.00 44.85      A    O  \\nATOM   3260  CB  PRO A 398      -4.604  11.304  19.484  1.00 42.19      A    C  \\nATOM   3261  CG  PRO A 398      -4.315  10.803  18.111  1.00 42.46      A    C  \\nATOM   3262  CD  PRO A 398      -2.904  11.226  17.819  1.00 44.13      A    C  \\nATOM   3263  N   SER A 399      -3.993   9.733  22.003  1.00 43.65      A    N  \\nATOM   3264  CA  SER A 399      -3.964   8.531  22.848  1.00 46.67      A    C  \\nATOM   3265  C   SER A 399      -4.635   7.335  22.179  1.00 47.19      A    C  \\nATOM   3266  O   SER A 399      -4.442   6.211  22.630  1.00 46.96      A    O  \\nATOM   3267  CB  SER A 399      -4.647   8.825  24.177  1.00 44.35      A    C  \\nATOM   3268  OG  SER A 399      -6.029   9.091  24.006  1.00 40.69      A    O  \\nATOM   3269  N   GLU A 400      -5.371   7.577  21.123  1.00 41.84      A    N  \\nATOM   3270  CA  GLU A 400      -6.014   6.472  20.431  1.00 44.65      A    C  \\nATOM   3271  C   GLU A 400      -5.048   5.667  19.612  1.00 44.41      A    C  \\nATOM   3272  O   GLU A 400      -5.198   4.461  19.495  1.00 43.16      A    O  \\nATOM   3273  CB  GLU A 400      -7.130   7.000  19.527  1.00 41.40      A    C  \\nATOM   3274  CG  GLU A 400      -8.294   7.624  20.299  1.00 40.60      A    C  \\nATOM   3275  CD  GLU A 400      -9.378   8.128  19.372  1.00 38.78      A    C  \\nATOM   3276  OE1 GLU A 400      -9.202   8.040  18.113  1.00 38.55      A    O  \\nATOM   3277  OE2 GLU A 400     -10.422   8.613  19.877  1.00 37.66      A    O1-\\nATOM   3278  N   PHE A 401      -4.029   6.331  19.088  1.00 37.66      A    N  \\nATOM   3279  CA  PHE A 401      -3.002   5.641  18.335  1.00 40.80      A    C  \\nATOM   3280  C   PHE A 401      -1.952   5.012  19.263  1.00 40.86      A    C  \\nATOM   3281  O   PHE A 401      -1.302   4.058  18.894  1.00 39.60      A    O  \\nATOM   3282  CB  PHE A 401      -2.313   6.583  17.352  1.00 37.09      A    C  \\nATOM   3283  CG  PHE A 401      -3.250   7.142  16.288  1.00 35.11      A    C  \\nATOM   3284  CD1 PHE A 401      -3.601   6.347  15.203  1.00 34.18      A    C  \\nATOM   3285  CD2 PHE A 401      -3.777   8.399  16.419  1.00 34.04      A    C  \\nATOM   3286  CE1 PHE A 401      -4.449   6.847  14.227  1.00 31.40      A    C  \\nATOM   3287  CE2 PHE A 401      -4.625   8.902  15.446  1.00 31.48      A    C  \\nATOM   3288  CZ  PHE A 401      -4.952   8.119  14.349  1.00 30.46      A    C  \\nATOM   3289  N   ALA A 402      -1.808   5.581  20.445  1.00 39.58      A    N  \\nATOM   3290  CA  ALA A 402      -0.842   5.038  21.401  1.00 42.07      A    C  \\nATOM   3291  C   ALA A 402      -1.347   3.759  22.050  1.00 42.55      A    C  \\nATOM   3292  O   ALA A 402      -0.546   2.951  22.503  1.00 41.78      A    O  \\nATOM   3293  CB  ALA A 402      -0.513   6.096  22.468  1.00 39.85      A    C  \\nATOM   3294  N   ASP A 403      -2.646   3.585  22.091  1.00 42.35      A    N  \\nATOM   3295  CA  ASP A 403      -3.208   2.365  22.670  1.00 44.62      A    C  \\nATOM   3296  C   ASP A 403      -3.111   1.182  21.740  1.00 44.78      A    C  \\nATOM   3297  O   ASP A 403      -3.250   0.037  22.198  1.00 43.63      A    O  \\nATOM   3298  CB  ASP A 403      -4.671   2.607  23.068  1.00 42.92      A    C  \\nATOM   3299  CG  ASP A 403      -4.812   3.550  24.242  1.00 42.16      A    C  \\nATOM   3300  OD1 ASP A 403      -3.895   3.596  25.074  1.00 37.92      A    O  \\nATOM   3301  OD2 ASP A 403      -5.840   4.260  24.316  1.00 38.22      A    O1-\\nATOM   3302  N   GLN A 404      -2.903   1.442  20.476  1.00 39.47      A    N  \\nATOM   3303  CA  GLN A 404      -2.810   0.335  19.517  1.00 43.09      A    C  \\nATOM   3304  C   GLN A 404      -1.447  -0.358  19.618  1.00 43.03      A    C  \\nATOM   3305  O   GLN A 404      -1.356  -1.566  19.351  1.00 41.73      A    O  \\nATOM   3306  CB  GLN A 404      -3.035   0.848  18.099  1.00 40.07      A    C  \\nATOM   3307  CG  GLN A 404      -4.445   1.331  17.853  1.00 39.31      A    C  \\nATOM   3308  CD  GLN A 404      -4.654   1.841  16.460  1.00 37.27      A    C  \\nATOM   3309  NE2 GLN A 404      -5.863   2.261  16.151  1.00 32.78      A    N  \\nATOM   3310  OE1 GLN A 404      -3.734   1.811  15.633  1.00 36.06      A    O  \\nATOM   3311  N   TYR A 405      -0.413   0.395  19.968  1.00 33.83      A    N  \\nATOM   3312  CA  TYR A 405       0.921  -0.172  20.023  1.00 37.14      A    C  \\nATOM   3313  C   TYR A 405       1.372  -0.423  21.456  1.00 37.09      A    C  \\nATOM   3314  O   TYR A 405       2.576  -0.412  21.738  1.00 35.76      A    O  \\nATOM   3315  CB  TYR A 405       1.893   0.761  19.299  1.00 33.74      A    C  \\nATOM   3316  CG  TYR A 405       1.620   0.893  17.816  1.00 32.54      A    C  \\nATOM   3317  CD1 TYR A 405       2.068  -0.068  16.915  1.00 31.35      A    C  \\nATOM   3318  CD2 TYR A 405       0.911   1.980  17.322  1.00 30.74      A    C  \\nATOM   3319  CE1 TYR A 405       1.809   0.050  15.571  1.00 27.13      A    C  \\nATOM   3320  CE2 TYR A 405       0.648   2.096  15.973  1.00 27.53      A    C  \\nATOM   3321  CZ  TYR A 405       1.116   1.120  15.115  1.00 26.24      A    C  \\nATOM   3322  OH  TYR A 405       0.877   1.246  13.764  1.00 31.37      A    O  \\nATOM   3323  N   THR A 406       0.427  -0.646  22.333  1.00 40.23      A    N  \\nATOM   3324  CA  THR A 406       0.748  -0.936  23.717  1.00 40.83      A    C  \\nATOM   3325  C   THR A 406       1.216  -2.387  23.864  1.00 41.03      A    C  \\nATOM   3326  O   THR A 406       0.801  -3.264  23.112  1.00 39.55      A    O  \\nATOM   3327  CB  THR A 406      -0.455  -0.683  24.614  1.00 38.35      A    C  \\nATOM   3328  CG2 THR A 406      -0.819   0.797  24.605  1.00 38.03      A    C  \\nATOM   3329  OG1 THR A 406      -1.572  -1.455  24.190  1.00 39.93      A    O  \\nATOM   3330  N   GLY A 407       2.057  -2.597  24.865  1.00 45.24      A    N  \\nATOM   3331  CA  GLY A 407       2.531  -3.933  25.115  1.00 45.40      A    C  \\nATOM   3332  C   GLY A 407       1.521  -4.850  25.761  1.00 47.53      A    C  \\nATOM   3333  O   GLY A 407       1.708  -6.068  25.788  1.00 47.22      A    O  \\nATOM   3334  N   GLU A 408       0.464  -4.306  26.314  1.00 41.89      A    N  \\nATOM   3335  CA  GLU A 408      -0.555  -5.111  26.954  1.00 45.53      A    C  \\nATOM   3336  C   GLU A 408      -1.371  -5.880  25.948  1.00 45.71      A    C  \\nATOM   3337  O   GLU A 408      -1.788  -7.008  26.222  1.00 44.58      A    O  \\nATOM   3338  CB  GLU A 408      -1.456  -4.230  27.817  1.00 43.03      A    C  \\nATOM   3339  CG  GLU A 408      -0.781  -3.585  28.992  1.00 41.49      A    C  \\nATOM   3340  CD  GLU A 408      -1.700  -2.740  29.835  1.00 39.10      A    C  \\nATOM   3341  OE1 GLU A 408      -2.900  -2.597  29.457  1.00 37.94      A    O  \\nATOM   3342  OE2 GLU A 408      -1.257  -2.213  30.877  1.00 36.32      A    O1-\\nATOM   3343  N   LYS A 409      -1.613  -5.271  24.810  1.00 38.80      A    N  \\nATOM   3344  CA  LYS A 409      -2.340  -6.000  23.756  1.00 41.52      A    C  \\nATOM   3345  C   LYS A 409      -1.476  -7.026  23.072  1.00 41.25      A    C  \\nATOM   3346  O   LYS A 409      -1.979  -7.993  22.522  1.00 40.02      A    O  \\nATOM   3347  CB  LYS A 409      -2.908  -5.020  22.719  1.00 38.67      A    C  \\nATOM   3348  CG  LYS A 409      -3.997  -4.132  23.281  1.00 37.34      A    C  \\nATOM   3349  CD  LYS A 409      -4.536  -3.203  22.219  1.00 35.78      A    C  \\nATOM   3350  CE  LYS A 409      -5.617  -2.287  22.790  1.00 29.88      A    C  \\nATOM   3351  NZ  LYS A 409      -6.180  -1.390  21.790  1.00 30.07      A    N1+\\nATOM   3352  N   ARG A 410      -0.156  -6.828  23.116  1.00 42.16      A    N  \\nATOM   3353  CA  ARG A 410       0.744  -7.809  22.525  1.00 45.39      A    C  \\nATOM   3354  C   ARG A 410       0.824  -9.082  23.377  1.00 44.91      A    C  \\nATOM   3355  O   ARG A 410       0.999 -10.191  22.852  1.00 44.29      A    O  \\nATOM   3356  CB  ARG A 410       2.134  -7.209  22.349  1.00 42.64      A    C  \\nATOM   3357  CG  ARG A 410       2.187  -6.099  21.324  1.00 41.14      A    C  \\nATOM   3358  CD  ARG A 410       3.577  -5.564  21.199  1.00 39.46      A    C  \\nATOM   3359  NE  ARG A 410       3.682  -4.491  20.222  1.00 36.15      A    N  \\nATOM   3360  CZ  ARG A 410       4.798  -3.847  19.912  1.00 33.76      A    C  \\nATOM   3361  NH1 ARG A 410       5.942  -4.166  20.537  1.00 31.23      A    N1+\\nATOM   3362  NH2 ARG A 410       4.790  -2.905  19.007  1.00 28.55      A    N  \\nATOM   3363  N   VAL A 411       0.703  -8.904  24.675  1.00 42.10      A    N  \\nATOM   3364  CA  VAL A 411       0.698 -10.074  25.567  1.00 45.52      A    C  \\nATOM   3365  C   VAL A 411      -0.602 -10.851  25.408  1.00 45.82      A    C  \\nATOM   3366  O   VAL A 411      -0.597 -12.085  25.424  1.00 45.37      A    O  \\nATOM   3367  CB  VAL A 411       0.908  -9.619  27.020  1.00 43.64      A    C  \\nATOM   3368  CG1 VAL A 411       2.261  -8.932  27.150  1.00 44.95      A    C  \\nATOM   3369  CG2 VAL A 411       0.825 -10.795  27.993  1.00 43.34      A    C  \\nATOM   3370  N   LYS A 412      -1.701 -10.134  25.237  1.00 43.74      A    N  \\nATOM   3371  CA  LYS A 412      -2.979 -10.799  25.033  1.00 47.50      A    C  \\nATOM   3372  C   LYS A 412      -3.058 -11.512  23.688  1.00 47.11      A    C  \\nATOM   3373  O   LYS A 412      -3.718 -12.541  23.565  1.00 45.85      A    O  \\nATOM   3374  CB  LYS A 412      -4.121  -9.775  25.132  1.00 44.75      A    C  \\nATOM   3375  CG  LYS A 412      -4.280  -9.209  26.531  1.00 42.95      A    C  \\nATOM   3376  CD  LYS A 412      -5.409  -8.200  26.567  1.00 40.51      A    C  \\nATOM   3377  CE  LYS A 412      -5.577  -7.590  27.955  1.00 34.83      A    C  \\nATOM   3378  NZ  LYS A 412      -6.667  -6.602  27.997  1.00 34.28      A    N1+\\nATOM   3379  N   SER A 413      -2.338 -10.996  22.707  1.00 41.19      A    N  \\nATOM   3380  CA  SER A 413      -2.290 -11.655  21.427  1.00 44.20      A    C  \\nATOM   3381  C   SER A 413      -1.429 -12.901  21.458  1.00 43.97      A    C  \\nATOM   3382  O   SER A 413      -1.679 -13.873  20.746  1.00 43.50      A    O  \\nATOM   3383  CB  SER A 413      -1.809 -10.714  20.328  1.00 41.98      A    C  \\nATOM   3384  OG  SER A 413      -0.465 -10.332  20.580  1.00 40.29      A    O  \\nATOM   3385  N   LYS A 414      -0.390 -12.885  22.306  1.00 39.74      A    N  \\nATOM   3386  CA  LYS A 414       0.470 -14.063  22.442  1.00 42.84      A    C  \\nATOM   3387  C   LYS A 414      -0.222 -15.135  23.268  1.00 42.69      A    C  \\nATOM   3388  O   LYS A 414      -0.018 -16.341  23.041  1.00 41.71      A    O  \\nATOM   3389  CB  LYS A 414       1.785 -13.688  23.102  1.00 40.17      A    C  \\nATOM   3390  CG  LYS A 414       2.674 -12.848  22.229  1.00 38.89      A    C  \\nATOM   3391  CD  LYS A 414       3.953 -12.464  22.938  1.00 37.16      A    C  \\nATOM   3392  CE  LYS A 414       4.842 -11.605  22.089  1.00 31.05      A    C  \\nATOM   3393  NZ  LYS A 414       6.103 -11.263  22.778  1.00 31.04      A    N1+\\nATOM   3394  N   GLN A 415      -1.040 -14.731  24.235  1.00 45.59      A    N  \\nATOM   3395  CA  GLN A 415      -1.776 -15.720  25.025  1.00 47.76      A    C  \\nATOM   3396  C   GLN A 415      -2.885 -16.385  24.230  1.00 48.11      A    C  \\nATOM   3397  O   GLN A 415      -3.215 -17.546  24.482  1.00 47.53      A    O  \\nATOM   3398  CB  GLN A 415      -2.338 -15.052  26.276  1.00 45.81      A    C  \\nATOM   3399  CG  GLN A 415      -1.290 -14.605  27.264  1.00 44.75      A    C  \\nATOM   3400  CD  GLN A 415      -1.883 -13.935  28.480  1.00 42.66      A    C  \\nATOM   3401  NE2 GLN A 415      -1.045 -13.611  29.447  1.00 35.69      A    N  \\nATOM   3402  OE1 GLN A 415      -3.094 -13.719  28.556  1.00 40.02      A    O  \\nATOM   3403  N   ALA A 416      -3.475 -15.645  23.323  1.00 38.72      A    N  \\nATOM   3404  CA  ALA A 416      -4.515 -16.217  22.495  1.00 41.40      A    C  \\nATOM   3405  C   ALA A 416      -3.928 -17.121  21.414  1.00 42.17      A    C  \\nATOM   3406  O   ALA A 416      -4.556 -18.104  21.001  1.00 41.19      A    O  \\nATOM   3407  CB  ALA A 416      -5.362 -15.119  21.862  1.00 38.74      A    C  \\nATOM   3408  N   ASN A 417      -2.729 -16.809  20.964  1.00 39.75      A    N  \\nATOM   3409  CA  ASN A 417      -2.079 -17.586  19.924  1.00 42.84      A    C  \\nATOM   3410  C   ASN A 417      -1.203 -18.705  20.457  1.00 42.73      A    C  \\nATOM   3411  O   ASN A 417      -0.586 -19.433  19.681  1.00 41.05      A    O  \\nATOM   3412  CB  ASN A 417      -1.259 -16.672  19.025  1.00 39.88      A    C  \\nATOM   3413  CG  ASN A 417      -2.108 -15.731  18.193  1.00 38.77      A    C  \\nATOM   3414  ND2 ASN A 417      -1.727 -14.446  18.136  1.00 35.44      A    N  \\nATOM   3415  OD1 ASN A 417      -3.101 -16.143  17.604  1.00 33.94      A    O  \\nATOM   3416  N   SER A 418      -1.112 -18.814  21.762  1.00 43.90      A    N  \\nATOM   3417  CA  SER A 418      -0.269 -19.865  22.359  1.00 46.51      A    C  \\nATOM   3418  C   SER A 418      -1.033 -21.195  22.323  1.00 46.43      A    C  \\nATOM   3419  O   SER A 418      -1.819 -21.492  23.234  1.00 45.34      A    O  \\nATOM   3420  CB  SER A 418       0.129 -19.489  23.766  1.00 43.65      A    C  \\nATOM   3421  OG  SER A 418      -1.025 -19.363  24.579  1.00 40.18      A    O  \\nATOM   3422  N   GLN A 419      -0.785 -21.961  21.319  1.00 48.10      A    N  \\nATOM   3423  CA  GLN A 419      -1.452 -23.252  21.127  1.00 50.29      A    C  \\nATOM   3424  C   GLN A 419      -0.472 -24.406  20.944  1.00 49.99      A    C  \\nATOM   3425  O   GLN A 419      -0.832 -25.552  21.118  1.00 50.07      A    O  \\nATOM   3426  CB  GLN A 419      -2.418 -23.219  19.942  1.00 48.50      A    C  \\nATOM   3427  CG  GLN A 419      -3.594 -22.299  20.165  1.00 46.51      A    C  \\nATOM   3428  CD  GLN A 419      -4.527 -22.255  18.978  1.00 43.00      A    C  \\nATOM   3429  NE2 GLN A 419      -5.615 -21.511  19.090  1.00 37.47      A    N  \\nATOM   3430  OE1 GLN A 419      -4.272 -22.888  17.952  1.00 41.24      A    O  \\nATOM   3431  N   TYR A 420       0.760 -24.102  20.572  1.00 41.73      A    N  \\nATOM   3432  CA  TYR A 420       1.746 -25.127  20.341  1.00 45.40      A    C  \\nATOM   3433  C   TYR A 420       2.480 -25.452  21.654  1.00 45.41      A    C  \\nATOM   3434  O   TYR A 420       3.574 -24.957  21.905  1.00 43.77      A    O  \\nATOM   3435  CB  TYR A 420       2.738 -24.699  19.257  1.00 41.63      A    C  \\nATOM   3436  CG  TYR A 420       2.115 -24.469  17.879  1.00 40.53      A    C  \\nATOM   3437  CD1 TYR A 420       1.775 -23.190  17.475  1.00 38.98      A    C  \\nATOM   3438  CD2 TYR A 420       1.883 -25.530  17.036  1.00 38.43      A    C  \\nATOM   3439  CE1 TYR A 420       1.199 -22.974  16.233  1.00 33.42      A    C  \\nATOM   3440  CE2 TYR A 420       1.321 -25.339  15.778  1.00 33.23      A    C  \\nATOM   3441  CZ  TYR A 420       0.957 -24.061  15.407  1.00 31.96      A    C  \\nATOM   3442  OH  TYR A 420       0.384 -23.853  14.168  1.00 35.03      A    O  \\nATOM   3443  N   HIS A 421       1.855 -26.263  22.474  1.00 45.72      A    N  \\nATOM   3444  CA  HIS A 421       2.439 -26.632  23.759  1.00 47.60      A    C  \\nATOM   3445  C   HIS A 421       2.956 -28.071  23.783  1.00 47.41      A    C  \\nATOM   3446  O   HIS A 421       3.722 -28.422  24.668  1.00 46.34      A    O  \\nATOM   3447  CB  HIS A 421       1.426 -26.425  24.852  1.00 45.19      A    C  \\nATOM   3448  CG  HIS A 421       0.987 -25.022  25.043  1.00 44.83      A    C  \\nATOM   3449  CD2 HIS A 421      -0.151 -24.358  24.693  1.00 42.29      A    C  \\nATOM   3450  ND1 HIS A 421       1.799 -24.084  25.652  1.00 40.78      A    N  \\nATOM   3451  CE1 HIS A 421       1.160 -22.921  25.689  1.00 39.39      A    C  \\nATOM   3452  NE2 HIS A 421      -0.017 -23.066  25.120  1.00 36.19      A    N  \\nATOM   3453  N   GLU A 422       2.506 -28.858  22.828  1.00 45.70      A    N  \\nATOM   3454  CA  GLU A 422       2.926 -30.254  22.775  1.00 48.09      A    C  \\nATOM   3455  C   GLU A 422       4.352 -30.428  22.264  1.00 48.54      A    C  \\nATOM   3456  O   GLU A 422       5.032 -31.378  22.649  1.00 47.34      A    O  \\nATOM   3457  CB  GLU A 422       1.948 -31.058  21.930  1.00 45.27      A    C  \\nATOM   3458  CG  GLU A 422       0.543 -31.127  22.503  1.00 44.47      A    C  \\nATOM   3459  CD  GLU A 422      -0.425 -31.900  21.638  1.00 41.36      A    C  \\nATOM   3460  OE1 GLU A 422      -0.032 -32.353  20.530  1.00 37.10      A    O  \\nATOM   3461  OE2 GLU A 422      -1.608 -32.085  22.060  1.00 33.79      A    O1-\\nATOM   3462  N   ASP A 423       4.773 -29.503  21.430  1.00 47.11      A    N  \\nATOM   3463  CA  ASP A 423       6.104 -29.598  20.845  1.00 50.27      A    C  \\nATOM   3464  C   ASP A 423       7.149 -28.814  21.609  1.00 50.79      A    C  \\nATOM   3465  O   ASP A 423       8.258 -28.595  21.090  1.00 48.89      A    O  \\nATOM   3466  CB  ASP A 423       6.083 -29.110  19.377  1.00 46.58      A    C  \\nATOM   3467  CG  ASP A 423       5.276 -30.048  18.478  1.00 44.85      A    C  \\nATOM   3468  OD1 ASP A 423       5.286 -31.261  18.749  1.00 39.98      A    O  \\nATOM   3469  OD2 ASP A 423       4.661 -29.545  17.505  1.00 40.28      A    O1-\\nATOM   3470  N   GLU A 424       6.816 -28.382  22.808  1.00 48.80      A    N  \\nATOM   3471  CA  GLU A 424       7.785 -27.629  23.592  1.00 52.28      A    C  \\nATOM   3472  C   GLU A 424       8.912 -28.505  24.115  1.00 52.49      A    C  \\nATOM   3473  O   GLU A 424      10.069 -28.083  24.134  1.00 51.45      A    O  \\nATOM   3474  CB  GLU A 424       7.085 -26.926  24.742  1.00 49.53      A    C  \\nATOM   3475  CG  GLU A 424       6.111 -25.818  24.326  1.00 47.89      A    C  \\nATOM   3476  CD  GLU A 424       5.472 -25.141  25.514  1.00 43.70      A    C  \\nATOM   3477  OE1 GLU A 424       5.726 -25.568  26.652  1.00 41.72      A    O  \\nATOM   3478  OE2 GLU A 424       4.713 -24.161  25.304  1.00 39.07      A    O1-\\nATOM   3479  N   ASP A 425       8.561 -29.723  24.508  1.00 51.66      A    N  \\nATOM   3480  CA  ASP A 425       9.593 -30.635  25.009  1.00 53.87      A    C  \\nATOM   3481  C   ASP A 425      10.508 -31.107  23.887  1.00 54.54      A    C  \\nATOM   3482  O   ASP A 425      11.688 -31.311  24.111  1.00 53.46      A    O  \\nATOM   3483  CB  ASP A 425       8.919 -31.833  25.689  1.00 50.73      A    C  \\nATOM   3484  CG  ASP A 425       8.229 -31.440  26.983  1.00 48.48      A    C  \\nATOM   3485  OD1 ASP A 425       8.677 -30.486  27.640  1.00 43.66      A    O  \\nATOM   3486  OD2 ASP A 425       7.219 -32.106  27.324  1.00 44.03      A    O1-\\nATOM   3487  N   GLU A 426       9.938 -31.271  22.721  1.00 51.46      A    N  \\nATOM   3488  CA  GLU A 426      10.779 -31.667  21.576  1.00 54.49      A    C  \\nATOM   3489  C   GLU A 426      11.645 -30.527  21.101  1.00 55.21      A    C  \\nATOM   3490  O   GLU A 426      12.731 -30.747  20.608  1.00 54.57      A    O  \\nATOM   3491  CB  GLU A 426       9.883 -32.177  20.445  1.00 51.73      A    C  \\nATOM   3492  CG  GLU A 426       9.137 -33.457  20.774  1.00 48.91      A    C  \\nATOM   3493  CD  GLU A 426       8.288 -33.954  19.634  1.00 44.90      A    C  \\nATOM   3494  OE1 GLU A 426       8.211 -33.245  18.585  1.00 42.66      A    O  \\nATOM   3495  OE2 GLU A 426       7.691 -35.042  19.737  1.00 39.77      A    O1-\\nATOM   3496  N   ASP A 427      11.145 -29.310  21.265  1.00 55.59      A    N  \\nATOM   3497  CA  ASP A 427      11.971 -28.159  20.909  1.00 56.91      A    C  \\nATOM   3498  C   ASP A 427      13.079 -27.933  21.919  1.00 58.19      A    C  \\nATOM   3499  O   ASP A 427      14.145 -27.440  21.557  1.00 57.35      A    O  \\nATOM   3500  CB  ASP A 427      11.093 -26.911  20.791  1.00 53.48      A    C  \\nATOM   3501  CG  ASP A 427      10.186 -26.948  19.576  1.00 50.47      A    C  \\nATOM   3502  OD1 ASP A 427      10.551 -27.584  18.567  1.00 47.05      A    O  \\nATOM   3503  OD2 ASP A 427       9.095 -26.348  19.646  1.00 48.27      A    O1-\\nATOM   3504  N   GLU A 428      12.832 -28.304  23.169  1.00 57.52      A    N  \\nATOM   3505  CA  GLU A 428      13.877 -28.199  24.178  1.00 60.99      A    C  \\nATOM   3506  C   GLU A 428      14.955 -29.236  23.945  1.00 62.06      A    C  \\nATOM   3507  O   GLU A 428      16.133 -28.997  24.234  1.00 62.95      A    O  \\nATOM   3508  CB  GLU A 428      13.285 -28.328  25.565  1.00 59.49      A    C  \\nATOM   3509  CG  GLU A 428      12.455 -27.149  26.013  1.00 55.73      A    C  \\nATOM   3510  CD  GLU A 428      11.885 -27.322  27.401  1.00 49.79      A    C  \\nATOM   3511  OE1 GLU A 428      12.103 -28.396  28.008  1.00 47.17      A    O  \\nATOM   3512  OE2 GLU A 428      11.215 -26.371  27.897  1.00 43.94      A    O1-\\nATOM   3513  N   LYS A 429      14.561 -30.399  23.460  1.00 56.56      A    N  \\nATOM   3514  CA  LYS A 429      15.542 -31.440  23.169  1.00 58.23      A    C  \\nATOM   3515  C   LYS A 429      16.380 -31.072  21.945  1.00 58.59      A    C  \\nATOM   3516  O   LYS A 429      17.578 -31.393  21.904  1.00 57.82      A    O  \\nATOM   3517  CB  LYS A 429      14.854 -32.783  22.932  1.00 55.46      A    C  \\nATOM   3518  CG  LYS A 429      14.212 -33.354  24.193  1.00 52.15      A    C  \\nATOM   3519  CD  LYS A 429      13.557 -34.667  23.911  1.00 48.64      A    C  \\nATOM   3520  CE  LYS A 429      12.921 -35.247  25.148  1.00 43.54      A    C  \\nATOM   3521  NZ  LYS A 429      12.225 -36.527  24.865  1.00 41.60      A    N1+\\nATOM   3522  N   THR A 430      15.793 -30.398  21.001  1.00 58.21      A    N  \\nATOM   3523  CA  THR A 430      16.562 -29.951  19.840  1.00 59.03      A    C  \\nATOM   3524  C   THR A 430      17.463 -28.777  20.193  1.00 59.21      A    C  \\nATOM   3525  O   THR A 430      18.550 -28.671  19.646  1.00 58.76      A    O  \\nATOM   3526  CB  THR A 430      15.631 -29.592  18.689  1.00 56.57      A    C  \\nATOM   3527  CG2 THR A 430      14.897 -30.831  18.214  1.00 53.12      A    C  \\nATOM   3528  OG1 THR A 430      14.680 -28.621  19.124  1.00 56.01      A    O  \\nATOM   3529  N   TYR A 431      17.010 -27.928  21.099  1.00 57.70      A    N  \\nATOM   3530  CA  TYR A 431      17.847 -26.807  21.524  1.00 60.22      A    C  \\nATOM   3531  C   TYR A 431      19.005 -27.277  22.383  1.00 61.64      A    C  \\nATOM   3532  O   TYR A 431      20.087 -26.704  22.331  1.00 62.22      A    O  \\nATOM   3533  CB  TYR A 431      17.030 -25.759  22.261  1.00 58.51      A    C  \\nATOM   3534  CG  TYR A 431      16.046 -25.001  21.393  1.00 55.38      A    C  \\nATOM   3535  CD1 TYR A 431      16.187 -24.966  20.019  1.00 50.94      A    C  \\nATOM   3536  CD2 TYR A 431      14.990 -24.319  21.959  1.00 49.86      A    C  \\nATOM   3537  CE1 TYR A 431      15.315 -24.260  19.231  1.00 45.38      A    C  \\nATOM   3538  CE2 TYR A 431      14.097 -23.591  21.186  1.00 45.23      A    C  \\nATOM   3539  CZ  TYR A 431      14.263 -23.583  19.814  1.00 43.65      A    C  \\nATOM   3540  OH  TYR A 431      13.371 -22.878  19.034  1.00 43.16      A    O  \\nATOM   3541  N   LYS A 432      18.776 -28.332  23.189  1.00 56.82      A    N  \\nATOM   3542  CA  LYS A 432      19.885 -28.907  23.926  1.00 60.77      A    C  \\nATOM   3543  C   LYS A 432      20.881 -29.616  23.026  1.00 60.25      A    C  \\nATOM   3544  O   LYS A 432      22.069 -29.661  23.325  1.00 60.89      A    O  \\nATOM   3545  CB  LYS A 432      19.363 -29.874  24.982  1.00 60.00      A    C  \\nATOM   3546  CG  LYS A 432      18.656 -29.214  26.159  1.00 57.24      A    C  \\nATOM   3547  CD  LYS A 432      18.168 -30.236  27.142  1.00 53.53      A    C  \\nATOM   3548  CE  LYS A 432      17.482 -29.571  28.330  1.00 50.79      A    C  \\nATOM   3549  NZ  LYS A 432      16.966 -30.557  29.300  1.00 47.18      A    N1+\\nATOM   3550  N   PHE A 433      20.385 -30.152  21.954  1.00 57.83      A    N  \\nATOM   3551  CA  PHE A 433      21.287 -30.767  20.966  1.00 60.76      A    C  \\nATOM   3552  C   PHE A 433      22.122 -29.712  20.259  1.00 60.96      A    C  \\nATOM   3553  O   PHE A 433      23.293 -29.962  19.946  1.00 60.91      A    O  \\nATOM   3554  CB  PHE A 433      20.463 -31.569  19.964  1.00 59.16      A    C  \\nATOM   3555  CG  PHE A 433      21.321 -32.231  18.900  1.00 56.46      A    C  \\nATOM   3556  CD1 PHE A 433      22.050 -33.377  19.184  1.00 53.71      A    C  \\nATOM   3557  CD2 PHE A 433      21.360 -31.710  17.630  1.00 52.75      A    C  \\nATOM   3558  CE1 PHE A 433      22.810 -33.978  18.212  1.00 50.84      A    C  \\nATOM   3559  CE2 PHE A 433      22.130 -32.307  16.653  1.00 49.15      A    C  \\nATOM   3560  CZ  PHE A 433      22.846 -33.445  16.954  1.00 48.60      A    C  \\nATOM   3561  N   ILE A 434      21.553 -28.549  20.051  1.00 56.68      A    N  \\nATOM   3562  CA  ILE A 434      22.313 -27.447  19.452  1.00 58.08      A    C  \\nATOM   3563  C   ILE A 434      23.354 -26.939  20.444  1.00 58.76      A    C  \\nATOM   3564  O   ILE A 434      24.491 -26.643  20.053  1.00 57.08      A    O  \\nATOM   3565  CB  ILE A 434      21.354 -26.325  18.990  1.00 55.17      A    C  \\nATOM   3566  CG1 ILE A 434      20.489 -26.837  17.833  1.00 58.10      A    C  \\nATOM   3567  CG2 ILE A 434      22.119 -25.096  18.559  1.00 53.56      A    C  \\nATOM   3568  CD1 ILE A 434      19.360 -25.908  17.476  1.00 50.37      A    C  \\nATOM   3569  N   GLU A 435      22.984 -26.874  21.721  1.00 59.00      A    N  \\nATOM   3570  CA  GLU A 435      23.912 -26.374  22.716  1.00 61.26      A    C  \\nATOM   3571  C   GLU A 435      25.067 -27.339  22.968  1.00 61.29      A    C  \\nATOM   3572  O   GLU A 435      26.202 -26.922  23.170  1.00 62.29      A    O  \\nATOM   3573  CB  GLU A 435      23.174 -26.110  24.035  1.00 60.62      A    C  \\nATOM   3574  CG  GLU A 435      22.221 -24.944  24.000  1.00 57.28      A    C  \\nATOM   3575  CD  GLU A 435      21.502 -24.733  25.315  1.00 52.75      A    C  \\nATOM   3576  OE1 GLU A 435      21.653 -25.610  26.208  1.00 51.32      A    O  \\nATOM   3577  OE2 GLU A 435      20.793 -23.716  25.469  1.00 47.10      A    O1-\\nATOM   3578  N   HIS A 436      24.768 -28.644  22.916  1.00 58.63      A    N  \\nATOM   3579  CA  HIS A 436      25.853 -29.614  23.026  1.00 60.81      A    C  \\nATOM   3580  C   HIS A 436      26.705 -29.666  21.775  1.00 59.61      A    C  \\nATOM   3581  O   HIS A 436      27.884 -29.963  21.838  1.00 61.07      A    O  \\nATOM   3582  CB  HIS A 436      25.317 -31.008  23.352  1.00 61.09      A    C  \\nATOM   3583  CG  HIS A 436      24.753 -31.118  24.736  1.00 57.61      A    C  \\nATOM   3584  CD2 HIS A 436      23.474 -31.292  25.162  1.00 52.26      A    C  \\nATOM   3585  ND1 HIS A 436      25.521 -31.080  25.878  1.00 50.05      A    N  \\nATOM   3586  CE1 HIS A 436      24.765 -31.240  26.943  1.00 50.13      A    C  \\nATOM   3587  NE2 HIS A 436      23.502 -31.363  26.530  1.00 48.46      A    N  \\nATOM   3588  N   ALA A 437      26.083 -29.415  20.662  1.00 56.61      A    N  \\nATOM   3589  CA  ALA A 437      26.831 -29.408  19.405  1.00 58.47      A    C  \\nATOM   3590  C   ALA A 437      27.760 -28.197  19.298  1.00 59.39      A    C  \\nATOM   3591  O   ALA A 437      28.834 -28.281  18.716  1.00 58.37      A    O  \\nATOM   3592  CB  ALA A 437      25.883 -29.425  18.222  1.00 55.84      A    C  \\nATOM   3593  N   GLN A 438      27.323 -27.081  19.878  1.00 60.68      A    N  \\nATOM   3594  CA  GLN A 438      28.170 -25.891  19.900  1.00 62.17      A    C  \\nATOM   3595  C   GLN A 438      29.355 -26.054  20.849  1.00 62.42      A    C  \\nATOM   3596  O   GLN A 438      30.426 -25.534  20.606  1.00 62.78      A    O  \\nATOM   3597  CB  GLN A 438      27.370 -24.652  20.283  1.00 60.36      A    C  \\nATOM   3598  CG  GLN A 438      26.371 -24.202  19.239  1.00 56.77      A    C  \\nATOM   3599  CD  GLN A 438      25.558 -22.991  19.697  1.00 53.25      A    C  \\nATOM   3600  NE2 GLN A 438      24.732 -22.466  18.797  1.00 48.16      A    N  \\nATOM   3601  OE1 GLN A 438      25.682 -22.566  20.833  1.00 52.00      A    O  \\nATOM   3602  N   GLN A 439      29.125 -26.811  21.937  1.00 64.62      A    N  \\nATOM   3603  CA  GLN A 439      30.208 -27.057  22.858  1.00 66.24      A    C  \\nATOM   3604  C   GLN A 439      31.213 -28.061  22.318  1.00 65.15      A    C  \\nATOM   3605  O   GLN A 439      32.396 -27.991  22.621  1.00 66.00      A    O  \\nATOM   3606  CB  GLN A 439      29.665 -27.552  24.200  1.00 65.33      A    C  \\nATOM   3607  CG  GLN A 439      28.892 -26.519  24.991  1.00 61.12      A    C  \\nATOM   3608  CD  GLN A 439      28.326 -27.052  26.256  1.00 56.51      A    C  \\nATOM   3609  NE2 GLN A 439      27.644 -26.185  27.006  1.00 50.81      A    N  \\nATOM   3610  OE1 GLN A 439      28.485 -28.218  26.598  1.00 55.28      A    O  \\nATOM   3611  N   ASP A 440      30.740 -29.008  21.508  1.00 59.95      A    N  \\nATOM   3612  CA  ASP A 440      31.641 -29.967  20.913  1.00 61.43      A    C  \\nATOM   3613  C   ASP A 440      32.500 -29.323  19.817  1.00 60.15      A    C  \\nATOM   3614  O   ASP A 440      33.617 -29.772  19.588  1.00 59.21      A    O  \\nATOM   3615  CB  ASP A 440      30.857 -31.159  20.344  1.00 59.08      A    C  \\nATOM   3616  CG  ASP A 440      30.281 -32.036  21.427  1.00 56.16      A    C  \\nATOM   3617  OD1 ASP A 440      30.885 -32.112  22.513  1.00 50.27      A    O  \\nATOM   3618  OD2 ASP A 440      29.220 -32.648  21.188  1.00 52.91      A    O1-\\nATOM   3619  N   VAL A 441      31.957 -28.322  19.179  1.00 58.27      A    N  \\nATOM   3620  CA  VAL A 441      32.743 -27.586  18.186  1.00 61.03      A    C  \\nATOM   3621  C   VAL A 441      33.754 -26.668  18.871  1.00 61.22      A    C  \\nATOM   3622  O   VAL A 441      34.914 -26.594  18.432  1.00 60.39      A    O  \\nATOM   3623  CB  VAL A 441      31.818 -26.805  17.240  1.00 58.20      A    C  \\nATOM   3624  CG1 VAL A 441      30.960 -27.776  16.417  1.00 59.45      A    C  \\nATOM   3625  CG2 VAL A 441      32.608 -25.944  16.284  1.00 57.34      A    C  \\nATOM   3626  N   GLU A 442      33.341 -26.044  19.938  1.00 61.03      A    N  \\nATOM   3627  CA  GLU A 442      34.226 -25.155  20.640  1.00 63.39      A    C  \\nATOM   3628  C   GLU A 442      35.334 -25.898  21.392  1.00 63.33      A    C  \\nATOM   3629  O   GLU A 442      36.417 -25.364  21.596  1.00 62.21      A    O  \\nATOM   3630  CB  GLU A 442      33.445 -24.276  21.614  1.00 60.78      A    C  \\nATOM   3631  CG  GLU A 442      32.543 -23.260  20.951  1.00 56.79      A    C  \\nATOM   3632  CD  GLU A 442      31.760 -22.431  21.933  1.00 52.77      A    C  \\nATOM   3633  OE1 GLU A 442      31.863 -22.731  23.155  1.00 49.96      A    O  \\nATOM   3634  OE2 GLU A 442      31.057 -21.462  21.518  1.00 45.17      A    O1-\\nATOM   3635  N   ASP A 443      35.078 -27.159  21.769  1.00 63.30      A    N  \\nATOM   3636  CA  ASP A 443      36.087 -27.967  22.413  1.00 64.89      A    C  \\nATOM   3637  C   ASP A 443      37.105 -28.481  21.406  1.00 62.96      A    C  \\nATOM   3638  O   ASP A 443      38.260 -28.753  21.775  1.00 61.76      A    O  \\nATOM   3639  CB  ASP A 443      35.468 -29.131  23.162  1.00 61.89      A    C  \\nATOM   3640  CG  ASP A 443      34.688 -28.686  24.396  1.00 59.40      A    C  \\nATOM   3641  OD1 ASP A 443      35.060 -27.640  24.978  1.00 53.24      A    O  \\nATOM   3642  OD2 ASP A 443      33.737 -29.375  24.786  1.00 55.55      A    O1-\\nATOM   3643  N   GLU A 444      36.692 -28.635  20.173  1.00 45.56      A    N  \\nATOM   3644  CA  GLU A 444      37.632 -29.072  19.132  1.00 50.99      A    C  \\nATOM   3645  C   GLU A 444      38.441 -27.911  18.596  1.00 49.51      A    C  \\nATOM   3646  O   GLU A 444      39.521 -28.117  18.070  1.00 45.94      A    O  \\nATOM   3647  CB  GLU A 444      36.883 -29.749  17.989  1.00 43.61      A    C  \\nATOM   3648  CG  GLU A 444      36.250 -31.077  18.362  1.00 42.24      A    C  \\nATOM   3649  CD  GLU A 444      35.467 -31.687  17.221  1.00 42.15      A    C  \\nATOM   3650  OE1 GLU A 444      35.308 -31.036  16.169  1.00 46.33      A    O  \\nATOM   3651  OE2 GLU A 444      34.991 -32.848  17.370  1.00 40.09      A    O1-\\nTER   \\nEND\\n\",\"pdb\");\n\tviewer_1742372256039599.setStyle({\"chain\": \"A\"},{\"cartoon\": {\"color\": \"#33ff33\"}});\n\tviewer_1742372256039599.zoomTo();\nviewer_1742372256039599.render();\n});\n</script>",
            "text/html": [
              "<div id=\"3dmolviewer_1742372256039599\"  style=\"position: relative; width: 640px; height: 480px;\">\n",
              "        <p id=\"3dmolwarning_1742372256039599\" style=\"background-color:#ffcccc;color:black\">3Dmol.js failed to load for some reason.  Please check your browser console for error messages.<br></p>\n",
              "        </div>\n",
              "<script>\n",
              "\n",
              "var loadScriptAsync = function(uri){\n",
              "  return new Promise((resolve, reject) => {\n",
              "    //this is to ignore the existence of requirejs amd\n",
              "    var savedexports, savedmodule;\n",
              "    if (typeof exports !== 'undefined') savedexports = exports;\n",
              "    else exports = {}\n",
              "    if (typeof module !== 'undefined') savedmodule = module;\n",
              "    else module = {}\n",
              "\n",
              "    var tag = document.createElement('script');\n",
              "    tag.src = uri;\n",
              "    tag.async = true;\n",
              "    tag.onload = () => {\n",
              "        exports = savedexports;\n",
              "        module = savedmodule;\n",
              "        resolve();\n",
              "    };\n",
              "  var firstScriptTag = document.getElementsByTagName('script')[0];\n",
              "  firstScriptTag.parentNode.insertBefore(tag, firstScriptTag);\n",
              "});\n",
              "};\n",
              "\n",
              "if(typeof $3Dmolpromise === 'undefined') {\n",
              "$3Dmolpromise = null;\n",
              "  $3Dmolpromise = loadScriptAsync('https://3dmol.org/build/3Dmol.js');\n",
              "}\n",
              "\n",
              "var viewer_1742372256039599 = null;\n",
              "var warn = document.getElementById(\"3dmolwarning_1742372256039599\");\n",
              "if(warn) {\n",
              "    warn.parentNode.removeChild(warn);\n",
              "}\n",
              "$3Dmolpromise.then(function() {\n",
              "viewer_1742372256039599 = $3Dmol.createViewer(document.getElementById(\"3dmolviewer_1742372256039599\"),{backgroundColor:\"white\"});\n",
              "viewer_1742372256039599.zoomTo();\n",
              "\tviewer_1742372256039599.addModel(\"ATOM      1  N   MET A   1     -27.698 -28.468 -31.491  1.00 37.85      A    N  \\nATOM      2  CA  MET A   1     -26.538 -28.314 -30.616  1.00 43.90      A    C  \\nATOM      3  C   MET A   1     -25.381 -27.672 -31.350  1.00 43.61      A    C  \\nATOM      4  O   MET A   1     -24.600 -26.941 -30.736  1.00 41.03      A    O  \\nATOM      5  CB  MET A   1     -26.113 -29.677 -30.027  1.00 37.01      A    C  \\nATOM      6  CG  MET A   1     -27.117 -30.312 -29.076  1.00 37.68      A    C  \\nATOM      7  SD  MET A   1     -26.655 -31.905 -28.452  1.00 37.85      A    S  \\nATOM      8  CE  MET A   1     -25.334 -31.405 -27.350  1.00 31.57      A    C  \\nATOM      9  N   SER A   2     -25.273 -28.007 -32.614  1.00 54.59      A    N  \\nATOM     10  CA  SER A   2     -24.216 -27.414 -33.427  1.00 59.61      A    C  \\nATOM     11  C   SER A   2     -24.461 -25.938 -33.664  1.00 60.18      A    C  \\nATOM     12  O   SER A   2     -23.539 -25.156 -33.832  1.00 57.16      A    O  \\nATOM     13  CB  SER A   2     -24.072 -28.141 -34.756  1.00 54.61      A    C  \\nATOM     14  OG  SER A   2     -25.265 -28.009 -35.519  1.00 51.31      A    O  \\nATOM     15  N   ASN A   3     -25.742 -25.559 -33.696  1.00 55.15      A    N  \\nATOM     16  CA  ASN A   3     -26.063 -24.138 -33.866  1.00 59.47      A    C  \\nATOM     17  C   ASN A   3     -25.781 -23.356 -32.598  1.00 59.93      A    C  \\nATOM     18  O   ASN A   3     -25.398 -22.188 -32.664  1.00 58.24      A    O  \\nATOM     19  CB  ASN A   3     -27.533 -23.968 -34.289  1.00 55.02      A    C  \\nATOM     20  CG  ASN A   3     -27.791 -24.444 -35.689  1.00 52.56      A    C  \\nATOM     21  ND2 ASN A   3     -28.932 -25.089 -35.886  1.00 47.22      A    N  \\nATOM     22  OD1 ASN A   3     -27.008 -24.202 -36.580  1.00 44.78      A    O  \\nATOM     23  N   LYS A   4     -25.979 -23.988 -31.443  1.00 53.58      A    N  \\nATOM     24  CA  LYS A   4     -25.708 -23.296 -30.178  1.00 58.31      A    C  \\nATOM     25  C   LYS A   4     -24.236 -23.097 -29.961  1.00 59.36      A    C  \\nATOM     26  O   LYS A   4     -23.807 -22.081 -29.396  1.00 58.08      A    O  \\nATOM     27  CB  LYS A   4     -26.313 -24.058 -29.010  1.00 54.65      A    C  \\nATOM     28  CG  LYS A   4     -27.840 -24.071 -29.000  1.00 52.08      A    C  \\nATOM     29  CD  LYS A   4     -28.393 -24.854 -27.861  1.00 47.90      A    C  \\nATOM     30  CE  LYS A   4     -29.918 -24.866 -27.833  1.00 39.43      A    C  \\nATOM     31  NZ  LYS A   4     -30.452 -25.610 -26.668  1.00 34.86      A    N1+\\nATOM     32  N   VAL A   5     -23.440 -24.074 -30.358  1.00 57.03      A    N  \\nATOM     33  CA  VAL A   5     -21.988 -23.940 -30.272  1.00 61.05      A    C  \\nATOM     34  C   VAL A   5     -21.470 -22.890 -31.213  1.00 61.96      A    C  \\nATOM     35  O   VAL A   5     -20.566 -22.118 -30.887  1.00 60.33      A    O  \\nATOM     36  CB  VAL A   5     -21.309 -25.303 -30.482  1.00 57.55      A    C  \\nATOM     37  CG1 VAL A   5     -21.748 -26.273 -29.402  1.00 57.63      A    C  \\nATOM     38  CG2 VAL A   5     -19.807 -25.168 -30.493  1.00 56.03      A    C  \\nATOM     39  N   SER A   6     -22.064 -22.832 -32.386  1.00 61.71      A    N  \\nATOM     40  CA  SER A   6     -21.648 -21.796 -33.346  1.00 65.53      A    C  \\nATOM     41  C   SER A   6     -22.087 -20.407 -32.900  1.00 66.06      A    C  \\nATOM     42  O   SER A   6     -21.425 -19.425 -33.203  1.00 64.27      A    O  \\nATOM     43  CB  SER A   6     -22.243 -22.109 -34.725  1.00 62.12      A    C  \\nATOM     44  OG  SER A   6     -23.675 -22.003 -34.697  1.00 54.71      A    O  \\nATOM     45  N   ARG A   7     -23.182 -20.340 -32.162  1.00 60.92      A    N  \\nATOM     46  CA  ARG A   7     -23.619 -19.039 -31.639  1.00 65.26      A    C  \\nATOM     47  C   ARG A   7     -22.756 -18.571 -30.502  1.00 66.29      A    C  \\nATOM     48  O   ARG A   7     -22.514 -17.349 -30.337  1.00 65.30      A    O  \\nATOM     49  CB  ARG A   7     -25.085 -19.092 -31.196  1.00 61.65      A    C  \\nATOM     50  CG  ARG A   7     -26.072 -19.183 -32.366  1.00 57.08      A    C  \\nATOM     51  CD  ARG A   7     -27.502 -19.318 -31.907  1.00 52.65      A    C  \\nATOM     52  NE  ARG A   7     -28.444 -19.422 -32.998  1.00 47.03      A    N  \\nATOM     53  CZ  ARG A   7     -29.753 -19.554 -32.840  1.00 45.97      A    C  \\nATOM     54  NH1 ARG A   7     -30.288 -19.572 -31.649  1.00 38.79      A    N1+\\nATOM     55  NH2 ARG A   7     -30.534 -19.678 -33.905  1.00 36.64      A    N  \\nATOM     56  N   THR A   8     -22.302 -19.501 -29.657  1.00 58.96      A    N  \\nATOM     57  CA  THR A   8     -21.379 -19.110 -28.585  1.00 61.14      A    C  \\nATOM     58  C   THR A   8     -20.036 -18.711 -29.121  1.00 62.16      A    C  \\nATOM     59  O   THR A   8     -19.371 -17.842 -28.560  1.00 61.06      A    O  \\nATOM     60  CB  THR A   8     -21.242 -20.260 -27.593  1.00 57.01      A    C  \\nATOM     61  CG2 THR A   8     -22.583 -20.559 -26.920  1.00 53.46      A    C  \\nATOM     62  OG1 THR A   8     -20.769 -21.436 -28.258  1.00 56.02      A    O  \\nATOM     63  N   ASP A   9     -19.602 -19.346 -30.199  1.00 65.02      A    N  \\nATOM     64  CA  ASP A   9     -18.359 -18.945 -30.841  1.00 69.20      A    C  \\nATOM     65  C   ASP A   9     -18.478 -17.597 -31.519  1.00 69.94      A    C  \\nATOM     66  O   ASP A   9     -17.516 -16.832 -31.565  1.00 69.21      A    O  \\nATOM     67  CB  ASP A   9     -17.886 -20.016 -31.839  1.00 66.86      A    C  \\nATOM     68  CG  ASP A   9     -17.472 -21.299 -31.149  1.00 62.83      A    C  \\nATOM     69  OD1 ASP A   9     -17.004 -21.220 -30.003  1.00 55.10      A    O  \\nATOM     70  OD2 ASP A   9     -17.604 -22.371 -31.777  1.00 56.63      A    O1-\\nATOM     71  N   SER A  10     -19.643 -17.311 -32.035  1.00 66.15      A    N  \\nATOM     72  CA  SER A  10     -19.871 -15.998 -32.620  1.00 68.97      A    C  \\nATOM     73  C   SER A  10     -19.962 -14.912 -31.544  1.00 69.07      A    C  \\nATOM     74  O   SER A  10     -19.538 -13.770 -31.771  1.00 68.69      A    O  \\nATOM     75  CB  SER A  10     -21.133 -15.999 -33.475  1.00 66.85      A    C  \\nATOM     76  OG  SER A  10     -22.283 -16.229 -32.663  1.00 57.90      A    O  \\nATOM     77  N   LEU A  11     -20.495 -15.271 -30.385  1.00 65.25      A    N  \\nATOM     78  CA  LEU A  11     -20.533 -14.318 -29.283  1.00 67.80      A    C  \\nATOM     79  C   LEU A  11     -19.159 -14.079 -28.687  1.00 68.57      A    C  \\nATOM     80  O   LEU A  11     -18.862 -12.999 -28.177  1.00 67.49      A    O  \\nATOM     81  CB  LEU A  11     -21.474 -14.819 -28.198  1.00 63.85      A    C  \\nATOM     82  CG  LEU A  11     -22.970 -14.756 -28.519  1.00 59.63      A    C  \\nATOM     83  CD1 LEU A  11     -23.436 -13.331 -28.658  1.00 60.06      A    C  \\nATOM     84  CD2 LEU A  11     -23.766 -15.467 -27.432  1.00 58.55      A    C  \\nATOM     85  N   LYS A  12     -18.288 -15.100 -28.727  1.00 65.13      A    N  \\nATOM     86  CA  LYS A  12     -16.927 -14.923 -28.277  1.00 66.72      A    C  \\nATOM     87  C   LYS A  12     -16.123 -14.024 -29.213  1.00 66.74      A    C  \\nATOM     88  O   LYS A  12     -15.278 -13.245 -28.773  1.00 66.72      A    O  \\nATOM     89  CB  LYS A  12     -16.221 -16.256 -28.129  1.00 65.04      A    C  \\nATOM     90  CG  LYS A  12     -16.689 -17.077 -26.932  1.00 60.90      A    C  \\nATOM     91  CD  LYS A  12     -16.019 -18.405 -26.851  1.00 58.20      A    C  \\nATOM     92  CE  LYS A  12     -16.515 -19.217 -25.668  1.00 52.14      A    C  \\nATOM     93  NZ  LYS A  12     -15.844 -20.524 -25.563  1.00 46.35      A    N1+\\nATOM     94  N   LYS A  13     -16.440 -14.093 -30.469  1.00 62.63      A    N  \\nATOM     95  CA  LYS A  13     -15.786 -13.216 -31.444  1.00 64.54      A    C  \\nATOM     96  C   LYS A  13     -16.295 -11.789 -31.361  1.00 63.85      A    C  \\nATOM     97  O   LYS A  13     -15.537 -10.837 -31.617  1.00 63.33      A    O  \\nATOM     98  CB  LYS A  13     -15.943 -13.757 -32.860  1.00 62.52      A    C  \\nATOM     99  CG  LYS A  13     -15.166 -15.039 -33.143  1.00 57.87      A    C  \\nATOM    100  CD  LYS A  13     -15.407 -15.520 -34.553  1.00 54.50      A    C  \\nATOM    101  CE  LYS A  13     -14.695 -16.788 -34.842  1.00 48.48      A    C  \\nATOM    102  NZ  LYS A  13     -14.879 -17.286 -36.194  1.00 43.57      A    N1+\\nATOM    103  N   THR A  14     -17.551 -11.640 -31.025  1.00 65.47      A    N  \\nATOM    104  CA  THR A  14     -18.099 -10.285 -30.860  1.00 65.69      A    C  \\nATOM    105  C   THR A  14     -17.641  -9.661 -29.532  1.00 64.98      A    C  \\nATOM    106  O   THR A  14     -17.450  -8.450 -29.482  1.00 64.14      A    O  \\nATOM    107  CB  THR A  14     -19.618 -10.292 -30.919  1.00 63.17      A    C  \\nATOM    108  CG2 THR A  14     -20.069 -10.794 -32.279  1.00 58.28      A    C  \\nATOM    109  OG1 THR A  14     -20.158 -11.138 -29.893  1.00 61.42      A    O  \\nATOM    110  N   LEU A  15     -17.459 -10.496 -28.518  1.00 59.61      A    N  \\nATOM    111  CA  LEU A  15     -16.943  -9.966 -27.266  1.00 61.72      A    C  \\nATOM    112  C   LEU A  15     -15.471  -9.600 -27.352  1.00 62.02      A    C  \\nATOM    113  O   LEU A  15     -15.035  -8.658 -26.717  1.00 61.67      A    O  \\nATOM    114  CB  LEU A  15     -17.169 -10.968 -26.133  1.00 59.47      A    C  \\nATOM    115  CG  LEU A  15     -18.601 -11.147 -25.625  1.00 55.97      A    C  \\nATOM    116  CD1 LEU A  15     -19.118  -9.875 -24.979  1.00 56.08      A    C  \\nATOM    117  CD2 LEU A  15     -18.691 -12.310 -24.661  1.00 55.18      A    C  \\nATOM    118  N   SER A  16     -14.742 -10.348 -28.128  1.00 65.39      A    N  \\nATOM    119  CA  SER A  16     -13.328 -10.050 -28.349  1.00 67.36      A    C  \\nATOM    120  C   SER A  16     -13.174  -8.761 -29.164  1.00 66.22      A    C  \\nATOM    121  O   SER A  16     -12.247  -7.960 -28.917  1.00 65.25      A    O  \\nATOM    122  CB  SER A  16     -12.591 -11.191 -29.028  1.00 65.27      A    C  \\nATOM    123  OG  SER A  16     -13.082 -11.393 -30.332  1.00 58.30      A    O  \\nATOM    124  N   ARG A  17     -14.038  -8.562 -30.098  1.00 60.22      A    N  \\nATOM    125  CA  ARG A  17     -14.012  -7.321 -30.900  1.00 63.48      A    C  \\nATOM    126  C   ARG A  17     -14.501  -6.104 -30.094  1.00 62.80      A    C  \\nATOM    127  O   ARG A  17     -13.988  -4.997 -30.292  1.00 61.27      A    O  \\nATOM    128  CB  ARG A  17     -14.868  -7.459 -32.160  1.00 60.92      A    C  \\nATOM    129  CG  ARG A  17     -14.315  -8.442 -33.188  1.00 57.06      A    C  \\nATOM    130  CD  ARG A  17     -15.243  -8.591 -34.374  1.00 53.33      A    C  \\nATOM    131  NE  ARG A  17     -14.731  -9.524 -35.353  1.00 47.78      A    N  \\nATOM    132  CZ  ARG A  17     -15.378  -9.901 -36.456  1.00 46.89      A    C  \\nATOM    133  NH1 ARG A  17     -16.575  -9.388 -36.716  1.00 40.05      A    N1+\\nATOM    134  NH2 ARG A  17     -14.821 -10.765 -37.285  1.00 38.60      A    N  \\nATOM    135  N   SER A  18     -15.435  -6.348 -29.192  1.00 54.68      A    N  \\nATOM    136  CA  SER A  18     -15.922  -5.246 -28.376  1.00 58.21      A    C  \\nATOM    137  C   SER A  18     -14.903  -4.869 -27.295  1.00 58.35      A    C  \\nATOM    138  O   SER A  18     -14.759  -3.697 -26.936  1.00 58.57      A    O  \\nATOM    139  CB  SER A  18     -17.265  -5.598 -27.729  1.00 55.43      A    C  \\nATOM    140  OG  SER A  18     -17.105  -6.662 -26.795  1.00 53.12      A    O  \\nATOM    141  N   LYS A  19     -14.171  -5.859 -26.771  1.00 54.50      A    N  \\nATOM    142  CA  LYS A  19     -13.099  -5.569 -25.816  1.00 56.57      A    C  \\nATOM    143  C   LYS A  19     -11.925  -4.848 -26.495  1.00 57.40      A    C  \\nATOM    144  O   LYS A  19     -11.285  -3.989 -25.882  1.00 57.19      A    O  \\nATOM    145  CB  LYS A  19     -12.588  -6.842 -25.175  1.00 54.13      A    C  \\nATOM    146  CG  LYS A  19     -13.531  -7.439 -24.152  1.00 51.30      A    C  \\nATOM    147  CD  LYS A  19     -12.980  -8.705 -23.594  1.00 47.99      A    C  \\nATOM    148  CE  LYS A  19     -13.938  -9.335 -22.588  1.00 43.17      A    C  \\nATOM    149  NZ  LYS A  19     -13.370 -10.562 -21.992  1.00 39.47      A    N1+\\nATOM    150  N   SER A  20     -11.643  -5.186 -27.726  1.00 56.66      A    N  \\nATOM    151  CA  SER A  20     -10.599  -4.467 -28.475  1.00 60.56      A    C  \\nATOM    152  C   SER A  20     -11.043  -3.061 -28.827  1.00 60.93      A    C  \\nATOM    153  O   SER A  20     -10.242  -2.150 -28.905  1.00 60.99      A    O  \\nATOM    154  CB  SER A  20     -10.242  -5.231 -29.729  1.00 58.78      A    C  \\nATOM    155  OG  SER A  20     -11.334  -5.289 -30.627  1.00 53.00      A    O  \\nATOM    156  N   SER A  21     -12.350  -2.876 -29.013  1.00 56.39      A    N  \\nATOM    157  CA  SER A  21     -12.846  -1.530 -29.301  1.00 60.18      A    C  \\nATOM    158  C   SER A  21     -12.861  -0.651 -28.042  1.00 60.44      A    C  \\nATOM    159  O   SER A  21     -12.642   0.548 -28.106  1.00 60.14      A    O  \\nATOM    160  CB  SER A  21     -14.250  -1.610 -29.910  1.00 58.28      A    C  \\nATOM    161  OG  SER A  21     -15.168  -2.108 -28.966  1.00 51.91      A    O  \\nATOM    162  N   LEU A  22     -13.120  -1.282 -26.899  1.00 57.13      A    N  \\nATOM    163  CA  LEU A  22     -13.090  -0.536 -25.635  1.00 60.02      A    C  \\nATOM    164  C   LEU A  22     -11.673  -0.156 -25.240  1.00 61.19      A    C  \\nATOM    165  O   LEU A  22     -11.443   0.905 -24.667  1.00 61.22      A    O  \\nATOM    166  CB  LEU A  22     -13.759  -1.334 -24.519  1.00 56.77      A    C  \\nATOM    167  CG  LEU A  22     -15.261  -1.492 -24.581  1.00 53.85      A    C  \\nATOM    168  CD1 LEU A  22     -15.936  -0.127 -24.470  1.00 53.60      A    C  \\nATOM    169  CD2 LEU A  22     -15.740  -2.418 -23.485  1.00 53.01      A    C  \\nATOM    170  N   THR A  23     -10.743  -1.045 -25.523  1.00 57.36      A    N  \\nATOM    171  CA  THR A  23      -9.328  -0.711 -25.272  1.00 60.66      A    C  \\nATOM    172  C   THR A  23      -8.827   0.384 -26.208  1.00 61.17      A    C  \\nATOM    173  O   THR A  23      -8.012   1.193 -25.823  1.00 61.07      A    O  \\nATOM    174  CB  THR A  23      -8.433  -1.940 -25.382  1.00 58.52      A    C  \\nATOM    175  CG2 THR A  23      -8.818  -2.960 -24.320  1.00 55.10      A    C  \\nATOM    176  OG1 THR A  23      -8.564  -2.536 -26.680  1.00 57.50      A    O  \\nATOM    177  N   ASN A  24      -9.345   0.389 -27.415  1.00 62.80      A    N  \\nATOM    178  CA  ASN A  24      -8.970   1.445 -28.350  1.00 63.27      A    C  \\nATOM    179  C   ASN A  24      -9.598   2.782 -27.973  1.00 63.00      A    C  \\nATOM    180  O   ASN A  24      -9.004   3.826 -28.164  1.00 63.35      A    O  \\nATOM    181  CB  ASN A  24      -9.389   1.060 -29.772  1.00 61.61      A    C  \\nATOM    182  CG  ASN A  24      -8.549  -0.063 -30.338  1.00 59.29      A    C  \\nATOM    183  ND2 ASN A  24      -9.197  -0.965 -31.048  1.00 51.62      A    N  \\nATOM    184  OD1 ASN A  24      -7.347  -0.111 -30.114  1.00 51.44      A    O  \\nATOM    185  N   LEU A  25     -10.813   2.721 -27.417  1.00 59.28      A    N  \\nATOM    186  CA  LEU A  25     -11.445   3.955 -26.946  1.00 61.39      A    C  \\nATOM    187  C   LEU A  25     -10.787   4.471 -25.677  1.00 62.03      A    C  \\nATOM    188  O   LEU A  25     -10.748   5.674 -25.424  1.00 61.73      A    O  \\nATOM    189  CB  LEU A  25     -12.915   3.723 -26.709  1.00 58.92      A    C  \\nATOM    190  CG  LEU A  25     -13.792   3.511 -27.939  1.00 55.14      A    C  \\nATOM    191  CD1 LEU A  25     -13.771   4.747 -28.830  1.00 55.42      A    C  \\nATOM    192  CD2 LEU A  25     -15.204   3.167 -27.536  1.00 54.56      A    C  \\nATOM    193  N   PHE A  26     -10.279   3.543 -24.861  1.00 53.08      A    N  \\nATOM    194  CA  PHE A  26      -9.603   3.937 -23.661  1.00 56.50      A    C  \\nATOM    195  C   PHE A  26      -8.226   4.543 -23.979  1.00 57.05      A    C  \\nATOM    196  O   PHE A  26      -7.760   5.452 -23.271  1.00 57.51      A    O  \\nATOM    197  CB  PHE A  26      -9.487   2.758 -22.711  1.00 53.93      A    C  \\nATOM    198  CG  PHE A  26      -8.852   3.071 -21.388  1.00 51.33      A    C  \\nATOM    199  CD1 PHE A  26      -9.525   3.775 -20.430  1.00 48.13      A    C  \\nATOM    200  CD2 PHE A  26      -7.565   2.645 -21.124  1.00 47.08      A    C  \\nATOM    201  CE1 PHE A  26      -8.955   4.074 -19.221  1.00 43.65      A    C  \\nATOM    202  CE2 PHE A  26      -6.988   2.927 -19.888  1.00 41.71      A    C  \\nATOM    203  CZ  PHE A  26      -7.675   3.651 -18.954  1.00 42.22      A    C  \\nATOM    204  N   LYS A  27      -7.588   4.046 -25.022  1.00 60.81      A    N  \\nATOM    205  CA  LYS A  27      -6.315   4.635 -25.461  1.00 63.46      A    C  \\nATOM    206  C   LYS A  27      -6.531   5.999 -26.102  1.00 63.22      A    C  \\nATOM    207  O   LYS A  27      -5.687   6.886 -25.963  1.00 62.78      A    O  \\nATOM    208  CB  LYS A  27      -5.610   3.699 -26.421  1.00 61.70      A    C  \\nATOM    209  CG  LYS A  27      -5.064   2.437 -25.791  1.00 59.74      A    C  \\nATOM    210  CD  LYS A  27      -4.427   1.544 -26.826  1.00 56.01      A    C  \\nATOM    211  CE  LYS A  27      -3.883   0.294 -26.199  1.00 52.11      A    C  \\nATOM    212  NZ  LYS A  27      -3.238  -0.609 -27.194  1.00 45.89      A    N1+\\nATOM    213  N   LYS A  28      -7.639   6.170 -26.781  1.00 56.47      A    N  \\nATOM    214  CA  LYS A  28      -7.963   7.481 -27.336  1.00 59.14      A    C  \\nATOM    215  C   LYS A  28      -8.265   8.502 -26.276  1.00 58.37      A    C  \\nATOM    216  O   LYS A  28      -7.980   9.693 -26.439  1.00 57.58      A    O  \\nATOM    217  CB  LYS A  28      -9.142   7.377 -28.310  1.00 56.81      A    C  \\nATOM    218  CG  LYS A  28      -8.836   6.666 -29.607  1.00 53.49      A    C  \\nATOM    219  CD  LYS A  28     -10.053   6.560 -30.481  1.00 50.79      A    C  \\nATOM    220  CE  LYS A  28      -9.733   5.821 -31.766  1.00 44.60      A    C  \\nATOM    221  NZ  LYS A  28     -10.920   5.705 -32.623  1.00 40.56      A    N1+\\nATOM    222  N   ASN A  29      -8.833   8.026 -25.162  1.00 53.60      A    N  \\nATOM    223  CA  ASN A  29      -9.089   8.951 -24.056  1.00 55.73      A    C  \\nATOM    224  C   ASN A  29      -7.805   9.347 -23.344  1.00 55.81      A    C  \\nATOM    225  O   ASN A  29      -7.670  10.459 -22.854  1.00 55.66      A    O  \\nATOM    226  CB  ASN A  29     -10.061   8.321 -23.058  1.00 52.64      A    C  \\nATOM    227  CG  ASN A  29     -11.469   8.202 -23.614  1.00 51.01      A    C  \\nATOM    228  ND2 ASN A  29     -12.097   7.085 -23.311  1.00 45.64      A    N  \\nATOM    229  OD1 ASN A  29     -11.962   9.072 -24.303  1.00 43.88      A    O  \\nATOM    230  N   LYS A  30      -6.855   8.430 -23.311  1.00 55.01      A    N  \\nATOM    231  CA  LYS A  30      -5.555   8.763 -22.743  1.00 57.35      A    C  \\nATOM    232  C   LYS A  30      -4.743   9.689 -23.633  1.00 56.95      A    C  \\nATOM    233  O   LYS A  30      -3.960  10.518 -23.143  1.00 56.26      A    O  \\nATOM    234  CB  LYS A  30      -4.759   7.493 -22.453  1.00 54.81      A    C  \\nATOM    235  CG  LYS A  30      -5.304   6.676 -21.308  1.00 51.51      A    C  \\nATOM    236  CD  LYS A  30      -4.441   5.464 -21.062  1.00 48.54      A    C  \\nATOM    237  CE  LYS A  30      -4.934   4.611 -19.897  1.00 42.06      A    C  \\nATOM    238  NZ  LYS A  30      -4.057   3.460 -19.617  1.00 39.20      A    N1+\\nATOM    239  N   ARG A  31      -4.933   9.592 -24.921  1.00 57.98      A    N  \\nATOM    240  CA  ARG A  31      -4.260  10.482 -25.843  1.00 60.57      A    C  \\nATOM    241  C   ARG A  31      -4.841  11.900 -25.791  1.00 60.12      A    C  \\nATOM    242  O   ARG A  31      -4.122  12.872 -25.975  1.00 58.98      A    O  \\nATOM    243  CB  ARG A  31      -4.356   9.941 -27.279  1.00 57.44      A    C  \\nATOM    244  CG  ARG A  31      -3.575   8.689 -27.546  1.00 54.47      A    C  \\nATOM    245  CD  ARG A  31      -3.782   8.207 -28.975  1.00 50.81      A    C  \\nATOM    246  NE  ARG A  31      -3.063   6.961 -29.213  1.00 44.43      A    N  \\nATOM    247  CZ  ARG A  31      -3.151   6.265 -30.359  1.00 43.97      A    C  \\nATOM    248  NH1 ARG A  31      -3.890   6.718 -31.343  1.00 36.29      A    N1+\\nATOM    249  NH2 ARG A  31      -2.489   5.136 -30.499  1.00 34.48      A    N  \\nATOM    250  N   LYS A  32      -6.151  11.970 -25.515  1.00 54.18      A    N  \\nATOM    251  CA  LYS A  32      -6.777  13.298 -25.395  1.00 56.12      A    C  \\nATOM    252  C   LYS A  32      -6.354  14.027 -24.122  1.00 55.12      A    C  \\nATOM    253  O   LYS A  32      -6.195  15.239 -24.142  1.00 55.10      A    O  \\nATOM    254  CB  LYS A  32      -8.295  13.163 -25.449  1.00 54.02      A    C  \\nATOM    255  CG  LYS A  32      -8.833  12.751 -26.813  1.00 52.32      A    C  \\nATOM    256  CD  LYS A  32     -10.341  12.582 -26.794  1.00 49.32      A    C  \\nATOM    257  CE  LYS A  32     -10.845  12.203 -28.154  1.00 44.14      A    C  \\nATOM    258  NZ  LYS A  32     -12.311  12.015 -28.185  1.00 40.04      A    N1+\\nATOM    259  N   HIS A  33      -6.176  13.262 -23.052  1.00 51.71      A    N  \\nATOM    260  CA  HIS A  33      -5.706  13.862 -21.825  1.00 53.98      A    C  \\nATOM    261  C   HIS A  33      -4.243  14.247 -21.906  1.00 52.97      A    C  \\nATOM    262  O   HIS A  33      -3.784  15.158 -21.238  1.00 52.35      A    O  \\nATOM    263  CB  HIS A  33      -5.951  12.964 -20.624  1.00 51.18      A    C  \\nATOM    264  CG  HIS A  33      -7.385  12.814 -20.247  1.00 50.14      A    C  \\nATOM    265  CD2 HIS A  33      -8.244  11.751 -20.330  1.00 46.85      A    C  \\nATOM    266  ND1 HIS A  33      -8.120  13.862 -19.720  1.00 43.45      A    N  \\nATOM    267  CE1 HIS A  33      -9.344  13.440 -19.471  1.00 43.10      A    C  \\nATOM    268  NE2 HIS A  33      -9.453  12.180 -19.818  1.00 40.39      A    N  \\nATOM    269  N   ALA A  34      -3.490  13.524 -22.715  1.00 51.12      A    N  \\nATOM    270  CA  ALA A  34      -2.072  13.824 -22.893  1.00 54.84      A    C  \\nATOM    271  C   ALA A  34      -1.863  15.063 -23.750  1.00 54.40      A    C  \\nATOM    272  O   ALA A  34      -0.959  15.846 -23.505  1.00 53.71      A    O  \\nATOM    273  CB  ALA A  34      -1.349  12.619 -23.503  1.00 52.74      A    C  \\nATOM    274  N   ARG A  35      -2.710  15.236 -24.751  1.00 49.90      A    N  \\nATOM    275  CA  ARG A  35      -2.607  16.406 -25.612  1.00 54.04      A    C  \\nATOM    276  C   ARG A  35      -3.064  17.694 -24.915  1.00 52.99      A    C  \\nATOM    277  O   ARG A  35      -2.541  18.756 -25.226  1.00 50.66      A    O  \\nATOM    278  CB  ARG A  35      -3.409  16.198 -26.887  1.00 50.29      A    C  \\nATOM    279  CG  ARG A  35      -2.874  15.159 -27.813  1.00 48.90      A    C  \\nATOM    280  CD  ARG A  35      -3.781  14.904 -29.007  1.00 43.94      A    C  \\nATOM    281  NE  ARG A  35      -3.279  13.885 -29.891  1.00 36.74      A    N  \\nATOM    282  CZ  ARG A  35      -3.906  13.478 -30.992  1.00 36.62      A    C  \\nATOM    283  NH1 ARG A  35      -5.060  14.022 -31.353  1.00 31.46      A    N1+\\nATOM    284  NH2 ARG A  35      -3.362  12.528 -31.758  1.00 29.02      A    N  \\nATOM    285  N   ASP A  36      -3.980  17.557 -23.998  1.00 46.65      A    N  \\nATOM    286  CA  ASP A  36      -4.424  18.736 -23.255  1.00 49.47      A    C  \\nATOM    287  C   ASP A  36      -3.387  19.172 -22.219  1.00 49.26      A    C  \\nATOM    288  O   ASP A  36      -3.234  20.343 -21.942  1.00 47.02      A    O  \\nATOM    289  CB  ASP A  36      -5.744  18.459 -22.570  1.00 46.18      A    C  \\nATOM    290  CG  ASP A  36      -6.907  18.324 -23.555  1.00 44.97      A    C  \\nATOM    291  OD1 ASP A  36      -6.819  18.898 -24.661  1.00 38.68      A    O  \\nATOM    292  OD2 ASP A  36      -7.890  17.635 -23.210  1.00 39.62      A    O1-\\nATOM    293  N   ASP A  37      -2.671  18.201 -21.656  1.00 46.49      A    N  \\nATOM    294  CA  ASP A  37      -1.644  18.530 -20.678  1.00 49.31      A    C  \\nATOM    295  C   ASP A  37      -0.361  19.062 -21.343  1.00 49.22      A    C  \\nATOM    296  O   ASP A  37       0.418  19.741 -20.696  1.00 47.13      A    O  \\nATOM    297  CB  ASP A  37      -1.313  17.337 -19.800  1.00 45.92      A    C  \\nATOM    298  CG  ASP A  37      -2.463  16.952 -18.877  1.00 44.38      A    C  \\nATOM    299  OD1 ASP A  37      -3.218  17.839 -18.447  1.00 37.22      A    O  \\nATOM    300  OD2 ASP A  37      -2.604  15.728 -18.580  1.00 36.99      A    O1-\\nATOM    301  N   ASP A  38      -0.169  18.741 -22.583  1.00 45.57      A    N  \\nATOM    302  CA  ASP A  38       1.015  19.202 -23.307  1.00 47.82      A    C  \\nATOM    303  C   ASP A  38       0.842  20.610 -23.861  1.00 47.90      A    C  \\nATOM    304  O   ASP A  38       1.813  21.218 -24.283  1.00 45.63      A    O  \\nATOM    305  CB  ASP A  38       1.373  18.233 -24.436  1.00 44.16      A    C  \\nATOM    306  CG  ASP A  38       1.850  16.885 -23.925  1.00 43.87      A    C  \\nATOM    307  OD1 ASP A  38       2.450  16.845 -22.852  1.00 36.34      A    O  \\nATOM    308  OD2 ASP A  38       1.596  15.868 -24.628  1.00 36.28      A    O1-\\nATOM    309  N   GLU A  39      -0.386  21.074 -23.892  1.00 47.71      A    N  \\nATOM    310  CA  GLU A  39      -0.644  22.415 -24.386  1.00 50.73      A    C  \\nATOM    311  C   GLU A  39      -0.261  23.448 -23.341  1.00 50.51      A    C  \\nATOM    312  O   GLU A  39      -0.711  23.421 -22.201  1.00 48.28      A    O  \\nATOM    313  CB  GLU A  39      -2.101  22.601 -24.770  1.00 47.56      A    C  \\nATOM    314  CG  GLU A  39      -2.526  21.768 -25.976  1.00 46.71      A    C  \\nATOM    315  CD  GLU A  39      -3.998  21.949 -26.324  1.00 43.31      A    C  \\nATOM    316  OE1 GLU A  39      -4.694  22.641 -25.556  1.00 39.43      A    O  \\nATOM    317  OE2 GLU A  39      -4.441  21.383 -27.348  1.00 36.98      A    O1-\\nATOM    318  N   ASP A  40       0.611  24.370 -23.741  1.00 46.59      A    N  \\nATOM    319  CA  ASP A  40       1.081  25.414 -22.861  1.00 47.74      A    C  \\nATOM    320  C   ASP A  40       0.245  26.660 -23.078  1.00 47.37      A    C  \\nATOM    321  O   ASP A  40       0.047  27.119 -24.197  1.00 46.52      A    O  \\nATOM    322  CB  ASP A  40       2.571  25.712 -23.095  1.00 44.64      A    C  \\nATOM    323  CG  ASP A  40       3.463  24.554 -22.696  1.00 43.33      A    C  \\nATOM    324  OD1 ASP A  40       3.076  23.775 -21.776  1.00 37.62      A    O  \\nATOM    325  OD2 ASP A  40       4.570  24.400 -23.287  1.00 37.54      A    O1-\\nATOM    326  N   ASP A  41      -0.213  27.207 -21.979  1.00 45.41      A    N  \\nATOM    327  CA  ASP A  41      -1.025  28.424 -22.011  1.00 47.31      A    C  \\nATOM    328  C   ASP A  41      -0.096  29.616 -21.764  1.00 47.10      A    C  \\nATOM    329  O   ASP A  41       0.506  29.744 -20.704  1.00 45.85      A    O  \\nATOM    330  CB  ASP A  41      -2.141  28.390 -20.999  1.00 44.01      A    C  \\nATOM    331  CG  ASP A  41      -3.203  27.352 -21.348  1.00 42.98      A    C  \\nATOM    332  OD1 ASP A  41      -3.375  27.068 -22.553  1.00 38.02      A    O  \\nATOM    333  OD2 ASP A  41      -3.853  26.838 -20.425  1.00 37.73      A    O1-\\nATOM    334  N   GLU A  42      -0.008  30.472 -22.743  1.00 40.91      A    N  \\nATOM    335  CA  GLU A  42       0.855  31.624 -22.670  1.00 43.44      A    C  \\nATOM    336  C   GLU A  42       0.088  32.771 -22.037  1.00 42.98      A    C  \\nATOM    337  O   GLU A  42      -0.971  33.190 -22.516  1.00 41.44      A    O  \\nATOM    338  CB  GLU A  42       1.397  32.039 -24.025  1.00 40.30      A    C  \\nATOM    339  CG  GLU A  42       2.334  31.020 -24.644  1.00 39.40      A    C  \\nATOM    340  CD  GLU A  42       2.880  31.459 -25.999  1.00 37.72      A    C  \\nATOM    341  OE1 GLU A  42       2.474  32.542 -26.486  1.00 34.74      A    O  \\nATOM    342  OE2 GLU A  42       3.723  30.755 -26.577  1.00 32.40      A    O1-\\nATOM    343  N   ASP A  43       0.629  33.262 -20.941  1.00 47.07      A    N  \\nATOM    344  CA  ASP A  43       0.028  34.389 -20.233  1.00 48.99      A    C  \\nATOM    345  C   ASP A  43       0.579  35.712 -20.775  1.00 48.10      A    C  \\nATOM    346  O   ASP A  43       1.315  35.754 -21.771  1.00 46.87      A    O  \\nATOM    347  CB  ASP A  43       0.279  34.258 -18.727  1.00 46.07      A    C  \\nATOM    348  CG  ASP A  43      -0.480  33.107 -18.103  1.00 44.83      A    C  \\nATOM    349  OD1 ASP A  43      -1.574  32.769 -18.607  1.00 39.17      A    O  \\nATOM    350  OD2 ASP A  43       0.005  32.548 -17.125  1.00 39.74      A    O1-\\nATOM    351  N   GLU A  44       0.185  36.772 -20.125  1.00 45.47      A    N  \\nATOM    352  CA  GLU A  44       0.656  38.090 -20.548  1.00 48.32      A    C  \\nATOM    353  C   GLU A  44       2.142  38.295 -20.256  1.00 47.79      A    C  \\nATOM    354  O   GLU A  44       2.839  39.015 -20.974  1.00 45.69      A    O  \\nATOM    355  CB  GLU A  44      -0.158  39.223 -19.892  1.00 44.96      A    C  \\nATOM    356  CG  GLU A  44      -1.612  39.249 -20.322  1.00 44.05      A    C  \\nATOM    357  CD  GLU A  44      -2.393  40.373 -19.620  1.00 42.26      A    C  \\nATOM    358  OE1 GLU A  44      -1.805  41.029 -18.732  1.00 38.80      A    O  \\nATOM    359  OE2 GLU A  44      -3.582  40.547 -19.969  1.00 36.49      A    O1-\\nATOM    360  N   ASN A  45       2.607  37.670 -19.199  1.00 45.81      A    N  \\nATOM    361  CA  ASN A  45       3.983  37.796 -18.795  1.00 47.51      A    C  \\nATOM    362  C   ASN A  45       4.874  36.719 -19.407  1.00 46.77      A    C  \\nATOM    363  O   ASN A  45       6.055  36.698 -19.140  1.00 45.46      A    O  \\nATOM    364  CB  ASN A  45       4.113  37.783 -17.279  1.00 44.53      A    C  \\nATOM    365  CG  ASN A  45       3.475  38.989 -16.636  1.00 43.03      A    C  \\nATOM    366  ND2 ASN A  45       2.810  38.782 -15.519  1.00 40.10      A    N  \\nATOM    367  OD1 ASN A  45       3.567  40.090 -17.158  1.00 37.33      A    O  \\nATOM    368  N   GLY A  46       4.278  35.816 -20.186  1.00 42.44      A    N  \\nATOM    369  CA  GLY A  46       5.045  34.746 -20.787  1.00 43.17      A    C  \\nATOM    370  C   GLY A  46       5.302  33.576 -19.843  1.00 44.40      A    C  \\nATOM    371  O   GLY A  46       6.242  32.791 -20.068  1.00 43.72      A    O  \\nATOM    372  N   ILE A  47       4.517  33.449 -18.793  1.00 44.58      A    N  \\nATOM    373  CA  ILE A  47       4.676  32.389 -17.813  1.00 47.13      A    C  \\nATOM    374  C   ILE A  47       3.914  31.186 -18.319  1.00 47.36      A    C  \\nATOM    375  O   ILE A  47       2.673  31.199 -18.411  1.00 46.29      A    O  \\nATOM    376  CB  ILE A  47       4.198  32.817 -16.425  1.00 44.74      A    C  \\nATOM    377  CG1 ILE A  47       5.022  33.988 -15.909  1.00 46.53      A    C  \\nATOM    378  CG2 ILE A  47       4.290  31.647 -15.455  1.00 43.63      A    C  \\nATOM    379  CD1 ILE A  47       4.530  34.606 -14.647  1.00 42.22      A    C  \\nATOM    380  N   SER A  48       4.633  30.128 -18.629  1.00 44.45      A    N  \\nATOM    381  CA  SER A  48       3.997  28.892 -19.094  1.00 48.79      A    C  \\nATOM    382  C   SER A  48       3.685  28.048 -17.881  1.00 48.22      A    C  \\nATOM    383  O   SER A  48       4.491  27.907 -16.955  1.00 47.46      A    O  \\nATOM    384  CB  SER A  48       4.891  28.146 -20.055  1.00 46.14      A    C  \\nATOM    385  OG  SER A  48       6.086  27.722 -19.420  1.00 43.11      A    O  \\nATOM    386  N   SER A  49       2.524  27.441 -17.878  1.00 44.55      A    N  \\nATOM    387  CA  SER A  49       2.071  26.582 -16.807  1.00 47.59      A    C  \\nATOM    388  C   SER A  49       2.127  25.143 -17.246  1.00 47.00      A    C  \\nATOM    389  O   SER A  49       1.558  24.757 -18.267  1.00 45.65      A    O  \\nATOM    390  CB  SER A  49       0.658  26.973 -16.357  1.00 44.76      A    C  \\nATOM    391  OG  SER A  49      -0.279  26.691 -17.396  1.00 40.63      A    O  \\nATOM    392  N   ASN A  50       2.820  24.345 -16.467  1.00 44.07      A    N  \\nATOM    393  CA  ASN A  50       2.923  22.931 -16.745  1.00 46.40      A    C  \\nATOM    394  C   ASN A  50       3.070  22.172 -15.433  1.00 46.30      A    C  \\nATOM    395  O   ASN A  50       4.018  22.392 -14.679  1.00 45.12      A    O  \\nATOM    396  CB  ASN A  50       4.108  22.613 -17.675  1.00 43.40      A    C  \\nATOM    397  CG  ASN A  50       4.048  21.180 -18.212  1.00 41.99      A    C  \\nATOM    398  ND2 ASN A  50       5.034  20.800 -19.021  1.00 37.75      A    N  \\nATOM    399  OD1 ASN A  50       3.125  20.421 -17.903  1.00 36.23      A    O  \\nATOM    400  N   ASP A  51       2.124  21.308 -15.164  1.00 45.55      A    N  \\nATOM    401  CA  ASP A  51       2.151  20.546 -13.913  1.00 47.56      A    C  \\nATOM    402  C   ASP A  51       3.143  19.409 -13.952  1.00 47.88      A    C  \\nATOM    403  O   ASP A  51       3.724  19.053 -12.932  1.00 46.90      A    O  \\nATOM    404  CB  ASP A  51       0.740  20.000 -13.618  1.00 43.81      A    C  \\nATOM    405  CG  ASP A  51      -0.249  21.101 -13.301  1.00 42.70      A    C  \\nATOM    406  OD1 ASP A  51       0.158  22.151 -12.757  1.00 37.73      A    O  \\nATOM    407  OD2 ASP A  51      -1.445  20.907 -13.633  1.00 37.69      A    O1-\\nATOM    408  N   LYS A  52       3.336  18.824 -15.130  1.00 43.86      A    N  \\nATOM    409  CA  LYS A  52       4.259  17.730 -15.244  1.00 46.14      A    C  \\nATOM    410  C   LYS A  52       5.714  18.180 -15.143  1.00 45.99      A    C  \\nATOM    411  O   LYS A  52       6.566  17.440 -14.647  1.00 45.25      A    O  \\nATOM    412  CB  LYS A  52       4.048  16.976 -16.568  1.00 43.40      A    C  \\nATOM    413  CG  LYS A  52       2.700  16.287 -16.665  1.00 42.81      A    C  \\nATOM    414  CD  LYS A  52       2.579  15.545 -17.972  1.00 40.30      A    C  \\nATOM    415  CE  LYS A  52       1.257  14.852 -18.099  1.00 35.00      A    C  \\nATOM    416  NZ  LYS A  52       1.141  14.129 -19.390  1.00 31.55      A    N1+\\nATOM    417  N   SER A  53       5.977  19.387 -15.604  1.00 45.28      A    N  \\nATOM    418  CA  SER A  53       7.329  19.928 -15.495  1.00 48.37      A    C  \\nATOM    419  C   SER A  53       7.651  20.359 -14.068  1.00 48.04      A    C  \\nATOM    420  O   SER A  53       8.814  20.307 -13.651  1.00 47.20      A    O  \\nATOM    421  CB  SER A  53       7.515  21.098 -16.440  1.00 45.82      A    C  \\nATOM    422  OG  SER A  53       6.682  22.163 -16.066  1.00 42.03      A    O  \\nATOM    423  N   LEU A  54       6.630  20.758 -13.315  1.00 44.05      A    N  \\nATOM    424  CA  LEU A  54       6.844  21.127 -11.927  1.00 47.06      A    C  \\nATOM    425  C   LEU A  54       7.105  19.921 -11.035  1.00 47.21      A    C  \\nATOM    426  O   LEU A  54       7.880  20.011 -10.085  1.00 46.14      A    O  \\nATOM    427  CB  LEU A  54       5.641  21.901 -11.388  1.00 44.08      A    C  \\nATOM    428  CG  LEU A  54       5.398  23.279 -11.994  1.00 42.46      A    C  \\nATOM    429  CD1 LEU A  54       6.550  24.206 -11.668  1.00 42.05      A    C  \\nATOM    430  CD2 LEU A  54       4.107  23.845 -11.491  1.00 41.49      A    C  \\nATOM    431  N   THR A  55       6.481  18.806 -11.373  1.00 45.15      A    N  \\nATOM    432  CA  THR A  55       6.722  17.598 -10.589  1.00 45.85      A    C  \\nATOM    433  C   THR A  55       8.118  17.011 -10.856  1.00 45.54      A    C  \\nATOM    434  O   THR A  55       8.719  16.435  -9.951  1.00 44.61      A    O  \\nATOM    435  CB  THR A  55       5.658  16.555 -10.885  1.00 42.76      A    C  \\nATOM    436  CG2 THR A  55       4.263  17.073 -10.522  1.00 39.92      A    C  \\nATOM    437  OG1 THR A  55       5.677  16.192 -12.247  1.00 41.52      A    O  \\nATOM    438  N   LYS A  56       8.588  17.145 -12.073  1.00 44.73      A    N  \\nATOM    439  CA  LYS A  56       9.931  16.685 -12.396  1.00 47.43      A    C  \\nATOM    440  C   LYS A  56      11.018  17.513 -11.741  1.00 46.72      A    C  \\nATOM    441  O   LYS A  56      12.068  16.996 -11.362  1.00 46.04      A    O  \\nATOM    442  CB  LYS A  56      10.142  16.657 -13.911  1.00 45.65      A    C  \\nATOM    443  CG  LYS A  56       9.318  15.609 -14.626  1.00 44.07      A    C  \\nATOM    444  CD  LYS A  56       9.561  15.657 -16.104  1.00 41.72      A    C  \\nATOM    445  CE  LYS A  56       8.742  14.608 -16.846  1.00 36.03      A    C  \\nATOM    446  NZ  LYS A  56       9.003  14.613 -18.306  1.00 34.01      A    N1+\\nATOM    447  N   ILE A  57      10.743  18.808 -11.602  1.00 39.53      A    N  \\nATOM    448  CA  ILE A  57      11.705  19.689 -10.921  1.00 41.97      A    C  \\nATOM    449  C   ILE A  57      11.720  19.398  -9.427  1.00 42.29      A    C  \\nATOM    450  O   ILE A  57      12.787  19.383  -8.810  1.00 41.09      A    O  \\nATOM    451  CB  ILE A  57      11.386  21.163 -11.227  1.00 39.09      A    C  \\nATOM    452  CG1 ILE A  57      11.601  21.452 -12.699  1.00 42.32      A    C  \\nATOM    453  CG2 ILE A  57      12.255  22.089 -10.389  1.00 39.71      A    C  \\nATOM    454  CD1 ILE A  57      11.113  22.801 -13.159  1.00 38.47      A    C  \\nATOM    455  N   ASP A  58      10.575  19.129  -8.867  1.00 40.54      A    N  \\nATOM    456  CA  ASP A  58      10.526  18.843  -7.435  1.00 43.26      A    C  \\nATOM    457  C   ASP A  58      11.162  17.508  -7.092  1.00 43.47      A    C  \\nATOM    458  O   ASP A  58      11.726  17.346  -6.017  1.00 41.78      A    O  \\nATOM    459  CB  ASP A  58       9.073  18.867  -6.945  1.00 40.03      A    C  \\nATOM    460  CG  ASP A  58       8.459  20.250  -6.965  1.00 38.96      A    C  \\nATOM    461  OD1 ASP A  58       9.201  21.228  -6.748  1.00 33.23      A    O  \\nATOM    462  OD2 ASP A  58       7.237  20.367  -7.178  1.00 32.13      A    O1-\\nATOM    463  N   ASN A  59      11.052  16.555  -8.001  1.00 44.95      A    N  \\nATOM    464  CA  ASN A  59      11.675  15.264  -7.776  1.00 47.05      A    C  \\nATOM    465  C   ASN A  59      13.192  15.355  -7.918  1.00 47.09      A    C  \\nATOM    466  O   ASN A  59      13.929  14.612  -7.264  1.00 45.06      A    O  \\nATOM    467  CB  ASN A  59      11.122  14.199  -8.716  1.00 43.24      A    C  \\nATOM    468  CG  ASN A  59       9.688  13.814  -8.405  1.00 42.67      A    C  \\nATOM    469  ND2 ASN A  59       8.885  13.579  -9.423  1.00 38.09      A    N  \\nATOM    470  OD1 ASN A  59       9.306  13.749  -7.234  1.00 36.98      A    O  \\nATOM    471  N   GLU A  60      13.666  16.242  -8.761  1.00 45.70      A    N  \\nATOM    472  CA  GLU A  60      15.086  16.447  -8.903  1.00 49.77      A    C  \\nATOM    473  C   GLU A  60      15.684  17.227  -7.736  1.00 49.36      A    C  \\nATOM    474  O   GLU A  60      16.852  17.076  -7.440  1.00 47.57      A    O  \\nATOM    475  CB  GLU A  60      15.396  17.144 -10.219  1.00 46.61      A    C  \\nATOM    476  CG  GLU A  60      15.113  16.296 -11.464  1.00 45.32      A    C  \\nATOM    477  CD  GLU A  60      15.382  17.044 -12.768  1.00 41.65      A    C  \\nATOM    478  OE1 GLU A  60      15.684  18.250 -12.718  1.00 38.82      A    O  \\nATOM    479  OE2 GLU A  60      15.277  16.402 -13.841  1.00 36.13      A    O1-\\nATOM    480  N   ASP A  61      14.861  18.054  -7.111  1.00 43.97      A    N  \\nATOM    481  CA  ASP A  61      15.355  18.806  -5.982  1.00 46.24      A    C  \\nATOM    482  C   ASP A  61      15.427  17.960  -4.705  1.00 46.34      A    C  \\nATOM    483  O   ASP A  61      16.274  18.221  -3.845  1.00 44.36      A    O  \\nATOM    484  CB  ASP A  61      14.475  20.025  -5.727  1.00 42.52      A    C  \\nATOM    485  CG  ASP A  61      14.619  21.077  -6.803  1.00 41.58      A    C  \\nATOM    486  OD1 ASP A  61      15.722  21.207  -7.367  1.00 35.92      A    O  \\nATOM    487  OD2 ASP A  61      13.624  21.786  -7.070  1.00 35.32      A    O1-\\nATOM    488  N   GLU A  62      14.556  16.980  -4.598  1.00 43.55      A    N  \\nATOM    489  CA  GLU A  62      14.563  16.114  -3.435  1.00 45.83      A    C  \\nATOM    490  C   GLU A  62      15.698  15.113  -3.480  1.00 45.66      A    C  \\nATOM    491  O   GLU A  62      16.126  14.603  -2.438  1.00 43.73      A    O  \\nATOM    492  CB  GLU A  62      13.234  15.392  -3.290  1.00 42.93      A    C  \\nATOM    493  CG  GLU A  62      12.062  16.281  -2.956  1.00 42.85      A    C  \\nATOM    494  CD  GLU A  62      10.751  15.551  -2.816  1.00 39.74      A    C  \\nATOM    495  OE1 GLU A  62      10.732  14.344  -3.137  1.00 35.68      A    O  \\nATOM    496  OE2 GLU A  62       9.750  16.180  -2.379  1.00 33.59      A    O1-\\nATOM    497  N   ASP A  63      16.197  14.807  -4.656  1.00 43.35      A    N  \\nATOM    498  CA  ASP A  63      17.291  13.858  -4.822  1.00 46.04      A    C  \\nATOM    499  C   ASP A  63      18.660  14.545  -4.733  1.00 45.97      A    C  \\nATOM    500  O   ASP A  63      19.665  13.844  -4.720  1.00 44.16      A    O  \\nATOM    501  CB  ASP A  63      17.153  13.124  -6.154  1.00 42.24      A    C  \\nATOM    502  CG  ASP A  63      15.978  12.173  -6.189  1.00 41.35      A    C  \\nATOM    503  OD1 ASP A  63      15.602  11.657  -5.105  1.00 37.05      A    O  \\nATOM    504  OD2 ASP A  63      15.425  11.929  -7.275  1.00 36.52      A    O1-\\nATOM    505  N   ASP A  64      18.661  15.844  -4.678  1.00 42.88      A    N  \\nATOM    506  CA  ASP A  64      19.931  16.549  -4.611  1.00 44.35      A    C  \\nATOM    507  C   ASP A  64      20.543  16.442  -3.227  1.00 44.62      A    C  \\nATOM    508  O   ASP A  64      19.844  16.472  -2.209  1.00 42.89      A    O  \\nATOM    509  CB  ASP A  64      19.751  18.021  -4.984  1.00 40.83      A    C  \\nATOM    510  CG  ASP A  64      19.442  18.224  -6.461  1.00 40.12      A    C  \\nATOM    511  OD1 ASP A  64      19.894  17.376  -7.290  1.00 35.63      A    O  \\nATOM    512  OD2 ASP A  64      18.744  19.203  -6.796  1.00 34.57      A    O1-\\nATOM    513  N   THR A  65      21.836  16.297  -3.214  1.00 46.92      A    N  \\nATOM    514  CA  THR A  65      22.554  16.185  -1.958  1.00 49.56      A    C  \\nATOM    515  C   THR A  65      22.859  17.546  -1.351  1.00 50.34      A    C  \\nATOM    516  O   THR A  65      22.979  17.658  -0.131  1.00 48.69      A    O  \\nATOM    517  CB  THR A  65      23.853  15.404  -2.189  1.00 45.75      A    C  \\nATOM    518  CG2 THR A  65      23.539  14.001  -2.707  1.00 44.35      A    C  \\nATOM    519  OG1 THR A  65      24.682  16.087  -3.120  1.00 46.27      A    O  \\nATOM    520  N   ASP A  66      22.996  18.544  -2.160  1.00 47.79      A    N  \\nATOM    521  CA  ASP A  66      23.314  19.892  -1.669  1.00 51.42      A    C  \\nATOM    522  C   ASP A  66      22.062  20.522  -1.100  1.00 52.41      A    C  \\nATOM    523  O   ASP A  66      21.036  20.667  -1.780  1.00 51.19      A    O  \\nATOM    524  CB  ASP A  66      23.884  20.764  -2.800  1.00 48.37      A    C  \\nATOM    525  CG  ASP A  66      25.253  20.278  -3.241  1.00 47.16      A    C  \\nATOM    526  OD1 ASP A  66      25.977  19.654  -2.426  1.00 40.28      A    O  \\nATOM    527  OD2 ASP A  66      25.597  20.497  -4.414  1.00 40.99      A    O1-\\nATOM    528  N   THR A  67      22.152  20.893   0.158  1.00 52.99      A    N  \\nATOM    529  CA  THR A  67      20.992  21.475   0.829  1.00 54.44      A    C  \\nATOM    530  C   THR A  67      20.832  22.966   0.510  1.00 54.84      A    C  \\nATOM    531  O   THR A  67      19.706  23.460   0.487  1.00 54.06      A    O  \\nATOM    532  CB  THR A  67      21.111  21.278   2.344  1.00 51.60      A    C  \\nATOM    533  CG2 THR A  67      21.162  19.789   2.653  1.00 48.14      A    C  \\nATOM    534  OG1 THR A  67      22.308  21.894   2.826  1.00 49.97      A    O  \\nATOM    535  N   SER A  68      21.927  23.654   0.276  1.00 54.08      A    N  \\nATOM    536  CA  SER A  68      21.835  25.089   0.018  1.00 57.41      A    C  \\nATOM    537  C   SER A  68      21.269  25.384  -1.354  1.00 57.50      A    C  \\nATOM    538  O   SER A  68      20.423  26.284  -1.517  1.00 55.74      A    O  \\nATOM    539  CB  SER A  68      23.214  25.745   0.177  1.00 53.95      A    C  \\nATOM    540  OG  SER A  68      24.111  25.284  -0.805  1.00 49.99      A    O  \\nATOM    541  N   ARG A  69      21.691  24.609  -2.330  1.00 57.23      A    N  \\nATOM    542  CA  ARG A  69      21.165  24.820  -3.702  1.00 59.30      A    C  \\nATOM    543  C   ARG A  69      19.726  24.373  -3.821  1.00 59.66      A    C  \\nATOM    544  O   ARG A  69      18.941  25.007  -4.543  1.00 58.97      A    O  \\nATOM    545  CB  ARG A  69      22.039  24.073  -4.718  1.00 56.46      A    C  \\nATOM    546  CG  ARG A  69      23.456  24.607  -4.814  1.00 53.52      A    C  \\nATOM    547  CD  ARG A  69      24.248  23.850  -5.828  1.00 49.90      A    C  \\nATOM    548  NE  ARG A  69      25.592  24.386  -5.945  1.00 45.58      A    N  \\nATOM    549  CZ  ARG A  69      26.550  23.883  -6.729  1.00 44.72      A    C  \\nATOM    550  NH1 ARG A  69      26.299  22.792  -7.463  1.00 38.83      A    N1+\\nATOM    551  NH2 ARG A  69      27.749  24.456  -6.779  1.00 36.22      A    N  \\nATOM    552  N   SER A  70      19.374  23.313  -3.105  1.00 54.93      A    N  \\nATOM    553  CA  SER A  70      18.003  22.855  -3.144  1.00 56.99      A    C  \\nATOM    554  C   SER A  70      17.070  23.810  -2.395  1.00 57.40      A    C  \\nATOM    555  O   SER A  70      15.887  23.907  -2.717  1.00 57.49      A    O  \\nATOM    556  CB  SER A  70      17.898  21.451  -2.581  1.00 54.91      A    C  \\nATOM    557  OG  SER A  70      18.235  21.439  -1.208  1.00 49.75      A    O  \\nATOM    558  N   HIS A  71      17.595  24.478  -1.376  1.00 55.95      A    N  \\nATOM    559  CA  HIS A  71      16.780  25.471  -0.672  1.00 57.03      A    C  \\nATOM    560  C   HIS A  71      16.515  26.706  -1.535  1.00 57.63      A    C  \\nATOM    561  O   HIS A  71      15.437  27.273  -1.460  1.00 58.13      A    O  \\nATOM    562  CB  HIS A  71      17.488  25.855   0.625  1.00 55.46      A    C  \\nATOM    563  CG  HIS A  71      16.740  26.866   1.431  1.00 52.61      A    C  \\nATOM    564  CD2 HIS A  71      16.995  28.180   1.663  1.00 47.89      A    C  \\nATOM    565  ND1 HIS A  71      15.571  26.583   2.108  1.00 46.38      A    N  \\nATOM    566  CE1 HIS A  71      15.144  27.668   2.718  1.00 44.27      A    C  \\nATOM    567  NE2 HIS A  71      15.992  28.652   2.478  1.00 41.79      A    N  \\nATOM    568  N   LYS A  72      17.491  27.065  -2.338  1.00 53.63      A    N  \\nATOM    569  CA  LYS A  72      17.309  28.224  -3.212  1.00 56.36      A    C  \\nATOM    570  C   LYS A  72      16.370  27.935  -4.369  1.00 56.27      A    C  \\nATOM    571  O   LYS A  72      15.602  28.805  -4.792  1.00 56.13      A    O  \\nATOM    572  CB  LYS A  72      18.650  28.724  -3.757  1.00 54.72      A    C  \\nATOM    573  CG  LYS A  72      19.576  29.304  -2.701  1.00 51.89      A    C  \\nATOM    574  CD  LYS A  72      20.882  29.758  -3.307  1.00 47.88      A    C  \\nATOM    575  CE  LYS A  72      21.816  30.335  -2.256  1.00 43.23      A    C  \\nATOM    576  NZ  LYS A  72      23.111  30.770  -2.827  1.00 39.42      A    N1+\\nATOM    577  N   ARG A  73      16.456  26.695  -4.884  1.00 55.15      A    N  \\nATOM    578  CA  ARG A  73      15.539  26.327  -5.979  1.00 56.50      A    C  \\nATOM    579  C   ARG A  73      14.106  26.206  -5.486  1.00 57.25      A    C  \\nATOM    580  O   ARG A  73      13.178  26.582  -6.197  1.00 57.23      A    O  \\nATOM    581  CB  ARG A  73      16.002  25.000  -6.588  1.00 53.75      A    C  \\nATOM    582  CG  ARG A  73      17.276  25.121  -7.376  1.00 50.32      A    C  \\nATOM    583  CD  ARG A  73      17.731  23.790  -7.934  1.00 46.61      A    C  \\nATOM    584  NE  ARG A  73      18.964  23.880  -8.667  1.00 41.70      A    N  \\nATOM    585  CZ  ARG A  73      19.597  22.839  -9.191  1.00 40.79      A    C  \\nATOM    586  NH1 ARG A  73      19.099  21.624  -9.078  1.00 34.94      A    N1+\\nATOM    587  NH2 ARG A  73      20.754  23.012  -9.839  1.00 32.52      A    N  \\nATOM    588  N   ARG A  74      13.937  25.683  -4.269  1.00 53.12      A    N  \\nATOM    589  CA  ARG A  74      12.578  25.589  -3.713  1.00 56.23      A    C  \\nATOM    590  C   ARG A  74      11.996  26.954  -3.366  1.00 56.92      A    C  \\nATOM    591  O   ARG A  74      10.792  27.135  -3.407  1.00 56.26      A    O  \\nATOM    592  CB  ARG A  74      12.581  24.687  -2.481  1.00 52.71      A    C  \\nATOM    593  CG  ARG A  74      12.813  23.216  -2.774  1.00 49.74      A    C  \\nATOM    594  CD  ARG A  74      12.902  22.402  -1.502  1.00 45.30      A    C  \\nATOM    595  NE  ARG A  74      13.179  20.988  -1.770  1.00 40.13      A    N  \\nATOM    596  CZ  ARG A  74      13.341  20.051  -0.824  1.00 38.74      A    C  \\nATOM    597  NH1 ARG A  74      13.239  20.381   0.455  1.00 33.39      A    N1+\\nATOM    598  NH2 ARG A  74      13.607  18.811  -1.166  1.00 31.35      A    N  \\nATOM    599  N   HIS A  75      12.869  27.893  -3.037  1.00 55.13      A    N  \\nATOM    600  CA  HIS A  75      12.407  29.232  -2.721  1.00 57.17      A    C  \\nATOM    601  C   HIS A  75      11.912  29.949  -3.986  1.00 57.87      A    C  \\nATOM    602  O   HIS A  75      10.935  30.689  -3.953  1.00 58.48      A    O  \\nATOM    603  CB  HIS A  75      13.502  30.047  -2.054  1.00 54.39      A    C  \\nATOM    604  CG  HIS A  75      13.079  31.432  -1.669  1.00 51.13      A    C  \\nATOM    605  CD2 HIS A  75      13.405  32.650  -2.187  1.00 47.76      A    C  \\nATOM    606  ND1 HIS A  75      12.214  31.696  -0.618  1.00 44.90      A    N  \\nATOM    607  CE1 HIS A  75      12.042  33.014  -0.542  1.00 43.81      A    C  \\nATOM    608  NE2 HIS A  75      12.751  33.602  -1.488  1.00 41.71      A    N  \\nATOM    609  N   VAL A  76      12.601  29.717  -5.103  1.00 54.41      A    N  \\nATOM    610  CA  VAL A  76      12.191  30.323  -6.368  1.00 57.12      A    C  \\nATOM    611  C   VAL A  76      10.937  29.634  -6.908  1.00 58.46      A    C  \\nATOM    612  O   VAL A  76      10.044  30.291  -7.446  1.00 58.91      A    O  \\nATOM    613  CB  VAL A  76      13.363  30.251  -7.396  1.00 55.05      A    C  \\nATOM    614  CG1 VAL A  76      14.523  31.083  -6.908  1.00 54.89      A    C  \\nATOM    615  CG2 VAL A  76      12.917  30.755  -8.769  1.00 53.41      A    C  \\nATOM    616  N   GLU A  77      10.887  28.308  -6.743  1.00 54.48      A    N  \\nATOM    617  CA  GLU A  77       9.699  27.571  -7.211  1.00 57.39      A    C  \\nATOM    618  C   GLU A  77       8.460  27.889  -6.391  1.00 58.20      A    C  \\nATOM    619  O   GLU A  77       7.374  27.994  -6.935  1.00 57.98      A    O  \\nATOM    620  CB  GLU A  77       9.971  26.076  -7.180  1.00 54.16      A    C  \\nATOM    621  CG  GLU A  77      10.969  25.624  -8.231  1.00 50.62      A    C  \\nATOM    622  CD  GLU A  77      11.268  24.120  -8.146  1.00 45.39      A    C  \\nATOM    623  OE1 GLU A  77      10.761  23.457  -7.219  1.00 43.08      A    O  \\nATOM    624  OE2 GLU A  77      12.023  23.621  -9.002  1.00 40.70      A    O1-\\nATOM    625  N   ASP A  78       8.651  28.078  -5.101  1.00 54.81      A    N  \\nATOM    626  CA  ASP A  78       7.505  28.452  -4.270  1.00 56.67      A    C  \\nATOM    627  C   ASP A  78       7.057  29.878  -4.522  1.00 57.27      A    C  \\nATOM    628  O   ASP A  78       5.873  30.190  -4.425  1.00 57.58      A    O  \\nATOM    629  CB  ASP A  78       7.835  28.257  -2.781  1.00 54.08      A    C  \\nATOM    630  CG  ASP A  78       7.910  26.797  -2.406  1.00 51.34      A    C  \\nATOM    631  OD1 ASP A  78       7.214  25.979  -3.016  1.00 46.99      A    O  \\nATOM    632  OD2 ASP A  78       8.680  26.448  -1.464  1.00 48.45      A    O1-\\nATOM    633  N   ALA A  79       7.999  30.748  -4.832  1.00 57.69      A    N  \\nATOM    634  CA  ALA A  79       7.640  32.128  -5.172  1.00 59.90      A    C  \\nATOM    635  C   ALA A  79       6.984  32.210  -6.536  1.00 60.64      A    C  \\nATOM    636  O   ALA A  79       6.110  33.041  -6.773  1.00 61.34      A    O  \\nATOM    637  CB  ALA A  79       8.856  33.011  -5.111  1.00 58.63      A    C  \\nATOM    638  N   ALA A  80       7.432  31.355  -7.457  1.00 56.51      A    N  \\nATOM    639  CA  ALA A  80       6.829  31.333  -8.782  1.00 58.82      A    C  \\nATOM    640  C   ALA A  80       5.447  30.671  -8.769  1.00 59.63      A    C  \\nATOM    641  O   ALA A  80       4.563  31.046  -9.537  1.00 60.34      A    O  \\nATOM    642  CB  ALA A  80       7.735  30.605  -9.771  1.00 57.24      A    C  \\nATOM    643  N   LYS A  81       5.278  29.678  -7.886  1.00 56.25      A    N  \\nATOM    644  CA  LYS A  81       3.966  29.050  -7.752  1.00 57.57      A    C  \\nATOM    645  C   LYS A  81       2.947  29.975  -7.098  1.00 57.29      A    C  \\nATOM    646  O   LYS A  81       1.776  29.950  -7.451  1.00 58.24      A    O  \\nATOM    647  CB  LYS A  81       4.068  27.745  -6.972  1.00 55.40      A    C  \\nATOM    648  CG  LYS A  81       4.782  26.624  -7.695  1.00 52.73      A    C  \\nATOM    649  CD  LYS A  81       4.971  25.426  -6.822  1.00 50.04      A    C  \\nATOM    650  CE  LYS A  81       5.745  24.323  -7.535  1.00 47.79      A    C  \\nATOM    651  NZ  LYS A  81       6.038  23.172  -6.668  1.00 43.12      A    N1+\\nATOM    652  N   GLU A  82       3.424  30.819  -6.205  1.00 51.77      A    N  \\nATOM    653  CA  GLU A  82       2.547  31.804  -5.608  1.00 54.45      A    C  \\nATOM    654  C   GLU A  82       2.150  32.891  -6.618  1.00 55.20      A    C  \\nATOM    655  O   GLU A  82       1.055  33.453  -6.542  1.00 56.06      A    O  \\nATOM    656  CB  GLU A  82       3.192  32.463  -4.397  1.00 52.42      A    C  \\nATOM    657  CG  GLU A  82       3.286  31.541  -3.188  1.00 49.58      A    C  \\nATOM    658  CD  GLU A  82       3.944  32.199  -2.005  1.00 45.47      A    C  \\nATOM    659  OE1 GLU A  82       4.434  33.369  -2.130  1.00 42.76      A    O  \\nATOM    660  OE2 GLU A  82       4.004  31.563  -0.924  1.00 40.65      A    O1-\\nATOM    661  N   LYS A  83       3.016  33.181  -7.561  1.00 53.67      A    N  \\nATOM    662  CA  LYS A  83       2.681  34.139  -8.599  1.00 54.98      A    C  \\nATOM    663  C   LYS A  83       1.696  33.558  -9.572  1.00 55.19      A    C  \\nATOM    664  O   LYS A  83       0.814  34.284 -10.078  1.00 55.81      A    O  \\nATOM    665  CB  LYS A  83       3.947  34.582  -9.337  1.00 52.55      A    C  \\nATOM    666  CG  LYS A  83       4.900  35.450  -8.499  1.00 48.99      A    C  \\nATOM    667  CD  LYS A  83       6.136  35.825  -9.277  1.00 45.52      A    C  \\nATOM    668  CE  LYS A  83       7.079  36.674  -8.467  1.00 40.03      A    C  \\nATOM    669  NZ  LYS A  83       8.265  37.080  -9.203  1.00 37.00      A    N1+\\nATOM    670  N   LEU A  84       1.808  32.260  -9.871  1.00 54.28      A    N  \\nATOM    671  CA  LEU A  84       0.847  31.622 -10.752  1.00 55.55      A    C  \\nATOM    672  C   LEU A  84      -0.524  31.498 -10.090  1.00 56.14      A    C  \\nATOM    673  O   LEU A  84      -1.562  31.591 -10.764  1.00 56.06      A    O  \\nATOM    674  CB  LEU A  84       1.342  30.238 -11.151  1.00 52.50      A    C  \\nATOM    675  CG  LEU A  84       2.570  30.177 -12.069  1.00 49.28      A    C  \\nATOM    676  CD1 LEU A  84       2.216  30.766 -13.421  1.00 49.96      A    C  \\nATOM    677  CD2 LEU A  84       3.053  28.757 -12.221  1.00 49.22      A    C  \\nATOM    678  N   ARG A  85      -0.525  31.294  -8.790  1.00 48.18      A    N  \\nATOM    679  CA  ARG A  85      -1.797  31.228  -8.070  1.00 49.39      A    C  \\nATOM    680  C   ARG A  85      -2.467  32.605  -8.001  1.00 50.15      A    C  \\nATOM    681  O   ARG A  85      -3.686  32.704  -7.987  1.00 49.28      A    O  \\nATOM    682  CB  ARG A  85      -1.577  30.636  -6.690  1.00 46.51      A    C  \\nATOM    683  CG  ARG A  85      -1.271  29.167  -6.665  1.00 43.88      A    C  \\nATOM    684  CD  ARG A  85      -1.073  28.662  -5.269  1.00 39.89      A    C  \\nATOM    685  NE  ARG A  85      -0.724  27.256  -5.217  1.00 36.76      A    N  \\nATOM    686  CZ  ARG A  85      -0.490  26.579  -4.104  1.00 36.15      A    C  \\nATOM    687  NH1 ARG A  85      -0.557  27.179  -2.941  1.00 32.28      A    N1+\\nATOM    688  NH2 ARG A  85      -0.173  25.295  -4.164  1.00 29.72      A    N  \\nATOM    689  N   LEU A  86      -1.680  33.634  -7.934  1.00 53.51      A    N  \\nATOM    690  CA  LEU A  86      -2.245  34.997  -7.909  1.00 55.19      A    C  \\nATOM    691  C   LEU A  86      -2.775  35.403  -9.293  1.00 55.79      A    C  \\nATOM    692  O   LEU A  86      -3.770  36.114  -9.392  1.00 56.35      A    O  \\nATOM    693  CB  LEU A  86      -1.210  35.995  -7.433  1.00 52.76      A    C  \\nATOM    694  CG  LEU A  86      -0.838  35.927  -5.950  1.00 48.93      A    C  \\nATOM    695  CD1 LEU A  86      -2.045  36.244  -5.082  1.00 47.95      A    C  \\nATOM    696  CD2 LEU A  86       0.290  36.882  -5.640  1.00 47.33      A    C  \\nATOM    697  N   ILE A  87      -2.083  34.956 -10.336  1.00 58.33      A    N  \\nATOM    698  CA  ILE A  87      -2.538  35.250 -11.700  1.00 58.41      A    C  \\nATOM    699  C   ILE A  87      -3.827  34.469 -11.999  1.00 58.31      A    C  \\nATOM    700  O   ILE A  87      -4.777  35.016 -12.578  1.00 58.11      A    O  \\nATOM    701  CB  ILE A  87      -1.439  34.937 -12.716  1.00 56.40      A    C  \\nATOM    702  CG1 ILE A  87      -0.273  35.922 -12.520  1.00 58.36      A    C  \\nATOM    703  CG2 ILE A  87      -1.938  35.024 -14.144  1.00 54.86      A    C  \\nATOM    704  CD1 ILE A  87       0.964  35.559 -13.285  1.00 53.19      A    C  \\nATOM    705  N   ALA A  88      -3.852  33.205 -11.620  1.00 50.24      A    N  \\nATOM    706  CA  ALA A  88      -5.034  32.383 -11.834  1.00 51.75      A    C  \\nATOM    707  C   ALA A  88      -6.201  32.835 -10.970  1.00 52.12      A    C  \\nATOM    708  O   ALA A  88      -7.357  32.676 -11.359  1.00 51.88      A    O  \\nATOM    709  CB  ALA A  88      -4.726  30.908 -11.580  1.00 49.09      A    C  \\nATOM    710  N   PHE A  89      -5.894  33.356  -9.806  1.00 48.02      A    N  \\nATOM    711  CA  PHE A  89      -6.946  33.904  -8.970  1.00 50.63      A    C  \\nATOM    712  C   PHE A  89      -7.492  35.213  -9.518  1.00 51.12      A    C  \\nATOM    713  O   PHE A  89      -8.699  35.465  -9.455  1.00 50.74      A    O  \\nATOM    714  CB  PHE A  89      -6.423  34.112  -7.532  1.00 48.33      A    C  \\nATOM    715  CG  PHE A  89      -7.469  34.614  -6.562  1.00 46.35      A    C  \\nATOM    716  CD1 PHE A  89      -8.452  33.748  -6.065  1.00 44.95      A    C  \\nATOM    717  CD2 PHE A  89      -7.477  35.924  -6.170  1.00 45.02      A    C  \\nATOM    718  CE1 PHE A  89      -9.402  34.198  -5.190  1.00 42.66      A    C  \\nATOM    719  CE2 PHE A  89      -8.411  36.382  -5.288  1.00 43.02      A    C  \\nATOM    720  CZ  PHE A  89      -9.380  35.530  -4.798  1.00 40.79      A    C  \\nATOM    721  N   GLN A  90      -6.631  36.046 -10.073  1.00 52.44      A    N  \\nATOM    722  CA  GLN A  90      -7.094  37.294 -10.660  1.00 53.99      A    C  \\nATOM    723  C   GLN A  90      -7.834  37.069 -11.974  1.00 54.22      A    C  \\nATOM    724  O   GLN A  90      -8.697  37.869 -12.330  1.00 54.33      A    O  \\nATOM    725  CB  GLN A  90      -5.914  38.232 -10.877  1.00 51.38      A    C  \\nATOM    726  CG  GLN A  90      -5.303  38.768  -9.597  1.00 49.74      A    C  \\nATOM    727  CD  GLN A  90      -4.080  39.633  -9.827  1.00 46.09      A    C  \\nATOM    728  NE2 GLN A  90      -3.492  40.148  -8.759  1.00 42.97      A    N  \\nATOM    729  OE1 GLN A  90      -3.665  39.842 -10.955  1.00 44.17      A    O  \\nATOM    730  N   LYS A  91      -7.489  35.994 -12.673  1.00 51.31      A    N  \\nATOM    731  CA  LYS A  91      -8.217  35.663 -13.900  1.00 52.74      A    C  \\nATOM    732  C   LYS A  91      -9.614  35.125 -13.613  1.00 53.12      A    C  \\nATOM    733  O   LYS A  91     -10.532  35.293 -14.408  1.00 51.97      A    O  \\nATOM    734  CB  LYS A  91      -7.445  34.694 -14.768  1.00 49.53      A    C  \\nATOM    735  CG  LYS A  91      -6.221  35.277 -15.423  1.00 47.99      A    C  \\nATOM    736  CD  LYS A  91      -5.501  34.252 -16.276  1.00 45.45      A    C  \\nATOM    737  CE  LYS A  91      -4.302  34.835 -16.948  1.00 42.17      A    C  \\nATOM    738  NZ  LYS A  91      -3.571  33.830 -17.781  1.00 38.07      A    N1+\\nATOM    739  N   ASN A  92      -9.773  34.462 -12.462  1.00 50.92      A    N  \\nATOM    740  CA  ASN A  92     -11.075  33.959 -12.091  1.00 51.60      A    C  \\nATOM    741  C   ASN A  92     -12.018  35.070 -11.657  1.00 51.57      A    C  \\nATOM    742  O   ASN A  92     -13.223  34.943 -11.771  1.00 50.26      A    O  \\nATOM    743  CB  ASN A  92     -10.942  32.923 -10.969  1.00 48.51      A    C  \\nATOM    744  CG  ASN A  92     -10.318  31.622 -11.420  1.00 46.83      A    C  \\nATOM    745  ND2 ASN A  92      -9.489  31.045 -10.588  1.00 46.27      A    N  \\nATOM    746  OD1 ASN A  92     -10.600  31.150 -12.520  1.00 44.39      A    O  \\nATOM    747  N   GLU A  93     -11.463  36.159 -11.160  1.00 49.52      A    N  \\nATOM    748  CA  GLU A  93     -12.276  37.315 -10.777  1.00 49.53      A    C  \\nATOM    749  C   GLU A  93     -12.720  38.115 -12.002  1.00 49.38      A    C  \\nATOM    750  O   GLU A  93     -13.786  38.730 -11.978  1.00 47.61      A    O  \\nATOM    751  CB  GLU A  93     -11.514  38.207  -9.809  1.00 45.46      A    C  \\nATOM    752  CG  GLU A  93     -11.245  37.579  -8.459  1.00 43.74      A    C  \\nATOM    753  CD  GLU A  93     -10.516  38.496  -7.517  1.00 40.38      A    C  \\nATOM    754  OE1 GLU A  93     -10.090  39.594  -7.961  1.00 38.45      A    O  \\nATOM    755  OE2 GLU A  93     -10.373  38.148  -6.317  1.00 37.28      A    O1-\\nATOM    756  N   LYS A  94     -11.896  38.132 -13.026  1.00 47.87      A    N  \\nATOM    757  CA  LYS A  94     -12.221  38.868 -14.250  1.00 49.27      A    C  \\nATOM    758  C   LYS A  94     -13.189  38.133 -15.156  1.00 48.98      A    C  \\nATOM    759  O   LYS A  94     -13.832  38.733 -16.019  1.00 47.69      A    O  \\nATOM    760  CB  LYS A  94     -10.957  39.209 -15.024  1.00 46.31      A    C  \\nATOM    761  CG  LYS A  94     -10.040  40.182 -14.317  1.00 45.00      A    C  \\nATOM    762  CD  LYS A  94      -8.813  40.491 -15.182  1.00 42.44      A    C  \\nATOM    763  CE  LYS A  94      -7.876  41.427 -14.458  1.00 38.00      A    C  \\nATOM    764  NZ  LYS A  94      -6.660  41.747 -15.261  1.00 34.15      A    N1+\\nATOM    765  N   LEU A  95     -13.282  36.798 -14.987  1.00 46.80      A    N  \\nATOM    766  CA  LEU A  95     -14.184  36.004 -15.796  1.00 47.87      A    C  \\nATOM    767  C   LEU A  95     -15.605  36.165 -15.312  1.00 48.08      A    C  \\nATOM    768  O   LEU A  95     -15.872  36.512 -14.153  1.00 47.19      A    O  \\nATOM    769  CB  LEU A  95     -13.788  34.530 -15.779  1.00 45.01      A    C  \\nATOM    770  CG  LEU A  95     -12.494  34.134 -16.478  1.00 44.00      A    C  \\nATOM    771  CD1 LEU A  95     -12.631  34.369 -17.971  1.00 42.56      A    C  \\nATOM    772  CD2 LEU A  95     -12.153  32.702 -16.179  1.00 42.03      A    C  \\nATOM    773  N   LEU A  96     -16.538  35.878 -16.200  1.00 49.24      A    N  \\nATOM    774  CA  LEU A  96     -17.959  35.950 -15.887  1.00 51.65      A    C  \\nATOM    775  C   LEU A  96     -18.470  34.598 -15.431  1.00 53.05      A    C  \\nATOM    776  O   LEU A  96     -17.732  33.601 -15.427  1.00 50.61      A    O  \\nATOM    777  CB  LEU A  96     -18.747  36.449 -17.098  1.00 46.01      A    C  \\nATOM    778  CG  LEU A  96     -18.437  37.874 -17.536  1.00 43.79      A    C  \\nATOM    779  CD1 LEU A  96     -18.787  38.866 -16.464  1.00 43.27      A    C  \\nATOM    780  CD2 LEU A  96     -19.196  38.199 -18.819  1.00 42.27      A    C  \\nATOM    781  N   ASP A  97     -19.704  34.549 -15.054  1.00 45.49      A    N  \\nATOM    782  CA  ASP A  97     -20.345  33.314 -14.601  1.00 46.75      A    C  \\nATOM    783  C   ASP A  97     -20.650  32.440 -15.788  1.00 47.94      A    C  \\nATOM    784  O   ASP A  97     -21.300  32.824 -16.725  1.00 46.23      A    O  \\nATOM    785  CB  ASP A  97     -21.603  33.620 -13.813  1.00 42.35      A    C  \\nATOM    786  CG  ASP A  97     -21.334  34.306 -12.486  1.00 41.88      A    C  \\nATOM    787  OD1 ASP A  97     -20.265  34.032 -11.881  1.00 37.36      A    O  \\nATOM    788  OD2 ASP A  97     -22.174  35.109 -12.054  1.00 35.97      A    O1-\\nATOM    789  N   GLY A  98     -20.150  31.228 -15.705  1.00 56.16      A    N  \\nATOM    790  CA  GLY A  98     -20.355  30.240 -16.766  1.00 57.28      A    C  \\nATOM    791  C   GLY A  98     -19.165  30.019 -17.678  1.00 60.56      A    C  \\nATOM    792  O   GLY A  98     -19.146  29.048 -18.429  1.00 59.19      A    O  \\nATOM    793  N   LYS A  99     -18.197  30.888 -17.598  1.00 62.20      A    N  \\nATOM    794  CA  LYS A  99     -17.010  30.779 -18.438  1.00 64.90      A    C  \\nATOM    795  C   LYS A  99     -15.868  30.157 -17.651  1.00 66.22      A    C  \\nATOM    796  O   LYS A  99     -15.800  30.219 -16.422  1.00 64.84      A    O  \\nATOM    797  CB  LYS A  99     -16.592  32.135 -19.006  1.00 61.00      A    C  \\nATOM    798  CG  LYS A  99     -17.582  32.706 -20.002  1.00 56.79      A    C  \\nATOM    799  CD  LYS A  99     -17.134  34.041 -20.533  1.00 52.17      A    C  \\nATOM    800  CE  LYS A  99     -18.111  34.607 -21.546  1.00 47.05      A    C  \\nATOM    801  NZ  LYS A  99     -17.727  35.941 -22.044  1.00 41.00      A    N1+\\nATOM    802  N   LEU A 100     -14.970  29.540 -18.385  1.00 69.28      A    N  \\nATOM    803  CA  LEU A 100     -13.839  28.854 -17.784  1.00 71.16      A    C  \\nATOM    804  C   LEU A 100     -12.578  29.172 -18.547  1.00 71.21      A    C  \\nATOM    805  O   LEU A 100     -12.600  29.255 -19.781  1.00 69.91      A    O  \\nATOM    806  CB  LEU A 100     -14.053  27.342 -17.744  1.00 68.26      A    C  \\nATOM    807  CG  LEU A 100     -15.144  26.805 -16.816  1.00 65.46      A    C  \\nATOM    808  CD1 LEU A 100     -14.821  27.124 -15.376  1.00 66.88      A    C  \\nATOM    809  CD2 LEU A 100     -15.316  25.295 -17.016  1.00 67.20      A    C  \\nATOM    810  N   PRO A 101     -11.446  29.338 -17.867  1.00 69.67      A    N  \\nATOM    811  CA  PRO A 101     -10.184  29.528 -18.573  1.00 71.25      A    C  \\nATOM    812  C   PRO A 101      -9.639  28.226 -19.108  1.00 72.12      A    C  \\nATOM    813  O   PRO A 101     -10.241  27.169 -18.933  1.00 70.80      A    O  \\nATOM    814  CB  PRO A 101      -9.278  30.141 -17.507  1.00 68.25      A    C  \\nATOM    815  CG  PRO A 101      -9.803  29.587 -16.219  1.00 67.27      A    C  \\nATOM    816  CD  PRO A 101     -11.283  29.518 -16.422  1.00 70.06      A    C  \\nATOM    817  N   LYS A 102      -8.492  28.324 -19.740  1.00 67.52      A    N  \\nATOM    818  CA  LYS A 102      -7.868  27.139 -20.325  1.00 68.15      A    C  \\nATOM    819  C   LYS A 102      -7.287  26.240 -19.242  1.00 69.24      A    C  \\nATOM    820  O   LYS A 102      -7.122  25.032 -19.463  1.00 69.17      A    O  \\nATOM    821  CB  LYS A 102      -6.781  27.513 -21.311  1.00 64.61      A    C  \\nATOM    822  CG  LYS A 102      -7.293  28.214 -22.579  1.00 60.18      A    C  \\nATOM    823  CD  LYS A 102      -6.164  28.526 -23.534  1.00 56.49      A    C  \\nATOM    824  CE  LYS A 102      -6.650  29.244 -24.774  1.00 51.08      A    C  \\nATOM    825  NZ  LYS A 102      -5.521  29.567 -25.692  1.00 43.94      A    N1+\\nATOM    826  N   GLU A 103      -6.958  26.813 -18.091  1.00 70.17      A    N  \\nATOM    827  CA  GLU A 103      -6.398  26.034 -17.018  1.00 70.51      A    C  \\nATOM    828  C   GLU A 103      -7.447  25.160 -16.334  1.00 72.04      A    C  \\nATOM    829  O   GLU A 103      -7.160  24.047 -15.919  1.00 70.82      A    O  \\nATOM    830  CB  GLU A 103      -5.714  26.942 -15.983  1.00 67.24      A    C  \\nATOM    831  CG  GLU A 103      -4.506  27.669 -16.547  1.00 62.69      A    C  \\nATOM    832  CD  GLU A 103      -3.850  28.578 -15.527  1.00 56.33      A    C  \\nATOM    833  OE1 GLU A 103      -4.353  28.658 -14.404  1.00 52.58      A    O  \\nATOM    834  OE2 GLU A 103      -2.778  29.194 -15.861  1.00 50.95      A    O1-\\nATOM    835  N   LEU A 104      -8.670  25.668 -16.253  1.00 72.31      A    N  \\nATOM    836  CA  LEU A 104      -9.743  24.896 -15.629  1.00 73.94      A    C  \\nATOM    837  C   LEU A 104     -10.303  23.861 -16.589  1.00 74.77      A    C  \\nATOM    838  O   LEU A 104     -11.017  22.950 -16.149  1.00 73.89      A    O  \\nATOM    839  CB  LEU A 104     -10.857  25.811 -15.144  1.00 71.56      A    C  \\nATOM    840  CG  LEU A 104     -10.506  26.750 -13.979  1.00 70.49      A    C  \\nATOM    841  CD1 LEU A 104     -10.084  25.944 -12.754  1.00 70.16      A    C  \\nATOM    842  CD2 LEU A 104     -11.658  27.678 -13.667  1.00 70.51      A    C  \\nATOM    843  N   ARG A 105     -10.052  23.980 -17.862  1.00 73.57      A    N  \\nATOM    844  CA  ARG A 105     -10.494  22.984 -18.832  1.00 74.21      A    C  \\nATOM    845  C   ARG A 105      -9.647  21.733 -18.770  1.00 74.57      A    C  \\nATOM    846  O   ARG A 105     -10.061  20.698 -19.279  1.00 74.18      A    O  \\nATOM    847  CB  ARG A 105     -10.440  23.566 -20.248  1.00 71.41      A    C  \\nATOM    848  CG  ARG A 105     -11.483  24.655 -20.521  1.00 67.24      A    C  \\nATOM    849  CD  ARG A 105     -11.341  25.189 -21.941  1.00 64.67      A    C  \\nATOM    850  NE  ARG A 105     -12.320  26.233 -22.208  1.00 61.82      A    N  \\nATOM    851  CZ  ARG A 105     -12.402  26.886 -23.359  1.00 59.29      A    C  \\nATOM    852  NH1 ARG A 105     -11.598  26.594 -24.352  1.00 50.75      A    N1+\\nATOM    853  NH2 ARG A 105     -13.319  27.830 -23.530  1.00 51.57      A    N  \\nATOM    854  N   LYS A 106      -8.506  21.821 -18.120  1.00 75.33      A    N  \\nATOM    855  CA  LYS A 106      -7.635  20.653 -17.998  1.00 75.74      A    C  \\nATOM    856  C   LYS A 106      -8.089  19.719 -16.901  1.00 76.73      A    C  \\nATOM    857  O   LYS A 106      -7.721  18.537 -16.928  1.00 75.54      A    O  \\nATOM    858  CB  LYS A 106      -6.183  21.104 -17.741  1.00 72.72      A    C  \\nATOM    859  CG  LYS A 106      -5.548  21.875 -18.863  1.00 67.54      A    C  \\nATOM    860  CD  LYS A 106      -4.126  22.240 -18.533  1.00 64.96      A    C  \\nATOM    861  CE  LYS A 106      -3.479  23.043 -19.623  1.00 59.59      A    C  \\nATOM    862  NZ  LYS A 106      -2.092  23.449 -19.288  1.00 51.56      A    N1+\\nATOM    863  N   TYR A 107      -8.860  20.208 -15.962  1.00 79.73      A    N  \\nATOM    864  CA  TYR A 107      -9.297  19.392 -14.825  1.00 81.94      A    C  \\nATOM    865  C   TYR A 107     -10.732  18.909 -14.952  1.00 83.25      A    C  \\nATOM    866  O   TYR A 107     -11.185  18.140 -14.133  1.00 81.87      A    O  \\nATOM    867  CB  TYR A 107      -9.142  20.162 -13.518  1.00 80.71      A    C  \\nATOM    868  CG  TYR A 107      -7.696  20.515 -13.192  1.00 80.76      A    C  \\nATOM    869  CD1 TYR A 107      -6.864  19.567 -12.624  1.00 79.71      A    C  \\nATOM    870  CD2 TYR A 107      -7.191  21.766 -13.461  1.00 80.20      A    C  \\nATOM    871  CE1 TYR A 107      -5.538  19.855 -12.317  1.00 77.71      A    C  \\nATOM    872  CE2 TYR A 107      -5.868  22.079 -13.170  1.00 79.29      A    C  \\nATOM    873  CZ  TYR A 107      -5.062  21.116 -12.604  1.00 77.97      A    C  \\nATOM    874  OH  TYR A 107      -3.731  21.405 -12.305  1.00 73.03      A    O  \\nATOM    875  N   ARG A 108     -11.437  19.362 -15.995  1.00 79.60      A    N  \\nATOM    876  CA  ARG A 108     -12.828  18.951 -16.172  1.00 80.64      A    C  \\nATOM    877  C   ARG A 108     -12.883  17.582 -16.854  1.00 81.20      A    C  \\nATOM    878  O   ARG A 108     -12.026  17.248 -17.660  1.00 80.99      A    O  \\nATOM    879  CB  ARG A 108     -13.609  19.988 -16.981  1.00 77.66      A    C  \\nATOM    880  CG  ARG A 108     -13.058  20.230 -18.379  1.00 69.88      A    C  \\nATOM    881  CD  ARG A 108     -13.858  21.298 -19.087  1.00 66.81      A    C  \\nATOM    882  NE  ARG A 108     -13.395  21.509 -20.474  1.00 63.05      A    N  \\nATOM    883  CZ  ARG A 108     -13.895  22.423 -21.296  1.00 58.19      A    C  \\nATOM    884  NH1 ARG A 108     -14.917  23.159 -20.909  1.00 50.40      A    N1+\\nATOM    885  NH2 ARG A 108     -13.381  22.562 -22.524  1.00 49.86      A    N  \\nATOM    886  N   PRO A 109     -13.897  16.772 -16.569  1.00 80.39      A    N  \\nATOM    887  CA  PRO A 109     -14.030  15.487 -17.253  1.00 80.32      A    C  \\nATOM    888  C   PRO A 109     -14.591  15.657 -18.661  1.00 80.00      A    C  \\nATOM    889  O   PRO A 109     -14.931  16.751 -19.101  1.00 77.90      A    O  \\nATOM    890  CB  PRO A 109     -15.015  14.740 -16.363  1.00 78.51      A    C  \\nATOM    891  CG  PRO A 109     -15.832  15.781 -15.747  1.00 79.08      A    C  \\nATOM    892  CD  PRO A 109     -14.905  16.944 -15.502  1.00 81.90      A    C  \\nATOM    893  N   GLN A 110     -14.643  14.527 -19.340  1.00 78.49      A    N  \\nATOM    894  CA  GLN A 110     -15.169  14.518 -20.702  1.00 77.99      A    C  \\nATOM    895  C   GLN A 110     -16.693  14.401 -20.699  1.00 78.24      A    C  \\nATOM    896  O   GLN A 110     -17.231  13.448 -20.133  1.00 75.78      A    O  \\nATOM    897  CB  GLN A 110     -14.548  13.388 -21.498  1.00 74.96      A    C  \\nATOM    898  CG  GLN A 110     -13.059  13.517 -21.716  1.00 68.17      A    C  \\nATOM    899  CD  GLN A 110     -12.484  12.365 -22.522  1.00 62.44      A    C  \\nATOM    900  NE2 GLN A 110     -11.207  12.463 -22.880  1.00 54.73      A    N  \\nATOM    901  OE1 GLN A 110     -13.189  11.403 -22.800  1.00 57.81      A    O  \\nATOM    902  N   GLY A 111     -17.330  15.372 -21.276  1.00 78.11      A    N  \\nATOM    903  CA  GLY A 111     -18.762  15.384 -21.361  1.00 77.63      A    C  \\nATOM    904  C   GLY A 111     -19.471  16.099 -20.237  1.00 79.18      A    C  \\nATOM    905  O   GLY A 111     -20.680  16.310 -20.300  1.00 77.04      A    O  \\nATOM    906  N   TYR A 112     -18.737  16.447 -19.195  1.00 80.06      A    N  \\nATOM    907  CA  TYR A 112     -19.292  17.116 -18.042  1.00 80.98      A    C  \\nATOM    908  C   TYR A 112     -18.666  18.477 -17.876  1.00 79.93      A    C  \\nATOM    909  O   TYR A 112     -17.515  18.711 -18.225  1.00 77.96      A    O  \\nATOM    910  CB  TYR A 112     -19.095  16.285 -16.770  1.00 80.20      A    C  \\nATOM    911  CG  TYR A 112     -19.864  14.984 -16.794  1.00 80.96      A    C  \\nATOM    912  CD1 TYR A 112     -21.189  14.933 -16.403  1.00 80.43      A    C  \\nATOM    913  CD2 TYR A 112     -19.259  13.795 -17.202  1.00 80.67      A    C  \\nATOM    914  CE1 TYR A 112     -21.895  13.746 -16.417  1.00 79.31      A    C  \\nATOM    915  CE2 TYR A 112     -19.952  12.607 -17.222  1.00 80.26      A    C  \\nATOM    916  CZ  TYR A 112     -21.271  12.606 -16.829  1.00 79.55      A    C  \\nATOM    917  OH  TYR A 112     -21.980  11.419 -16.873  1.00 76.11      A    O  \\nATOM    918  N   SER A 113     -19.443  19.386 -17.306  1.00 80.47      A    N  \\nATOM    919  CA  SER A 113     -19.005  20.770 -17.096  1.00 80.27      A    C  \\nATOM    920  C   SER A 113     -18.318  20.880 -15.750  1.00 80.90      A    C  \\nATOM    921  O   SER A 113     -18.162  19.909 -14.997  1.00 79.25      A    O  \\nATOM    922  CB  SER A 113     -20.201  21.713 -17.177  1.00 76.90      A    C  \\nATOM    923  OG  SER A 113     -21.130  21.454 -16.146  1.00 70.73      A    O  \\nATOM    924  N   LEU A 114     -17.893  22.094 -15.427  1.00 80.74      A    N  \\nATOM    925  CA  LEU A 114     -17.199  22.381 -14.186  1.00 80.94      A    C  \\nATOM    926  C   LEU A 114     -17.602  23.766 -13.696  1.00 79.85      A    C  \\nATOM    927  O   LEU A 114     -17.569  24.719 -14.479  1.00 76.62      A    O  \\nATOM    928  CB  LEU A 114     -15.691  22.281 -14.358  1.00 79.88      A    C  \\nATOM    929  CG  LEU A 114     -14.809  22.509 -13.113  1.00 79.41      A    C  \\nATOM    930  CD1 LEU A 114     -15.060  21.416 -12.087  1.00 79.69      A    C  \\nATOM    931  CD2 LEU A 114     -13.332  22.566 -13.493  1.00 79.48      A    C  \\nATOM    932  N   ASN A 115     -17.961  23.844 -12.445  1.00 81.50      A    N  \\nATOM    933  CA  ASN A 115     -18.353  25.109 -11.859  1.00 80.60      A    C  \\nATOM    934  C   ASN A 115     -17.137  25.968 -11.556  1.00 79.84      A    C  \\nATOM    935  O   ASN A 115     -16.004  25.469 -11.473  1.00 77.40      A    O  \\nATOM    936  CB  ASN A 115     -19.180  24.915 -10.581  1.00 78.98      A    C  \\nATOM    937  CG  ASN A 115     -20.556  24.336 -10.870  1.00 76.80      A    C  \\nATOM    938  ND2 ASN A 115     -21.006  23.429 -10.017  1.00 71.36      A    N  \\nATOM    939  OD1 ASN A 115     -21.205  24.686 -11.850  1.00 70.51      A    O  \\nATOM    940  N   LEU A 116     -17.383  27.247 -11.362  1.00 75.70      A    N  \\nATOM    941  CA  LEU A 116     -16.297  28.178 -11.079  1.00 75.16      A    C  \\nATOM    942  C   LEU A 116     -15.946  28.133  -9.589  1.00 75.63      A    C  \\nATOM    943  O   LEU A 116     -16.816  27.877  -8.736  1.00 74.27      A    O  \\nATOM    944  CB  LEU A 116     -16.687  29.600 -11.459  1.00 71.97      A    C  \\nATOM    945  CG  LEU A 116     -16.861  29.891 -12.943  1.00 69.28      A    C  \\nATOM    946  CD1 LEU A 116     -15.558  29.691 -13.676  1.00 67.97      A    C  \\nATOM    947  CD2 LEU A 116     -17.383  31.296 -13.171  1.00 67.43      A    C  \\nATOM    948  N   PRO A 117     -14.689  28.377  -9.241  1.00 77.57      A    N  \\nATOM    949  CA  PRO A 117     -14.319  28.408  -7.815  1.00 76.94      A    C  \\nATOM    950  C   PRO A 117     -14.841  29.678  -7.146  1.00 76.67      A    C  \\nATOM    951  O   PRO A 117     -15.059  30.689  -7.805  1.00 74.60      A    O  \\nATOM    952  CB  PRO A 117     -12.808  28.369  -7.856  1.00 75.11      A    C  \\nATOM    953  CG  PRO A 117     -12.460  28.968  -9.176  1.00 73.88      A    C  \\nATOM    954  CD  PRO A 117     -13.538  28.495 -10.096  1.00 76.27      A    C  \\nATOM    955  N   PRO A 118     -15.054  29.625  -5.838  1.00 79.55      A    N  \\nATOM    956  CA  PRO A 118     -15.559  30.810  -5.136  1.00 78.86      A    C  \\nATOM    957  C   PRO A 118     -14.456  31.862  -4.987  1.00 77.65      A    C  \\nATOM    958  O   PRO A 118     -13.274  31.579  -5.149  1.00 75.20      A    O  \\nATOM    959  CB  PRO A 118     -15.995  30.260  -3.775  1.00 78.32      A    C  \\nATOM    960  CG  PRO A 118     -15.129  29.096  -3.590  1.00 78.87      A    C  \\nATOM    961  CD  PRO A 118     -14.958  28.480  -4.936  1.00 81.76      A    C  \\nATOM    962  N   LYS A 119     -14.878  33.081  -4.669  1.00 72.67      A    N  \\nATOM    963  CA  LYS A 119     -13.942  34.205  -4.537  1.00 71.08      A    C  \\nATOM    964  C   LYS A 119     -13.757  34.673  -3.086  1.00 71.07      A    C  \\nATOM    965  O   LYS A 119     -12.736  35.263  -2.766  1.00 67.34      A    O  \\nATOM    966  CB  LYS A 119     -14.393  35.388  -5.393  1.00 66.92      A    C  \\nATOM    967  CG  LYS A 119     -14.386  35.087  -6.881  1.00 61.96      A    C  \\nATOM    968  CD  LYS A 119     -14.881  36.287  -7.690  1.00 56.94      A    C  \\nATOM    969  CE  LYS A 119     -14.894  35.984  -9.153  1.00 51.38      A    C  \\nATOM    970  NZ  LYS A 119     -15.400  37.103  -9.957  1.00 44.79      A    N1+\\nATOM    971  N   ASP A 120     -14.752  34.392  -2.265  1.00 74.15      A    N  \\nATOM    972  CA  ASP A 120     -14.688  34.854  -0.879  1.00 73.76      A    C  \\nATOM    973  C   ASP A 120     -13.915  33.861  -0.010  1.00 74.28      A    C  \\nATOM    974  O   ASP A 120     -12.855  34.171   0.521  1.00 72.08      A    O  \\nATOM    975  CB  ASP A 120     -16.091  35.075  -0.323  1.00 69.80      A    C  \\nATOM    976  CG  ASP A 120     -16.820  36.199  -1.020  1.00 64.96      A    C  \\nATOM    977  OD1 ASP A 120     -16.141  37.164  -1.452  1.00 59.24      A    O  \\nATOM    978  OD2 ASP A 120     -18.062  36.150  -1.095  1.00 63.94      A    O1-\\nATOM    979  N   ARG A 121     -14.469  32.670   0.132  1.00 76.73      A    N  \\nATOM    980  CA  ARG A 121     -13.866  31.624   0.936  1.00 77.93      A    C  \\nATOM    981  C   ARG A 121     -12.824  30.871   0.106  1.00 78.94      A    C  \\nATOM    982  O   ARG A 121     -12.917  30.852  -1.125  1.00 78.50      A    O  \\nATOM    983  CB  ARG A 121     -14.942  30.694   1.462  1.00 76.55      A    C  \\nATOM    984  CG  ARG A 121     -15.729  29.992   0.390  1.00 74.83      A    C  \\nATOM    985  CD  ARG A 121     -16.808  29.123   0.989  1.00 74.67      A    C  \\nATOM    986  NE  ARG A 121     -17.567  28.367  -0.025  1.00 71.43      A    N  \\nATOM    987  CZ  ARG A 121     -18.586  27.575   0.217  1.00 72.42      A    C  \\nATOM    988  NH1 ARG A 121     -19.026  27.425   1.439  1.00 61.81      A    N1+\\nATOM    989  NH2 ARG A 121     -19.183  26.961  -0.791  1.00 65.73      A    N  \\nATOM    990  N   PRO A 122     -11.852  30.247   0.757  1.00 79.88      A    N  \\nATOM    991  CA  PRO A 122     -10.875  29.466  -0.001  1.00 80.37      A    C  \\nATOM    992  C   PRO A 122     -11.493  28.179  -0.554  1.00 82.44      A    C  \\nATOM    993  O   PRO A 122     -12.542  27.727  -0.094  1.00 81.54      A    O  \\nATOM    994  CB  PRO A 122      -9.791  29.144   1.035  1.00 77.55      A    C  \\nATOM    995  CG  PRO A 122     -10.502  29.226   2.318  1.00 74.78      A    C  \\nATOM    996  CD  PRO A 122     -11.504  30.316   2.176  1.00 76.97      A    C  \\nATOM    997  N   ILE A 123     -10.829  27.632  -1.533  1.00 83.65      A    N  \\nATOM    998  CA  ILE A 123     -11.341  26.480  -2.242  1.00 85.51      A    C  \\nATOM    999  C   ILE A 123     -11.089  25.235  -1.401  1.00 86.91      A    C  \\nATOM   1000  O   ILE A 123      -9.952  24.941  -1.025  1.00 85.93      A    O  \\nATOM   1001  CB  ILE A 123     -10.690  26.333  -3.627  1.00 83.90      A    C  \\nATOM   1002  CG1 ILE A 123     -10.972  27.574  -4.445  1.00 81.35      A    C  \\nATOM   1003  CG2 ILE A 123     -11.224  25.101  -4.360  1.00 82.91      A    C  \\nATOM   1004  CD1 ILE A 123     -10.188  27.658  -5.704  1.00 77.22      A    C  \\nATOM   1005  N   ARG A 124     -12.127  24.484  -1.108  1.00 89.19      A    N  \\nATOM   1006  CA  ARG A 124     -12.038  23.288  -0.301  1.00 90.59      A    C  \\nATOM   1007  C   ARG A 124     -12.217  22.075  -1.204  1.00 91.25      A    C  \\nATOM   1008  O   ARG A 124     -13.310  21.832  -1.725  1.00 90.70      A    O  \\nATOM   1009  CB  ARG A 124     -13.114  23.307   0.803  1.00 90.67      A    C  \\nATOM   1010  CG  ARG A 124     -12.928  24.438   1.810  1.00 88.53      A    C  \\nATOM   1011  CD  ARG A 124     -14.048  24.482   2.833  1.00 86.72      A    C  \\nATOM   1012  NE  ARG A 124     -14.003  23.284   3.683  1.00 81.43      A    N  \\nATOM   1013  CZ  ARG A 124     -13.289  23.203   4.797  1.00 80.88      A    C  \\nATOM   1014  NH1 ARG A 124     -12.572  24.240   5.188  1.00 69.83      A    N1+\\nATOM   1015  NH2 ARG A 124     -13.319  22.097   5.508  1.00 71.58      A    N  \\nATOM   1016  N   ILE A 125     -11.154  21.342  -1.393  1.00 92.65      A    N  \\nATOM   1017  CA  ILE A 125     -11.158  20.170  -2.260  1.00 92.72      A    C  \\nATOM   1018  C   ILE A 125     -11.149  18.930  -1.397  1.00 93.42      A    C  \\nATOM   1019  O   ILE A 125     -10.269  18.753  -0.549  1.00 92.65      A    O  \\nATOM   1020  CB  ILE A 125      -9.953  20.177  -3.228  1.00 91.56      A    C  \\nATOM   1021  CG1 ILE A 125     -10.032  21.376  -4.155  1.00 89.83      A    C  \\nATOM   1022  CG2 ILE A 125      -9.916  18.884  -4.029  1.00 90.56      A    C  \\nATOM   1023  CD1 ILE A 125      -8.825  21.533  -5.019  1.00 86.82      A    C  \\nATOM   1024  N   TYR A 126     -12.090  18.066  -1.613  1.00 92.99      A    N  \\nATOM   1025  CA  TYR A 126     -12.221  16.821  -0.861  1.00 93.54      A    C  \\nATOM   1026  C   TYR A 126     -11.576  15.704  -1.676  1.00 93.22      A    C  \\nATOM   1027  O   TYR A 126     -11.962  15.447  -2.806  1.00 91.91      A    O  \\nATOM   1028  CB  TYR A 126     -13.676  16.506  -0.548  1.00 93.41      A    C  \\nATOM   1029  CG  TYR A 126     -13.882  15.163   0.102  1.00 93.89      A    C  \\nATOM   1030  CD1 TYR A 126     -13.597  14.975   1.443  1.00 92.71      A    C  \\nATOM   1031  CD2 TYR A 126     -14.380  14.096  -0.644  1.00 92.25      A    C  \\nATOM   1032  CE1 TYR A 126     -13.791  13.739   2.037  1.00 91.71      A    C  \\nATOM   1033  CE2 TYR A 126     -14.568  12.879  -0.046  1.00 91.48      A    C  \\nATOM   1034  CZ  TYR A 126     -14.283  12.703   1.291  1.00 91.54      A    C  \\nATOM   1035  OH  TYR A 126     -14.489  11.490   1.874  1.00 89.12      A    O  \\nATOM   1036  N   ALA A 127     -10.583  15.061  -1.098  1.00 93.90      A    N  \\nATOM   1037  CA  ALA A 127      -9.893  13.957  -1.758  1.00 93.04      A    C  \\nATOM   1038  C   ALA A 127     -10.427  12.638  -1.205  1.00 91.86      A    C  \\nATOM   1039  O   ALA A 127     -10.232  12.299  -0.032  1.00 89.94      A    O  \\nATOM   1040  CB  ALA A 127      -8.381  14.049  -1.556  1.00 92.36      A    C  \\nATOM   1041  N   ASP A 128     -11.090  11.904  -2.068  1.00 88.62      A    N  \\nATOM   1042  CA  ASP A 128     -11.721  10.640  -1.688  1.00 86.17      A    C  \\nATOM   1043  C   ASP A 128     -10.764   9.506  -1.998  1.00 85.24      A    C  \\nATOM   1044  O   ASP A 128     -10.273   9.373  -3.115  1.00 80.58      A    O  \\nATOM   1045  CB  ASP A 128     -13.030  10.448  -2.415  1.00 82.21      A    C  \\nATOM   1046  CG  ASP A 128     -13.860   9.303  -1.868  1.00 78.96      A    C  \\nATOM   1047  OD1 ASP A 128     -13.408   8.647  -0.889  1.00 70.75      A    O  \\nATOM   1048  OD2 ASP A 128     -14.948   9.041  -2.406  1.00 72.99      A    O1-\\nATOM   1049  N   GLY A 129     -10.524   8.682  -1.013  1.00 89.01      A    N  \\nATOM   1050  CA  GLY A 129      -9.615   7.551  -1.194  1.00 88.31      A    C  \\nATOM   1051  C   GLY A 129      -9.534   6.725   0.079  1.00 89.35      A    C  \\nATOM   1052  O   GLY A 129      -9.901   7.185   1.169  1.00 87.97      A    O  \\nATOM   1053  N   VAL A 130      -9.065   5.515  -0.073  1.00 88.55      A    N  \\nATOM   1054  CA  VAL A 130      -8.858   4.649   1.087  1.00 89.18      A    C  \\nATOM   1055  C   VAL A 130      -7.577   4.999   1.794  1.00 90.53      A    C  \\nATOM   1056  O   VAL A 130      -7.531   5.026   3.014  1.00 88.69      A    O  \\nATOM   1057  CB  VAL A 130      -8.888   3.172   0.662  1.00 87.15      A    C  \\nATOM   1058  CG1 VAL A 130     -10.213   2.862  -0.042  1.00 84.22      A    C  \\nATOM   1059  CG2 VAL A 130      -8.682   2.269   1.844  1.00 84.65      A    C  \\nATOM   1060  N   PHE A 131      -6.545   5.302   1.031  1.00 89.84      A    N  \\nATOM   1061  CA  PHE A 131      -5.232   5.690   1.526  1.00 90.84      A    C  \\nATOM   1062  C   PHE A 131      -4.597   4.602   2.375  1.00 90.76      A    C  \\nATOM   1063  O   PHE A 131      -3.980   4.871   3.385  1.00 86.98      A    O  \\nATOM   1064  CB  PHE A 131      -5.267   7.027   2.272  1.00 91.44      A    C  \\nATOM   1065  CG  PHE A 131      -5.724   8.196   1.418  1.00 92.86      A    C  \\nATOM   1066  CD1 PHE A 131      -4.860   8.780   0.517  1.00 91.32      A    C  \\nATOM   1067  CD2 PHE A 131      -7.022   8.645   1.492  1.00 91.63      A    C  \\nATOM   1068  CE1 PHE A 131      -5.269   9.828  -0.265  1.00 90.23      A    C  \\nATOM   1069  CE2 PHE A 131      -7.433   9.710   0.705  1.00 90.98      A    C  \\nATOM   1070  CZ  PHE A 131      -6.568  10.290  -0.179  1.00 90.23      A    C  \\nATOM   1071  N   ASP A 132      -4.781   3.357   1.936  1.00 89.51      A    N  \\nATOM   1072  CA  ASP A 132      -4.130   2.244   2.600  1.00 88.82      A    C  \\nATOM   1073  C   ASP A 132      -2.722   2.086   2.022  1.00 89.04      A    C  \\nATOM   1074  O   ASP A 132      -2.536   2.085   0.817  1.00 86.98      A    O  \\nATOM   1075  CB  ASP A 132      -4.947   0.969   2.399  1.00 87.50      A    C  \\nATOM   1076  CG  ASP A 132      -4.466  -0.199   3.239  1.00 87.90      A    C  \\nATOM   1077  OD1 ASP A 132      -3.662   0.040   4.172  1.00 82.93      A    O  \\nATOM   1078  OD2 ASP A 132      -4.896  -1.335   2.973  1.00 83.03      A    O1-\\nATOM   1079  N   LEU A 133      -1.751   1.963   2.920  1.00 91.30      A    N  \\nATOM   1080  CA  LEU A 133      -0.326   1.873   2.544  1.00 91.04      A    C  \\nATOM   1081  C   LEU A 133       0.054   3.047   1.661  1.00 90.65      A    C  \\nATOM   1082  O   LEU A 133       0.233   2.916   0.446  1.00 87.17      A    O  \\nATOM   1083  CB  LEU A 133      -0.028   0.543   1.844  1.00 89.71      A    C  \\nATOM   1084  CG  LEU A 133      -0.187  -0.702   2.716  1.00 89.36      A    C  \\nATOM   1085  CD1 LEU A 133       0.861  -0.707   3.815  1.00 88.25      A    C  \\nATOM   1086  CD2 LEU A 133      -0.100  -1.944   1.872  1.00 87.84      A    C  \\nATOM   1087  N   PHE A 134       0.189   4.192   2.313  1.00 92.69      A    N  \\nATOM   1088  CA  PHE A 134       0.399   5.457   1.623  1.00 92.54      A    C  \\nATOM   1089  C   PHE A 134       1.669   5.450   0.782  1.00 92.18      A    C  \\nATOM   1090  O   PHE A 134       2.771   5.368   1.290  1.00 89.20      A    O  \\nATOM   1091  CB  PHE A 134       0.434   6.591   2.679  1.00 91.61      A    C  \\nATOM   1092  CG  PHE A 134       0.391   7.981   2.071  1.00 92.68      A    C  \\nATOM   1093  CD1 PHE A 134      -0.800   8.539   1.686  1.00 90.37      A    C  \\nATOM   1094  CD2 PHE A 134       1.559   8.721   1.938  1.00 90.03      A    C  \\nATOM   1095  CE1 PHE A 134      -0.857   9.800   1.167  1.00 88.01      A    C  \\nATOM   1096  CE2 PHE A 134       1.511   9.997   1.413  1.00 88.23      A    C  \\nATOM   1097  CZ  PHE A 134       0.303  10.544   1.032  1.00 88.42      A    C  \\nATOM   1098  N   HIS A 135       1.474   5.491  -0.527  1.00 91.26      A    N  \\nATOM   1099  CA  HIS A 135       2.562   5.347  -1.468  1.00 89.96      A    C  \\nATOM   1100  C   HIS A 135       2.771   6.671  -2.226  1.00 89.31      A    C  \\nATOM   1101  O   HIS A 135       2.238   7.710  -1.857  1.00 89.33      A    O  \\nATOM   1102  CB  HIS A 135       2.311   4.187  -2.435  1.00 87.96      A    C  \\nATOM   1103  CG  HIS A 135       0.987   4.221  -3.109  1.00 90.50      A    C  \\nATOM   1104  CD2 HIS A 135      -0.192   3.628  -2.791  1.00 83.90      A    C  \\nATOM   1105  ND1 HIS A 135       0.728   4.972  -4.244  1.00 84.74      A    N  \\nATOM   1106  CE1 HIS A 135      -0.527   4.825  -4.591  1.00 83.82      A    C  \\nATOM   1107  NE2 HIS A 135      -1.115   4.010  -3.736  1.00 85.67      A    N  \\nATOM   1108  N   LEU A 136       3.542   6.598  -3.283  1.00 90.64      A    N  \\nATOM   1109  CA  LEU A 136       3.854   7.780  -4.072  1.00 88.68      A    C  \\nATOM   1110  C   LEU A 136       2.644   8.264  -4.895  1.00 88.68      A    C  \\nATOM   1111  O   LEU A 136       2.537   9.457  -5.180  1.00 87.12      A    O  \\nATOM   1112  CB  LEU A 136       5.052   7.499  -4.973  1.00 86.01      A    C  \\nATOM   1113  CG  LEU A 136       5.606   8.632  -5.816  1.00 81.23      A    C  \\nATOM   1114  CD1 LEU A 136       6.133   9.750  -4.919  1.00 81.57      A    C  \\nATOM   1115  CD2 LEU A 136       6.730   8.095  -6.689  1.00 80.45      A    C  \\nATOM   1116  N   GLY A 137       1.798   7.326  -5.270  1.00 89.22      A    N  \\nATOM   1117  CA  GLY A 137       0.624   7.713  -6.049  1.00 88.84      A    C  \\nATOM   1118  C   GLY A 137      -0.329   8.616  -5.302  1.00 89.81      A    C  \\nATOM   1119  O   GLY A 137      -0.940   9.489  -5.889  1.00 88.52      A    O  \\nATOM   1120  N   HIS A 138      -0.442   8.373  -3.995  1.00 90.70      A    N  \\nATOM   1121  CA  HIS A 138      -1.285   9.261  -3.197  1.00 90.50      A    C  \\nATOM   1122  C   HIS A 138      -0.641  10.633  -3.020  1.00 90.43      A    C  \\nATOM   1123  O   HIS A 138      -1.344  11.628  -2.924  1.00 90.18      A    O  \\nATOM   1124  CB  HIS A 138      -1.585   8.609  -1.868  1.00 91.30      A    C  \\nATOM   1125  CG  HIS A 138      -2.423   7.391  -1.939  1.00 92.64      A    C  \\nATOM   1126  CD2 HIS A 138      -3.760   7.214  -1.955  1.00 88.95      A    C  \\nATOM   1127  ND1 HIS A 138      -1.873   6.133  -2.039  1.00 87.65      A    N  \\nATOM   1128  CE1 HIS A 138      -2.846   5.248  -2.089  1.00 88.28      A    C  \\nATOM   1129  NE2 HIS A 138      -4.004   5.886  -2.044  1.00 88.52      A    N  \\nATOM   1130  N   MET A 139       0.662  10.655  -2.915  1.00 90.83      A    N  \\nATOM   1131  CA  MET A 139       1.342  11.939  -2.756  1.00 89.52      A    C  \\nATOM   1132  C   MET A 139       1.278  12.788  -4.010  1.00 88.80      A    C  \\nATOM   1133  O   MET A 139       1.215  14.019  -3.941  1.00 88.27      A    O  \\nATOM   1134  CB  MET A 139       2.812  11.736  -2.342  1.00 88.90      A    C  \\nATOM   1135  CG  MET A 139       2.967  11.098  -0.953  1.00 80.57      A    C  \\nATOM   1136  SD  MET A 139       4.693  10.951  -0.433  1.00 68.07      A    S  \\nATOM   1137  CE  MET A 139       5.209   9.543  -1.391  1.00 60.92      A    C  \\nATOM   1138  N   LYS A 140       1.282  12.116  -5.163  1.00 87.96      A    N  \\nATOM   1139  CA  LYS A 140       1.150  12.864  -6.421  1.00 86.28      A    C  \\nATOM   1140  C   LYS A 140      -0.272  13.315  -6.662  1.00 86.34      A    C  \\nATOM   1141  O   LYS A 140      -0.511  14.357  -7.291  1.00 85.21      A    O  \\nATOM   1142  CB  LYS A 140       1.634  12.008  -7.594  1.00 83.93      A    C  \\nATOM   1143  CG  LYS A 140       3.134  11.777  -7.617  1.00 81.69      A    C  \\nATOM   1144  CD  LYS A 140       3.557  10.978  -8.845  1.00 77.28      A    C  \\nATOM   1145  CE  LYS A 140       5.040  10.747  -8.880  1.00 73.42      A    C  \\nATOM   1146  NZ  LYS A 140       5.449   9.926 -10.055  1.00 65.90      A    N1+\\nATOM   1147  N   GLN A 141      -1.238  12.525  -6.185  1.00 89.23      A    N  \\nATOM   1148  CA  GLN A 141      -2.639  12.919  -6.343  1.00 89.01      A    C  \\nATOM   1149  C   GLN A 141      -2.976  14.102  -5.430  1.00 89.20      A    C  \\nATOM   1150  O   GLN A 141      -3.682  15.019  -5.824  1.00 88.50      A    O  \\nATOM   1151  CB  GLN A 141      -3.553  11.742  -6.036  1.00 88.51      A    C  \\nATOM   1152  CG  GLN A 141      -5.034  12.051  -6.263  1.00 83.54      A    C  \\nATOM   1153  CD  GLN A 141      -5.932  10.869  -5.980  1.00 81.70      A    C  \\nATOM   1154  NE2 GLN A 141      -7.218  11.034  -6.210  1.00 72.78      A    N  \\nATOM   1155  OE1 GLN A 141      -5.445   9.821  -5.558  1.00 75.54      A    O  \\nATOM   1156  N   LEU A 142      -2.458  14.063  -4.208  1.00 91.74      A    N  \\nATOM   1157  CA  LEU A 142      -2.697  15.162  -3.282  1.00 91.84      A    C  \\nATOM   1158  C   LEU A 142      -1.930  16.411  -3.691  1.00 90.95      A    C  \\nATOM   1159  O   LEU A 142      -2.351  17.525  -3.383  1.00 89.68      A    O  \\nATOM   1160  CB  LEU A 142      -2.348  14.756  -1.856  1.00 92.72      A    C  \\nATOM   1161  CG  LEU A 142      -3.220  13.661  -1.242  1.00 93.18      A    C  \\nATOM   1162  CD1 LEU A 142      -4.648  14.124  -1.110  1.00 92.13      A    C  \\nATOM   1163  CD2 LEU A 142      -2.673  13.222   0.105  1.00 92.70      A    C  \\nATOM   1164  N   GLU A 143      -0.817  16.219  -4.354  1.00 88.78      A    N  \\nATOM   1165  CA  GLU A 143      -0.074  17.358  -4.880  1.00 87.20      A    C  \\nATOM   1166  C   GLU A 143      -0.837  18.036  -6.025  1.00 86.57      A    C  \\nATOM   1167  O   GLU A 143      -0.850  19.263  -6.105  1.00 85.16      A    O  \\nATOM   1168  CB  GLU A 143       1.322  16.918  -5.335  1.00 85.27      A    C  \\nATOM   1169  CG  GLU A 143       2.208  18.056  -5.787  1.00 83.37      A    C  \\nATOM   1170  CD  GLU A 143       3.577  17.596  -6.239  1.00 80.96      A    C  \\nATOM   1171  OE1 GLU A 143       3.840  16.382  -6.165  1.00 76.36      A    O  \\nATOM   1172  OE2 GLU A 143       4.409  18.438  -6.633  1.00 73.12      A    O1-\\nATOM   1173  N   GLN A 144      -1.475  17.234  -6.857  1.00 88.12      A    N  \\nATOM   1174  CA  GLN A 144      -2.293  17.808  -7.941  1.00 87.64      A    C  \\nATOM   1175  C   GLN A 144      -3.541  18.469  -7.400  1.00 87.82      A    C  \\nATOM   1176  O   GLN A 144      -4.044  19.411  -8.017  1.00 87.34      A    O  \\nATOM   1177  CB  GLN A 144      -2.661  16.706  -8.928  1.00 86.88      A    C  \\nATOM   1178  CG  GLN A 144      -1.509  16.149  -9.724  1.00 84.51      A    C  \\nATOM   1179  CD  GLN A 144      -1.920  15.018 -10.652  1.00 82.90      A    C  \\nATOM   1180  NE2 GLN A 144      -0.977  14.484 -11.394  1.00 74.14      A    N  \\nATOM   1181  OE1 GLN A 144      -3.089  14.644 -10.717  1.00 76.58      A    O  \\nATOM   1182  N   CYS A 145      -4.040  17.998  -6.271  1.00 89.09      A    N  \\nATOM   1183  CA  CYS A 145      -5.185  18.651  -5.666  1.00 89.21      A    C  \\nATOM   1184  C   CYS A 145      -4.818  19.978  -5.033  1.00 87.91      A    C  \\nATOM   1185  O   CYS A 145      -5.560  20.939  -5.081  1.00 85.57      A    O  \\nATOM   1186  CB  CYS A 145      -5.822  17.736  -4.619  1.00 90.28      A    C  \\nATOM   1187  SG  CYS A 145      -6.519  16.221  -5.303  1.00 89.50      A    S  \\nATOM   1188  N   LYS A 146      -3.623  20.028  -4.428  1.00 87.62      A    N  \\nATOM   1189  CA  LYS A 146      -3.206  21.252  -3.751  1.00 85.58      A    C  \\nATOM   1190  C   LYS A 146      -2.745  22.308  -4.738  1.00 84.46      A    C  \\nATOM   1191  O   LYS A 146      -3.000  23.484  -4.533  1.00 81.98      A    O  \\nATOM   1192  CB  LYS A 146      -2.083  20.927  -2.762  1.00 84.29      A    C  \\nATOM   1193  CG  LYS A 146      -1.679  22.084  -1.892  1.00 82.46      A    C  \\nATOM   1194  CD  LYS A 146      -2.798  22.545  -0.980  1.00 81.66      A    C  \\nATOM   1195  CE  LYS A 146      -2.354  23.662  -0.066  1.00 80.32      A    C  \\nATOM   1196  NZ  LYS A 146      -3.474  24.153   0.774  1.00 76.88      A    N1+\\nATOM   1197  N   LYS A 147      -2.122  21.889  -5.792  1.00 81.52      A    N  \\nATOM   1198  CA  LYS A 147      -1.579  22.799  -6.803  1.00 79.98      A    C  \\nATOM   1199  C   LYS A 147      -2.555  23.098  -7.939  1.00 79.96      A    C  \\nATOM   1200  O   LYS A 147      -2.141  23.608  -8.972  1.00 77.83      A    O  \\nATOM   1201  CB  LYS A 147      -0.266  22.288  -7.354  1.00 77.99      A    C  \\nATOM   1202  CG  LYS A 147       0.862  22.222  -6.351  1.00 74.35      A    C  \\nATOM   1203  CD  LYS A 147       2.125  21.646  -6.968  1.00 70.97      A    C  \\nATOM   1204  CE  LYS A 147       3.265  21.610  -5.977  1.00 66.82      A    C  \\nATOM   1205  NZ  LYS A 147       4.474  21.003  -6.548  1.00 59.13      A    N1+\\nATOM   1206  N   SER A 148      -3.812  22.759  -7.746  1.00 81.66      A    N  \\nATOM   1207  CA  SER A 148      -4.802  23.014  -8.792  1.00 80.72      A    C  \\nATOM   1208  C   SER A 148      -5.273  24.442  -8.775  1.00 78.91      A    C  \\nATOM   1209  O   SER A 148      -5.241  25.138  -9.792  1.00 75.69      A    O  \\nATOM   1210  CB  SER A 148      -5.974  22.056  -8.616  1.00 80.76      A    C  \\nATOM   1211  OG  SER A 148      -6.639  22.253  -7.375  1.00 75.68      A    O  \\nATOM   1212  N   PHE A 149      -5.704  24.900  -7.612  1.00 79.11      A    N  \\nATOM   1213  CA  PHE A 149      -6.211  26.252  -7.435  1.00 78.73      A    C  \\nATOM   1214  C   PHE A 149      -5.426  26.970  -6.358  1.00 77.93      A    C  \\nATOM   1215  O   PHE A 149      -4.885  26.319  -5.464  1.00 76.13      A    O  \\nATOM   1216  CB  PHE A 149      -7.687  26.235  -7.051  1.00 78.15      A    C  \\nATOM   1217  CG  PHE A 149      -8.607  25.641  -8.101  1.00 79.05      A    C  \\nATOM   1218  CD1 PHE A 149      -9.020  26.383  -9.186  1.00 79.09      A    C  \\nATOM   1219  CD2 PHE A 149      -9.032  24.323  -7.992  1.00 79.37      A    C  \\nATOM   1220  CE1 PHE A 149      -9.843  25.842 -10.133  1.00 76.95      A    C  \\nATOM   1221  CE2 PHE A 149      -9.865  23.779  -8.940  1.00 78.80      A    C  \\nATOM   1222  CZ  PHE A 149     -10.273  24.530 -10.021  1.00 76.81      A    C  \\nATOM   1223  N   PRO A 150      -5.345  28.285  -6.401  1.00 74.26      A    N  \\nATOM   1224  CA  PRO A 150      -4.696  29.018  -5.309  1.00 72.57      A    C  \\nATOM   1225  C   PRO A 150      -5.584  29.057  -4.056  1.00 72.53      A    C  \\nATOM   1226  O   PRO A 150      -6.805  29.074  -4.154  1.00 67.92      A    O  \\nATOM   1227  CB  PRO A 150      -4.504  30.426  -5.882  1.00 68.93      A    C  \\nATOM   1228  CG  PRO A 150      -5.583  30.557  -6.911  1.00 68.28      A    C  \\nATOM   1229  CD  PRO A 150      -5.710  29.174  -7.499  1.00 70.81      A    C  \\nATOM   1230  N   ASN A 151      -4.956  29.031  -2.916  1.00 76.24      A    N  \\nATOM   1231  CA  ASN A 151      -5.629  29.027  -1.632  1.00 75.91      A    C  \\nATOM   1232  C   ASN A 151      -6.564  27.840  -1.492  1.00 77.79      A    C  \\nATOM   1233  O   ASN A 151      -7.754  27.970  -1.258  1.00 75.60      A    O  \\nATOM   1234  CB  ASN A 151      -6.372  30.339  -1.360  1.00 71.09      A    C  \\nATOM   1235  CG  ASN A 151      -5.435  31.529  -1.178  1.00 63.30      A    C  \\nATOM   1236  ND2 ASN A 151      -5.798  32.648  -1.762  1.00 57.97      A    N  \\nATOM   1237  OD1 ASN A 151      -4.420  31.402  -0.532  1.00 55.83      A    O  \\nATOM   1238  N   VAL A 152      -6.002  26.663  -1.650  1.00 82.53      A    N  \\nATOM   1239  CA  VAL A 152      -6.769  25.424  -1.615  1.00 84.83      A    C  \\nATOM   1240  C   VAL A 152      -6.561  24.741  -0.275  1.00 85.19      A    C  \\nATOM   1241  O   VAL A 152      -5.416  24.513   0.150  1.00 83.27      A    O  \\nATOM   1242  CB  VAL A 152      -6.377  24.479  -2.757  1.00 84.64      A    C  \\nATOM   1243  CG1 VAL A 152      -6.764  25.108  -4.073  1.00 83.98      A    C  \\nATOM   1244  CG2 VAL A 152      -7.038  23.126  -2.598  1.00 83.44      A    C  \\nATOM   1245  N   THR A 153      -7.625  24.430   0.388  1.00 87.18      A    N  \\nATOM   1246  CA  THR A 153      -7.607  23.621   1.593  1.00 87.92      A    C  \\nATOM   1247  C   THR A 153      -7.969  22.204   1.201  1.00 89.21      A    C  \\nATOM   1248  O   THR A 153      -9.089  21.939   0.745  1.00 87.94      A    O  \\nATOM   1249  CB  THR A 153      -8.578  24.145   2.656  1.00 87.02      A    C  \\nATOM   1250  CG2 THR A 153      -8.255  25.567   3.017  1.00 81.51      A    C  \\nATOM   1251  OG1 THR A 153      -9.913  24.086   2.138  1.00 79.86      A    O  \\nATOM   1252  N   LEU A 154      -7.034  21.307   1.365  1.00 91.56      A    N  \\nATOM   1253  CA  LEU A 154      -7.232  19.917   0.976  1.00 92.21      A    C  \\nATOM   1254  C   LEU A 154      -7.784  19.121   2.150  1.00 92.23      A    C  \\nATOM   1255  O   LEU A 154      -7.202  19.128   3.247  1.00 91.14      A    O  \\nATOM   1256  CB  LEU A 154      -5.908  19.309   0.476  1.00 92.31      A    C  \\nATOM   1257  CG  LEU A 154      -5.989  17.867  -0.036  1.00 92.04      A    C  \\nATOM   1258  CD1 LEU A 154      -6.897  17.786  -1.244  1.00 90.43      A    C  \\nATOM   1259  CD2 LEU A 154      -4.599  17.365  -0.392  1.00 91.11      A    C  \\nATOM   1260  N   ILE A 155      -8.862  18.441   1.915  1.00 91.50      A    N  \\nATOM   1261  CA  ILE A 155      -9.539  17.645   2.928  1.00 91.33      A    C  \\nATOM   1262  C   ILE A 155      -9.552  16.205   2.445  1.00 91.73      A    C  \\nATOM   1263  O   ILE A 155     -10.050  15.906   1.365  1.00 90.75      A    O  \\nATOM   1264  CB  ILE A 155     -10.982  18.140   3.171  1.00 90.88      A    C  \\nATOM   1265  CG1 ILE A 155     -10.953  19.603   3.649  1.00 89.19      A    C  \\nATOM   1266  CG2 ILE A 155     -11.651  17.267   4.240  1.00 90.21      A    C  \\nATOM   1267  CD1 ILE A 155     -12.323  20.223   3.709  1.00 85.22      A    C  \\nATOM   1268  N   CYS A 156      -9.004  15.324   3.252  1.00 92.62      A    N  \\nATOM   1269  CA  CYS A 156      -8.925  13.906   2.912  1.00 92.63      A    C  \\nATOM   1270  C   CYS A 156      -9.872  13.109   3.780  1.00 91.83      A    C  \\nATOM   1271  O   CYS A 156      -9.778  13.135   5.023  1.00 90.21      A    O  \\nATOM   1272  CB  CYS A 156      -7.505  13.366   3.049  1.00 92.30      A    C  \\nATOM   1273  SG  CYS A 156      -6.389  14.048   1.796  1.00 89.41      A    S  \\nATOM   1274  N   GLY A 157     -10.765  12.395   3.167  1.00 92.79      A    N  \\nATOM   1275  CA  GLY A 157     -11.709  11.565   3.885  1.00 91.86      A    C  \\nATOM   1276  C   GLY A 157     -11.247  10.132   3.948  1.00 92.15      A    C  \\nATOM   1277  O   GLY A 157     -11.037   9.474   2.951  1.00 90.01      A    O  \\nATOM   1278  N   VAL A 158     -11.106   9.661   5.178  1.00 89.92      A    N  \\nATOM   1279  CA  VAL A 158     -10.694   8.293   5.457  1.00 89.88      A    C  \\nATOM   1280  C   VAL A 158     -11.920   7.502   5.864  1.00 89.40      A    C  \\nATOM   1281  O   VAL A 158     -12.498   7.762   6.920  1.00 88.13      A    O  \\nATOM   1282  CB  VAL A 158      -9.606   8.238   6.531  1.00 89.45      A    C  \\nATOM   1283  CG1 VAL A 158      -8.394   9.000   6.101  1.00 88.31      A    C  \\nATOM   1284  CG2 VAL A 158      -9.250   6.791   6.856  1.00 87.13      A    C  \\nATOM   1285  N   PRO A 159     -12.367   6.562   5.056  1.00 89.24      A    N  \\nATOM   1286  CA  PRO A 159     -13.495   5.731   5.454  1.00 88.03      A    C  \\nATOM   1287  C   PRO A 159     -13.111   4.730   6.522  1.00 87.59      A    C  \\nATOM   1288  O   PRO A 159     -11.948   4.449   6.740  1.00 86.09      A    O  \\nATOM   1289  CB  PRO A 159     -13.923   5.041   4.146  1.00 87.03      A    C  \\nATOM   1290  CG  PRO A 159     -12.666   4.975   3.365  1.00 86.59      A    C  \\nATOM   1291  CD  PRO A 159     -11.911   6.243   3.687  1.00 88.39      A    C  \\nATOM   1292  N   SER A 160     -14.125   4.210   7.189  1.00 86.04      A    N  \\nATOM   1293  CA  SER A 160     -13.918   3.266   8.260  1.00 84.76      A    C  \\nATOM   1294  C   SER A 160     -13.468   1.913   7.718  1.00 85.11      A    C  \\nATOM   1295  O   SER A 160     -13.533   1.650   6.524  1.00 83.56      A    O  \\nATOM   1296  CB  SER A 160     -15.164   3.114   9.106  1.00 83.09      A    C  \\nATOM   1297  OG  SER A 160     -16.195   2.521   8.350  1.00 77.61      A    O  \\nATOM   1298  N   ASP A 161     -13.028   1.062   8.634  1.00 84.56      A    N  \\nATOM   1299  CA  ASP A 161     -12.586  -0.260   8.230  1.00 82.69      A    C  \\nATOM   1300  C   ASP A 161     -13.757  -1.154   7.835  1.00 83.02      A    C  \\nATOM   1301  O   ASP A 161     -13.596  -2.060   7.015  1.00 82.14      A    O  \\nATOM   1302  CB  ASP A 161     -11.791  -0.923   9.337  1.00 81.93      A    C  \\nATOM   1303  CG  ASP A 161     -10.438  -0.264   9.582  1.00 82.07      A    C  \\nATOM   1304  OD1 ASP A 161      -9.860   0.278   8.604  1.00 78.05      A    O  \\nATOM   1305  OD2 ASP A 161      -9.958  -0.298  10.728  1.00 78.09      A    O1-\\nATOM   1306  N   LYS A 162     -14.928  -0.901   8.400  1.00 81.31      A    N  \\nATOM   1307  CA  LYS A 162     -16.093  -1.684   8.072  1.00 79.71      A    C  \\nATOM   1308  C   LYS A 162     -16.598  -1.407   6.666  1.00 80.10      A    C  \\nATOM   1309  O   LYS A 162     -17.043  -2.325   5.977  1.00 79.39      A    O  \\nATOM   1310  CB  LYS A 162     -17.213  -1.434   9.086  1.00 77.90      A    C  \\nATOM   1311  CG  LYS A 162     -16.896  -1.884  10.494  1.00 70.98      A    C  \\nATOM   1312  CD  LYS A 162     -18.065  -1.604  11.428  1.00 64.80      A    C  \\nATOM   1313  CE  LYS A 162     -17.699  -2.056  12.850  1.00 57.79      A    C  \\nATOM   1314  NZ  LYS A 162     -18.817  -1.839  13.794  1.00 49.52      A    N1+\\nATOM   1315  N   VAL A 163     -16.541  -0.166   6.252  1.00 79.83      A    N  \\nATOM   1316  CA  VAL A 163     -16.991   0.191   4.918  1.00 79.22      A    C  \\nATOM   1317  C   VAL A 163     -15.991  -0.293   3.876  1.00 79.26      A    C  \\nATOM   1318  O   VAL A 163     -16.378  -0.821   2.831  1.00 78.44      A    O  \\nATOM   1319  CB  VAL A 163     -17.214   1.714   4.821  1.00 78.13      A    C  \\nATOM   1320  CG1 VAL A 163     -18.215   2.172   5.850  1.00 75.73      A    C  \\nATOM   1321  CG2 VAL A 163     -17.680   2.103   3.430  1.00 76.30      A    C  \\nATOM   1322  N   THR A 164     -14.713  -0.143   4.147  1.00 82.87      A    N  \\nATOM   1323  CA  THR A 164     -13.692  -0.586   3.215  1.00 82.97      A    C  \\nATOM   1324  C   THR A 164     -13.628  -2.103   3.130  1.00 81.67      A    C  \\nATOM   1325  O   THR A 164     -13.347  -2.637   2.057  1.00 79.16      A    O  \\nATOM   1326  CB  THR A 164     -12.324  -0.036   3.578  1.00 83.99      A    C  \\nATOM   1327  CG2 THR A 164     -12.329   1.487   3.505  1.00 78.35      A    C  \\nATOM   1328  OG1 THR A 164     -11.983  -0.432   4.903  1.00 77.22      A    O  \\nATOM   1329  N   HIS A 165     -13.897  -2.793   4.246  1.00 82.37      A    N  \\nATOM   1330  CA  HIS A 165     -13.858  -4.241   4.210  1.00 80.31      A    C  \\nATOM   1331  C   HIS A 165     -15.080  -4.799   3.494  1.00 80.60      A    C  \\nATOM   1332  O   HIS A 165     -14.995  -5.866   2.880  1.00 78.79      A    O  \\nATOM   1333  CB  HIS A 165     -13.775  -4.811   5.632  1.00 78.58      A    C  \\nATOM   1334  CG  HIS A 165     -12.405  -4.607   6.266  1.00 67.88      A    C  \\nATOM   1335  CD2 HIS A 165     -11.294  -4.036   5.785  1.00 62.20      A    C  \\nATOM   1336  ND1 HIS A 165     -12.130  -5.039   7.539  1.00 61.80      A    N  \\nATOM   1337  CE1 HIS A 165     -10.864  -4.735   7.804  1.00 60.53      A    C  \\nATOM   1338  NE2 HIS A 165     -10.349  -4.133   6.773  1.00 57.65      A    N  \\nATOM   1339  N   LYS A 166     -16.197  -4.122   3.598  1.00 75.46      A    N  \\nATOM   1340  CA  LYS A 166     -17.412  -4.615   2.942  1.00 73.64      A    C  \\nATOM   1341  C   LYS A 166     -17.437  -4.279   1.467  1.00 74.27      A    C  \\nATOM   1342  O   LYS A 166     -17.835  -5.103   0.649  1.00 73.37      A    O  \\nATOM   1343  CB  LYS A 166     -18.641  -4.011   3.653  1.00 71.38      A    C  \\nATOM   1344  CG  LYS A 166     -19.950  -4.530   3.112  1.00 66.03      A    C  \\nATOM   1345  CD  LYS A 166     -21.110  -3.978   3.874  1.00 61.31      A    C  \\nATOM   1346  CE  LYS A 166     -22.437  -4.518   3.312  1.00 54.88      A    C  \\nATOM   1347  NZ  LYS A 166     -23.606  -3.970   4.020  1.00 48.47      A    N1+\\nATOM   1348  N   LEU A 167     -17.051  -3.080   1.118  1.00 76.15      A    N  \\nATOM   1349  CA  LEU A 167     -17.135  -2.656  -0.281  1.00 75.99      A    C  \\nATOM   1350  C   LEU A 167     -15.898  -3.059  -1.067  1.00 75.60      A    C  \\nATOM   1351  O   LEU A 167     -15.988  -3.776  -2.067  1.00 73.56      A    O  \\nATOM   1352  CB  LEU A 167     -17.331  -1.150  -0.367  1.00 74.83      A    C  \\nATOM   1353  CG  LEU A 167     -18.651  -0.607   0.234  1.00 74.33      A    C  \\nATOM   1354  CD1 LEU A 167     -19.851  -1.181  -0.486  1.00 73.24      A    C  \\nATOM   1355  CD2 LEU A 167     -18.647   0.903   0.193  1.00 72.64      A    C  \\nATOM   1356  N   LYS A 168     -14.753  -2.599  -0.616  1.00 76.58      A    N  \\nATOM   1357  CA  LYS A 168     -13.519  -2.846  -1.377  1.00 76.65      A    C  \\nATOM   1358  C   LYS A 168     -12.903  -4.179  -0.972  1.00 75.98      A    C  \\nATOM   1359  O   LYS A 168     -12.835  -5.106  -1.775  1.00 72.79      A    O  \\nATOM   1360  CB  LYS A 168     -12.524  -1.715  -1.181  1.00 74.98      A    C  \\nATOM   1361  CG  LYS A 168     -11.259  -1.842  -1.993  1.00 72.18      A    C  \\nATOM   1362  CD  LYS A 168     -10.374  -0.610  -1.802  1.00 70.76      A    C  \\nATOM   1363  CE  LYS A 168      -9.100  -0.702  -2.608  1.00 68.33      A    C  \\nATOM   1364  NZ  LYS A 168      -8.228   0.483  -2.452  1.00 61.91      A    N1+\\nATOM   1365  N   GLY A 169     -12.420  -4.258   0.239  1.00 79.12      A    N  \\nATOM   1366  CA  GLY A 169     -11.778  -5.465   0.740  1.00 78.46      A    C  \\nATOM   1367  C   GLY A 169     -11.047  -5.215   2.041  1.00 80.44      A    C  \\nATOM   1368  O   GLY A 169     -11.309  -4.237   2.738  1.00 78.57      A    O  \\nATOM   1369  N   LEU A 170     -10.130  -6.092   2.343  1.00 80.33      A    N  \\nATOM   1370  CA  LEU A 170      -9.355  -5.978   3.554  1.00 80.41      A    C  \\nATOM   1371  C   LEU A 170      -8.309  -4.902   3.407  1.00 82.57      A    C  \\nATOM   1372  O   LEU A 170      -7.775  -4.657   2.316  1.00 81.24      A    O  \\nATOM   1373  CB  LEU A 170      -8.690  -7.299   3.914  1.00 77.63      A    C  \\nATOM   1374  CG  LEU A 170      -9.621  -8.448   4.273  1.00 74.44      A    C  \\nATOM   1375  CD1 LEU A 170     -10.404  -8.133   5.545  1.00 73.97      A    C  \\nATOM   1376  CD2 LEU A 170      -8.846  -9.751   4.430  1.00 74.11      A    C  \\nATOM   1377  N   THR A 171      -7.959  -4.279   4.517  1.00 83.12      A    N  \\nATOM   1378  CA  THR A 171      -6.949  -3.233   4.577  1.00 84.53      A    C  \\nATOM   1379  C   THR A 171      -5.846  -3.647   5.506  1.00 84.53      A    C  \\nATOM   1380  O   THR A 171      -6.077  -4.352   6.500  1.00 82.07      A    O  \\nATOM   1381  CB  THR A 171      -7.528  -1.895   5.019  1.00 84.21      A    C  \\nATOM   1382  CG2 THR A 171      -8.598  -1.452   4.037  1.00 81.05      A    C  \\nATOM   1383  OG1 THR A 171      -8.118  -2.018   6.337  1.00 80.44      A    O  \\nATOM   1384  N   VAL A 172      -4.643  -3.218   5.214  1.00 87.78      A    N  \\nATOM   1385  CA  VAL A 172      -3.493  -3.558   6.040  1.00 86.63      A    C  \\nATOM   1386  C   VAL A 172      -3.404  -2.653   7.233  1.00 86.59      A    C  \\nATOM   1387  O   VAL A 172      -3.414  -3.087   8.389  1.00 83.49      A    O  \\nATOM   1388  CB  VAL A 172      -2.195  -3.502   5.212  1.00 85.49      A    C  \\nATOM   1389  CG1 VAL A 172      -2.260  -4.475   4.036  1.00 84.89      A    C  \\nATOM   1390  CG2 VAL A 172      -0.992  -3.825   6.088  1.00 85.04      A    C  \\nATOM   1391  N   LEU A 173      -3.367  -1.347   6.994  1.00 87.57      A    N  \\nATOM   1392  CA  LEU A 173      -3.343  -0.372   8.072  1.00 87.49      A    C  \\nATOM   1393  C   LEU A 173      -4.758  -0.121   8.593  1.00 88.05      A    C  \\nATOM   1394  O   LEU A 173      -5.736  -0.281   7.852  1.00 86.07      A    O  \\nATOM   1395  CB  LEU A 173      -2.702   0.940   7.624  1.00 87.60      A    C  \\nATOM   1396  CG  LEU A 173      -1.234   0.907   7.203  1.00 88.16      A    C  \\nATOM   1397  CD1 LEU A 173      -0.353   0.550   8.390  1.00 84.56      A    C  \\nATOM   1398  CD2 LEU A 173      -0.825   2.261   6.640  1.00 85.93      A    C  \\nATOM   1399  N   THR A 174      -4.843   0.264   9.823  1.00 85.45      A    N  \\nATOM   1400  CA  THR A 174      -6.137   0.595  10.430  1.00 83.75      A    C  \\nATOM   1401  C   THR A 174      -6.536   2.016   9.990  1.00 84.57      A    C  \\nATOM   1402  O   THR A 174      -5.766   2.743   9.391  1.00 84.43      A    O  \\nATOM   1403  CB  THR A 174      -6.083   0.508  11.931  1.00 82.60      A    C  \\nATOM   1404  CG2 THR A 174      -5.663  -0.906  12.348  1.00 77.85      A    C  \\nATOM   1405  OG1 THR A 174      -5.131   1.442  12.436  1.00 78.51      A    O  \\nATOM   1406  N   ASP A 175      -7.777   2.355  10.317  1.00 85.48      A    N  \\nATOM   1407  CA  ASP A 175      -8.257   3.685   9.956  1.00 83.74      A    C  \\nATOM   1408  C   ASP A 175      -7.586   4.771  10.762  1.00 84.54      A    C  \\nATOM   1409  O   ASP A 175      -7.387   5.883  10.272  1.00 84.25      A    O  \\nATOM   1410  CB  ASP A 175      -9.780   3.747  10.114  1.00 81.70      A    C  \\nATOM   1411  CG  ASP A 175     -10.240   3.337  11.493  1.00 79.72      A    C  \\nATOM   1412  OD1 ASP A 175      -9.394   2.967  12.337  1.00 73.50      A    O  \\nATOM   1413  OD2 ASP A 175     -11.470   3.366  11.751  1.00 74.71      A    O1-\\nATOM   1414  N   LYS A 176      -7.234   4.469  11.986  1.00 84.08      A    N  \\nATOM   1415  CA  LYS A 176      -6.519   5.442  12.808  1.00 83.24      A    C  \\nATOM   1416  C   LYS A 176      -5.098   5.660  12.277  1.00 84.61      A    C  \\nATOM   1417  O   LYS A 176      -4.584   6.775  12.371  1.00 84.14      A    O  \\nATOM   1418  CB  LYS A 176      -6.480   4.991  14.255  1.00 81.44      A    C  \\nATOM   1419  CG  LYS A 176      -7.847   4.951  14.918  1.00 73.20      A    C  \\nATOM   1420  CD  LYS A 176      -7.750   4.483  16.341  1.00 67.46      A    C  \\nATOM   1421  CE  LYS A 176      -9.132   4.471  16.979  1.00 58.90      A    C  \\nATOM   1422  NZ  LYS A 176      -9.091   4.027  18.383  1.00 50.89      A    N1+\\nATOM   1423  N   GLN A 177      -4.497   4.630  11.748  1.00 86.50      A    N  \\nATOM   1424  CA  GLN A 177      -3.189   4.739  11.161  1.00 85.85      A    C  \\nATOM   1425  C   GLN A 177      -3.250   5.435   9.804  1.00 86.62      A    C  \\nATOM   1426  O   GLN A 177      -2.313   6.150   9.435  1.00 87.49      A    O  \\nATOM   1427  CB  GLN A 177      -2.552   3.370  11.023  1.00 86.05      A    C  \\nATOM   1428  CG  GLN A 177      -2.277   2.711  12.348  1.00 85.87      A    C  \\nATOM   1429  CD  GLN A 177      -1.674   1.332  12.198  1.00 83.49      A    C  \\nATOM   1430  NE2 GLN A 177      -1.330   0.718  13.304  1.00 78.61      A    N  \\nATOM   1431  OE1 GLN A 177      -1.525   0.823  11.083  1.00 81.80      A    O  \\nATOM   1432  N   ARG A 178      -4.329   5.237   9.086  1.00 90.06      A    N  \\nATOM   1433  CA  ARG A 178      -4.496   5.933   7.810  1.00 90.15      A    C  \\nATOM   1434  C   ARG A 178      -4.748   7.401   8.028  1.00 90.20      A    C  \\nATOM   1435  O   ARG A 178      -4.364   8.218   7.214  1.00 90.42      A    O  \\nATOM   1436  CB  ARG A 178      -5.632   5.292   6.983  1.00 90.15      A    C  \\nATOM   1437  CG  ARG A 178      -5.302   3.903   6.495  1.00 89.48      A    C  \\nATOM   1438  CD  ARG A 178      -6.460   3.347   5.686  1.00 89.06      A    C  \\nATOM   1439  NE  ARG A 178      -7.618   3.056   6.520  1.00 85.53      A    N  \\nATOM   1440  CZ  ARG A 178      -8.811   2.736   6.026  1.00 88.28      A    C  \\nATOM   1441  NH1 ARG A 178      -8.986   2.696   4.718  1.00 76.61      A    N1+\\nATOM   1442  NH2 ARG A 178      -9.811   2.490   6.835  1.00 79.32      A    N  \\nATOM   1443  N   CYS A 179      -5.400   7.754   9.125  1.00 89.54      A    N  \\nATOM   1444  CA  CYS A 179      -5.587   9.153   9.446  1.00 89.16      A    C  \\nATOM   1445  C   CYS A 179      -4.287   9.778   9.954  1.00 88.94      A    C  \\nATOM   1446  O   CYS A 179      -4.054  10.965   9.743  1.00 87.70      A    O  \\nATOM   1447  CB  CYS A 179      -6.683   9.324  10.489  1.00 88.00      A    C  \\nATOM   1448  SG  CYS A 179      -8.321   8.881   9.894  1.00 84.39      A    S  \\nATOM   1449  N   GLU A 180      -3.460   8.977  10.594  1.00 87.08      A    N  \\nATOM   1450  CA  GLU A 180      -2.195   9.514  11.077  1.00 85.28      A    C  \\nATOM   1451  C   GLU A 180      -1.218   9.795   9.955  1.00 85.96      A    C  \\nATOM   1452  O   GLU A 180      -0.477  10.774  10.001  1.00 85.40      A    O  \\nATOM   1453  CB  GLU A 180      -1.575   8.568  12.105  1.00 83.58      A    C  \\nATOM   1454  CG  GLU A 180      -0.326   9.121  12.786  1.00 75.21      A    C  \\nATOM   1455  CD  GLU A 180       0.221   8.179  13.838  1.00 70.27      A    C  \\nATOM   1456  OE1 GLU A 180      -0.365   7.097  14.040  1.00 66.78      A    O  \\nATOM   1457  OE2 GLU A 180       1.239   8.543  14.495  1.00 61.00      A    O1-\\nATOM   1458  N   THR A 181      -1.223   8.965   8.933  1.00 89.58      A    N  \\nATOM   1459  CA  THR A 181      -0.342   9.166   7.795  1.00 89.51      A    C  \\nATOM   1460  C   THR A 181      -0.766  10.380   6.962  1.00 89.79      A    C  \\nATOM   1461  O   THR A 181       0.080  10.966   6.297  1.00 88.37      A    O  \\nATOM   1462  CB  THR A 181      -0.286   7.925   6.912  1.00 89.39      A    C  \\nATOM   1463  CG2 THR A 181       0.329   6.777   7.645  1.00 83.92      A    C  \\nATOM   1464  OG1 THR A 181      -1.608   7.564   6.500  1.00 81.65      A    O  \\nATOM   1465  N   LEU A 182      -2.019  10.736   7.018  1.00 90.72      A    N  \\nATOM   1466  CA  LEU A 182      -2.469  11.917   6.305  1.00 90.50      A    C  \\nATOM   1467  C   LEU A 182      -2.239  13.202   7.073  1.00 89.52      A    C  \\nATOM   1468  O   LEU A 182      -2.143  14.264   6.483  1.00 87.86      A    O  \\nATOM   1469  CB  LEU A 182      -3.942  11.799   5.937  1.00 91.21      A    C  \\nATOM   1470  CG  LEU A 182      -4.282  10.732   4.904  1.00 91.77      A    C  \\nATOM   1471  CD1 LEU A 182      -3.618  11.068   3.582  1.00 91.16      A    C  \\nATOM   1472  CD2 LEU A 182      -5.778  10.619   4.722  1.00 91.41      A    C  \\nATOM   1473  N   THR A 183      -2.143  13.112   8.403  1.00 87.90      A    N  \\nATOM   1474  CA  THR A 183      -1.873  14.325   9.194  1.00 84.82      A    C  \\nATOM   1475  C   THR A 183      -0.438  14.784   9.013  1.00 83.79      A    C  \\nATOM   1476  O   THR A 183      -0.170  15.977   9.046  1.00 81.54      A    O  \\nATOM   1477  CB  THR A 183      -2.158  14.079  10.671  1.00 83.29      A    C  \\nATOM   1478  CG2 THR A 183      -3.635  13.849  10.869  1.00 72.50      A    C  \\nATOM   1479  OG1 THR A 183      -1.428  12.961  11.147  1.00 74.45      A    O  \\nATOM   1480  N   HIS A 184       0.471  13.845   8.843  1.00 86.15      A    N  \\nATOM   1481  CA  HIS A 184       1.869  14.189   8.649  1.00 84.17      A    C  \\nATOM   1482  C   HIS A 184       2.201  14.491   7.211  1.00 85.17      A    C  \\nATOM   1483  O   HIS A 184       3.349  14.793   6.894  1.00 83.07      A    O  \\nATOM   1484  CB  HIS A 184       2.761  13.053   9.162  1.00 82.17      A    C  \\nATOM   1485  CG  HIS A 184       2.710  12.850  10.642  1.00 80.30      A    C  \\nATOM   1486  CD2 HIS A 184       2.128  11.859  11.374  1.00 75.43      A    C  \\nATOM   1487  ND1 HIS A 184       3.268  13.718  11.537  1.00 72.82      A    N  \\nATOM   1488  CE1 HIS A 184       3.050  13.288  12.773  1.00 73.48      A    C  \\nATOM   1489  NE2 HIS A 184       2.362  12.162  12.699  1.00 74.05      A    N  \\nATOM   1490  N   CYS A 185       1.216  14.430   6.336  1.00 87.29      A    N  \\nATOM   1491  CA  CYS A 185       1.445  14.768   4.935  1.00 86.96      A    C  \\nATOM   1492  C   CYS A 185       1.542  16.277   4.778  1.00 86.14      A    C  \\nATOM   1493  O   CYS A 185       1.073  17.050   5.623  1.00 82.61      A    O  \\nATOM   1494  CB  CYS A 185       0.334  14.205   4.049  1.00 86.47      A    C  \\nATOM   1495  SG  CYS A 185       0.588  14.526   2.291  1.00 83.47      A    S  \\nATOM   1496  N   ARG A 186       2.179  16.701   3.731  1.00 84.99      A    N  \\nATOM   1497  CA  ARG A 186       2.437  18.114   3.543  1.00 82.37      A    C  \\nATOM   1498  C   ARG A 186       1.219  18.868   3.005  1.00 81.96      A    C  \\nATOM   1499  O   ARG A 186       0.888  19.951   3.460  1.00 79.74      A    O  \\nATOM   1500  CB  ARG A 186       3.617  18.331   2.576  1.00 79.26      A    C  \\nATOM   1501  CG  ARG A 186       4.020  19.766   2.376  1.00 71.01      A    C  \\nATOM   1502  CD  ARG A 186       5.227  19.890   1.474  1.00 64.21      A    C  \\nATOM   1503  NE  ARG A 186       5.607  21.267   1.222  1.00 54.81      A    N  \\nATOM   1504  CZ  ARG A 186       6.631  21.638   0.472  1.00 56.29      A    C  \\nATOM   1505  NH1 ARG A 186       7.411  20.700  -0.088  1.00 44.71      A    N1+\\nATOM   1506  NH2 ARG A 186       6.913  22.924   0.311  1.00 45.10      A    N  \\nATOM   1507  N   TRP A 187       0.579  18.287   2.003  1.00 86.10      A    N  \\nATOM   1508  CA  TRP A 187      -0.510  18.972   1.320  1.00 85.20      A    C  \\nATOM   1509  C   TRP A 187      -1.835  18.773   1.991  1.00 86.44      A    C  \\nATOM   1510  O   TRP A 187      -2.800  19.484   1.650  1.00 83.49      A    O  \\nATOM   1511  CB  TRP A 187      -0.562  18.486  -0.137  1.00 83.66      A    C  \\nATOM   1512  CG  TRP A 187       0.676  18.829  -0.894  1.00 85.39      A    C  \\nATOM   1513  CD1 TRP A 187       0.985  20.013  -1.470  1.00 81.73      A    C  \\nATOM   1514  CD2 TRP A 187       1.799  17.975  -1.152  1.00 83.72      A    C  \\nATOM   1515  CE2 TRP A 187       2.721  18.689  -1.899  1.00 82.38      A    C  \\nATOM   1516  CE3 TRP A 187       2.073  16.650  -0.833  1.00 83.27      A    C  \\nATOM   1517  NE1 TRP A 187       2.209  19.936  -2.083  1.00 77.31      A    N  \\nATOM   1518  CZ2 TRP A 187       3.922  18.138  -2.323  1.00 76.97      A    C  \\nATOM   1519  CZ3 TRP A 187       3.269  16.122  -1.253  1.00 76.67      A    C  \\nATOM   1520  CH2 TRP A 187       4.177  16.845  -1.983  1.00 74.46      A    C  \\nATOM   1521  N   VAL A 188      -1.935  17.840   2.920  1.00 88.29      A    N  \\nATOM   1522  CA  VAL A 188      -3.202  17.582   3.587  1.00 89.02      A    C  \\nATOM   1523  C   VAL A 188      -3.400  18.573   4.713  1.00 88.00      A    C  \\nATOM   1524  O   VAL A 188      -2.514  18.737   5.566  1.00 85.18      A    O  \\nATOM   1525  CB  VAL A 188      -3.279  16.138   4.099  1.00 89.85      A    C  \\nATOM   1526  CG1 VAL A 188      -3.221  15.186   2.919  1.00 88.20      A    C  \\nATOM   1527  CG2 VAL A 188      -4.569  15.913   4.858  1.00 88.40      A    C  \\nATOM   1528  N   ASP A 189      -4.530  19.237   4.749  1.00 87.20      A    N  \\nATOM   1529  CA  ASP A 189      -4.820  20.213   5.785  1.00 85.15      A    C  \\nATOM   1530  C   ASP A 189      -5.761  19.640   6.824  1.00 85.26      A    C  \\nATOM   1531  O   ASP A 189      -5.481  19.714   8.020  1.00 82.81      A    O  \\nATOM   1532  CB  ASP A 189      -5.406  21.481   5.171  1.00 83.25      A    C  \\nATOM   1533  CG  ASP A 189      -4.412  22.208   4.296  1.00 82.90      A    C  \\nATOM   1534  OD1 ASP A 189      -3.216  22.201   4.622  1.00 79.43      A    O  \\nATOM   1535  OD2 ASP A 189      -4.842  22.788   3.279  1.00 78.91      A    O1-\\nATOM   1536  N   GLU A 190      -6.877  19.089   6.377  1.00 87.68      A    N  \\nATOM   1537  CA  GLU A 190      -7.856  18.539   7.301  1.00 86.74      A    C  \\nATOM   1538  C   GLU A 190      -8.151  17.105   6.916  1.00 87.39      A    C  \\nATOM   1539  O   GLU A 190      -8.316  16.786   5.748  1.00 87.16      A    O  \\nATOM   1540  CB  GLU A 190      -9.149  19.350   7.281  1.00 85.49      A    C  \\nATOM   1541  CG  GLU A 190      -9.001  20.773   7.802  1.00 76.93      A    C  \\nATOM   1542  CD  GLU A 190     -10.295  21.573   7.730  1.00 76.27      A    C  \\nATOM   1543  OE1 GLU A 190     -11.314  21.024   7.264  1.00 73.32      A    O  \\nATOM   1544  OE2 GLU A 190     -10.290  22.754   8.156  1.00 66.56      A    O1-\\nATOM   1545  N   VAL A 191      -8.237  16.251   7.920  1.00 86.82      A    N  \\nATOM   1546  CA  VAL A 191      -8.531  14.829   7.713  1.00 86.96      A    C  \\nATOM   1547  C   VAL A 191      -9.853  14.520   8.395  1.00 86.59      A    C  \\nATOM   1548  O   VAL A 191      -9.995  14.681   9.591  1.00 83.88      A    O  \\nATOM   1549  CB  VAL A 191      -7.402  13.931   8.256  1.00 85.82      A    C  \\nATOM   1550  CG1 VAL A 191      -6.115  14.184   7.526  1.00 85.25      A    C  \\nATOM   1551  CG2 VAL A 191      -7.817  12.479   8.160  1.00 85.69      A    C  \\nATOM   1552  N   ILE A 192     -10.802  14.076   7.600  1.00 87.34      A    N  \\nATOM   1553  CA  ILE A 192     -12.107  13.695   8.115  1.00 85.98      A    C  \\nATOM   1554  C   ILE A 192     -12.090  12.179   8.318  1.00 85.72      A    C  \\nATOM   1555  O   ILE A 192     -12.022  11.420   7.347  1.00 84.60      A    O  \\nATOM   1556  CB  ILE A 192     -13.234  14.109   7.173  1.00 85.16      A    C  \\nATOM   1557  CG1 ILE A 192     -13.266  15.640   7.029  1.00 84.61      A    C  \\nATOM   1558  CG2 ILE A 192     -14.585  13.603   7.677  1.00 84.48      A    C  \\nATOM   1559  CD1 ILE A 192     -14.217  16.157   6.001  1.00 81.32      A    C  \\nATOM   1560  N   ALA A 193     -12.182  11.772   9.556  1.00 85.23      A    N  \\nATOM   1561  CA  ALA A 193     -12.240  10.342   9.873  1.00 84.18      A    C  \\nATOM   1562  C   ALA A 193     -13.676   9.846   9.734  1.00 84.39      A    C  \\nATOM   1563  O   ALA A 193     -14.617  10.613   9.879  1.00 81.41      A    O  \\nATOM   1564  CB  ALA A 193     -11.705  10.063  11.264  1.00 82.42      A    C  \\nATOM   1565  N   ASN A 194     -13.791   8.568   9.454  1.00 83.83      A    N  \\nATOM   1566  CA  ASN A 194     -15.088   7.915   9.246  1.00 83.84      A    C  \\nATOM   1567  C   ASN A 194     -15.857   8.594   8.116  1.00 84.32      A    C  \\nATOM   1568  O   ASN A 194     -17.016   9.002   8.280  1.00 81.98      A    O  \\nATOM   1569  CB  ASN A 194     -15.918   7.857  10.528  1.00 81.26      A    C  \\nATOM   1570  CG  ASN A 194     -15.292   6.972  11.585  1.00 76.81      A    C  \\nATOM   1571  ND2 ASN A 194     -15.196   7.485  12.795  1.00 71.01      A    N  \\nATOM   1572  OD1 ASN A 194     -14.909   5.843  11.307  1.00 72.24      A    O  \\nATOM   1573  N   ALA A 195     -15.227   8.693   6.983  1.00 88.10      A    N  \\nATOM   1574  CA  ALA A 195     -15.852   9.304   5.829  1.00 88.35      A    C  \\nATOM   1575  C   ALA A 195     -16.700   8.282   5.089  1.00 87.69      A    C  \\nATOM   1576  O   ALA A 195     -16.347   7.089   5.057  1.00 85.74      A    O  \\nATOM   1577  CB  ALA A 195     -14.789   9.871   4.886  1.00 88.36      A    C  \\nATOM   1578  N   PRO A 196     -17.805   8.689   4.490  1.00 87.77      A    N  \\nATOM   1579  CA  PRO A 196     -18.629   7.740   3.744  1.00 87.14      A    C  \\nATOM   1580  C   PRO A 196     -17.972   7.353   2.434  1.00 86.65      A    C  \\nATOM   1581  O   PRO A 196     -17.161   8.093   1.888  1.00 85.00      A    O  \\nATOM   1582  CB  PRO A 196     -19.932   8.495   3.524  1.00 85.59      A    C  \\nATOM   1583  CG  PRO A 196     -19.503   9.913   3.512  1.00 85.06      A    C  \\nATOM   1584  CD  PRO A 196     -18.404  10.023   4.530  1.00 87.73      A    C  \\nATOM   1585  N   TRP A 197     -18.352   6.180   1.944  1.00 82.94      A    N  \\nATOM   1586  CA  TRP A 197     -17.788   5.690   0.694  1.00 82.48      A    C  \\nATOM   1587  C   TRP A 197     -18.353   6.473  -0.504  1.00 82.14      A    C  \\nATOM   1588  O   TRP A 197     -17.611   6.837  -1.409  1.00 79.73      A    O  \\nATOM   1589  CB  TRP A 197     -18.054   4.208   0.536  1.00 81.32      A    C  \\nATOM   1590  CG  TRP A 197     -17.468   3.601  -0.692  1.00 80.60      A    C  \\nATOM   1591  CD1 TRP A 197     -18.107   3.352  -1.863  1.00 78.21      A    C  \\nATOM   1592  CD2 TRP A 197     -16.106   3.145  -0.873  1.00 80.26      A    C  \\nATOM   1593  CE2 TRP A 197     -16.001   2.648  -2.168  1.00 79.08      A    C  \\nATOM   1594  CE3 TRP A 197     -14.982   3.144  -0.057  1.00 79.13      A    C  \\nATOM   1595  NE1 TRP A 197     -17.245   2.784  -2.759  1.00 76.66      A    N  \\nATOM   1596  CZ2 TRP A 197     -14.790   2.117  -2.666  1.00 75.18      A    C  \\nATOM   1597  CZ3 TRP A 197     -13.790   2.632  -0.542  1.00 74.96      A    C  \\nATOM   1598  CH2 TRP A 197     -13.708   2.098  -1.864  1.00 74.24      A    C  \\nATOM   1599  N   CYS A 198     -19.650   6.690  -0.485  1.00 83.80      A    N  \\nATOM   1600  CA  CYS A 198     -20.313   7.441  -1.541  1.00 83.95      A    C  \\nATOM   1601  C   CYS A 198     -20.457   8.902  -1.104  1.00 84.54      A    C  \\nATOM   1602  O   CYS A 198     -21.059   9.188  -0.085  1.00 82.46      A    O  \\nATOM   1603  CB  CYS A 198     -21.681   6.857  -1.871  1.00 81.19      A    C  \\nATOM   1604  SG  CYS A 198     -21.565   5.217  -2.603  1.00 76.12      A    S  \\nATOM   1605  N   VAL A 199     -19.885   9.800  -1.901  1.00 86.66      A    N  \\nATOM   1606  CA  VAL A 199     -19.926  11.212  -1.581  1.00 86.82      A    C  \\nATOM   1607  C   VAL A 199     -21.280  11.765  -1.973  1.00 86.27      A    C  \\nATOM   1608  O   VAL A 199     -21.634  11.852  -3.156  1.00 83.44      A    O  \\nATOM   1609  CB  VAL A 199     -18.801  11.959  -2.321  1.00 86.14      A    C  \\nATOM   1610  CG1 VAL A 199     -17.446  11.429  -1.877  1.00 81.45      A    C  \\nATOM   1611  CG2 VAL A 199     -18.873  13.448  -2.004  1.00 79.76      A    C  \\nATOM   1612  N   THR A 200     -22.046  12.142  -0.988  1.00 86.90      A    N  \\nATOM   1613  CA  THR A 200     -23.381  12.696  -1.190  1.00 86.51      A    C  \\nATOM   1614  C   THR A 200     -23.313  14.224  -1.218  1.00 87.40      A    C  \\nATOM   1615  O   THR A 200     -22.373  14.818  -0.696  1.00 87.05      A    O  \\nATOM   1616  CB  THR A 200     -24.334  12.231  -0.097  1.00 84.29      A    C  \\nATOM   1617  CG2 THR A 200     -24.414  10.720  -0.063  1.00 80.71      A    C  \\nATOM   1618  OG1 THR A 200     -23.869  12.702   1.188  1.00 81.85      A    O  \\nATOM   1619  N   PRO A 201     -24.280  14.875  -1.842  1.00 87.39      A    N  \\nATOM   1620  CA  PRO A 201     -24.291  16.328  -1.822  1.00 86.47      A    C  \\nATOM   1621  C   PRO A 201     -24.592  16.935  -0.465  1.00 87.93      A    C  \\nATOM   1622  O   PRO A 201     -24.276  18.106  -0.222  1.00 86.83      A    O  \\nATOM   1623  CB  PRO A 201     -25.371  16.719  -2.856  1.00 84.20      A    C  \\nATOM   1624  CG  PRO A 201     -26.213  15.520  -2.935  1.00 82.80      A    C  \\nATOM   1625  CD  PRO A 201     -25.312  14.354  -2.785  1.00 85.76      A    C  \\nATOM   1626  N   GLU A 202     -25.179  16.152   0.434  1.00 87.07      A    N  \\nATOM   1627  CA  GLU A 202     -25.373  16.641   1.798  1.00 86.60      A    C  \\nATOM   1628  C   GLU A 202     -24.046  16.697   2.547  1.00 87.98      A    C  \\nATOM   1629  O   GLU A 202     -23.836  17.575   3.387  1.00 87.22      A    O  \\nATOM   1630  CB  GLU A 202     -26.378  15.754   2.534  1.00 85.00      A    C  \\nATOM   1631  CG  GLU A 202     -26.743  16.234   3.923  1.00 75.68      A    C  \\nATOM   1632  CD  GLU A 202     -27.782  15.352   4.582  1.00 65.34      A    C  \\nATOM   1633  OE1 GLU A 202     -28.175  14.323   3.963  1.00 60.44      A    O  \\nATOM   1634  OE2 GLU A 202     -28.205  15.670   5.719  1.00 56.28      A    O1-\\nATOM   1635  N   PHE A 203     -23.167  15.777   2.253  1.00 88.92      A    N  \\nATOM   1636  CA  PHE A 203     -21.849  15.779   2.846  1.00 89.02      A    C  \\nATOM   1637  C   PHE A 203     -20.997  16.945   2.313  1.00 89.17      A    C  \\nATOM   1638  O   PHE A 203     -20.155  17.475   3.035  1.00 88.50      A    O  \\nATOM   1639  CB  PHE A 203     -21.165  14.453   2.565  1.00 89.56      A    C  \\nATOM   1640  CG  PHE A 203     -19.780  14.353   3.139  1.00 90.87      A    C  \\nATOM   1641  CD1 PHE A 203     -19.598  14.148   4.508  1.00 89.66      A    C  \\nATOM   1642  CD2 PHE A 203     -18.650  14.451   2.322  1.00 90.28      A    C  \\nATOM   1643  CE1 PHE A 203     -18.345  14.045   5.045  1.00 88.31      A    C  \\nATOM   1644  CE2 PHE A 203     -17.376  14.347   2.878  1.00 89.69      A    C  \\nATOM   1645  CZ  PHE A 203     -17.225  14.132   4.229  1.00 88.98      A    C  \\nATOM   1646  N   LEU A 204     -21.266  17.349   1.101  1.00 92.16      A    N  \\nATOM   1647  CA  LEU A 204     -20.535  18.483   0.535  1.00 91.71      A    C  \\nATOM   1648  C   LEU A 204     -20.967  19.790   1.180  1.00 91.19      A    C  \\nATOM   1649  O   LEU A 204     -20.149  20.694   1.366  1.00 90.00      A    O  \\nATOM   1650  CB  LEU A 204     -20.760  18.552  -0.974  1.00 91.33      A    C  \\nATOM   1651  CG  LEU A 204     -20.242  17.389  -1.825  1.00 90.82      A    C  \\nATOM   1652  CD1 LEU A 204     -18.749  17.267  -1.683  1.00 89.00      A    C  \\nATOM   1653  CD2 LEU A 204     -20.655  17.569  -3.254  1.00 89.24      A    C  \\nATOM   1654  N   LYS A 205     -22.242  19.896   1.511  1.00 89.64      A    N  \\nATOM   1655  CA  LYS A 205     -22.747  21.147   2.099  1.00 88.13      A    C  \\nATOM   1656  C   LYS A 205     -22.404  21.270   3.550  1.00 87.61      A    C  \\nATOM   1657  O   LYS A 205     -22.229  22.386   4.061  1.00 85.64      A    O  \\nATOM   1658  CB  LYS A 205     -24.257  21.243   1.891  1.00 87.91      A    C  \\nATOM   1659  CG  LYS A 205     -24.699  21.397   0.439  1.00 76.74      A    C  \\nATOM   1660  CD  LYS A 205     -26.197  21.482   0.328  1.00 71.72      A    C  \\nATOM   1661  CE  LYS A 205     -26.644  21.629  -1.130  1.00 61.47      A    C  \\nATOM   1662  NZ  LYS A 205     -28.114  21.676  -1.267  1.00 53.66      A    N1+\\nATOM   1663  N   ASP A 206     -22.335  20.147   4.263  1.00 87.92      A    N  \\nATOM   1664  CA  ASP A 206     -22.028  20.206   5.677  1.00 86.03      A    C  \\nATOM   1665  C   ASP A 206     -20.569  20.555   5.922  1.00 86.68      A    C  \\nATOM   1666  O   ASP A 206     -20.251  21.258   6.876  1.00 84.29      A    O  \\nATOM   1667  CB  ASP A 206     -22.372  18.877   6.338  1.00 85.05      A    C  \\nATOM   1668  CG  ASP A 206     -23.857  18.630   6.387  1.00 80.68      A    C  \\nATOM   1669  OD1 ASP A 206     -24.626  19.597   6.479  1.00 75.53      A    O  \\nATOM   1670  OD2 ASP A 206     -24.254  17.456   6.347  1.00 75.78      A    O1-\\nATOM   1671  N   HIS A 207     -19.667  20.080   5.065  1.00 87.51      A    N  \\nATOM   1672  CA  HIS A 207     -18.264  20.344   5.231  1.00 86.38      A    C  \\nATOM   1673  C   HIS A 207     -17.752  21.454   4.310  1.00 86.55      A    C  \\nATOM   1674  O   HIS A 207     -16.577  21.772   4.332  1.00 86.14      A    O  \\nATOM   1675  CB  HIS A 207     -17.481  19.057   4.944  1.00 85.80      A    C  \\nATOM   1676  CG  HIS A 207     -17.699  18.004   5.978  1.00 86.36      A    C  \\nATOM   1677  CD2 HIS A 207     -17.026  17.680   7.086  1.00 79.56      A    C  \\nATOM   1678  ND1 HIS A 207     -18.782  17.148   5.901  1.00 79.11      A    N  \\nATOM   1679  CE1 HIS A 207     -18.745  16.327   6.926  1.00 79.94      A    C  \\nATOM   1680  NE2 HIS A 207     -17.689  16.599   7.650  1.00 80.95      A    N  \\nATOM   1681  N   LYS A 208     -18.656  22.053   3.556  1.00 88.59      A    N  \\nATOM   1682  CA  LYS A 208     -18.354  23.143   2.645  1.00 88.21      A    C  \\nATOM   1683  C   LYS A 208     -17.261  22.751   1.640  1.00 89.43      A    C  \\nATOM   1684  O   LYS A 208     -16.235  23.404   1.527  1.00 88.16      A    O  \\nATOM   1685  CB  LYS A 208     -17.979  24.409   3.393  1.00 85.92      A    C  \\nATOM   1686  CG  LYS A 208     -19.122  24.994   4.210  1.00 81.78      A    C  \\nATOM   1687  CD  LYS A 208     -18.681  26.209   4.977  1.00 75.61      A    C  \\nATOM   1688  CE  LYS A 208     -19.818  26.789   5.818  1.00 69.90      A    C  \\nATOM   1689  NZ  LYS A 208     -19.391  27.935   6.635  1.00 59.63      A    N1+\\nATOM   1690  N   ILE A 209     -17.499  21.680   0.938  1.00 90.35      A    N  \\nATOM   1691  CA  ILE A 209     -16.559  21.140  -0.031  1.00 91.38      A    C  \\nATOM   1692  C   ILE A 209     -16.935  21.643  -1.408  1.00 90.91      A    C  \\nATOM   1693  O   ILE A 209     -18.060  21.458  -1.869  1.00 89.69      A    O  \\nATOM   1694  CB  ILE A 209     -16.510  19.617   0.018  1.00 91.98      A    C  \\nATOM   1695  CG1 ILE A 209     -15.983  19.154   1.383  1.00 91.06      A    C  \\nATOM   1696  CG2 ILE A 209     -15.665  19.067  -1.092  1.00 91.22      A    C  \\nATOM   1697  CD1 ILE A 209     -16.176  17.696   1.619  1.00 87.33      A    C  \\nATOM   1698  N   ASP A 210     -16.000  22.315  -2.067  1.00 90.99      A    N  \\nATOM   1699  CA  ASP A 210     -16.268  22.887  -3.379  1.00 89.77      A    C  \\nATOM   1700  C   ASP A 210     -16.083  21.856  -4.485  1.00 89.33      A    C  \\nATOM   1701  O   ASP A 210     -16.956  21.751  -5.376  1.00 87.57      A    O  \\nATOM   1702  CB  ASP A 210     -15.362  24.083  -3.627  1.00 88.70      A    C  \\nATOM   1703  CG  ASP A 210     -15.681  25.249  -2.700  1.00 88.47      A    C  \\nATOM   1704  OD1 ASP A 210     -16.859  25.394  -2.306  1.00 84.09      A    O  \\nATOM   1705  OD2 ASP A 210     -14.764  25.997  -2.366  1.00 83.73      A    O1-\\nATOM   1706  N   TYR A 211     -15.005  21.121  -4.434  1.00 90.35      A    N  \\nATOM   1707  CA  TYR A 211     -14.690  20.137  -5.473  1.00 90.47      A    C  \\nATOM   1708  C   TYR A 211     -14.320  18.804  -4.851  1.00 91.29      A    C  \\nATOM   1709  O   TYR A 211     -13.835  18.754  -3.715  1.00 91.32      A    O  \\nATOM   1710  CB  TYR A 211     -13.572  20.624  -6.394  1.00 89.70      A    C  \\nATOM   1711  CG  TYR A 211     -13.899  21.862  -7.172  1.00 89.33      A    C  \\nATOM   1712  CD1 TYR A 211     -14.703  21.803  -8.304  1.00 88.02      A    C  \\nATOM   1713  CD2 TYR A 211     -13.392  23.105  -6.806  1.00 88.10      A    C  \\nATOM   1714  CE1 TYR A 211     -14.997  22.925  -9.055  1.00 86.31      A    C  \\nATOM   1715  CE2 TYR A 211     -13.680  24.249  -7.520  1.00 87.33      A    C  \\nATOM   1716  CZ  TYR A 211     -14.469  24.153  -8.641  1.00 86.54      A    C  \\nATOM   1717  OH  TYR A 211     -14.764  25.272  -9.374  1.00 84.47      A    O  \\nATOM   1718  N   VAL A 212     -14.528  17.763  -5.616  1.00 92.19      A    N  \\nATOM   1719  CA  VAL A 212     -14.146  16.410  -5.231  1.00 92.44      A    C  \\nATOM   1720  C   VAL A 212     -13.098  15.936  -6.208  1.00 91.99      A    C  \\nATOM   1721  O   VAL A 212     -13.346  15.848  -7.397  1.00 90.62      A    O  \\nATOM   1722  CB  VAL A 212     -15.345  15.452  -5.201  1.00 92.39      A    C  \\nATOM   1723  CG1 VAL A 212     -16.374  15.910  -4.166  1.00 91.26      A    C  \\nATOM   1724  CG2 VAL A 212     -14.884  14.037  -4.878  1.00 91.46      A    C  \\nATOM   1725  N   ALA A 213     -11.911  15.643  -5.694  1.00 92.50      A    N  \\nATOM   1726  CA  ALA A 213     -10.797  15.239  -6.538  1.00 91.88      A    C  \\nATOM   1727  C   ALA A 213     -10.704  13.708  -6.569  1.00 90.83      A    C  \\nATOM   1728  O   ALA A 213     -10.667  13.056  -5.529  1.00 88.78      A    O  \\nATOM   1729  CB  ALA A 213      -9.493  15.837  -6.040  1.00 91.38      A    C  \\nATOM   1730  N   HIS A 214     -10.656  13.201  -7.778  1.00 89.56      A    N  \\nATOM   1731  CA  HIS A 214     -10.508  11.766  -7.985  1.00 87.93      A    C  \\nATOM   1732  C   HIS A 214     -10.040  11.537  -9.428  1.00 87.31      A    C  \\nATOM   1733  O   HIS A 214     -10.006  12.461 -10.232  1.00 86.12      A    O  \\nATOM   1734  CB  HIS A 214     -11.810  11.003  -7.675  1.00 86.33      A    C  \\nATOM   1735  CG  HIS A 214     -11.614   9.533  -7.491  1.00 85.54      A    C  \\nATOM   1736  CD2 HIS A 214     -11.244   8.801  -6.424  1.00 81.24      A    C  \\nATOM   1737  ND1 HIS A 214     -11.800   8.644  -8.525  1.00 80.61      A    N  \\nATOM   1738  CE1 HIS A 214     -11.555   7.418  -8.088  1.00 80.21      A    C  \\nATOM   1739  NE2 HIS A 214     -11.237   7.473  -6.818  1.00 81.48      A    N  \\nATOM   1740  N   ASP A 215      -9.654  10.321  -9.713  1.00 85.40      A    N  \\nATOM   1741  CA  ASP A 215      -9.163   9.994 -11.062  1.00 83.47      A    C  \\nATOM   1742  C   ASP A 215     -10.308   9.868 -12.053  1.00 82.79      A    C  \\nATOM   1743  O   ASP A 215     -11.486   9.784 -11.699  1.00 79.77      A    O  \\nATOM   1744  CB  ASP A 215      -8.326   8.716 -11.010  1.00 79.42      A    C  \\nATOM   1745  CG  ASP A 215      -9.061   7.564 -10.370  1.00 75.94      A    C  \\nATOM   1746  OD1 ASP A 215     -10.181   7.765  -9.876  1.00 67.42      A    O  \\nATOM   1747  OD2 ASP A 215      -8.520   6.434 -10.364  1.00 72.01      A    O1-\\nATOM   1748  N   ASP A 216      -9.947   9.847 -13.324  1.00 78.25      A    N  \\nATOM   1749  CA  ASP A 216     -10.928   9.790 -14.396  1.00 76.57      A    C  \\nATOM   1750  C   ASP A 216     -11.527   8.397 -14.541  1.00 76.76      A    C  \\nATOM   1751  O   ASP A 216     -12.585   8.232 -15.166  1.00 73.87      A    O  \\nATOM   1752  CB  ASP A 216     -10.305  10.234 -15.724  1.00 72.38      A    C  \\nATOM   1753  CG  ASP A 216     -11.337  10.585 -16.784  1.00 66.38      A    C  \\nATOM   1754  OD1 ASP A 216     -12.536  10.619 -16.451  1.00 60.39      A    O  \\nATOM   1755  OD2 ASP A 216     -10.942  10.840 -17.924  1.00 62.80      A    O1-\\nATOM   1756  N   ILE A 217     -10.859   7.386 -14.001  1.00 75.05      A    N  \\nATOM   1757  CA  ILE A 217     -11.357   6.005 -14.100  1.00 75.04      A    C  \\nATOM   1758  C   ILE A 217     -12.590   5.846 -13.223  1.00 75.71      A    C  \\nATOM   1759  O   ILE A 217     -12.512   6.053 -12.011  1.00 72.85      A    O  \\nATOM   1760  CB  ILE A 217     -10.262   5.020 -13.685  1.00 71.54      A    C  \\nATOM   1761  CG1 ILE A 217      -9.028   5.180 -14.587  1.00 67.71      A    C  \\nATOM   1762  CG2 ILE A 217     -10.760   3.585 -13.781  1.00 66.75      A    C  \\nATOM   1763  CD1 ILE A 217      -7.826   4.405 -14.119  1.00 62.08      A    C  \\nATOM   1764  N   PRO A 218     -13.729   5.462 -13.803  1.00 75.36      A    N  \\nATOM   1765  CA  PRO A 218     -14.926   5.288 -12.990  1.00 74.88      A    C  \\nATOM   1766  C   PRO A 218     -14.867   4.007 -12.177  1.00 74.77      A    C  \\nATOM   1767  O   PRO A 218     -14.344   2.980 -12.608  1.00 70.73      A    O  \\nATOM   1768  CB  PRO A 218     -16.058   5.228 -14.034  1.00 71.28      A    C  \\nATOM   1769  CG  PRO A 218     -15.380   4.686 -15.233  1.00 69.96      A    C  \\nATOM   1770  CD  PRO A 218     -13.988   5.258 -15.218  1.00 72.61      A    C  \\nATOM   1771  N   TYR A 219     -15.390   4.064 -10.983  1.00 74.15      A    N  \\nATOM   1772  CA  TYR A 219     -15.471   2.918 -10.099  1.00 74.74      A    C  \\nATOM   1773  C   TYR A 219     -16.812   2.254 -10.258  1.00 74.19      A    C  \\nATOM   1774  O   TYR A 219     -17.852   2.893 -10.149  1.00 71.43      A    O  \\nATOM   1775  CB  TYR A 219     -15.227   3.307  -8.648  1.00 72.24      A    C  \\nATOM   1776  CG  TYR A 219     -15.176   2.128  -7.689  1.00 68.07      A    C  \\nATOM   1777  CD1 TYR A 219     -16.228   1.830  -6.888  1.00 63.35      A    C  \\nATOM   1778  CD2 TYR A 219     -14.035   1.321  -7.634  1.00 62.81      A    C  \\nATOM   1779  CE1 TYR A 219     -16.192   0.750  -6.005  1.00 59.17      A    C  \\nATOM   1780  CE2 TYR A 219     -13.986   0.253  -6.774  1.00 59.12      A    C  \\nATOM   1781  CZ  TYR A 219     -15.055  -0.016  -5.954  1.00 58.66      A    C  \\nATOM   1782  OH  TYR A 219     -14.987  -1.096  -5.089  1.00 54.54      A    O  \\nATOM   1783  N   VAL A 220     -16.779   0.953 -10.483  1.00 69.53      A    N  \\nATOM   1784  CA  VAL A 220     -17.979   0.174 -10.692  1.00 68.96      A    C  \\nATOM   1785  C   VAL A 220     -18.429  -0.384  -9.353  1.00 69.36      A    C  \\nATOM   1786  O   VAL A 220     -17.690  -1.083  -8.662  1.00 66.74      A    O  \\nATOM   1787  CB  VAL A 220     -17.745  -0.968 -11.685  1.00 64.44      A    C  \\nATOM   1788  CG1 VAL A 220     -17.306  -0.424 -13.024  1.00 63.30      A    C  \\nATOM   1789  CG2 VAL A 220     -19.003  -1.783 -11.830  1.00 62.03      A    C  \\nATOM   1790  N   SER A 221     -19.630  -0.044  -8.971  1.00 71.53      A    N  \\nATOM   1791  CA  SER A 221     -20.217  -0.500  -7.740  1.00 71.49      A    C  \\nATOM   1792  C   SER A 221     -21.513  -1.258  -8.041  1.00 71.45      A    C  \\nATOM   1793  O   SER A 221     -21.948  -1.352  -9.171  1.00 69.07      A    O  \\nATOM   1794  CB  SER A 221     -20.495   0.659  -6.779  1.00 67.93      A    C  \\nATOM   1795  OG  SER A 221     -21.492   1.529  -7.316  1.00 64.04      A    O  \\nATOM   1796  N   ALA A 222     -22.094  -1.809  -6.966  1.00 66.03      A    N  \\nATOM   1797  CA  ALA A 222     -23.350  -2.546  -7.138  1.00 66.35      A    C  \\nATOM   1798  C   ALA A 222     -24.515  -1.610  -7.344  1.00 67.49      A    C  \\nATOM   1799  O   ALA A 222     -25.508  -1.975  -7.967  1.00 64.66      A    O  \\nATOM   1800  CB  ALA A 222     -23.573  -3.429  -5.923  1.00 62.30      A    C  \\nATOM   1801  N   ASP A 223     -24.422  -0.417  -6.806  1.00 67.03      A    N  \\nATOM   1802  CA  ASP A 223     -25.515   0.545  -6.938  1.00 67.51      A    C  \\nATOM   1803  C   ASP A 223     -25.516   1.194  -8.311  1.00 68.64      A    C  \\nATOM   1804  O   ASP A 223     -26.549   1.233  -8.991  1.00 65.49      A    O  \\nATOM   1805  CB  ASP A 223     -25.420   1.642  -5.843  1.00 62.47      A    C  \\nATOM   1806  CG  ASP A 223     -25.662   1.073  -4.453  1.00 58.69      A    C  \\nATOM   1807  OD1 ASP A 223     -26.399   0.078  -4.331  1.00 53.15      A    O  \\nATOM   1808  OD2 ASP A 223     -25.099   1.633  -3.486  1.00 56.33      A    O1-\\nATOM   1809  N   SER A 224     -24.385   1.695  -8.731  1.00 71.90      A    N  \\nATOM   1810  CA  SER A 224     -24.287   2.428  -9.990  1.00 72.49      A    C  \\nATOM   1811  C   SER A 224     -22.995   2.040 -10.711  1.00 73.15      A    C  \\nATOM   1812  O   SER A 224     -22.082   1.458 -10.104  1.00 69.69      A    O  \\nATOM   1813  CB  SER A 224     -24.338   3.937  -9.752  1.00 69.41      A    C  \\nATOM   1814  OG  SER A 224     -23.214   4.374  -9.030  1.00 63.84      A    O  \\nATOM   1815  N   ASP A 225     -22.957   2.371 -11.965  1.00 69.39      A    N  \\nATOM   1816  CA  ASP A 225     -21.748   2.077 -12.753  1.00 70.97      A    C  \\nATOM   1817  C   ASP A 225     -20.603   3.018 -12.388  1.00 73.28      A    C  \\nATOM   1818  O   ASP A 225     -19.456   2.667 -12.609  1.00 71.17      A    O  \\nATOM   1819  CB  ASP A 225     -22.054   2.218 -14.247  1.00 66.08      A    C  \\nATOM   1820  CG  ASP A 225     -22.977   1.128 -14.739  1.00 61.94      A    C  \\nATOM   1821  OD1 ASP A 225     -22.913  -0.007 -14.215  1.00 55.24      A    O  \\nATOM   1822  OD2 ASP A 225     -23.774   1.405 -15.673  1.00 56.17      A    O1-\\nATOM   1823  N   ASP A 226     -20.935   4.163 -11.855  1.00 79.38      A    N  \\nATOM   1824  CA  ASP A 226     -19.914   5.121 -11.421  1.00 80.39      A    C  \\nATOM   1825  C   ASP A 226     -20.431   5.840 -10.176  1.00 81.00      A    C  \\nATOM   1826  O   ASP A 226     -21.555   6.333 -10.166  1.00 80.23      A    O  \\nATOM   1827  CB  ASP A 226     -19.603   6.150 -12.527  1.00 78.63      A    C  \\nATOM   1828  CG  ASP A 226     -18.354   6.979 -12.225  1.00 77.95      A    C  \\nATOM   1829  OD1 ASP A 226     -17.729   6.766 -11.178  1.00 74.65      A    O  \\nATOM   1830  OD2 ASP A 226     -18.039   7.863 -13.041  1.00 74.83      A    O1-\\nATOM   1831  N   ILE A 227     -19.603   5.874  -9.142  1.00 80.32      A    N  \\nATOM   1832  CA  ILE A 227     -20.027   6.517  -7.896  1.00 81.22      A    C  \\nATOM   1833  C   ILE A 227     -19.968   8.041  -8.048  1.00 82.41      A    C  \\nATOM   1834  O   ILE A 227     -20.778   8.742  -7.440  1.00 81.18      A    O  \\nATOM   1835  CB  ILE A 227     -19.127   6.022  -6.743  1.00 78.59      A    C  \\nATOM   1836  CG1 ILE A 227     -19.254   4.502  -6.601  1.00 71.74      A    C  \\nATOM   1837  CG2 ILE A 227     -19.522   6.687  -5.422  1.00 70.64      A    C  \\nATOM   1838  CD1 ILE A 227     -18.340   3.914  -5.592  1.00 67.19      A    C  \\nATOM   1839  N   TYR A 228     -19.074   8.537  -8.837  1.00 85.68      A    N  \\nATOM   1840  CA  TYR A 228     -18.859   9.964  -9.030  1.00 86.52      A    C  \\nATOM   1841  C   TYR A 228     -19.692  10.550 -10.144  1.00 86.31      A    C  \\nATOM   1842  O   TYR A 228     -19.334  11.588 -10.720  1.00 85.70      A    O  \\nATOM   1843  CB  TYR A 228     -17.381  10.257  -9.254  1.00 87.07      A    C  \\nATOM   1844  CG  TYR A 228     -16.517   9.884  -8.077  1.00 87.58      A    C  \\nATOM   1845  CD1 TYR A 228     -16.410  10.707  -6.977  1.00 86.89      A    C  \\nATOM   1846  CD2 TYR A 228     -15.803   8.684  -8.092  1.00 87.27      A    C  \\nATOM   1847  CE1 TYR A 228     -15.620  10.373  -5.909  1.00 85.73      A    C  \\nATOM   1848  CE2 TYR A 228     -15.013   8.337  -7.016  1.00 87.02      A    C  \\nATOM   1849  CZ  TYR A 228     -14.926   9.186  -5.919  1.00 85.75      A    C  \\nATOM   1850  OH  TYR A 228     -14.148   8.852  -4.845  1.00 83.42      A    O  \\nATOM   1851  N   LYS A 229     -20.809   9.902 -10.478  1.00 85.06      A    N  \\nATOM   1852  CA  LYS A 229     -21.699  10.428 -11.512  1.00 84.55      A    C  \\nATOM   1853  C   LYS A 229     -22.425  11.697 -11.106  1.00 84.83      A    C  \\nATOM   1854  O   LYS A 229     -22.459  12.656 -11.888  1.00 84.61      A    O  \\nATOM   1855  CB  LYS A 229     -22.702   9.380 -11.980  1.00 83.19      A    C  \\nATOM   1856  CG  LYS A 229     -22.075   8.261 -12.813  1.00 77.89      A    C  \\nATOM   1857  CD  LYS A 229     -23.134   7.329 -13.342  1.00 71.64      A    C  \\nATOM   1858  CE  LYS A 229     -22.512   6.212 -14.168  1.00 65.74      A    C  \\nATOM   1859  NZ  LYS A 229     -23.528   5.262 -14.672  1.00 56.51      A    N1+\\nATOM   1860  N   PRO A 230     -23.052  11.784  -9.919  1.00 86.16      A    N  \\nATOM   1861  CA  PRO A 230     -23.716  13.036  -9.575  1.00 85.92      A    C  \\nATOM   1862  C   PRO A 230     -22.775  14.196  -9.256  1.00 86.79      A    C  \\nATOM   1863  O   PRO A 230     -23.184  15.344  -9.316  1.00 85.94      A    O  \\nATOM   1864  CB  PRO A 230     -24.568  12.684  -8.352  1.00 84.28      A    C  \\nATOM   1865  CG  PRO A 230     -23.812  11.588  -7.707  1.00 83.15      A    C  \\nATOM   1866  CD  PRO A 230     -23.240  10.806  -8.842  1.00 85.50      A    C  \\nATOM   1867  N   ILE A 231     -21.523  13.872  -8.937  1.00 88.52      A    N  \\nATOM   1868  CA  ILE A 231     -20.525  14.937  -8.721  1.00 88.85      A    C  \\nATOM   1869  C   ILE A 231     -20.155  15.576 -10.050  1.00 88.83      A    C  \\nATOM   1870  O   ILE A 231     -19.975  16.789 -10.137  1.00 88.44      A    O  \\nATOM   1871  CB  ILE A 231     -19.304  14.379  -7.993  1.00 89.19      A    C  \\nATOM   1872  CG1 ILE A 231     -19.722  13.721  -6.676  1.00 87.67      A    C  \\nATOM   1873  CG2 ILE A 231     -18.298  15.472  -7.696  1.00 87.77      A    C  \\nATOM   1874  CD1 ILE A 231     -18.630  12.994  -5.999  1.00 84.19      A    C  \\nATOM   1875  N   LYS A 232     -20.060  14.767 -11.079  1.00 87.77      A    N  \\nATOM   1876  CA  LYS A 232     -19.769  15.289 -12.421  1.00 87.21      A    C  \\nATOM   1877  C   LYS A 232     -20.956  16.065 -12.989  1.00 86.96      A    C  \\nATOM   1878  O   LYS A 232     -20.764  16.972 -13.785  1.00 85.67      A    O  \\nATOM   1879  CB  LYS A 232     -19.385  14.136 -13.349  1.00 86.49      A    C  \\nATOM   1880  CG  LYS A 232     -18.086  13.440 -12.985  1.00 85.88      A    C  \\nATOM   1881  CD  LYS A 232     -17.784  12.288 -13.900  1.00 84.20      A    C  \\nATOM   1882  CE  LYS A 232     -16.510  11.577 -13.505  1.00 81.79      A    C  \\nATOM   1883  NZ  LYS A 232     -16.234  10.412 -14.390  1.00 77.49      A    N1+\\nATOM   1884  N   GLU A 233     -22.163  15.685 -12.603  1.00 85.24      A    N  \\nATOM   1885  CA  GLU A 233     -23.345  16.401 -13.090  1.00 85.01      A    C  \\nATOM   1886  C   GLU A 233     -23.531  17.735 -12.380  1.00 84.82      A    C  \\nATOM   1887  O   GLU A 233     -24.064  18.677 -12.969  1.00 83.17      A    O  \\nATOM   1888  CB  GLU A 233     -24.582  15.551 -12.904  1.00 83.75      A    C  \\nATOM   1889  CG  GLU A 233     -24.621  14.310 -13.782  1.00 80.44      A    C  \\nATOM   1890  CD  GLU A 233     -25.878  13.473 -13.563  1.00 76.61      A    C  \\nATOM   1891  OE1 GLU A 233     -26.678  13.817 -12.679  1.00 71.73      A    O  \\nATOM   1892  OE2 GLU A 233     -26.051  12.459 -14.290  1.00 69.28      A    O1-\\nATOM   1893  N   MET A 234     -23.098  17.811 -11.139  1.00 87.10      A    N  \\nATOM   1894  CA  MET A 234     -23.158  19.076 -10.406  1.00 86.46      A    C  \\nATOM   1895  C   MET A 234     -22.014  20.011 -10.749  1.00 86.24      A    C  \\nATOM   1896  O   MET A 234     -22.045  21.162 -10.337  1.00 85.09      A    O  \\nATOM   1897  CB  MET A 234     -23.167  18.818  -8.888  1.00 85.97      A    C  \\nATOM   1898  CG  MET A 234     -24.437  18.163  -8.379  1.00 83.40      A    C  \\nATOM   1899  SD  MET A 234     -24.358  17.798  -6.623  1.00 82.88      A    S  \\nATOM   1900  CE  MET A 234     -24.575  19.435  -5.936  1.00 71.06      A    C  \\nATOM   1901  N   GLY A 235     -21.033  19.527 -11.459  1.00 86.01      A    N  \\nATOM   1902  CA  GLY A 235     -19.894  20.338 -11.809  1.00 85.55      A    C  \\nATOM   1903  C   GLY A 235     -18.876  20.502 -10.717  1.00 86.32      A    C  \\nATOM   1904  O   GLY A 235     -18.134  21.487 -10.695  1.00 84.67      A    O  \\nATOM   1905  N   LYS A 236     -18.814  19.539  -9.815  1.00 88.26      A    N  \\nATOM   1906  CA  LYS A 236     -17.868  19.575  -8.698  1.00 88.79      A    C  \\nATOM   1907  C   LYS A 236     -16.961  18.370  -8.719  1.00 89.40      A    C  \\nATOM   1908  O   LYS A 236     -16.885  17.617  -7.760  1.00 88.76      A    O  \\nATOM   1909  CB  LYS A 236     -18.589  19.690  -7.379  1.00 88.32      A    C  \\nATOM   1910  CG  LYS A 236     -19.379  20.950  -7.165  1.00 87.85      A    C  \\nATOM   1911  CD  LYS A 236     -20.055  20.987  -5.822  1.00 85.45      A    C  \\nATOM   1912  CE  LYS A 236     -20.836  22.276  -5.627  1.00 82.97      A    C  \\nATOM   1913  NZ  LYS A 236     -21.515  22.316  -4.309  1.00 75.07      A    N1+\\nATOM   1914  N   PHE A 237     -16.273  18.166  -9.801  1.00 89.40      A    N  \\nATOM   1915  CA  PHE A 237     -15.384  17.026  -9.961  1.00 89.97      A    C  \\nATOM   1916  C   PHE A 237     -14.109  17.437 -10.662  1.00 89.40      A    C  \\nATOM   1917  O   PHE A 237     -14.115  17.977 -11.754  1.00 88.38      A    O  \\nATOM   1918  CB  PHE A 237     -16.083  15.905 -10.745  1.00 89.70      A    C  \\nATOM   1919  CG  PHE A 237     -15.246  14.645 -10.875  1.00 90.00      A    C  \\nATOM   1920  CD1 PHE A 237     -15.129  13.778  -9.801  1.00 89.34      A    C  \\nATOM   1921  CD2 PHE A 237     -14.595  14.349 -12.061  1.00 89.03      A    C  \\nATOM   1922  CE1 PHE A 237     -14.365  12.631  -9.907  1.00 88.31      A    C  \\nATOM   1923  CE2 PHE A 237     -13.828  13.204 -12.165  1.00 88.53      A    C  \\nATOM   1924  CZ  PHE A 237     -13.732  12.343 -11.091  1.00 87.99      A    C  \\nATOM   1925  N   LEU A 238     -12.987  17.157 -10.006  1.00 89.04      A    N  \\nATOM   1926  CA  LEU A 238     -11.676  17.479 -10.536  1.00 88.86      A    C  \\nATOM   1927  C   LEU A 238     -10.945  16.205 -10.913  1.00 89.13      A    C  \\nATOM   1928  O   LEU A 238     -10.765  15.325 -10.056  1.00 88.09      A    O  \\nATOM   1929  CB  LEU A 238     -10.859  18.268  -9.524  1.00 87.45      A    C  \\nATOM   1930  CG  LEU A 238     -11.365  19.674  -9.191  1.00 85.41      A    C  \\nATOM   1931  CD1 LEU A 238     -11.366  20.543 -10.428  1.00 82.73      A    C  \\nATOM   1932  CD2 LEU A 238     -10.516  20.298  -8.106  1.00 83.28      A    C  \\nATOM   1933  N   THR A 239     -10.525  16.097 -12.145  1.00 88.08      A    N  \\nATOM   1934  CA  THR A 239      -9.817  14.918 -12.614  1.00 87.88      A    C  \\nATOM   1935  C   THR A 239      -8.347  15.041 -12.237  1.00 87.40      A    C  \\nATOM   1936  O   THR A 239      -7.676  16.016 -12.579  1.00 85.11      A    O  \\nATOM   1937  CB  THR A 239      -9.966  14.735 -14.124  1.00 86.76      A    C  \\nATOM   1938  CG2 THR A 239     -11.430  14.545 -14.474  1.00 79.99      A    C  \\nATOM   1939  OG1 THR A 239      -9.469  15.916 -14.788  1.00 79.25      A    O  \\nATOM   1940  N   THR A 240      -7.867  14.031 -11.513  1.00 88.01      A    N  \\nATOM   1941  CA  THR A 240      -6.468  13.974 -11.111  1.00 87.27      A    C  \\nATOM   1942  C   THR A 240      -5.831  12.747 -11.755  1.00 85.89      A    C  \\nATOM   1943  O   THR A 240      -6.374  11.652 -11.658  1.00 83.43      A    O  \\nATOM   1944  CB  THR A 240      -6.286  13.937  -9.611  1.00 87.25      A    C  \\nATOM   1945  CG2 THR A 240      -6.911  15.179  -8.979  1.00 84.13      A    C  \\nATOM   1946  OG1 THR A 240      -6.927  12.775  -9.061  1.00 83.60      A    O  \\nATOM   1947  N   GLN A 241      -4.696  12.951 -12.369  1.00 83.03      A    N  \\nATOM   1948  CA  GLN A 241      -4.007  11.873 -13.039  1.00 81.17      A    C  \\nATOM   1949  C   GLN A 241      -3.257  11.015 -12.017  1.00 80.97      A    C  \\nATOM   1950  O   GLN A 241      -2.405  11.515 -11.290  1.00 79.25      A    O  \\nATOM   1951  CB  GLN A 241      -3.042  12.413 -14.098  1.00 77.72      A    C  \\nATOM   1952  CG  GLN A 241      -3.712  13.141 -15.228  1.00 69.18      A    C  \\nATOM   1953  CD  GLN A 241      -2.726  13.655 -16.258  1.00 63.22      A    C  \\nATOM   1954  NE2 GLN A 241      -3.231  14.167 -17.391  1.00 54.57      A    N  \\nATOM   1955  OE1 GLN A 241      -1.504  13.599 -16.061  1.00 57.66      A    O  \\nATOM   1956  N   ARG A 242      -3.573   9.740 -11.992  1.00 76.15      A    N  \\nATOM   1957  CA  ARG A 242      -2.932   8.824 -11.091  1.00 74.92      A    C  \\nATOM   1958  C   ARG A 242      -1.673   8.257 -11.758  1.00 75.58      A    C  \\nATOM   1959  O   ARG A 242      -1.485   8.356 -12.973  1.00 73.92      A    O  \\nATOM   1960  CB  ARG A 242      -3.853   7.659 -10.684  1.00 71.50      A    C  \\nATOM   1961  CG  ARG A 242      -5.043   8.130  -9.860  1.00 65.85      A    C  \\nATOM   1962  CD  ARG A 242      -5.933   6.949  -9.471  1.00 63.31      A    C  \\nATOM   1963  NE  ARG A 242      -5.282   6.091  -8.488  1.00 59.55      A    N  \\nATOM   1964  CZ  ARG A 242      -5.831   4.971  -8.035  1.00 57.74      A    C  \\nATOM   1965  NH1 ARG A 242      -7.029   4.590  -8.472  1.00 50.34      A    N1+\\nATOM   1966  NH2 ARG A 242      -5.226   4.256  -7.108  1.00 49.99      A    N  \\nATOM   1967  N   THR A 243      -0.810   7.705 -10.942  1.00 78.58      A    N  \\nATOM   1968  CA  THR A 243       0.404   7.082 -11.441  1.00 78.39      A    C  \\nATOM   1969  C   THR A 243       0.099   5.696 -11.996  1.00 78.41      A    C  \\nATOM   1970  O   THR A 243      -0.946   5.103 -11.723  1.00 75.16      A    O  \\nATOM   1971  CB  THR A 243       1.465   6.978 -10.356  1.00 76.95      A    C  \\nATOM   1972  CG2 THR A 243       1.792   8.356  -9.801  1.00 73.99      A    C  \\nATOM   1973  OG1 THR A 243       0.994   6.150  -9.305  1.00 75.30      A    O  \\nATOM   1974  N   ASN A 244       1.018   5.188 -12.770  1.00 76.38      A    N  \\nATOM   1975  CA  ASN A 244       0.856   3.872 -13.382  1.00 75.61      A    C  \\nATOM   1976  C   ASN A 244       1.718   2.850 -12.684  1.00 76.08      A    C  \\nATOM   1977  O   ASN A 244       2.913   3.057 -12.503  1.00 72.61      A    O  \\nATOM   1978  CB  ASN A 244       1.206   3.920 -14.867  1.00 71.02      A    C  \\nATOM   1979  CG  ASN A 244       0.208   4.740 -15.670  1.00 64.22      A    C  \\nATOM   1980  ND2 ASN A 244       0.720   5.519 -16.623  1.00 58.17      A    N  \\nATOM   1981  OD1 ASN A 244      -0.987   4.667 -15.453  1.00 55.86      A    O  \\nATOM   1982  N   GLY A 245       1.087   1.759 -12.254  1.00 75.88      A    N  \\nATOM   1983  CA  GLY A 245       1.829   0.694 -11.614  1.00 74.74      A    C  \\nATOM   1984  C   GLY A 245       2.090   0.904 -10.124  1.00 76.84      A    C  \\nATOM   1985  O   GLY A 245       2.727   0.073  -9.509  1.00 74.30      A    O  \\nATOM   1986  N   ILE A 246       1.613   2.000  -9.569  1.00 80.18      A    N  \\nATOM   1987  CA  ILE A 246       1.820   2.303  -8.155  1.00 80.38      A    C  \\nATOM   1988  C   ILE A 246       0.483   2.480  -7.508  1.00 81.52      A    C  \\nATOM   1989  O   ILE A 246      -0.211   3.468  -7.738  1.00 79.37      A    O  \\nATOM   1990  CB  ILE A 246       2.690   3.555  -7.954  1.00 78.46      A    C  \\nATOM   1991  CG1 ILE A 246       4.073   3.359  -8.591  1.00 69.47      A    C  \\nATOM   1992  CG2 ILE A 246       2.848   3.849  -6.477  1.00 71.92      A    C  \\nATOM   1993  CD1 ILE A 246       4.918   4.605  -8.568  1.00 65.94      A    C  \\nATOM   1994  N   SER A 247       0.110   1.514  -6.710  1.00 82.00      A    N  \\nATOM   1995  CA  SER A 247      -1.152   1.524  -5.975  1.00 82.70      A    C  \\nATOM   1996  C   SER A 247      -1.006   0.667  -4.735  1.00 84.39      A    C  \\nATOM   1997  O   SER A 247       0.026   0.046  -4.493  1.00 83.53      A    O  \\nATOM   1998  CB  SER A 247      -2.307   1.021  -6.852  1.00 79.36      A    C  \\nATOM   1999  OG  SER A 247      -2.148  -0.355  -7.138  1.00 75.37      A    O  \\nATOM   2000  N   THR A 248      -2.053   0.657  -3.931  1.00 84.93      A    N  \\nATOM   2001  CA  THR A 248      -2.034  -0.193  -2.757  1.00 85.21      A    C  \\nATOM   2002  C   THR A 248      -2.144  -1.647  -3.111  1.00 84.62      A    C  \\nATOM   2003  O   THR A 248      -1.620  -2.512  -2.414  1.00 83.76      A    O  \\nATOM   2004  CB  THR A 248      -3.164   0.198  -1.810  1.00 86.03      A    C  \\nATOM   2005  CG2 THR A 248      -3.000   1.622  -1.364  1.00 82.93      A    C  \\nATOM   2006  OG1 THR A 248      -4.433   0.052  -2.483  1.00 82.04      A    O  \\nATOM   2007  N   SER A 249      -2.822  -1.941  -4.206  1.00 81.59      A    N  \\nATOM   2008  CA  SER A 249      -2.971  -3.326  -4.639  1.00 79.69      A    C  \\nATOM   2009  C   SER A 249      -1.667  -3.856  -5.225  1.00 79.49      A    C  \\nATOM   2010  O   SER A 249      -1.390  -5.057  -5.133  1.00 78.62      A    O  \\nATOM   2011  CB  SER A 249      -4.091  -3.459  -5.662  1.00 77.41      A    C  \\nATOM   2012  OG  SER A 249      -5.345  -3.115  -5.074  1.00 71.30      A    O  \\nATOM   2013  N   ASP A 250      -0.895  -2.984  -5.819  1.00 82.56      A    N  \\nATOM   2014  CA  ASP A 250       0.383  -3.419  -6.370  1.00 80.92      A    C  \\nATOM   2015  C   ASP A 250       1.376  -3.754  -5.279  1.00 81.52      A    C  \\nATOM   2016  O   ASP A 250       2.236  -4.624  -5.461  1.00 80.71      A    O  \\nATOM   2017  CB  ASP A 250       0.970  -2.344  -7.289  1.00 78.81      A    C  \\nATOM   2018  CG  ASP A 250       0.137  -2.140  -8.537  1.00 76.14      A    C  \\nATOM   2019  OD1 ASP A 250      -0.465  -3.123  -9.015  1.00 73.79      A    O  \\nATOM   2020  OD2 ASP A 250       0.063  -0.995  -9.042  1.00 73.33      A    O1-\\nATOM   2021  N   ILE A 251       1.269  -3.058  -4.156  1.00 85.18      A    N  \\nATOM   2022  CA  ILE A 251       2.151  -3.344  -3.015  1.00 85.52      A    C  \\nATOM   2023  C   ILE A 251       1.803  -4.704  -2.420  1.00 84.61      A    C  \\nATOM   2024  O   ILE A 251       2.672  -5.478  -2.045  1.00 83.98      A    O  \\nATOM   2025  CB  ILE A 251       2.081  -2.220  -1.977  1.00 87.12      A    C  \\nATOM   2026  CG1 ILE A 251       2.534  -0.913  -2.602  1.00 87.33      A    C  \\nATOM   2027  CG2 ILE A 251       2.940  -2.545  -0.776  1.00 87.10      A    C  \\nATOM   2028  CD1 ILE A 251       2.247   0.295  -1.740  1.00 86.03      A    C  \\nATOM   2029  N   ILE A 252       0.502  -4.996  -2.338  1.00 82.76      A    N  \\nATOM   2030  CA  ILE A 252       0.058  -6.268  -1.809  1.00 81.65      A    C  \\nATOM   2031  C   ILE A 252       0.469  -7.412  -2.728  1.00 80.21      A    C  \\nATOM   2032  O   ILE A 252       0.884  -8.479  -2.253  1.00 78.67      A    O  \\nATOM   2033  CB  ILE A 252      -1.469  -6.251  -1.561  1.00 82.09      A    C  \\nATOM   2034  CG1 ILE A 252      -1.801  -5.202  -0.495  1.00 83.18      A    C  \\nATOM   2035  CG2 ILE A 252      -1.983  -7.606  -1.130  1.00 82.03      A    C  \\nATOM   2036  CD1 ILE A 252      -3.267  -4.930  -0.372  1.00 81.74      A    C  \\nATOM   2037  N   THR A 253       0.368  -7.206  -4.015  1.00 77.36      A    N  \\nATOM   2038  CA  THR A 253       0.800  -8.230  -4.952  1.00 74.75      A    C  \\nATOM   2039  C   THR A 253       2.301  -8.440  -4.932  1.00 74.22      A    C  \\nATOM   2040  O   THR A 253       2.780  -9.529  -5.178  1.00 72.97      A    O  \\nATOM   2041  CB  THR A 253       0.339  -7.921  -6.369  1.00 71.24      A    C  \\nATOM   2042  CG2 THR A 253      -1.171  -7.883  -6.429  1.00 68.06      A    C  \\nATOM   2043  OG1 THR A 253       0.854  -6.635  -6.765  1.00 71.11      A    O  \\nATOM   2044  N   LYS A 254       3.042  -7.379  -4.615  1.00 78.36      A    N  \\nATOM   2045  CA  LYS A 254       4.488  -7.518  -4.479  1.00 77.40      A    C  \\nATOM   2046  C   LYS A 254       4.854  -8.287  -3.217  1.00 77.14      A    C  \\nATOM   2047  O   LYS A 254       5.897  -8.953  -3.177  1.00 76.75      A    O  \\nATOM   2048  CB  LYS A 254       5.183  -6.150  -4.466  1.00 75.96      A    C  \\nATOM   2049  CG  LYS A 254       5.216  -5.491  -5.828  1.00 70.50      A    C  \\nATOM   2050  CD  LYS A 254       5.860  -4.122  -5.746  1.00 67.41      A    C  \\nATOM   2051  CE  LYS A 254       5.933  -3.456  -7.097  1.00 62.76      A    C  \\nATOM   2052  NZ  LYS A 254       6.564  -2.105  -7.029  1.00 56.34      A    N1+\\nATOM   2053  N   ILE A 255       4.025  -8.168  -2.198  1.00 78.24      A    N  \\nATOM   2054  CA  ILE A 255       4.289  -8.912  -0.966  1.00 78.25      A    C  \\nATOM   2055  C   ILE A 255       3.921 -10.377  -1.153  1.00 77.19      A    C  \\nATOM   2056  O   ILE A 255       4.651 -11.273  -0.708  1.00 76.50      A    O  \\nATOM   2057  CB  ILE A 255       3.509  -8.286   0.223  1.00 79.22      A    C  \\nATOM   2058  CG1 ILE A 255       4.056  -6.889   0.480  1.00 80.53      A    C  \\nATOM   2059  CG2 ILE A 255       3.649  -9.110   1.487  1.00 79.89      A    C  \\nATOM   2060  CD1 ILE A 255       3.192  -6.064   1.397  1.00 79.49      A    C  \\nATOM   2061  N   ILE A 256       2.805 -10.640  -1.834  1.00 72.72      A    N  \\nATOM   2062  CA  ILE A 256       2.333 -12.002  -2.043  1.00 70.40      A    C  \\nATOM   2063  C   ILE A 256       3.266 -12.746  -2.976  1.00 70.30      A    C  \\nATOM   2064  O   ILE A 256       3.592 -13.910  -2.746  1.00 68.44      A    O  \\nATOM   2065  CB  ILE A 256       0.890 -11.991  -2.571  1.00 67.62      A    C  \\nATOM   2066  CG1 ILE A 256      -0.052 -11.510  -1.453  1.00 69.21      A    C  \\nATOM   2067  CG2 ILE A 256       0.467 -13.371  -3.031  1.00 65.91      A    C  \\nATOM   2068  CD1 ILE A 256      -1.435 -11.198  -1.950  1.00 64.14      A    C  \\nATOM   2069  N   ARG A 257       3.706 -12.097  -4.042  1.00 68.31      A    N  \\nATOM   2070  CA  ARG A 257       4.628 -12.767  -4.977  1.00 67.99      A    C  \\nATOM   2071  C   ARG A 257       5.937 -13.165  -4.293  1.00 68.80      A    C  \\nATOM   2072  O   ARG A 257       6.436 -14.270  -4.509  1.00 67.01      A    O  \\nATOM   2073  CB  ARG A 257       4.930 -11.832  -6.171  1.00 64.42      A    C  \\nATOM   2074  CG  ARG A 257       3.790 -11.654  -7.115  1.00 58.91      A    C  \\nATOM   2075  CD  ARG A 257       4.106 -10.638  -8.179  1.00 54.92      A    C  \\nATOM   2076  NE  ARG A 257       2.998 -10.422  -9.115  1.00 48.67      A    N  \\nATOM   2077  CZ  ARG A 257       3.015  -9.560 -10.106  1.00 47.19      A    C  \\nATOM   2078  NH1 ARG A 257       4.088  -8.820 -10.324  1.00 41.47      A    N1+\\nATOM   2079  NH2 ARG A 257       1.953  -9.446 -10.903  1.00 39.56      A    N  \\nATOM   2080  N   ASP A 258       6.471 -12.276  -3.472  1.00 69.78      A    N  \\nATOM   2081  CA  ASP A 258       7.696 -12.570  -2.736  1.00 69.26      A    C  \\nATOM   2082  C   ASP A 258       7.393 -12.497  -1.247  1.00 69.86      A    C  \\nATOM   2083  O   ASP A 258       7.854 -11.593  -0.541  1.00 68.94      A    O  \\nATOM   2084  CB  ASP A 258       8.809 -11.603  -3.129  1.00 67.03      A    C  \\nATOM   2085  CG  ASP A 258       9.293 -11.805  -4.552  1.00 64.99      A    C  \\nATOM   2086  OD1 ASP A 258       9.280 -12.936  -5.024  1.00 62.97      A    O  \\nATOM   2087  OD2 ASP A 258       9.690 -10.792  -5.181  1.00 64.14      A    O1-\\nATOM   2088  N   TYR A 259       6.608 -13.472  -0.794  1.00 68.08      A    N  \\nATOM   2089  CA  TYR A 259       6.250 -13.480   0.624  1.00 67.82      A    C  \\nATOM   2090  C   TYR A 259       7.410 -13.977   1.489  1.00 68.71      A    C  \\nATOM   2091  O   TYR A 259       7.649 -13.480   2.579  1.00 68.28      A    O  \\nATOM   2092  CB  TYR A 259       5.006 -14.343   0.839  1.00 64.70      A    C  \\nATOM   2093  CG  TYR A 259       4.576 -14.431   2.292  1.00 62.03      A    C  \\nATOM   2094  CD1 TYR A 259       3.945 -13.350   2.895  1.00 58.04      A    C  \\nATOM   2095  CD2 TYR A 259       4.790 -15.581   3.035  1.00 58.46      A    C  \\nATOM   2096  CE1 TYR A 259       3.530 -13.399   4.191  1.00 56.28      A    C  \\nATOM   2097  CE2 TYR A 259       4.383 -15.636   4.355  1.00 56.44      A    C  \\nATOM   2098  CZ  TYR A 259       3.757 -14.540   4.927  1.00 54.63      A    C  \\nATOM   2099  OH  TYR A 259       3.348 -14.610   6.245  1.00 48.37      A    O  \\nATOM   2100  N   ASP A 260       8.090 -15.020   1.010  1.00 64.50      A    N  \\nATOM   2101  CA  ASP A 260       9.168 -15.632   1.798  1.00 63.88      A    C  \\nATOM   2102  C   ASP A 260      10.342 -14.700   1.983  1.00 64.90      A    C  \\nATOM   2103  O   ASP A 260      11.026 -14.758   3.017  1.00 63.91      A    O  \\nATOM   2104  CB  ASP A 260       9.582 -16.953   1.150  1.00 60.23      A    C  \\nATOM   2105  CG  ASP A 260       9.960 -16.789  -0.310  1.00 56.11      A    C  \\nATOM   2106  OD1 ASP A 260       9.865 -15.673  -0.857  1.00 52.59      A    O  \\nATOM   2107  OD2 ASP A 260      10.368 -17.799  -0.926  1.00 54.43      A    O1-\\nATOM   2108  N   LYS A 261      10.558 -13.827   1.027  1.00 68.39      A    N  \\nATOM   2109  CA  LYS A 261      11.681 -12.892   1.107  1.00 67.89      A    C  \\nATOM   2110  C   LYS A 261      11.474 -11.848   2.191  1.00 68.55      A    C  \\nATOM   2111  O   LYS A 261      12.398 -11.500   2.918  1.00 68.15      A    O  \\nATOM   2112  CB  LYS A 261      11.915 -12.214  -0.234  1.00 65.69      A    C  \\nATOM   2113  CG  LYS A 261      12.374 -13.158  -1.319  1.00 63.55      A    C  \\nATOM   2114  CD  LYS A 261      12.573 -12.442  -2.625  1.00 62.18      A    C  \\nATOM   2115  CE  LYS A 261      13.016 -13.405  -3.708  1.00 58.97      A    C  \\nATOM   2116  NZ  LYS A 261      13.209 -12.737  -5.036  1.00 52.66      A    N1+\\nATOM   2117  N   TYR A 262      10.246 -11.350   2.313  1.00 71.62      A    N  \\nATOM   2118  CA  TYR A 262       9.935 -10.387   3.367  1.00 71.51      A    C  \\nATOM   2119  C   TYR A 262       9.821 -11.067   4.727  1.00 70.73      A    C  \\nATOM   2120  O   TYR A 262      10.091 -10.445   5.745  1.00 71.49      A    O  \\nATOM   2121  CB  TYR A 262       8.671  -9.616   3.037  1.00 72.36      A    C  \\nATOM   2122  CG  TYR A 262       8.809  -8.723   1.827  1.00 73.62      A    C  \\nATOM   2123  CD1 TYR A 262       9.393  -7.490   1.933  1.00 73.90      A    C  \\nATOM   2124  CD2 TYR A 262       8.361  -9.134   0.582  1.00 73.44      A    C  \\nATOM   2125  CE1 TYR A 262       9.506  -6.656   0.829  1.00 70.83      A    C  \\nATOM   2126  CE2 TYR A 262       8.467  -8.334  -0.516  1.00 71.95      A    C  \\nATOM   2127  CZ  TYR A 262       9.061  -7.095  -0.408  1.00 71.70      A    C  \\nATOM   2128  OH  TYR A 262       9.153  -6.280  -1.486  1.00 66.28      A    O  \\nATOM   2129  N   LEU A 263       9.400 -12.329   4.707  1.00 62.29      A    N  \\nATOM   2130  CA  LEU A 263       9.295 -13.061   5.979  1.00 62.19      A    C  \\nATOM   2131  C   LEU A 263      10.692 -13.337   6.554  1.00 62.91      A    C  \\nATOM   2132  O   LEU A 263      10.896 -13.210   7.773  1.00 62.50      A    O  \\nATOM   2133  CB  LEU A 263       8.527 -14.366   5.762  1.00 59.52      A    C  \\nATOM   2134  CG  LEU A 263       8.304 -15.251   6.986  1.00 55.51      A    C  \\nATOM   2135  CD1 LEU A 263       7.456 -14.515   8.011  1.00 55.42      A    C  \\nATOM   2136  CD2 LEU A 263       7.643 -16.550   6.569  1.00 54.76      A    C  \\nATOM   2137  N   LEU A 264      11.615 -13.690   5.697  1.00 59.56      A    N  \\nATOM   2138  CA  LEU A 264      12.992 -13.896   6.160  1.00 59.52      A    C  \\nATOM   2139  C   LEU A 264      13.649 -12.602   6.574  1.00 60.33      A    C  \\nATOM   2140  O   LEU A 264      14.520 -12.618   7.453  1.00 60.31      A    O  \\nATOM   2141  CB  LEU A 264      13.800 -14.578   5.078  1.00 56.60      A    C  \\nATOM   2142  CG  LEU A 264      13.416 -16.026   4.752  1.00 52.70      A    C  \\nATOM   2143  CD1 LEU A 264      13.712 -16.914   5.954  1.00 52.85      A    C  \\nATOM   2144  CD2 LEU A 264      14.161 -16.518   3.523  1.00 51.87      A    C  \\nATOM   2145  N   ARG A 265      13.241 -11.499   5.989  1.00 64.10      A    N  \\nATOM   2146  CA  ARG A 265      13.768 -10.203   6.393  1.00 65.18      A    C  \\nATOM   2147  C   ARG A 265      13.275  -9.845   7.767  1.00 64.86      A    C  \\nATOM   2148  O   ARG A 265      14.011  -9.230   8.555  1.00 64.91      A    O  \\nATOM   2149  CB  ARG A 265      13.392  -9.141   5.342  1.00 64.09      A    C  \\nATOM   2150  CG  ARG A 265      14.029  -7.790   5.598  1.00 61.28      A    C  \\nATOM   2151  CD  ARG A 265      13.501  -6.783   4.580  1.00 60.42      A    C  \\nATOM   2152  NE  ARG A 265      12.090  -6.446   4.810  1.00 57.50      A    N  \\nATOM   2153  CZ  ARG A 265      11.396  -5.612   4.081  1.00 56.87      A    C  \\nATOM   2154  NH1 ARG A 265      11.963  -4.987   3.055  1.00 51.79      A    N1+\\nATOM   2155  NH2 ARG A 265      10.116  -5.378   4.349  1.00 50.06      A    N  \\nATOM   2156  N   ASN A 266      12.053 -10.206   8.079  1.00 66.96      A    N  \\nATOM   2157  CA  ASN A 266      11.492  -9.866   9.389  1.00 66.38      A    C  \\nATOM   2158  C   ASN A 266      12.146 -10.667  10.508  1.00 65.89      A    C  \\nATOM   2159  O   ASN A 266      12.335 -10.172  11.619  1.00 65.19      A    O  \\nATOM   2160  CB  ASN A 266       9.989 -10.077   9.398  1.00 64.69      A    C  \\nATOM   2161  CG  ASN A 266       9.260  -9.078   8.524  1.00 63.60      A    C  \\nATOM   2162  ND2 ASN A 266       8.231  -9.538   7.837  1.00 60.58      A    N  \\nATOM   2163  OD1 ASN A 266       9.608  -7.904   8.489  1.00 60.83      A    O  \\nATOM   2164  N   PHE A 267      12.497 -11.931  10.226  1.00 55.86      A    N  \\nATOM   2165  CA  PHE A 267      13.178 -12.712  11.223  1.00 55.60      A    C  \\nATOM   2166  C   PHE A 267      14.663 -12.354  11.291  1.00 56.04      A    C  \\nATOM   2167  O   PHE A 267      15.300 -12.574  12.317  1.00 56.08      A    O  \\nATOM   2168  CB  PHE A 267      13.032 -14.205  10.928  1.00 53.02      A    C  \\nATOM   2169  CG  PHE A 267      11.662 -14.749  11.233  1.00 50.33      A    C  \\nATOM   2170  CD1 PHE A 267      11.275 -14.956  12.543  1.00 47.99      A    C  \\nATOM   2171  CD2 PHE A 267      10.803 -15.045  10.213  1.00 47.33      A    C  \\nATOM   2172  CE1 PHE A 267      10.034 -15.474  12.829  1.00 45.28      A    C  \\nATOM   2173  CE2 PHE A 267       9.548 -15.552  10.494  1.00 45.75      A    C  \\nATOM   2174  CZ  PHE A 267       9.169 -15.762  11.809  1.00 42.07      A    C  \\nATOM   2175  N   ALA A 268      15.195 -11.812  10.224  1.00 57.41      A    N  \\nATOM   2176  CA  ALA A 268      16.603 -11.394  10.247  1.00 58.19      A    C  \\nATOM   2177  C   ALA A 268      16.783 -10.116  11.059  1.00 58.35      A    C  \\nATOM   2178  O   ALA A 268      17.778  -9.957  11.777  1.00 57.88      A    O  \\nATOM   2179  CB  ALA A 268      17.119 -11.162   8.837  1.00 55.74      A    C  \\nATOM   2180  N   ARG A 269      15.814  -9.231  10.982  1.00 61.80      A    N  \\nATOM   2181  CA  ARG A 269      15.868  -7.995  11.764  1.00 61.58      A    C  \\nATOM   2182  C   ARG A 269      15.577  -8.276  13.245  1.00 61.09      A    C  \\nATOM   2183  O   ARG A 269      16.133  -7.598  14.121  1.00 59.70      A    O  \\nATOM   2184  CB  ARG A 269      14.839  -6.982  11.236  1.00 59.01      A    C  \\nATOM   2185  CG  ARG A 269      15.216  -6.446   9.852  1.00 55.29      A    C  \\nATOM   2186  CD  ARG A 269      14.174  -5.478   9.312  1.00 51.42      A    C  \\nATOM   2187  NE  ARG A 269      14.132  -4.233  10.046  1.00 46.43      A    N  \\nATOM   2188  CZ  ARG A 269      13.141  -3.881  10.851  1.00 46.39      A    C  \\nATOM   2189  NH1 ARG A 269      12.084  -4.687  11.052  1.00 40.02      A    N1+\\nATOM   2190  NH2 ARG A 269      13.197  -2.708  11.459  1.00 39.13      A    N  \\nATOM   2191  N   GLY A 270      14.752  -9.255  13.492  1.00 53.37      A    N  \\nATOM   2192  CA  GLY A 270      14.481  -9.645  14.874  1.00 53.72      A    C  \\nATOM   2193  C   GLY A 270      15.634 -10.408  15.500  1.00 54.21      A    C  \\nATOM   2194  O   GLY A 270      15.789 -10.405  16.731  1.00 53.34      A    O  \\nATOM   2195  N   ALA A 271      16.411 -11.086  14.696  1.00 51.02      A    N  \\nATOM   2196  CA  ALA A 271      17.565 -11.816  15.199  1.00 51.49      A    C  \\nATOM   2197  C   ALA A 271      18.741 -10.896  15.483  1.00 52.25      A    C  \\nATOM   2198  O   ALA A 271      19.572 -11.210  16.354  1.00 51.27      A    O  \\nATOM   2199  CB  ALA A 271      17.982 -12.902  14.211  1.00 48.40      A    C  \\nATOM   2200  N   THR A 272      18.837  -9.800  14.767  1.00 52.10      A    N  \\nATOM   2201  CA  THR A 272      19.939  -8.878  15.005  1.00 52.72      A    C  \\nATOM   2202  C   THR A 272      19.663  -7.976  16.186  1.00 52.70      A    C  \\nATOM   2203  O   THR A 272      20.607  -7.533  16.855  1.00 51.68      A    O  \\nATOM   2204  CB  THR A 272      20.202  -8.038  13.757  1.00 49.80      A    C  \\nATOM   2205  CG2 THR A 272      20.619  -8.922  12.597  1.00 46.35      A    C  \\nATOM   2206  OG1 THR A 272      19.010  -7.309  13.390  1.00 48.37      A    O  \\nATOM   2207  N   ARG A 273      18.410  -7.679  16.457  1.00 53.75      A    N  \\nATOM   2208  CA  ARG A 273      18.063  -6.789  17.566  1.00 54.89      A    C  \\nATOM   2209  C   ARG A 273      17.998  -7.543  18.889  1.00 54.16      A    C  \\nATOM   2210  O   ARG A 273      18.423  -7.005  19.924  1.00 52.71      A    O  \\nATOM   2211  CB  ARG A 273      16.731  -6.097  17.299  1.00 52.50      A    C  \\nATOM   2212  CG  ARG A 273      16.764  -5.135  16.153  1.00 50.25      A    C  \\nATOM   2213  CD  ARG A 273      15.417  -4.520  15.891  1.00 47.08      A    C  \\nATOM   2214  NE  ARG A 273      15.455  -3.562  14.764  1.00 43.35      A    N  \\nATOM   2215  CZ  ARG A 273      14.394  -2.871  14.354  1.00 43.67      A    C  \\nATOM   2216  NH1 ARG A 273      13.226  -3.022  14.955  1.00 37.95      A    N1+\\nATOM   2217  NH2 ARG A 273      14.502  -2.015  13.350  1.00 36.03      A    N  \\nATOM   2218  N   GLN A 274      17.482  -8.753  18.871  1.00 48.86      A    N  \\nATOM   2219  CA  GLN A 274      17.340  -9.522  20.081  1.00 50.31      A    C  \\nATOM   2220  C   GLN A 274      18.563 -10.385  20.325  1.00 50.13      A    C  \\nATOM   2221  O   GLN A 274      19.382 -10.628  19.447  1.00 48.88      A    O  \\nATOM   2222  CB  GLN A 274      16.092 -10.408  20.021  1.00 47.53      A    C  \\nATOM   2223  CG  GLN A 274      14.793  -9.608  19.947  1.00 46.71      A    C  \\nATOM   2224  CD  GLN A 274      13.561 -10.483  19.835  1.00 44.54      A    C  \\nATOM   2225  NE2 GLN A 274      12.383  -9.856  19.870  1.00 39.59      A    N  \\nATOM   2226  OE1 GLN A 274      13.654 -11.707  19.728  1.00 41.99      A    O  \\nATOM   2227  N   GLU A 275      18.674 -10.891  21.549  1.00 46.67      A    N  \\nATOM   2228  CA  GLU A 275      19.783 -11.728  21.923  1.00 48.58      A    C  \\nATOM   2229  C   GLU A 275      19.464 -13.176  21.562  1.00 48.69      A    C  \\nATOM   2230  O   GLU A 275      18.341 -13.568  21.315  1.00 47.07      A    O  \\nATOM   2231  CB  GLU A 275      20.066 -11.609  23.403  1.00 45.47      A    C  \\nATOM   2232  CG  GLU A 275      20.535 -10.244  23.862  1.00 44.17      A    C  \\nATOM   2233  CD  GLU A 275      20.784 -10.182  25.353  1.00 41.37      A    C  \\nATOM   2234  OE1 GLU A 275      20.481 -11.175  26.045  1.00 39.37      A    O  \\nATOM   2235  OE2 GLU A 275      21.273  -9.125  25.851  1.00 37.50      A    O1-\\nATOM   2236  N   LEU A 276      20.521 -13.956  21.536  1.00 45.52      A    N  \\nATOM   2237  CA  LEU A 276      20.412 -15.380  21.197  1.00 48.78      A    C  \\nATOM   2238  C   LEU A 276      20.347 -16.203  22.487  1.00 49.53      A    C  \\nATOM   2239  O   LEU A 276      21.364 -16.455  23.117  1.00 48.06      A    O  \\nATOM   2240  CB  LEU A 276      21.582 -15.835  20.330  1.00 44.34      A    C  \\nATOM   2241  CG  LEU A 276      21.716 -15.126  18.975  1.00 42.71      A    C  \\nATOM   2242  CD1 LEU A 276      20.507 -15.455  18.109  1.00 42.89      A    C  \\nATOM   2243  CD2 LEU A 276      23.000 -15.566  18.256  1.00 41.95      A    C  \\nATOM   2244  N   ASN A 277      19.168 -16.612  22.840  1.00 47.85      A    N  \\nATOM   2245  CA  ASN A 277      18.928 -17.360  24.053  1.00 50.43      A    C  \\nATOM   2246  C   ASN A 277      18.034 -18.548  23.759  1.00 51.25      A    C  \\nATOM   2247  O   ASN A 277      17.549 -18.729  22.637  1.00 50.30      A    O  \\nATOM   2248  CB  ASN A 277      18.303 -16.478  25.142  1.00 46.59      A    C  \\nATOM   2249  CG  ASN A 277      19.258 -15.446  25.703  1.00 45.14      A    C  \\nATOM   2250  ND2 ASN A 277      18.760 -14.244  25.947  1.00 42.40      A    N  \\nATOM   2251  OD1 ASN A 277      20.440 -15.734  25.909  1.00 40.90      A    O  \\nATOM   2252  N   VAL A 278      17.846 -19.358  24.764  1.00 54.41      A    N  \\nATOM   2253  CA  VAL A 278      16.972 -20.534  24.619  1.00 56.14      A    C  \\nATOM   2254  C   VAL A 278      15.515 -20.099  24.537  1.00 57.77      A    C  \\nATOM   2255  O   VAL A 278      14.773 -20.554  23.654  1.00 57.22      A    O  \\nATOM   2256  CB  VAL A 278      17.181 -21.495  25.793  1.00 53.31      A    C  \\nATOM   2257  CG1 VAL A 278      18.627 -21.997  25.806  1.00 54.36      A    C  \\nATOM   2258  CG2 VAL A 278      16.237 -22.692  25.686  1.00 52.18      A    C  \\nATOM   2259  N   SER A 279      15.116 -19.202  25.412  1.00 57.85      A    N  \\nATOM   2260  CA  SER A 279      13.743 -18.695  25.397  1.00 59.69      A    C  \\nATOM   2261  C   SER A 279      13.483 -17.827  24.189  1.00 60.29      A    C  \\nATOM   2262  O   SER A 279      12.332 -17.724  23.734  1.00 59.88      A    O  \\nATOM   2263  CB  SER A 279      13.454 -17.918  26.686  1.00 57.67      A    C  \\nATOM   2264  OG  SER A 279      14.213 -16.750  26.734  1.00 52.89      A    O  \\nATOM   2265  N   TRP A 280      14.503 -17.196  23.665  1.00 54.83      A    N  \\nATOM   2266  CA  TRP A 280      14.342 -16.408  22.470  1.00 56.36      A    C  \\nATOM   2267  C   TRP A 280      14.158 -17.309  21.241  1.00 57.04      A    C  \\nATOM   2268  O   TRP A 280      13.395 -17.000  20.339  1.00 55.89      A    O  \\nATOM   2269  CB  TRP A 280      15.556 -15.473  22.257  1.00 53.25      A    C  \\nATOM   2270  CG  TRP A 280      15.426 -14.594  21.054  1.00 53.02      A    C  \\nATOM   2271  CD1 TRP A 280      14.660 -13.492  20.974  1.00 49.92      A    C  \\nATOM   2272  CD2 TRP A 280      16.079 -14.774  19.801  1.00 51.60      A    C  \\nATOM   2273  CE2 TRP A 280      15.657 -13.755  18.990  1.00 49.22      A    C  \\nATOM   2274  CE3 TRP A 280      16.962 -15.731  19.321  1.00 45.24      A    C  \\nATOM   2275  NE1 TRP A 280      14.808 -12.948  19.701  1.00 47.01      A    N  \\nATOM   2276  CZ2 TRP A 280      16.099 -13.637  17.671  1.00 44.61      A    C  \\nATOM   2277  CZ3 TRP A 280      17.397 -15.606  18.016  1.00 45.83      A    C  \\nATOM   2278  CH2 TRP A 280      16.971 -14.563  17.206  1.00 44.46      A    C  \\nATOM   2279  N   LEU A 281      14.856 -18.438  21.227  1.00 52.63      A    N  \\nATOM   2280  CA  LEU A 281      14.692 -19.385  20.130  1.00 54.49      A    C  \\nATOM   2281  C   LEU A 281      13.371 -20.135  20.222  1.00 55.57      A    C  \\nATOM   2282  O   LEU A 281      12.831 -20.526  19.179  1.00 55.42      A    O  \\nATOM   2283  CB  LEU A 281      15.862 -20.371  20.100  1.00 50.89      A    C  \\nATOM   2284  CG  LEU A 281      17.229 -19.813  19.694  1.00 48.21      A    C  \\nATOM   2285  CD1 LEU A 281      17.205 -19.321  18.252  1.00 46.77      A    C  \\nATOM   2286  CD2 LEU A 281      18.316 -20.861  19.896  1.00 46.35      A    C  \\nATOM   2287  N   LYS A 282      12.880 -20.332  21.413  1.00 55.98      A    N  \\nATOM   2288  CA  LYS A 282      11.577 -20.989  21.547  1.00 57.77      A    C  \\nATOM   2289  C   LYS A 282      10.450 -20.105  21.047  1.00 59.00      A    C  \\nATOM   2290  O   LYS A 282       9.518 -20.575  20.414  1.00 59.18      A    O  \\nATOM   2291  CB  LYS A 282      11.309 -21.345  23.017  1.00 54.84      A    C  \\nATOM   2292  CG  LYS A 282      12.151 -22.501  23.552  1.00 52.06      A    C  \\nATOM   2293  CD  LYS A 282      11.827 -22.774  25.003  1.00 48.89      A    C  \\nATOM   2294  CE  LYS A 282      12.679 -23.907  25.549  1.00 45.72      A    C  \\nATOM   2295  NZ  LYS A 282      12.405 -24.160  26.972  1.00 40.61      A    N1+\\nATOM   2296  N   LYS A 283      10.573 -18.816  21.290  1.00 56.67      A    N  \\nATOM   2297  CA  LYS A 283       9.575 -17.890  20.776  1.00 58.11      A    C  \\nATOM   2298  C   LYS A 283       9.689 -17.678  19.281  1.00 59.04      A    C  \\nATOM   2299  O   LYS A 283       8.677 -17.466  18.605  1.00 58.50      A    O  \\nATOM   2300  CB  LYS A 283       9.676 -16.535  21.519  1.00 54.96      A    C  \\nATOM   2301  CG  LYS A 283       9.301 -16.584  22.954  1.00 52.26      A    C  \\nATOM   2302  CD  LYS A 283       9.461 -15.233  23.614  1.00 48.79      A    C  \\nATOM   2303  CE  LYS A 283       9.105 -15.291  25.085  1.00 44.26      A    C  \\nATOM   2304  NZ  LYS A 283       9.240 -14.009  25.770  1.00 39.34      A    N1+\\nATOM   2305  N   ASN A 284      10.897 -17.789  18.766  1.00 53.57      A    N  \\nATOM   2306  CA  ASN A 284      11.070 -17.691  17.337  1.00 55.79      A    C  \\nATOM   2307  C   ASN A 284      10.558 -18.925  16.617  1.00 56.56      A    C  \\nATOM   2308  O   ASN A 284      10.040 -18.812  15.501  1.00 55.63      A    O  \\nATOM   2309  CB  ASN A 284      12.558 -17.467  16.989  1.00 53.30      A    C  \\nATOM   2310  CG  ASN A 284      13.017 -16.061  17.336  1.00 50.93      A    C  \\nATOM   2311  ND2 ASN A 284      14.232 -15.970  17.829  1.00 48.84      A    N  \\nATOM   2312  OD1 ASN A 284      12.291 -15.100  17.153  1.00 47.07      A    O  \\nATOM   2313  N   GLU A 285      10.671 -20.092  17.259  1.00 54.31      A    N  \\nATOM   2314  CA  GLU A 285      10.121 -21.291  16.656  1.00 56.08      A    C  \\nATOM   2315  C   GLU A 285       8.596 -21.280  16.720  1.00 57.31      A    C  \\nATOM   2316  O   GLU A 285       7.939 -21.780  15.800  1.00 57.18      A    O  \\nATOM   2317  CB  GLU A 285      10.670 -22.544  17.320  1.00 53.46      A    C  \\nATOM   2318  CG  GLU A 285      12.154 -22.784  17.092  1.00 50.08      A    C  \\nATOM   2319  CD  GLU A 285      12.470 -23.196  15.677  1.00 46.15      A    C  \\nATOM   2320  OE1 GLU A 285      11.633 -23.886  15.052  1.00 44.02      A    O  \\nATOM   2321  OE2 GLU A 285      13.569 -22.850  15.177  1.00 41.84      A    O1-\\nATOM   2322  N   LEU A 286       8.044 -20.748  17.776  1.00 63.69      A    N  \\nATOM   2323  CA  LEU A 286       6.607 -20.649  17.898  1.00 66.01      A    C  \\nATOM   2324  C   LEU A 286       6.039 -19.640  16.898  1.00 67.24      A    C  \\nATOM   2325  O   LEU A 286       4.966 -19.846  16.339  1.00 67.91      A    O  \\nATOM   2326  CB  LEU A 286       6.167 -20.323  19.310  1.00 64.15      A    C  \\nATOM   2327  CG  LEU A 286       6.446 -21.392  20.370  1.00 59.92      A    C  \\nATOM   2328  CD1 LEU A 286       5.653 -22.655  20.079  1.00 60.30      A    C  \\nATOM   2329  CD2 LEU A 286       6.103 -20.857  21.753  1.00 59.50      A    C  \\nATOM   2330  N   GLU A 287       6.737 -18.542  16.723  1.00 62.38      A    N  \\nATOM   2331  CA  GLU A 287       6.311 -17.571  15.743  1.00 63.81      A    C  \\nATOM   2332  C   GLU A 287       6.545 -18.056  14.332  1.00 64.38      A    C  \\nATOM   2333  O   GLU A 287       5.842 -17.688  13.402  1.00 64.68      A    O  \\nATOM   2334  CB  GLU A 287       7.005 -16.235  15.981  1.00 61.76      A    C  \\nATOM   2335  CG  GLU A 287       6.558 -15.519  17.232  1.00 57.60      A    C  \\nATOM   2336  CD  GLU A 287       7.264 -14.197  17.459  1.00 53.13      A    C  \\nATOM   2337  OE1 GLU A 287       8.175 -13.855  16.661  1.00 51.52      A    O  \\nATOM   2338  OE2 GLU A 287       6.937 -13.502  18.447  1.00 48.15      A    O1-\\nATOM   2339  N   PHE A 288       7.553 -18.905  14.174  1.00 53.50      A    N  \\nATOM   2340  CA  PHE A 288       7.819 -19.502  12.868  1.00 55.28      A    C  \\nATOM   2341  C   PHE A 288       6.712 -20.448  12.458  1.00 56.85      A    C  \\nATOM   2342  O   PHE A 288       6.269 -20.452  11.305  1.00 57.15      A    O  \\nATOM   2343  CB  PHE A 288       9.167 -20.244  12.908  1.00 52.18      A    C  \\nATOM   2344  CG  PHE A 288       9.519 -20.952  11.634  1.00 49.57      A    C  \\nATOM   2345  CD1 PHE A 288       9.954 -20.235  10.537  1.00 46.85      A    C  \\nATOM   2346  CD2 PHE A 288       9.438 -22.326  11.545  1.00 45.86      A    C  \\nATOM   2347  CE1 PHE A 288      10.287 -20.878   9.362  1.00 43.75      A    C  \\nATOM   2348  CE2 PHE A 288       9.780 -22.962  10.379  1.00 42.78      A    C  \\nATOM   2349  CZ  PHE A 288      10.221 -22.250   9.282  1.00 40.83      A    C  \\nATOM   2350  N   LYS A 289       6.243 -21.283  13.396  1.00 58.48      A    N  \\nATOM   2351  CA  LYS A 289       5.143 -22.199  13.131  1.00 59.79      A    C  \\nATOM   2352  C   LYS A 289       3.842 -21.461  12.907  1.00 60.99      A    C  \\nATOM   2353  O   LYS A 289       2.993 -21.918  12.159  1.00 61.42      A    O  \\nATOM   2354  CB  LYS A 289       4.994 -23.210  14.268  1.00 57.02      A    C  \\nATOM   2355  CG  LYS A 289       6.164 -24.169  14.358  1.00 53.86      A    C  \\nATOM   2356  CD  LYS A 289       5.986 -25.165  15.480  1.00 49.88      A    C  \\nATOM   2357  CE  LYS A 289       7.157 -26.099  15.617  1.00 47.03      A    C  \\nATOM   2358  NZ  LYS A 289       7.000 -27.050  16.729  1.00 42.86      A    N1+\\nATOM   2359  N   LYS A 290       3.674 -20.333  13.572  1.00 65.50      A    N  \\nATOM   2360  CA  LYS A 290       2.479 -19.538  13.373  1.00 66.36      A    C  \\nATOM   2361  C   LYS A 290       2.460 -18.850  12.027  1.00 67.04      A    C  \\nATOM   2362  O   LYS A 290       1.404 -18.744  11.399  1.00 68.01      A    O  \\nATOM   2363  CB  LYS A 290       2.317 -18.499  14.489  1.00 64.94      A    C  \\nATOM   2364  CG  LYS A 290       1.958 -19.091  15.822  1.00 61.33      A    C  \\nATOM   2365  CD  LYS A 290       1.834 -18.012  16.876  1.00 58.35      A    C  \\nATOM   2366  CE  LYS A 290       1.460 -18.602  18.229  1.00 54.25      A    C  \\nATOM   2367  NZ  LYS A 290       1.337 -17.569  19.273  1.00 47.76      A    N1+\\nATOM   2368  N   HIS A 291       3.612 -18.438  11.554  1.00 57.56      A    N  \\nATOM   2369  CA  HIS A 291       3.679 -17.860  10.221  1.00 58.73      A    C  \\nATOM   2370  C   HIS A 291       3.550 -18.897   9.135  1.00 59.51      A    C  \\nATOM   2371  O   HIS A 291       3.022 -18.611   8.064  1.00 59.35      A    O  \\nATOM   2372  CB  HIS A 291       4.987 -17.081  10.042  1.00 55.70      A    C  \\nATOM   2373  CG  HIS A 291       5.063 -15.814  10.840  1.00 52.91      A    C  \\nATOM   2374  CD2 HIS A 291       5.881 -15.463  11.861  1.00 50.18      A    C  \\nATOM   2375  ND1 HIS A 291       4.231 -14.756  10.623  1.00 51.69      A    N  \\nATOM   2376  CE1 HIS A 291       4.525 -13.791  11.479  1.00 49.34      A    C  \\nATOM   2377  NE2 HIS A 291       5.515 -14.184  12.234  1.00 47.23      A    N  \\nATOM   2378  N   ILE A 292       4.006 -20.131   9.408  1.00 56.14      A    N  \\nATOM   2379  CA  ILE A 292       3.827 -21.207   8.438  1.00 57.03      A    C  \\nATOM   2380  C   ILE A 292       2.369 -21.625   8.391  1.00 58.34      A    C  \\nATOM   2381  O   ILE A 292       1.813 -21.910   7.329  1.00 58.65      A    O  \\nATOM   2382  CB  ILE A 292       4.771 -22.377   8.730  1.00 53.81      A    C  \\nATOM   2383  CG1 ILE A 292       6.218 -21.958   8.473  1.00 54.85      A    C  \\nATOM   2384  CG2 ILE A 292       4.456 -23.582   7.872  1.00 52.15      A    C  \\nATOM   2385  CD1 ILE A 292       7.234 -22.964   8.936  1.00 49.01      A    C  \\nATOM   2386  N   LYS A 293       1.737 -21.693   9.561  1.00 64.60      A    N  \\nATOM   2387  CA  LYS A 293       0.340 -22.064   9.619  1.00 64.85      A    C  \\nATOM   2388  C   LYS A 293      -0.561 -21.016   8.959  1.00 65.80      A    C  \\nATOM   2389  O   LYS A 293      -1.514 -21.377   8.267  1.00 67.05      A    O  \\nATOM   2390  CB  LYS A 293      -0.094 -22.288  11.087  1.00 63.56      A    C  \\nATOM   2391  CG  LYS A 293      -1.526 -22.785  11.252  1.00 59.57      A    C  \\nATOM   2392  CD  LYS A 293      -1.823 -23.043  12.703  1.00 55.64      A    C  \\nATOM   2393  CE  LYS A 293      -3.275 -23.510  12.872  1.00 50.53      A    C  \\nATOM   2394  NZ  LYS A 293      -3.609 -23.797  14.267  1.00 45.06      A    N1+\\nATOM   2395  N   ASP A 294      -0.266 -19.774   9.180  1.00 65.96      A    N  \\nATOM   2396  CA  ASP A 294      -1.033 -18.703   8.524  1.00 67.57      A    C  \\nATOM   2397  C   ASP A 294      -0.719 -18.642   7.041  1.00 67.61      A    C  \\nATOM   2398  O   ASP A 294      -1.563 -18.224   6.247  1.00 68.20      A    O  \\nATOM   2399  CB  ASP A 294      -0.758 -17.359   9.172  1.00 66.61      A    C  \\nATOM   2400  CG  ASP A 294      -1.288 -17.284  10.597  1.00 64.15      A    C  \\nATOM   2401  OD1 ASP A 294      -2.305 -17.951  10.881  1.00 62.41      A    O  \\nATOM   2402  OD2 ASP A 294      -0.690 -16.538  11.407  1.00 63.34      A    O1-\\nATOM   2403  N   PHE A 295       0.463 -19.058   6.664  1.00 57.44      A    N  \\nATOM   2404  CA  PHE A 295       0.799 -19.097   5.244  1.00 58.98      A    C  \\nATOM   2405  C   PHE A 295       0.117 -20.252   4.543  1.00 60.03      A    C  \\nATOM   2406  O   PHE A 295      -0.287 -20.132   3.383  1.00 59.89      A    O  \\nATOM   2407  CB  PHE A 295       2.310 -19.153   5.065  1.00 55.36      A    C  \\nATOM   2408  CG  PHE A 295       2.758 -19.241   3.627  1.00 52.14      A    C  \\nATOM   2409  CD1 PHE A 295       2.706 -18.123   2.828  1.00 49.86      A    C  \\nATOM   2410  CD2 PHE A 295       3.232 -20.427   3.099  1.00 49.64      A    C  \\nATOM   2411  CE1 PHE A 295       3.104 -18.190   1.512  1.00 48.11      A    C  \\nATOM   2412  CE2 PHE A 295       3.629 -20.482   1.782  1.00 48.47      A    C  \\nATOM   2413  CZ  PHE A 295       3.571 -19.372   0.997  1.00 44.84      A    C  \\nATOM   2414  N   ARG A 296      -0.009 -21.388   5.228  1.00 57.09      A    N  \\nATOM   2415  CA  ARG A 296      -0.710 -22.520   4.634  1.00 58.55      A    C  \\nATOM   2416  C   ARG A 296      -2.209 -22.288   4.570  1.00 59.87      A    C  \\nATOM   2417  O   ARG A 296      -2.859 -22.760   3.628  1.00 60.17      A    O  \\nATOM   2418  CB  ARG A 296      -0.427 -23.790   5.444  1.00 55.58      A    C  \\nATOM   2419  CG  ARG A 296       1.005 -24.296   5.330  1.00 52.17      A    C  \\nATOM   2420  CD  ARG A 296       1.215 -25.548   6.156  1.00 49.12      A    C  \\nATOM   2421  NE  ARG A 296       2.570 -26.054   6.015  1.00 45.18      A    N  \\nATOM   2422  CZ  ARG A 296       3.034 -27.121   6.674  1.00 43.98      A    C  \\nATOM   2423  NH1 ARG A 296       2.230 -27.795   7.478  1.00 39.04      A    N1+\\nATOM   2424  NH2 ARG A 296       4.276 -27.525   6.490  1.00 36.90      A    N  \\nATOM   2425  N   GLU A 297      -2.766 -21.594   5.525  1.00 67.67      A    N  \\nATOM   2426  CA  GLU A 297      -4.180 -21.235   5.460  1.00 68.20      A    C  \\nATOM   2427  C   GLU A 297      -4.452 -20.216   4.367  1.00 68.36      A    C  \\nATOM   2428  O   GLU A 297      -5.526 -20.243   3.766  1.00 68.82      A    O  \\nATOM   2429  CB  GLU A 297      -4.649 -20.721   6.799  1.00 67.12      A    C  \\nATOM   2430  CG  GLU A 297      -4.703 -21.777   7.887  1.00 63.67      A    C  \\nATOM   2431  CD  GLU A 297      -5.159 -21.226   9.229  1.00 59.86      A    C  \\nATOM   2432  OE1 GLU A 297      -5.387 -20.002   9.329  1.00 59.07      A    O  \\nATOM   2433  OE2 GLU A 297      -5.266 -22.025  10.184  1.00 55.07      A    O1-\\nATOM   2434  N   TYR A 298      -3.511 -19.355   4.145  1.00 62.23      A    N  \\nATOM   2435  CA  TYR A 298      -3.645 -18.425   3.029  1.00 63.02      A    C  \\nATOM   2436  C   TYR A 298      -3.345 -19.103   1.687  1.00 63.30      A    C  \\nATOM   2437  O   TYR A 298      -3.844 -18.659   0.663  1.00 62.66      A    O  \\nATOM   2438  CB  TYR A 298      -2.712 -17.229   3.235  1.00 61.00      A    C  \\nATOM   2439  CG  TYR A 298      -2.821 -16.158   2.184  1.00 58.42      A    C  \\nATOM   2440  CD1 TYR A 298      -3.948 -15.350   2.125  1.00 57.33      A    C  \\nATOM   2441  CD2 TYR A 298      -1.814 -15.964   1.230  1.00 56.63      A    C  \\nATOM   2442  CE1 TYR A 298      -4.055 -14.360   1.159  1.00 54.73      A    C  \\nATOM   2443  CE2 TYR A 298      -1.915 -14.972   0.282  1.00 55.13      A    C  \\nATOM   2444  CZ  TYR A 298      -3.047 -14.187   0.228  1.00 53.18      A    C  \\nATOM   2445  OH  TYR A 298      -3.159 -13.232  -0.733  1.00 53.48      A    O  \\nATOM   2446  N   PHE A 299      -2.580 -20.149   1.729  1.00 57.25      A    N  \\nATOM   2447  CA  PHE A 299      -2.250 -20.875   0.482  1.00 58.93      A    C  \\nATOM   2448  C   PHE A 299      -3.431 -21.723   0.028  1.00 59.31      A    C  \\nATOM   2449  O   PHE A 299      -3.675 -21.819  -1.197  1.00 58.41      A    O  \\nATOM   2450  CB  PHE A 299      -1.009 -21.738   0.697  1.00 55.81      A    C  \\nATOM   2451  CG  PHE A 299      -0.578 -22.502  -0.555  1.00 53.11      A    C  \\nATOM   2452  CD1 PHE A 299      -0.890 -23.829  -0.706  1.00 50.45      A    C  \\nATOM   2453  CD2 PHE A 299       0.128 -21.853  -1.545  1.00 50.36      A    C  \\nATOM   2454  CE1 PHE A 299      -0.504 -24.517  -1.820  1.00 48.93      A    C  \\nATOM   2455  CE2 PHE A 299       0.512 -22.523  -2.681  1.00 49.14      A    C  \\nATOM   2456  CZ  PHE A 299       0.204 -23.860  -2.823  1.00 47.05      A    C  \\nATOM   2457  N   LYS A 300      -4.125 -22.341   0.959  1.00 62.15      A    N  \\nATOM   2458  CA  LYS A 300      -5.289 -23.160   0.608  1.00 62.76      A    C  \\nATOM   2459  C   LYS A 300      -6.440 -22.312   0.082  1.00 62.76      A    C  \\nATOM   2460  O   LYS A 300      -7.266 -22.834  -0.666  1.00 62.62      A    O  \\nATOM   2461  CB  LYS A 300      -5.739 -23.981   1.806  1.00 60.68      A    C  \\nATOM   2462  CG  LYS A 300      -4.800 -25.072   2.213  1.00 56.81      A    C  \\nATOM   2463  CD  LYS A 300      -5.306 -25.819   3.422  1.00 53.11      A    C  \\nATOM   2464  CE  LYS A 300      -4.335 -26.925   3.845  1.00 48.11      A    C  \\nATOM   2465  NZ  LYS A 300      -4.858 -27.684   5.009  1.00 42.99      A    N1+\\nATOM   2466  N   LYS A 301      -6.507 -21.079   0.486  1.00 67.15      A    N  \\nATOM   2467  CA  LYS A 301      -7.568 -20.195  -0.024  1.00 67.58      A    C  \\nATOM   2468  C   LYS A 301      -7.282 -19.735  -1.436  1.00 67.43      A    C  \\nATOM   2469  O   LYS A 301      -8.164 -19.773  -2.305  1.00 66.17      A    O  \\nATOM   2470  CB  LYS A 301      -7.781 -19.019   0.892  1.00 65.84      A    C  \\nATOM   2471  CG  LYS A 301      -8.333 -19.370   2.259  1.00 61.02      A    C  \\nATOM   2472  CD  LYS A 301      -8.520 -18.138   3.115  1.00 57.63      A    C  \\nATOM   2473  CE  LYS A 301      -9.064 -18.523   4.484  1.00 51.45      A    C  \\nATOM   2474  NZ  LYS A 301      -9.271 -17.330   5.340  1.00 46.40      A    N1+\\nATOM   2475  N   ASN A 302      -6.058 -19.328  -1.668  1.00 63.85      A    N  \\nATOM   2476  CA  ASN A 302      -5.657 -18.834  -2.986  1.00 64.33      A    C  \\nATOM   2477  C   ASN A 302      -4.705 -19.833  -3.630  1.00 63.77      A    C  \\nATOM   2478  O   ASN A 302      -3.495 -19.669  -3.620  1.00 62.94      A    O  \\nATOM   2479  CB  ASN A 302      -5.011 -17.456  -2.895  1.00 61.51      A    C  \\nATOM   2480  CG  ASN A 302      -5.995 -16.400  -2.424  1.00 59.53      A    C  \\nATOM   2481  ND2 ASN A 302      -5.557 -15.548  -1.506  1.00 58.61      A    N  \\nATOM   2482  OD1 ASN A 302      -7.120 -16.349  -2.891  1.00 58.31      A    O  \\nATOM   2483  N   GLN A 303      -5.303 -20.891  -4.184  1.00 58.78      A    N  \\nATOM   2484  CA  GLN A 303      -4.503 -21.917  -4.865  1.00 58.76      A    C  \\nATOM   2485  C   GLN A 303      -4.294 -21.609  -6.344  1.00 58.05      A    C  \\nATOM   2486  O   GLN A 303      -3.294 -22.008  -6.919  1.00 56.75      A    O  \\nATOM   2487  CB  GLN A 303      -5.208 -23.272  -4.735  1.00 55.23      A    C  \\nATOM   2488  CG  GLN A 303      -5.244 -23.800  -3.303  1.00 52.62      A    C  \\nATOM   2489  CD  GLN A 303      -5.931 -25.140  -3.206  1.00 48.76      A    C  \\nATOM   2490  NE2 GLN A 303      -5.905 -25.738  -2.030  1.00 44.97      A    N  \\nATOM   2491  OE1 GLN A 303      -6.477 -25.644  -4.197  1.00 47.28      A    O  \\nATOM   2492  N   GLU A 304      -5.248 -20.921  -6.928  1.00 54.72      A    N  \\nATOM   2493  CA  GLU A 304      -5.149 -20.633  -8.360  1.00 55.49      A    C  \\nATOM   2494  C   GLU A 304      -4.185 -19.488  -8.661  1.00 55.76      A    C  \\nATOM   2495  O   GLU A 304      -3.508 -19.503  -9.687  1.00 54.20      A    O  \\nATOM   2496  CB  GLU A 304      -6.514 -20.319  -8.937  1.00 51.82      A    C  \\nATOM   2497  CG  GLU A 304      -7.479 -21.508  -8.886  1.00 49.50      A    C  \\nATOM   2498  CD  GLU A 304      -8.844 -21.154  -9.434  1.00 46.27      A    C  \\nATOM   2499  OE1 GLU A 304      -9.079 -19.975  -9.756  1.00 44.11      A    O  \\nATOM   2500  OE2 GLU A 304      -9.700 -22.068  -9.533  1.00 41.74      A    O1-\\nATOM   2501  N   SER A 305      -4.155 -18.520  -7.778  1.00 59.03      A    N  \\nATOM   2502  CA  SER A 305      -3.282 -17.374  -8.030  1.00 60.08      A    C  \\nATOM   2503  C   SER A 305      -1.855 -17.637  -7.598  1.00 59.71      A    C  \\nATOM   2504  O   SER A 305      -0.897 -17.322  -8.312  1.00 58.85      A    O  \\nATOM   2505  CB  SER A 305      -3.817 -16.120  -7.310  1.00 57.46      A    C  \\nATOM   2506  OG  SER A 305      -3.802 -16.340  -5.907  1.00 53.75      A    O  \\nATOM   2507  N   LEU A 306      -1.695 -18.228  -6.410  1.00 54.24      A    N  \\nATOM   2508  CA  LEU A 306      -0.363 -18.498  -5.884  1.00 55.78      A    C  \\nATOM   2509  C   LEU A 306       0.173 -19.823  -6.439  1.00 55.46      A    C  \\nATOM   2510  O   LEU A 306      -0.541 -20.790  -6.571  1.00 53.78      A    O  \\nATOM   2511  CB  LEU A 306      -0.368 -18.550  -4.367  1.00 52.24      A    C  \\nATOM   2512  CG  LEU A 306      -0.766 -17.270  -3.624  1.00 50.12      A    C  \\nATOM   2513  CD1 LEU A 306       0.251 -16.173  -3.933  1.00 51.16      A    C  \\nATOM   2514  CD2 LEU A 306      -0.848 -17.517  -2.130  1.00 50.31      A    C  \\nATOM   2515  N   ASN A 307       1.442 -19.826  -6.724  1.00 51.14      A    N  \\nATOM   2516  CA  ASN A 307       2.112 -21.004  -7.241  1.00 52.39      A    C  \\nATOM   2517  C   ASN A 307       3.397 -21.233  -6.445  1.00 53.09      A    C  \\nATOM   2518  O   ASN A 307       4.501 -21.127  -6.962  1.00 50.68      A    O  \\nATOM   2519  CB  ASN A 307       2.398 -20.873  -8.739  1.00 47.69      A    C  \\nATOM   2520  CG  ASN A 307       1.145 -20.903  -9.571  1.00 46.24      A    C  \\nATOM   2521  ND2 ASN A 307       1.081 -20.015 -10.562  1.00 44.65      A    N  \\nATOM   2522  OD1 ASN A 307       0.251 -21.701  -9.335  1.00 42.44      A    O  \\nATOM   2523  N   VAL A 308       3.213 -21.567  -5.163  1.00 49.22      A    N  \\nATOM   2524  CA  VAL A 308       4.349 -21.759  -4.280  1.00 50.57      A    C  \\nATOM   2525  C   VAL A 308       4.200 -23.118  -3.589  1.00 51.48      A    C  \\nATOM   2526  O   VAL A 308       3.128 -23.731  -3.617  1.00 49.48      A    O  \\nATOM   2527  CB  VAL A 308       4.469 -20.644  -3.233  1.00 46.01      A    C  \\nATOM   2528  CG1 VAL A 308       4.857 -19.326  -3.880  1.00 46.83      A    C  \\nATOM   2529  CG2 VAL A 308       3.182 -20.479  -2.478  1.00 45.05      A    C  \\nATOM   2530  N   ASN A 309       5.265 -23.544  -2.973  1.00 54.55      A    N  \\nATOM   2531  CA  ASN A 309       5.288 -24.785  -2.220  1.00 56.41      A    C  \\nATOM   2532  C   ASN A 309       5.613 -24.462  -0.772  1.00 57.38      A    C  \\nATOM   2533  O   ASN A 309       6.644 -23.866  -0.479  1.00 57.07      A    O  \\nATOM   2534  CB  ASN A 309       6.296 -25.762  -2.788  1.00 52.72      A    C  \\nATOM   2535  CG  ASN A 309       5.910 -26.272  -4.163  1.00 50.46      A    C  \\nATOM   2536  ND2 ASN A 309       6.885 -26.354  -5.048  1.00 49.15      A    N  \\nATOM   2537  OD1 ASN A 309       4.759 -26.594  -4.424  1.00 46.35      A    O  \\nATOM   2538  N   SER A 310       4.726 -24.891   0.114  1.00 54.78      A    N  \\nATOM   2539  CA  SER A 310       4.922 -24.591   1.533  1.00 56.78      A    C  \\nATOM   2540  C   SER A 310       6.045 -25.431   2.138  1.00 58.09      A    C  \\nATOM   2541  O   SER A 310       6.706 -25.000   3.095  1.00 57.22      A    O  \\nATOM   2542  CB  SER A 310       3.622 -24.798   2.315  1.00 52.84      A    C  \\nATOM   2543  OG  SER A 310       3.262 -26.181   2.304  1.00 49.29      A    O  \\nATOM   2544  N   LYS A 311       6.245 -26.636   1.621  1.00 56.12      A    N  \\nATOM   2545  CA  LYS A 311       7.306 -27.486   2.144  1.00 58.32      A    C  \\nATOM   2546  C   LYS A 311       8.682 -26.968   1.748  1.00 59.71      A    C  \\nATOM   2547  O   LYS A 311       9.623 -27.053   2.544  1.00 59.52      A    O  \\nATOM   2548  CB  LYS A 311       7.137 -28.918   1.595  1.00 55.13      A    C  \\nATOM   2549  CG  LYS A 311       5.912 -29.640   2.094  1.00 52.04      A    C  \\nATOM   2550  CD  LYS A 311       5.842 -31.037   1.542  1.00 49.03      A    C  \\nATOM   2551  CE  LYS A 311       4.634 -31.778   2.050  1.00 44.83      A    C  \\nATOM   2552  NZ  LYS A 311       4.532 -33.154   1.504  1.00 40.66      A    N1+\\nATOM   2553  N   ASP A 312       8.782 -26.415   0.581  1.00 60.89      A    N  \\nATOM   2554  CA  ASP A 312      10.069 -25.835   0.143  1.00 62.42      A    C  \\nATOM   2555  C   ASP A 312      10.379 -24.546   0.884  1.00 63.58      A    C  \\nATOM   2556  O   ASP A 312      11.544 -24.259   1.176  1.00 63.16      A    O  \\nATOM   2557  CB  ASP A 312      10.038 -25.593  -1.373  1.00 59.78      A    C  \\nATOM   2558  CG  ASP A 312       9.943 -26.875  -2.161  1.00 56.71      A    C  \\nATOM   2559  OD1 ASP A 312      10.498 -27.888  -1.701  1.00 53.67      A    O  \\nATOM   2560  OD2 ASP A 312       9.363 -26.848  -3.253  1.00 55.67      A    O1-\\nATOM   2561  N   LEU A 313       9.356 -23.769   1.159  1.00 57.83      A    N  \\nATOM   2562  CA  LEU A 313       9.571 -22.528   1.899  1.00 59.57      A    C  \\nATOM   2563  C   LEU A 313       9.890 -22.785   3.366  1.00 60.31      A    C  \\nATOM   2564  O   LEU A 313      10.592 -22.000   3.997  1.00 59.44      A    O  \\nATOM   2565  CB  LEU A 313       8.352 -21.618   1.788  1.00 56.77      A    C  \\nATOM   2566  CG  LEU A 313       8.079 -21.022   0.408  1.00 53.36      A    C  \\nATOM   2567  CD1 LEU A 313       9.215 -20.119   0.023  1.00 52.98      A    C  \\nATOM   2568  CD2 LEU A 313       6.757 -20.299   0.393  1.00 53.09      A    C  \\nATOM   2569  N   TYR A 314       9.371 -23.896   3.884  1.00 55.12      A    N  \\nATOM   2570  CA  TYR A 314       9.712 -24.251   5.262  1.00 56.73      A    C  \\nATOM   2571  C   TYR A 314      11.173 -24.650   5.370  1.00 58.22      A    C  \\nATOM   2572  O   TYR A 314      11.841 -24.317   6.366  1.00 57.90      A    O  \\nATOM   2573  CB  TYR A 314       8.813 -25.380   5.746  1.00 53.68      A    C  \\nATOM   2574  CG  TYR A 314       9.079 -25.800   7.171  1.00 50.58      A    C  \\nATOM   2575  CD1 TYR A 314       8.607 -25.060   8.228  1.00 47.88      A    C  \\nATOM   2576  CD2 TYR A 314       9.801 -26.965   7.453  1.00 47.59      A    C  \\nATOM   2577  CE1 TYR A 314       8.828 -25.449   9.539  1.00 45.12      A    C  \\nATOM   2578  CE2 TYR A 314      10.023 -27.359   8.751  1.00 46.02      A    C  \\nATOM   2579  CZ  TYR A 314       9.526 -26.594   9.777  1.00 42.55      A    C  \\nATOM   2580  OH  TYR A 314       9.766 -26.970  11.079  1.00 43.32      A    O  \\nATOM   2581  N   PHE A 315      11.690 -25.359   4.397  1.00 59.20      A    N  \\nATOM   2582  CA  PHE A 315      13.099 -25.772   4.407  1.00 60.83      A    C  \\nATOM   2583  C   PHE A 315      14.005 -24.576   4.254  1.00 61.69      A    C  \\nATOM   2584  O   PHE A 315      15.068 -24.527   4.886  1.00 61.63      A    O  \\nATOM   2585  CB  PHE A 315      13.360 -26.788   3.302  1.00 57.85      A    C  \\nATOM   2586  CG  PHE A 315      12.704 -28.133   3.496  1.00 54.25      A    C  \\nATOM   2587  CD1 PHE A 315      12.418 -28.580   4.771  1.00 52.19      A    C  \\nATOM   2588  CD2 PHE A 315      12.387 -28.938   2.413  1.00 51.97      A    C  \\nATOM   2589  CE1 PHE A 315      11.840 -29.801   4.960  1.00 50.37      A    C  \\nATOM   2590  CE2 PHE A 315      11.800 -30.159   2.614  1.00 51.06      A    C  \\nATOM   2591  CZ  PHE A 315      11.515 -30.595   3.890  1.00 47.36      A    C  \\nATOM   2592  N   GLU A 316      13.621 -23.623   3.436  1.00 61.76      A    N  \\nATOM   2593  CA  GLU A 316      14.482 -22.476   3.192  1.00 62.99      A    C  \\nATOM   2594  C   GLU A 316      14.502 -21.533   4.395  1.00 63.44      A    C  \\nATOM   2595  O   GLU A 316      15.558 -21.004   4.731  1.00 63.44      A    O  \\nATOM   2596  CB  GLU A 316      14.001 -21.718   1.942  1.00 60.29      A    C  \\nATOM   2597  CG  GLU A 316      14.167 -22.519   0.659  1.00 56.21      A    C  \\nATOM   2598  CD  GLU A 316      13.658 -21.770  -0.564  1.00 52.62      A    C  \\nATOM   2599  OE1 GLU A 316      13.081 -20.670  -0.375  1.00 50.12      A    O  \\nATOM   2600  OE2 GLU A 316      13.836 -22.263  -1.707  1.00 47.21      A    O1-\\nATOM   2601  N   VAL A 317      13.376 -21.360   5.027  1.00 58.87      A    N  \\nATOM   2602  CA  VAL A 317      13.333 -20.516   6.232  1.00 59.90      A    C  \\nATOM   2603  C   VAL A 317      14.043 -21.183   7.402  1.00 60.91      A    C  \\nATOM   2604  O   VAL A 317      14.744 -20.523   8.155  1.00 60.00      A    O  \\nATOM   2605  CB  VAL A 317      11.892 -20.162   6.571  1.00 56.17      A    C  \\nATOM   2606  CG1 VAL A 317      11.289 -19.313   5.459  1.00 57.57      A    C  \\nATOM   2607  CG2 VAL A 317      11.774 -19.399   7.852  1.00 55.60      A    C  \\nATOM   2608  N   ARG A 318      13.884 -22.495   7.512  1.00 57.24      A    N  \\nATOM   2609  CA  ARG A 318      14.509 -23.234   8.591  1.00 58.52      A    C  \\nATOM   2610  C   ARG A 318      16.029 -23.196   8.479  1.00 59.95      A    C  \\nATOM   2611  O   ARG A 318      16.750 -23.043   9.483  1.00 60.91      A    O  \\nATOM   2612  CB  ARG A 318      14.002 -24.654   8.662  1.00 55.65      A    C  \\nATOM   2613  CG  ARG A 318      14.593 -25.515   9.777  1.00 52.40      A    C  \\nATOM   2614  CD  ARG A 318      14.158 -25.018  11.126  1.00 49.60      A    C  \\nATOM   2615  NE  ARG A 318      14.665 -25.876  12.191  1.00 46.99      A    N  \\nATOM   2616  CZ  ARG A 318      14.537 -25.612  13.470  1.00 46.35      A    C  \\nATOM   2617  NH1 ARG A 318      13.934 -24.487  13.861  1.00 41.12      A    N1+\\nATOM   2618  NH2 ARG A 318      15.002 -26.438  14.370  1.00 39.54      A    N  \\nATOM   2619  N   GLU A 319      16.544 -23.377   7.274  1.00 63.08      A    N  \\nATOM   2620  CA  GLU A 319      17.969 -23.376   7.051  1.00 63.57      A    C  \\nATOM   2621  C   GLU A 319      18.596 -22.009   7.283  1.00 63.61      A    C  \\nATOM   2622  O   GLU A 319      19.723 -21.913   7.745  1.00 63.62      A    O  \\nATOM   2623  CB  GLU A 319      18.306 -23.894   5.656  1.00 61.21      A    C  \\nATOM   2624  CG  GLU A 319      18.021 -25.366   5.444  1.00 57.51      A    C  \\nATOM   2625  CD  GLU A 319      18.340 -25.827   4.040  1.00 53.42      A    C  \\nATOM   2626  OE1 GLU A 319      18.778 -24.997   3.200  1.00 53.28      A    O  \\nATOM   2627  OE2 GLU A 319      18.161 -27.035   3.754  1.00 48.80      A    O1-\\nATOM   2628  N   ILE A 320      17.849 -20.990   6.962  1.00 64.74      A    N  \\nATOM   2629  CA  ILE A 320      18.339 -19.630   7.182  1.00 64.98      A    C  \\nATOM   2630  C   ILE A 320      18.337 -19.300   8.672  1.00 65.23      A    C  \\nATOM   2631  O   ILE A 320      19.249 -18.648   9.159  1.00 63.60      A    O  \\nATOM   2632  CB  ILE A 320      17.526 -18.621   6.357  1.00 62.08      A    C  \\nATOM   2633  CG1 ILE A 320      17.757 -18.865   4.866  1.00 62.42      A    C  \\nATOM   2634  CG2 ILE A 320      17.882 -17.197   6.694  1.00 59.93      A    C  \\nATOM   2635  CD1 ILE A 320      16.842 -18.081   3.959  1.00 56.87      A    C  \\nATOM   2636  N   LEU A 321      17.334 -19.783   9.382  1.00 61.02      A    N  \\nATOM   2637  CA  LEU A 321      17.294 -19.538  10.817  1.00 61.67      A    C  \\nATOM   2638  C   LEU A 321      18.370 -20.329  11.552  1.00 62.45      A    C  \\nATOM   2639  O   LEU A 321      18.833 -19.907  12.606  1.00 61.64      A    O  \\nATOM   2640  CB  LEU A 321      15.924 -19.892  11.386  1.00 58.14      A    C  \\nATOM   2641  CG  LEU A 321      14.732 -19.001  11.028  1.00 54.34      A    C  \\nATOM   2642  CD1 LEU A 321      14.954 -17.574  11.530  1.00 54.20      A    C  \\nATOM   2643  CD2 LEU A 321      13.451 -19.581  11.580  1.00 52.96      A    C  \\nATOM   2644  N   LEU A 322      18.723 -21.492  11.025  1.00 61.44      A    N  \\nATOM   2645  CA  LEU A 322      19.821 -22.247  11.620  1.00 62.28      A    C  \\nATOM   2646  C   LEU A 322      21.191 -21.658  11.306  1.00 62.76      A    C  \\nATOM   2647  O   LEU A 322      22.086 -21.724  12.133  1.00 62.07      A    O  \\nATOM   2648  CB  LEU A 322      19.757 -23.716  11.160  1.00 59.53      A    C  \\nATOM   2649  CG  LEU A 322      18.604 -24.533  11.716  1.00 56.42      A    C  \\nATOM   2650  CD1 LEU A 322      18.719 -24.661  13.226  1.00 57.13      A    C  \\nATOM   2651  CD2 LEU A 322      18.551 -25.913  11.063  1.00 55.93      A    C  \\nATOM   2652  N   ARG A 323      21.302 -21.056  10.152  1.00 64.20      A    N  \\nATOM   2653  CA  ARG A 323      22.570 -20.451   9.789  1.00 64.68      A    C  \\nATOM   2654  C   ARG A 323      22.798 -19.099  10.479  1.00 64.03      A    C  \\nATOM   2655  O   ARG A 323      23.939 -18.691  10.664  1.00 63.44      A    O  \\nATOM   2656  CB  ARG A 323      22.684 -20.252   8.282  1.00 62.11      A    C  \\nATOM   2657  CG  ARG A 323      22.810 -21.525   7.484  1.00 58.40      A    C  \\nATOM   2658  CD  ARG A 323      22.876 -21.256   6.010  1.00 56.34      A    C  \\nATOM   2659  NE  ARG A 323      22.977 -22.501   5.221  1.00 54.52      A    N  \\nATOM   2660  CZ  ARG A 323      23.062 -22.539   3.909  1.00 52.28      A    C  \\nATOM   2661  NH1 ARG A 323      23.025 -21.426   3.197  1.00 47.11      A    N1+\\nATOM   2662  NH2 ARG A 323      23.174 -23.710   3.279  1.00 46.12      A    N  \\nATOM   2663  N   LYS A 324      21.715 -18.464  10.867  1.00 60.10      A    N  \\nATOM   2664  CA  LYS A 324      21.866 -17.172  11.569  1.00 60.73      A    C  \\nATOM   2665  C   LYS A 324      22.021 -17.339  13.073  1.00 61.06      A    C  \\nATOM   2666  O   LYS A 324      22.711 -16.557  13.698  1.00 60.19      A    O  \\nATOM   2667  CB  LYS A 324      20.664 -16.283  11.266  1.00 57.64      A    C  \\nATOM   2668  CG  LYS A 324      20.642 -15.751   9.857  1.00 53.78      A    C  \\nATOM   2669  CD  LYS A 324      19.439 -14.875   9.595  1.00 50.58      A    C  \\nATOM   2670  CE  LYS A 324      19.445 -14.330   8.175  1.00 46.12      A    C  \\nATOM   2671  NZ  LYS A 324      18.289 -13.449   7.884  1.00 42.49      A    N1+\\nATOM   2672  N   THR A 325      21.410 -18.372  13.620  1.00 58.79      A    N  \\nATOM   2673  CA  THR A 325      21.568 -18.605  15.060  1.00 59.46      A    C  \\nATOM   2674  C   THR A 325      22.861 -19.361  15.353  1.00 60.07      A    C  \\nATOM   2675  O   THR A 325      23.615 -18.971  16.260  1.00 59.18      A    O  \\nATOM   2676  CB  THR A 325      20.365 -19.409  15.588  1.00 56.75      A    C  \\nATOM   2677  CG2 THR A 325      19.100 -18.577  15.447  1.00 52.62      A    C  \\nATOM   2678  OG1 THR A 325      20.226 -20.633  14.864  1.00 55.04      A    O  \\nATOM   2679  N   LEU A 326      23.115 -20.420  14.616  1.00 60.59      A    N  \\nATOM   2680  CA  LEU A 326      24.337 -21.176  14.821  1.00 61.30      A    C  \\nATOM   2681  C   LEU A 326      25.526 -20.487  14.179  1.00 60.95      A    C  \\nATOM   2682  O   LEU A 326      26.627 -20.471  14.757  1.00 59.06      A    O  \\nATOM   2683  CB  LEU A 326      24.194 -22.583  14.254  1.00 58.88      A    C  \\nATOM   2684  CG  LEU A 326      23.178 -23.498  14.947  1.00 56.48      A    C  \\nATOM   2685  CD1 LEU A 326      23.592 -23.760  16.376  1.00 58.55      A    C  \\nATOM   2686  CD2 LEU A 326      23.007 -24.784  14.186  1.00 57.63      A    C  \\nATOM   2687  N   GLY A 327      25.330 -19.922  13.031  1.00 62.24      A    N  \\nATOM   2688  CA  GLY A 327      26.399 -19.306  12.273  1.00 63.98      A    C  \\nATOM   2689  C   GLY A 327      26.753 -20.083  11.037  1.00 65.61      A    C  \\nATOM   2690  O   GLY A 327      26.444 -21.278  10.937  1.00 64.13      A    O  \\nATOM   2691  N   ASN A 328      27.391 -19.424  10.119  1.00 63.43      A    N  \\nATOM   2692  CA  ASN A 328      27.718 -20.053   8.858  1.00 64.47      A    C  \\nATOM   2693  C   ASN A 328      28.825 -21.086   9.020  1.00 65.06      A    C  \\nATOM   2694  O   ASN A 328      28.843 -22.098   8.308  1.00 64.95      A    O  \\nATOM   2695  CB  ASN A 328      28.142 -19.006   7.816  1.00 61.79      A    C  \\nATOM   2696  CG  ASN A 328      26.972 -18.157   7.378  1.00 58.82      A    C  \\nATOM   2697  ND2 ASN A 328      27.201 -16.862   7.252  1.00 54.91      A    N  \\nATOM   2698  OD1 ASN A 328      25.883 -18.654   7.149  1.00 54.40      A    O  \\nATOM   2699  N   LYS A 329      29.724 -20.848   9.965  1.00 62.74      A    N  \\nATOM   2700  CA  LYS A 329      30.815 -21.789  10.191  1.00 63.63      A    C  \\nATOM   2701  C   LYS A 329      30.354 -23.001  10.969  1.00 64.55      A    C  \\nATOM   2702  O   LYS A 329      30.758 -24.140  10.666  1.00 63.45      A    O  \\nATOM   2703  CB  LYS A 329      31.973 -21.103  10.933  1.00 60.72      A    C  \\nATOM   2704  CG  LYS A 329      32.662 -20.038  10.105  1.00 57.63      A    C  \\nATOM   2705  CD  LYS A 329      33.793 -19.396  10.873  1.00 52.41      A    C  \\nATOM   2706  CE  LYS A 329      34.510 -18.351  10.086  1.00 47.66      A    C  \\nATOM   2707  NZ  LYS A 329      35.619 -17.706  10.818  1.00 40.83      A    N1+\\nATOM   2708  N   LEU A 330      29.510 -22.808  11.957  1.00 61.23      A    N  \\nATOM   2709  CA  LEU A 330      29.016 -23.910  12.775  1.00 63.23      A    C  \\nATOM   2710  C   LEU A 330      28.035 -24.791  12.021  1.00 64.11      A    C  \\nATOM   2711  O   LEU A 330      27.965 -26.000  12.238  1.00 63.75      A    O  \\nATOM   2712  CB  LEU A 330      28.381 -23.398  14.058  1.00 60.71      A    C  \\nATOM   2713  CG  LEU A 330      29.312 -22.730  15.067  1.00 56.38      A    C  \\nATOM   2714  CD1 LEU A 330      30.355 -23.716  15.560  1.00 57.26      A    C  \\nATOM   2715  CD2 LEU A 330      28.533 -22.119  16.201  1.00 57.27      A    C  \\nATOM   2716  N   TYR A 331      27.241 -24.174  11.147  1.00 62.80      A    N  \\nATOM   2717  CA  TYR A 331      26.277 -24.926  10.377  1.00 63.73      A    C  \\nATOM   2718  C   TYR A 331      26.962 -25.828   9.348  1.00 63.87      A    C  \\nATOM   2719  O   TYR A 331      26.521 -26.944   9.098  1.00 63.36      A    O  \\nATOM   2720  CB  TYR A 331      25.302 -23.990   9.666  1.00 61.67      A    C  \\nATOM   2721  CG  TYR A 331      24.270 -24.677   8.817  1.00 60.78      A    C  \\nATOM   2722  CD1 TYR A 331      23.212 -25.341   9.404  1.00 60.26      A    C  \\nATOM   2723  CD2 TYR A 331      24.371 -24.653   7.435  1.00 59.88      A    C  \\nATOM   2724  CE1 TYR A 331      22.267 -25.988   8.617  1.00 60.10      A    C  \\nATOM   2725  CE2 TYR A 331      23.435 -25.284   6.660  1.00 59.99      A    C  \\nATOM   2726  CZ  TYR A 331      22.380 -25.959   7.251  1.00 59.38      A    C  \\nATOM   2727  OH  TYR A 331      21.429 -26.583   6.490  1.00 55.56      A    O  \\nATOM   2728  N   LYS A 332      28.046 -25.353   8.782  1.00 65.21      A    N  \\nATOM   2729  CA  LYS A 332      28.798 -26.160   7.810  1.00 65.61      A    C  \\nATOM   2730  C   LYS A 332      29.575 -27.269   8.519  1.00 65.50      A    C  \\nATOM   2731  O   LYS A 332      29.790 -28.333   7.948  1.00 63.90      A    O  \\nATOM   2732  CB  LYS A 332      29.750 -25.306   6.998  1.00 63.34      A    C  \\nATOM   2733  CG  LYS A 332      29.012 -24.373   6.037  1.00 59.55      A    C  \\nATOM   2734  CD  LYS A 332      29.992 -23.533   5.238  1.00 56.04      A    C  \\nATOM   2735  CE  LYS A 332      29.278 -22.587   4.272  1.00 51.67      A    C  \\nATOM   2736  NZ  LYS A 332      30.196 -21.751   3.498  1.00 46.04      A    N1+\\nATOM   2737  N   LYS A 333      29.982 -27.030   9.764  1.00 64.86      A    N  \\nATOM   2738  CA  LYS A 333      30.699 -28.057  10.522  1.00 64.99      A    C  \\nATOM   2739  C   LYS A 333      29.765 -29.151  10.998  1.00 64.92      A    C  \\nATOM   2740  O   LYS A 333      30.181 -30.297  11.112  1.00 64.68      A    O  \\nATOM   2741  CB  LYS A 333      31.466 -27.434  11.682  1.00 63.09      A    C  \\nATOM   2742  CG  LYS A 333      32.638 -26.594  11.266  1.00 59.81      A    C  \\nATOM   2743  CD  LYS A 333      33.327 -26.006  12.464  1.00 56.60      A    C  \\nATOM   2744  CE  LYS A 333      34.498 -25.132  12.052  1.00 52.19      A    C  \\nATOM   2745  NZ  LYS A 333      35.202 -24.557  13.205  1.00 46.95      A    N1+\\nATOM   2746  N   LEU A 334      28.522 -28.798  11.305  1.00 63.19      A    N  \\nATOM   2747  CA  LEU A 334      27.548 -29.792  11.715  1.00 63.56      A    C  \\nATOM   2748  C   LEU A 334      27.112 -30.666  10.533  1.00 62.89      A    C  \\nATOM   2749  O   LEU A 334      26.793 -31.844  10.714  1.00 61.20      A    O  \\nATOM   2750  CB  LEU A 334      26.329 -29.136  12.350  1.00 60.18      A    C  \\nATOM   2751  CG  LEU A 334      26.551 -28.541  13.736  1.00 57.36      A    C  \\nATOM   2752  CD1 LEU A 334      26.910 -29.620  14.734  1.00 57.80      A    C  \\nATOM   2753  CD2 LEU A 334      25.318 -27.781  14.175  1.00 57.16      A    C  \\nATOM   2754  N   LEU A 335      27.069 -30.083   9.371  1.00 63.84      A    N  \\nATOM   2755  CA  LEU A 335      26.794 -30.879   8.164  1.00 65.17      A    C  \\nATOM   2756  C   LEU A 335      27.986 -31.726   7.764  1.00 64.17      A    C  \\nATOM   2757  O   LEU A 335      27.823 -32.817   7.214  1.00 62.41      A    O  \\nATOM   2758  CB  LEU A 335      26.400 -29.969   7.008  1.00 62.17      A    C  \\nATOM   2759  CG  LEU A 335      25.062 -29.263   7.164  1.00 59.03      A    C  \\nATOM   2760  CD1 LEU A 335      23.933 -30.283   7.263  1.00 59.66      A    C  \\nATOM   2761  CD2 LEU A 335      24.837 -28.316   5.985  1.00 59.12      A    C  \\nATOM   2762  N   GLY A 336      29.186 -31.255   8.050  1.00 64.37      A    N  \\nATOM   2763  CA  GLY A 336      30.375 -32.019   7.735  1.00 63.54      A    C  \\nATOM   2764  C   GLY A 336      30.648 -33.149   8.700  1.00 63.88      A    C  \\nATOM   2765  O   GLY A 336      31.228 -34.173   8.334  1.00 60.51      A    O  \\nATOM   2766  N   ASN A 337      30.209 -33.009   9.957  1.00 59.43      A    N  \\nATOM   2767  CA  ASN A 337      30.382 -34.040  10.946  1.00 59.98      A    C  \\nATOM   2768  C   ASN A 337      29.372 -35.178  10.803  1.00 59.19      A    C  \\nATOM   2769  O   ASN A 337      29.526 -36.193  11.480  1.00 56.48      A    O  \\nATOM   2770  CB  ASN A 337      30.276 -33.454  12.355  1.00 56.17      A    C  \\nATOM   2771  CG  ASN A 337      31.485 -32.626  12.707  1.00 53.49      A    C  \\nATOM   2772  ND2 ASN A 337      31.263 -31.520  13.385  1.00 50.58      A    N  \\nATOM   2773  OD1 ASN A 337      32.619 -32.966  12.361  1.00 48.24      A    O  \\nATOM   2774  N   ASN A 338      28.343 -34.989  10.002  1.00 53.39      A    N  \\nATOM   2775  CA  ASN A 338      27.295 -35.970   9.712  1.00 54.38      A    C  \\nATOM   2776  C   ASN A 338      26.556 -36.383  10.970  1.00 54.54      A    C  \\nATOM   2777  O   ASN A 338      26.138 -37.522  11.118  1.00 51.84      A    O  \\nATOM   2778  CB  ASN A 338      27.840 -37.194   8.977  1.00 48.85      A    C  \\nATOM   2779  CG  ASN A 338      28.297 -36.879   7.564  1.00 46.59      A    C  \\nATOM   2780  ND2 ASN A 338      29.421 -37.461   7.169  1.00 43.79      A    N  \\nATOM   2781  OD1 ASN A 338      27.665 -36.119   6.862  1.00 41.98      A    O  \\nATOM   2782  N   LYS A 339      26.378 -35.430  11.894  1.00 49.42      A    N  \\nATOM   2783  CA  LYS A 339      25.630 -35.701  13.106  1.00 52.03      A    C  \\nATOM   2784  C   LYS A 339      24.268 -34.997  13.097  1.00 52.51      A    C  \\nATOM   2785  O   LYS A 339      23.257 -35.598  13.422  1.00 51.45      A    O  \\nATOM   2786  CB  LYS A 339      26.425 -35.275  14.338  1.00 48.27      A    C  \\nATOM   2787  CG  LYS A 339      27.651 -36.098  14.599  1.00 46.53      A    C  \\nATOM   2788  CD  LYS A 339      28.421 -35.640  15.830  1.00 43.83      A    C  \\nATOM   2789  CE  LYS A 339      29.636 -36.504  16.096  1.00 38.18      A    C  \\nATOM   2790  NZ  LYS A 339      30.374 -36.071  17.295  1.00 35.60      A    N1+\\nATOM   2791  N   PHE A 340      24.264 -33.759  12.694  1.00 53.67      A    N  \\nATOM   2792  CA  PHE A 340      23.057 -32.955  12.654  1.00 57.10      A    C  \\nATOM   2793  C   PHE A 340      22.459 -32.999  11.253  1.00 57.68      A    C  \\nATOM   2794  O   PHE A 340      23.072 -32.501  10.304  1.00 56.24      A    O  \\nATOM   2795  CB  PHE A 340      23.380 -31.526  13.089  1.00 54.53      A    C  \\nATOM   2796  CG  PHE A 340      22.167 -30.601  13.051  1.00 52.46      A    C  \\nATOM   2797  CD1 PHE A 340      21.184 -30.681  14.025  1.00 51.22      A    C  \\nATOM   2798  CD2 PHE A 340      22.056 -29.661  12.039  1.00 51.02      A    C  \\nATOM   2799  CE1 PHE A 340      20.096 -29.836  13.989  1.00 48.48      A    C  \\nATOM   2800  CE2 PHE A 340      20.977 -28.790  12.011  1.00 49.58      A    C  \\nATOM   2801  CZ  PHE A 340      20.008 -28.885  12.978  1.00 47.70      A    C  \\nATOM   2802  N   ASN A 341      21.289 -33.583  11.156  1.00 56.67      A    N  \\nATOM   2803  CA  ASN A 341      20.588 -33.656   9.880  1.00 59.14      A    C  \\nATOM   2804  C   ASN A 341      19.407 -32.706   9.913  1.00 59.50      A    C  \\nATOM   2805  O   ASN A 341      18.382 -33.015  10.506  1.00 57.75      A    O  \\nATOM   2806  CB  ASN A 341      20.113 -35.095   9.605  1.00 55.77      A    C  \\nATOM   2807  CG  ASN A 341      21.264 -36.036   9.341  1.00 53.12      A    C  \\nATOM   2808  ND2 ASN A 341      21.177 -37.232   9.931  1.00 49.88      A    N  \\nATOM   2809  OD1 ASN A 341      22.197 -35.716   8.632  1.00 46.92      A    O  \\nATOM   2810  N   PRO A 342      19.535 -31.537   9.336  1.00 51.59      A    N  \\nATOM   2811  CA  PRO A 342      18.448 -30.558   9.396  1.00 53.55      A    C  \\nATOM   2812  C   PRO A 342      17.277 -30.921   8.512  1.00 54.03      A    C  \\nATOM   2813  O   PRO A 342      16.153 -30.444   8.754  1.00 53.19      A    O  \\nATOM   2814  CB  PRO A 342      19.122 -29.275   8.918  1.00 50.80      A    C  \\nATOM   2815  CG  PRO A 342      20.167 -29.740   7.985  1.00 50.96      A    C  \\nATOM   2816  CD  PRO A 342      20.676 -31.015   8.566  1.00 51.47      A    C  \\nATOM   2817  N   LYS A 343      17.473 -31.758   7.531  1.00 48.90      A    N  \\nATOM   2818  CA  LYS A 343      16.366 -32.153   6.645  1.00 51.77      A    C  \\nATOM   2819  C   LYS A 343      15.408 -33.091   7.361  1.00 51.92      A    C  \\nATOM   2820  O   LYS A 343      14.211 -33.049   7.087  1.00 49.98      A    O  \\nATOM   2821  CB  LYS A 343      16.905 -32.808   5.386  1.00 48.66      A    C  \\nATOM   2822  CG  LYS A 343      17.651 -31.829   4.469  1.00 46.58      A    C  \\nATOM   2823  CD  LYS A 343      18.165 -32.533   3.229  1.00 43.86      A    C  \\nATOM   2824  CE  LYS A 343      18.924 -31.589   2.339  1.00 39.67      A    C  \\nATOM   2825  NZ  LYS A 343      19.434 -32.247   1.104  1.00 38.46      A    N1+\\nATOM   2826  N   ASN A 344      15.932 -33.927   8.254  1.00 52.34      A    N  \\nATOM   2827  CA  ASN A 344      15.059 -34.822   8.998  1.00 53.13      A    C  \\nATOM   2828  C   ASN A 344      14.291 -34.077  10.068  1.00 53.75      A    C  \\nATOM   2829  O   ASN A 344      13.193 -34.471  10.434  1.00 52.71      A    O  \\nATOM   2830  CB  ASN A 344      15.876 -35.953   9.613  1.00 50.55      A    C  \\nATOM   2831  CG  ASN A 344      16.424 -36.908   8.574  1.00 48.16      A    C  \\nATOM   2832  ND2 ASN A 344      17.633 -37.361   8.767  1.00 46.30      A    N  \\nATOM   2833  OD1 ASN A 344      15.739 -37.236   7.607  1.00 43.84      A    O  \\nATOM   2834  N   ILE A 345      14.854 -33.007  10.585  1.00 48.88      A    N  \\nATOM   2835  CA  ILE A 345      14.144 -32.191  11.562  1.00 51.23      A    C  \\nATOM   2836  C   ILE A 345      13.044 -31.378  10.901  1.00 51.51      A    C  \\nATOM   2837  O   ILE A 345      11.951 -31.244  11.442  1.00 50.32      A    O  \\nATOM   2838  CB  ILE A 345      15.128 -31.289  12.333  1.00 47.72      A    C  \\nATOM   2839  CG1 ILE A 345      16.125 -32.151  13.138  1.00 51.32      A    C  \\nATOM   2840  CG2 ILE A 345      14.409 -30.358  13.287  1.00 48.05      A    C  \\nATOM   2841  CD1 ILE A 345      17.224 -31.335  13.770  1.00 45.14      A    C  \\nATOM   2842  N   HIS A 346      13.341 -30.854   9.736  1.00 50.11      A    N  \\nATOM   2843  CA  HIS A 346      12.328 -30.086   9.012  1.00 52.18      A    C  \\nATOM   2844  C   HIS A 346      11.177 -30.975   8.546  1.00 53.19      A    C  \\nATOM   2845  O   HIS A 346      10.035 -30.557   8.567  1.00 52.56      A    O  \\nATOM   2846  CB  HIS A 346      12.944 -29.382   7.814  1.00 48.24      A    C  \\nATOM   2847  CG  HIS A 346      13.778 -28.188   8.178  1.00 45.78      A    C  \\nATOM   2848  CD2 HIS A 346      13.459 -26.873   8.150  1.00 43.33      A    C  \\nATOM   2849  ND1 HIS A 346      15.104 -28.301   8.602  1.00 43.17      A    N  \\nATOM   2850  CE1 HIS A 346      15.534 -27.079   8.823  1.00 39.93      A    C  \\nATOM   2851  NE2 HIS A 346      14.586 -26.199   8.559  1.00 36.23      A    N  \\nATOM   2852  N   SER A 347      11.506 -32.217   8.173  1.00 52.90      A    N  \\nATOM   2853  CA  SER A 347      10.450 -33.160   7.768  1.00 54.94      A    C  \\nATOM   2854  C   SER A 347       9.654 -33.667   8.952  1.00 55.23      A    C  \\nATOM   2855  O   SER A 347       8.489 -34.027   8.806  1.00 54.46      A    O  \\nATOM   2856  CB  SER A 347      11.060 -34.315   6.993  1.00 51.85      A    C  \\nATOM   2857  OG  SER A 347      11.868 -35.123   7.832  1.00 50.69      A    O  \\nATOM   2858  N   HIS A 348      10.279 -33.732  10.116  1.00 54.28      A    N  \\nATOM   2859  CA  HIS A 348       9.586 -34.191  11.312  1.00 57.01      A    C  \\nATOM   2860  C   HIS A 348       8.621 -33.130  11.840  1.00 57.52      A    C  \\nATOM   2861  O   HIS A 348       7.590 -33.448  12.413  1.00 58.33      A    O  \\nATOM   2862  CB  HIS A 348      10.573 -34.585  12.392  1.00 55.25      A    C  \\nATOM   2863  CG  HIS A 348       9.930 -35.077  13.651  1.00 52.92      A    C  \\nATOM   2864  CD2 HIS A 348       9.605 -36.340  14.032  1.00 49.47      A    C  \\nATOM   2865  ND1 HIS A 348       9.602 -34.243  14.673  1.00 47.23      A    N  \\nATOM   2866  CE1 HIS A 348       9.099 -34.960  15.661  1.00 47.31      A    C  \\nATOM   2867  NE2 HIS A 348       9.072 -36.225  15.309  1.00 45.74      A    N  \\nATOM   2868  N   GLU A 349       8.976 -31.866  11.653  1.00 50.31      A    N  \\nATOM   2869  CA  GLU A 349       8.105 -30.790  12.140  1.00 52.92      A    C  \\nATOM   2870  C   GLU A 349       6.962 -30.497  11.195  1.00 53.79      A    C  \\nATOM   2871  O   GLU A 349       5.938 -29.982  11.629  1.00 54.28      A    O  \\nATOM   2872  CB  GLU A 349       8.931 -29.529  12.384  1.00 51.20      A    C  \\nATOM   2873  CG  GLU A 349       9.918 -29.642  13.531  1.00 48.15      A    C  \\nATOM   2874  CD  GLU A 349      10.723 -28.374  13.749  1.00 44.93      A    C  \\nATOM   2875  OE1 GLU A 349      10.602 -27.434  12.943  1.00 43.12      A    O  \\nATOM   2876  OE2 GLU A 349      11.483 -28.314  14.737  1.00 40.64      A    O1-\\nATOM   2877  N   ILE A 350       7.147 -30.801   9.934  1.00 47.84      A    N  \\nATOM   2878  CA  ILE A 350       6.073 -30.557   8.954  1.00 49.39      A    C  \\nATOM   2879  C   ILE A 350       4.918 -31.534   9.183  1.00 50.49      A    C  \\nATOM   2880  O   ILE A 350       3.748 -31.153   9.067  1.00 50.53      A    O  \\nATOM   2881  CB  ILE A 350       6.634 -30.647   7.515  1.00 46.77      A    C  \\nATOM   2882  CG1 ILE A 350       7.504 -29.436   7.211  1.00 46.91      A    C  \\nATOM   2883  CG2 ILE A 350       5.501 -30.737   6.494  1.00 44.50      A    C  \\nATOM   2884  CD1 ILE A 350       8.262 -29.518   5.922  1.00 43.25      A    C  \\nATOM   2885  N   LYS A 351       5.242 -32.778   9.528  1.00 52.81      A    N  \\nATOM   2886  CA  LYS A 351       4.187 -33.798   9.745  1.00 54.40      A    C  \\nATOM   2887  C   LYS A 351       3.339 -33.436  10.951  1.00 54.85      A    C  \\nATOM   2888  O   LYS A 351       2.148 -33.671  10.954  1.00 55.00      A    O  \\nATOM   2889  CB  LYS A 351       4.819 -35.159   9.912  1.00 52.14      A    C  \\nATOM   2890  CG  LYS A 351       5.440 -35.704   8.641  1.00 49.21      A    C  \\nATOM   2891  CD  LYS A 351       6.068 -37.060   8.882  1.00 45.98      A    C  \\nATOM   2892  CE  LYS A 351       6.719 -37.595   7.623  1.00 41.29      A    C  \\nATOM   2893  NZ  LYS A 351       7.305 -38.928   7.798  1.00 36.02      A    N1+\\nATOM   2894  N   LYS A 352       3.963 -32.904  11.990  1.00 53.92      A    N  \\nATOM   2895  CA  LYS A 352       3.207 -32.488  13.165  1.00 56.28      A    C  \\nATOM   2896  C   LYS A 352       2.410 -31.205  12.918  1.00 57.03      A    C  \\nATOM   2897  O   LYS A 352       1.358 -31.003  13.556  1.00 57.11      A    O  \\nATOM   2898  CB  LYS A 352       4.130 -32.316  14.386  1.00 54.34      A    C  \\nATOM   2899  CG  LYS A 352       4.725 -33.602  14.879  1.00 52.12      A    C  \\nATOM   2900  CD  LYS A 352       5.572 -33.394  16.116  1.00 48.94      A    C  \\nATOM   2901  CE  LYS A 352       6.167 -34.678  16.608  1.00 45.23      A    C  \\nATOM   2902  NZ  LYS A 352       6.993 -34.508  17.817  1.00 40.17      A    N1+\\nATOM   2903  N   LEU A 353       2.891 -30.375  12.063  1.00 48.40      A    N  \\nATOM   2904  CA  LEU A 353       2.144 -29.178  11.704  1.00 50.68      A    C  \\nATOM   2905  C   LEU A 353       0.968 -29.502  10.786  1.00 51.05      A    C  \\nATOM   2906  O   LEU A 353      -0.085 -28.861  10.877  1.00 51.54      A    O  \\nATOM   2907  CB  LEU A 353       3.040 -28.143  11.051  1.00 48.85      A    C  \\nATOM   2908  CG  LEU A 353       4.056 -27.457  11.953  1.00 46.49      A    C  \\nATOM   2909  CD1 LEU A 353       3.357 -26.598  12.993  1.00 46.94      A    C  \\nATOM   2910  CD2 LEU A 353       4.990 -26.582  11.119  1.00 46.22      A    C  \\nATOM   2911  N   GLN A 354       1.126 -30.474   9.925  1.00 51.17      A    N  \\nATOM   2912  CA  GLN A 354       0.061 -30.849   9.007  1.00 53.23      A    C  \\nATOM   2913  C   GLN A 354      -1.089 -31.562   9.720  1.00 54.03      A    C  \\nATOM   2914  O   GLN A 354      -2.238 -31.462   9.313  1.00 54.54      A    O  \\nATOM   2915  CB  GLN A 354       0.577 -31.728   7.881  1.00 51.09      A    C  \\nATOM   2916  CG  GLN A 354       1.451 -31.020   6.877  1.00 48.30      A    C  \\nATOM   2917  CD  GLN A 354       1.947 -31.914   5.779  1.00 45.03      A    C  \\nATOM   2918  NE2 GLN A 354       2.659 -31.336   4.813  1.00 41.01      A    N  \\nATOM   2919  OE1 GLN A 354       1.703 -33.115   5.810  1.00 43.47      A    O  \\nATOM   2920  N   ASP A 355      -0.750 -32.288  10.787  1.00 55.89      A    N  \\nATOM   2921  CA  ASP A 355      -1.803 -32.983  11.543  1.00 56.34      A    C  \\nATOM   2922  C   ASP A 355      -2.601 -32.031  12.399  1.00 56.74      A    C  \\nATOM   2923  O   ASP A 355      -3.725 -32.343  12.790  1.00 56.79      A    O  \\nATOM   2924  CB  ASP A 355      -1.179 -34.072  12.410  1.00 54.51      A    C  \\nATOM   2925  CG  ASP A 355      -0.605 -35.210  11.588  1.00 52.00      A    C  \\nATOM   2926  OD1 ASP A 355      -1.120 -35.485  10.504  1.00 49.43      A    O  \\nATOM   2927  OD2 ASP A 355       0.357 -35.834  12.065  1.00 50.37      A    O1-\\nATOM   2928  N   ARG A 356      -2.042 -30.866  12.722  1.00 49.35      A    N  \\nATOM   2929  CA  ARG A 356      -2.751 -29.886  13.538  1.00 51.35      A    C  \\nATOM   2930  C   ARG A 356      -3.570 -28.896  12.717  1.00 51.14      A    C  \\nATOM   2931  O   ARG A 356      -4.345 -28.143  13.292  1.00 50.83      A    O  \\nATOM   2932  CB  ARG A 356      -1.792 -29.134  14.437  1.00 49.51      A    C  \\nATOM   2933  CG  ARG A 356      -1.154 -29.973  15.503  1.00 47.28      A    C  \\nATOM   2934  CD  ARG A 356      -0.150 -29.159  16.325  1.00 45.06      A    C  \\nATOM   2935  NE  ARG A 356       0.497 -29.962  17.340  1.00 42.56      A    N  \\nATOM   2936  CZ  ARG A 356       1.394 -29.497  18.204  1.00 40.03      A    C  \\nATOM   2937  NH1 ARG A 356       1.771 -28.209  18.107  1.00 34.31      A    N1+\\nATOM   2938  NH2 ARG A 356       1.960 -30.271  19.075  1.00 32.32      A    N  \\nATOM   2939  N   LEU A 357      -3.445 -28.927  11.419  1.00 46.91      A    N  \\nATOM   2940  CA  LEU A 357      -4.247 -28.053  10.570  1.00 49.01      A    C  \\nATOM   2941  C   LEU A 357      -5.712 -28.528  10.490  1.00 49.32      A    C  \\nATOM   2942  O   LEU A 357      -6.609 -27.719  10.336  1.00 49.36      A    O  \\nATOM   2943  CB  LEU A 357      -3.675 -27.982   9.153  1.00 47.01      A    C  \\nATOM   2944  CG  LEU A 357      -2.324 -27.277   8.998  1.00 44.47      A    C  \\nATOM   2945  CD1 LEU A 357      -2.408 -25.813   9.396  1.00 44.28      A    C  \\nATOM   2946  CD2 LEU A 357      -1.836 -27.417   7.571  1.00 43.56      A    C  \\nATOM   2947  N   SER A 358      -5.907 -29.830  10.590  1.00 50.46      A    N  \\nATOM   2948  CA  SER A 358      -7.256 -30.358  10.555  1.00 51.95      A    C  \\nATOM   2949  C   SER A 358      -8.009 -30.167  11.860  1.00 51.21      A    C  \\nATOM   2950  O   SER A 358      -9.222 -29.958  11.869  1.00 49.91      A    O  \\nATOM   2951  CB  SER A 358      -7.238 -31.850  10.175  1.00 49.28      A    C  \\nATOM   2952  OG  SER A 358      -6.599 -32.616  11.175  1.00 44.19      A    O  \\nATOM   2953  N   ASP A 359      -7.294 -30.221  12.966  1.00 53.13      A    N  \\nATOM   2954  CA  ASP A 359      -7.946 -30.140  14.263  1.00 54.39      A    C  \\nATOM   2955  C   ASP A 359      -7.824 -28.770  14.902  1.00 53.73      A    C  \\nATOM   2956  O   ASP A 359      -8.806 -28.164  15.315  1.00 52.88      A    O  \\nATOM   2957  CB  ASP A 359      -7.365 -31.206  15.210  1.00 52.73      A    C  \\nATOM   2958  CG  ASP A 359      -7.700 -32.619  14.761  1.00 50.09      A    C  \\nATOM   2959  OD1 ASP A 359      -8.779 -32.819  14.178  1.00 45.01      A    O  \\nATOM   2960  OD2 ASP A 359      -6.866 -33.519  14.997  1.00 45.73      A    O1-\\nATOM   2961  N   ILE A 360      -6.598 -28.293  15.057  1.00 47.55      A    N  \\nATOM   2962  CA  ILE A 360      -6.363 -27.032  15.751  1.00 50.07      A    C  \\nATOM   2963  C   ILE A 360      -6.689 -25.869  14.827  1.00 49.73      A    C  \\nATOM   2964  O   ILE A 360      -6.124 -25.767  13.730  1.00 48.31      A    O  \\nATOM   2965  CB  ILE A 360      -4.927 -26.930  16.282  1.00 46.57      A    C  \\nATOM   2966  CG1 ILE A 360      -4.675 -28.048  17.314  1.00 49.68      A    C  \\nATOM   2967  CG2 ILE A 360      -4.687 -25.575  16.921  1.00 45.58      A    C  \\nATOM   2968  CD1 ILE A 360      -3.252 -28.131  17.765  1.00 44.40      A    C  \\nATOM   2969  N   SER A 361      -7.581 -25.032  15.254  1.00 44.13      A    N  \\nATOM   2970  CA  SER A 361      -7.894 -23.788  14.536  1.00 46.47      A    C  \\nATOM   2971  C   SER A 361      -7.260 -22.625  15.287  1.00 46.39      A    C  \\nATOM   2972  O   SER A 361      -7.607 -22.342  16.431  1.00 44.65      A    O  \\nATOM   2973  CB  SER A 361      -9.389 -23.607  14.359  1.00 43.06      A    C  \\nATOM   2974  OG  SER A 361     -10.003 -23.442  15.630  1.00 38.67      A    O  \\nATOM   2975  N   ASP A 362      -6.315 -21.998  14.648  1.00 50.98      A    N  \\nATOM   2976  CA  ASP A 362      -5.644 -20.859  15.248  1.00 53.14      A    C  \\nATOM   2977  C   ASP A 362      -6.248 -19.580  14.738  1.00 52.80      A    C  \\nATOM   2978  O   ASP A 362      -6.742 -19.520  13.607  1.00 51.90      A    O  \\nATOM   2979  CB  ASP A 362      -4.131 -20.903  14.935  1.00 50.56      A    C  \\nATOM   2980  CG  ASP A 362      -3.332 -19.922  15.795  1.00 49.17      A    C  \\nATOM   2981  OD1 ASP A 362      -3.914 -19.209  16.625  1.00 44.21      A    O  \\nATOM   2982  OD2 ASP A 362      -2.091 -19.893  15.577  1.00 45.90      A    O1-\\nATOM   2983  N   SER A 363      -6.210 -18.560  15.570  1.00 46.81      A    N  \\nATOM   2984  CA  SER A 363      -6.700 -17.241  15.182  1.00 49.36      A    C  \\nATOM   2985  C   SER A 363      -5.579 -16.475  14.517  1.00 49.32      A    C  \\nATOM   2986  O   SER A 363      -4.550 -16.204  15.126  1.00 48.19      A    O  \\nATOM   2987  CB  SER A 363      -7.239 -16.473  16.376  1.00 45.66      A    C  \\nATOM   2988  OG  SER A 363      -6.204 -16.200  17.314  1.00 43.34      A    O  \\nATOM   2989  N   SER A 364      -5.776 -16.148  13.271  1.00 46.37      A    N  \\nATOM   2990  CA  SER A 364      -4.799 -15.394  12.516  1.00 49.50      A    C  \\nATOM   2991  C   SER A 364      -5.296 -13.995  12.250  1.00 49.31      A    C  \\nATOM   2992  O   SER A 364      -6.394 -13.605  12.655  1.00 48.10      A    O  \\nATOM   2993  CB  SER A 364      -4.486 -16.095  11.203  1.00 45.80      A    C  \\nATOM   2994  OG  SER A 364      -5.627 -16.115  10.372  1.00 43.35      A    O  \\nATOM   2995  N   SER A 365      -4.482 -13.222  11.593  1.00 47.56      A    N  \\nATOM   2996  CA  SER A 365      -4.878 -11.848  11.253  1.00 50.85      A    C  \\nATOM   2997  C   SER A 365      -5.966 -11.821  10.196  1.00 51.16      A    C  \\nATOM   2998  O   SER A 365      -6.796 -10.920  10.191  1.00 49.83      A    O  \\nATOM   2999  CB  SER A 365      -3.665 -11.056  10.780  1.00 47.22      A    C  \\nATOM   3000  OG  SER A 365      -3.172 -11.564   9.552  1.00 43.68      A    O  \\nATOM   3001  N   GLN A 366      -5.957 -12.818   9.338  1.00 51.38      A    N  \\nATOM   3002  CA  GLN A 366      -7.001 -12.888   8.314  1.00 53.32      A    C  \\nATOM   3003  C   GLN A 366      -8.359 -13.260   8.909  1.00 53.07      A    C  \\nATOM   3004  O   GLN A 366      -9.389 -12.771   8.468  1.00 52.60      A    O  \\nATOM   3005  CB  GLN A 366      -6.623 -13.879   7.225  1.00 50.54      A    C  \\nATOM   3006  CG  GLN A 366      -5.389 -13.498   6.416  1.00 48.45      A    C  \\nATOM   3007  CD  GLN A 366      -5.038 -14.511   5.371  1.00 45.57      A    C  \\nATOM   3008  NE2 GLN A 366      -4.007 -14.228   4.585  1.00 40.62      A    N  \\nATOM   3009  OE1 GLN A 366      -5.695 -15.548   5.268  1.00 42.82      A    O  \\nATOM   3010  N   LYS A 367      -8.326 -14.143   9.910  1.00 45.79      A    N  \\nATOM   3011  CA  LYS A 367      -9.569 -14.544  10.552  1.00 48.80      A    C  \\nATOM   3012  C   LYS A 367     -10.150 -13.431  11.411  1.00 48.27      A    C  \\nATOM   3013  O   LYS A 367     -11.373 -13.303  11.539  1.00 46.59      A    O  \\nATOM   3014  CB  LYS A 367      -9.355 -15.795  11.405  1.00 45.46      A    C  \\nATOM   3015  CG  LYS A 367      -9.010 -17.038  10.593  1.00 44.28      A    C  \\nATOM   3016  CD  LYS A 367      -8.843 -18.242  11.483  1.00 41.28      A    C  \\nATOM   3017  CE  LYS A 367      -8.453 -19.473  10.692  1.00 36.30      A    C  \\nATOM   3018  NZ  LYS A 367      -8.324 -20.666  11.551  1.00 33.39      A    N1+\\nATOM   3019  N   ASP A 368      -9.283 -12.630  12.014  1.00 45.48      A    N  \\nATOM   3020  CA  ASP A 368      -9.772 -11.486  12.780  1.00 47.57      A    C  \\nATOM   3021  C   ASP A 368     -10.353 -10.420  11.874  1.00 47.40      A    C  \\nATOM   3022  O   ASP A 368     -11.291  -9.725  12.271  1.00 45.46      A    O  \\nATOM   3023  CB  ASP A 368      -8.636 -10.900  13.604  1.00 44.69      A    C  \\nATOM   3024  CG  ASP A 368      -8.191 -11.833  14.723  1.00 43.71      A    C  \\nATOM   3025  OD1 ASP A 368      -9.022 -12.577  15.265  1.00 37.86      A    O  \\nATOM   3026  OD2 ASP A 368      -6.999 -11.801  15.057  1.00 37.14      A    O1-\\nATOM   3027  N   ASN A 369      -9.814 -10.306  10.699  1.00 47.15      A    N  \\nATOM   3028  CA  ASN A 369     -10.377  -9.354   9.741  1.00 49.04      A    C  \\nATOM   3029  C   ASN A 369     -11.705  -9.839   9.177  1.00 49.39      A    C  \\nATOM   3030  O   ASN A 369     -12.566  -9.020   8.846  1.00 47.36      A    O  \\nATOM   3031  CB  ASN A 369      -9.370  -9.085   8.613  1.00 45.33      A    C  \\nATOM   3032  CG  ASN A 369      -8.152  -8.325   9.086  1.00 45.02      A    C  \\nATOM   3033  ND2 ASN A 369      -6.975  -8.700   8.604  1.00 40.99      A    N  \\nATOM   3034  OD1 ASN A 369      -8.255  -7.387   9.864  1.00 39.77      A    O  \\nATOM   3035  N   ASP A 370     -11.860 -11.134   9.065  1.00 44.16      A    N  \\nATOM   3036  CA  ASP A 370     -13.107 -11.693   8.567  1.00 46.74      A    C  \\nATOM   3037  C   ASP A 370     -14.221 -11.550   9.614  1.00 46.85      A    C  \\nATOM   3038  O   ASP A 370     -15.389 -11.372   9.253  1.00 45.06      A    O  \\nATOM   3039  CB  ASP A 370     -12.944 -13.145   8.178  1.00 43.89      A    C  \\nATOM   3040  CG  ASP A 370     -12.112 -13.342   6.932  1.00 43.13      A    C  \\nATOM   3041  OD1 ASP A 370     -12.128 -12.453   6.068  1.00 36.68      A    O  \\nATOM   3042  OD2 ASP A 370     -11.451 -14.406   6.821  1.00 36.59      A    O1-\\nATOM   3043  N   ASP A 371     -13.844 -11.596  10.876  1.00 46.25      A    N  \\nATOM   3044  CA  ASP A 371     -14.834 -11.372  11.918  1.00 47.83      A    C  \\nATOM   3045  C   ASP A 371     -15.255  -9.920  11.991  1.00 47.61      A    C  \\nATOM   3046  O   ASP A 371     -16.400  -9.607  12.307  1.00 46.07      A    O  \\nATOM   3047  CB  ASP A 371     -14.318 -11.842  13.272  1.00 44.62      A    C  \\nATOM   3048  CG  ASP A 371     -14.159 -13.351  13.361  1.00 44.05      A    C  \\nATOM   3049  OD1 ASP A 371     -14.961 -14.062  12.711  1.00 37.18      A    O  \\nATOM   3050  OD2 ASP A 371     -13.261 -13.800  14.066  1.00 37.31      A    O1-\\nATOM   3051  N   GLU A 372     -14.344  -9.029  11.683  1.00 40.22      A    N  \\nATOM   3052  CA  GLU A 372     -14.700  -7.633  11.612  1.00 43.10      A    C  \\nATOM   3053  C   GLU A 372     -15.569  -7.328  10.406  1.00 42.86      A    C  \\nATOM   3054  O   GLU A 372     -16.409  -6.434  10.439  1.00 40.66      A    O  \\nATOM   3055  CB  GLU A 372     -13.416  -6.767  11.563  1.00 39.73      A    C  \\nATOM   3056  CG  GLU A 372     -12.602  -6.807  12.824  1.00 39.13      A    C  \\nATOM   3057  CD  GLU A 372     -11.339  -5.981  12.741  1.00 37.12      A    C  \\nATOM   3058  OE1 GLU A 372     -11.074  -5.416  11.661  1.00 33.50      A    O  \\nATOM   3059  OE2 GLU A 372     -10.603  -5.893  13.755  1.00 31.29      A    O1-\\nATOM   3060  N   ASP A 373     -15.340  -8.061   9.334  1.00 43.46      A    N  \\nATOM   3061  CA  ASP A 373     -16.184  -7.895   8.157  1.00 45.20      A    C  \\nATOM   3062  C   ASP A 373     -17.568  -8.490   8.374  1.00 44.74      A    C  \\nATOM   3063  O   ASP A 373     -18.560  -7.941   7.853  1.00 42.97      A    O  \\nATOM   3064  CB  ASP A 373     -15.510  -8.543   6.934  1.00 41.71      A    C  \\nATOM   3065  CG  ASP A 373     -14.278  -7.801   6.478  1.00 40.91      A    C  \\nATOM   3066  OD1 ASP A 373     -14.217  -6.559   6.627  1.00 35.19      A    O  \\nATOM   3067  OD2 ASP A 373     -13.334  -8.452   5.949  1.00 35.21      A    O1-\\nATOM   3068  N   GLU A 374     -17.630  -9.561   9.113  1.00 40.60      A    N  \\nATOM   3069  CA  GLU A 374     -18.922 -10.133   9.421  1.00 42.74      A    C  \\nATOM   3070  C   GLU A 374     -19.696  -9.230  10.405  1.00 42.50      A    C  \\nATOM   3071  O   GLU A 374     -20.915  -9.147  10.332  1.00 40.47      A    O  \\nATOM   3072  CB  GLU A 374     -18.806 -11.544   9.979  1.00 39.94      A    C  \\nATOM   3073  CG  GLU A 374     -18.361 -12.558   8.962  1.00 39.00      A    C  \\nATOM   3074  CD  GLU A 374     -18.187 -13.947   9.553  1.00 37.10      A    C  \\nATOM   3075  OE1 GLU A 374     -18.339 -14.109  10.774  1.00 33.26      A    O  \\nATOM   3076  OE2 GLU A 374     -17.883 -14.899   8.772  1.00 31.41      A    O1-\\nATOM   3077  N   ASP A 375     -18.974  -8.615  11.312  1.00 45.23      A    N  \\nATOM   3078  CA  ASP A 375     -19.608  -7.669  12.221  1.00 47.32      A    C  \\nATOM   3079  C   ASP A 375     -20.044  -6.391  11.478  1.00 46.84      A    C  \\nATOM   3080  O   ASP A 375     -20.984  -5.740  11.895  1.00 45.13      A    O  \\nATOM   3081  CB  ASP A 375     -18.656  -7.299  13.367  1.00 43.74      A    C  \\nATOM   3082  CG  ASP A 375     -18.422  -8.469  14.312  1.00 42.74      A    C  \\nATOM   3083  OD1 ASP A 375     -19.327  -9.318  14.458  1.00 36.90      A    O  \\nATOM   3084  OD2 ASP A 375     -17.310  -8.511  14.891  1.00 37.25      A    O1-\\nATOM   3085  N   ALA A 376     -19.338  -6.069  10.445  1.00 41.56      A    N  \\nATOM   3086  CA  ALA A 376     -19.727  -4.927   9.635  1.00 43.52      A    C  \\nATOM   3087  C   ALA A 376     -20.914  -5.238   8.740  1.00 43.56      A    C  \\nATOM   3088  O   ALA A 376     -21.684  -4.345   8.384  1.00 42.52      A    O  \\nATOM   3089  CB  ALA A 376     -18.565  -4.443   8.772  1.00 41.26      A    C  \\nATOM   3090  N   ASP A 377     -21.067  -6.514   8.395  1.00 45.13      A    N  \\nATOM   3091  CA  ASP A 377     -22.205  -6.916   7.591  1.00 47.57      A    C  \\nATOM   3092  C   ASP A 377     -23.490  -6.959   8.420  1.00 47.24      A    C  \\nATOM   3093  O   ASP A 377     -24.588  -6.747   7.880  1.00 45.71      A    O  \\nATOM   3094  CB  ASP A 377     -21.944  -8.280   6.933  1.00 44.31      A    C  \\nATOM   3095  CG  ASP A 377     -20.888  -8.225   5.861  1.00 42.62      A    C  \\nATOM   3096  OD1 ASP A 377     -20.783  -7.200   5.166  1.00 37.23      A    O  \\nATOM   3097  OD2 ASP A 377     -20.138  -9.231   5.711  1.00 37.56      A    O1-\\nATOM   3098  N   GLU A 378     -23.334  -7.277   9.703  1.00 43.38      A    N  \\nATOM   3099  CA  GLU A 378     -24.511  -7.293  10.542  1.00 46.15      A    C  \\nATOM   3100  C   GLU A 378     -24.983  -5.906  10.890  1.00 45.83      A    C  \\nATOM   3101  O   GLU A 378     -26.194  -5.698  11.101  1.00 44.31      A    O  \\nATOM   3102  CB  GLU A 378     -24.213  -8.086  11.817  1.00 43.81      A    C  \\nATOM   3103  CG  GLU A 378     -23.969  -9.560  11.589  1.00 43.19      A    C  \\nATOM   3104  CD  GLU A 378     -23.641 -10.300  12.869  1.00 40.31      A    C  \\nATOM   3105  OE1 GLU A 378     -23.514  -9.634  13.933  1.00 37.73      A    O  \\nATOM   3106  OE2 GLU A 378     -23.512 -11.555  12.823  1.00 36.03      A    O1-\\nATOM   3107  N   GLU A 379     -24.070  -4.977  11.027  1.00 44.02      A    N  \\nATOM   3108  CA  GLU A 379     -24.417  -3.605  11.281  1.00 47.55      A    C  \\nATOM   3109  C   GLU A 379     -24.750  -2.885   9.996  1.00 47.19      A    C  \\nATOM   3110  O   GLU A 379     -24.538  -3.358   8.910  1.00 44.76      A    O  \\nATOM   3111  CB  GLU A 379     -23.281  -2.899  12.019  1.00 43.42      A    C  \\nATOM   3112  CG  GLU A 379     -23.029  -3.430  13.417  1.00 42.49      A    C  \\nATOM   3113  CD  GLU A 379     -21.862  -2.736  14.097  1.00 39.67      A    C  \\nATOM   3114  OE1 GLU A 379     -21.208  -1.869  13.470  1.00 35.85      A    O  \\nATOM   3115  OE2 GLU A 379     -21.588  -3.043  15.286  1.00 33.38      A    O1-\\nATOM   3116  N   SER A 380     -25.304  -1.684  10.175  1.00 40.67      A    N  \\nATOM   3117  CA  SER A 380     -25.706  -0.889   9.028  1.00 42.89      A    C  \\nATOM   3118  C   SER A 380     -24.493  -0.292   8.345  1.00 42.33      A    C  \\nATOM   3119  O   SER A 380     -23.504   0.066   8.992  1.00 41.03      A    O  \\nATOM   3120  CB  SER A 380     -26.687   0.196   9.427  1.00 39.52      A    C  \\nATOM   3121  OG  SER A 380     -26.055   1.126  10.316  1.00 37.88      A    O  \\nATOM   3122  N   SER A 381     -24.563  -0.180   7.062  1.00 42.87      A    N  \\nATOM   3123  CA  SER A 381     -23.474   0.387   6.266  1.00 47.10      A    C  \\nATOM   3124  C   SER A 381     -23.496   1.917   6.254  1.00 46.87      A    C  \\nATOM   3125  O   SER A 381     -22.568   2.530   5.743  1.00 45.45      A    O  \\nATOM   3126  CB  SER A 381     -23.527  -0.132   4.840  1.00 43.63      A    C  \\nATOM   3127  OG  SER A 381     -24.702   0.315   4.184  1.00 39.97      A    O  \\nATOM   3128  N   GLU A 382     -24.576   2.490   6.761  1.00 45.12      A    N  \\nATOM   3129  CA  GLU A 382     -24.681   3.955   6.770  1.00 47.81      A    C  \\nATOM   3130  C   GLU A 382     -23.793   4.535   7.860  1.00 47.96      A    C  \\nATOM   3131  O   GLU A 382     -23.719   4.016   8.974  1.00 46.59      A    O  \\nATOM   3132  CB  GLU A 382     -26.120   4.385   6.995  1.00 45.32      A    C  \\nATOM   3133  CG  GLU A 382     -27.031   4.012   5.839  1.00 44.10      A    C  \\nATOM   3134  CD  GLU A 382     -28.455   4.453   6.047  1.00 40.77      A    C  \\nATOM   3135  OE1 GLU A 382     -28.783   4.959   7.152  1.00 39.39      A    O  \\nATOM   3136  OE2 GLU A 382     -29.295   4.243   5.129  1.00 36.80      A    O1-\\nATOM   3137  N   ILE A 383     -23.160   5.632   7.542  1.00 42.09      A    N  \\nATOM   3138  CA  ILE A 383     -22.263   6.300   8.452  1.00 46.50      A    C  \\nATOM   3139  C   ILE A 383     -22.973   7.516   9.019  1.00 47.21      A    C  \\nATOM   3140  O   ILE A 383     -23.452   8.369   8.266  1.00 45.68      A    O  \\nATOM   3141  CB  ILE A 383     -20.945   6.705   7.739  1.00 42.59      A    C  \\nATOM   3142  CG1 ILE A 383     -20.192   5.455   7.283  1.00 45.99      A    C  \\nATOM   3143  CG2 ILE A 383     -20.058   7.522   8.689  1.00 43.08      A    C  \\nATOM   3144  CD1 ILE A 383     -18.987   5.774   6.438  1.00 41.45      A    C  \\nATOM   3145  N   ASP A 384     -23.053   7.588  10.321  1.00 48.34      A    N  \\nATOM   3146  CA  ASP A 384     -23.695   8.711  10.974  1.00 50.27      A    C  \\nATOM   3147  C   ASP A 384     -22.785   9.943  10.943  1.00 50.63      A    C  \\nATOM   3148  O   ASP A 384     -21.566   9.825  10.827  1.00 49.72      A    O  \\nATOM   3149  CB  ASP A 384     -24.087   8.382  12.419  1.00 47.63      A    C  \\nATOM   3150  CG  ASP A 384     -25.171   7.320  12.468  1.00 46.05      A    C  \\nATOM   3151  OD1 ASP A 384     -26.041   7.311  11.568  1.00 41.00      A    O  \\nATOM   3152  OD2 ASP A 384     -25.157   6.524  13.420  1.00 41.06      A    O1-\\nATOM   3153  N   ALA A 385     -23.406  11.090  11.073  1.00 47.29      A    N  \\nATOM   3154  CA  ALA A 385     -22.624  12.330  11.094  1.00 50.71      A    C  \\nATOM   3155  C   ALA A 385     -22.040  12.627  12.445  1.00 51.59      A    C  \\nATOM   3156  O   ALA A 385     -21.085  13.393  12.574  1.00 51.50      A    O  \\nATOM   3157  CB  ALA A 385     -23.497  13.506  10.653  1.00 48.58      A    C  \\nATOM   3158  N   GLU A 386     -22.599  12.011  13.471  1.00 45.25      A    N  \\nATOM   3159  CA  GLU A 386     -22.105  12.236  14.813  1.00 48.99      A    C  \\nATOM   3160  C   GLU A 386     -20.829  11.468  15.103  1.00 49.40      A    C  \\nATOM   3161  O   GLU A 386     -20.046  11.857  15.988  1.00 49.43      A    O  \\nATOM   3162  CB  GLU A 386     -23.171  11.863  15.837  1.00 47.44      A    C  \\nATOM   3163  CG  GLU A 386     -24.427  12.707  15.767  1.00 45.49      A    C  \\nATOM   3164  CD  GLU A 386     -25.443  12.341  16.810  1.00 42.48      A    C  \\nATOM   3165  OE1 GLU A 386     -25.219  11.343  17.538  1.00 41.24      A    O  \\nATOM   3166  OE2 GLU A 386     -26.485  13.041  16.917  1.00 37.75      A    O1-\\nATOM   3167  N   GLU A 387     -20.592  10.396  14.371  1.00 42.74      A    N  \\nATOM   3168  CA  GLU A 387     -19.388   9.584  14.559  1.00 45.91      A    C  \\nATOM   3169  C   GLU A 387     -18.196  10.132  13.788  1.00 45.86      A    C  \\nATOM   3170  O   GLU A 387     -17.110   9.603  13.904  1.00 45.81      A    O  \\nATOM   3171  CB  GLU A 387     -19.669   8.138  14.150  1.00 44.31      A    C  \\nATOM   3172  CG  GLU A 387     -20.670   7.428  15.025  1.00 43.12      A    C  \\nATOM   3173  CD  GLU A 387     -20.942   6.015  14.579  1.00 41.40      A    C  \\nATOM   3174  OE1 GLU A 387     -20.440   5.622  13.502  1.00 40.58      A    O  \\nATOM   3175  OE2 GLU A 387     -21.631   5.278  15.320  1.00 36.88      A    O1-\\nATOM   3176  N   ARG A 388     -18.420  11.148  13.002  1.00 44.12      A    N  \\nATOM   3177  CA  ARG A 388     -17.332  11.717  12.216  1.00 47.27      A    C  \\nATOM   3178  C   ARG A 388     -16.453  12.588  13.098  1.00 47.14      A    C  \\nATOM   3179  O   ARG A 388     -16.905  13.353  13.931  1.00 47.32      A    O  \\nATOM   3180  CB  ARG A 388     -17.887  12.541  11.058  1.00 45.36      A    C  \\nATOM   3181  CG  ARG A 388     -18.684  11.729  10.064  1.00 43.51      A    C  \\nATOM   3182  CD  ARG A 388     -19.364  12.599   9.030  1.00 41.10      A    C  \\nATOM   3183  NE  ARG A 388     -20.175  11.791   8.117  1.00 37.13      A    N  \\nATOM   3184  CZ  ARG A 388     -20.970  12.300   7.179  1.00 37.06      A    C  \\nATOM   3185  NH1 ARG A 388     -21.086  13.611   7.036  1.00 33.10      A    N1+\\nATOM   3186  NH2 ARG A 388     -21.647  11.496   6.396  1.00 30.47      A    N  \\nATOM   3187  N   GLU A 389     -15.162  12.456  12.893  1.00 46.00      A    N  \\nATOM   3188  CA  GLU A 389     -14.180  13.174  13.663  1.00 48.91      A    C  \\nATOM   3189  C   GLU A 389     -13.173  13.830  12.741  1.00 48.98      A    C  \\nATOM   3190  O   GLU A 389     -12.679  13.225  11.817  1.00 50.22      A    O  \\nATOM   3191  CB  GLU A 389     -13.478  12.281  14.670  1.00 47.90      A    C  \\nATOM   3192  CG  GLU A 389     -14.370  11.752  15.772  1.00 46.76      A    C  \\nATOM   3193  CD  GLU A 389     -13.640  10.873  16.768  1.00 43.08      A    C  \\nATOM   3194  OE1 GLU A 389     -12.461  10.581  16.523  1.00 42.13      A    O  \\nATOM   3195  OE2 GLU A 389     -14.257  10.471  17.765  1.00 37.41      A    O1-\\nATOM   3196  N   ALA A 390     -12.881  15.057  13.043  1.00 46.90      A    N  \\nATOM   3197  CA  ALA A 390     -11.894  15.806  12.271  1.00 49.76      A    C  \\nATOM   3198  C   ALA A 390     -10.585  15.830  13.053  1.00 50.18      A    C  \\nATOM   3199  O   ALA A 390     -10.515  16.321  14.175  1.00 50.73      A    O  \\nATOM   3200  CB  ALA A 390     -12.364  17.236  12.007  1.00 48.47      A    C  \\nATOM   3201  N   ILE A 391      -9.584  15.271  12.444  1.00 45.11      A    N  \\nATOM   3202  CA  ILE A 391      -8.277  15.167  13.086  1.00 48.63      A    C  \\nATOM   3203  C   ILE A 391      -7.455  16.401  12.744  1.00 48.08      A    C  \\nATOM   3204  O   ILE A 391      -7.169  16.684  11.564  1.00 47.91      A    O  \\nATOM   3205  CB  ILE A 391      -7.552  13.878  12.675  1.00 46.74      A    C  \\nATOM   3206  CG1 ILE A 391      -8.356  12.646  13.138  1.00 51.88      A    C  \\nATOM   3207  CG2 ILE A 391      -6.142  13.841  13.255  1.00 47.37      A    C  \\nATOM   3208  CD1 ILE A 391      -7.868  11.359  12.569  1.00 46.27      A    C  \\nATOM   3209  N   ILE A 392      -7.090  17.122  13.753  1.00 40.46      A    N  \\nATOM   3210  CA  ILE A 392      -6.253  18.298  13.601  1.00 44.20      A    C  \\nATOM   3211  C   ILE A 392      -4.788  17.875  13.654  1.00 44.31      A    C  \\nATOM   3212  O   ILE A 392      -4.399  17.007  14.427  1.00 43.88      A    O  \\nATOM   3213  CB  ILE A 392      -6.586  19.349  14.678  1.00 41.52      A    C  \\nATOM   3214  CG1 ILE A 392      -8.050  19.758  14.568  1.00 46.67      A    C  \\nATOM   3215  CG2 ILE A 392      -5.695  20.584  14.542  1.00 42.53      A    C  \\nATOM   3216  CD1 ILE A 392      -8.534  20.606  15.730  1.00 42.83      A    C  \\nATOM   3217  N   ARG A 393      -3.996  18.497  12.826  1.00 46.33      A    N  \\nATOM   3218  CA  ARG A 393      -2.567  18.181  12.803  1.00 48.29      A    C  \\nATOM   3219  C   ARG A 393      -1.922  18.577  14.128  1.00 47.79      A    C  \\nATOM   3220  O   ARG A 393      -2.102  19.666  14.645  1.00 46.94      A    O  \\nATOM   3221  CB  ARG A 393      -1.878  18.889  11.647  1.00 45.81      A    C  \\nATOM   3222  CG  ARG A 393      -2.263  18.405  10.267  1.00 44.71      A    C  \\nATOM   3223  CD  ARG A 393      -1.509  19.123   9.191  1.00 43.31      A    C  \\nATOM   3224  NE  ARG A 393      -1.845  18.644   7.837  1.00 40.64      A    N  \\nATOM   3225  CZ  ARG A 393      -1.317  19.108   6.732  1.00 39.44      A    C  \\nATOM   3226  NH1 ARG A 393      -0.408  20.063   6.789  1.00 34.32      A    N1+\\nATOM   3227  NH2 ARG A 393      -1.677  18.610   5.553  1.00 32.16      A    N  \\nATOM   3228  N   SER A 394      -1.129  17.649  14.630  1.00 42.26      A    N  \\nATOM   3229  CA  SER A 394      -0.443  17.878  15.899  1.00 44.37      A    C  \\nATOM   3230  C   SER A 394       0.920  17.222  15.833  1.00 44.14      A    C  \\nATOM   3231  O   SER A 394       1.213  16.377  15.009  1.00 43.75      A    O  \\nATOM   3232  CB  SER A 394      -1.266  17.330  17.085  1.00 42.10      A    C  \\nATOM   3233  OG  SER A 394      -1.359  15.917  17.018  1.00 38.85      A    O  \\nATOM   3234  N   LYS A 395       1.754  17.655  16.764  1.00 38.19      A    N  \\nATOM   3235  CA  LYS A 395       3.122  17.111  16.822  1.00 42.74      A    C  \\nATOM   3236  C   LYS A 395       3.130  15.722  17.416  1.00 42.64      A    C  \\nATOM   3237  O   LYS A 395       3.735  14.792  16.876  1.00 41.25      A    O  \\nATOM   3238  CB  LYS A 395       4.005  18.031  17.641  1.00 39.12      A    C  \\nATOM   3239  CG  LYS A 395       4.258  19.366  16.967  1.00 38.36      A    C  \\nATOM   3240  CD  LYS A 395       5.172  20.238  17.795  1.00 36.91      A    C  \\nATOM   3241  CE  LYS A 395       5.446  21.562  17.168  1.00 32.19      A    C  \\nATOM   3242  NZ  LYS A 395       6.345  22.408  17.952  1.00 30.48      A    N1+\\nATOM   3243  N   SER A 396       2.462  15.565  18.534  1.00 40.03      A    N  \\nATOM   3244  CA  SER A 396       2.388  14.271  19.212  1.00 43.33      A    C  \\nATOM   3245  C   SER A 396       1.147  13.520  18.751  1.00 43.16      A    C  \\nATOM   3246  O   SER A 396       0.122  14.147  18.470  1.00 42.47      A    O  \\nATOM   3247  CB  SER A 396       2.370  14.445  20.728  1.00 40.95      A    C  \\nATOM   3248  OG  SER A 396       1.209  15.144  21.134  1.00 38.39      A    O  \\nATOM   3249  N   PRO A 397       1.193  12.218  18.705  1.00 38.17      A    N  \\nATOM   3250  CA  PRO A 397      -0.002  11.433  18.341  1.00 40.22      A    C  \\nATOM   3251  C   PRO A 397      -1.030  11.442  19.475  1.00 40.57      A    C  \\nATOM   3252  O   PRO A 397      -0.682  11.626  20.638  1.00 39.95      A    O  \\nATOM   3253  CB  PRO A 397       0.544  10.039  18.080  1.00 37.64      A    C  \\nATOM   3254  CG  PRO A 397       1.769   9.975  18.920  1.00 38.07      A    C  \\nATOM   3255  CD  PRO A 397       2.374  11.342  18.870  1.00 39.56      A    C  \\nATOM   3256  N   PRO A 398      -2.282  11.230  19.169  1.00 41.20      A    N  \\nATOM   3257  CA  PRO A 398      -3.282  11.164  20.226  1.00 44.56      A    C  \\nATOM   3258  C   PRO A 398      -3.189   9.831  20.976  1.00 45.33      A    C  \\nATOM   3259  O   PRO A 398      -2.446   8.919  20.592  1.00 44.85      A    O  \\nATOM   3260  CB  PRO A 398      -4.604  11.304  19.484  1.00 42.19      A    C  \\nATOM   3261  CG  PRO A 398      -4.315  10.803  18.111  1.00 42.46      A    C  \\nATOM   3262  CD  PRO A 398      -2.904  11.226  17.819  1.00 44.13      A    C  \\nATOM   3263  N   SER A 399      -3.993   9.733  22.003  1.00 43.65      A    N  \\nATOM   3264  CA  SER A 399      -3.964   8.531  22.848  1.00 46.67      A    C  \\nATOM   3265  C   SER A 399      -4.635   7.335  22.179  1.00 47.19      A    C  \\nATOM   3266  O   SER A 399      -4.442   6.211  22.630  1.00 46.96      A    O  \\nATOM   3267  CB  SER A 399      -4.647   8.825  24.177  1.00 44.35      A    C  \\nATOM   3268  OG  SER A 399      -6.029   9.091  24.006  1.00 40.69      A    O  \\nATOM   3269  N   GLU A 400      -5.371   7.577  21.123  1.00 41.84      A    N  \\nATOM   3270  CA  GLU A 400      -6.014   6.472  20.431  1.00 44.65      A    C  \\nATOM   3271  C   GLU A 400      -5.048   5.667  19.612  1.00 44.41      A    C  \\nATOM   3272  O   GLU A 400      -5.198   4.461  19.495  1.00 43.16      A    O  \\nATOM   3273  CB  GLU A 400      -7.130   7.000  19.527  1.00 41.40      A    C  \\nATOM   3274  CG  GLU A 400      -8.294   7.624  20.299  1.00 40.60      A    C  \\nATOM   3275  CD  GLU A 400      -9.378   8.128  19.372  1.00 38.78      A    C  \\nATOM   3276  OE1 GLU A 400      -9.202   8.040  18.113  1.00 38.55      A    O  \\nATOM   3277  OE2 GLU A 400     -10.422   8.613  19.877  1.00 37.66      A    O1-\\nATOM   3278  N   PHE A 401      -4.029   6.331  19.088  1.00 37.66      A    N  \\nATOM   3279  CA  PHE A 401      -3.002   5.641  18.335  1.00 40.80      A    C  \\nATOM   3280  C   PHE A 401      -1.952   5.012  19.263  1.00 40.86      A    C  \\nATOM   3281  O   PHE A 401      -1.302   4.058  18.894  1.00 39.60      A    O  \\nATOM   3282  CB  PHE A 401      -2.313   6.583  17.352  1.00 37.09      A    C  \\nATOM   3283  CG  PHE A 401      -3.250   7.142  16.288  1.00 35.11      A    C  \\nATOM   3284  CD1 PHE A 401      -3.601   6.347  15.203  1.00 34.18      A    C  \\nATOM   3285  CD2 PHE A 401      -3.777   8.399  16.419  1.00 34.04      A    C  \\nATOM   3286  CE1 PHE A 401      -4.449   6.847  14.227  1.00 31.40      A    C  \\nATOM   3287  CE2 PHE A 401      -4.625   8.902  15.446  1.00 31.48      A    C  \\nATOM   3288  CZ  PHE A 401      -4.952   8.119  14.349  1.00 30.46      A    C  \\nATOM   3289  N   ALA A 402      -1.808   5.581  20.445  1.00 39.58      A    N  \\nATOM   3290  CA  ALA A 402      -0.842   5.038  21.401  1.00 42.07      A    C  \\nATOM   3291  C   ALA A 402      -1.347   3.759  22.050  1.00 42.55      A    C  \\nATOM   3292  O   ALA A 402      -0.546   2.951  22.503  1.00 41.78      A    O  \\nATOM   3293  CB  ALA A 402      -0.513   6.096  22.468  1.00 39.85      A    C  \\nATOM   3294  N   ASP A 403      -2.646   3.585  22.091  1.00 42.35      A    N  \\nATOM   3295  CA  ASP A 403      -3.208   2.365  22.670  1.00 44.62      A    C  \\nATOM   3296  C   ASP A 403      -3.111   1.182  21.740  1.00 44.78      A    C  \\nATOM   3297  O   ASP A 403      -3.250   0.037  22.198  1.00 43.63      A    O  \\nATOM   3298  CB  ASP A 403      -4.671   2.607  23.068  1.00 42.92      A    C  \\nATOM   3299  CG  ASP A 403      -4.812   3.550  24.242  1.00 42.16      A    C  \\nATOM   3300  OD1 ASP A 403      -3.895   3.596  25.074  1.00 37.92      A    O  \\nATOM   3301  OD2 ASP A 403      -5.840   4.260  24.316  1.00 38.22      A    O1-\\nATOM   3302  N   GLN A 404      -2.903   1.442  20.476  1.00 39.47      A    N  \\nATOM   3303  CA  GLN A 404      -2.810   0.335  19.517  1.00 43.09      A    C  \\nATOM   3304  C   GLN A 404      -1.447  -0.358  19.618  1.00 43.03      A    C  \\nATOM   3305  O   GLN A 404      -1.356  -1.566  19.351  1.00 41.73      A    O  \\nATOM   3306  CB  GLN A 404      -3.035   0.848  18.099  1.00 40.07      A    C  \\nATOM   3307  CG  GLN A 404      -4.445   1.331  17.853  1.00 39.31      A    C  \\nATOM   3308  CD  GLN A 404      -4.654   1.841  16.460  1.00 37.27      A    C  \\nATOM   3309  NE2 GLN A 404      -5.863   2.261  16.151  1.00 32.78      A    N  \\nATOM   3310  OE1 GLN A 404      -3.734   1.811  15.633  1.00 36.06      A    O  \\nATOM   3311  N   TYR A 405      -0.413   0.395  19.968  1.00 33.83      A    N  \\nATOM   3312  CA  TYR A 405       0.921  -0.172  20.023  1.00 37.14      A    C  \\nATOM   3313  C   TYR A 405       1.372  -0.423  21.456  1.00 37.09      A    C  \\nATOM   3314  O   TYR A 405       2.576  -0.412  21.738  1.00 35.76      A    O  \\nATOM   3315  CB  TYR A 405       1.893   0.761  19.299  1.00 33.74      A    C  \\nATOM   3316  CG  TYR A 405       1.620   0.893  17.816  1.00 32.54      A    C  \\nATOM   3317  CD1 TYR A 405       2.068  -0.068  16.915  1.00 31.35      A    C  \\nATOM   3318  CD2 TYR A 405       0.911   1.980  17.322  1.00 30.74      A    C  \\nATOM   3319  CE1 TYR A 405       1.809   0.050  15.571  1.00 27.13      A    C  \\nATOM   3320  CE2 TYR A 405       0.648   2.096  15.973  1.00 27.53      A    C  \\nATOM   3321  CZ  TYR A 405       1.116   1.120  15.115  1.00 26.24      A    C  \\nATOM   3322  OH  TYR A 405       0.877   1.246  13.764  1.00 31.37      A    O  \\nATOM   3323  N   THR A 406       0.427  -0.646  22.333  1.00 40.23      A    N  \\nATOM   3324  CA  THR A 406       0.748  -0.936  23.717  1.00 40.83      A    C  \\nATOM   3325  C   THR A 406       1.216  -2.387  23.864  1.00 41.03      A    C  \\nATOM   3326  O   THR A 406       0.801  -3.264  23.112  1.00 39.55      A    O  \\nATOM   3327  CB  THR A 406      -0.455  -0.683  24.614  1.00 38.35      A    C  \\nATOM   3328  CG2 THR A 406      -0.819   0.797  24.605  1.00 38.03      A    C  \\nATOM   3329  OG1 THR A 406      -1.572  -1.455  24.190  1.00 39.93      A    O  \\nATOM   3330  N   GLY A 407       2.057  -2.597  24.865  1.00 45.24      A    N  \\nATOM   3331  CA  GLY A 407       2.531  -3.933  25.115  1.00 45.40      A    C  \\nATOM   3332  C   GLY A 407       1.521  -4.850  25.761  1.00 47.53      A    C  \\nATOM   3333  O   GLY A 407       1.708  -6.068  25.788  1.00 47.22      A    O  \\nATOM   3334  N   GLU A 408       0.464  -4.306  26.314  1.00 41.89      A    N  \\nATOM   3335  CA  GLU A 408      -0.555  -5.111  26.954  1.00 45.53      A    C  \\nATOM   3336  C   GLU A 408      -1.371  -5.880  25.948  1.00 45.71      A    C  \\nATOM   3337  O   GLU A 408      -1.788  -7.008  26.222  1.00 44.58      A    O  \\nATOM   3338  CB  GLU A 408      -1.456  -4.230  27.817  1.00 43.03      A    C  \\nATOM   3339  CG  GLU A 408      -0.781  -3.585  28.992  1.00 41.49      A    C  \\nATOM   3340  CD  GLU A 408      -1.700  -2.740  29.835  1.00 39.10      A    C  \\nATOM   3341  OE1 GLU A 408      -2.900  -2.597  29.457  1.00 37.94      A    O  \\nATOM   3342  OE2 GLU A 408      -1.257  -2.213  30.877  1.00 36.32      A    O1-\\nATOM   3343  N   LYS A 409      -1.613  -5.271  24.810  1.00 38.80      A    N  \\nATOM   3344  CA  LYS A 409      -2.340  -6.000  23.756  1.00 41.52      A    C  \\nATOM   3345  C   LYS A 409      -1.476  -7.026  23.072  1.00 41.25      A    C  \\nATOM   3346  O   LYS A 409      -1.979  -7.993  22.522  1.00 40.02      A    O  \\nATOM   3347  CB  LYS A 409      -2.908  -5.020  22.719  1.00 38.67      A    C  \\nATOM   3348  CG  LYS A 409      -3.997  -4.132  23.281  1.00 37.34      A    C  \\nATOM   3349  CD  LYS A 409      -4.536  -3.203  22.219  1.00 35.78      A    C  \\nATOM   3350  CE  LYS A 409      -5.617  -2.287  22.790  1.00 29.88      A    C  \\nATOM   3351  NZ  LYS A 409      -6.180  -1.390  21.790  1.00 30.07      A    N1+\\nATOM   3352  N   ARG A 410      -0.156  -6.828  23.116  1.00 42.16      A    N  \\nATOM   3353  CA  ARG A 410       0.744  -7.809  22.525  1.00 45.39      A    C  \\nATOM   3354  C   ARG A 410       0.824  -9.082  23.377  1.00 44.91      A    C  \\nATOM   3355  O   ARG A 410       0.999 -10.191  22.852  1.00 44.29      A    O  \\nATOM   3356  CB  ARG A 410       2.134  -7.209  22.349  1.00 42.64      A    C  \\nATOM   3357  CG  ARG A 410       2.187  -6.099  21.324  1.00 41.14      A    C  \\nATOM   3358  CD  ARG A 410       3.577  -5.564  21.199  1.00 39.46      A    C  \\nATOM   3359  NE  ARG A 410       3.682  -4.491  20.222  1.00 36.15      A    N  \\nATOM   3360  CZ  ARG A 410       4.798  -3.847  19.912  1.00 33.76      A    C  \\nATOM   3361  NH1 ARG A 410       5.942  -4.166  20.537  1.00 31.23      A    N1+\\nATOM   3362  NH2 ARG A 410       4.790  -2.905  19.007  1.00 28.55      A    N  \\nATOM   3363  N   VAL A 411       0.703  -8.904  24.675  1.00 42.10      A    N  \\nATOM   3364  CA  VAL A 411       0.698 -10.074  25.567  1.00 45.52      A    C  \\nATOM   3365  C   VAL A 411      -0.602 -10.851  25.408  1.00 45.82      A    C  \\nATOM   3366  O   VAL A 411      -0.597 -12.085  25.424  1.00 45.37      A    O  \\nATOM   3367  CB  VAL A 411       0.908  -9.619  27.020  1.00 43.64      A    C  \\nATOM   3368  CG1 VAL A 411       2.261  -8.932  27.150  1.00 44.95      A    C  \\nATOM   3369  CG2 VAL A 411       0.825 -10.795  27.993  1.00 43.34      A    C  \\nATOM   3370  N   LYS A 412      -1.701 -10.134  25.237  1.00 43.74      A    N  \\nATOM   3371  CA  LYS A 412      -2.979 -10.799  25.033  1.00 47.50      A    C  \\nATOM   3372  C   LYS A 412      -3.058 -11.512  23.688  1.00 47.11      A    C  \\nATOM   3373  O   LYS A 412      -3.718 -12.541  23.565  1.00 45.85      A    O  \\nATOM   3374  CB  LYS A 412      -4.121  -9.775  25.132  1.00 44.75      A    C  \\nATOM   3375  CG  LYS A 412      -4.280  -9.209  26.531  1.00 42.95      A    C  \\nATOM   3376  CD  LYS A 412      -5.409  -8.200  26.567  1.00 40.51      A    C  \\nATOM   3377  CE  LYS A 412      -5.577  -7.590  27.955  1.00 34.83      A    C  \\nATOM   3378  NZ  LYS A 412      -6.667  -6.602  27.997  1.00 34.28      A    N1+\\nATOM   3379  N   SER A 413      -2.338 -10.996  22.707  1.00 41.19      A    N  \\nATOM   3380  CA  SER A 413      -2.290 -11.655  21.427  1.00 44.20      A    C  \\nATOM   3381  C   SER A 413      -1.429 -12.901  21.458  1.00 43.97      A    C  \\nATOM   3382  O   SER A 413      -1.679 -13.873  20.746  1.00 43.50      A    O  \\nATOM   3383  CB  SER A 413      -1.809 -10.714  20.328  1.00 41.98      A    C  \\nATOM   3384  OG  SER A 413      -0.465 -10.332  20.580  1.00 40.29      A    O  \\nATOM   3385  N   LYS A 414      -0.390 -12.885  22.306  1.00 39.74      A    N  \\nATOM   3386  CA  LYS A 414       0.470 -14.063  22.442  1.00 42.84      A    C  \\nATOM   3387  C   LYS A 414      -0.222 -15.135  23.268  1.00 42.69      A    C  \\nATOM   3388  O   LYS A 414      -0.018 -16.341  23.041  1.00 41.71      A    O  \\nATOM   3389  CB  LYS A 414       1.785 -13.688  23.102  1.00 40.17      A    C  \\nATOM   3390  CG  LYS A 414       2.674 -12.848  22.229  1.00 38.89      A    C  \\nATOM   3391  CD  LYS A 414       3.953 -12.464  22.938  1.00 37.16      A    C  \\nATOM   3392  CE  LYS A 414       4.842 -11.605  22.089  1.00 31.05      A    C  \\nATOM   3393  NZ  LYS A 414       6.103 -11.263  22.778  1.00 31.04      A    N1+\\nATOM   3394  N   GLN A 415      -1.040 -14.731  24.235  1.00 45.59      A    N  \\nATOM   3395  CA  GLN A 415      -1.776 -15.720  25.025  1.00 47.76      A    C  \\nATOM   3396  C   GLN A 415      -2.885 -16.385  24.230  1.00 48.11      A    C  \\nATOM   3397  O   GLN A 415      -3.215 -17.546  24.482  1.00 47.53      A    O  \\nATOM   3398  CB  GLN A 415      -2.338 -15.052  26.276  1.00 45.81      A    C  \\nATOM   3399  CG  GLN A 415      -1.290 -14.605  27.264  1.00 44.75      A    C  \\nATOM   3400  CD  GLN A 415      -1.883 -13.935  28.480  1.00 42.66      A    C  \\nATOM   3401  NE2 GLN A 415      -1.045 -13.611  29.447  1.00 35.69      A    N  \\nATOM   3402  OE1 GLN A 415      -3.094 -13.719  28.556  1.00 40.02      A    O  \\nATOM   3403  N   ALA A 416      -3.475 -15.645  23.323  1.00 38.72      A    N  \\nATOM   3404  CA  ALA A 416      -4.515 -16.217  22.495  1.00 41.40      A    C  \\nATOM   3405  C   ALA A 416      -3.928 -17.121  21.414  1.00 42.17      A    C  \\nATOM   3406  O   ALA A 416      -4.556 -18.104  21.001  1.00 41.19      A    O  \\nATOM   3407  CB  ALA A 416      -5.362 -15.119  21.862  1.00 38.74      A    C  \\nATOM   3408  N   ASN A 417      -2.729 -16.809  20.964  1.00 39.75      A    N  \\nATOM   3409  CA  ASN A 417      -2.079 -17.586  19.924  1.00 42.84      A    C  \\nATOM   3410  C   ASN A 417      -1.203 -18.705  20.457  1.00 42.73      A    C  \\nATOM   3411  O   ASN A 417      -0.586 -19.433  19.681  1.00 41.05      A    O  \\nATOM   3412  CB  ASN A 417      -1.259 -16.672  19.025  1.00 39.88      A    C  \\nATOM   3413  CG  ASN A 417      -2.108 -15.731  18.193  1.00 38.77      A    C  \\nATOM   3414  ND2 ASN A 417      -1.727 -14.446  18.136  1.00 35.44      A    N  \\nATOM   3415  OD1 ASN A 417      -3.101 -16.143  17.604  1.00 33.94      A    O  \\nATOM   3416  N   SER A 418      -1.112 -18.814  21.762  1.00 43.90      A    N  \\nATOM   3417  CA  SER A 418      -0.269 -19.865  22.359  1.00 46.51      A    C  \\nATOM   3418  C   SER A 418      -1.033 -21.195  22.323  1.00 46.43      A    C  \\nATOM   3419  O   SER A 418      -1.819 -21.492  23.234  1.00 45.34      A    O  \\nATOM   3420  CB  SER A 418       0.129 -19.489  23.766  1.00 43.65      A    C  \\nATOM   3421  OG  SER A 418      -1.025 -19.363  24.579  1.00 40.18      A    O  \\nATOM   3422  N   GLN A 419      -0.785 -21.961  21.319  1.00 48.10      A    N  \\nATOM   3423  CA  GLN A 419      -1.452 -23.252  21.127  1.00 50.29      A    C  \\nATOM   3424  C   GLN A 419      -0.472 -24.406  20.944  1.00 49.99      A    C  \\nATOM   3425  O   GLN A 419      -0.832 -25.552  21.118  1.00 50.07      A    O  \\nATOM   3426  CB  GLN A 419      -2.418 -23.219  19.942  1.00 48.50      A    C  \\nATOM   3427  CG  GLN A 419      -3.594 -22.299  20.165  1.00 46.51      A    C  \\nATOM   3428  CD  GLN A 419      -4.527 -22.255  18.978  1.00 43.00      A    C  \\nATOM   3429  NE2 GLN A 419      -5.615 -21.511  19.090  1.00 37.47      A    N  \\nATOM   3430  OE1 GLN A 419      -4.272 -22.888  17.952  1.00 41.24      A    O  \\nATOM   3431  N   TYR A 420       0.760 -24.102  20.572  1.00 41.73      A    N  \\nATOM   3432  CA  TYR A 420       1.746 -25.127  20.341  1.00 45.40      A    C  \\nATOM   3433  C   TYR A 420       2.480 -25.452  21.654  1.00 45.41      A    C  \\nATOM   3434  O   TYR A 420       3.574 -24.957  21.905  1.00 43.77      A    O  \\nATOM   3435  CB  TYR A 420       2.738 -24.699  19.257  1.00 41.63      A    C  \\nATOM   3436  CG  TYR A 420       2.115 -24.469  17.879  1.00 40.53      A    C  \\nATOM   3437  CD1 TYR A 420       1.775 -23.190  17.475  1.00 38.98      A    C  \\nATOM   3438  CD2 TYR A 420       1.883 -25.530  17.036  1.00 38.43      A    C  \\nATOM   3439  CE1 TYR A 420       1.199 -22.974  16.233  1.00 33.42      A    C  \\nATOM   3440  CE2 TYR A 420       1.321 -25.339  15.778  1.00 33.23      A    C  \\nATOM   3441  CZ  TYR A 420       0.957 -24.061  15.407  1.00 31.96      A    C  \\nATOM   3442  OH  TYR A 420       0.384 -23.853  14.168  1.00 35.03      A    O  \\nATOM   3443  N   HIS A 421       1.855 -26.263  22.474  1.00 45.72      A    N  \\nATOM   3444  CA  HIS A 421       2.439 -26.632  23.759  1.00 47.60      A    C  \\nATOM   3445  C   HIS A 421       2.956 -28.071  23.783  1.00 47.41      A    C  \\nATOM   3446  O   HIS A 421       3.722 -28.422  24.668  1.00 46.34      A    O  \\nATOM   3447  CB  HIS A 421       1.426 -26.425  24.852  1.00 45.19      A    C  \\nATOM   3448  CG  HIS A 421       0.987 -25.022  25.043  1.00 44.83      A    C  \\nATOM   3449  CD2 HIS A 421      -0.151 -24.358  24.693  1.00 42.29      A    C  \\nATOM   3450  ND1 HIS A 421       1.799 -24.084  25.652  1.00 40.78      A    N  \\nATOM   3451  CE1 HIS A 421       1.160 -22.921  25.689  1.00 39.39      A    C  \\nATOM   3452  NE2 HIS A 421      -0.017 -23.066  25.120  1.00 36.19      A    N  \\nATOM   3453  N   GLU A 422       2.506 -28.858  22.828  1.00 45.70      A    N  \\nATOM   3454  CA  GLU A 422       2.926 -30.254  22.775  1.00 48.09      A    C  \\nATOM   3455  C   GLU A 422       4.352 -30.428  22.264  1.00 48.54      A    C  \\nATOM   3456  O   GLU A 422       5.032 -31.378  22.649  1.00 47.34      A    O  \\nATOM   3457  CB  GLU A 422       1.948 -31.058  21.930  1.00 45.27      A    C  \\nATOM   3458  CG  GLU A 422       0.543 -31.127  22.503  1.00 44.47      A    C  \\nATOM   3459  CD  GLU A 422      -0.425 -31.900  21.638  1.00 41.36      A    C  \\nATOM   3460  OE1 GLU A 422      -0.032 -32.353  20.530  1.00 37.10      A    O  \\nATOM   3461  OE2 GLU A 422      -1.608 -32.085  22.060  1.00 33.79      A    O1-\\nATOM   3462  N   ASP A 423       4.773 -29.503  21.430  1.00 47.11      A    N  \\nATOM   3463  CA  ASP A 423       6.104 -29.598  20.845  1.00 50.27      A    C  \\nATOM   3464  C   ASP A 423       7.149 -28.814  21.609  1.00 50.79      A    C  \\nATOM   3465  O   ASP A 423       8.258 -28.595  21.090  1.00 48.89      A    O  \\nATOM   3466  CB  ASP A 423       6.083 -29.110  19.377  1.00 46.58      A    C  \\nATOM   3467  CG  ASP A 423       5.276 -30.048  18.478  1.00 44.85      A    C  \\nATOM   3468  OD1 ASP A 423       5.286 -31.261  18.749  1.00 39.98      A    O  \\nATOM   3469  OD2 ASP A 423       4.661 -29.545  17.505  1.00 40.28      A    O1-\\nATOM   3470  N   GLU A 424       6.816 -28.382  22.808  1.00 48.80      A    N  \\nATOM   3471  CA  GLU A 424       7.785 -27.629  23.592  1.00 52.28      A    C  \\nATOM   3472  C   GLU A 424       8.912 -28.505  24.115  1.00 52.49      A    C  \\nATOM   3473  O   GLU A 424      10.069 -28.083  24.134  1.00 51.45      A    O  \\nATOM   3474  CB  GLU A 424       7.085 -26.926  24.742  1.00 49.53      A    C  \\nATOM   3475  CG  GLU A 424       6.111 -25.818  24.326  1.00 47.89      A    C  \\nATOM   3476  CD  GLU A 424       5.472 -25.141  25.514  1.00 43.70      A    C  \\nATOM   3477  OE1 GLU A 424       5.726 -25.568  26.652  1.00 41.72      A    O  \\nATOM   3478  OE2 GLU A 424       4.713 -24.161  25.304  1.00 39.07      A    O1-\\nATOM   3479  N   ASP A 425       8.561 -29.723  24.508  1.00 51.66      A    N  \\nATOM   3480  CA  ASP A 425       9.593 -30.635  25.009  1.00 53.87      A    C  \\nATOM   3481  C   ASP A 425      10.508 -31.107  23.887  1.00 54.54      A    C  \\nATOM   3482  O   ASP A 425      11.688 -31.311  24.111  1.00 53.46      A    O  \\nATOM   3483  CB  ASP A 425       8.919 -31.833  25.689  1.00 50.73      A    C  \\nATOM   3484  CG  ASP A 425       8.229 -31.440  26.983  1.00 48.48      A    C  \\nATOM   3485  OD1 ASP A 425       8.677 -30.486  27.640  1.00 43.66      A    O  \\nATOM   3486  OD2 ASP A 425       7.219 -32.106  27.324  1.00 44.03      A    O1-\\nATOM   3487  N   GLU A 426       9.938 -31.271  22.721  1.00 51.46      A    N  \\nATOM   3488  CA  GLU A 426      10.779 -31.667  21.576  1.00 54.49      A    C  \\nATOM   3489  C   GLU A 426      11.645 -30.527  21.101  1.00 55.21      A    C  \\nATOM   3490  O   GLU A 426      12.731 -30.747  20.608  1.00 54.57      A    O  \\nATOM   3491  CB  GLU A 426       9.883 -32.177  20.445  1.00 51.73      A    C  \\nATOM   3492  CG  GLU A 426       9.137 -33.457  20.774  1.00 48.91      A    C  \\nATOM   3493  CD  GLU A 426       8.288 -33.954  19.634  1.00 44.90      A    C  \\nATOM   3494  OE1 GLU A 426       8.211 -33.245  18.585  1.00 42.66      A    O  \\nATOM   3495  OE2 GLU A 426       7.691 -35.042  19.737  1.00 39.77      A    O1-\\nATOM   3496  N   ASP A 427      11.145 -29.310  21.265  1.00 55.59      A    N  \\nATOM   3497  CA  ASP A 427      11.971 -28.159  20.909  1.00 56.91      A    C  \\nATOM   3498  C   ASP A 427      13.079 -27.933  21.919  1.00 58.19      A    C  \\nATOM   3499  O   ASP A 427      14.145 -27.440  21.557  1.00 57.35      A    O  \\nATOM   3500  CB  ASP A 427      11.093 -26.911  20.791  1.00 53.48      A    C  \\nATOM   3501  CG  ASP A 427      10.186 -26.948  19.576  1.00 50.47      A    C  \\nATOM   3502  OD1 ASP A 427      10.551 -27.584  18.567  1.00 47.05      A    O  \\nATOM   3503  OD2 ASP A 427       9.095 -26.348  19.646  1.00 48.27      A    O1-\\nATOM   3504  N   GLU A 428      12.832 -28.304  23.169  1.00 57.52      A    N  \\nATOM   3505  CA  GLU A 428      13.877 -28.199  24.178  1.00 60.99      A    C  \\nATOM   3506  C   GLU A 428      14.955 -29.236  23.945  1.00 62.06      A    C  \\nATOM   3507  O   GLU A 428      16.133 -28.997  24.234  1.00 62.95      A    O  \\nATOM   3508  CB  GLU A 428      13.285 -28.328  25.565  1.00 59.49      A    C  \\nATOM   3509  CG  GLU A 428      12.455 -27.149  26.013  1.00 55.73      A    C  \\nATOM   3510  CD  GLU A 428      11.885 -27.322  27.401  1.00 49.79      A    C  \\nATOM   3511  OE1 GLU A 428      12.103 -28.396  28.008  1.00 47.17      A    O  \\nATOM   3512  OE2 GLU A 428      11.215 -26.371  27.897  1.00 43.94      A    O1-\\nATOM   3513  N   LYS A 429      14.561 -30.399  23.460  1.00 56.56      A    N  \\nATOM   3514  CA  LYS A 429      15.542 -31.440  23.169  1.00 58.23      A    C  \\nATOM   3515  C   LYS A 429      16.380 -31.072  21.945  1.00 58.59      A    C  \\nATOM   3516  O   LYS A 429      17.578 -31.393  21.904  1.00 57.82      A    O  \\nATOM   3517  CB  LYS A 429      14.854 -32.783  22.932  1.00 55.46      A    C  \\nATOM   3518  CG  LYS A 429      14.212 -33.354  24.193  1.00 52.15      A    C  \\nATOM   3519  CD  LYS A 429      13.557 -34.667  23.911  1.00 48.64      A    C  \\nATOM   3520  CE  LYS A 429      12.921 -35.247  25.148  1.00 43.54      A    C  \\nATOM   3521  NZ  LYS A 429      12.225 -36.527  24.865  1.00 41.60      A    N1+\\nATOM   3522  N   THR A 430      15.793 -30.398  21.001  1.00 58.21      A    N  \\nATOM   3523  CA  THR A 430      16.562 -29.951  19.840  1.00 59.03      A    C  \\nATOM   3524  C   THR A 430      17.463 -28.777  20.193  1.00 59.21      A    C  \\nATOM   3525  O   THR A 430      18.550 -28.671  19.646  1.00 58.76      A    O  \\nATOM   3526  CB  THR A 430      15.631 -29.592  18.689  1.00 56.57      A    C  \\nATOM   3527  CG2 THR A 430      14.897 -30.831  18.214  1.00 53.12      A    C  \\nATOM   3528  OG1 THR A 430      14.680 -28.621  19.124  1.00 56.01      A    O  \\nATOM   3529  N   TYR A 431      17.010 -27.928  21.099  1.00 57.70      A    N  \\nATOM   3530  CA  TYR A 431      17.847 -26.807  21.524  1.00 60.22      A    C  \\nATOM   3531  C   TYR A 431      19.005 -27.277  22.383  1.00 61.64      A    C  \\nATOM   3532  O   TYR A 431      20.087 -26.704  22.331  1.00 62.22      A    O  \\nATOM   3533  CB  TYR A 431      17.030 -25.759  22.261  1.00 58.51      A    C  \\nATOM   3534  CG  TYR A 431      16.046 -25.001  21.393  1.00 55.38      A    C  \\nATOM   3535  CD1 TYR A 431      16.187 -24.966  20.019  1.00 50.94      A    C  \\nATOM   3536  CD2 TYR A 431      14.990 -24.319  21.959  1.00 49.86      A    C  \\nATOM   3537  CE1 TYR A 431      15.315 -24.260  19.231  1.00 45.38      A    C  \\nATOM   3538  CE2 TYR A 431      14.097 -23.591  21.186  1.00 45.23      A    C  \\nATOM   3539  CZ  TYR A 431      14.263 -23.583  19.814  1.00 43.65      A    C  \\nATOM   3540  OH  TYR A 431      13.371 -22.878  19.034  1.00 43.16      A    O  \\nATOM   3541  N   LYS A 432      18.776 -28.332  23.189  1.00 56.82      A    N  \\nATOM   3542  CA  LYS A 432      19.885 -28.907  23.926  1.00 60.77      A    C  \\nATOM   3543  C   LYS A 432      20.881 -29.616  23.026  1.00 60.25      A    C  \\nATOM   3544  O   LYS A 432      22.069 -29.661  23.325  1.00 60.89      A    O  \\nATOM   3545  CB  LYS A 432      19.363 -29.874  24.982  1.00 60.00      A    C  \\nATOM   3546  CG  LYS A 432      18.656 -29.214  26.159  1.00 57.24      A    C  \\nATOM   3547  CD  LYS A 432      18.168 -30.236  27.142  1.00 53.53      A    C  \\nATOM   3548  CE  LYS A 432      17.482 -29.571  28.330  1.00 50.79      A    C  \\nATOM   3549  NZ  LYS A 432      16.966 -30.557  29.300  1.00 47.18      A    N1+\\nATOM   3550  N   PHE A 433      20.385 -30.152  21.954  1.00 57.83      A    N  \\nATOM   3551  CA  PHE A 433      21.287 -30.767  20.966  1.00 60.76      A    C  \\nATOM   3552  C   PHE A 433      22.122 -29.712  20.259  1.00 60.96      A    C  \\nATOM   3553  O   PHE A 433      23.293 -29.962  19.946  1.00 60.91      A    O  \\nATOM   3554  CB  PHE A 433      20.463 -31.569  19.964  1.00 59.16      A    C  \\nATOM   3555  CG  PHE A 433      21.321 -32.231  18.900  1.00 56.46      A    C  \\nATOM   3556  CD1 PHE A 433      22.050 -33.377  19.184  1.00 53.71      A    C  \\nATOM   3557  CD2 PHE A 433      21.360 -31.710  17.630  1.00 52.75      A    C  \\nATOM   3558  CE1 PHE A 433      22.810 -33.978  18.212  1.00 50.84      A    C  \\nATOM   3559  CE2 PHE A 433      22.130 -32.307  16.653  1.00 49.15      A    C  \\nATOM   3560  CZ  PHE A 433      22.846 -33.445  16.954  1.00 48.60      A    C  \\nATOM   3561  N   ILE A 434      21.553 -28.549  20.051  1.00 56.68      A    N  \\nATOM   3562  CA  ILE A 434      22.313 -27.447  19.452  1.00 58.08      A    C  \\nATOM   3563  C   ILE A 434      23.354 -26.939  20.444  1.00 58.76      A    C  \\nATOM   3564  O   ILE A 434      24.491 -26.643  20.053  1.00 57.08      A    O  \\nATOM   3565  CB  ILE A 434      21.354 -26.325  18.990  1.00 55.17      A    C  \\nATOM   3566  CG1 ILE A 434      20.489 -26.837  17.833  1.00 58.10      A    C  \\nATOM   3567  CG2 ILE A 434      22.119 -25.096  18.559  1.00 53.56      A    C  \\nATOM   3568  CD1 ILE A 434      19.360 -25.908  17.476  1.00 50.37      A    C  \\nATOM   3569  N   GLU A 435      22.984 -26.874  21.721  1.00 59.00      A    N  \\nATOM   3570  CA  GLU A 435      23.912 -26.374  22.716  1.00 61.26      A    C  \\nATOM   3571  C   GLU A 435      25.067 -27.339  22.968  1.00 61.29      A    C  \\nATOM   3572  O   GLU A 435      26.202 -26.922  23.170  1.00 62.29      A    O  \\nATOM   3573  CB  GLU A 435      23.174 -26.110  24.035  1.00 60.62      A    C  \\nATOM   3574  CG  GLU A 435      22.221 -24.944  24.000  1.00 57.28      A    C  \\nATOM   3575  CD  GLU A 435      21.502 -24.733  25.315  1.00 52.75      A    C  \\nATOM   3576  OE1 GLU A 435      21.653 -25.610  26.208  1.00 51.32      A    O  \\nATOM   3577  OE2 GLU A 435      20.793 -23.716  25.469  1.00 47.10      A    O1-\\nATOM   3578  N   HIS A 436      24.768 -28.644  22.916  1.00 58.63      A    N  \\nATOM   3579  CA  HIS A 436      25.853 -29.614  23.026  1.00 60.81      A    C  \\nATOM   3580  C   HIS A 436      26.705 -29.666  21.775  1.00 59.61      A    C  \\nATOM   3581  O   HIS A 436      27.884 -29.963  21.838  1.00 61.07      A    O  \\nATOM   3582  CB  HIS A 436      25.317 -31.008  23.352  1.00 61.09      A    C  \\nATOM   3583  CG  HIS A 436      24.753 -31.118  24.736  1.00 57.61      A    C  \\nATOM   3584  CD2 HIS A 436      23.474 -31.292  25.162  1.00 52.26      A    C  \\nATOM   3585  ND1 HIS A 436      25.521 -31.080  25.878  1.00 50.05      A    N  \\nATOM   3586  CE1 HIS A 436      24.765 -31.240  26.943  1.00 50.13      A    C  \\nATOM   3587  NE2 HIS A 436      23.502 -31.363  26.530  1.00 48.46      A    N  \\nATOM   3588  N   ALA A 437      26.083 -29.415  20.662  1.00 56.61      A    N  \\nATOM   3589  CA  ALA A 437      26.831 -29.408  19.405  1.00 58.47      A    C  \\nATOM   3590  C   ALA A 437      27.760 -28.197  19.298  1.00 59.39      A    C  \\nATOM   3591  O   ALA A 437      28.834 -28.281  18.716  1.00 58.37      A    O  \\nATOM   3592  CB  ALA A 437      25.883 -29.425  18.222  1.00 55.84      A    C  \\nATOM   3593  N   GLN A 438      27.323 -27.081  19.878  1.00 60.68      A    N  \\nATOM   3594  CA  GLN A 438      28.170 -25.891  19.900  1.00 62.17      A    C  \\nATOM   3595  C   GLN A 438      29.355 -26.054  20.849  1.00 62.42      A    C  \\nATOM   3596  O   GLN A 438      30.426 -25.534  20.606  1.00 62.78      A    O  \\nATOM   3597  CB  GLN A 438      27.370 -24.652  20.283  1.00 60.36      A    C  \\nATOM   3598  CG  GLN A 438      26.371 -24.202  19.239  1.00 56.77      A    C  \\nATOM   3599  CD  GLN A 438      25.558 -22.991  19.697  1.00 53.25      A    C  \\nATOM   3600  NE2 GLN A 438      24.732 -22.466  18.797  1.00 48.16      A    N  \\nATOM   3601  OE1 GLN A 438      25.682 -22.566  20.833  1.00 52.00      A    O  \\nATOM   3602  N   GLN A 439      29.125 -26.811  21.937  1.00 64.62      A    N  \\nATOM   3603  CA  GLN A 439      30.208 -27.057  22.858  1.00 66.24      A    C  \\nATOM   3604  C   GLN A 439      31.213 -28.061  22.318  1.00 65.15      A    C  \\nATOM   3605  O   GLN A 439      32.396 -27.991  22.621  1.00 66.00      A    O  \\nATOM   3606  CB  GLN A 439      29.665 -27.552  24.200  1.00 65.33      A    C  \\nATOM   3607  CG  GLN A 439      28.892 -26.519  24.991  1.00 61.12      A    C  \\nATOM   3608  CD  GLN A 439      28.326 -27.052  26.256  1.00 56.51      A    C  \\nATOM   3609  NE2 GLN A 439      27.644 -26.185  27.006  1.00 50.81      A    N  \\nATOM   3610  OE1 GLN A 439      28.485 -28.218  26.598  1.00 55.28      A    O  \\nATOM   3611  N   ASP A 440      30.740 -29.008  21.508  1.00 59.95      A    N  \\nATOM   3612  CA  ASP A 440      31.641 -29.967  20.913  1.00 61.43      A    C  \\nATOM   3613  C   ASP A 440      32.500 -29.323  19.817  1.00 60.15      A    C  \\nATOM   3614  O   ASP A 440      33.617 -29.772  19.588  1.00 59.21      A    O  \\nATOM   3615  CB  ASP A 440      30.857 -31.159  20.344  1.00 59.08      A    C  \\nATOM   3616  CG  ASP A 440      30.281 -32.036  21.427  1.00 56.16      A    C  \\nATOM   3617  OD1 ASP A 440      30.885 -32.112  22.513  1.00 50.27      A    O  \\nATOM   3618  OD2 ASP A 440      29.220 -32.648  21.188  1.00 52.91      A    O1-\\nATOM   3619  N   VAL A 441      31.957 -28.322  19.179  1.00 58.27      A    N  \\nATOM   3620  CA  VAL A 441      32.743 -27.586  18.186  1.00 61.03      A    C  \\nATOM   3621  C   VAL A 441      33.754 -26.668  18.871  1.00 61.22      A    C  \\nATOM   3622  O   VAL A 441      34.914 -26.594  18.432  1.00 60.39      A    O  \\nATOM   3623  CB  VAL A 441      31.818 -26.805  17.240  1.00 58.20      A    C  \\nATOM   3624  CG1 VAL A 441      30.960 -27.776  16.417  1.00 59.45      A    C  \\nATOM   3625  CG2 VAL A 441      32.608 -25.944  16.284  1.00 57.34      A    C  \\nATOM   3626  N   GLU A 442      33.341 -26.044  19.938  1.00 61.03      A    N  \\nATOM   3627  CA  GLU A 442      34.226 -25.155  20.640  1.00 63.39      A    C  \\nATOM   3628  C   GLU A 442      35.334 -25.898  21.392  1.00 63.33      A    C  \\nATOM   3629  O   GLU A 442      36.417 -25.364  21.596  1.00 62.21      A    O  \\nATOM   3630  CB  GLU A 442      33.445 -24.276  21.614  1.00 60.78      A    C  \\nATOM   3631  CG  GLU A 442      32.543 -23.260  20.951  1.00 56.79      A    C  \\nATOM   3632  CD  GLU A 442      31.760 -22.431  21.933  1.00 52.77      A    C  \\nATOM   3633  OE1 GLU A 442      31.863 -22.731  23.155  1.00 49.96      A    O  \\nATOM   3634  OE2 GLU A 442      31.057 -21.462  21.518  1.00 45.17      A    O1-\\nATOM   3635  N   ASP A 443      35.078 -27.159  21.769  1.00 63.30      A    N  \\nATOM   3636  CA  ASP A 443      36.087 -27.967  22.413  1.00 64.89      A    C  \\nATOM   3637  C   ASP A 443      37.105 -28.481  21.406  1.00 62.96      A    C  \\nATOM   3638  O   ASP A 443      38.260 -28.753  21.775  1.00 61.76      A    O  \\nATOM   3639  CB  ASP A 443      35.468 -29.131  23.162  1.00 61.89      A    C  \\nATOM   3640  CG  ASP A 443      34.688 -28.686  24.396  1.00 59.40      A    C  \\nATOM   3641  OD1 ASP A 443      35.060 -27.640  24.978  1.00 53.24      A    O  \\nATOM   3642  OD2 ASP A 443      33.737 -29.375  24.786  1.00 55.55      A    O1-\\nATOM   3643  N   GLU A 444      36.692 -28.635  20.173  1.00 45.56      A    N  \\nATOM   3644  CA  GLU A 444      37.632 -29.072  19.132  1.00 50.99      A    C  \\nATOM   3645  C   GLU A 444      38.441 -27.911  18.596  1.00 49.51      A    C  \\nATOM   3646  O   GLU A 444      39.521 -28.117  18.070  1.00 45.94      A    O  \\nATOM   3647  CB  GLU A 444      36.883 -29.749  17.989  1.00 43.61      A    C  \\nATOM   3648  CG  GLU A 444      36.250 -31.077  18.362  1.00 42.24      A    C  \\nATOM   3649  CD  GLU A 444      35.467 -31.687  17.221  1.00 42.15      A    C  \\nATOM   3650  OE1 GLU A 444      35.308 -31.036  16.169  1.00 46.33      A    O  \\nATOM   3651  OE2 GLU A 444      34.991 -32.848  17.370  1.00 40.09      A    O1-\\nTER   \\nEND\\n\",\"pdb\");\n",
              "\tviewer_1742372256039599.setStyle({\"chain\": \"A\"},{\"cartoon\": {\"color\": \"#33ff33\"}});\n",
              "\tviewer_1742372256039599.zoomTo();\n",
              "viewer_1742372256039599.render();\n",
              "});\n",
              "</script>"
            ]
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "Dropdown(description='Sampling methodology:', layout=Layout(height='30px', width='500px'), options=('argmax', ‚Ä¶"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "95dcb311094f44fdba62a5253d83b0bf"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "BoundedIntText(value=1, description='Sampling num:', layout=Layout(display='none', height='30px', width='500px‚Ä¶"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "0a0feface3a4407686eef7e3a90be88e"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "HTML(value='<p><em>For each position, Saprot will predict the probability of every amino acid occuring at this‚Ä¶"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "0a753a8a60614f65bb440c80727d76f9"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "HTML(value='<h3>---------------------------------------------------------------------------</h3>')"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "cd907850ca024b20971715368f018664"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "Button(button_style='info', description='Predict protein sequence', disabled=True, layout=Layout(height='30px'‚Ä¶"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "30fe7b0e433a4bf19dc8bda8c5660cde"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "HTML(value='<p><strong>Predicted sequences are listed below:</strong></p>\\n<ol>\\n<li>LEELLELLEELEEELEELEEELEEL‚Ä¶"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "ab859c29b67641249ed3a391d6a3f607"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "Button(button_style='success', description='Download File', style=ButtonStyle())"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "4902a6111f87440db2145b576fc75a8e"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "HTML(value='<h3>You could use ESMFold to evaluate the quality of the predicted sequence</h3>\\n<h4><font color=‚Ä¶"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "c99ee45b34174aabab065748325949fc"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "Text(value='', layout=Layout(height='30px', width='500px'), placeholder='Copy the predicted sequence and paste‚Ä¶"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "94ecdefcd9594129b025c2a87e05d83b"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "Button(button_style='info', description='Predict structure', layout=Layout(height='30px', width='500px'), styl‚Ä¶"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "d626ed4d4b604fc4a7019e321ce91a22"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "HTML(value='<p><strong><font color=red>Note: At any time you can click below buttons to stop and restart</font‚Ä¶"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "99cac01948ec49e28342c51c0f8a7cb7"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "HBox(children=(Button(button_style='success', description='Go back', style=ButtonStyle()), Button(button_style‚Ä¶"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "3eccfa1db44a4344b43dc726d20a512b"
            }
          },
          "metadata": {}
        },
        {
          "output_type": "display_data",
          "data": {
            "text/plain": [
              "HTML(value='<p><strong>Go back:</strong> Stop the running task and go back to the first interface</p>\\n<p><str‚Ä¶"
            ],
            "application/vnd.jupyter.widget-view+json": {
              "version_major": 2,
              "version_minor": 0,
              "model_id": "fa4c378424fe4ffc903b08584dd6ae47"
            }
          },
          "metadata": {}
        }
      ]
    }
  ],
  "metadata": {
    "kernelspec": {
      "display_name": "Python 3",
      "name": "python3"
    },
    "language_info": {
      "codemirror_mode": {
        "name": "ipython",
        "version": 2
      },
      "file_extension": ".py",
      "mimetype": "text/x-python",
      "name": "python",
      "nbconvert_exporter": "python",
      "pygments_lexer": "ipython2",
      "version": "2.7.6"
    },
    "colab": {
      "provenance": [],
      "gpuType": "T4"
    },
    "accelerator": "GPU",
    "widgets": {
      "application/vnd.jupyter.widget-state+json": {
        "cb892f8a799b49a79e68f86a2214cc32": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_31116bf082be4db4a0445dacfd75adf8",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_b2c0ad6e0eec45b0b7d715432b5ede67",
            "value": "<h2>Please choose what you want to do with ColabSaprot</h2>"
          }
        },
        "31116bf082be4db4a0445dacfd75adf8": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "b2c0ad6e0eec45b0b7d715432b5ede67": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "6f56172384284742809974592f2c56f9": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_e97ed5724e2a49ddbf3fd14ef1470b55",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_905790c74150464f8727a378ab71a248",
            "value": "<p><a href='https://github.com/westlake-repl/SaprotHub/wiki/SaprotHub-v2-(latest)#introduction-of-options' target='blank'>Introduction of these options</a></p>"
          }
        },
        "e97ed5724e2a49ddbf3fd14ef1470b55": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "905790c74150464f8727a378ab71a248": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "afd0e84440484bdf97b1a8252bda0249": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ButtonView",
            "button_style": "",
            "description": "I want to train my own model",
            "disabled": false,
            "icon": "",
            "layout": "IPY_MODEL_7c90febfd5394f648248398805e99620",
            "style": "IPY_MODEL_c329a7db5be0435595c2bbdf7b072de8",
            "tooltip": ""
          }
        },
        "7c90febfd5394f648248398805e99620": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": "30px",
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "400px"
          }
        },
        "c329a7db5be0435595c2bbdf7b072de8": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "button_color": null,
            "font_weight": ""
          }
        },
        "4f0f730b7266466b81143e5cc98f1f38": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ButtonView",
            "button_style": "",
            "description": "I want to use existing models to make prediction",
            "disabled": false,
            "icon": "",
            "layout": "IPY_MODEL_5944035516eb48efa3bab5e5d4c047e0",
            "style": "IPY_MODEL_84c915f4c24b47019c3a57fd64584b0c",
            "tooltip": ""
          }
        },
        "5944035516eb48efa3bab5e5d4c047e0": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": "30px",
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "400px"
          }
        },
        "84c915f4c24b47019c3a57fd64584b0c": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "button_color": null,
            "font_weight": ""
          }
        },
        "1e3e43d7104a46bc82b43ca8d0cbc786": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ButtonView",
            "button_style": "",
            "description": "I want to share my model publicly",
            "disabled": false,
            "icon": "",
            "layout": "IPY_MODEL_153fcbc5e217499eb8d0d404d2ab9915",
            "style": "IPY_MODEL_2ab7a019c0af47e78954085d5df3bac7",
            "tooltip": ""
          }
        },
        "153fcbc5e217499eb8d0d404d2ab9915": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": "30px",
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "400px"
          }
        },
        "2ab7a019c0af47e78954085d5df3bac7": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "button_color": null,
            "font_weight": ""
          }
        },
        "99cac01948ec49e28342c51c0f8a7cb7": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_0883c3d8f0f94f56b46bcbb4d1997c48",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_c2399df31a0540ed802cd557481b3dbd",
            "value": "<p><strong><font color=red>Note: At any time you can click below buttons to stop and restart</font></strong></p>"
          }
        },
        "0883c3d8f0f94f56b46bcbb4d1997c48": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "c2399df31a0540ed802cd557481b3dbd": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "3eccfa1db44a4344b43dc726d20a512b": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_75d32a15b59449c78511319420a6335e",
              "IPY_MODEL_f4c926ba549c4e8cb73d505ed00eebdc",
              "IPY_MODEL_234301fa55ac438d99d13083ca14747e"
            ],
            "layout": "IPY_MODEL_53f47a56a11b45ed962800ebce4e0326"
          }
        },
        "75d32a15b59449c78511319420a6335e": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ButtonView",
            "button_style": "success",
            "description": "Go back",
            "disabled": false,
            "icon": "",
            "layout": "IPY_MODEL_59ac90a30a644eeaabc24b8cafd11278",
            "style": "IPY_MODEL_a584fa36c41f457c949110f63f22ee2b",
            "tooltip": ""
          }
        },
        "f4c926ba549c4e8cb73d505ed00eebdc": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ButtonView",
            "button_style": "success",
            "description": "Refresh",
            "disabled": false,
            "icon": "",
            "layout": "IPY_MODEL_e944b600328242e4a2b37ee73f370037",
            "style": "IPY_MODEL_a34d2e740cf849408137abc6a474f818",
            "tooltip": ""
          }
        },
        "234301fa55ac438d99d13083ca14747e": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ButtonView",
            "button_style": "danger",
            "description": "Stop",
            "disabled": false,
            "icon": "",
            "layout": "IPY_MODEL_bebe165d768d44689d04b95e5529d1a4",
            "style": "IPY_MODEL_d8b0829952da4a70ac337f26256af992",
            "tooltip": ""
          }
        },
        "53f47a56a11b45ed962800ebce4e0326": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "59ac90a30a644eeaabc24b8cafd11278": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "a584fa36c41f457c949110f63f22ee2b": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "button_color": null,
            "font_weight": ""
          }
        },
        "e944b600328242e4a2b37ee73f370037": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "a34d2e740cf849408137abc6a474f818": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "button_color": null,
            "font_weight": ""
          }
        },
        "bebe165d768d44689d04b95e5529d1a4": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "d8b0829952da4a70ac337f26256af992": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "button_color": null,
            "font_weight": ""
          }
        },
        "fa4c378424fe4ffc903b08584dd6ae47": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_b8e0bee67db8463c9924cd7c09f4ca6e",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_1dec01a5339c4cf69f8f80437f220a30",
            "value": "<p><strong>Go back:</strong> Stop the running task and go back to the first interface</p>\n<p><strong>Refresh</strong>: Stop the running task and refresh the current interface</p>\n<p><strong>Stop</strong>: Stop the running task</p>"
          }
        },
        "b8e0bee67db8463c9924cd7c09f4ca6e": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "1dec01a5339c4cf69f8f80437f220a30": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "fdcc3a61ae9c4844980c0f3fe6d4597a": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_6ef3b636cc274069946cbffb38cc190a",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_6d813806bc49426396e6b0fd77669627",
            "value": "<h2>ColabSaprot supports multiple prediction tasks, which one would you like to choose?</h2>"
          }
        },
        "6ef3b636cc274069946cbffb38cc190a": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "6d813806bc49426396e6b0fd77669627": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "e33c5526e0a741fa8b625653280c6fae": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_131a7eb8af1c4e359243b1c77270cd4f",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_126cc102032b483288f5368954881958",
            "value": "<h3>---------------------------------------------------------------------------</h3>"
          }
        },
        "131a7eb8af1c4e359243b1c77270cd4f": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "126cc102032b483288f5368954881958": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "444a0ec1fea2499ea163722c54831b38": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ButtonView",
            "button_style": "info",
            "description": "Protein property prediction",
            "disabled": false,
            "icon": "",
            "layout": "IPY_MODEL_3757ba3aeafe4941b9f31a16cc972b5d",
            "style": "IPY_MODEL_429d2ee9e7ab4e809306043612194632",
            "tooltip": ""
          }
        },
        "3757ba3aeafe4941b9f31a16cc972b5d": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": "30px",
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "429d2ee9e7ab4e809306043612194632": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "button_color": null,
            "font_weight": ""
          }
        },
        "776ad3846df34250a96370dfe2ed8832": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_3290bce4312345edbb9efdcc218ddda3",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_bc4c5346a9aa4df990c197a8c44f70f8",
            "value": "<p>This section enables property prediction using well-trained models for various protein characteristics, including stability, subcellular localization, and solubility, etc. Users can leverage either models available in <a href='https://huggingface.co/SaProtHub' target='blank'>SaprotHub</a> or their previously trained models for predictions. The model input for predicting should be consistent with its training format.</p>"
          }
        },
        "3290bce4312345edbb9efdcc218ddda3": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "bc4c5346a9aa4df990c197a8c44f70f8": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "49168b26cc2c4f13bf6c368841ba975e": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ButtonView",
            "button_style": "info",
            "description": "Mutational effect prediction",
            "disabled": false,
            "icon": "",
            "layout": "IPY_MODEL_71557d0108404be8aced4d707743dc59",
            "style": "IPY_MODEL_db95609d0f1143759c5938be8a88ddd9",
            "tooltip": ""
          }
        },
        "71557d0108404be8aced4d707743dc59": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": "30px",
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "db95609d0f1143759c5938be8a88ddd9": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "button_color": null,
            "font_weight": ""
          }
        },
        "91557ea908cf4b2680a193fe32d96001": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_55937092eb6b4bfeb492b1b0b66fb996",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_d7e95855731e4332a46f9e9f9711b63e",
            "value": "<p>The Mutational Effect Prediction section utilizes the Saprot 650M model. By analyzing the predicted mutation scores, users can quickly identify mutations that are likely to enhance specific protein functions, such as enzyme activity.</p>"
          }
        },
        "55937092eb6b4bfeb492b1b0b66fb996": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "d7e95855731e4332a46f9e9f9711b63e": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "6ff6e7cd96044210a83d26e6f3030f9e": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ButtonView",
            "button_style": "info",
            "description": "Protein sequence design",
            "disabled": false,
            "icon": "",
            "layout": "IPY_MODEL_d15a3542c1564708a1cb5d96e5a084a5",
            "style": "IPY_MODEL_c15dc1d27091430c83e8c99524ac7a38",
            "tooltip": ""
          }
        },
        "d15a3542c1564708a1cb5d96e5a084a5": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": "30px",
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "c15dc1d27091430c83e8c99524ac7a38": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "button_color": null,
            "font_weight": ""
          }
        },
        "feddedbe2ee54a9cac2fbea88e27172e": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_7f45b74269744c05ba7a99af9423c01b",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_3fd98403ec1a402a8fec39c898bc3134",
            "value": "<p>The Sequence Design section enables generation of diverse protein sequences compatible with a given structural backbone. Users can input their desired backbone coordinates and obtain novel sequences optimized for that scaffold.</p>"
          }
        },
        "7f45b74269744c05ba7a99af9423c01b": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "3fd98403ec1a402a8fec39c898bc3134": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "579c36f72f5e48b7a1319f34b150dc6d": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ButtonView",
            "button_style": "info",
            "description": "Obtain protein-level embeddings",
            "disabled": false,
            "icon": "",
            "layout": "IPY_MODEL_1ad70eee4d07494bb9d14d378b75aafd",
            "style": "IPY_MODEL_7fa1247efd5a4cd1af121a241dcf56cf",
            "tooltip": ""
          }
        },
        "1ad70eee4d07494bb9d14d378b75aafd": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": "30px",
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "7fa1247efd5a4cd1af121a241dcf56cf": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "button_color": null,
            "font_weight": ""
          }
        },
        "4381953e5b964201b48c3e0f07c6a7a8": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_74583fff751a4c0daa90522e3e23319d",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_f6ee8c85fccd4f998fada4bde46598b8",
            "value": "<p>The Protein Embedding section enables extraction of sequence or structure embeddings using either standard Saprot models or custom fine-tuned models for specialized analysis tasks.</p>"
          }
        },
        "74583fff751a4c0daa90522e3e23319d": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "f6ee8c85fccd4f998fada4bde46598b8": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "033205f8a7714146b4b750323bf9a3e2": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_107447b6cb0e4e8685008bdc5445e6c0",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_93ed434d4e4248e9888c12916376514e",
            "value": "<h2>Protein sequence design</h2>"
          }
        },
        "107447b6cb0e4e8685008bdc5445e6c0": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "93ed434d4e4248e9888c12916376514e": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "54822490622c4729b0d93f9bf9d8f536": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_78124bcbe7ac4464aca0e97785d389ea",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_f7ef258e9cf540939f611f10ae48d5d2",
            "value": "<h3>Please upload the structure backbone:</h3>"
          }
        },
        "78124bcbe7ac4464aca0e97785d389ea": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "f7ef258e9cf540939f611f10ae48d5d2": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "43a6c8d9b31345fb815e8e2d04d142a7": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_e6c963ef9e7b446aaa8889d7a252143a",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_ed9665034f78414fb70918210e013070",
            "value": "<p>Chain (to be extracted from the structure):</p>"
          }
        },
        "e6c963ef9e7b446aaa8889d7a252143a": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "ed9665034f78414fb70918210e013070": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "e5a2715f8f514d3c9f7eb18bc4ca198f": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "TextModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "TextModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "TextView",
            "continuous_update": true,
            "description": "",
            "description_tooltip": null,
            "disabled": false,
            "layout": "IPY_MODEL_f3f62515c2e243b59f7f82a07c8b307d",
            "placeholder": "Enter the name of chain here",
            "style": "IPY_MODEL_3086f99a43fd4867aa06fc1985adf9cd",
            "value": "A"
          }
        },
        "f3f62515c2e243b59f7f82a07c8b307d": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": "30px",
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "3086f99a43fd4867aa06fc1985adf9cd": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "745169f17570408dbea231bfd45888a1": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_0c4f36c100ef49c8b99bb92f51417803",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_9fab2e2685d4443799ab51bed7057ca0",
            "value": "<p>Upload the protein structure:</p>"
          }
        },
        "0c4f36c100ef49c8b99bb92f51417803": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "9fab2e2685d4443799ab51bed7057ca0": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "e481bd8d7a844e74acc75e6a7e2d0ecb": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "RadioButtonsModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "RadioButtonsModel",
            "_options_labels": [
              "Yes",
              "No"
            ],
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "RadioButtonsView",
            "description": "Is the file size > 5MB?",
            "description_tooltip": null,
            "disabled": false,
            "index": 1,
            "layout": "IPY_MODEL_16417102ff5d41c1a83e61c56f425de9",
            "style": "IPY_MODEL_fabede3b814e4e47a2b9199c1ae0458c"
          }
        },
        "16417102ff5d41c1a83e61c56f425de9": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "max-content"
          }
        },
        "fabede3b814e4e47a2b9199c1ae0458c": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": "initial"
          }
        },
        "842be5fc77934a7d930230ec77c072a7": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_5dde927371f549b2a56d0944ec07ff2d",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_4ca8b09e41644f05a7f1ff178b138418",
            "value": "<p><a href='https://github.com/westlake-repl/SaprotHub/wiki/SaprotHub-v2-%28latest%29#get-your-file-from-google-drive' target='blank'>Quick tutorial</a></p>"
          }
        },
        "5dde927371f549b2a56d0944ec07ff2d": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": "none",
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "4ca8b09e41644f05a7f1ff178b138418": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "e26629d88f764328bd324abc05bac619": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "TextModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "TextModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "TextView",
            "continuous_update": true,
            "description": "File path:",
            "description_tooltip": null,
            "disabled": false,
            "layout": "IPY_MODEL_0318445a8dd14013abb3e8c2270eabbe",
            "placeholder": "Upload your file to Google Drive and copy its path into this box",
            "style": "IPY_MODEL_aff7949f8ce14342981f650d3e6c5327",
            "value": ""
          }
        },
        "0318445a8dd14013abb3e8c2270eabbe": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": "none",
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": "30px",
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "aff7949f8ce14342981f650d3e6c5327": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "c4af9f968ff14e70a3c436b4acf8cf84": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ButtonView",
            "button_style": "info",
            "description": "Upload your file",
            "disabled": false,
            "icon": "",
            "layout": "IPY_MODEL_39234706a8b145c08bd4f3070c2511e6",
            "style": "IPY_MODEL_3596f01850f349ce88e8be87abdc18cb",
            "tooltip": ""
          }
        },
        "39234706a8b145c08bd4f3070c2511e6": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": "30px",
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "3596f01850f349ce88e8be87abdc18cb": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "button_color": null,
            "font_weight": ""
          }
        },
        "75b99b20537944aeb1deea199c756e41": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_3ccc6beee0cd40c98001062e97634d4c",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_3212c28719a24c50a77be0a886681d4c",
            "value": "\n      <input type=\"file\" id=\"files-c74dcf53-1627-4983-af3f-65d88250f630\" name=\"files[]\"  disabled\n        style=\"border:none\" /><br><font color=red>Note: If you upload the same file, the upload will be ignored.</font></br>\n      <output id=\"result-c74dcf53-1627-4983-af3f-65d88250f630\">\n      Upload widget is only available when the cell has been executed in the\n      current browser session. Please rerun this cell to enable.\n      </output>\n      <script>// Copyright 2017 Google LLC\n//\n// Licensed under the Apache License, Version 2.0 (the \"License\");\n// you may not use this file except in compliance with the License.\n// You may obtain a copy of the License at\n//\n//      http://www.apache.org/licenses/LICENSE-2.0\n//\n// Unless required by applicable law or agreed to in writing, software\n// distributed under the License is distributed on an \"AS IS\" BASIS,\n// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n// See the License for the specific language governing permissions and\n// limitations under the License.\n\n/**\n * @fileoverview Helpers for google.colab Python module.\n */\n(function(scope) {\nfunction span(text, styleAttributes = {}) {\n  const element = document.createElement('span');\n  element.textContent = text;\n  for (const key of Object.keys(styleAttributes)) {\n    element.style[key] = styleAttributes[key];\n  }\n  return element;\n}\n\n// Max number of bytes which will be uploaded at a time.\nconst MAX_PAYLOAD_SIZE = 100 * 1024;\n\nfunction _uploadFiles(inputId, outputId) {\n  const steps = uploadFilesStep(inputId, outputId);\n  const outputElement = document.getElementById(outputId);\n  // Cache steps on the outputElement to make it available for the next call\n  // to uploadFilesContinue from Python.\n  outputElement.steps = steps;\n\n  return _uploadFilesContinue(outputId);\n}\n\n// This is roughly an async generator (not supported in the browser yet),\n// where there are multiple asynchronous steps and the Python side is going\n// to poll for completion of each step.\n// This uses a Promise to block the python side on completion of each step,\n// then passes the result of the previous step as the input to the next step.\nfunction _uploadFilesContinue(outputId) {\n  const outputElement = document.getElementById(outputId);\n  const steps = outputElement.steps;\n\n  const next = steps.next(outputElement.lastPromiseValue);\n  return Promise.resolve(next.value.promise).then((value) => {\n    // Cache the last promise value to make it available to the next\n    // step of the generator.\n    outputElement.lastPromiseValue = value;\n    return next.value.response;\n  });\n}\n\n/**\n * Generator function which is called between each async step of the upload\n * process.\n * @param {string} inputId Element ID of the input file picker element.\n * @param {string} outputId Element ID of the output display.\n * @return {!Iterable<!Object>} Iterable of next steps.\n */\nfunction* uploadFilesStep(inputId, outputId) {\n  const inputElement = document.getElementById(inputId);\n  inputElement.disabled = false;\n\n  const outputElement = document.getElementById(outputId);\n  outputElement.innerHTML = '';\n\n  const pickedPromise = new Promise((resolve) => {\n    inputElement.addEventListener('change', (e) => {\n      resolve(e.target.files);\n    });\n  });\n\n  const cancel = document.createElement('button');\n  inputElement.parentElement.appendChild(cancel);\n  cancel.textContent = 'Cancel upload';\n  const cancelPromise = new Promise((resolve) => {\n    cancel.onclick = () => {\n      resolve(null);\n    };\n  });\n\n  // Wait for the user to pick the files.\n  const files = yield {\n    promise: Promise.race([pickedPromise, cancelPromise]),\n    response: {\n      action: 'starting',\n    }\n  };\n\n  cancel.remove();\n\n  // Disable the input element since further picks are not allowed.\n  inputElement.disabled = true;\n\n  if (!files) {\n    return {\n      response: {\n        action: 'complete',\n      }\n    };\n  }\n\n  for (const file of files) {\n    const li = document.createElement('li');\n    li.append(span(file.name, {fontWeight: 'bold'}));\n    li.append(span(\n        `(${file.type || 'n/a'}) - ${file.size} bytes, ` +\n        `last modified: ${\n            file.lastModifiedDate ? file.lastModifiedDate.toLocaleDateString() :\n                                    'n/a'} - `));\n    const percent = span('0% done');\n    li.appendChild(percent);\n\n    outputElement.appendChild(li);\n\n    const fileDataPromise = new Promise((resolve) => {\n      const reader = new FileReader();\n      reader.onload = (e) => {\n        resolve(e.target.result);\n      };\n      reader.readAsArrayBuffer(file);\n    });\n    // Wait for the data to be ready.\n    let fileData = yield {\n      promise: fileDataPromise,\n      response: {\n        action: 'continue',\n      }\n    };\n\n    // Use a chunked sending to avoid message size limits. See b/62115660.\n    let position = 0;\n    do {\n      const length = Math.min(fileData.byteLength - position, MAX_PAYLOAD_SIZE);\n      const chunk = new Uint8Array(fileData, position, length);\n      position += length;\n\n      const base64 = btoa(String.fromCharCode.apply(null, chunk));\n      yield {\n        response: {\n          action: 'append',\n          file: file.name,\n          data: base64,\n        },\n      };\n\n      let percentDone = fileData.byteLength === 0 ?\n          100 :\n          Math.round((position / fileData.byteLength) * 100);\n      percent.textContent = `${percentDone}% done`;\n\n    } while (position < fileData.byteLength);\n  }\n\n  // All done.\n  yield {\n    response: {\n      action: 'complete',\n    }\n  };\n}\n\nscope.google = scope.google || {};\nscope.google.colab = scope.google.colab || {};\nscope.google.colab._files = {\n  _uploadFiles,\n  _uploadFilesContinue,\n};\n})(self);\n</script> "
          }
        },
        "3ccc6beee0cd40c98001062e97634d4c": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": "none",
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "3212c28719a24c50a77be0a886681d4c": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "0999c826dff54b2e8767ee4f85c7b58e": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ButtonView",
            "button_style": "info",
            "description": "Submit",
            "disabled": false,
            "icon": "",
            "layout": "IPY_MODEL_cd5978e89963496284182842414ba373",
            "style": "IPY_MODEL_865069f1a237426ea1beb1b37867db4b",
            "tooltip": ""
          }
        },
        "cd5978e89963496284182842414ba373": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": "30px",
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "865069f1a237426ea1beb1b37867db4b": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "button_color": null,
            "font_weight": ""
          }
        },
        "c7a6d65400324c5ea6ce76d80ddaf921": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_9c606953d00c44d0b1067f37eafcdae6",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_0cee1d8fb7ee44dd8305c34d24772beb",
            "value": "<h2>Protein sequence design</h2>"
          }
        },
        "9c606953d00c44d0b1067f37eafcdae6": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "0cee1d8fb7ee44dd8305c34d24772beb": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "700fe055b07d466ca49bb28f45d1dd8c": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_1ecfcf1451f542f4a2426cf600c5405d",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_83ef374662834a9facb743377207acac",
            "value": "<h3>Please upload the structure backbone:</h3>"
          }
        },
        "1ecfcf1451f542f4a2426cf600c5405d": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "83ef374662834a9facb743377207acac": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "3ecc0304993a4badac618fc1f072f045": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_a72202a60c9648f3889546e5c5fb85e5",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_425e2e46cf4948749afee64d51772e35",
            "value": "<p>Chain (to be extracted from the structure):</p>"
          }
        },
        "a72202a60c9648f3889546e5c5fb85e5": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "425e2e46cf4948749afee64d51772e35": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "b8bd64125e7c4082bfa7992d56c1f745": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "TextModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "TextModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "TextView",
            "continuous_update": true,
            "description": "",
            "description_tooltip": null,
            "disabled": false,
            "layout": "IPY_MODEL_42142b648d25435592c2cd412abae2c8",
            "placeholder": "Enter the name of chain here",
            "style": "IPY_MODEL_fafe0a31a1b24e6b81f4dbf808ef882f",
            "value": "A"
          }
        },
        "42142b648d25435592c2cd412abae2c8": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": "30px",
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "fafe0a31a1b24e6b81f4dbf808ef882f": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "0bde97d05de44b5190444541ad0ec38a": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_685e62f8fe1943ff82ff1f0c642f57b6",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_0527f077b3914d84b3225016150abc48",
            "value": "<p>Upload the protein structure:</p>"
          }
        },
        "685e62f8fe1943ff82ff1f0c642f57b6": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "0527f077b3914d84b3225016150abc48": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "eb6e64c5b0b34d8081c88e032aaf6fdc": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "RadioButtonsModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "RadioButtonsModel",
            "_options_labels": [
              "Yes",
              "No"
            ],
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "RadioButtonsView",
            "description": "Is the file size > 5MB?",
            "description_tooltip": null,
            "disabled": false,
            "index": 1,
            "layout": "IPY_MODEL_a54569f2806a4f3daf0ec81078787519",
            "style": "IPY_MODEL_8d4889f9613b48d4a41616a9bc99add5"
          }
        },
        "a54569f2806a4f3daf0ec81078787519": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "max-content"
          }
        },
        "8d4889f9613b48d4a41616a9bc99add5": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": "initial"
          }
        },
        "c2c9fec6578346478b6244b3a6db2176": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_3d82d2dd59c142b5b351df4f715cb565",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_6bbe3cf2f3d7437b99b3882309d5c344",
            "value": "<p><a href='https://github.com/westlake-repl/SaprotHub/wiki/SaprotHub-v2-%28latest%29#get-your-file-from-google-drive' target='blank'>Quick tutorial</a></p>"
          }
        },
        "3d82d2dd59c142b5b351df4f715cb565": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": "none",
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "6bbe3cf2f3d7437b99b3882309d5c344": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "e556d9a6341b4b959bc1267167860e5f": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "TextModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "TextModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "TextView",
            "continuous_update": true,
            "description": "File path:",
            "description_tooltip": null,
            "disabled": false,
            "layout": "IPY_MODEL_3c37c9fc693a4cab88d5f346ad686463",
            "placeholder": "Upload your file to Google Drive and copy its path into this box",
            "style": "IPY_MODEL_a54de0c45e6e47418945f198f3fc0500",
            "value": ""
          }
        },
        "3c37c9fc693a4cab88d5f346ad686463": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": "none",
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": "30px",
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "a54de0c45e6e47418945f198f3fc0500": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "53393a656e7f4450ae08541b972b4d75": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ButtonView",
            "button_style": "info",
            "description": "001-chai1.pdb",
            "disabled": false,
            "icon": "",
            "layout": "IPY_MODEL_d4a09839fa964cc4ab0a63dfb798db70",
            "style": "IPY_MODEL_0d718fcee69d49f9a4d6953603c8e912",
            "tooltip": ""
          }
        },
        "d4a09839fa964cc4ab0a63dfb798db70": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": "30px",
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "0d718fcee69d49f9a4d6953603c8e912": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "button_color": null,
            "font_weight": ""
          }
        },
        "572888144abf4345ae5b0f41f0457acc": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_7032ff2e022746e39d9389e28c67b5e5",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_915f0e154d304c7d8cb6f2ae183c355e",
            "value": "\n      <input type=\"file\" id=\"files-7c75b2e4-9bd8-4539-b4fd-ac9789867c03\" name=\"files[]\"  disabled\n        style=\"border:none\" /><br><font color=red>Note: If you upload the same file, the upload will be ignored.</font></br>\n      <output id=\"result-7c75b2e4-9bd8-4539-b4fd-ac9789867c03\">\n      Upload widget is only available when the cell has been executed in the\n      current browser session. Please rerun this cell to enable.\n      </output>\n      <script>// Copyright 2017 Google LLC\n//\n// Licensed under the Apache License, Version 2.0 (the \"License\");\n// you may not use this file except in compliance with the License.\n// You may obtain a copy of the License at\n//\n//      http://www.apache.org/licenses/LICENSE-2.0\n//\n// Unless required by applicable law or agreed to in writing, software\n// distributed under the License is distributed on an \"AS IS\" BASIS,\n// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\n// See the License for the specific language governing permissions and\n// limitations under the License.\n\n/**\n * @fileoverview Helpers for google.colab Python module.\n */\n(function(scope) {\nfunction span(text, styleAttributes = {}) {\n  const element = document.createElement('span');\n  element.textContent = text;\n  for (const key of Object.keys(styleAttributes)) {\n    element.style[key] = styleAttributes[key];\n  }\n  return element;\n}\n\n// Max number of bytes which will be uploaded at a time.\nconst MAX_PAYLOAD_SIZE = 100 * 1024;\n\nfunction _uploadFiles(inputId, outputId) {\n  const steps = uploadFilesStep(inputId, outputId);\n  const outputElement = document.getElementById(outputId);\n  // Cache steps on the outputElement to make it available for the next call\n  // to uploadFilesContinue from Python.\n  outputElement.steps = steps;\n\n  return _uploadFilesContinue(outputId);\n}\n\n// This is roughly an async generator (not supported in the browser yet),\n// where there are multiple asynchronous steps and the Python side is going\n// to poll for completion of each step.\n// This uses a Promise to block the python side on completion of each step,\n// then passes the result of the previous step as the input to the next step.\nfunction _uploadFilesContinue(outputId) {\n  const outputElement = document.getElementById(outputId);\n  const steps = outputElement.steps;\n\n  const next = steps.next(outputElement.lastPromiseValue);\n  return Promise.resolve(next.value.promise).then((value) => {\n    // Cache the last promise value to make it available to the next\n    // step of the generator.\n    outputElement.lastPromiseValue = value;\n    return next.value.response;\n  });\n}\n\n/**\n * Generator function which is called between each async step of the upload\n * process.\n * @param {string} inputId Element ID of the input file picker element.\n * @param {string} outputId Element ID of the output display.\n * @return {!Iterable<!Object>} Iterable of next steps.\n */\nfunction* uploadFilesStep(inputId, outputId) {\n  const inputElement = document.getElementById(inputId);\n  inputElement.disabled = false;\n\n  const outputElement = document.getElementById(outputId);\n  outputElement.innerHTML = '';\n\n  const pickedPromise = new Promise((resolve) => {\n    inputElement.addEventListener('change', (e) => {\n      resolve(e.target.files);\n    });\n  });\n\n  const cancel = document.createElement('button');\n  inputElement.parentElement.appendChild(cancel);\n  cancel.textContent = 'Cancel upload';\n  const cancelPromise = new Promise((resolve) => {\n    cancel.onclick = () => {\n      resolve(null);\n    };\n  });\n\n  // Wait for the user to pick the files.\n  const files = yield {\n    promise: Promise.race([pickedPromise, cancelPromise]),\n    response: {\n      action: 'starting',\n    }\n  };\n\n  cancel.remove();\n\n  // Disable the input element since further picks are not allowed.\n  inputElement.disabled = true;\n\n  if (!files) {\n    return {\n      response: {\n        action: 'complete',\n      }\n    };\n  }\n\n  for (const file of files) {\n    const li = document.createElement('li');\n    li.append(span(file.name, {fontWeight: 'bold'}));\n    li.append(span(\n        `(${file.type || 'n/a'}) - ${file.size} bytes, ` +\n        `last modified: ${\n            file.lastModifiedDate ? file.lastModifiedDate.toLocaleDateString() :\n                                    'n/a'} - `));\n    const percent = span('0% done');\n    li.appendChild(percent);\n\n    outputElement.appendChild(li);\n\n    const fileDataPromise = new Promise((resolve) => {\n      const reader = new FileReader();\n      reader.onload = (e) => {\n        resolve(e.target.result);\n      };\n      reader.readAsArrayBuffer(file);\n    });\n    // Wait for the data to be ready.\n    let fileData = yield {\n      promise: fileDataPromise,\n      response: {\n        action: 'continue',\n      }\n    };\n\n    // Use a chunked sending to avoid message size limits. See b/62115660.\n    let position = 0;\n    do {\n      const length = Math.min(fileData.byteLength - position, MAX_PAYLOAD_SIZE);\n      const chunk = new Uint8Array(fileData, position, length);\n      position += length;\n\n      const base64 = btoa(String.fromCharCode.apply(null, chunk));\n      yield {\n        response: {\n          action: 'append',\n          file: file.name,\n          data: base64,\n        },\n      };\n\n      let percentDone = fileData.byteLength === 0 ?\n          100 :\n          Math.round((position / fileData.byteLength) * 100);\n      percent.textContent = `${percentDone}% done`;\n\n    } while (position < fileData.byteLength);\n  }\n\n  // All done.\n  yield {\n    response: {\n      action: 'complete',\n    }\n  };\n}\n\nscope.google = scope.google || {};\nscope.google.colab = scope.google.colab || {};\nscope.google.colab._files = {\n  _uploadFiles,\n  _uploadFilesContinue,\n};\n})(self);\n</script> "
          }
        },
        "7032ff2e022746e39d9389e28c67b5e5": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": "none",
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "915f0e154d304c7d8cb6f2ae183c355e": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "ac45630d0bb74b3694e9fe9231cc96eb": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ButtonView",
            "button_style": "info",
            "description": "Submit",
            "disabled": false,
            "icon": "",
            "layout": "IPY_MODEL_8f7c63d564804352a86173123e9c567d",
            "style": "IPY_MODEL_7f3eb6775b0b4049852cf45bc3c325fa",
            "tooltip": ""
          }
        },
        "8f7c63d564804352a86173123e9c567d": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": "30px",
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "7f3eb6775b0b4049852cf45bc3c325fa": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "button_color": null,
            "font_weight": ""
          }
        },
        "b157ba138a314ff48d8247dc1895b970": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_d22367a764864b198755c00beb4e8a9b",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_2cacff63343949c1ac6412a465eb6fb7",
            "value": "<p><strong>Foldseek sequence of chain A (\"#\" means low pLDDT positions that are masked):</strong></p>\n<p>dvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvlvvlqpdwdqdpvrdtdhpvvvvvvvvvlvpdpdpvvsvvvvvvvvvvllvvqvvvlvvlvpddfpdddslqvsfaappfaagradnpdaaeeeeedqcfppdvqvlqqlvcqqsshhryayeyehaaqqlccqwprhgddhqlvsqvvqlprdshakykyndanqddpvncvvvvhqayedandadcgpvdggscpvcrhvnrythgdtdpqddpvnvvvvcqqvvlvvvlvvvvvvpvvddqdlvvvvvvvlvvlvvllvvvvvcvvpvppddddslvsllvsqvvvccsnlhvvlvvvcvvvppddsvvvsvvvvvvsvvcsvpddaddpvnvsvvsnvvsvvddsdddpvpmdmdgrdddrdvvvvvvpdvvvvvvncvrrpdppvcpvvsvvvvvvsvvsvvvvvvd</p>\n<p><strong>Backbone visualization of 1742372195.4881818.pdb (chains are displayed with different colors):</strong></p>"
          }
        },
        "d22367a764864b198755c00beb4e8a9b": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "2cacff63343949c1ac6412a465eb6fb7": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "95dcb311094f44fdba62a5253d83b0bf": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DropdownModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DropdownModel",
            "_options_labels": [
              "argmax",
              "multinomial"
            ],
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "DropdownView",
            "description": "Sampling methodology:",
            "description_tooltip": null,
            "disabled": false,
            "index": 0,
            "layout": "IPY_MODEL_9ae785c8c8304b12a3926c9a7e506adb",
            "style": "IPY_MODEL_7e20bb2da89e4753b57e2a9678d8a9a3"
          }
        },
        "9ae785c8c8304b12a3926c9a7e506adb": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": "30px",
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "7e20bb2da89e4753b57e2a9678d8a9a3": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": "initial"
          }
        },
        "0a0feface3a4407686eef7e3a90be88e": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "BoundedIntTextModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "BoundedIntTextModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "IntTextView",
            "continuous_update": false,
            "description": "Sampling num:",
            "description_tooltip": null,
            "disabled": false,
            "layout": "IPY_MODEL_25cc738bc6ca4e9cb817e4d4633a3fa5",
            "max": 100,
            "min": 1,
            "step": 1,
            "style": "IPY_MODEL_1a4c9ca971b24e6084038b1d718c6de0",
            "value": 1
          }
        },
        "25cc738bc6ca4e9cb817e4d4633a3fa5": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": "none",
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": "30px",
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "1a4c9ca971b24e6084038b1d718c6de0": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": "initial"
          }
        },
        "0a753a8a60614f65bb440c80727d76f9": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_230d4790e9cb4b058926eaf064405ea1",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_31dae71c0ae04187ad1802e8df826f87",
            "value": "<p><em>For each position, Saprot will predict the probability of every amino acid occuring at this position. </em><strong>argmax</strong><em> means the model will choose the amino acid with highest probability as its prediction. </em><strong>multinomial</strong><em> means the model will sample amino acids based on the probability distribution.</em></p>"
          }
        },
        "230d4790e9cb4b058926eaf064405ea1": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "31dae71c0ae04187ad1802e8df826f87": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "cd907850ca024b20971715368f018664": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_e68866bbfa3e4d2eaeec08ad4c9c8afa",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_0bbcc91ab0ab4a9fb91c5b996e0c1829",
            "value": "<h3>---------------------------------------------------------------------------</h3>"
          }
        },
        "e68866bbfa3e4d2eaeec08ad4c9c8afa": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "0bbcc91ab0ab4a9fb91c5b996e0c1829": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "30fe7b0e433a4bf19dc8bda8c5660cde": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ButtonView",
            "button_style": "info",
            "description": "Predict protein sequence",
            "disabled": false,
            "icon": "",
            "layout": "IPY_MODEL_835135854bde4df58361f55bdfee936d",
            "style": "IPY_MODEL_b879a1097e8747dc90e2fd4873fe4a61",
            "tooltip": ""
          }
        },
        "835135854bde4df58361f55bdfee936d": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": "30px",
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "b879a1097e8747dc90e2fd4873fe4a61": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "button_color": null,
            "font_weight": ""
          }
        },
        "18f9708b2aee49a18045725bed317cff": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_e13621fc126142378f91e6865624b4d2",
              "IPY_MODEL_f055a639b2c7476c9a9c160f48d3cd24",
              "IPY_MODEL_ec9dee9766004a01ba65a23bc867342d"
            ],
            "layout": "IPY_MODEL_bfe0d6e243c745c7b4a91d024fd4d1f1"
          }
        },
        "e13621fc126142378f91e6865624b4d2": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_14290117e08945b2995351621d915666",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_41c2e69509d743cc8b6b33c5a5b916a6",
            "value": "tokenizer_config.json:‚Äá100%"
          }
        },
        "f055a639b2c7476c9a9c160f48d3cd24": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_021de2c3ab284151aa226fcee9596873",
            "max": 135,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_cfcd072b06434b0281391d86555d834f",
            "value": 135
          }
        },
        "ec9dee9766004a01ba65a23bc867342d": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_04e1fc6fc23f475fa6ecdc3210225928",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_3516f2d47c8e4aa6aa1173e56c776589",
            "value": "‚Äá135/135‚Äá[00:00&lt;00:00,‚Äá9.67kB/s]"
          }
        },
        "bfe0d6e243c745c7b4a91d024fd4d1f1": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "14290117e08945b2995351621d915666": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "41c2e69509d743cc8b6b33c5a5b916a6": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "021de2c3ab284151aa226fcee9596873": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "cfcd072b06434b0281391d86555d834f": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "04e1fc6fc23f475fa6ecdc3210225928": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "3516f2d47c8e4aa6aa1173e56c776589": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "448afc138e0a4888905e6d4b780a55dd": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_f89bc13783314b379a6ce2b3a4fe2f43",
              "IPY_MODEL_663b48cc54484b6b8eee3ebc7e0c30d3",
              "IPY_MODEL_14315ef2e22b468d8eb538c77662cb47"
            ],
            "layout": "IPY_MODEL_0b0cba32e3694402bc9036abab46993e"
          }
        },
        "f89bc13783314b379a6ce2b3a4fe2f43": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_f9134f53502d4adfb65692b0422bc101",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_5f30f4b7a5a348468ec91651bec24e88",
            "value": "vocab.txt:‚Äá100%"
          }
        },
        "663b48cc54484b6b8eee3ebc7e0c30d3": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_5ba9fc68af354bbf896746fa8cb1ebf0",
            "max": 1353,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_0c5af4b1b42040eebdaab9ad9d14ce19",
            "value": 1353
          }
        },
        "14315ef2e22b468d8eb538c77662cb47": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_66f274d78e444de193e952c37087a51c",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_ac397aa11a1149f79d48735027d704eb",
            "value": "‚Äá1.35k/1.35k‚Äá[00:00&lt;00:00,‚Äá123kB/s]"
          }
        },
        "0b0cba32e3694402bc9036abab46993e": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "f9134f53502d4adfb65692b0422bc101": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "5f30f4b7a5a348468ec91651bec24e88": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "5ba9fc68af354bbf896746fa8cb1ebf0": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "0c5af4b1b42040eebdaab9ad9d14ce19": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "66f274d78e444de193e952c37087a51c": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "ac397aa11a1149f79d48735027d704eb": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "9a8173f1fe3f4125a509823737170499": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_4dde9740edb5472083597c01d496d88b",
              "IPY_MODEL_b884f523af2f47f18f69da35baed22e8",
              "IPY_MODEL_1617bd4931114311b1e0f5eb93887582"
            ],
            "layout": "IPY_MODEL_20d4faece52248b58dda81eaa21b6fc0"
          }
        },
        "4dde9740edb5472083597c01d496d88b": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_690daf1037fc44e7bb2081bb4b925e83",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_d822bdc17f904f5aac71464412167942",
            "value": "special_tokens_map.json:‚Äá100%"
          }
        },
        "b884f523af2f47f18f69da35baed22e8": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_b793a44d6d4749fcac9240417c3cb87f",
            "max": 125,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_b53d5879bde54f3cbb6b59dea6ac46cf",
            "value": 125
          }
        },
        "1617bd4931114311b1e0f5eb93887582": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_a53fe7a3ab984f36836ca6b1947f6c52",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_1fd5878a9c454e11a6b5517b66a36403",
            "value": "‚Äá125/125‚Äá[00:00&lt;00:00,‚Äá9.18kB/s]"
          }
        },
        "20d4faece52248b58dda81eaa21b6fc0": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "690daf1037fc44e7bb2081bb4b925e83": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "d822bdc17f904f5aac71464412167942": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "b793a44d6d4749fcac9240417c3cb87f": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "b53d5879bde54f3cbb6b59dea6ac46cf": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "a53fe7a3ab984f36836ca6b1947f6c52": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "1fd5878a9c454e11a6b5517b66a36403": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "03962db06c784275bc2e16a81c253b41": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_c942691f13934b7f86a82d7daad16097",
              "IPY_MODEL_d9cb38272f76435e989dd432a26e68a2",
              "IPY_MODEL_5e060167622c4f3ea4c29a0626d6c0e4"
            ],
            "layout": "IPY_MODEL_6d7d6979ffe74f468b7eb6b3c3aafafb"
          }
        },
        "c942691f13934b7f86a82d7daad16097": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_e987f8de9db644b2b2226a5163a2fa0d",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_1341d23d8a5e49aa85595ebec1b72220",
            "value": "config.json:‚Äá100%"
          }
        },
        "d9cb38272f76435e989dd432a26e68a2": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_f6dc9ae6b23e4ada9c3414915e647b17",
            "max": 778,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_e9aa8bbc4b8d4589ba653cc69237d70b",
            "value": 778
          }
        },
        "5e060167622c4f3ea4c29a0626d6c0e4": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_3b4323756ca64e0f89afc0fc8c61418e",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_0ddd93e7bab947ba94864af87b6362c3",
            "value": "‚Äá778/778‚Äá[00:00&lt;00:00,‚Äá70.5kB/s]"
          }
        },
        "6d7d6979ffe74f468b7eb6b3c3aafafb": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "e987f8de9db644b2b2226a5163a2fa0d": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "1341d23d8a5e49aa85595ebec1b72220": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "f6dc9ae6b23e4ada9c3414915e647b17": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "e9aa8bbc4b8d4589ba653cc69237d70b": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "3b4323756ca64e0f89afc0fc8c61418e": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "0ddd93e7bab947ba94864af87b6362c3": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "05201e9c8a6544deaeed712140073e81": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HBoxModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HBoxModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HBoxView",
            "box_style": "",
            "children": [
              "IPY_MODEL_e91d225d33cb4d3eb8bb8cf34d8749aa",
              "IPY_MODEL_571d2b84906d4a87adfe77f1449ffbcb",
              "IPY_MODEL_b155c5def25b46728fe8f03050f72f2b"
            ],
            "layout": "IPY_MODEL_ee34ea4748ae4d3d89320b31d6662f79"
          }
        },
        "e91d225d33cb4d3eb8bb8cf34d8749aa": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_7ce642e79f574c7ebc79cb52050eb31c",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_1ceb03de8458467d8da1231757f982b3",
            "value": "pytorch_model.bin:‚Äá100%"
          }
        },
        "571d2b84906d4a87adfe77f1449ffbcb": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "FloatProgressModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "FloatProgressModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ProgressView",
            "bar_style": "success",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_55138d9d4e584e90b60c687f2848c430",
            "max": 2606485237,
            "min": 0,
            "orientation": "horizontal",
            "style": "IPY_MODEL_8f54f69eb0264944879f003de7da4b43",
            "value": 2606485237
          }
        },
        "b155c5def25b46728fe8f03050f72f2b": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_5ad7202b8dff481d80be9d1d0686588e",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_c54c486c9f8d4151aae04cf93d09c6f5",
            "value": "‚Äá2.61G/2.61G‚Äá[00:23&lt;00:00,‚Äá144MB/s]"
          }
        },
        "ee34ea4748ae4d3d89320b31d6662f79": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "7ce642e79f574c7ebc79cb52050eb31c": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "1ceb03de8458467d8da1231757f982b3": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "55138d9d4e584e90b60c687f2848c430": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "8f54f69eb0264944879f003de7da4b43": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ProgressStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ProgressStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "bar_color": null,
            "description_width": ""
          }
        },
        "5ad7202b8dff481d80be9d1d0686588e": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "c54c486c9f8d4151aae04cf93d09c6f5": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "ab859c29b67641249ed3a391d6a3f607": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_8403502d130f4e9abf7eba0091f48091",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_08c185d7c7204904b9d9ef3a285be90f",
            "value": "<p><strong>Predicted sequences are listed below:</strong></p>\n<ol>\n<li>LEELLELLEELEEELEELEEELEELEEELEELLEELESLTITLPDGEEVSLEELLELLEELLSIEDEEERLELLEELLEEFLELIEEEREELLSLPPGEVPPELRKFIPPGYKGKPPDPDKPVRVYCDGIFDLFHSGHMRALRQAKNLFPNTYVIVGVCNDEDTHRFKGKTVMNEWERAESLLHCRWVDEVIIGAPWVTTLELLDRFGIDYVAHDDIPTSSADGDDVYAPLKKAGRFVATQRTEGVSTSDIITRIIRNYYKYLRRNFLRELEENEIDPEELLEELKELLKLLLELLEELEENPEGFKLDEEELKKELLEELKEKLLGPLLEKLSKLLPGLPRELLKKLLEKLIEELLENLPPPSLEELLELLKELAKKLPLEVDPSKWIRLKKGPLIDLEELLKGDEEYLEELLELLGVPEEELEEILELLEELLELLEELKKL</li>\n</ol>\n<p>You can click the button to dowanload the predictions.</p>"
          }
        },
        "8403502d130f4e9abf7eba0091f48091": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "08c185d7c7204904b9d9ef3a285be90f": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "4902a6111f87440db2145b576fc75a8e": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ButtonView",
            "button_style": "success",
            "description": "Download File",
            "disabled": false,
            "icon": "",
            "layout": "IPY_MODEL_fc066b6a54fc4acdbff9e4ed9bdb43a4",
            "style": "IPY_MODEL_b02e285f17c6494da4582accc2e945bc",
            "tooltip": ""
          }
        },
        "fc066b6a54fc4acdbff9e4ed9bdb43a4": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "b02e285f17c6494da4582accc2e945bc": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "button_color": null,
            "font_weight": ""
          }
        },
        "c99ee45b34174aabab065748325949fc": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "HTMLModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "HTMLModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "HTMLView",
            "description": "",
            "description_tooltip": null,
            "layout": "IPY_MODEL_fd244b32346142cca415edf38d2056c6",
            "placeholder": "‚Äã",
            "style": "IPY_MODEL_cdeb7cbf4af74aa793149643a26e3403",
            "value": "<h3>You could use ESMFold to evaluate the quality of the predicted sequence</h3>\n<h4><font color=red>Warning: If you are using Google T4 GPU, it will run out of memory!</font></h4>"
          }
        },
        "fd244b32346142cca415edf38d2056c6": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": null,
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": null
          }
        },
        "cdeb7cbf4af74aa793149643a26e3403": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": ""
          }
        },
        "94ecdefcd9594129b025c2a87e05d83b": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "TextModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "TextModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "TextView",
            "continuous_update": true,
            "description": "",
            "description_tooltip": null,
            "disabled": false,
            "layout": "IPY_MODEL_47359149efa2418e81cc4d23307b1414",
            "placeholder": "Copy the predicted sequence and paste it here",
            "style": "IPY_MODEL_3ed70a3857a24b39b71ba5a32dcdcb90",
            "value": ""
          }
        },
        "47359149efa2418e81cc4d23307b1414": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": "30px",
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "3ed70a3857a24b39b71ba5a32dcdcb90": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "DescriptionStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "DescriptionStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "description_width": "initial"
          }
        },
        "d626ed4d4b604fc4a7019e321ce91a22": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonModel",
          "model_module_version": "1.5.0",
          "state": {
            "_dom_classes": [],
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/controls",
            "_view_module_version": "1.5.0",
            "_view_name": "ButtonView",
            "button_style": "info",
            "description": "Predict structure",
            "disabled": false,
            "icon": "",
            "layout": "IPY_MODEL_5ca5b1edd3284635b61bf9023433b924",
            "style": "IPY_MODEL_d0d00befdc3946b4ac72d08259c5eec2",
            "tooltip": ""
          }
        },
        "5ca5b1edd3284635b61bf9023433b924": {
          "model_module": "@jupyter-widgets/base",
          "model_name": "LayoutModel",
          "model_module_version": "1.2.0",
          "state": {
            "_model_module": "@jupyter-widgets/base",
            "_model_module_version": "1.2.0",
            "_model_name": "LayoutModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "LayoutView",
            "align_content": null,
            "align_items": null,
            "align_self": null,
            "border": null,
            "bottom": null,
            "display": null,
            "flex": null,
            "flex_flow": null,
            "grid_area": null,
            "grid_auto_columns": null,
            "grid_auto_flow": null,
            "grid_auto_rows": null,
            "grid_column": null,
            "grid_gap": null,
            "grid_row": null,
            "grid_template_areas": null,
            "grid_template_columns": null,
            "grid_template_rows": null,
            "height": "30px",
            "justify_content": null,
            "justify_items": null,
            "left": null,
            "margin": null,
            "max_height": null,
            "max_width": null,
            "min_height": null,
            "min_width": null,
            "object_fit": null,
            "object_position": null,
            "order": null,
            "overflow": null,
            "overflow_x": null,
            "overflow_y": null,
            "padding": null,
            "right": null,
            "top": null,
            "visibility": null,
            "width": "500px"
          }
        },
        "d0d00befdc3946b4ac72d08259c5eec2": {
          "model_module": "@jupyter-widgets/controls",
          "model_name": "ButtonStyleModel",
          "model_module_version": "1.5.0",
          "state": {
            "_model_module": "@jupyter-widgets/controls",
            "_model_module_version": "1.5.0",
            "_model_name": "ButtonStyleModel",
            "_view_count": null,
            "_view_module": "@jupyter-widgets/base",
            "_view_module_version": "1.2.0",
            "_view_name": "StyleView",
            "button_color": null,
            "font_weight": ""
          }
        }
      }
    }
  },
  "nbformat": 4,
  "nbformat_minor": 5
}